--- xulaye\scripts\lk#xula.baf	Tue Jan 15 13:04:24 2013
+++ C:\BWP Patchstudio\patched files\xulaye\scripts\lk#xula.baf	Tue Jun 30 21:32:57 2015
@@ -246,6 +246,7 @@
     !See([ENEMY])
     !PartyHasItem("lk#xarm")
     Global("LK#XulQuestReminder","GLOBAL",0)
+    GlobalLT("LK#XULQUEST","GLOBAL",7)
 THEN
   RESPONSE #100
     SetGlobal("LK#XulQuestReminder","GLOBAL",1)
@@ -260,6 +261,7 @@
     !See([ENEMY])
     !PartyHasItem("lk#xarm")
     Global("LK#XulQuestReminder","GLOBAL",1)
+    GlobalLT("LK#XULQUEST","GLOBAL",7)
 THEN
   RESPONSE #100
     MoveToObject(Player1)
@@ -276,6 +278,7 @@
     !See([ENEMY])                       
     !PartyHasItem("lk#xarm")
     Global("LK#XulQuestReminder","GLOBAL",2)
+    GlobalLT("LK#XULQUEST","GLOBAL",7)
 THEN
   RESPONSE #100
     IncrementGlobal("LK#XulQuestReminder","GLOBAL",1)
@@ -290,6 +293,7 @@
     !See([ENEMY])
     !PartyHasItem("lk#xarm")
     Global("LK#XulQuestReminder","GLOBAL",3)
+    GlobalLT("LK#XULQUEST","GLOBAL",7)
 THEN
   RESPONSE #100
     MoveToObject(Player1)
