--- Rolles\DLG\rolles.d	Sat Sep 13 11:04:20 2003
+++ C:\BWP Patchstudio\patched files\Rolles\DLG\rolles.d	Sat Apr 04 02:00:40 2015
@@ -387,28 +387,28 @@
 
 IF ~~ THEN BEGIN s#swfree
     SAY @147 IF ~~ THEN REPLY @013 GOTO Upgrade
-    IF ~PartyHasItem("sw1h26") PartyHasItem("sw1h27") PartyHasItem("sw1h35")
+    IF ~PartyHasItem("sw1h26") PartyHasItem("sw1h27") PartyHasItem("sw1h30")
 PartyHasItem("sw1h36") NumItemsPartyGT("scrl1q",1) NumItemsPartyGT("potn14",1) PartyHasItem("potn45")
         PartyGoldGT(14999)~
       THEN REPLY @148
       DO ~TakePartyGold(15000)      DestroyGold(15000)
           TakePartyItemNum("sw1h26",1)   DestroyItem("sw1h26")
           TakePartyItemNum("sw1h27",1)   DestroyItem("sw1h27")
-          TakePartyItemNum("sw1h35",1)   DestroyItem("sw1h35")
+          TakePartyItemNum("sw1h30",1)   DestroyItem("sw1h30")
           TakePartyItemNum("sw1h36",1)   DestroyItem("sw1h36")
           TakePartyItemNum("scrl1q",2)   DestroyItem("scrl1q")
           TakePartyItemNum("potn14",2)   DestroyItem("potn14")
           TakePartyItemNum("potn45",1)   DestroyItem("potn45")
           GiveItemCreate("s#sw1h02",Player1,1,1,1)
           CreateVisualEffect("spcrtwpn",[365.420])~ EXIT
-    IF ~PartyHasItem("sw1h26") PartyHasItem("sw1h27") PartyHasItem("sw1h35")
+    IF ~PartyHasItem("sw1h26") PartyHasItem("sw1h27") PartyHasItem("sw1h30")
 PartyHasItem("sw1h36") NumItemsPartyGT("scrl1q",1) NumItemsPartyGT("potn14",1) PartyHasItem("potn45")
         PartyGoldGT(14999)~
       THEN REPLY @149
       DO ~TakePartyGold(15000)      DestroyGold(15000)
           TakePartyItemNum("sw1h26",1)   DestroyItem("sw1h26")
           TakePartyItemNum("sw1h27",1)   DestroyItem("sw1h27")
-          TakePartyItemNum("sw1h35",1)   DestroyItem("sw1h35")
+          TakePartyItemNum("sw1h30",1)   DestroyItem("sw1h30")
           TakePartyItemNum("sw1h36",1)   DestroyItem("sw1h36")
           TakePartyItemNum("scrl1q",2)   DestroyItem("scrl1q")
           TakePartyItemNum("potn14",2)   DestroyItem("potn14")
