--- rr/setup-rr.tp2	2017-05-01 14:40:56.000000000 +0200
+++ Fixpack/rr/setup-rr.tp2	2017-06-22 16:22:24.987000100 +0200
@@ -10,6 +10,17 @@
 VERSION ~v4.91~
 
 ALWAYS
+  ACTION_IF GAME_IS ~eet~ BEGIN
+    OUTER_SET bg2_chapter = 12
+  END ELSE BEGIN
+    OUTER_SET bg2_chapter = 0
+  END
+  OUTER_FOR (i=1; i<=10; i=i+1) BEGIN
+    OUTER_SET bg2_chapter = bg2_chapter + 1
+    OUTER_SPRINT name_source ~bg2_chapter_%i%~
+    OUTER_SET EVAL ~%name_source%~ = bg2_chapter
+  END
+
   INCLUDE ~rr/lib/rr#afix.tph~                                        // Include the Rogue Rebalancing Assorted Fixes macro
   INCLUDE ~rr/lib/rr#bg1v.tpa~                                        // Include the BG1 variables macro (used for proper Tutu and BGT compatibility)
   INCLUDE ~rr/lib/rr#sp2in.tph~                                       // Include my custom "Spell to Innate" function
@@ -3770,7 +3781,7 @@
 COPY_EXISTING ~BALDUR.BCS~ override                                        // delay Malchor Harpell's appearance until the CoC encounter is over, also delay quests etc
   DECOMPILE_AND_PATCH BEGIN
     REPLACE_TEXTUALLY EXACT_MATCH ~!Exists("c6harp")~ ~!Exists("c6harp") !AreaCheck("RR#001") !AreaCheck("AR0800")~
-    REPLACE_TEXTUALLY CASE_INSENSITIVE EVALUATE_REGEXP ~\(^ +!Global("Chapter","GLOBAL",4)[ %LNL%]+!Global("Chapter","GLOBAL",5)[ %LNL%]+!Global("Chapter","GLOBAL",7)\)~ ~\1 !AreaCheck("RR#001")~
+    REPLACE_TEXTUALLY CASE_INSENSITIVE EVALUATE_REGEXP ~\(^ +!Global("Chapter","GLOBAL",%bg2_chapter_4%)[ %LNL%]+!Global("Chapter","GLOBAL",%bg2_chapter_5%)[ %LNL%]+!Global("Chapter","GLOBAL",%bg2_chapter_7%)\)~ ~\1 !AreaCheck("RR#001")~
   END
 UNLESS ~16510 0 1 0 0 "RR#001" "" OB~
 BUT_ONLY
@@ -3789,7 +3800,7 @@
               minsc.bcs   override
               yoshimo.bcs override
   DECOMPILE_AND_PATCH BEGIN
-    REPLACE_TEXTUALLY CASE_INSENSITIVE EVALUATE_REGEXP ~^\( +!Global("Chapter","GLOBAL",4)[ %LNL%]+!Global("Chapter","GLOBAL",5)[ %LNL%]+!Global("Chapter","GLOBAL",7)\)~ ~\1 !AreaCheck("RR#001")~
+    REPLACE_TEXTUALLY CASE_INSENSITIVE EVALUATE_REGEXP ~^\( +!Global("Chapter","GLOBAL",%bg2_chapter_4%)[ %LNL%]+!Global("Chapter","GLOBAL",%bg2_chapter_5%)[ %LNL%]+!Global("Chapter","GLOBAL",%bg2_chapter_7%)\)~ ~\1 !AreaCheck("RR#001")~
   END
 UNLESS ~16510 0 1 0 0 "RR#001" "" OB~
 BUT_ONLY
