--- Setup-Solaufein.tp2	Sat Aug 15 22:54:34 2009
+++ C:\BWP Patchstudio\patched files\Setup-Solaufein.tp2	Mon Oct 14 22:51:25 2013
@@ -9,7 +9,7 @@
 BACKUP ~solarom/backup~ // where should we put backup files? 
 // where should tell the user to send email if something goes wrong? 
 AUTHOR "weimer@cs.berkeley.edu (http://www.weidu.org)"
-
+VERSION "v104 BWP Fix"
 // The ALLOW_MISSING declaration is optional. If present, it gives a 
 // list of files that are allowed to be "not found" during the 
 // installation. 
