--- TomeAndBlood/data/components/setup_spelltweaks.tpa	2016-02-19 18:22:19.070478000 -0500
+++ Fixpack/TomeAndBlood/data/components/setup_spelltweaks.tpa	2016-12-07 18:56:43.784200600 -0500
@@ -479,7 +479,7 @@
 	END
 
 //Blindness
-	ACTION_IF FILE_EXISTS_IN_GAME ~spwi106.spl~ THEN BEGIN
+	ACTION_IF (FILE_EXISTS_IN_GAME ~spwi106.spl~) AND (NOT GAME_IS ~iwdee~) THEN BEGIN
 		COPY_EXISTING ~spwi106.spl~ ~override~
 		ADD_SPELL ~override/spwi106.spl~ 2 2 WIZARD_BLINDNESS_TNB
 			WRITE_LONG 0x34 2
@@ -522,8 +522,8 @@
 					PATCH_IF (class == 1 || class == 13 || class == 14 || class == 7 || class == 10 || class == 17 || class == 19 || class == 5) BEGIN // wizard casters
 						REMOVE_MEMORIZED_SPELL ~spwi106~
 						REMOVE_KNOWN_SPELL ~spwi106~
-						ADD_KNOWN_SPELL ~%spell_res%~ #0 ~wizard~
-						ADD_MEMORIZED_SPELL ~%spell_res%~ #0 ~wizard~
+//						ADD_KNOWN_SPELL ~%spell_res%~ #1 ~wizard~
+//						ADD_MEMORIZED_SPELL ~%spell_res%~ #1 ~wizard~
 					END
 				END
 			END
@@ -531,7 +531,7 @@
 	END
 
 //Deafness
-	ACTION_IF FILE_EXISTS_IN_GAME ~spwi223.spl~ THEN BEGIN
+	ACTION_IF (FILE_EXISTS_IN_GAME ~spwi223.spl~) AND (NOT GAME_IS ~iwdee~) THEN BEGIN
 		COPY_EXISTING ~spwi223.spl~ ~override~
 		ADD_SPELL ~override/spwi223.spl~ 2 1 WIZARD_DEAFNESS_TNB
 			WRITE_LONG 0x34 1
@@ -724,8 +724,8 @@
 					PATCH_IF (class == 1 || class == 13 || class == 14 || class == 7 || class == 10 || class == 17 || class == 19 || class == 5) BEGIN // wizard casters
 						REMOVE_MEMORIZED_SPELL ~spwi722~
 						REMOVE_KNOWN_SPELL ~spwi722~
-						ADD_KNOWN_SPELL ~%spell_res%~ #0 ~wizard~
-						ADD_MEMORIZED_SPELL ~%spell_res%~ #0 ~wizard~
+//						ADD_KNOWN_SPELL ~%spell_res%~ #7 ~wizard~
+//						ADD_MEMORIZED_SPELL ~%spell_res%~ #7 ~wizard~
 					END
 				END
 			END
@@ -748,6 +748,7 @@
 		ACTION_IF FILE_EXISTS_IN_GAME ~hidespl.2da~ BEGIN
 			APPEND ~hidespl.2da~ ~spwi811	1		0~
 		END
+		ACTION_IF FILE_EXISTS_IN_GAME ~scrl9f.itm~ THEN BEGIN
 		LAF RES_NUM_OF_SPELL_NAME
 			STR_VAR spell_name = ~WIZARD_SYMBOL_FEAR_TNB~
 			RET spell_res
@@ -776,8 +777,8 @@
 					PATCH_IF (class == 1 || class == 13 || class == 14 || class == 7 || class == 10 || class == 17 || class == 19 || class == 5) BEGIN // wizard casters
 						REMOVE_MEMORIZED_SPELL ~spwi811~
 						REMOVE_KNOWN_SPELL ~spwi811~
-						ADD_KNOWN_SPELL ~%spell_res%~ #0 ~wizard~
-						ADD_MEMORIZED_SPELL ~%spell_res%~ #0 ~wizard~
+//						ADD_KNOWN_SPELL ~%spell_res%~ #6 ~wizard~
+//						ADD_MEMORIZED_SPELL ~%spell_res%~ #6 ~wizard~
 					END
 				END
 			END
@@ -828,8 +829,8 @@
 					PATCH_IF (class == 1 || class == 13 || class == 14 || class == 7 || class == 10 || class == 17 || class == 19 || class == 5) BEGIN // wizard casters
 						REMOVE_MEMORIZED_SPELL ~spwi811~
 						REMOVE_KNOWN_SPELL ~spwi811~
-						ADD_KNOWN_SPELL ~%spell_res%~ #0 ~wizard~
-						ADD_MEMORIZED_SPELL ~%spell_res%~ #0 ~wizard~
+//						ADD_KNOWN_SPELL ~%spell_res%~ #7 ~wizard~
+//						ADD_MEMORIZED_SPELL ~%spell_res%~ #7 ~wizard~
 					END
 				END
 			END
@@ -866,8 +867,8 @@
 					PATCH_IF (class == 1 || class == 13 || class == 14 || class == 7 || class == 10 || class == 17 || class == 19 || class == 5) BEGIN // wizard casters
 						REMOVE_MEMORIZED_SPELL ~spwi915~
 						REMOVE_KNOWN_SPELL ~spwi915~
-						ADD_KNOWN_SPELL ~%spell_res%~ #0 ~wizard~
-						ADD_MEMORIZED_SPELL ~%spell_res%~ #0 ~wizard~
+//						ADD_KNOWN_SPELL ~%spell_res%~ #7 ~wizard~
+//						ADD_MEMORIZED_SPELL ~%spell_res%~ #7 ~wizard~
 					END
 				END
 			END
@@ -1420,8 +1421,8 @@
 					PATCH_IF (class == 1 || class == 13 || class == 14 || class == 7 || class == 10 || class == 17 || class == 19 || class == 5) BEGIN // wizard casters
 						REMOVE_MEMORIZED_SPELL ~spwi102~
 						REMOVE_KNOWN_SPELL ~spwi102~
-						ADD_KNOWN_SPELL ~%spell_res%~ #0 ~wizard~
-						ADD_MEMORIZED_SPELL ~%spell_res%~ #0 ~wizard~
+//						ADD_KNOWN_SPELL ~%spell_res%~ #1 ~wizard~
+//						ADD_MEMORIZED_SPELL ~%spell_res%~ #1 ~wizard~
 					END
 				END
 			END
@@ -1524,8 +1525,8 @@
 					PATCH_IF (class == 1 || class == 13 || class == 14 || class == 7 || class == 10 || class == 17 || class == 19 || class == 5) BEGIN // wizard casters
 						REMOVE_MEMORIZED_SPELL ~spwi120~
 						REMOVE_KNOWN_SPELL ~spwi120~
-						ADD_KNOWN_SPELL ~%spell_res%~ #0 ~wizard~
-						ADD_MEMORIZED_SPELL ~%spell_res%~ #0 ~wizard~
+//						ADD_KNOWN_SPELL ~%spell_res%~ #1 ~wizard~
+//						ADD_MEMORIZED_SPELL ~%spell_res%~ #1 ~wizard~
 					END
 				END
 			END
@@ -1578,8 +1579,8 @@
 					PATCH_IF (class == 1 || class == 13 || class == 14 || class == 7 || class == 10 || class == 17 || class == 19 || class == 5) BEGIN // wizard casters
 						REMOVE_MEMORIZED_SPELL ~spwi212~
 						REMOVE_KNOWN_SPELL ~spwi212~
-						ADD_KNOWN_SPELL ~%spell_res%~ #0 ~wizard~
-						ADD_MEMORIZED_SPELL ~%spell_res%~ #0 ~wizard~
+//						ADD_KNOWN_SPELL ~%spell_res%~ #2 ~wizard~
+//						ADD_MEMORIZED_SPELL ~%spell_res%~ #2 ~wizard~
 					END
 				END
 			END
@@ -1616,8 +1617,8 @@
 					PATCH_IF (class == 1 || class == 13 || class == 14 || class == 7 || class == 10 || class == 17 || class == 19 || class == 5) BEGIN // wizard casters
 						REMOVE_MEMORIZED_SPELL ~spwi209~
 						REMOVE_KNOWN_SPELL ~spwi209~
-						ADD_KNOWN_SPELL ~%spell_res%~ #0 ~wizard~
-						ADD_MEMORIZED_SPELL ~%spell_res%~ #0 ~wizard~
+//						ADD_KNOWN_SPELL ~%spell_res%~ #0 ~wizard~
+//						ADD_MEMORIZED_SPELL ~%spell_res%~ #0 ~wizard~
 					END
 				END
 			END
@@ -1725,8 +1726,8 @@
 					PATCH_IF (class == 1 || class == 13 || class == 14 || class == 7 || class == 10 || class == 17 || class == 19 || class == 5) BEGIN // wizard casters
 						REMOVE_MEMORIZED_SPELL ~spwi811~
 						REMOVE_KNOWN_SPELL ~spwi811~
-						ADD_KNOWN_SPELL ~%spell_res%~ #0 ~wizard~
-						ADD_MEMORIZED_SPELL ~%spell_res%~ #0 ~wizard~
+//						ADD_KNOWN_SPELL ~%spell_res%~ #6 ~wizard~
+//						ADD_MEMORIZED_SPELL ~%spell_res%~ #6 ~wizard~
 					END
 				END
 			END
@@ -1777,8 +1778,8 @@
 					PATCH_IF (class == 1 || class == 13 || class == 14 || class == 7 || class == 10 || class == 17 || class == 19 || class == 5) BEGIN // wizard casters
 						REMOVE_MEMORIZED_SPELL ~spwi916~
 						REMOVE_KNOWN_SPELL ~spwi916~
-						ADD_KNOWN_SPELL ~%spell_res%~ #0 ~wizard~
-						ADD_MEMORIZED_SPELL ~%spell_res%~ #0 ~wizard~
+//						ADD_KNOWN_SPELL ~%spell_res%~ #7 ~wizard~
+//						ADD_MEMORIZED_SPELL ~%spell_res%~ #7 ~wizard~
 					END
 				END
 			END
@@ -1815,8 +1816,8 @@
 					PATCH_IF (class == 1 || class == 13 || class == 14 || class == 7 || class == 10 || class == 17 || class == 19 || class == 5) BEGIN // wizard casters
 						REMOVE_MEMORIZED_SPELL ~spwi915~
 						REMOVE_KNOWN_SPELL ~spwi915~
-						ADD_KNOWN_SPELL ~%spell_res%~ #0 ~wizard~
-						ADD_MEMORIZED_SPELL ~%spell_res%~ #0 ~wizard~
+//						ADD_KNOWN_SPELL ~%spell_res%~ #7 ~wizard~
+//						ADD_MEMORIZED_SPELL ~%spell_res%~ #7 ~wizard~
 					END
 				END
 			END
@@ -1855,8 +1856,8 @@
 					PATCH_IF (class == 1 || class == 13 || class == 14 || class == 7 || class == 10 || class == 17 || class == 19 || class == 5) BEGIN // wizard casters
 						REMOVE_MEMORIZED_SPELL ~spwi914~
 						REMOVE_KNOWN_SPELL ~spwi914~
-						ADD_KNOWN_SPELL ~%spell_res%~ #0 ~wizard~
-						ADD_MEMORIZED_SPELL ~%spell_res%~ #0 ~wizard~
+//						ADD_KNOWN_SPELL ~%spell_res%~ #7 ~wizard~
+//						ADD_MEMORIZED_SPELL ~%spell_res%~ #7 ~wizard~
 					END
 				END
 			END
@@ -2019,5 +2020,6 @@
 		END
 	END
 END 
+END
  
 CLEAR_ARRAYS 
