--- setup-foodmod.tp2	Sun Feb 13 04:04:46 2011
+++ C:\BWP Patchstudio\patched files\setup-foodmod.tp2	Wed Oct 02 09:39:45 2013
@@ -1,7 +1,7 @@
 BACKUP ~foodmod\backup~
 AUTHOR ~belladonna@chosenofmystra.net~
 
-VERSION ~v1.03~
+VERSION ~v1.03 BWP Fix~
 
 AUTO_TRA ~foodmod\%s~
 
@@ -287,7 +287,7 @@
 
 EXTEND_BOTTOM ~ar1200.bcs~ ~foodmod/bafs/cml1200.baf~
 
-EXTEND_BOTTOM ~ar1300.bcs~ ~foodmod/bafs/cml1300.baf~
+EXTEND_BOTTOM ~ar3000.bcs~ ~foodmod/bafs/cml1300.baf~
 
 EXTEND_BOTTOM ~ar2010.bcs~ ~foodmod/bafs/cml2010.baf~
 
