--- Setup-Grimuars.tp2	Sat Aug 29 03:32:18 2009
+++ C:\BWP Patchstudio\patched files\Setup-Grimuars.tp2	Sat Oct 05 12:04:45 2013
@@ -1,5 +1,6 @@
 BACKUP ~grimuar/Backup~
 AUTHOR ~Rughalt (rughalt@gmail.com)~
+VERSION ~v3.3 BWP Patching version~
 AUTO_TRA ~Grimuar/Translation/%s~
 LANGUAGE ~English~
          ~english~
@@ -20,53 +21,77 @@
 BEGIN @22
 
 
-COPY ~grimuar/creatures/notactics/shadra01.cre~ ~Override/shadra01.cre~
-COPY ~grimuar/creatures/notactics/pmaster.cre~ ~Override/Pmaster.cre~
-COPY ~grimuar/creatures/notactics/hlshang.cre~ ~Override/hlshang.cre~
-COPY ~grimuar/creatures/notactics/Hlkoshi.cre~ ~Override/Hlkoshi.cre~
-COPY ~grimuar/creatures/notactics/hldemi.cre~ ~Override/hldemi.cre~
-COPY ~grimuar/creatures/notactics/gaal01.cre~ ~Override/gaal01.cre~
-COPY ~grimuar/creatures/notactics/gaal02.cre~ ~Override/gaal02.cre~
-COPY ~grimuar/creatures/notactics/torgal.cre~ ~Override/torgal.cre~
-COPY ~grimuar/creatures/notactics/draug.cre~ ~Override/draug.cre~
-COPY ~grimuar/creatures/notactics/trrak01.cre~ ~Override/trrak01.cre~
-COPY ~grimuar/items/book08.itm~ ~Override/book08.itm~
-COPY ~grimuar/items/book07.itm~ ~Override/book07.itm~
-COPY ~grimuar/items/r#grim01.itm~ ~Override/r#grim01.itm~
-SAY NAME1 #7150
-  SAY NAME2 @3
-  SAY UNIDENTIFIED_DESC #17278
-  SAY DESC @2
-  SAY 0x10e @23
-COPY ~grimuar/items/r#grim02.itm~ ~Override/r#grim02.itm~
-SAY NAME1 #7150
-  SAY NAME2 @4
-  SAY UNIDENTIFIED_DESC #17278
-  SAY DESC @1
-  SAY 0x10e @24
-
-COPY ~grimuar/items/r#grim03.itm~ ~Override/r#grim03.itm~
-SAY NAME1 #7150
-  SAY NAME2 @5
-  SAY UNIDENTIFIED_DESC #17278
-  SAY DESC @6
-  SAY 0x10e @25
-
-
-BEGIN @26
-
-COPY ~grimuar/creatures/withtactics/shadra01.cre~ ~Override/shadra01.cre~
-COPY ~grimuar/creatures/withtactics/pmaster.cre~ ~Override/Pmaster.cre~
-COPY ~grimuar/creatures/withtactics/hlshang.cre~ ~Override/hlshang.cre~
-COPY ~grimuar/creatures/withtactics/Hlkoshi.cre~ ~Override/Hlkoshi.cre~
-COPY ~grimuar/creatures/withtactics/hldemi.cre~ ~Override/hldemi.cre~
-COPY ~grimuar/creatures/notactics/gaal01.cre~ ~Override/gaal01.cre~
-COPY ~grimuar/creatures/notactics/gaal02.cre~ ~Override/gaal02.cre~
-COPY ~grimuar/creatures/withtactics/torgal.cre~ ~Override/torgal.cre~
-COPY ~grimuar/creatures/withtactics/draug.cre~ ~Override/draug.cre~
-COPY ~grimuar/creatures/withtactics/trrak01.cre~ ~Override/trrak01.cre~
-COPY ~grimuar/items/book08.itm~ ~Override/book08.itm~
-COPY ~grimuar/items/book07.itm~ ~Override/book07.itm~
+COPY_EXISTING ~shadra01.cre~	~override/shadra01.cre~
+  PATCH_IF (SOURCE_SIZE > 0x2d3) BEGIN
+    ADD_CRE_ITEM ~book03~ #0 #20 #20 ~NONE~ ~INV~
+  END
+BUT_ONLY_IF_IT_CHANGES
+
+COPY_EXISTING ~Pmaster.cre~		~override/Pmaster.cre~
+  PATCH_IF (SOURCE_SIZE > 0x2d3) BEGIN
+    ADD_CRE_ITEM ~book04~ #0 #20 #20 ~NONE~ ~INV~  
+  END
+BUT_ONLY_IF_IT_CHANGES
+
+COPY_EXISTING ~hlshang.cre~		~override/hlshang.cre~
+  PATCH_IF (SOURCE_SIZE > 0x2d3) BEGIN
+    ADD_CRE_ITEM ~book06~ #0 #20 #20 ~NONE~ ~INV~
+  END
+BUT_ONLY_IF_IT_CHANGES
+
+COPY_EXISTING ~Hlkoshi.cre~		~override/Hlkoshi.cre~
+  PATCH_IF (SOURCE_SIZE > 0x2d3) BEGIN
+    ADD_CRE_ITEM ~book05~ #0 #20 #20 ~NONE~ ~INV~
+  END
+BUT_ONLY_IF_IT_CHANGES
+
+COPY_EXISTING ~hldemi.cre~		~override/hldemi.cre~
+  PATCH_IF (SOURCE_SIZE > 0x2d3) BEGIN
+    ADD_CRE_ITEM ~book08~ #0 #20 #20 ~NONE~ ~INV~
+  END
+BUT_ONLY_IF_IT_CHANGES
+
+COPY_EXISTING ~gaal01.cre~		~override/gaal01.cre~
+  PATCH_IF (SOURCE_SIZE > 0x2d3) BEGIN
+    ADD_CRE_ITEM ~book07~ #0 #20 #20 ~NONE~ ~INV~
+  END
+BUT_ONLY_IF_IT_CHANGES
+
+COPY_EXISTING ~gaal02.cre~		~override/gaal02.cre~
+  PATCH_IF (SOURCE_SIZE > 0x2d3) BEGIN
+    ADD_CRE_ITEM ~book07~ #0 #20 #20 ~NONE~ ~INV~
+  END
+BUT_ONLY_IF_IT_CHANGES
+
+COPY_EXISTING ~torgal.cre~		~override/torgal.cre~
+  PATCH_IF (SOURCE_SIZE > 0x2d3) BEGIN
+    ADD_CRE_ITEM ~r#grim02~ #0 #20 #20 ~NONE~ ~INV~
+  END
+BUT_ONLY_IF_IT_CHANGES
+
+COPY_EXISTING ~draug.cre~		~override/draug.cre~
+  PATCH_IF (SOURCE_SIZE > 0x2d3) BEGIN
+    ADD_CRE_ITEM ~r#grim01~ #0 #20 #20 ~NONE~ ~INV~
+  END
+BUT_ONLY_IF_IT_CHANGES
+
+COPY_EXISTING ~trrak01.cre~	~override/trrak01.cre~
+  PATCH_IF (SOURCE_SIZE > 0x2d3) BEGIN
+    ADD_CRE_ITEM ~r#grim03~ #0 #20 #20 ~NONE~ ~INV~
+  END
+BUT_ONLY_IF_IT_CHANGES
+
+COPY_EXISTING ~book08.itm~ ~override/book08.itm~
+  PATCH_IF (SOURCE_SIZE > 0x71) THEN BEGIN 
+    WRITE_SHORT 0x1c 0x0b
+  END
+BUT_ONLY_IF_IT_CHANGES
+COPY_EXISTING ~book07.itm~ ~override/book07.itm~
+  PATCH_IF (SOURCE_SIZE > 0x71) THEN BEGIN 
+    WRITE_SHORT 0x1c 0x0b
+  END
+BUT_ONLY_IF_IT_CHANGES
+
 COPY ~grimuar/items/r#grim01.itm~ ~Override/r#grim01.itm~
 SAY NAME1 #7150
   SAY NAME2 @3
