--- Eilistraee/SCRIPTS/AR0334.baf	2010-10-09 18:46:36.000000000 -0400
+++ Fixpack/Eilistraee/SCRIPTS/AR0334.baf	2016-11-25 15:21:05.154483000 -0500
@@ -1,29 +1,32 @@
 IF
-Global("Dozen","GLOBAL",1)
  Global("ForgeStuff","GLOBAL",1)
+ Global("EsxCromDozen","GLOBAL",1)
 THEN
  RESPONSE #100
-     SetGlobal("Dozen","GLOBAL",0)
-     TakePartyItem("ESxTOOT")
-     TakePartyItem("DWDust")
-     TakePartyItem("MISC27")
+     SetGlobal("EsxCromDozen","GLOBAL",0)
+     TakePartyItemNum("ESxTOOT",1) // lich tooth
+     TakePartyItemNum("DWDust",1)  // adamantine dust
+     TakePartyItemNum("MISC27",1)  // moonstone gem
      DestroyItem("ESxTOOT")
-     DestroyItem("AdamentineDust")
+     DestroyItem("DWDUST")
+     DestroyItem("MISC27")
      GiveItemCreate("ESxDAG",PLAYER1,0,0,0) // completed item
      ActionOverride("wsmith01",StartDialogueNoSet([PC]))
 END
 
 IF
-Global("13thMoon","GLOBAL",1)
  Global("ForgeStuff","GLOBAL",1)
+ Global("EsxCrom13thMoon","GLOBAL",1)
 THEN
  RESPONSE #100
-     SetGlobal("13thMoon","GLOBAL",0)
-     TakePartyItem("ESxTOOT")
-     TakePartyItem("DWDust")
-     TakePartyItem("MISC27")
-     TakePartyItem("MISC8U")
+     SetGlobal("EsxCrom13thMoon","GLOBAL",0)
+     TakePartyItemNum("ESxTOOT",1) // lich tooth
+     TakePartyItemNum("DWDust",1)  // adamantine dust
+     TakePartyItemNum("MISC27",1)  // moonstone gem
+     TakePartyItemNum("MISC8U",1)  // silver sword
      DestroyItem("ESxTOOT")
+     DestroyItem("DWDUST")
+     DestroyItem("MISC27")
      DestroyItem("MISC8U")
      GiveItemCreate("ESxDAG2",PLAYER1,0,0,0) // completed item
      ActionOverride("wsmith01",StartDialogueNoSet([PC]))
