--- pale/fl#add_kit_ee.tpa	2013-11-12 19:51:00.000000000 -0500
+++ Fixpack/pale/fl#add_kit_ee.tpa	2016-11-27 13:02:53.548667400 -0500
@@ -1,10 +1,11 @@
 /*
+ * fl#add_kit_ee - An ADD_KIT supplement for EE-type games
  * Author: Fredrik Lindgren, a.k.a., Wisp
  * Licence: public domain
- * Date: 2013-11-11
+ * Date: 2016-04-17
  */
 
-OUTER_SET fl#add_kit_ee#version = 1
+OUTER_SPRINT fl#add_kit_ee#version "1.1.3"
 
 DEFINE_ACTION_FUNCTION fl#add_kit_ee#get_row
   STR_VAR
@@ -115,19 +116,27 @@
     clasiskl = ""
     clasthac = ""
     thiefscl = ""
+    sneakatt = ""
+    crippstr = ""
   RET
     backstab
     thiefskl
     traplimt
+    sneakatt
+    crippstr
 BEGIN
   ACTION_IF kit_class != 4 AND ("%backstab%" STR_CMP "" OR
                                 "%thiefskl%" STR_CMP "" OR
-                                "%traplimt%" STR_CMP "")
+                                "%traplimt%" STR_CMP "" OR
+                                "%sneakatt%" STR_CMP "" OR
+                                "%crippstr%" STR_CMP "")
   BEGIN
-    WARN ~WARNING: the arguments backstab, thiefskl and traplimt should only be provided for thief kits~
+    WARN ~WARNING: the arguments for thief-only 2DAs should only be provided for thief kits~
     OUTER_SPRINT backstab ""
     OUTER_SPRINT thiefskl ""
     OUTER_SPRINT traplimt ""
+    OUTER_SPRINT sneakatt ""
+    OUTER_SPRINT crippstr ""
   END
 END
 
@@ -136,6 +145,7 @@
     row = ""
     column = ""
     req_column_count = 2
+    value = ""
   RET
     value
 BEGIN
@@ -155,15 +165,50 @@
       END
     END
   END
-  PATCH_IF !VARIABLE_IS_SET value BEGIN
-    PATCH_FAIL "ERROR: fl#add_kit_ee#get_2da_value could not find row %row% and column %column% with %req_column_count% required columns"
+  PATCH_IF "%value%" STRING_EQUAL "" BEGIN
+    PATCH_FAIL "ERROR: fl#add_kit_ee failed to retrieve a value for column %column% and row %row% in %SOURCE_FILE%"
+  END
+END
+
+DEFINE_PATCH_MACRO fl#add_kit_ee#validate_kitlist_values BEGIN
+  PATCH_IF IS_AN_INT kit_number AND kit_number < 0 BEGIN
+    PATCH_FAIL ~ERROR: fl#add_kit_ee could not find the kit %kit_name% in kitlist.2da~
+  END
+  PATCH_IF !IS_AN_INT kit_number BEGIN
+    PATCH_FAIL ~ERROR: fl#add_kit_ee could not read a valid kit number for kit %kit_name% from kitlist.2da~
+  END
+  PATCH_IF !IS_AN_INT kit_lower BEGIN
+    PATCH_FAIL ~ERROR: fl#add_kit_ee could not read a valid lower strref for kit %kit_name% from kitlist.2da~
+  END
+  PATCH_IF !IS_AN_INT kit_mixed BEGIN
+    PATCH_FAIL ~ERROR: fl#add_kit_ee could not read a valid mixed strref for kit %kit_name% from kitlist.2da~
+  END
+  PATCH_IF !IS_AN_INT kit_desc BEGIN
+    PATCH_FAIL ~ERROR: fl#add_kit_ee could not read a valid help strref for kit %kit_name% from kitlist.2da~
+  END
+  PATCH_IF !IS_AN_INT kit_class BEGIN
+    PATCH_FAIL ~ERROR: fl#add_kit_ee could not read a valid class for kit %kit_name% from kitlist.2da~
+  END
+END
+
+DEFINE_PATCH_MACRO fl#add_kit_ee#validate_kitids_value BEGIN
+  PATCH_IF !"%number%" STRING_MATCHES_REGEXP "0x\([0-9]+\)" = 0 BEGIN
+    PATCH_FAIL ~ERROR: fl#add_kit_ee could not read a valid kit number for %kit_name% from kit.ids~
+  END
+END
+
+DEFINE_PATCH_MACRO fl#add_kit_ee#validate_clastext_value BEGIN
+  PATCH_IF !IS_AN_INT value BEGIN
+    PATCH_FAIL ~ERROR: fl#add_kit_ee could not read a valid value for column %column% for class %class% from clastext.2da~
   END
 END
 
 DEFINE_ACTION_FUNCTION fl#add_kit_ee
   INT_VAR
     biography = "-2"
+    fallen = 0
     briefdesc = "-2"
+    fallen_notice = "-2"
   STR_VAR
     kit_name = ""
     backstab = ""
@@ -177,15 +222,14 @@
     clasiskl = ""
     clasthac = ""
     thiefscl = ""
+    sneakatt = ""
+    crippstr = ""
 BEGIN
+  ACTION_IF FILE_EXISTS_IN_GAME monkfist.2da BEGIN // "Rule" file as-of-yet unique to EE games
   PRINT "Doing EE-type kit extensions for kit %kit_name%"
-  ACTION_IF "%kit_name%" STRING_EQUAL "" OR
-            NOT FILE_CONTAINS_EVALUATED (kitlist.2da "%kit_name%")
-  BEGIN
-    FAIL ~The variable kit_name is required to name a valid kit~
-  END
 
-  ACTION_IF GAME_IS ~bgee bg2ee~ BEGIN
+    OUTER_SET kit_number = "-1"
+
     COPY_EXISTING kitlist.2da override
       READ_2DA_ENTRIES_NOW kitlist 9
       FOR (i = 0; i < kitlist; ++i) BEGIN
@@ -198,11 +242,9 @@
           READ_2DA_ENTRY_FORMER kitlist i 8 kit_class
         END
       END
+      LPM fl#add_kit_ee#validate_kitlist_values
     BUT_ONLY
 
-    ACTION_IF !VARIABLE_IS_SET kit_number BEGIN
-      FAIL ~The kit %kit_name% was not found in kitlist.2da~
-    END
     LAF fl#add_kit_ee#validate
       INT_VAR
         kit_number
@@ -220,10 +262,14 @@
         clasiskl
         clasthac
         thiefscl
+        sneakatt
+        crippstr
       RET
         backstab
         thiefskl
         traplimt
+        sneakatt
+        crippstr
     END
 
     COPY_EXISTING kitlist.2da override
@@ -236,6 +282,7 @@
               READ_2DA_ENTRY_FORMER file i 0 number
             END
           END
+          LPM fl#add_kit_ee#validate_kitids_value
         BUT_ONLY
       END
       INNER_PATCH_SAVE kitids "%number%" BEGIN
@@ -252,35 +299,38 @@
       LOOKUP_IDS_SYMBOL_OF_INT class class kit_class
     END
 
-    ACTION_IF biography < "-1" OR briefidesc < "-1" BEGIN
+    ACTION_IF biography < "-1" OR briefdesc < "-1" OR fallen_notice < "-1" BEGIN
       COPY_EXISTING clastext.2da override
-        PATCH_IF biography < "-1" BEGIN
+        PATCH_FOR_EACH column IN biography briefdesc fallen_notice BEGIN
+          PATCH_IF EVAL "%column%" < "-1" AND FILE_CONTAINS_EVALUATED (clastext.2da "%column%") BEGIN
           LPF fl#add_kit_ee#get_2da_value
             STR_VAR
               row = EVAL "%class%"
-              column = biography
+                column
               req_column_count = 8
             RET
-              biography = value
-          END
+                value
         END
-        PATCH_IF briefdesc < "-1" BEGIN
-          LPF fl#add_kit_ee#get_2da_value
-            STR_VAR
-              row = EVAL "%class%"
-              column = briefdesc
-              req_column_count = 8
-            RET
-              briefdesc = value
+            LPM fl#add_kit_ee#validate_clastext_value
+            SET EVAL "%column%" = value
           END
         END
       BUT_ONLY
     END
 
-    OUTER_SPRINT clastext "%kit_name% %kit_class% %kit_number% %kit_lower% %kit_desc% %kit_mixed% %biography% 0 %briefdesc%"
+    OUTER_SPRINT clastext "%kit_name% %kit_class% %kit_number% %kit_lower% %kit_desc% %kit_mixed% %biography% %fallen% %briefdesc%"
+
+    ACTION_IF FILE_CONTAINS_EVALUATED (clastext.2da "FALLEN_NOTICE") BEGIN
+      OUTER_SPRINT clastext "%clastext% %fallen_notice%"
+    END
+
     APPEND clastext.2da "%clastext%" UNLESS "%kit_name%"
 
-    ACTION_FOR_EACH item IN
+    ACTION_IF FILE_EXISTS_IN_GAME ~sodcltxt.2da~ BEGIN
+      APPEND sodcltxt.2da "%clastext%" UNLESS "%kit_name%"
+    END
+
+    ACTION_DEFINE_ARRAY row_files BEGIN
                     backstab
                     clsrcreq
                     clswpbon
@@ -288,7 +338,19 @@
                     numwslot
                     thiefskl
                     traplimt
-    BEGIN
+      sneakatt
+      crippstr
+    END
+
+    ACTION_DEFINE_ARRAY col_files BEGIN
+      clascolr
+      clasiskl
+      clasthac
+      thiefscl
+    END
+
+    ACTION_PHP_EACH row_files AS _ => item BEGIN
+      ACTION_IF FILE_EXISTS_IN_GAME "%item%.2da" BEGIN
       ACTION_IF FILE_CONTAINS_EVALUATED ("%item%.2da" "^[ %TAB%]*%class%[ %TAB%]+") BEGIN
         OUTER_SPRINT clone "%class%"
       END ELSE BEGIN
@@ -302,13 +364,10 @@
           clone
       END
     END
+    END
 
-    ACTION_FOR_EACH item IN
-                    clascolr
-                    clasiskl
-                    clasthac
-                    thiefscl
-    BEGIN
+    ACTION_PHP_EACH col_files AS _ => item BEGIN
+      ACTION_IF FILE_EXISTS_IN_GAME "%item%.2da" BEGIN
       LAF fl#add_kit_ee#add_column
         STR_VAR
           kit_name
@@ -317,25 +376,26 @@
           clone = EVAL "%class%"
       END
     END
+    END
 
-    ACTION_FOR_EACH file IN
-                    backstab
-                    clascolr
-                    clasiskl
-                    clastext
-                    clasthac
-                    clsrcreq
-                    clswpbon
-                    hpclass
-                    numwslot
-                    thiefscl
-                    thiefskl
-                    traplimt
-                    kitlist
-    BEGIN
+    ACTION_DEFINE_ARRAY cleanup BEGIN kitlist END
+    OUTER_SET index = 1
+
+    ACTION_PHP_EACH row_files AS _ => file BEGIN
+      OUTER_SPRINT $cleanup("%index%") "%file%"
+      OUTER_SET ++index
+    END
+    ACTION_PHP_EACH col_files AS _ => file BEGIN
+      OUTER_SPRINT $cleanup("%index%") "%file%"
+      OUTER_SET ++index
+    END
+
+    ACTION_PHP_EACH cleanup AS _ => file BEGIN
+      ACTION_IF FILE_EXISTS_IN_GAME "%file%.2da" BEGIN
       COPY_EXISTING "%file%.2da" override
         PRETTY_PRINT_2DA
       BUT_ONLY
     END
   END
 END
+END
