--- Sellswords/Setup-Sellswords.tp2	Thu Nov 17 01:00:03 2016
+++ Fixpack/Sellswords/Setup-Sellswords.tp2	Thu Nov 17 01:00:10 2016
@@ -1,12 +1,20 @@
 BACKUP ~Sellswords/Backup~
 AUTHOR ~http://forums.pocketplane.net~
-VERSION ~v5~
+VERSION ~v5 BWP Fix~
 
 README ~Sellswords/%LANGUAGE%/Readme-Sellswords.txt~ ~Sellswords/Readme-Sellswords.txt~ 
 
 ALWAYS
-  // Borrowed from Edwin Romance: convert strings to UTF-8 for BGEE/BG2EE
-  INCLUDE ~%MOD_FOLDER%/handle_charsets.tpa~
+  ACTION_IF GAME_IS ~eet~ BEGIN
+    OUTER_SET bg2_chapter = 12
+  END ELSE BEGIN
+    OUTER_SET bg2_chapter = 0
+  END
+  OUTER_FOR (i=1; i<=10; i=i+1) BEGIN
+    OUTER_SET bg2_chapter = bg2_chapter + 1
+    OUTER_SPRINT name_source ~bg2_chapter_%i%~
+    OUTER_SET EVAL ~%name_source%~ = bg2_chapter
+  END
 
   // No exception, everything needs to be converted
   ACTION_DEFINE_ARRAY O#noconvert BEGIN END
@@ -14,10 +22,10 @@
   // Only setup-Mod.tra needs to be reloaded
   ACTION_DEFINE_ARRAY O#reload BEGIN Setup-Sellswords END
 
-  // There is no "language" folder so the root of all languages in %MOD_FOLDER%
+  // There is no "language" folder so the root of all languages is %MOD_FOLDER%
   LAF HANDLE_CHARSETS
     INT_VAR
-      infer_charset = 1
+      infer_charsets = 1
     STR_VAR
       tra_path = EVAL ~%MOD_FOLDER%~
       // charset_table = O#charsets // Included for illustrative purposes.
@@ -46,7 +54,7 @@
 	      ~AM2203.wav~   ~override/O#2203.wav~
 	      ~AM2204.wav~   ~override/O#2204.wav~
 
-ACTION_IF GAME_IS ~BG2EE~ BEGIN
+ACTION_IF GAME_IS ~BG2EE EET~ BEGIN
 LOAD_TRA ~Sellswords/%LANGUAGE%/Setup-Sellswords-EE.tra~
 ADD_JOURNAL @9999 @10000 @10001
 ADD_JOURNAL @10002 @10003 @10004 @10005 @10006 @10007 @10008 @10009 @10010 @10011 @10012 @10013 @10014 @10015 @10016 @10017 @10018 @10019 @10020 @10021 @10022 @10023 @10024 @10025 @10026 
@@ -121,7 +129,7 @@
 	~Sellswords/Scripts/O#LLTALK.baf~
 	~Sellswords/Scripts/O#LLZEDE.baf~
 
-EXTEND_TOP ~AR0700.bcs~ ~Sellswords/Scripts/AR0700.baf~
+EXTEND_TOP ~AR0700.bcs~ ~Sellswords/Scripts/AR0700.baf~ EVALUATE_BUFFER
 EXTEND_TOP ~AR0701.bcs~ ~Sellswords/Scripts/AR0701.baf~
 
 // Drow items
