--- Star.tp2	Sun Jul 22 20:53:27 2012
+++ C:\BWP Patchstudio\patched files\Star.tp2	Tue Aug 02 09:16:06 2016
@@ -1,5 +1,6 @@
 BACKUP ~Star\backup~
 AUTHOR ~Olryx2@Yahoo.co.uk~
+VERSION ~v1.93 BWP Fix~
 AUTO_TRA ~Star/%s~
 ALLOW_MISSING ~aR4500.bcs~
 ~FATESP.dlg~
@@ -77,8 +78,8 @@
   SAY HIDDEN_IN_SHADOWS ~~ [Innoc5]
   SAY SET_A_TRAP ~~ [Innoc7]
   WRITE_SHORT   0x244 0
-  WRITE_BYTE    0x246 ~%StarHL%~
-  WRITE_BYTE    0x247 0x40
+  WRITE_SHORT   0x246 0x4000 + ~%StarHL%~
+//WRITE_BYTE    0x247 0x40
   SAY NAME1 ~Silver Star~
   SAY NAME2 ~Silver Star~
   SAY BIO @6
@@ -144,7 +145,11 @@
 REPLACE 4004 @24
 REPLACE 4005 @25
 
-COPY ~Star\Creature\SHAPE.cre~ ~override\SHAPE.cre~
+COPY_EXISTING ~SHAPE.cre~ ~override~
+  PATCH_IF SOURCE_SIZE > 0x2d3 BEGIN
+    ADD_CRE_ITEM ~D#CRYBL1~ #0 #0 #0 ~NONE~ ~INV~
+  END
+BUT_ONLY
 COPY ~Star\creature\D#Mary.cre~ ~override\D#Mary.cre~
  SAY NAME1 @26
  SAY NAME2 @26
