--- fadingpromises/dialogue/b!aidan.d	2007-12-24 12:37:36.000000000 -0500
+++ Fixpack/fadingpromises/dialogue/b!aidan.d	2016-11-21 11:40:52.041008100 -0500
@@ -1111,7 +1111,7 @@
 
 IF ~~ BA3.8
 SAY @394
-IF ~~ THEN EXIT
+IF ~~ THEN + BA3.18
 END
 
 IF ~~ BA3.9
@@ -1434,11 +1434,11 @@
 
 IF ~~ BA4.30
 SAY @515
-+ ~GlobalLT("Chapter","GLOBAL",5) Global("B!AidanIrenicus","LOCALS",0)~ + @516 DO ~SetGlobal("B!AidanIrenicus","LOCALS",1)~ + BA4.18
-+ ~GlobalGT("Chapter","GLOBAL",4) Global("B!AidanIrenicus","LOCALS",0)~ + @517 DO ~SetGlobal("B!AidanIrenicus","LOCALS",1)~ + BA4.19
-+ ~GlobalLT("Chapter","GLOBAL",5) Global("B!AidanImoen","LOCALS",0)~ + @518 DO ~IncrementGlobal("B!AidanLove","GLOBAL",1) SetGlobal("B!AidanImoen","LOCALS",1)~ + BA4.20
-+ ~GlobalGT("Chapter","GLOBAL",4) Global("B!AidanImoen","LOCALS",0)~ + @519 DO ~IncrementGlobal("B!AidanLove","GLOBAL",1) SetGlobal("B!AidanImoen","LOCALS",1)~ + BA4.21
-+ ~Global("Chapter","GLOBAL",6) Global("B!AidanRhynnLanthorn","LOCALS",0)~ + @520 DO ~SetGlobal("B!AidanRhynnLanthorn","LOCALS",1)~ + BA4.22
++ ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_5%) Global("B!AidanIrenicus","LOCALS",0)~ + @516 DO ~SetGlobal("B!AidanIrenicus","LOCALS",1)~ + BA4.18
++ ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_4%) Global("B!AidanIrenicus","LOCALS",0)~ + @517 DO ~SetGlobal("B!AidanIrenicus","LOCALS",1)~ + BA4.19
++ ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_5%) Global("B!AidanImoen","LOCALS",0)~ + @518 DO ~IncrementGlobal("B!AidanLove","GLOBAL",1) SetGlobal("B!AidanImoen","LOCALS",1)~ + BA4.20
++ ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_4%) Global("B!AidanImoen","LOCALS",0)~ + @519 DO ~IncrementGlobal("B!AidanLove","GLOBAL",1) SetGlobal("B!AidanImoen","LOCALS",1)~ + BA4.21
++ ~Global("Chapter","GLOBAL",%bg2_chapter_6%) Global("B!AidanRhynnLanthorn","LOCALS",0)~ + @520 DO ~SetGlobal("B!AidanRhynnLanthorn","LOCALS",1)~ + BA4.22
 + ~Global("B!AidanDarkForces","LOCALS",0)~ + @521 DO ~IncrementGlobal("B!AidanLove","GLOBAL",1) SetGlobal("B!AidanDarkForces","LOCALS",1)~ + BA4.23
 + ~Kit(Player1,GODLATHANDER) Global("B!AidanUndeadLegion","LOCALS",0)~ + @522 DO ~SetGlobal("B!AidanUndeadLegion","LOCALS",1)~ + BA4.24
 + ~Global("B!AidanStayAlive","LOCALS",0)~ + @523 DO ~SetGlobal("B!AidanStayAlive","LOCALS",1)~ + BA4.25
@@ -1574,12 +1574,12 @@
 
 IF ~~ BA4.55
 SAY @557
-IF ~~ THEN EXIT
+IF ~~ THEN + BA4.59
 END
 
 IF ~~ BA4.56
 SAY @558
-IF ~~ THEN + BA4.59
+IF ~~ THEN EXIT
 END
 
 IF ~~ BA4.57
