--- Yvette\Dialogues\DrawNAT.d	Fri Nov 14 00:23:54 2014
+++ C:\BWP Patchstudio\patched files\Yvette\Dialogues\DrawNAT.d	Wed Nov 11 13:51:33 2015
@@ -190,7 +190,7 @@
 =
 @78 /* I have seen <PRO_HISHER> story. A childhood in the citadel, a best friend, a father. Everything was so real, it felt as though I was in the middle of it. Who was I in the dream? I don't know. I will never forget it - the seagulls that called as they soared on the wind, the crashing waves on the cliffs... I saw the hero's face once - before <PRO_HESHE> fell asleep, <PRO_HESHE> wondered if it was time to leave. */
 =
-@79 /* At last <PRO_HESHE> left the citadel. I saw <PRO_HERHIM> weeks later, with several friends - a harsh but kindly elven woman, a bard who wanted to travel and learn more about the world, a mysterious elf who never told <PRO_HERHIM> their reason for joining the group... Only a few days later, they faced an evil creature - a wrathful spirit who wanted to rule the land as a tyrant. "You shall never prevail!" the hero taunted. The ensuing battle took two days, but the hero and <PRO_HISHER> companions at last defeated the creature. Then the spirit explained why it wanted to claim the land. Long ago, a past ruler of this land destroyed a temple dedicated to this spirit, slaying all who remembered its existence. The spirit wanted to avenge the fallen. After telling its story, the spirit died its final death, but that isn't the end of the story. */
+@79 /* At last <PRO_HESHE> left the citadel. I saw <PRO_HIMHER> weeks later, with several friends - a harsh but kindly elven woman, a bard who wanted to travel and learn more about the world, a mysterious elf who never told <PRO_HIMHER> their reason for joining the group... Only a few days later, they faced an evil creature - a wrathful spirit who wanted to rule the land as a tyrant. "You shall never prevail!" the hero taunted. The ensuing battle took two days, but the hero and <PRO_HISHER> companions at last defeated the creature. Then the spirit explained why it wanted to claim the land. Long ago, a past ruler of this land destroyed a temple dedicated to this spirit, slaying all who remembered its existence. The spirit wanted to avenge the fallen. After telling its story, the spirit died its final death, but that isn't the end of the story. */
 =
 @80 /* The hero built a new temple for the spirit and those who had died all those years ago. I joined them, because I thought it was a good idea. We built a tower with a huge monument inside. Then the spirit's last priest arrived, and he could hardly believe his eyes! He remained in the new temple, aiding the adventurers who required it. The temple's tragic past was forgotten, put aside in favor of its glorious future. It was then a place of happiness, and a symbol of how fate can change. */
 IF~~THEN GOTO YvP3xT4
@@ -258,7 +258,7 @@
 IF~~THEN BEGIN YvP4x06
 SAY @104 /* Really? Well, I think differently - it seems like it should be something magical. */
 =
-@105 /* I don't know if I asked because it's something I can't experience now, or because I *did* experience it... Still, I think it's best to see the world as a magical place, dear <PRO_LORDLADY>. */
+@105 /* I don't know if I asked because it's something I can't experience now, or because I *did* experience it... Still, I think it's best to see the world as a magical place, dear <PRO_LADYLORD>. */
 IF~~THEN GOTO YvP4x07
 END
 
@@ -277,7 +277,7 @@
 END
 
 IF~~THEN BEGIN YvP4x08
-SAY @110 /* You know - when I looked at you, just now, I realized that you are...kind. Not everyone would give me so much attention. If I had a body I would kiss your cheek, my <PRO_LORDLADY>. You understand - to show how much I appreciate what you do for me. */
+SAY @110 /* You know - when I looked at you, just now, I realized that you are...kind. Not everyone would give me so much attention. If I had a body I would kiss your cheek, my <PRO_LADYLORD>. You understand - to show how much I appreciate what you do for me. */
 IF~~THEN REPLY @111 /* I'm honored to talk with someone as pleasant as you, Yvette. I don't need a reward for that. */ GOTO YvP4x09
 IF~~THEN REPLY @112 /* Thank you, Yvette. I bet you say that to all your friends. (Laugh.) */ GOTO YvP4x10
 IF~~THEN REPLY @113 /* I don't care. If you really cared, you would leave me alone. */ GOTO YvP4x02
@@ -324,7 +324,7 @@
 END
 
 IF~~THEN BEGIN YvP5x03
-SAY @129 /* Exactly, my <PRO_LORDLADY>. I couldn't have said it better myself. I am glad we agree with on this, <CHARNAME>. */
+SAY @129 /* Exactly, my <PRO_LADYLORD>. I couldn't have said it better myself. I am glad we agree with on this, <CHARNAME>. */
 IF~~THEN GOTO YvP5x05
 END
 
