--- Yvette\Setup-Yvette.tp2	Mon Jan 26 03:38:40 2015
+++ C:\BWP Patchstudio\patched files\Yvette\Setup-Yvette.tp2	Mon May 11 09:38:34 2015
@@ -1,7 +1,7 @@
 BACKUP ~Yvette\backup~
 AUTHOR ~Lava Del'Vortel~
 
-VERSION ~2.0_Isaya~
+VERSION ~2.0_Isaya BWP Fix~
 
 README ~Yvette/translations/%LANGUAGE%/Readme.txt~ ~Yvette/Readme.txt~
 
@@ -26,7 +26,7 @@
 
 ADD_KIT ~YxKit~
 ~YxKit 0 0 0 0 0 0 0 0~
-~YxKit 0 0 0 0 1 1 0 1 0 0 0 0 0 0 0 0 1 1 0 0 1 1 1 0 0 0 0 1 1 1 1 1 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0~
+~YxKit 0 0 0 0 1 1 0 1 0 0 0 0 0 0 0 0 1 1 0 0 1 1 1 0 0 0 0 1 1 1 1 1 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0~
 ~YxKit 0 0 0 0 9 0~
 ~YxKit 0 0 0 0 0 0~
 ~YxKit 0 0 0 0 0 0~
