--- Yvette/Dialogues/YxYveJ.d	2014-11-14 00:25:56.000000000 -0500
+++ Fixpack/Yvette/DIALOGUES/YxYveJ.d	2016-11-21 22:53:26.212658800 -0500
@@ -30,7 +30,7 @@
 
 ///////Book Gift from charname
 
-IF~HasItem("YxBook",Myself) Global("YvetteHasBook","GLOBAL",0) !Global("YvetteRomanceActive","GLOBAL",4) GlobalLT("CHAPTER","GLOBAL",4)~THEN BEGIN YvBox00
+IF~HasItem("YxBook",Myself) Global("YvetteHasBook","GLOBAL",0) !Global("YvetteRomanceActive","GLOBAL",4) GlobalLT("CHAPTER","GLOBAL",%bg2_chapter_4%)~THEN BEGIN YvBox00
 SAY @4 /* Is this book for me? Don't tell me...this is "Sune's Book of Chants"! I've seen a copy only once before! */
 IF~~THEN REPLY @5 /* And do you see any other Heartwarder here, my fair lady? Of course it's for you! */ GOTO YvBox01
 IF~~THEN REPLY @6 /* Well, yes! I hope you'll like it. */ GOTO YvBox01
@@ -120,7 +120,7 @@
 
 /////////////... and now I know how to do that!
 
-IF~Global("YvetteTurnUndead","GLOBAL",2) GlobalGT("CHAPTER","GLOBAL",5) AreaCheck("AR0800") !Global("YvetteRomanceActive","GLOBAL",4)~THEN BEGIN YVxTUx20
+IF~Global("YvetteTurnUndead","GLOBAL",2) GlobalGT("CHAPTER","GLOBAL",%bg2_chapter_5%) AreaCheck("AR0800") !Global("YvetteRomanceActive","GLOBAL",4)~THEN BEGIN YVxTUx20
 SAY @37 /* I just wanted to tell you that I think I now know how to turn undead now, and just in time too it seems. Now I'm sure we will win! */
 IF~~THEN REPLY @38 /* Good and thank you. I'm glad you've told me. We'll fight together, and Bodhi will fall! */ GOTO YVxTUx21
 IF~~THEN REPLY @39 /* (Nod and start walking again.) */ GOTO YVxTUx22
