--- Yvette/Scripts/YxYve.baf	2012-02-25 13:52:48.000000000 -0500
+++ Fixpack/Yvette/SCRIPTS/YxYve.baf	2016-11-26 01:12:48.960672600 -0500
@@ -36,7 +36,7 @@
 IF
 InParty(Myself)
 Global("YvetteTurnUndead","GLOBAL",2)
-GlobalGT("CHAPTER","GLOBAL",5)
+GlobalGT("CHAPTER","GLOBAL",%bg2_chapter_5%)
 AreaCheck("AR0800")
 THEN
 RESPONSE #100
@@ -69,7 +69,7 @@
 !AreaType(DUNGEON)
 !See([ENEMY])
 CombatCounter(0)
-GlobalLT("CHAPTER","GLOBAL",4)
+GlobalLT("CHAPTER","GLOBAL",%bg2_chapter_4%)
 RealGlobalTimerExpired("YvetteRomanceTime","GLOBAL")
 OR(2)
 Global("YvetteRomanceActive","GLOBAL",2)
@@ -104,7 +104,7 @@
 !AreaCheck("ARY001")
 InParty(Myself)
 CombatCounter(0)
-GlobalLT("CHAPTER","GLOBAL",4)
+GlobalLT("CHAPTER","GLOBAL",%bg2_chapter_4%)
 RealGlobalTimerExpired("YvetteRomanceTime","GLOBAL")
 OR(2)
 Global("YvetteRomanceActive","GLOBAL",2)
@@ -135,7 +135,7 @@
 IF
 InParty(Myself)
 Global("YvetteRomanceActive","GLOBAL",1)
-GlobalLT("CHAPTER","GLOBAL",4)
+GlobalLT("CHAPTER","GLOBAL",%bg2_chapter_4%)
 Global("YLie","LOCALS",1)
 THEN
 RESPONSE #100
@@ -145,7 +145,7 @@
 
 IF
 InParty(Myself)
-GlobalLT("CHAPTER","GLOBAL",4)
+GlobalLT("CHAPTER","GLOBAL",%bg2_chapter_4%)
 Global("YvetteRomanceActive","GLOBAL",1)
 Global("YLie","LOCALS",2)
 RealGlobalTimerExpired("YvetteLie","GLOBAL")
@@ -156,7 +156,7 @@
 
 IF
 Global("YvetteRomanceTalk","GLOBAL",17)
-GlobalLT("CHAPTER","GLOBAL",4)
+GlobalLT("CHAPTER","GLOBAL",%bg2_chapter_4%)
 Global("YStraw","GLOBAL",1)
 THEN
 RESPONSE #100
@@ -167,7 +167,7 @@
 
 IF
 Global("YvetteEmily","GLOBAL",2)
-GlobalLT("CHAPTER","GLOBAL",4)
+GlobalLT("CHAPTER","GLOBAL",%bg2_chapter_4%)
 InParty(Myself)
 CombatCounter(0)
 RealGlobalTimerExpired("YvetteRomanceTime","GLOBAL")
@@ -185,7 +185,7 @@
 InParty(Myself)
 Global("YvetteRomanceTalk","GLOBAL",30)
 Global("YvetteRomanceActive","GLOBAL",1)
-GlobalLT("CHAPTER","GLOBAL",4)
+GlobalLT("CHAPTER","GLOBAL",%bg2_chapter_4%)
 THEN
 RESPONSE #100
 SetGlobal("YvetteRomanceTalk","GLOBAL",31)
@@ -198,7 +198,7 @@
 Global("YvetteRomanceActive","GLOBAL",1)
 Global("YvetteRomanceTalk","GLOBAL",31)
 GlobalTimerExpired("YvetteSleepTimer","GLOBAL")
-GlobalLT("CHAPTER","GLOBAL",4)
+GlobalLT("CHAPTER","GLOBAL",%bg2_chapter_4%)
 THEN
 RESPONSE #100
 PlaySong(0)
@@ -210,7 +210,7 @@
 InParty(Myself)
 Global("YvetteRomanceTalk","GLOBAL",36)
 Global("YvetteRomanceActive","GLOBAL",1)
-GlobalLT("CHAPTER","GLOBAL",4)
+GlobalLT("CHAPTER","GLOBAL",%bg2_chapter_4%)
 AreaCheck("AR0020")
 THEN
 RESPONSE #100
@@ -292,7 +292,7 @@
 InParty(Myself)
 Global("YvetteRomanceTalk","GLOBAL",38)
 Global("YvetteRomanceActive","GLOBAL",1)
-GlobalLT("CHAPTER","GLOBAL",4)
+GlobalLT("CHAPTER","GLOBAL",%bg2_chapter_4%)
 AreaCheck("ARY001")
 THEN
 RESPONSE #100
@@ -336,7 +336,7 @@
 END
 
 IF
-Global("CHAPTER","GLOBAL",4)
+Global("CHAPTER","GLOBAL",%bg2_chapter_4%)
 GlobalLT("YvetteRomanceTalk","GLOBAL",40)
 Global("YvetteRomanceActive","GLOBAL",1)
 THEN
@@ -417,7 +417,7 @@
 !See([ENEMY])
 HasItem("YxBook",Myself)
 Global("YvetteHasBook","GLOBAL",0)
-GlobalLT("CHAPTER","GLOBAL",4)
+GlobalLT("CHAPTER","GLOBAL",%bg2_chapter_4%)
 THEN
 RESPONSE #100
 StartDialog("YxYveJ",Player1)
