--- NeJ2\baf\VPCutT20.baf	Fri Jan 02 22:36:30 2004
+++ C:\BWP Patchstudio\patched files\NeJ2\baf\VPCUTT20.BAF	Fri Oct 10 04:14:45 2014
@@ -2,7 +2,7 @@
 	True()
 THEN
 	RESPONSE #100
-		CutSceneId("TuPhin")
+		CutSceneId(Player1)
 		FadeToColor([20.0],0)
 		Wait(1)
 		ActionOverride(Player2,MakeUnselectable(1))
@@ -14,9 +14,9 @@
 		Wait(1)
 		FadeFromColor([20.0],0)
 		Wait(1)
-		GiveItem("ANTIMAG",Player1) // Antimagic Shell
-		AddXPObject(Player1,21000)
-		ReallyForceSpell(Myself,DRYAD_TELEPORT)
+		ActionOverride("TuPhin",GiveItem("ANTIMAG",Player1)) // Antimagic Shell
+		ActionOverride("TuPhin",AddXPObject(Player1,15750))
+		ActionOverride("TuPhin",ReallyForceSpell(Myself,DRYAD_TELEPORT)) // SPWI995.SPL (Dimension Door)
 		Wait(2)
 		EndCutSceneMode()
 END
