--- item_rev\lib\armor_penalty_table.txt	Sun Jul 17 01:13:52 2011
+++ C:\BWP Patchstudio\patched files\item_rev\lib\armor_penalty_table.txt	Mon May 28 22:25:02 2012
@@ -1,14 +1,19 @@
 
-armor     spell_failure  casting_speed  stealth  dexterity  movement  speed_factor
+armor     stealth  spell_failure  casting_speed  dexterity  movement  speed_factor  phys_res
 
-leather              10              2        0          0         0             0
-studded              15              2       20          0         0             1
-hide                 20              2       30          0        10             2
-chain                30              3       40          5        10             2
-splint               40              3       50         10        20             3
-plate                40              4       60         10        20             4
-full                 50              4       80         15        30             5
+leather         0             10              2          0         0             0         5
+studded        20             15              2          0         0             1         5
+hide           30             20              2          0        10             1         5
+chain          40             30              3          5        10             2        10
+splint         50             40              3         10        20             2        10
+plate          60             40              4         10        20             3        15
+full           80             50              4         15        30             4        15
 
-scale               -10             -1      -10         -5       -10            -1
-mithral             -20             -1      -30         -5       -10            -2
-elven               -30             -3      -30        -10       -20            -3
\ No newline at end of file
+scale         -10            -10             -1         -5       -10             0
+mithral       -30            -20             -1         -5       -10            -1
+elven         -30            -30             -3        -10       -20            -3
+
+buckler         0              5              1                                   
+small          10              5              1                                   
+medium         30             15              2                                   
+tower          80             50              3                                   
\ No newline at end of file
