--- item_rev\lib\descriptions.tpa	Tue Jul 12 22:39:10 2011
+++ C:\BWP Patchstudio\patched files\item_rev\lib\descriptions.tpa	Sun Dec 18 10:56:18 2011
@@ -14,7 +14,8 @@
 
 // stolen from BG2 Tweak Pack at G3 and modified to take parameter ~%text_update%~, the macro name of the REPLACE_TEXTUALLY to execute
 DEFINE_PATCH_MACRO ~update_item_descriptions~ BEGIN
-  FOR (index = 0x50 ; index < 0x55 ; index = index + 0x04) BEGIN // loop through descriptions
+  ident=1
+  FOR (index = 0x54 ; index >= 0x50 ; index -= 4) BEGIN // loop through descriptions
     READ_LONG "%index%" "valid"
     PATCH_IF ("%valid%" < 2147483646) AND ("%valid%" >= 0) BEGIN // verify description is valid
       READ_STRREF "%index%" "description"
@@ -23,6 +24,7 @@
       END
       SAY_EVALUATED "%index%" ~%new_desc%~
  //     PATCH_PRINT ~%new_desc%~
+      ident -=1
     END
   END
 END
\ No newline at end of file
