--- item_rev\components\dual_wielding.tpa	Mon Aug 01 04:59:12 2011
+++ C:\BWP Patchstudio\patched files\item_rev\components\dual_wielding.tpa	Sat Dec 17 22:26:16 2011
@@ -1,5 +1,11 @@
+INCLUDE ~item_rev/lib/identify_fix.tpa~
+INCLUDE ~item_rev/lib/identify_weapon.tpa~
+
+//////////////////////////////////////////////
+//  patching
+
 COPY_EXISTING_REGEXP GLOB ~^.+\.itm$~ override  PATCH_IF SOURCE_SIZE>0x71 BEGIN
-  PATCH_INCLUDE ~item_rev/lib/identify_weapon.tpa~
+  LPM identify_item
 
   PATCH_IF dagger || short_sword || ninjato || wakizashi || club || handaxe
                   || (~%SOURCE_RES%~ STRING_COMPARE_REGEXP ~\(blun28\|blun29\)~)=0 BEGIN // Storm Star +3/+5
