--- item_rev\components\enchantment_speed_factor.tpa	Mon Aug 01 05:02:20 2011
+++ C:\BWP Patchstudio\patched files\item_rev\components\enchantment_speed_factor.tpa	Mon May 28 22:25:02 2012
@@ -1,301 +1,165 @@
+INCLUDE ~item_rev/lib/compatibility.tpa~
 INCLUDE ~item_rev/lib/descriptions.tpa~
-INCLUDE ~item_rev/lib/fixpack.tpa~
+ACTION_IF comp_ir_wc=0 && comp_ir_sf=0 BEGIN // only run it once
+  INCLUDE ~item_rev/lib/fixpack.tpa~
+END
 INCLUDE ~item_rev/lib/identify_fix.tpa~
-
-// code herein is a lite version of Weapon Changes
+INCLUDE ~item_rev/lib/identify_weapon.tpa~
+INCLUDE ~item_rev/lib/weapon_changes_main.tpa~
+OUTER_SPRINT debug_file ~item_rev/debug/weapon_debug.log~
 
 /////////////////////////////////////////////////////
-// speed factor table
+//  Read tables
 
-ACTION_IF MOD_IS_INSTALLED ~item_rev/item_rev.tp2~ 17 BEGIN // check is Weapon Changes component is already installed
-  OUTER_SET wc=1
+ACTION_IF comp_ir_wc BEGIN
+  OUTER_SPRINT old_weapon_table ~item_rev/lib/irv3_weapon_stats.txt~
+  OUTER_SPRINT new_weapon_table ~item_rev/lib/irv3_weapon_stats.txt~
 END ELSE BEGIN
-  OUTER_SET wc=0
+  OUTER_SPRINT old_weapon_table ~item_rev/lib/bg2_weapon_stats.txt~
+  OUTER_SPRINT new_weapon_table ~item_rev/lib/bg2_weapon_stats.txt~
 END
-
-OUTER_SET sf_dagger          = (wc=0)? 2 : 1
-OUTER_SET sf_short_sword     = (wc=0)? 3 : 3
-OUTER_SET sf_long_sword      = (wc=0)? 5 : 5
-OUTER_SET sf_scimitar        = (wc=0)? 5 : 5
-OUTER_SET sf_battleaxe       = (wc=0)? 7 : 6
-OUTER_SET sf_bastard_sword   = (wc=0)? 8 : 7
-OUTER_SET sf_ninjato         = (wc=0)? 4 : 2
-OUTER_SET sf_wakizashi       = (wc=0)? 3 : 3
-OUTER_SET sf_katana          = (wc=0)? 4 : 5
-OUTER_SET sf_club            = (wc=0)? 4 : 3
-OUTER_SET sf_hammer          = (wc=0)? 4 : 5
-OUTER_SET sf_mace            = (wc=0)? 7 : 6
-OUTER_SET sf_morning_star    = (wc=0)? 7 : 7
-OUTER_SET sf_flail           = (wc=0)? 7 : 8
-OUTER_SET sf_staff           = (wc=0)? 4 : 4
-OUTER_SET sf_spear           = (wc=0)? 6 : 6
-OUTER_SET sf_halberd         = (wc=0)? 9 : 8
-OUTER_SET sf_twohanded_sword = (wc=0)? 10 : 10
-OUTER_SET sf_dart            = (wc=0)? 2 : 2
-OUTER_SET sf_handaxe         = (wc=0)? 4 : 4
-OUTER_SET sf_sling           = (wc=0)? 6 : 6
-OUTER_SET sf_short_bow       = (wc=0)? 6 : 6
-OUTER_SET sf_comp_long_bow   = (wc=0)? 7 : 7
-OUTER_SET sf_long_bow        = (wc=0)? 8 : 8
-OUTER_SET sf_light_xbow      = (wc=0)? 5 : 5
-OUTER_SET sf_heavy_xbow      = (wc=0)? 10 : 10
+LAM wc_table_read
 
 /////////////////////////////////////////////////////
-// Main patching macro
-
-DEFINE_PATCH_MACRO wc_patch BEGIN
-  READ_LONG  0x64 ab_off
-  READ_SHORT 0x68 ab_num
-  reupdate=1
-  FOR (i=0;i<ab_num;i+=1) BEGIN
-    patched=0
-    off=ab_off+i*0x38
-    READ_BYTE  off      type
-    READ_SHORT off+0x12 speed
-
-    delta_speed=0
-    unid_old_speed=speed
-
-    update_line=1
-    LPM ~%macro%~
-
-    PATCH_IF patched=1 BEGIN
-      new_speed= ( (speed+delta_speed) < 0 ) ? 0 : ( (speed+delta_speed) > 10 ) ? 10 : (speed+delta_speed) // [0;10]
-
-      WRITE_SHORT off+0x12 new_speed
-
-      PATCH_IF reupdate=1 BEGIN // update description only once
-        reupdate=0
-        SPRINT text_update wc_update
-        LPM update_item_descriptions
-      END
-    END
-
-  END
-END
-
-/////////////////////////////////////////////////////
-// Main description update macro
-
-// enable logging
-<<<<<<<<ag#dummy
+//  Special cases
 
+DEFINE_PATCH_MACRO special BEGIN
+  TO_LOWER ~SOURCE_RES~
 
->>>>>>>>
-COPY ~ag#dummy~ ~item_rev/debug/enchantment_speed.log~
-
-DEFINE_PATCH_MACRO wc_update BEGIN
-  PATCH_IF index=0x54 BEGIN
-    SPRINT colon @100100
-
-    // speed factor
-    PATCH_IF delta_speed !=0 BEGIN // SF was changed
-      nspeed=new_speed
-      ospeed=speed
-
-      linenum=0
-      SPRINT t1 @100101
-      REPLACE_EVALUATE ~%t1%~ BEGIN
-        log=0
-        linenum+=1
-        SPRINT line ~%MATCH1%~
-        PATCH_IF linenum=update_line BEGIN
-          INNER_PATCH_SAVE line ~%line%~ BEGIN
-            once=0
-            REPLACE_EVALUATE ~%ospeed%~ BEGIN
-              PATCH_IF once=0 BEGIN
-                ospeed=nspeed
-                once=1
-                log=1
-              END
-            END ~%ospeed%~
-          END
-        END
-        PATCH_IF log=0 BEGIN
-          INNER_ACTION BEGIN
-            APPEND_OUTER ~item_rev/debug/enchantment_speed.log~ ~%SOURCE_RES%   - Speed Factor~
-          END
-        END
-      END ~%line%~
-
-    END
-
-  END
-
-END
+  PATCH_IF type=1 || type=2 || type=4 BEGIN // melee/ranged/launcher
 
-/////////////////////////////////////////////////////
+  // mark the item as ready for updating
+  // also we can unflag it later if needed
+  patched=1
+
+  /////////////////////////
+  //    global stuff     //
+  /////////////////////////
 
-// generic
-DEFINE_PATCH_MACRO sf BEGIN
-  PATCH_IF type=1 || type=2 || type=4 BEGIN // melee/ranged/launcher
-    unid_old_speed= EVAL ~sf_%weapon%~
-    delta_speed=unid_old_speed - speed - extra
-  SET patched=1 END
-END
+  // set speed to base value
+  iden_delta_speed = (EVAL ~old_%itemtype%_speed~) - speed
 
-// halberd
-DEFINE_PATCH_MACRO halberd BEGIN
-  PATCH_IF type=1 BEGIN // melee
-    reupdate=1
+  // halberds, dual mode
+  PATCH_IF halberd && comp_ir_wc && type=1 BEGIN // melee
     READ_SHORT off+0x1c damtype
-    PATCH_IF damtype=1 BEGIN // piercing
-      unid_old_speed=sf_halberd
-      delta_speed=unid_old_speed - speed
-    END
     PATCH_IF damtype=3 BEGIN // slashing
-      unid_old_speed=sf_halberd + ( (wc=0)? 0 : 1 )
-      delta_speed=unid_old_speed - speed
-      update_line=2
+      iden_delta_speed +=1
+      reupdate=1
+      update_line = 2
     END
-  SET patched=1 END
-END
-
-/////////////////////////////////////////////////////
-// Actual patching
-
-COPY_EXISTING_REGEXP GLOB ~^.+\.itm$~ override PATCH_IF SOURCE_SIZE>0x71 BEGIN
-  PATCH_INCLUDE ~item_rev/lib/identify_weapon.tpa~
-  SPRINT macro ~sf~
-
-  extra=0
-  PATCH_IF ~%SOURCE_RES%~ STRING_EQUAL_CASE ~blun28~ || ~%SOURCE_RES%~ STRING_EQUAL_CASE ~blun29~ BEGIN // Storm Star
-    extra=2
   END
 
-  PATCH_MATCH 1 WITH
+  /////////////////////////
+  //    weird things     //
+  /////////////////////////
 
-    dagger
-    BEGIN
-      SPRINT weapon ~dagger~ LPM wc_patch
-    END
-
-    short_sword && (~%SOURCE_RES%~ STRING_COMPARE_CASE ~sw1h25~) // not Kundane +1
-    BEGIN
-      SPRINT weapon ~short_sword~ LPM wc_patch
-    END
+  // vanilla bugs that i'm too lazy to deal with in 'fixpack.tpa'
+  // Demi had kindly fixed most of them within IR's main component :bow: , so I see no reason to bother 
+  //
+  // also items that intentionally don't match their descriptions
 
-    long_sword && (~%SOURCE_RES%~ STRING_COMPARE_CASE ~sw1h77~) // not Answerer
-    BEGIN
-      SPRINT weapon ~long_sword~ LPM wc_patch
-    END
+  PATCH_IF comp_ir=0 BEGIN
 
-    scimitar
+    // Throwing Axe +2, Mace, Werebane, Spear, Sword of Balduran, The Vampire's Revenge,
+    // Spider's Bane - bugged and seemingly unused, don't touch
+    PATCH_IF
+      ~%SOURCE_RES%~ STRING_EQUAL ~ax1h06~ ||
+      ~%SOURCE_RES%~ STRING_EQUAL ~blun11~ ||
+      ~%SOURCE_RES%~ STRING_EQUAL ~dagg09~ ||
+      ~%SOURCE_RES%~ STRING_EQUAL ~sper04~ ||
+      ~%SOURCE_RES%~ STRING_EQUAL ~sw1h18~ ||
+      ~%SOURCE_RES%~ STRING_EQUAL ~sw1h19~ ||
+      ~%SOURCE_RES%~ STRING_EQUAL ~sw2h13~
     BEGIN
-      SPRINT weapon ~scimitar~ LPM wc_patch
+      patched=0
     END
 
-    battleaxe
+    // Flasher Mates, Bruenor's Axe, Flame Blade, Flasher Launcher - bugged yet used, but don't touch for now
+    PATCH_IF
+      ~%SOURCE_RES%~ STRING_EQUAL ~bolt07~   ||
+      ~%SOURCE_RES%~ STRING_EQUAL ~bruenaxe~ ||
+      ~%SOURCE_RES%~ STRING_EQUAL ~sw1h11~   ||
+      ~%SOURCE_RES%~ STRING_EQUAL ~xbow12~
     BEGIN
-      SPRINT weapon ~battleaxe~ LPM wc_patch
+      patched=0
     END
 
-    bastard_sword
-    BEGIN
-      SPRINT weapon ~bastard_sword~ LPM wc_patch
-    END
+  END ELSE BEGIN // comp_ir=0
 
-    ninjato
+    // Spear, Flame Blade - still bugged and seemingly unused, don't touch
+    PATCH_IF
+      ~%SOURCE_RES%~ STRING_EQUAL ~sper04~ ||
+      ~%SOURCE_RES%~ STRING_EQUAL ~sw1h11~
     BEGIN
-      SPRINT weapon ~ninjato~ LPM wc_patch
+      patched=0
     END
 
-    wakizashi
-    BEGIN
-      SPRINT weapon ~wakizashi~ LPM wc_patch
-    END
+  END // comp_ir=0
 
-    katana
-    BEGIN
-      SPRINT weapon ~katana~ LPM wc_patch
-    END
+  /////////////////////////
+  //  "hardcoded" items  //
+  /////////////////////////
 
-    club
-    BEGIN
-      SPRINT weapon ~club~ LPM wc_patch
-    END
+  // hardcode special items
 
-    hammer
-    BEGIN
-      SPRINT weapon ~hammer~ LPM wc_patch
-    END
-
-    mace
-    BEGIN
-      SPRINT weapon ~mace~ LPM wc_patch
-    END
-
-    morning_star
-    BEGIN
-      SPRINT weapon ~morning_star~ LPM wc_patch
-    END
-
-    flail
-    BEGIN
-      SPRINT weapon ~flail~ LPM wc_patch
-    END
+  // Staff Spear +2 - use spear's speed
+  PATCH_IF
+    ~%SOURCE_RES%~ STRING_EQUAL ~staf07~
+  BEGIN
+    iden_delta_speed  = old_spear_speed - speed
+  END
 
-    staff
-    BEGIN
-      SPRINT weapon ~staff~ LPM wc_patch
-    END
+  // Moonblade - use longsword's stats
+  PATCH_IF
+    ~%SOURCE_RES%~ STRING_EQUAL ~sw1h13~
+  BEGIN
+    iden_delta_speed  = old_long_sword_speed - speed
+  END
 
-    spear
-    BEGIN
-      SPRINT weapon ~spear~ LPM wc_patch
-    END
+  // Kundane +2, The Answerer +4 - they have fixed values, don't touch
+  PATCH_IF
+    ~%SOURCE_RES%~ STRING_EQUAL ~sw1h25~ ||
+    ~%SOURCE_RES%~ STRING_EQUAL ~sw1h77~
+  BEGIN
+    iden_delta_speed = 0
+  END
 
-    halberd
-    BEGIN
-      SPRINT macro ~halberd~
-      SPRINT weapon ~halberd~ LPM wc_patch
-    END
+  // IR-specific
+  PATCH_IF comp_ir BEGIN
 
-    twohanded_sword
+    // Storm Star +3, Storm Star +4 - they have +2 speed bonus
+    PATCH_IF
+      ~%SOURCE_RES%~ STRING_EQUAL ~blun28~ ||
+      ~%SOURCE_RES%~ STRING_EQUAL ~blun29~
     BEGIN
-      SPRINT weapon ~twohanded_sword~ LPM wc_patch
+      iden_delta_speed -=2
     END
 
-    dart
+    // Blade of Sharpness +1 - use short sword's speed
+    PATCH_IF
+      ~%SOURCE_RES%~ STRING_EQUAL ~wa2s1h~
     BEGIN
-      SPRINT weapon ~dart~ LPM wc_patch
+      iden_delta_speed = old_short_sword_speed - speed
     END
 
-    handaxe
-    BEGIN
-      SPRINT weapon ~handaxe~ LPM wc_patch
-    END
+  END
 
-    sling
-    BEGIN
-      SPRINT weapon ~sling~ LPM wc_patch
-    END
+  /////////////////////////
 
-    short_bow
-    BEGIN
-      SPRINT weapon ~short_bow~ LPM wc_patch
-    END
+  END // type=1 || type=2 || type=4
 
-    comp_long_bow
-    BEGIN
-      SPRINT weapon ~comp_long_bow~ LPM wc_patch
-    END
-
-    long_bow
-    BEGIN
-      SPRINT weapon ~long_bow~ LPM wc_patch
-    END
+END
 
-    light_xbow
-    BEGIN
-      SPRINT weapon ~light_xbow~ LPM wc_patch
-    END
+/////////////////////////////////////////////////////
+//  Patching
 
-    heavy_xbow
-    BEGIN
-      SPRINT weapon ~heavy_xbow~ LPM wc_patch
-    END
+COPY_EXISTING_REGEXP GLOB ~^.+\.itm$~ override PATCH_IF SOURCE_SIZE>0x71 BEGIN
+  LPM identify_item
+  PATCH_IF is_weapon BEGIN
+    SPRINT macro special
+    LPM wc_patch
+  END
+END BUT_ONLY
 
-  DEFAULT  END
+/////////////////////////////////////////////////////
 
-END BUT_ONLY
