--- item_rev\components\items_of_protection.tpa	Sat Nov 22 21:44:08 2008
+++ C:\BWP Patchstudio\patched files\item_rev\components\items_of_protection.tpa	Mon May 28 22:25:02 2012
@@ -1,25 +1,20 @@
-INCLUDE ~item_rev/lib/macros.tpa~
 
 COPY_EXISTING ~itemexcl.2da~ ~override~
-  COUNT_2DA_ROWS 2 num_rows
+  READ_2DA_ENTRIES_NOW num_rows 2
   FOR (i = 1; i < num_rows; i += 1) BEGIN
-    READ_2DA_ENTRY %i% 0 2 current_item
+    READ_2DA_ENTRY_FORMER num_rows i 0 current_item
     SET isArmor = 0
-    INNER_ACTION BEGIN
-      ACTION_IF (FILE_EXISTS_IN_GAME ~%current_item%.itm~) BEGIN
-        COPY_EXISTING ~%current_item%.itm~ ~override~
-          READ_SHORT 0x1c item_type
-          PATCH_IF (%item_type% == 0x02) BEGIN
-            SET isArmor = 1
-          END
-          BUT_ONLY_IF_IT_CHANGES
+    PATCH_IF (FILE_EXISTS_IN_GAME ~%current_item%.itm~) BEGIN
+      INNER_PATCH_FILE ~%current_item%.itm~ BEGIN
+        READ_SHORT 0x1c item_type
+        PATCH_IF (%item_type% == 0x02) BEGIN
+          SET isArmor = 1
+        END
       END
     END
     PATCH_IF (%isArmor% == 1) BEGIN
-      REPLACE_TEXTUALLY ~[%lnl%%mnl%%wnl%][ %tab%]*%current_item%[ %tab%].*~
+      REPLACE_TEXTUALLY ~[%LNL%%MNL%%WNL%][ %TAB%]*%current_item%[ %TAB%].*~
         ~~
-      SET num_rows -= 1
-      SET i -= 1
     END
   END
   BUT_ONLY_IF_IT_CHANGES
