--- item_rev\languages\english\mapping of strings.txt	Thu Jul 28 17:05:54 2011
+++ C:\BWP Patchstudio\patched files\item_rev\languages\english\mapping of strings.txt	Sat Dec 17 22:26:18 2011
@@ -3,12 +3,14 @@
 
 [100;999] - item upgrades
 
-[1000;2999] - Item Revisions (the core component)
+[1000;2999], 21651 - Item Revisions (the core component)
 
 [3000;3999] - Potion Revisions
 
 [4000;4999] - tooltips
 
+[5000;5999] - muliclasses
+
 [9000;9999] - installation warnings
 
 [50000;50099] - Masterwork Weapons
@@ -22,5 +24,5 @@
 
 
 free space:
-[5000;8999], [10000;49999], [50100;99999], [110000;199999], [210000;+]
+[6000;8999], [10000;21650], [21652;49999], [50100;99999], [110000;199999], [210000;+]
 
