--- item_rev\languages\english\potion_descriptions.tra	Tue Jan 18 06:42:38 2011
+++ C:\BWP Patchstudio\patched files\item_rev\languages\english\potion_descriptions.tra	Sat Dec 17 22:26:18 2011
@@ -145,7 +145,7 @@
 Usable By:
  All~
 @3058=~Potion of Insulation~
-@3059=~This potion bestows upon the person drinking it 50% resistance from all electrical-based attacks, whether magical or non-magical in origin. This is on top of any other resistance the creature might have, but multiple uses of this potion are not cumulative. The effect lasts for 5 turns, and cannot be dispelled
+@3059=~This potion bestows upon the person drinking it 50% resistance from all electrical-based attacks, whether magical or non-magical in origin. This is on top of any other resistance the creature might have, but multiple uses of this potion are not cumulative. The effect lasts for 5 turns, and cannot be dispelled.
 
 Not Usable By:
  Wizard Slayer~
