--- item_rev\languages\english\proficiencies.tra	Fri Nov 21 16:01:34 2008
+++ C:\BWP Patchstudio\patched files\item_rev\languages\english\proficiencies.tra	Sat Dec 17 22:26:18 2011
@@ -10,7 +10,7 @@
 @100002 = ~Katana~
 @100003 = ~KATANA/WAKIZASHI:  Katanas and wakizashis are the finest examples of the swordsmith's craft, and require great care to use.  As skill develops, the wielder and blade become a single being, dedicated completely to the art of warfare.  These weapons are hard to find outside of Kara-Tur, so care must be taken when considering this proficiency.  After all, skill with katanas and wakizashis is only helpful if you can find them.
 
-WARNING:  Magical katanas are very rare in Baldur's Gate 2!
+WARNING:  Magical katanas and wakizashis are very rare in Baldur's Gate 2!
 
 <DOTS1>
 <DOTS2>
