--- item_rev\components\thieving_armor.tpa	Wed Aug 03 18:12:30 2011
+++ C:\BWP Patchstudio\patched files\item_rev\components\thieving_armor.tpa	Mon May 28 22:25:02 2012
@@ -1,6 +1,10 @@
 INCLUDE ~item_rev/lib/descriptions.tpa~
 INCLUDE ~item_rev/lib/armor_list.tpa~
+INCLUDE ~item_rev/lib/common_shield_list.tpa~
+INCLUDE ~item_rev/lib/multiclass_standard.tpa~
 
+//////////////////////////////////////////////
+//  descriptions
 
 DEFINE_PATCH_MACRO ~cleanup~ BEGIN
   SPRINT t1 @100325
@@ -24,11 +28,14 @@
 END
 
 
+//////////////////////////////////////////////
+//  reading table
+
 COPY - ~item_rev/lib/armor_penalty_table.txt~ override
-  FOR (i=1;i<11;i+=1) BEGIN
-    READ_2DA_ENTRY i 3 7 ~value_%i%~
-  END
+  READ_2DA_ENTRIES_NOW rn 2
 
+//////////////////////////////////////////////
+//  patching armors
 
 ACTION_PHP_EACH armor AS ind => res BEGIN
   ACTION_IF FILE_EXISTS_IN_GAME ~%ind%.itm~ BEGIN
@@ -64,10 +71,10 @@
         subst = ( res >> 3 ) & 3 // 1=mithral, 2=scale
         elven = ( res >> 5 ) & 1
 
-        value= EVAL ~value_%armor%~
-        PATCH_IF subst=2 BEGIN value+= EVAL ~value_8~ END // scale
-        PATCH_IF subst=1 BEGIN value+= EVAL ~value_9~ END // mithral
-        PATCH_IF elven BEGIN value+= EVAL ~value_10~ END
+        READ_2DA_ENTRY_FORMER rn armor 1 value
+        PATCH_IF subst=2 BEGIN value+= EVAL ~rn_8_1~ END // scale
+        PATCH_IF subst=1 BEGIN value+= EVAL ~rn_9_1~ END // mithral
+        PATCH_IF elven BEGIN value+= EVAL ~rn_10_1~ END
         value= value>0 ? value : 0
 
         PATCH_IF value BEGIN
@@ -81,3 +88,75 @@
     END BUT_ONLY
   END
 END
+
+//////////////////////////////////////////////
+//  patching shields
+
+ACTION_IF option=2 BEGIN
+
+LAM shields
+
+COPY_EXISTING_REGEXP ~^.+\.itm$~ override PATCH_IF SOURCE_SIZE>0x71 BEGIN
+  READ_SHORT 0x1c item
+  READ_ASCII 0x22 anim (2)
+
+  type=0
+  PATCH_IF item=190 || (item=0xc && (~%anim%~ STRING_EQUAL_CASE ~d1~) ) BEGIN // buckler
+    LPM cleanup
+    type=11
+    WRITE_SHORT 0x1c 0xc
+  END
+  PATCH_IF item=191 || (item=0xc && (~%anim%~ STRING_EQUAL_CASE ~d2~) ) BEGIN // light
+    LPM cleanup
+    type=12
+    WRITE_SHORT 0x1c 0xc
+  END
+  PATCH_IF item=192 || (item=0xc && (~%anim%~ STRING_EQUAL_CASE ~d3~) ) BEGIN // heavy
+    LPM cleanup
+    type=13
+    WRITE_SHORT 0x1c 0xc
+  END
+  PATCH_IF item=194 || (item=0xc && (~%anim%~ STRING_EQUAL_CASE ~d4~) ) BEGIN // tower
+    LPM cleanup
+    type=14
+    WRITE_SHORT 0x1c 0xc
+  END
+
+  PATCH_IF type BEGIN
+    READ_2DA_ENTRY_FORMER rn type 1 value
+    PATCH_IF value>0 BEGIN
+      LPF ADD_ITEM_EQEFFECT INT_VAR opcode=59 target=1 timing=2 parameter1=(0 - value) END
+      LPF ADD_ITEM_EQEFFECT INT_VAR opcode=275 target=1 timing=2 parameter1=(0 - value) END
+      SPRINT text_update stealth
+      LPM update_item_descriptions
+    END
+  END
+
+END BUT_ONLY
+
+END
+
+//////////////////////////////////////////////
+//  class descriptions
+
+OUTER_PATCH ~~ BEGIN
+  GET_STRREF 9572 ft
+  INNER_PATCH_SAVE ft ~%ft%~ BEGIN
+    REPLACE_TEXTUALLY ~%van_thv%~ ~%new_thv%~
+    REPLACE_TEXTUALLY ~%gtu_thv%~ ~%new_thv%~
+  END
+  GET_STRREF 9576 fmt
+  INNER_PATCH_SAVE fmt ~%fmt%~ BEGIN
+    REPLACE_TEXTUALLY ~%van_thv%~ ~%new_thv%~
+    REPLACE_TEXTUALLY ~%gtu_thv%~ ~%new_thv%~
+  END
+  GET_STRREF 9578 tc
+  INNER_PATCH_SAVE tc ~%tc%~ BEGIN
+    REPLACE_TEXTUALLY ~%van_thv%~ ~%new_thv%~
+    REPLACE_TEXTUALLY ~%gtu_thv%~ ~%new_thv%~
+  END
+END
+STRING_SET 9572 ~%ft%~
+STRING_SET 9576 ~%fmt%~
+STRING_SET 9578 ~%tc%~
+
