--- setup-ashesofembers.tp2	Mon Jan 21 12:59:42 2008
+++ C:\BWP Patchstudio\patched files\setup-ashesofembers.tp2	Fri Sep 27 21:47:12 2013
@@ -1,5 +1,6 @@
 BACKUP ~AshesofEmbers\Backup~
 AUTHOR ~ghreyfain@yahoo.ca~
+VERSION ~v27 BWP Fix~
 
 ALWAYS
 PRINT @600
@@ -577,9 +578,7 @@
   SAY UNIDENTIFIED_DESC @24
 
 COPY_EXISTING ~CEFGT01.cre~ ~override\CEFGT01.cre~
-  WRITE_ASCII 0x40c ~Bola04
-
-~
+  WRITE_ASCII 0x40c ~Bola04~ #8
   WRITE_SHORT 0x434 0x0002
   WRITE_SHORT 0x456 0xffff
 
@@ -2149,7 +2148,7 @@
   SAY DESC @523
 
 COPY_EXISTING ~slmage1.cre~ ~override\slmage1.cre~
-  WRITE_ASCII 0x4f8 ~scal04~
+  WRITE_ASCII 0x4f8 ~scal04~ #8
   WRITE_BYTE 0x4fe 0
   WRITE_BYTE 0x4ff 0
 WRITE_SHORT 0x50e 0x0003
@@ -8718,27 +8717,27 @@
 SET_2DA_ENTRY 9  51 5 ~1~ // monk = 51
 SET_2DA_ENTRY 10 51 5 ~1~
 SET_2DA_ENTRY 11 51 5 ~1~
-SET_2DA_ENTRY 12 51 5 ~0~ // two-handed
-SET_2DA_ENTRY 13 51 5 ~1~
+SET_2DA_ENTRY 12 51 5 ~1~ 
+SET_2DA_ENTRY 13 51 5 ~0~ // two-handed sword
 SET_2DA_ENTRY 14 51 5 ~1~
 SET_2DA_ENTRY 15 51 5 ~1~
 SET_2DA_ENTRY 16 51 5 ~1~
 SET_2DA_ENTRY 17 51 5 ~1~
-SET_2DA_ENTRY 18 51 5 ~0~ // 2-h
-SET_2DA_ENTRY 19 51 5 ~0~ // 2-h
-SET_2DA_ENTRY 20 51 5 ~1~
+SET_2DA_ENTRY 18 51 5 ~1~ 
+SET_2DA_ENTRY 19 51 5 ~0~ // spear
+SET_2DA_ENTRY 20 51 5 ~0~ // halberd
 SET_2DA_ENTRY 21 51 5 ~1~
-SET_2DA_ENTRY 22 51 5 ~0~ // 2-h
-SET_2DA_ENTRY 23 51 5 ~0~
-SET_2DA_ENTRY 24 51 5 ~0~
-SET_2DA_ENTRY 25 51 5 ~0~
-SET_2DA_ENTRY 26 51 5 ~1~
+SET_2DA_ENTRY 22 51 5 ~1~ 
+SET_2DA_ENTRY 23 51 5 ~0~ // quarterstaff
+SET_2DA_ENTRY 24 51 5 ~0~ // crossbow
+SET_2DA_ENTRY 25 51 5 ~0~ // longbow
+SET_2DA_ENTRY 26 51 5 ~0~ // shortbow
 SET_2DA_ENTRY 27 51 5 ~1~
-SET_2DA_ENTRY 28 51 5 ~0~ // 2-handed
-SET_2DA_ENTRY 29 51 5 ~1~ // swrod&shield
-SET_2DA_ENTRY 30 51 5 ~1~ // single-weapon
-SET_2DA_ENTRY 31 51 5 ~1~ // dual-wield
-SET_2DA_ENTRY 32 51 5 ~1~ // dual-wield
+SET_2DA_ENTRY 28 51 5 ~1~ 
+SET_2DA_ENTRY 29 51 5 ~0~ // 2-handed
+SET_2DA_ENTRY 30 51 5 ~0~ // sword and shield
+SET_2DA_ENTRY 31 51 5 ~1~ // single-weapon
+SET_2DA_ENTRY 32 51 5 ~0~ // dual-wield
 
 /*                        														13						16		17	18		19		20		21	   22		23	24	25		26	27	28		29	30		31	     32		33	34		35	     36      37		38	39		40		41	42	43	44		45		46		47	48	49  	  50	  51	     52		53
               ID      NAME_REF DESC_REF    MAGE    FIGHTER CLERIC  THIEF   BARD    PALADIN DRUID   RANGER  FIGHTER_MAGE    FIGHTER_CLERIC  FIGHTER_THIEF   FIGHTER_MAGE_THIEF  MAGE_THIEF  CLERIC_MAGE CLERIC_THIEF    FIGHTER_DRUID   FIGHTER_MAGE_CLERIC CLERIC_RANGER   ABJURER CONJURER    DIVINER ENCHANTER   ILLUSIONIST INVOKER NECROMANCER TRANSMUTER  BERSERKER   WIZARD_SLAYER   KENSAI  CAVALIER    INQUISITOR  UNDEAD_HUNTER   FERALAN STALKER BEASTMASTER ASSASIN BOUNTY_HUNTER   SWASHBUCKLER    BLADE   JESTER  SKALD   TOTEMIC_DRUID   SHAPESHIFTER    BEAST_FRIEND    TALOS   HELM    LATHANDER  MONK  BARBARIAN  WILDMAGE
@@ -10169,23 +10168,23 @@
    READ_ASCII ("%items_offset%" + ("%items_num%" * 0x14)) "itemname"
     PATCH_IF ("%itemname%" STR_CMP "bull01" = 0 OR "%itemname%" STR_CMP "BULL01" = 0) //1819047266)
      BEGIN
-      WRITE_ASCII ("%items_offset%" + ("%items_num%" * 0x14)) ~bolt01~
+      WRITE_ASCII ("%items_offset%" + ("%items_num%" * 0x14)) ~bolt01~ #8
      END
     PATCH_IF ("%itemname%" STR_CMP "bull02" = 0 OR "%itemname%" STR_CMP "BULL02" = 0) //1819047266)
      BEGIN
-      WRITE_ASCII ("%items_offset%" + ("%items_num%" * 0x14)) ~bolt02~
+      WRITE_ASCII ("%items_offset%" + ("%items_num%" * 0x14)) ~bolt02~ #8
      END
     PATCH_IF ("%itemname%" STR_CMP "bull03" = 0 OR "%itemname%" STR_CMP "BULL03" = 0) //1819047266)
      BEGIN
-      WRITE_ASCII ("%items_offset%" + ("%items_num%" * 0x14)) ~bolt03~
+      WRITE_ASCII ("%items_offset%" + ("%items_num%" * 0x14)) ~bolt03~ #8
      END
     PATCH_IF ("%itemname%" STR_CMP "blun33" = 0 OR "%itemname%" STR_CMP "BLUN33" = 0) // 1853189218)
      BEGIN
-      WRITE_ASCII ("%items_offset%" + ("%items_num%" * 0x14)) ~blun32~
+      WRITE_ASCII ("%items_offset%" + ("%items_num%" * 0x14)) ~blun32~ #8
      END
     PATCH_IF ("%itemname%" STR_CMP "slng04" = 0 OR "%itemname%" STR_CMP "SLNG04" = 0) // 1735289971)
      BEGIN
-      WRITE_ASCII ("%items_offset%" + ("%items_num%" * 0x14)) ~xbow09~
+      WRITE_ASCII ("%items_offset%" + ("%items_num%" * 0x14)) ~xbow09~ #8
      END
  END
 BUT_ONLY_IF_IT_CHANGES
@@ -10238,7 +10237,7 @@
    READ_ASCII ("%items_offset%" + ("%items_num%" * 0x14)) "itemname"
     PATCH_IF ("%itemname%" STR_CMP "xbow04" = 0 OR "%itemname%" STR_CMP "XBOW04" = 0) //1819047266)
      BEGIN
-      WRITE_ASCII ("%items_offset%" + ("%items_num%" * 0x14)) ~sper01~
+      WRITE_ASCII ("%items_offset%" + ("%items_num%" * 0x14)) ~sper01~ #8
      END
  END
 BUT_ONLY_IF_IT_CHANGES
@@ -10281,7 +10280,7 @@
    READ_ASCII ("%items_offset%" + ("%items_num%" * 0x14)) "itemname"
     PATCH_IF ("%itemname%" STR_CMP "xbow04" = 0 OR "%itemname%" STR_CMP "XBOW04" = 0) //1819047266)
      BEGIN
-      WRITE_ASCII ("%items_offset%" + ("%items_num%" * 0x14)) ~sper01~
+      WRITE_ASCII ("%items_offset%" + ("%items_num%" * 0x14)) ~sper01~ #8
      END
  END
 BUT_ONLY_IF_IT_CHANGES
@@ -10324,7 +10323,7 @@
    READ_ASCII ("%items_offset%" + ("%items_num%" * 0x14)) "itemname"
     PATCH_IF ("%itemname%" STR_CMP "XBOW04" = 0 OR "%itemname%" STR_CMP "xbow04" = 0) //1819047266)
      BEGIN
-      WRITE_ASCII ("%items_offset%" + ("%items_num%" * 0x14)) ~sper01~
+      WRITE_ASCII ("%items_offset%" + ("%items_num%" * 0x14)) ~sper01~ #8
      END
  END
 BUT_ONLY_IF_IT_CHANGES
@@ -10378,7 +10377,7 @@
    READ_ASCII ("%items_offset%" + ("%items_num%" * 0x14)) "itemname"
     PATCH_IF ("%itemname%" STR_CMP "xbow04" = 0 OR "%itemname%" STR_CMP "XBOW04" = 0) //1819047266)
      BEGIN
-      WRITE_ASCII ("%items_offset%" + ("%items_num%" * 0x14)) ~sper01~
+      WRITE_ASCII ("%items_offset%" + ("%items_num%" * 0x14)) ~sper01~ #8
      END
  END
 BUT_ONLY_IF_IT_CHANGES
@@ -10426,7 +10425,7 @@
    READ_ASCII ("%items_offset%" + ("%items_num%" * 0x14)) "itemname"
     PATCH_IF ("%itemname%" STR_CMP "xbow07" = 0 OR "%itemname%" STR_CMP "XBOW07" = 0) //1819047266)
      BEGIN
-      WRITE_ASCII ("%items_offset%" + ("%items_num%" * 0x14)) ~sper05~
+      WRITE_ASCII ("%items_offset%" + ("%items_num%" * 0x14)) ~sper05~ #8
      END
  END
 BUT_ONLY_IF_IT_CHANGES
