--- banterpack/dialogue/banter-soa.d	2008-05-18 09:10:40.000000000 -0400
+++ Fixpack/banterpack/dialogue/banter-soa.d	2016-11-26 02:07:14.339954300 -0500
@@ -1171,7 +1171,8 @@
 REPLACE BVICONI
 
 IF
-~Gender(Player1,FEMALE)
+~%CLEVER_EDITORIAL_COMMENT%
+Gender(Player1,FEMALE)
 !StateCheck(Player1,STATE_SLEEPING)
 Global("BViconia12","LOCALS",0)~ THEN BEGIN 27
   SAY #8657 /* ~What brought you to the Life?  Surely some measure of madness.~ [VICONI66] */
