--- banterpack/scripts/keldorn.baf	2003-09-21 09:24:02.000000000 -0400
+++ Fixpack/banterpack/scripts/keldorn.baf	2016-11-21 16:27:17.038476900 -0500
@@ -10,7 +10,7 @@
 IF
 InParty(Myself)
 InParty("Minsc")
-Global("Chapter","GLOBAL",6)
+Global("Chapter","GLOBAL",%bg2_chapter_6%)
 AreaCheck("AR0808")
 Global("FWBanterMinscVamp","LOCALS",0)
 THEN
@@ -24,7 +24,7 @@
 InParty("Minsc")
 See("Minsc")
 !Dead("Minsc")
-Global("Chapter","GLOBAL",6)
+Global("Chapter","GLOBAL",%bg2_chapter_6%)
 RealGlobalTimerExpired("FWKeldornMinscVamp","LOCALS")
 CombatCounter(0)
 Global("FWBanterMinscVamp","LOCALS",1)
