--- setup-banterpack.tp2	2013-11-15 11:57:46.000000000 -0500
+++ Fixpack/setup-banterpack.tp2	2016-11-22 18:42:41.002966800 -0500
@@ -1,8 +1,22 @@
 BACKUP ~banterpack\Backup~
 AUTHOR ~jcompton@pocketplane.net~
-VERSION ~V14~
 
-ALWAYS AT_INTERACTIVE_EXIT ~VIEW banterpack\banter-readme.txt~ END
+VERSION ~v14 BWP Fix~
+
+ALWAYS
+  ACTION_IF GAME_IS ~eet~ BEGIN
+    OUTER_SET bg2_chapter = 12
+  END ELSE BEGIN
+    OUTER_SET bg2_chapter = 0
+  END
+  OUTER_FOR (i=1; i<=10; i=i+1) BEGIN
+    OUTER_SET bg2_chapter = bg2_chapter + 1
+    OUTER_SPRINT name_source ~bg2_chapter_%i%~
+    OUTER_SET EVAL ~%name_source%~ = bg2_chapter
+  END
+
+  AT_INTERACTIVE_EXIT ~VIEW banterpack\banter-readme.txt~
+END
 
 LANGUAGE ~English~
          ~english~
@@ -55,7 +69,7 @@
     OUTER_SPRINT ~CLEVER_EDITORIAL_COMMENT~ ~~
 END
 
-ACTION_IF GAME_IS BG2EE THEN BEGIN
+ACTION_IF GAME_IS ~bg2ee eet~ THEN BEGIN
   LOAD_TRA ~banterpack/%language%/setup-ee.tra~
 END
 
@@ -82,7 +96,7 @@
 EXTEND_TOP ~haerdali.bcs~ ~banterpack/scripts/haerdali.baf~
 EXTEND_TOP ~mazzy.bcs~ ~banterpack/scripts/mazzy.baf~
 EXTEND_TOP ~jan.bcs~ ~banterpack/scripts/jan.baf~
-EXTEND_TOP ~keldorn.bcs~ ~banterpack/scripts/keldorn.baf~
+EXTEND_TOP ~keldorn.bcs~ ~banterpack/scripts/keldorn.baf~ EVALUATE_BUFFER
 EXTEND_BOTTOM ~vicond.bcs~ ~banterpack/scripts/viconiad.baf~
 EXTEND_TOP ~viconia.bcs~ ~banterpack/scripts/viconia.baf~
 
@@ -119,7 +133,7 @@
 
 
 BEGIN @14
-REQUIRE_PREDICATE FILE_EXISTS_IN_GAME ~mel01.cre~ @15
+REQUIRE_PREDICATE GAME_INCLUDES ~tob~ @15
 
 EXTEND_TOP ~vico25.bcs~ ~banterpack/scripts/vico25.baf~
 
