--- sovereign\Scripts\Crsarfgt.baf	Sun Dec 23 23:54:57 2007
+++ C:\BWP Patchstudio\patched files\sovereign\Scripts\Crsarfgt.baf	Thu Apr 01 16:36:03 2010
@@ -16,7 +16,7 @@
 IF
 	Global("Stoneskin","LOCALS",1)
 	CheckStatLT(Myself,1,STONESKINS)
-	!Dead("yself)")
+	!StateCheck(Myself,STATE_REALLY_DEAD)
 THEN
 	RESPONSE #100
 		ReallyForceSpell(Myself,WIZARD_STONE_SKIN)
@@ -27,7 +27,7 @@
 	See(NearestEnemyOf(Myself))
 	TookDamage()
 	Global("Prep2","LOCALS",0)
-	!Dead("yself)")
+	!StateCheck(Myself,STATE_REALLY_DEAD)
 THEN
 	RESPONSE #100
 		ReallyForceSpell(Myself,WIZARD_IMPROVED_MANTLE)
