--- RE/Laran/Dialogue/B!LARAN2.D	2007-08-26 04:48:46.000000000 -0400
+++ Fixpack/RE/Laran/Dialogue/B!LARAN2.D	2016-11-21 22:55:14.713945100 -0500
@@ -1,7 +1,7 @@
 /* Kulyok: separating two encounters into two different files, so that SoA only players may have a go, too */
 /* Saradush - encounter */
 CHAIN
-IF ~Global("B!LaranFlirt","GLOBAL",1) GlobalGT("Chapter","GLOBAL",7)~ THEN ~B!LARAN~ 2Laran1
+IF ~Global("B!LaranFlirt","GLOBAL",1) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN ~B!LARAN~ 2Laran1
 @0 
 DO ~SetGlobal("B!LaranFlirt","GLOBAL",2)~
 == ~ANOME25J~ IF ~Global("AnomenRomanceActive","GLOBAL",2) GlobalGT("B!RE_AnoComplain","GLOBAL",0) InParty("Anomen") InMyArea("Anomen") !StateCheck("Anomen",CD_STATE_NOTVALID)~ THEN @1
