--- RE/Ribald/dialogue/RE_Riba.d	2007-08-26 04:49:20.000000000 -0400
+++ Fixpack/RE/Ribald/dialogue/RE_Riba.d	2016-11-21 22:55:11.416177900 -0500
@@ -943,7 +943,7 @@
 SAY @271
 IF ~~ THEN REPLY @272 GOTO InYourArms
 IF ~~ THEN REPLY @273 GOTO Bhaal
-IF ~GlobalGT("Chapter","GLOBAL",5)~ THEN REPLY @274 GOTO Underdark
+IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)~ THEN REPLY @274 GOTO Underdark
 IF ~~ THEN REPLY @275 GOTO SayMyName
 IF ~~ THEN REPLY @276 GOTO TiredOfThisGame
 END
@@ -1014,13 +1014,13 @@
 SAY @303
 = @304
 IF ~~ THEN REPLY @305 GOTO MyHeritage
-IF ~GlobalGT("Chapter","GLOBAL",5)~ THEN REPLY @274 GOTO Underdark
+IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)~ THEN REPLY @274 GOTO Underdark
 IF ~~ THEN REPLY @289 GOTO TitForTat
 END
 
 IF ~~ THEN BEGIN MyHeritage
 SAY @306
-IF ~GlobalGT("Chapter","GLOBAL",5)~ THEN REPLY @307 GOTO GiantLobsterGirl
+IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)~ THEN REPLY @307 GOTO GiantLobsterGirl
 IF ~~ THEN REPLY @308 GOTO NotRidiculous
 IF ~~ THEN REPLY @309 GOTO IKnowIt
 IF ~~ THEN REPLY @310 GOTO TitForTat
@@ -1442,7 +1442,7 @@
 IF WEIGHT #-1
 ~Global("RE_RibaldSpecialTalk","GLOBAL",14)
 Global("RE_RibaldDump","GLOBAL",0)
-GlobalGT("Chapter","GLOBAL",5)~ THEN BEGIN FinalTalk
+GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)~ THEN BEGIN FinalTalk
 SAY @421
 IF ~~ THEN REPLY @422 DO ~IncrementGlobal("RE_RibaldSpecialTalk","GLOBAL",1) RealSetGlobalTimer("RE_RibaldTimer","GLOBAL",1800)~ GOTO JustArrived
 IF ~~ THEN REPLY @423 DO ~IncrementGlobal("RE_RibaldSpecialTalk","GLOBAL",1) RealSetGlobalTimer("RE_RibaldTimer","GLOBAL",1800)~ GOTO EverComeBack
