--- RE/Ryan/RE_Ryan.d	2007-08-26 04:49:20.000000000 -0400
+++ Fixpack/RE/Ryan/RE_Ryan.d	2016-11-21 22:55:00.242559400 -0500
@@ -3,10 +3,10 @@
 + ~IsGabber(Player1) Gender(Player1,FEMALE) Global("PaladinOrder","GLOBAL",1) Global("RE_RyanTrawl2","GLOBAL",0)~ + @1 DO ~SetGlobal("RE_RyanTrawl2","GLOBAL",1)~ EXTERN KAYL2 R2
 + ~IsGabber(Player1) Gender(Player1,FEMALE) Global("PaladinOrder","GLOBAL",1) Global("RE_RyanTrawl3","GLOBAL",0)~ + @2 DO ~SetGlobal("RE_RyanTrawl3","GLOBAL",1)~ EXTERN KAYL2 R3
 + ~IsGabber(Player1) Gender(Player1,FEMALE) Global("PaladinOrder","GLOBAL",1) OR(3) Global("RE_RyanTrawl1","GLOBAL",1) Global("RE_RyanTrawl2","GLOBAL",1) Global("RE_RyanTrawl3","GLOBAL",1) Global("RE_RyanTrawl4","GLOBAL",0)~ + @3 DO ~SetGlobal("RE_RyanTrawl4","GLOBAL",1)~ EXTERN KAYL2 R4
-+ ~IsGabber(Player1) Gender(Player1,FEMALE) Global("PaladinOrder","GLOBAL",1) !Dead("C6Bodhi") Global("Chapter","GLOBAL",6) Global("RE_RyanTrawl5","GLOBAL",0)~ + @4 DO ~SetGlobal("RE_RyanTrawl5","GLOBAL",1)~ EXTERN KAYL2 R5
-+ ~IsGabber(Player1) Gender(Player1,FEMALE) Global("PaladinOrder","GLOBAL",1) Dead("C6Bodhi") GlobalGT("Chapter","GLOBAL",5) Global("RE_RyanTrawl6","GLOBAL",0)~ + @5 DO ~SetGlobal("RE_RyanTrawl6","GLOBAL",1)~ EXTERN KAYL2 R6
++ ~IsGabber(Player1) Gender(Player1,FEMALE) Global("PaladinOrder","GLOBAL",1) !Dead("C6Bodhi") Global("Chapter","GLOBAL",%bg2_chapter_6%) Global("RE_RyanTrawl5","GLOBAL",0)~ + @4 DO ~SetGlobal("RE_RyanTrawl5","GLOBAL",1)~ EXTERN KAYL2 R5
++ ~IsGabber(Player1) Gender(Player1,FEMALE) Global("PaladinOrder","GLOBAL",1) Dead("C6Bodhi") GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%) Global("RE_RyanTrawl6","GLOBAL",0)~ + @5 DO ~SetGlobal("RE_RyanTrawl6","GLOBAL",1)~ EXTERN KAYL2 R6
 + ~IsGabber(Player1) Gender(Player1,FEMALE) Global("PaladinOrder","GLOBAL",1) Global("RE_RyanTrawl1","GLOBAL",1) Global("RE_RyanTrawl2","GLOBAL",1) Global("RE_RyanTrawl3","GLOBAL",1) Global("RE_RyanTrawl7","GLOBAL",0)~ + @6 DO ~SetGlobal("RE_RyanTrawl7","GLOBAL",1) SetGlobal("RE_RyanFlirt","GLOBAL",1)~ EXTERN KAYL2 R7
-+ ~IsGabber(Player1) Gender(Player1,FEMALE) Global("PaladinOrder","GLOBAL",1) GlobalGT("Chapter","GLOBAL",3)~ + @7 EXTERN KAYL2 R0
++ ~IsGabber(Player1) Gender(Player1,FEMALE) Global("PaladinOrder","GLOBAL",1) GlobalGT("Chapter","GLOBAL",%bg2_chapter_3%)~ + @7 EXTERN KAYL2 R0
 END
 
 INTERJECT KAYL2 33 RE_RyanTrawl
@@ -16,8 +16,8 @@
 + ~Global("RE_RyanTrawl2","GLOBAL",0)~ + @1 DO ~SetGlobal("RE_RyanTrawl2","GLOBAL",1) SetGlobal("RE_RyanTrawl","GLOBAL",0)~ EXTERN KAYL2 R2
 + ~Global("RE_RyanTrawl3","GLOBAL",0)~ + @2 DO ~SetGlobal("RE_RyanTrawl3","GLOBAL",1) SetGlobal("RE_RyanTrawl","GLOBAL",0)~ EXTERN KAYL2 R3
 + ~OR(3) Global("RE_RyanTrawl1","GLOBAL",1) Global("RE_RyanTrawl2","GLOBAL",1) Global("RE_RyanTrawl3","GLOBAL",1) Global("RE_RyanTrawl4","GLOBAL",0)~ + @3 DO ~SetGlobal("RE_RyanTrawl4","GLOBAL",1) SetGlobal("RE_RyanTrawl","GLOBAL",0)~ EXTERN KAYL2 R4
-+ ~!Dead("C6Bodhi") Global("Chapter","GLOBAL",6) Global("RE_RyanTrawl5","GLOBAL",0)~ + @4 DO ~SetGlobal("RE_RyanTrawl5","GLOBAL",1) SetGlobal("RE_RyanTrawl","GLOBAL",0)~ EXTERN KAYL2 R5
-+ ~Dead("C6Bodhi") GlobalGT("Chapter","GLOBAL",5) Global("RE_RyanTrawl6","GLOBAL",0)~ + @5 DO ~SetGlobal("RE_RyanTrawl6","GLOBAL",1) SetGlobal("RE_RyanTrawl","GLOBAL",0)~ EXTERN KAYL2 R6
++ ~!Dead("C6Bodhi") Global("Chapter","GLOBAL",%bg2_chapter_6%) Global("RE_RyanTrawl5","GLOBAL",0)~ + @4 DO ~SetGlobal("RE_RyanTrawl5","GLOBAL",1) SetGlobal("RE_RyanTrawl","GLOBAL",0)~ EXTERN KAYL2 R5
++ ~Dead("C6Bodhi") GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%) Global("RE_RyanTrawl6","GLOBAL",0)~ + @5 DO ~SetGlobal("RE_RyanTrawl6","GLOBAL",1) SetGlobal("RE_RyanTrawl","GLOBAL",0)~ EXTERN KAYL2 R6
 + ~Global("RE_RyanTrawl1","GLOBAL",1) Global("RE_RyanTrawl2","GLOBAL",1) Global("RE_RyanTrawl3","GLOBAL",1) Global("RE_RyanTrawl7","GLOBAL",0)~ + @6 DO ~SetGlobal("RE_RyanTrawl7","GLOBAL",1) SetGlobal("RE_RyanFlirt","GLOBAL",1) SetGlobal("RE_RyanTrawl","GLOBAL",0)~ EXTERN KAYL2 R7
 ++ @9 DO ~SetGlobal("RE_RyanTrawl","GLOBAL",0)~ EXTERN KAYL2 R0
 
