--- RE/Valygar/Valy25.baf	2013-07-30 11:39:18.000000000 -0400
+++ Fixpack/RE/Valygar/Valy25.baf	2016-11-21 22:55:03.579243200 -0500
@@ -23,7 +23,7 @@
 InParty(Myself)
 Global("RE_ValygarToB","GLOBAL",3)
 GlobalGT("TalkedSarMel01","GLOBAL",0)
-Global("Chapter","GLOBAL",8)
+Global("Chapter","GLOBAL",%bg2_chapter_8%)
 See(Player1)
 !StateCheck(Player1,CD_STATE_NOTVALID) 
 !StateCheck(Myself,CD_STATE_NOTVALID)
@@ -85,7 +85,7 @@
 RealGlobalTimerExpired("RE_ValygarToBTimer","GLOBAL")
 InParty(Myself)
 !AreaType(DUNGEON)
-!Global("Chapter","GLOBAL",10)
+!Global("Chapter","GLOBAL",%bg2_chapter_10%)
 See(Player1)
 !StateCheck(Player1,CD_STATE_NOTVALID) 
 !StateCheck(Myself,CD_STATE_NOTVALID)
@@ -106,7 +106,7 @@
 RealGlobalTimerExpired("RE_ValygarToBTimer","GLOBAL")
 InParty(Myself)
 !AreaType(DUNGEON)
-!Global("Chapter","GLOBAL",10)
+!Global("Chapter","GLOBAL",%bg2_chapter_10%)
 See(Player1)
 !StateCheck(Player1,CD_STATE_NOTVALID) 
 !StateCheck(Myself,CD_STATE_NOTVALID)
