--- mazzy/scripts/mazz25.baf	2013-03-02 07:38:02.000000000 -0500
+++ Fixpack/mazzy/scripts/mazz25.baf	2016-11-26 02:07:17.217900100 -0500
@@ -38,7 +38,7 @@
     InParty(Myself)
     See(Player1)
     Global("SaradushSacked","GLOBAL",1)
-    Global("I#MazzyFriendTalks","GLOBAL",2)
+    Global("I#MazzyToBFriendTalks","GLOBAL",2)
 THEN
   RESPONSE #100
     RealSetGlobalTimer("I#MazzyToBFriendTalksTimer","GLOBAL",600)
@@ -60,6 +60,6 @@
     Global("I#MazzyToBFriendTalks","GLOBAL",4)
 THEN
   RESPONSE #100
-    IncrementGlobal("I#MazzyFriendTalks","GLOBAL",1)
-    RealSetGlobalTimer("I#MazzyFriendTalksTimer","GLOBAL",1800)
+    IncrementGlobal("I#MazzyToBFriendTalks","GLOBAL",1)
+    RealSetGlobalTimer("I#MazzyToBFriendTalksTimer","GLOBAL",1800)
 END
