--- mazzy/setup-mazzy.tp2	2014-04-20 06:03:56.995000000 -0400
+++ Fixpack/mazzy/setup-mazzy.tp2	2016-11-21 10:32:59.441832600 -0500
@@ -11,7 +11,7 @@
 AUTHOR ~kae@spellholdstudios.net~
 
 // Version
-VERSION ~v2.2_Isaya~
+VERSION ~v2.2_Isaya BWP Fix~
 
 // Readme
 // Default to English readme if not found in the selected language
@@ -55,7 +55,7 @@
 
 // Component
 BEGIN @0
-  REQUIRE_PREDICATE GAME_IS ~bg2ee tob bgt~ @1
+  REQUIRE_PREDICATE GAME_IS ~bg2ee eet tob bgt~ @1
 
 
 // --------------------------------------
