--- setup-vynd.tp2	Wed Sep 24 17:05:31 2014
+++ C:\BWP Patchstudio\patched files\setup-vynd.tp2	Sun Nov 29 08:18:43 2015
@@ -1,5 +1,6 @@
 BACKUP ~Vynd/backup~
 AUTHOR ~Glam Vrock~
+VERSION ~1 BWP Fix~
 README ~Vynd/VyndReadme.html~
 
 // Crossplatform var stuff blatantly stolen from cmorgan/gavin/bg1npc!
@@ -175,7 +176,12 @@
 WRITE_ASCII 0x280 ~NIMBUL~
 BUT_ONLY_IF_IT_CHANGES
 
-
+ACTION_IF GAME_IS ~bgee bg2ee~ THEN BEGIN
+  OUTER_SPRINT "BlackguardKitCheck" "!Kit(Player1,Blackguard)"
+END
+ELSE BEGIN
+  OUTER_SPRINT "BlackguardKitCheck" ""
+END
 
 COMPILE EVALUATE_BUFFER ~Vynd/Dialogue/GV#VYND.d~
 COMPILE EVALUATE_BUFFER ~Vynd/Dialogue/GV#VYNDJ.d~
