--- spell_rev/components/nwn_spell_deflection.tpa	Sat Dec 03 20:42:53 2016
+++ Fixpack/spell_rev/components/nwn_spell_deflection.tpa	Sun Jan 22 10:54:54 2017
@@ -108,29 +108,29 @@
   OUTER_SET $spell("balth12")=1
 END
 
-ACTION_IF FILE_EXISTS_IN_GAME ~bdneoawe.spl~ BEGIN
+/*ACTION_IF FILE_EXISTS_IN_GAME ~bdneoawe.spl~ BEGIN
   OUTER_SET $spell("bdneoawe")=1
-END
+END */
 
-ACTION_IF FILE_EXISTS_IN_GAME ~bw05sp00.spl~ BEGIN
+/*ACTION_IF FILE_EXISTS_IN_GAME ~bw05sp00.spl~ BEGIN
   OUTER_SET $spell("bw05sp00")=1
-END
+END */
 
-ACTION_IF FILE_EXISTS_IN_GAME ~bw05sp00.spl~ BEGIN
+/*ACTION_IF FILE_EXISTS_IN_GAME ~bw05sp00.spl~ BEGIN
   OUTER_SET $spell("bw05sp00")=1
-END
+END */
 
-ACTION_IF FILE_EXISTS_IN_GAME ~bw05sp01.spl~ BEGIN
+/*ACTION_IF FILE_EXISTS_IN_GAME ~bw05sp01.spl~ BEGIN
   OUTER_SET $spell("bw05sp01")=1
-END
+END */
 
-ACTION_IF FILE_EXISTS_IN_GAME ~bw05sp04.spl~ BEGIN
+/*ACTION_IF FILE_EXISTS_IN_GAME ~bw05sp04.spl~ BEGIN
   OUTER_SET $spell("bw05sp04")=1
-END
+END */
 
-ACTION_IF FILE_EXISTS_IN_GAME ~bw05sp05.spl~ BEGIN
+/*ACTION_IF FILE_EXISTS_IN_GAME ~bw05sp05.spl~ BEGIN
   OUTER_SET $spell("bw05sp05")=1
-END
+END */
 
 ACTION_IF FILE_EXISTS_IN_GAME ~cvchao.spl~ BEGIN
   OUTER_SET $spell("cvchao")=1
@@ -172,9 +172,9 @@
   OUTER_SET $spell("mels545")=1
 END
 
-ACTION_IF FILE_EXISTS_IN_GAME ~melspl07.spl~ BEGIN
+/*ACTION_IF FILE_EXISTS_IN_GAME ~melspl07.spl~ BEGIN
   OUTER_SET $spell("melspl07")=1
-END
+END */
 
 ACTION_IF FILE_EXISTS_IN_GAME ~myst10.spl~ BEGIN
   OUTER_SET $spell("myst10")=1
@@ -184,30 +184,30 @@
   OUTER_SET $spell("nabpara")=1
 END
 
-ACTION_IF FILE_EXISTS_IN_GAME ~ohbdemi2.spl~ BEGIN
+/*ACTION_IF FILE_EXISTS_IN_GAME ~ohbdemi2.spl~ BEGIN
   OUTER_SET $spell("ohbdemi2")=1
-END
+END */
 
-ACTION_IF FILE_EXISTS_IN_GAME ~ntspwi81.spl~ BEGIN
+/*ACTION_IF FILE_EXISTS_IN_GAME ~ntspwi81.spl~ BEGIN
   OUTER_SET $spell("ntspwi81")=1
-END
+END */
 
 ACTION_IF FILE_EXISTS_IN_GAME ~ohbicew.spl~ BEGIN
   OUTER_SET $spell("ohbicew")=1
 END
 
 
-ACTION_IF FILE_EXISTS_IN_GAME ~ohbquake.spl~ BEGIN
+/*ACTION_IF FILE_EXISTS_IN_GAME ~ohbquake.spl~ BEGIN
   OUTER_SET $spell("ohbquake")=1
-END
+END */
 
 ACTION_IF FILE_EXISTS_IN_GAME ~ohbwfog.spl~ BEGIN
   OUTER_SET $spell("ohbwfog")=1
 END
 
-ACTION_IF FILE_EXISTS_IN_GAME ~ohbwi304.spl~ BEGIN
+/*ACTION_IF FILE_EXISTS_IN_GAME ~ohbwi304.spl~ BEGIN
   OUTER_SET $spell("ohbwi304")=1
-END
+END */
 
 ACTION_IF FILE_EXISTS_IN_GAME ~ohrdark.spl~ BEGIN
   OUTER_SET $spell("ohrdark")=1
@@ -217,21 +217,21 @@
   OUTER_SET $spell("ohrfear")=1
 END
 
-ACTION_IF FILE_EXISTS_IN_GAME ~spblun29.spl~ BEGIN
+/*ACTION_IF FILE_EXISTS_IN_GAME ~spblun29.spl~ BEGIN
   OUTER_SET $spell("spblun29")=1
-END
+END */
 
-ACTION_IF FILE_EXISTS_IN_GAME ~spcl911b.spl~ BEGIN
+/*ACTION_IF FILE_EXISTS_IN_GAME ~spcl911b.spl~ BEGIN
   OUTER_SET $spell("spcl911b")=1
-END
+END */
 
-ACTION_IF FILE_EXISTS_IN_GAME ~spcryo01.spl~ BEGIN
+/*ACTION_IF FILE_EXISTS_IN_GAME ~spcryo01.spl~ BEGIN
   OUTER_SET $spell("spcryo01")=1
-END
+END */
 
-ACTION_IF FILE_EXISTS_IN_GAME ~spctmd01.spl~ BEGIN
+/*ACTION_IF FILE_EXISTS_IN_GAME ~spctmd01.spl~ BEGIN
   OUTER_SET $spell("spctmd01")=1
-END
+END */
 
 
 ACTION_IF FILE_EXISTS_IN_GAME ~spin560.spl~ BEGIN
@@ -335,9 +335,9 @@
   OUTER_SET $spell("spinds2")=1
 END
 
-ACTION_IF FILE_EXISTS_IN_GAME ~spogre01.spl~ BEGIN
+/*ACTION_IF FILE_EXISTS_IN_GAME ~spogre01.spl~ BEGIN
   OUTER_SET $spell("spogre01")=1
-END
+END */
 
 ACTION_IF FILE_EXISTS_IN_GAME ~sppr250.spl~ BEGIN
   OUTER_SET $spell("sppr250")=1
