--- eetact2/setup-eetact2.tp2	Mon Jan 23 10:56:16 2017
+++ Fixpack/eetact2/setup-eetact2.tp2	Mon Jan 23 10:57:37 2017
@@ -16,7 +16,7 @@
 Adaptation for BG2EE by Musigny on https://forums.beamdog.com"
 
 // !!! This is an alpha version !!!
-VERSION ~v0.73 alpha 2016022103~
+VERSION ~v0.73 alpha 2016022103 BWP Fix~
 README ~eetact2/readme.txt~
 //MODDER
 
@@ -95,7 +95,7 @@
 ///////////////////////////////////////////////////////////////////////
 ///////////////////////////////////////////////////////////////////////
 BEGIN @10000 DESIGNATED 0
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 //DEPRECATED ~dummy component~
 
 	// Script core
@@ -113,7 +113,7 @@
 BEGIN @4000 DESIGNATED 100
 //GROUP @10020
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 REQUIRE_PREDICATE !MOD_IS_INSTALLED "setup-stratagems.tp2" "8000" @10012
 //DEPRECATED ~dummy component~
 
@@ -138,7 +138,7 @@
 BEGIN @4004 DESIGNATED 110
 //GROUP @10020
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 //DEPRECATED ~dummy component~
 
 	// Variables
@@ -159,7 +159,7 @@
 BEGIN @4005 DESIGNATED 120
 //GROUP @10020
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 REQUIRE_PREDICATE ((!MOD_IS_INSTALLED "setup-stratagems.tp2" "8060") AND
 				  (!MOD_IS_INSTALLED "setup-stratagems.tp2" "8061")) @10012
 //DEPRECATED ~dummy component~
@@ -182,7 +182,7 @@
 BEGIN @4006 DESIGNATED 130
 //GROUP @10020
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 REQUIRE_PREDICATE !MOD_IS_INSTALLED "setup-stratagems.tp2" "8110" @10012
 //DEPRECATED ~dummy component~
 
@@ -204,7 +204,7 @@
 BEGIN @4007 DESIGNATED 140
 //GROUP @10020
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 REQUIRE_PREDICATE ((!MOD_IS_INSTALLED "setup-stratagems.tp2" "8080") AND
 				  (!MOD_IS_INSTALLED "setup-stratagems.tp2" "8081")) @10012
 //DEPRECATED ~dummy component~
@@ -227,7 +227,7 @@
 BEGIN @4008 DESIGNATED 150
 //GROUP @10020
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 //DEPRECATED ~dummy component~
 
 	// Variables
@@ -248,7 +248,7 @@
 BEGIN @4009 DESIGNATED 160
 //GROUP @10020
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 //DEPRECATED ~dummy component~
 
 	// Variables
@@ -269,7 +269,7 @@
 BEGIN @4010 DESIGNATED 170
 //GROUP @10021
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 //DEPRECATED ~dummy component~
 
 	// Variables
@@ -290,7 +290,7 @@
 BEGIN @4011 DESIGNATED 180
 //GROUP @10020
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 //DEPRECATED ~dummy component~
 
 	// Variables
@@ -311,7 +311,7 @@
 BEGIN @4012 DESIGNATED 190
 //GROUP @10020
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 //DEPRECATED ~dummy component~
 
 	// Variables
@@ -332,7 +332,7 @@
 BEGIN @4013 DESIGNATED 200
 //GROUP @10020
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 //DEPRECATED ~dummy component~
 
 	// Variables
@@ -353,7 +353,7 @@
 BEGIN @4014 DESIGNATED 210
 //GROUP @10020
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 //DEPRECATED ~dummy component~
 
 	// Variables
@@ -374,7 +374,7 @@
 BEGIN @4015 DESIGNATED 220
 //GROUP @10021
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 REQUIRE_PREDICATE ((!MOD_IS_INSTALLED "setup-stratagems.tp2" "6510") AND
 				  (!MOD_IS_INSTALLED "setup-stratagems.tp2" "6511") AND
 				  (!MOD_IS_INSTALLED "setup-atweaks.tp2" "160")) @10012
@@ -398,7 +398,7 @@
 BEGIN @10230 DESIGNATED 230 SUBCOMPONENT @4016
 //GROUP @10020
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 //DEPRECATED ~dummy component~
 
 	// Variables
@@ -415,7 +415,7 @@
 BEGIN @10231 DESIGNATED 231 SUBCOMPONENT @4016
 //GROUP @10020
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 //DEPRECATED ~dummy component~
 
 	// Variables
@@ -432,7 +432,7 @@
 BEGIN @10232 DESIGNATED 232 SUBCOMPONENT @4016
 //GROUP @10020
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 //DEPRECATED ~dummy component~
 
 	// Variables
@@ -455,7 +455,7 @@
 BEGIN @4017 DESIGNATED 240
 //GROUP @10020
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 //DEPRECATED ~dummy component~
 
 	// Variables
@@ -476,7 +476,7 @@
 BEGIN @4018 DESIGNATED 250
 //GROUP @10020
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 //DEPRECATED ~dummy component~
 
 	// Variables
@@ -497,7 +497,7 @@
 BEGIN @4019 DESIGNATED 260
 //GROUP @10020
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 //DEPRECATED ~dummy component~
 
 	// Variables
@@ -521,7 +521,7 @@
 BEGIN @4020 DESIGNATED 270
 //GROUP @10021
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 REQUIRE_PREDICATE !MOD_IS_INSTALLED "setup-stratagems.tp2" "8050" @10012
 //DEPRECATED ~dummy component~
 
@@ -546,7 +546,7 @@
 BEGIN @4021 DESIGNATED 280
 //GROUP @10021
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 REQUIRE_PREDICATE !MOD_IS_INSTALLED "setup-stratagems.tp2" "8050" @10012
 //DEPRECATED ~dummy component~
 
@@ -568,7 +568,7 @@
 BEGIN @4022 DESIGNATED 290
 //GROUP @10021
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 REQUIRE_PREDICATE !MOD_IS_INSTALLED "setup-atweaks.tp2" "160" @10012
 //DEPRECATED ~dummy component~
 
@@ -593,7 +593,7 @@
 BEGIN @4023 DESIGNATED 300
 //GROUP @10021
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 REQUIRE_PREDICATE ((!MOD_IS_INSTALLED "setup-stratagems.tp2" "6560") AND
 				  (!MOD_IS_INSTALLED "setup-stratagems.tp2" "6561") AND
 				  (!MOD_IS_INSTALLED "setup-stratagems.tp2" "6562") AND
@@ -621,7 +621,7 @@
 BEGIN @4024 DESIGNATED 310
 //GROUP @10021
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 REQUIRE_PREDICATE ((!MOD_IS_INSTALLED "setup-stratagems.tp2" "6540") AND
 				  (!MOD_IS_INSTALLED "setup-stratagems.tp2" "6541")) @10012
 //DEPRECATED ~dummy component~
@@ -644,7 +644,7 @@
 BEGIN @4025 DESIGNATED 320
 //GROUP @10021
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 REQUIRE_PREDICATE ((!MOD_IS_INSTALLED "setup-stratagems.tp2" "6550") AND
 				  (!MOD_IS_INSTALLED "setup-stratagems.tp2" "6551") AND
 				  (!MOD_IS_INSTALLED "setup-stratagems.tp2" "6552") AND
@@ -669,7 +669,7 @@
 BEGIN @4026 DESIGNATED 330
 //GROUP @10021
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 REQUIRE_PREDICATE !MOD_IS_INSTALLED "setup-stratagems.tp2" "6580" @10012
 //DEPRECATED ~dummy component~
 
@@ -691,7 +691,7 @@
 BEGIN @4027 DESIGNATED 340
 //GROUP @10021
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 REQUIRE_PREDICATE ((!MOD_IS_INSTALLED "setup-stratagems.tp2" "6030") AND
 				  (!MOD_IS_INSTALLED "setup-stratagems.tp2" "6031") AND
 				  (!MOD_IS_INSTALLED "setup-stratagems.tp2" "6032") AND
@@ -717,7 +717,7 @@
 BEGIN @4030 DESIGNATED 350
 //GROUP @10021
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 REQUIRE_PREDICATE !MOD_IS_INSTALLED "setup-stratagems.tp2" "6500" @10012
 //DEPRECATED ~dummy component~
 
@@ -742,7 +742,7 @@
 BEGIN @4031 DESIGNATED 360
 //GROUP @10021
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 REQUIRE_PREDICATE !MOD_IS_INSTALLED "setup-atweaks.tp2" "185" @10012
 //DEPRECATED ~dummy component~
 
@@ -764,7 +764,7 @@
 BEGIN @4032 DESIGNATED 370
 //GROUP @10021
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 //DEPRECATED ~dummy component~
 
 	// Variables
@@ -785,7 +785,7 @@
 BEGIN @4033 DESIGNATED 380
 //GROUP @10021
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 REQUIRE_PREDICATE ((!MOD_IS_INSTALLED "setup-d0questpack.tp2" "11") AND
 				  (!MOD_IS_INSTALLED "setup-d0questpack.tp2" "12")) @10012
 //DEPRECATED ~dummy component~
@@ -808,7 +808,7 @@
 BEGIN @4034 DESIGNATED 390
 //GROUP @10021
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 //DEPRECATED ~dummy component~
 
 	// Variables
@@ -829,7 +829,7 @@
 BEGIN @4035 DESIGNATED 400
 //GROUP @10021
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 //DEPRECATED ~dummy component~
 
 	// Variables
@@ -850,7 +850,7 @@
 BEGIN @4036 DESIGNATED 410
 //GROUP @10021
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 //DEPRECATED ~dummy component~
 
 	// Variables
@@ -871,7 +871,7 @@
 BEGIN @4037 DESIGNATED 420
 //GROUP @10021
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
 REQUIRE_PREDICATE ((!MOD_IS_INSTALLED "setup-stratagems.tp2" "6510") AND
 				  (!MOD_IS_INSTALLED "setup-stratagems.tp2" "6511") AND
@@ -901,7 +901,7 @@
 BEGIN @10430 DESIGNATED 430 SUBCOMPONENT @4038
 //GROUP @10020
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 //DEPRECATED ~dummy component~
 
 	// Variables
@@ -918,7 +918,7 @@
 BEGIN @10431 DESIGNATED 431 SUBCOMPONENT @4038
 //GROUP @10020
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 //DEPRECATED ~dummy component~
 
 	// Variables
@@ -935,7 +935,7 @@
 BEGIN @10432 DESIGNATED 432 SUBCOMPONENT @4038
 //GROUP @10020
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 //DEPRECATED ~dummy component~
 
 	// Variables
@@ -958,7 +958,7 @@
 BEGIN @4039 DESIGNATED 440
 //GROUP @10021
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 REQUIRE_PREDICATE !MOD_IS_INSTALLED "setup-stratagems.tp2" "8160" @10012
 //DEPRECATED ~dummy component~
 
@@ -980,7 +980,7 @@
 BEGIN @4040 DESIGNATED 450
 //GROUP @10020
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 //DEPRECATED ~dummy component~
 
 	// Variables - placeholder (no folder)
@@ -1000,7 +1000,7 @@
 BEGIN @4028 DESIGNATED 460
 //GROUP @10022
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 //DEPRECATED ~dummy component~
 
 	// Variables
@@ -1024,7 +1024,7 @@
 BEGIN @4029 DESIGNATED 470
 //GROUP @10022
 REQUIRE_COMPONENT "setup-eetact2.tp2" "0" @10011
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 //DEPRECATED ~dummy component~
 
 	// Variables
@@ -1050,7 +1050,7 @@
 ///////////////////////////////////////////////////////////////////////
 /*
 BEGIN ~dummy Tactics2~
-REQUIRE_PREDICATE GAME_IS ~bg2ee~ @10010
+REQUIRE_PREDICATE GAME_IS ~bg2ee eet~ @10010
 //DEPRECATED ~dummy component~
 
 	// Script core
