--- Bdash\BGTComp\trans\olAR0602.BAF	Sat Sep 13 02:03:30 2014
+++ C:\BWP Patchstudio\patched files\Bdash\BGTComp\trans\olAR0602.BAF	Sat Sep 10 19:38:00 2016
@@ -4,6 +4,7 @@
 	Global("JaheiraImport","GLOBAL",2)
 	Global("ImoenImport","GLOBAL",2)
 	Global("BGImportIncomplete","AR0602",0)
+	Gender(Player1,MALE)
 THEN
 	RESPONSE #100
 		SetGlobal("TakeImportItems","AR0602",1)
@@ -13,16 +14,17 @@
 		ActionOverride("Shelf1",TakeItemListPartyNum("IMPORT03",1))
 		SmallWait(4)
 		TakePartyGold(2147483647)
+		ActionOverride(Player1,MakeUnselectable(0))
 		ActionOverride(Player1,DestroyAllEquipment())
 		ActionOverride(Player2,DestroyAllEquipment())
 		ActionOverride(Player3,DestroyAllEquipment())
 		ActionOverride(Player4,DestroyAllEquipment())
 		ActionOverride(Player5,DestroyAllEquipment())
 		ActionOverride(Player6,DestroyAllEquipment())
-		CreateCreature("CSJON",[3210.3304],10) // Mage
+		CreateCreature("CSJON",[3210.3304],10) // Magier
 		ActionOverride(Player1,PlayDead(165))
 		Wait(1)
 		FadeFromColor([30.0],0)
 		StartCutSceneMode()
-		StartCutScene("NewGame")
+		StartCutScene("CBNEWGMM")
 END
