--- JklHel\Helarine_BGEE.Tp2	Fri Apr 12 17:03:28 2013
+++ D:\EE_Patchstudio\patched files\JklHel\Helarine_BGEE.Tp2	Tue Jan 19 21:58:35 2016
@@ -1,8 +1,57 @@
 BACKUP ~JklHel/backup~
  
 AUTHOR ~Jackkel Dragon~
+
+VERSION ~1.0 alpha BWP Fix~
+
+AUTO_TRA "JklHel/tra/%s"//%"
+
+ALWAYS
+ACTION_IF !FILE_EXISTS ~JklHel/charsets.mrk~ BEGIN
+
+ACTION_IF "%WEIDU_OS%" STRING_EQUAL_CASE ~win32~ BEGIN
+  AT_NOW ~if not exist "iconv" mkdir iconv~
+  AT_NOW ~if not exist "iconv\iconv.exe" copy "BiG World Fixpack\_utils\iconv" "iconv"~
+END
+
+ACTION_DEFINE_ARRAY tra#reload BEGIN setup END
+ACTION_DEFINE_ARRAY tra#noconvert BEGIN END
+
+LAF HANDLE_CHARSETS
+	INT_VAR
+		infer_charset = 1
+	STR_VAR
+		tra_path = EVAL ~JklHel/tra~
+		noconvert_array = tra#noconvert
+		reload_array = tra#reload
+		iconv_path = ~iconv~
+END
+		
+LOAD_TRA ~JklHel/tra/%LANGUAGE%/Setup.tra~
+COPY_EXISTING ~sw1h01.itm~ ~JklHel/charsets.mrk~
+END
+
+ACTION_IF GAME_IS ~eet~ BEGIN
+OUTER_SPRINT "eet_var" "_"
+OUTER_SPRINT "Candlekeep_Catacombs_L1_BCS" "BG2615"
+OUTER_SPRINT "FriendlyArmInn_BCS" "BG2300"
+OUTER_SPRINT "FriendlyArmInn_L1" "BG2301"
+OUTER_SPRINT "FriendlyArmInn" "BG2300"
+END
+
+ACTION_IF GAME_IS ~bgee~ BEGIN
+OUTER_SPRINT "eet_var" ""
+OUTER_SPRINT "Candlekeep_Catacombs_L1_BCS" "AR2615"
+OUTER_SPRINT "FriendlyArmInn_BCS" "AR2300"
+OUTER_SPRINT "FriendlyArmInn_L1" "AR2301"
+OUTER_SPRINT "FriendlyArmInn" "AR2300"
+END
+
+END
+
+LANGUAGE ~English~ ~english~ ~JklHel/tra/english/setup.tra~
  
-BEGIN ~Helarine for BGEE (alpha)~
+BEGIN @1
 
 // stolen from requiem's kitpack
 // ---------------------------------------DPKELEMVOR----------------------------------
@@ -46,35 +95,38 @@
 ~CHAN09 * HELM07 BAG26 RING06 RING31 * BOOT01 AMUL20 BRAC10 BELT06 AROW11,40 BULL03,40 BOLT06,40 POTN52,5 POTN04,2 POTN14,5 HAMM07 SW1H27 STAF08~
 
 // description
-SAY ~Doomguide~
-SAY ~Doomguide Of Kelemvor~
-SAY ~DOOMGUIDE OF KELEMVOR: Fair yet cold, Kelemvor is the god of death and the dead, he promotes death as a natural part of existence, something to be embraced and not feared, as such his clergy are often charged with arranging funerals and rites for the recently deceased.
-
-Advantages:
-
-- Gains domain spells relevant to the portfolio of the deity:
-- 1st Level: Chill Touch
-- 4th Level: Hold Undead
-- 7th Level: Emotion: Hopelessness
-- 10th Level: Raise Dead
-- 13th Level: Bolt of Glory
-
-Disadvantages:
-- None
-~
+SAY @2
+SAY @3
+SAY @4
+
+INCLUDE ~JklHel/fl#add_kit_ee.tpa~
+
+LAF fl#add_kit_ee
+	INT_VAR
+		biography = RESOLVE_STR_REF (@4)
+		briefdesc = RESOLVE_STR_REF (@3)
+	STR_VAR
+		kit_name = ~HLKELEMVOR~
+		clswpbon = ~0 0 3~
+		numwslot = ~2~
+		clascolr = ~30 23 64 23 30~ //took the colours from the creature
+		hpclass = ~HPPRS~
+		clsrcreq = ~1 0 0 0 0 0 0~ //I think, only humans could use this kit...
+		clasthac = ~0~
+END
 
 //-------------------------DOMAIN SPELLS-------------------------------------------
 
 COPY ~JklHel\kits\HEL_KELEMVOR\SPHLKEL1.spl~ ~override\SPHLKEL1.spl~
-SAY NAME1 ~Chill Touch~
+SAY NAME1 @5
 COPY ~JklHel\kits\HEL_KELEMVOR\SPHLKEL2.spl~ ~override\SPHLKEL2.spl~
-SAY NAME1 ~Hold Undead~
+SAY NAME1 @6
 COPY ~JklHel\kits\HEL_KELEMVOR\SPHLKEL3.spl~ ~override\SPHLKEL3.spl~
-SAY NAME1 ~Emotion: Hopelessness~
+SAY NAME1 @7
 COPY ~JklHel\kits\HEL_KELEMVOR\SPHLKEL4.spl~ ~override\SPHLKEL4.spl~
-SAY NAME1 ~Raise Dead~
+SAY NAME1 @8
 COPY ~JklHel\kits\HEL_KELEMVOR\SPHLKEL5.spl~ ~override\SPHLKEL5.spl~
-SAY NAME1 ~Bolt of Glory~
+SAY NAME1 @9
 
 //--------------------------SYMBOL, SPELL-ITEM-BAM-----------------------------
 
@@ -112,9 +164,9 @@
  
 /* This allows for your NPC's name to be shown in the game during conversations and also on the character sheet*/
  
-SAY NAME1 ~Helarine~
+SAY NAME1 @10
  
-SAY NAME2 ~Helarine~
+SAY NAME2 @10
 
 // Kit stuff
 WRITE_SHORT 0x244 0 // doesn't change  
@@ -123,63 +175,72 @@
  
 /* This is the biography. In NearInfinity, go to that side tab and find CHR. Open that and find Sam.BIO You can create your own bio this way by simply deleting the default PC bio and typing your own. Save this.*/
  
-SAY BIO ~When asked about her past, Helarine becomes quiet and hugs herself tightly. She explains that she once was the servant of an important deity that perished during the Time of Troubles, but refuses to specify which one. After a period of faithlessness, she began her service to the recently ascended God of Death, Kelemvor. She claims that she had to go through a significant change of personality to adhere to Kelemvor's wishes, but she doesn't tell you what she was like before her conversion to Kelemvor's clergy. Once the tale is told, she becomes distant and formal once more, leaving further questions unanswered.~
+SAY BIO @11
  
 /* These are the different things that your NPC will say when they respond to your commands.*/
  
-SAY MORALE ~This fight is going poorly...~
+SAY MORALE @12
  
-SAY UNHAPPY_SERIOUS ~Though my heart wishes to stay, Kelemvor may force my hand.~
+SAY UNHAPPY_SERIOUS @13
  
-SAY UNHAPPY_BREAKING ~I apologize... my duty to Kelemvor comes first.~
+SAY UNHAPPY_BREAKING @14
  
-SAY LEADER ~If the company requires my leadership.~
+SAY LEADER @15
  
-SAY TIRED ~We shouldn't continue until everyone is fully rested.~
+SAY TIRED @16
  
-SAY BATTLE_CRY1 ~Don't cross me!~
+SAY BATTLE_CRY1 @17
  
-SAY BATTLE_CRY2 ~I won't hold back!~
+SAY BATTLE_CRY2 @18
  
-SAY BATTLE_CRY3 ~Your soul has lingered on this realm for too long!~
+SAY BATTLE_CRY3 @19
  
-SAY DAMAGE ~Ugh!~
+SAY DAMAGE @20
  
-SAY DYING ~Kelemvor... guide me...~
+SAY DYING @21
  
-SAY HURT ~I can't keep this up much longer...~
+SAY HURT @22
  
-SAY SELECT_COMMON1 ~At your service.~
+SAY SELECT_COMMON1 @23
  
-SAY SELECT_COMMON2 ~I'm here.~
+SAY SELECT_COMMON2 @24
  
-SAY SELECT_COMMON3 ~Anything troubling you?~
+SAY SELECT_COMMON3 @25
  
-SAY SELECT_COMMON4 ~Your orders?~
+SAY SELECT_COMMON4 @26
  
-SAY SELECT_ACTION1 ~As you wish.~
+SAY SELECT_ACTION1 @27
  
-SAY SELECT_ACTION2 ~I'll get right to it.~
+SAY SELECT_ACTION2 @28
  
-SAY SELECT_ACTION3 ~That should be simple enough.~
+SAY SELECT_ACTION3 @29
 
-SAY SELECT_RARE1 ~I live to serve.~
+SAY SELECT_RARE1 @30
  
-SAY CRITICAL_HIT ~That looked painful...~
+SAY CRITICAL_HIT @31
  
-SAY CRITICAL_MISS ~I shouldn't fumble like this...~
+SAY CRITICAL_MISS @32
  
-SAY TARGET_IMMUNE ~I need a different weapon.~
- 
-SAY INVENTORY_FULL ~I... don't think I can carry this all.~
+SAY TARGET_IMMUNE @33
  
+SAY INVENTORY_FULL @34
+
+//Dorn Romance Check
+<<<<<<<< .../inlined/jkl_dornj.d
+ADD_TRANS_ACTION DORNJ BEGIN 214 END BEGIN END ~SetGlobal("JKLHEL_DornRomance","Global",2)~
+
+>>>>>>>>
+
+COMPILE ~.../inlined/jkl_dornj.d~
+
+
 // This is compiling Sam's dialogue
  
 COMPILE ~JklHel/dialogue/jkl_hel_main.d~
 
 //COMPILE ~JklHel/dialogue/jkl_hel_gabber.d~
 
-COMPILE ~JklHel/dialogue/jkl_hel_partyactions.d~
+COMPILE ~JklHel/dialogue/jkl_hel_partyactions.d~ EVAL
 
 COMPILE ~JklHel/dialogue/jkl_hel_banter.d~
 
@@ -189,27 +250,27 @@
  
 // This is adding our area file to the game
  
-EXTEND_TOP ~AR2300.bcs~ ~JklHel/scripts/AR2300.baf~
+EXTEND_TOP ~%FriendlyArmInn_BCS%.bcs~ ~JklHel/scripts/AR2300.baf~ EVAL
 
-EXTEND_TOP ~AR2615.bcs~ ~JklHel/scripts/AR2615.baf~
+EXTEND_TOP ~%Candlekeep_Catacombs_L1_BCS%.bcs~ ~JklHel/scripts/AR2615.baf~
 
 // extend NPC scripts for banter accelerators
 
-EXTEND_BOTTOM ~FALDORN.bcs~ ~JklHel/scripts/jklhel_faldorn.baf~
+EXTEND_BOTTOM ~FALDORN%eet_var%.bcs~ ~JklHel/scripts/jklhel_faldorn.baf~
 
 EXTEND_BOTTOM ~ALORA.bcs~ ~JklHel/scripts/jklhel_alora.baf~
 
-EXTEND_BOTTOM ~BRANWEN.bcs~ ~JklHel/scripts/jklhel_branwen.baf~
+EXTEND_BOTTOM ~BRANWEN%eet_var%.bcs~ ~JklHel/scripts/jklhel_branwen.baf~
 
-EXTEND_BOTTOM ~DORN.bcs~ ~JklHel/scripts/jklhel_dorn.baf~
+EXTEND_BOTTOM ~DORN%eet_var%.bcs~ ~JklHel/scripts/jklhel_dorn.baf~
 
-EXTEND_BOTTOM ~EDWIN.bcs~ ~JklHel/scripts/jklhel_edwin.baf~
+EXTEND_BOTTOM ~EDWIN%eet_var%.bcs~ ~JklHel/scripts/jklhel_edwin.baf~
 
 EXTEND_BOTTOM ~ELDOTH.bcs~ ~JklHel/scripts/jklhel_eldoth.baf~
 
 EXTEND_BOTTOM ~KIVAN.bcs~ ~JklHel/scripts/jklhel_kivan.baf~
 
-EXTEND_BOTTOM ~NEERA.bcs~ ~JklHel/scripts/jklhel_neera.baf~
+EXTEND_BOTTOM ~NEERA%eet_var%.bcs~ ~JklHel/scripts/jklhel_neera.baf~
 
 EXTEND_BOTTOM ~XAN.bcs~ ~JklHel/scripts/jklhel_xan.baf~
 
