--- JklHel\dialogue\jkl_hel_main.d	Tue Mar 26 20:21:52 2013
+++ D:\EE_Patchstudio\patched files\JklHel\dialogue\jkl_hel_main.d	Sun Jan 10 15:08:57 2016
@@ -7,20 +7,20 @@
 Global("JKL_Helarine_BG1_track","GLOBAL",2)
 Global("JKL_Helarine_ReadytoTalk","GLOBAL",1)~
 THEN JKLHELJ DeathTalk1
-~If I may ask, <CHARNAME>, what inspired you to face this iron crisis? Most don't have your initiative and are willing to wait and hope it sorts itself out.~
+@0
 DO ~SetGlobal("JKL_Helarine_BG1_track","GLOBAL",3)
 SetGlobal("JKL_Helarine_ReadytoTalk","GLOBAL",0)
 RealSetGlobalTimer("JKL_Helarine_BG1_track_timer","GLOBAL",1800)~
 END
 
-++ ~I didn't ask for this responsibility. It fell upon me when my foster father was killed on the road from Candlekeep.~ + DeathTalk1-1
-++ ~Someone had to take action. It might as well be me.~ + DeathTalk1-3
-++ ~I decided it was time to strike out on my own and make my mark on the Sword Coast.~ + DeathTalk1-2
+++ @1 + DeathTalk1-1
+++ @2 + DeathTalk1-3
+++ @3 + DeathTalk1-2
 
 CHAIN
 IF ~~
 THEN JKLHELJ DeathTalk1-1
-~That sounds tragic, but such things seem to happen in times like these. I hope that his soul is at peace now.~
+@4
 END
 
 IF ~~ THEN GOTO DeathTalk1-4
@@ -28,7 +28,7 @@
 CHAIN
 IF ~~
 THEN JKLHELJ DeathTalk1-2
-~So this adventure is for personal glory? It certainly is the climate that could create a hero.~
+@5
 END
 
 IF ~~ THEN GOTO DeathTalk1-4
@@ -36,7 +36,7 @@
 CHAIN
 IF ~~
 THEN JKLHELJ DeathTalk1-3
-~I suspect many would find it admirable that you thought that way.~
+@6
 END
 
 IF ~~ THEN GOTO DeathTalk1-4
@@ -44,18 +44,18 @@
 CHAIN
 IF ~~
 THEN JKLHELJ DeathTalk1-4
-~You've set out on a difficult road, <CHARNAME>. Kelemvor may well be seeing one or more of this company before the end of your task. I wonder: are you prepared to watch friends and allies die? Are you prepared to die yourself?~
+@7
 END
 
-++ ~I plan on keeping as many of my allies alive as possible.~ + DeathTalk1-5
-++ ~Few people are prepared to die. I doubt we'll know whether I'm ready until death comes for me.~ + DeathTalk1-6
-++ ~If my allies fall, they have failed me. There is no reason to mourn weakness.~ + DeathTalk1-7
+++ @8 + DeathTalk1-5
+++ @9 + DeathTalk1-6
+++ @10 + DeathTalk1-7
 
 
 CHAIN
 IF ~~
 THEN JKLHELJ DeathTalk1-5
-~That doesn't answer my question, though I believe I can read an answer into your unwillingness to answer.~
+@11
 END
 
 IF ~~ THEN GOTO DeathTalk1-8
@@ -63,7 +63,7 @@
 CHAIN
 IF ~~
 THEN JKLHELJ DeathTalk1-6
-~Wisely put. I expected a more definate answer, but I should expect one unaccustomed to death to avoid answering such a question until they know what they speak of.~
+@12
 END
 
 IF ~~ THEN GOTO DeathTalk1-8
@@ -71,7 +71,7 @@
 CHAIN
 IF ~~
 THEN JKLHELJ DeathTalk1-7
-~That is a callous attitude, and you may wish to conceal it from your other allies. Few will accept you as leader if you air such opinions.~
+@13
 END
 
 IF ~~ THEN GOTO DeathTalk1-8
@@ -79,18 +79,18 @@
 CHAIN
 IF ~~
 THEN JKLHELJ DeathTalk1-8
-~Death is something I am familiar with... and I know that death always accompanies times such as this.~
+@14
 END
 
-++ ~What makes you say that?~ + DeathTalk1-9
-++ ~Are you speaking from experience?~ + DeathTalk1-10
-++ ~Indeed it will, if anyone is foolhardy enough to cross me.~ + DeathTalk1-11
+++ @15 + DeathTalk1-9
+++ @16 + DeathTalk1-10
+++ @17 + DeathTalk1-11
 
 
 CHAIN
 IF ~~
 THEN JKLHELJ DeathTalk1-9
-~You've studied the Time of Troubles, have you not? You may have been young, but it occurred only a decade or so ago.~
+@18
 END
 
 IF ~~ THEN GOTO DeathTalk1-12
@@ -98,7 +98,7 @@
 CHAIN
 IF ~~
 THEN JKLHELJ DeathTalk1-10
-~You could say that... I witnessed the Time of Troubles firsthand. Not all of us were cloistered in Candlekeep for the past decade or so.~
+@19
 END
 
 IF ~~ THEN GOTO DeathTalk1-12
@@ -106,7 +106,7 @@
 CHAIN
 IF ~~
 THEN JKLHELJ DeathTalk1-11
-~It's exactly that kind of attitude that prevailed during the Time of Troubles. I suppose recent history may not be a favorite of Candlekeep's monks?~
+@20
 END
 
 IF ~~ THEN GOTO DeathTalk1-12
@@ -114,17 +114,17 @@
 CHAIN
 IF ~~
 THEN JKLHELJ DeathTalk1-12
-~Those days saw conflict and change, and not even gods were safe from death. Ironically, the gods associated with death before that time all fell, leaving Cyric as god of the dead.~
+@21
 END
 
-++ ~Isn't Kelemvor the god of the dead?~ + DeathTalk1-13
-++ ~What's your point?~ + DeathTalk1-14
-++ ~I'm growing tired of this conversation.~ + DeathTalk1-15
+++ @22 + DeathTalk1-13
+++ @23 + DeathTalk1-14
+++ @24 + DeathTalk1-15
 
 CHAIN
 IF ~~
 THEN JKLHELJ DeathTalk1-13
-~He is, but his ascension to godhood wasn't until a few years ago, after the chaos of the Time of Troubles.~
+@25
 END
 
 IF ~~ THEN GOTO DeathTalk1-16
@@ -132,7 +132,7 @@
 CHAIN
 IF ~~
 THEN JKLHELJ DeathTalk1-14
-~Death comes to everyone, without exception.~
+@26
 END
 
 IF ~~ THEN GOTO DeathTalk1-16
@@ -140,7 +140,7 @@
 CHAIN
 IF ~~
 THEN JKLHELJ DeathTalk1-15
-~I'm clearly rambling, then. I'll try to finish quickly.~
+@27
 END
 
 IF ~~ THEN GOTO DeathTalk1-16
@@ -148,8 +148,8 @@
 CHAIN
 IF ~~
 THEN JKLHELJ DeathTalk1-16
-~I feel I've been unclear... I only mean to make sure you understand that death will be a fact of life for you from now on.~
-=~Perhaps I've overstepped my boundries. I apologize. You'll understand what I meant to tell you soon enough.~
+@28
+=@29
 EXIT
 
 // Death Talk 2
@@ -159,20 +159,20 @@
 Global("JKL_Helarine_BG1_track","GLOBAL",4)
 Global("JKL_Helarine_ReadytoTalk","GLOBAL",1)~
 THEN JKLHELJ DeathTalk2
-~Do you ever regret your actions, <CHARNAME>?~
+@30
 DO ~SetGlobal("JKL_Helarine_BG1_track","GLOBAL",5)
 SetGlobal("JKL_Helarine_ReadytoTalk","GLOBAL",0)
 RealSetGlobalTimer("JKL_Helarine_BG1_track_timer","GLOBAL",1800)~
 END
 
-++ ~Sometimes. Do you have something specific in mind?~ + DeathTalk2-1
-++ ~What made you bring this up?~ + DeathTalk2-1
-++ ~Why would I regret anything I've done?~ + DeathTalk2-2
+++ @31 + DeathTalk2-1
+++ @32 + DeathTalk2-1
+++ @33 + DeathTalk2-2
 
 CHAIN
 IF ~~
 THEN JKLHELJ DeathTalk2-1
-~We've travelled the Sword Coast for some time now, and many have died at your hand or those of your companions.~
+@34
 END
 
 IF ~~ THEN GOTO DeathTalk2-3
@@ -180,7 +180,7 @@
 CHAIN
 IF ~~
 THEN JKLHELJ DeathTalk2-2
-~One who doesn't feel regret doesn't learn from their failures. But what I mean is that we've killed a lot of people across the Sword Coast.~
+@35
 END
 
 IF ~~ THEN GOTO DeathTalk2-3
@@ -188,17 +188,17 @@
 CHAIN
 IF ~~
 THEN JKLHELJ DeathTalk2-3
-~While I doubt we'd see eye-to-eye with many of our foes, they had lives just as we do. And now those lives have been extinguised.~
+@36
 END
 
-++ ~Sometimes I feel surprised at myself. I was never this violent back at Candlekeep.~ + DeathTalk2-4
-++ ~We fight in self-defense and for a greater good, it's not us who should have heavy hearts.~ + DeathTalk2-5
-++ ~They stood between us and our goals. I won't mourn fools.~ + DeathTalk2-6
+++ @37 + DeathTalk2-4
+++ @38 + DeathTalk2-5
+++ @39 + DeathTalk2-6
 
 CHAIN
 IF ~~
 THEN JKLHELJ DeathTalk2-4
-~I'm not surprised that you've changed. The Sword Coast isn't quite as safe as your home in Candlekeep.~
+@40
 END
 
 IF ~~ THEN GOTO DeathTalk2-7
@@ -206,7 +206,7 @@
 CHAIN
 IF ~~
 THEN JKLHELJ DeathTalk2-5
-~You may think those noble causes, but to others it will seem a justification.~
+@41
 END
 
 IF ~~ THEN GOTO DeathTalk2-7
@@ -214,7 +214,7 @@
 CHAIN
 IF ~~
 THEN JKLHELJ DeathTalk2-6
-~Seeing your enemies as little more than obstacles and fools will only fuel future violence and death.~
+@42
 END
 
 IF ~~ THEN GOTO DeathTalk2-7
@@ -222,18 +222,18 @@
 CHAIN
 IF ~~
 THEN JKLHELJ DeathTalk2-7
-~This iron crisis has made people desperate. More and more are turning to violence to survive.~
-=~We'll be seeing much more death before this is over... so I want you to remember that while we may kill to survive, we must show the proper respect for death.~
+@43
+=@44
 END
 
-++ ~I didn't plan on desecrating any of our fallen enemies.~ + DeathTalk2-8
-++ ~I'm not sure I follow you...~ + DeathTalk2-9
-++ ~I don't like being preached to.~ + DeathTalk2-10
+++ @45 + DeathTalk2-8
+++ @46 + DeathTalk2-9
+++ @47 + DeathTalk2-10
 
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk2-8
-~That's not what I meant, though respecting the dead does show reverance for death itself.~
+@48
 END
 
 IF ~~ THEN GOTO DeathTalk2-11
@@ -241,7 +241,7 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk2-9
-~I'll try to elaborate, then. I didn't mean to confuse you.~
+@49
 END
 
 IF ~~ THEN GOTO DeathTalk2-11
@@ -250,7 +250,7 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk2-10
-~I'm not trying to upset you, only make you understand what you've gotten yourself into.~
+@50
 END
 
 IF ~~ THEN GOTO DeathTalk2-11
@@ -258,30 +258,30 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk2-11
-~What I mean to say is that, while death will follow us, do not allow yourself to become desensitized to it.~
-=~Death is a transition from one life to another... but when its effect is devalued or embraced many suffer from its misuse.~
+@51
+=@52
 END
 
-++ ~I'll keep that in mind.~ + DeathTalk2-12
-++ ~You spend a lot of time thinking about death, it seems.~ + DeathTalk2-13
-++ ~Are you getting to your point anytime soon, or should I just start ignoring you?~ + DeathTalk2-14
+++ @53 + DeathTalk2-12
+++ @54 + DeathTalk2-13
+++ @55 + DeathTalk2-14
 
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk2-12
-~That's all I can ask.~
+@56
 EXIT
 
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk2-13
-~I've had a lot of time to ponder death, and many reasons to do so. But I think I've said enough for the time being.~
+@57
 EXIT
 
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk2-14
-~I'm finished. Continue as you will.~
+@58
 EXIT
 
 // Death Talk 3
@@ -291,20 +291,20 @@
 Global("JKL_Helarine_BG1_track","GLOBAL",6)
 Global("JKL_Helarine_ReadytoTalk","GLOBAL",1)~
 THEN JKLHELJ DeathTalk3
-~We've seen a great deal of danger since we've met, <CHARNAME>. Have you ever been afraid that you might not make it through the next battle?~
+@59
 DO ~SetGlobal("JKL_Helarine_BG1_track","GLOBAL",7)
 SetGlobal("JKL_Helarine_ReadytoTalk","GLOBAL",0)
 RealSetGlobalTimer("JKL_Helarine_BG1_track_timer","GLOBAL",1800)~
 END
 
-++ ~I do feel a bit frightened at times, but I try to keep calm during battle.~ + DeathTalk3-1
-++ ~I try not to think about what could happen.~ + DeathTalk3-2
-++ ~I acknowledge the possibility of dying, but leave fear to the cowards.~ + DeathTalk3-3
+++ @60 + DeathTalk3-1
+++ @61 + DeathTalk3-2
+++ @62 + DeathTalk3-3
 
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk3-1
-~A sound plan. Panicking would probably only raise the chances of being killed.~
+@63
 END
 
 IF ~~ THEN GOTO DeathTalk3-4
@@ -312,7 +312,7 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk3-2
-~I suppose that could help overcome the fear during combat, but that sort of thinking seems short-sighted.~
+@64
 END
 
 IF ~~ THEN GOTO DeathTalk3-4
@@ -320,7 +320,7 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk3-3
-~Fear may be debilitating to many, but it is a natural reaction that has little to do with whether one is brave or cowardly.~
+@65
 END
 
 IF ~~ THEN GOTO DeathTalk3-4
@@ -328,22 +328,22 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk3-4
-~I've been watching the way you fight. There seems to be something fueling your ability to slay our foes. If it isn't a fear of dying, what could that motivation be?~
+@66
 END
 
-++ ~I'm honestly not sure what it could be.~ + DeathTalk3-5
-IF ~ReputationLT(Player1,8)~ THEN REPLY ~I like to think I'm making the Sword Coast a safer place.~ + DeathTalk3-6 // low reputation version (need trigger!)
-IF ~!ReputationLT(Player1,8)~ THEN REPLY ~I like to think I'm making the Sword Coast a safer place.~ + DeathTalk3-6 // med/high reputation version (need trigger!)
-++ ~We live in a harsh world, if my travels are anything to go by. I'm just trying to get by.~ + DeathTalk3-8
-++ ~I deserve respect, and that only seems to come from the slaying of monsters and men.~ + DeathTalk3-9
-IF ~!Dead("NEERA") GlobalGT("NEERA_ROMANCE","GLOBAL",1)~ THEN REPLY ~I'm just trying to protect the people I care about.~ + DeathTalk3-10 // Neera version
-IF ~!Dead("RASAAD") !GlobalGT("NEERA_ROMANCE","GLOBAL",1) GlobalGT("RASAAD_ROMANCE","GLOBAL",1)~ THEN REPLY ~I'm just trying to protect the people I care about.~ + DeathTalk3-10 // Rasaad version
-//IF ~!Dead("DORN") !GlobalGT("NEERA_ROMANCE","GLOBAL",1) !GlobalGT("RASAAD_ROMANCE","GLOBAL",1) Global("DORN_ROMANCE_2","LOCALS",2)~ THEN REPLY ~I'm just trying to protect the people I care about.~ + DeathTalk3-10 // Dorn version
+++ @67 + DeathTalk3-5
+IF ~ReputationLT(Player1,8)~ THEN REPLY @68 + DeathTalk3-6 // low reputation version (need trigger!)
+IF ~!ReputationLT(Player1,8)~ THEN REPLY @68 + DeathTalk3-6 // med/high reputation version (need trigger!)
+++ @69 + DeathTalk3-8
+++ @70 + DeathTalk3-9
+IF ~!Dead("NEERA") Global("NEERA_ROMANCE","GLOBAL",2)~ THEN REPLY @71 + DeathTalk3-10 // Neera version
+IF ~!Dead("RASAAD") !Global("NEERA_ROMANCE","GLOBAL",2) Global("RASAAD_ROMANCE","GLOBAL",2)~ THEN REPLY @71 + DeathTalk3-10 // Rasaad version
+IF ~!Dead("DORN") !Global("NEERA_ROMANCE","GLOBAL",2) !Global("RASAAD_ROMANCE","GLOBAL",2) Global("JKLHEL_DornRomance","Global",2)~ THEN REPLY @71 + DeathTalk3-10 // Dorn version
 
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk3-5
-~Whatever drives you, it's made you an effective killer.~
+@72
 END
 
 IF ~~ THEN GOTO DeathTalk3-13
@@ -351,7 +351,7 @@
 CHAIN
 IF~~ // low rep version
 THEN JKLHELJ DeathTalk3-6
-~That sounds like a noble goal, but I wonder if others would agree with that claim. At any rate, you've become an effective killer in your travels.~
+@73
 END
 
 IF ~~ THEN GOTO DeathTalk3-13
@@ -359,7 +359,7 @@
 CHAIN
 IF~~ // high rep version
 THEN JKLHELJ DeathTalk3-7
-~Perhaps you are, although many wish that safety didn't exact so high a price in blood. At any rate, you've become an effective killer in your travels.~
+@74
 END
 
 IF ~~ THEN GOTO DeathTalk3-13
@@ -367,7 +367,7 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk3-8
-~It's interesting that you feel that way, since most people aren't subjected to the traumas you've faced since leaving Candlekeep. It's clear that you've been shaped into a killer by your experience.~
+@75
 END
 
 IF ~~ THEN GOTO DeathTalk3-13
@@ -375,31 +375,31 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk3-9
-~There are many who would disagree with you, but I can see why you might think that slaughter is the only road to respect. It certainly explains how you've become such an accomplished killer.~
+@76
 END
 
 IF ~~ THEN GOTO DeathTalk3-13
 
 CHAIN
-IF~GlobalGT("NEERA_ROMANCE","GLOBAL",1)~
+IF~~
 THEN JKLHELJ DeathTalk3-10
-~Do you mean Neera? While she is hunted by the Red Wizards, she isn't entirely defenseless on her own. Concern for her safety does explain why you have learned to kill so well, though.~
+@77
 END
 
 IF ~~ THEN GOTO DeathTalk3-13
 
 CHAIN
-IF~!GlobalGT("NEERA_ROMANCE","GLOBAL",1) GlobalGT("RASAAD_ROMANCE","GLOBAL",1)~
+IF~~
 THEN JKLHELJ DeathTalk3-11
-~If you're speaking of Rasaad, he's perfectly capable of defending himself. Your association with him might explain how you've become such a skilled killer, however.~
+@78
 END
 
 IF ~~ THEN GOTO DeathTalk3-13
 
 CHAIN
-IF~!GlobalGT("NEERA_ROMANCE","GLOBAL",1) !GlobalGT("RASAAD_ROMANCE","GLOBAL",1) Global("DORN_ROMANCE_2","LOCALS",2)~
+IF~~
 THEN JKLHELJ DeathTalk3-12
-~If anyone is capable of handling things on their own, it would be Dorn. While I doubt you are really concerned for his well-being, his influence may well have shaped you into the killer you've become.~
+@79
 END
 
 IF ~~ THEN GOTO DeathTalk3-13
@@ -407,29 +407,29 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk3-13
-~For whatever reason, it feels as though there's more to your gift for killing than what one might suspect. Maybe it's just my imagination... I'm been exposed to a number of murderers in my lifetime and might have generalized the signs too freely.~
+@80
 END
 
-++ ~Why did you know so many murderers?~ + DeathTalk3-14
-++ ~I'd rather not talk about murder right now...~ + DeathTalk3-15
-++ ~Could we talk about this some other time?~ + DeathTalk3-16
+++ @81 + DeathTalk3-14
+++ @82 + DeathTalk3-15
+++ @83 + DeathTalk3-16
 
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk3-14
-~It's a long story, and not one I'd like to relate right now. Perhaps some other time. For now, let's move on.~
+@84
 EXIT
 
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk3-15
-~I can understand that. Let's move on.~
+@85
 EXIT
 
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk3-16
-~Of course, I didn't mean to take up too much of your time. Let's move on.~
+@86
 EXIT
 
 // Death Talk 4 (always after bandit camp or later)
@@ -439,18 +439,18 @@
 Global("JKL_Helarine_BG1_story1","GLOBAL",0)
 GlobalGT("Chapter","GLOBAL",3)~
 THEN JKLHELJ DeathTalk4
-~Something about you makes me feel uneasy, <CHARNAME>. You managed to survive facing an entire bandit camp with only a small company of allies... not only survive but kill the leaders of the bandits.~
+@87
 DO ~SetGlobal("JKL_Helarine_BG1_story1","GLOBAL",1)~
 END
 
-++ ~I'm a bit surprised myself. I never thought I'd best a few dozen bandits with so little help.~ + DeathTalk4-1
-++ ~I feel far more powerful than I was back in Candlekeep... it's like something inside me was finally set free.~ + DeathTalk4-2
+++ @88 + DeathTalk4-1
+++ @89 + DeathTalk4-2
 
 
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk4-1
-~Exactly... there are, of course, others that might have been able to do what you have. But they all have years if not decades of experience. In all my days of adventuring, I have never seen such a growth in power as I see you now.~
+@90
 END
 
 IF ~~ THEN GOTO DeathTalk4-3
@@ -458,7 +458,7 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk4-2
-~I'm not sure if this hidden power should be set free. I'm sure others could have done what you have... but they all have years of experience. In all my days of adventuring, I have never seen such a growth in power as I see you now.~
+@91
 END
 
 IF ~~ THEN GOTO DeathTalk4-3
@@ -466,17 +466,17 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk4-3
-~This surge of power worries me. I feel like I've seen a similar growth before... during a period of training I underwent fifteen years ago.~
+@92
 END
 
-++ ~Was this when you first became a cleric?~ + DeathTalk4-4
-++ ~What kind of training would involve such a change in one's ability?~ + DeathTalk4-5
-++ ~Tell me more about this training... I want to know how to keep this going.~ + DeathTalk4-6
+++ @93 + DeathTalk4-4
+++ @94 + DeathTalk4-5
+++ @95 + DeathTalk4-6
 
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk4-4
-~Yes. While training as an acolyte, I was asked to kill an enemy that threatened my mentor. This enemy had minions, and so I had to elminate a number of them to reach my target.~
+@96
 END
 
 IF ~~ THEN GOTO DeathTalk4-7
@@ -484,7 +484,7 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk4-5
-~A violent kind of training. While training as an acolyte, I was asked to kill an enemy that threatened my mentor. This enemy had minions, and so I had to elminate a number of them to reach my target.~
+@97
 END
 
 IF ~~ THEN GOTO DeathTalk4-7
@@ -492,7 +492,7 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk4-6
-~I can't say how to maintain this growth, only when I felt it before. While training as an acolyte, I was asked to kill an enemy that threatened my mentor. This enemy had minions, and so I had to elminate a number of them to reach my target.~
+@98
 END
 
 IF ~~ THEN GOTO DeathTalk4-7
@@ -500,16 +500,16 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk4-7
-~With each kill, I felt a surge of power... like a gift representing approval. By the time I had my mentor's enemy at my mercy, I felt like I could take on an army without aid.~
+@99
 END
 
-++ ~What god did you worship that required such slaughter?~ + DeathTalk4-8
-++ ~And yet you don't have this power now... why is that?~ + DeathTalk4-9
+++ @100 + DeathTalk4-8
+++ @101 + DeathTalk4-9
 
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk4-8
-~It... would be better for you not to know. What matters is that I don't have this power any longer.~
+@102
 END
 
 IF ~~ THEN GOTO DeathTalk4-10
@@ -517,7 +517,7 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk4-9
-~It faded once I returned to my more mundane training... whatever entity granted me the power felt I no longer needed it, I assume.~
+@103
 END
 
 IF ~~ THEN GOTO DeathTalk4-10
@@ -525,17 +525,17 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk4-10
-~I don't know if my temporary gift is at all related to your sudden growth... in fact, I don't believe it's even possible for the two to be connected... but whatever the source there is a common goal in this power: death.~
+@104
 END
 
-++ ~Is it possible that Kelemvor has granted me power? He is the god of the dead, after all.~ + DeathTalk4-11
-++ ~I suppose I should be careful, then. I don't want this power getting out of control.~ + DeathTalk4-12
-++ ~Whatever this power is, I intend to keep it as long as I can.~ + DeathTalk4-13
+++ @105 + DeathTalk4-11
+++ @106 + DeathTalk4-12
+++ @107 + DeathTalk4-13
 
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk4-11
-~No... this power is nothing like Kelemvor's blessings. It revels in death... its source is likely far more sinister.~
+@108
 END
 
 IF ~~ THEN GOTO DeathTalk4-14
@@ -543,7 +543,7 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk4-12
-~That would be wise. This power revels in death... its source is certainly sinister in nature.~
+@109
 END
 
 IF ~~ THEN GOTO DeathTalk4-14
@@ -551,7 +551,7 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk4-13
-~You may wish to reconsider... this power revels in death. I'm certain that it originated in a sinister source.~
+@110
 END
 
 IF ~~ THEN GOTO DeathTalk4-14
@@ -559,7 +559,7 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk4-14
-~Excuse me... there is something I have to contemplate. Be wary of your new power, as I'm sure it hasn't been granted for the sake of good deeds.~
+@111
 EXIT
 
 // Death Talk 5 (always after BG enter or later)
@@ -569,18 +569,18 @@
 Global("JKL_Helarine_BG1_story2","GLOBAL",0)
 GlobalGT("Chapter","GLOBAL",4)~
 THEN JKLHELJ DeathTalk5
-~<CHARNAME>, do you fear death? I'm sure I asked this before, but this time I mean a permanent death... where nothing of you remains.~
+@112
 DO ~SetGlobal("JKL_Helarine_BG1_story2","GLOBAL",1)~
 END
 
-++ ~What brought this on? Is there something wrong?~ + DeathTalk5-1
-++ ~What sort of power could end someone's existance entirely?~ + DeathTalk5-2
-++ ~Are you trying to frighten me? Get to the point.~ + DeathTalk5-3
+++ @113 + DeathTalk5-1
+++ @114 + DeathTalk5-2
+++ @115 + DeathTalk5-3
 
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk5-1
-~There is something wrong... I've remembered another time I witness someone gaining as much power as you have in a short time.~
+@116
 END
 
 IF ~~ THEN GOTO DeathTalk5-4
@@ -588,7 +588,7 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk5-2
-~If my memory isn't decieving me, your power can end a life forever. I've remembered another time I witness someone gaining as much power as you have in a short time.~
+@117
 END
 
 IF ~~ THEN GOTO DeathTalk5-4
@@ -596,7 +596,7 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk5-3
-~I'm not trying to frighten you, only warn you of something. I've remembered another time I witness someone gaining as much power as you have in a short time.~
+@118
 END
 
 IF ~~ THEN GOTO DeathTalk5-4
@@ -604,20 +604,20 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk5-4
-~About... seven years ago I travelled with a young man, barely more than a boy. I was training him as an acolyte for the god I once followed... though I expected my efforts to be in vain, as that god had been killed during the Time of Troubles.~
-=~I was... excited when he manifested the power of my god. I thought that perhaps the god had simply been in hiding, and had chosen this boy to return to power.~
-=~The boy grew immensly powerful in a matter of months... he was the perfect killer by the end of the year. But worshippers of Cyric discovered him and eventually overwhelmed him.~
-=~As I crept away from the carnage to avoid a similar fate, I looked back... to see the boy disintegrate into ashes that in turn faded away.~
+@119
+=@120
+=@121
+=@122
 END
 
-++ ~That sounds like a horrible price for power...~ + DeathTalk5-5
-++ ~Isn't there magic to bring back those who die in such terrible ways?~ + DeathTalk5-6
-++ ~The boy should have known better than to reveal his power. It's wasted on the dead.~ + DeathTalk5-7
+++ @123 + DeathTalk5-5
+++ @124 + DeathTalk5-6
+++ @125 + DeathTalk5-7
 
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk5-5
-~It's even more horrifying to witness... especially if you had pinned all your hopes on the person who faded away.~
+@126
 END
 
 IF ~~ THEN GOTO DeathTalk5-8
@@ -625,7 +625,7 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk5-6
-~There may be, but there are only a scant few people in the realms capable of such feats... the boy in my story is almost certainly gone forever.~
+@127
 END
 
 IF ~~ THEN GOTO DeathTalk5-8
@@ -633,7 +633,7 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk5-7
-~Such power is dangerous even among the living. But I agree with you in one respect... if there is no way to rid yourself of this power, it is best kept hidden.~
+@128
 END
 
 IF ~~ THEN GOTO DeathTalk5-8
@@ -641,8 +641,8 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk5-8
-~Your power feels much like the one that infused the boy I trained all those years ago... if it comes from the same source, it's entirely possible that you'll fade away forever once you die.~
-=~I can't ask you to be any more careful than you have been so far... but keep in mind that power like yours always comes with a price.~
+@129
+=@130
 EXIT
 
 // Death Talk 6 (always after CK bhaal reveal or later)
@@ -653,69 +653,69 @@
 GlobalGT("Chapter","GLOBAL",5)
 Global("JKLHEL_BhaalReveal","GLOBAL",1)~
 THEN JKLHELJ DeathTalk6
-~I shouldn't have doubted. I should have known better... I should have... damn!~
-DO ~Global("JKL_Helarine_BG1_story3","GLOBAL",1)
+@131
+DO ~SetGlobal("JKL_Helarine_BG1_story3","GLOBAL",1)
 SetGlobal("JKLHEL_BhaalReveal","GLOBAL",2)~
 END
 
-++ ~Helarine, this is hardly the time to break down! What's gotten into you?~ + DeathTalk6-1
-++ ~What shouldn't you have doubted? What are you sobbing about?~ + DeathTalk6-1
+++ @132 + DeathTalk6-1
+++ @133 + DeathTalk6-1
 
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk6-1
-~Your power... it was so familiar to me because it's the same power I was trained to use so many years ago. The power of Bhaal.~
+@134
 END
 
-++ ~We couldn't have known that I was the <SONDAUGHTER> of Bhaal, Helarine.~ + DeathTalk6-2
-++ ~That's it? Get a hold of yourself, Helarine. We have more pressing troubles!~ + DeathTalk6-2
+++ @135 + DeathTalk6-2
+++ @136 + DeathTalk6-2
 
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk6-2
-~You don't understand, <CHARNAME>. I don't expect you to. You haven't died in the name of a dead god.~
+@137
 END
 
-++ ~Hold on, what are you saying?~ + DeathTalk6-3
-++ ~What have you been hiding from me?~ + DeathTalk6-3
-++ ~Out with it! I deserve to know why this is such a problem for you.~ + DeathTalk6-3
+++ @138 + DeathTalk6-3
+++ @139 + DeathTalk6-3
+++ @140 + DeathTalk6-3
 
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk6-3
-~During the Time of Troubles, my parents, priests of Bhaal, were killed by followers of Cyric after Bhaal's death. I barely escaped the attack, and only because no one recognized me as an acolyte of Bhaal.~
-=~For years after the Time of Troubles, I hunted down and murdered worshippers of Cyric, praying that if I did enough damage to his followers that other supporters of Bhaal would try to resurrect him to punish Cyric for murdering him.~
-=~I met another of your kind, a son of Bhaal, and I thought he would bring about Bhaal's rebirth... but then Cyric's followers stole that from me as well.~
+@141
+=@142
+=@143
 END
 
-++ ~I don't understand. You said you died, but you're still here.~ + DeathTalk6-4
-++ ~Get to the point, Helarine! You claimed you died for Bhaal, why do you still draw breath?~ + DeathTalk6-4
+++ @144 + DeathTalk6-4
+++ @145 + DeathTalk6-4
 
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk6-4
-~With the son of Bhaal dead and gone, I became desperate. I killed more than I needed to, and eventually I picked a target far beyond my means.~
-=~This was almost two years ago, when Kelemvor took on the mantle of god of the dead. The priest I had set out to kill had converted to Kelemvor, and was powerful enough to kill me with a single spell.~
-=~In the Fugue Plane, I was terrified of my judgement. My god was dead, there was no place for me but the Wall of the Faithless. In desperation, I offered prayer to Kelemvor.~
-=~Kelemvor must have communicated with the priest I had tried to kill, because I was raised from the dead by that very woman. That is how I came to serve Kelemvor.~
+@146
+=@147
+=@148
+=@149
 END
 
-++ ~That still doesn't explain why you nearly broke down just now.~ + DeathTalk6-5
-++ ~A fine story, but I want to know why you can't get a hold of yourself.~ + DeathTalk6-5
+++ @150 + DeathTalk6-5
+++ @151 + DeathTalk6-5
 
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk6-5
-~Don't you understand? I spent ten years killing for a dead god, and I finally am given the chance to redeem myself. Now... everything I've tried to forsake is incarnate in you! How can you not understand why I might be upset?!~
+@152
 END
 
-++ ~Helarine, this isn't as bad as it looks. I'm not Bhaal. We need you to keep your head until we get out of here, then we can talk.~ + DeathTalk6-6
-++ ~We don't have time for this, Helarine! Pull yourself together unless you want to be left behind.~ + DeathTalk6-6
+++ @153 + DeathTalk6-6
+++ @154 + DeathTalk6-6
 
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk6-6
-~I... you're right, I'm overreacting. Give me a moment to... calm down. We... no. I don't want to talk about this. I have to deal with this on my own. Lead... lead on.~
+@155
 EXIT
 
 // Death Talk 7 (4th friendship track convo)
@@ -725,45 +725,45 @@
 Global("JKL_Helarine_BG1_track","GLOBAL",8)
 Global("JKL_Helarine_ReadytoTalk","GLOBAL",1)~
 THEN JKLHELJ DeathTalk7
-~<CHARNAME>, I'm curious about something. What do you plan to do once this iron crisis is dealt with?~
+@156
 DO ~SetGlobal("JKL_Helarine_BG1_track","GLOBAL",9)
 SetGlobal("JKL_Helarine_ReadytoTalk","GLOBAL",0)~
 END
 
-++ ~Is that a trick question?...~ + DeathTalk7-1
-++ ~I haven't given it much thought yet.~ + DeathTalk7-2
-++ ~I've been too busy with my adventures to consider that.~ + DeathTalk7-2
+++ @157 + DeathTalk7-1
+++ @158 + DeathTalk7-2
+++ @159 + DeathTalk7-2
 
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk7-1
-~No, not a trick question. I'm just curious. You don't strike me as the type to settle down... I figure you'll continue to travel even after the iron crisis has been dealt with.~
+@160
 END
 
-++ ~What do you plan to do once this is over?~ + DeathTalk7-3
+++ @161 + DeathTalk7-3
 
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk7-2
-~I thought as much. You don't strike me as the type to settle down... I figure you'll continue to travel even after the iron crisis has been dealt with.~
+@162
 END
 
-++ ~What do you plan to do once this is over?~ + DeathTalk7-3
+++ @161 + DeathTalk7-3
 
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk7-3
-~I still have to check on someone I was supposed to meet in Daggerford, but after that I'm not sure. I've heard rumors that one of my order in someplace called Tradesmeet in Amn needed assistance... perhaps I'll head there next.~
+@163
 END
 
-++ ~I'll miss having you around. You've been a great help to this group.~ + DeathTalk7-4
-++ ~I hope things in Tradesmeet aren't as hectic as they are here.~ + DeathTalk7-5
-++ ~Seems fitting that you get sent on long journeys by your fellows. They must not be able to stand you for long.~ + DeathTalk7-6
+++ @164 + DeathTalk7-4
+++ @165 + DeathTalk7-5
+++ @166 + DeathTalk7-6
 
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk7-4
-~That's kind of you to say so. I'll probably miss the excitement of being at the heart of a crisis like this... but I won't miss the stress.~
+@167
 END
 
 IF ~~ THEN GOTO DeathTalk7-7
@@ -771,7 +771,7 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk7-5
-~I'm sure the city itself is fine... though perhaps that won't be the case when I arrive. I only hope the Doomguide staying there is still safe when I reach him.~
+@168
 END
 
 IF ~~ THEN GOTO DeathTalk7-7
@@ -779,7 +779,7 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk7-6
-~I'm going to pretend that's an insult to other Kelemvorites and not an insult meant for me. You may wish to watch your tongue when speaking to your other allies, <CHARNAME>.~
+@169
 END
 
 IF ~~ THEN GOTO DeathTalk7-7
@@ -787,16 +787,16 @@
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk7-7
-~At any rate, we still have to find a way to end the Sword Coast's problems.~
+@170
 END
 
-IF ~Global("JKLHEL_BhaalReveal","GLOBAL",2)~ THEN REPLY ~If we foil Sarevok's plans, I'm sure the rest will sort itself out.~ + DeathTalk7-8
-IF ~GlobalGT("CHAPTER","GLOBAL",3)~ THEN REPLY ~If we can find a way to stop the Iron Throne's plans, I'm sure the rest will sort itself out.~ + DeathTalk7-8
-IF ~GlobalLT("CHAPTER","GLOBAL",4)~ THEN REPLY ~If we can figure out and stop whoever started this iron plague, I'm sure the rest will sort itself out.~ + DeathTalk7-8
+IF ~Global("JKLHEL_BhaalReveal","GLOBAL",2)~ THEN REPLY @171 + DeathTalk7-8
+IF ~GlobalGT("CHAPTER","GLOBAL",3)~ THEN REPLY @172 + DeathTalk7-8
+IF ~GlobalLT("CHAPTER","GLOBAL",4)~ THEN REPLY @173 + DeathTalk7-8
 
 CHAIN
 IF~~
 THEN JKLHELJ DeathTalk7-8
-~I hope you're right. It would be a shame to make plans for what we'll do after this is all over then end up failing.~
-=~I've taken enough of your time. Let's bring this iron crisis to an end. Everything dies eventually, after all, and this conflict will be no different.~
+@174
+=@175
 EXIT
