--- worgas\are\W_0602.baf	Sat Apr 14 23:48:52 2007
+++ C:\BWP Patchstudio\patched files\worgas\are\W_0602.baf	Thu Apr 29 13:00:09 2010
@@ -5,13 +5,13 @@
 SetGlobal("ZarixExists","AR0602",1)
 CreateCreature("zarix",[1399.1495],2)
 StorePartyLocations()
-    ActionOverride(Player1,LeaveAreaLUA("AR0070","",[0.0],0)
+    ActionOverride(Player1,LeaveAreaLUA("AR0070","",[0.0],0))
     ActionOverride(Player2,LeaveAreaLUA("AR0070","",[0.0],0))
     ActionOverride(Player3,LeaveAreaLUA("AR0070","",[0.0],0))
     ActionOverride(Player4,LeaveAreaLUA("AR0070","",[0.0],0))
     ActionOverride(Player5,LeaveAreaLUA("AR0070","",[0.0],0))
     ActionOverride(Player6,LeaveAreaLUA("AR0070","",[0.0],0))
-   ActionOverride(Player1,MakeUnselectable(20)
+   ActionOverride(Player1,MakeUnselectable(20))
     ActionOverride(Player2,MakeUnselectable(20))
     ActionOverride(Player3,MakeUnselectable(20))
     ActionOverride(Player4,MakeUnselectable(20))
@@ -26,7 +26,7 @@
 THEN
 RESPONSE #100
 MoveViewPoint([3654.2889],INSTANT) 
-    ActionOverride(Player1,MakeUnselectable(20)
+    ActionOverride(Player1,MakeUnselectable(20))
     ActionOverride(Player2,MakeUnselectable(20))
     ActionOverride(Player3,MakeUnselectable(20))
     ActionOverride(Player4,MakeUnselectable(20))
@@ -53,7 +53,7 @@
 ActionOverride("Worgas",PlayDead(60))
 Wait(2)
 CreateVisualEffectObject("spcontin","Worgas")
-ActionOverride("Worgas",DisplayStringHead(Myself,60613)
+ActionOverride("Worgas",DisplayStringHead(Myself,60613))
 EndCutSceneMode()
 END
 
