--- Setup-RFWIP.tp2	Sat Jul 10 02:29:50 2004
+++ C:\BWP Patchstudio\patched files\Setup-RFWIP.tp2	Tue Jun 30 21:44:25 2015
@@ -2,6 +2,7 @@
 
 BACKUP ~RFWIP/Backup~
 AUTHOR ~mrennigma@aol.com~
+VERSION ~v1.0 BWP Fix~
 AUTO_TRA ~RFWIP/Translations/%s~
 ALLOW_MISSING ~AR6400.bcs~ ~Baldur25.bcs~
 
@@ -135,8 +136,8 @@
 COPY ~RFWIP/Items/FWRing09.itm~ ~override/FWRing09.itm~ SAY NAME2 @242 SAY DESC @243
 COPY ~RFWIP/Items/FWRing0A.itm~ ~override/FWRing0A.itm~ SAY NAME2 @244 SAY DESC @245
 
-EXTEND_BOTTOM ~Baldur.bcs~ ~RFWIP/Scripts/Sting.bcs~
-EXTEND_BOTTOM ~Baldur25.bcs~ ~RFWIP/Scripts/Sting.bcs~
+// EXTEND_BOTTOM ~Baldur.bcs~ ~RFWIP/Scripts/Sting.baf~
+// EXTEND_BOTTOM ~Baldur25.bcs~ ~RFWIP/Scripts/Sting.baf~
 /* V0.2.1 */
 
 //COPY ~RFWIP/Spells/FWWI901.spl~  ~override/FWWI901.spl~  SAY 0x08 @10 SAY 0x50 @11
@@ -148,8 +149,8 @@
 
 COPY ~RFWIP/Creatures/FWShop01.cre~ ~override/FWShop01.cre~ SAY NAME1 #9408 SAY NAME2 #9408 
 
-EXTEND_TOP ~AR0700.bcs~ ~RFWIP/Scripts/WPPatch.bcs~
-EXTEND_TOP ~AR6400.bcs~ ~RFWIP/Scripts/SarPatch.bcs~
+EXTEND_TOP ~AR0700.bcs~ ~RFWIP/Scripts/WPPatch.baf~
+EXTEND_TOP ~AR6400.bcs~ ~RFWIP/Scripts/SarPatch.baf~
 
 COPY ~RFWIP/Stores/FWStor01.sto~ ~override/FWStor01.sto~ SAY 0xC @184
 
@@ -169,7 +170,7 @@
 COPY ~RFWIP/Items/FWBelt04.itm~ ~override/FWBelt04.itm~ SAY NAME2 @172 SAY DESC @173
 COPY ~RFWIP/Items/FWBelt05.itm~ ~override/FWBelt05.itm~ SAY NAME2 @174 SAY DESC @175
 COPY ~RFWIP/Items/FWBelt06.itm~ ~override/FWBelt06.itm~
-EXTEND_BOTTOM ~Anom25.bcs~ ~RFWIP/Scripts/AnoHoly.bcs~
-EXTEND_BOTTOM ~Aeri25.bcs~ ~RFWIP/Scripts/AerHoly.bcs~
-EXTEND_BOTTOM ~Vico25.bcs~ ~RFWIP/Scripts/VicHoly.bcs~
+EXTEND_BOTTOM ~Anom25.bcs~ ~RFWIP/Scripts/AnoHoly.baf~
+EXTEND_BOTTOM ~Aeri25.bcs~ ~RFWIP/Scripts/AerHoly.baf~
+EXTEND_BOTTOM ~Vico25.bcs~ ~RFWIP/Scripts/VicHoly.baf~
 
