--- WildMage/Setup-WildMage.TP2	2009-07-17 12:05:36.312500000 -0400
+++ Fixpack/WildMage/Setup-WildMage.TP2	2016-11-28 00:11:52.139501100 -0500
@@ -1,5 +1,25 @@
 BACKUP ~WildMage/backup~
 AUTHOR ~jo.ge@gmx.net~
+VERSION ~v1.7 BWP Fix~
+
+ALWAYS
+  ACTION_IF ENGINE_IS ~bgee bg2ee iwdee eet~ BEGIN
+    ACTION_DEFINE_ARRAY charsetsNoConvertArray BEGIN END
+    ACTION_DEFINE_ARRAY charsetsReloadArray BEGIN text END
+    ACTION_IF (~%WEIDU_OS%~ STRING_EQUAL_CASE ~win32~) THEN BEGIN // HANDLE_CHARSETS doesn't work with iconv_path containing spaces
+      AT_NOW ~if not exist "iconv" move "BiG World Fixpack\_utils\iconv" "iconv"~
+    END
+    LAF ~HANDLE_CHARSETS~
+      INT_VAR
+        infer_charsets = 1
+      STR_VAR
+        tra_path = EVAL "%MOD_FOLDER%/translations"
+        iconv_path = ~iconv~ // provided by BiG World Fixpack
+        noconvert_array = charsetsNoConvertArray
+        reload_array = charsetsReloadArray
+    END
+  END
+END
 
 LANGUAGE ~English~
          ~english~
@@ -28,7 +48,7 @@
 //******************************************************
 
 BEGIN @1
-REQUIRE_FILE ~Data/25Dialog.bif~ @10
+REQUIRE_PREDICATE GAME_IS ~tob tutu tutu_totsc bgee bg2ee iwdee eet~ @10
 
 //------------------------------------------------------
 //		Mod-start in Baldur.BCS
@@ -328,10 +348,17 @@
 // BGT Compatibility (Retain Spellbook on Transfer from BG1 to BG2)
 //-----------------------------------------------------------------
 
-ACTION_IF FILE_EXISTS ~DATA/BG1ARE.BIF~ THEN BEGIN
 
-EXTEND_TOP ~aram00.bcs~ ~WildMage/0/baf/wm_aram00.baf~
-EXTEND_TOP ~ar0602.bcs~ ~WildMage/0/baf/wm_ar0602.baf~
+
+ACTION_IF GAME_IS ~bgt eet~ BEGIN
+
+  ACTION_IF GAME_IS ~bgt~ BEGIN
+    EXTEND_TOP ~aram00.bcs~   ~WildMage/0/baf/wm_aram00.baf~ // extend BGT transition script
+  END ELSE ACTION_IF GAME_IS ~eet~ BEGIN  
+    EXTEND_TOP ~K#TELBGT.BCS~ ~WildMage/0/baf/wm_aram00.baf~ // extend EET transition script
+  END
+
+  EXTEND_TOP ~ar0602.bcs~ ~WildMage/0/baf/wm_ar0602.baf~ // after transition, place book behind portrait
 
 END
 
@@ -636,20 +663,20 @@
 //-----------------------------------------------------------------------------------
 
 COPY_EXISTING ~baldur.bcs~ ~override~
-DECOMPILE_BCS_TO_BAF
+  DECOMPILE_AND_PATCH BEGIN
 REPLACE_TEXTUALLY ~Global("wm_start","GLOBAL",0)~
 ~Global("wm_start","GLOBAL",0)
 Global("ENDOFBG1","GLOBAL",2)
 !Global("wm_bgt","GLOBAL",2)~
-COMPILE_BAF_TO_BCS
+  END
 
 COPY_EXISTING ~baldur25.bcs~ ~override~
-DECOMPILE_BCS_TO_BAF
+  DECOMPILE_AND_PATCH BEGIN
 REPLACE_TEXTUALLY ~Global("wm_start","GLOBAL",0)~
 ~Global("wm_start","GLOBAL",0)
 Global("ENDOFBG1","GLOBAL",2)
 !Global("wm_bgt","GLOBAL",2)~
-COMPILE_BAF_TO_BCS
+  END
 
 EXTEND_TOP ~baldur.bcs~ ~WildMage/4/baf/BALDUR_ADDBGT.BAF~
 EXTEND_TOP ~baldur25.bcs~ ~WildMage/4/baf/BALDUR_ADDBGT.BAF~
@@ -674,23 +701,23 @@
   SAY DESC @69999
   
 
-ACTION_IF FILE_EXISTS ~DATA/BG1ARE.BIF~ THEN BEGIN			// Outside Candlekeep BGT
-
-EXTEND_TOP ~ar6526.bcs~ ~WildMage/4/baf/wm_bgt.baf~
-COPY_EXISTING ~Gorion.cre~ ~override~
-ADD_CRE_ITEM ~wm_scrl~ #0 #0 #0 ~IDENTIFIED~ ~INV16~
-
+ACTION_IF GAME_IS ~bgee~ BEGIN
+  OUTER_SPRINT gorion_cre ~GORION~
+  OUTER_SPRINT Candlekeep_Ch6_BCS ~AR2626~
+END ELSE ACTION_IF GAME_IS ~eet~ BEGIN
+  OUTER_SPRINT gorion_cre ~GORION~
+  OUTER_SPRINT Candlekeep_Ch6_BCS ~BG2626~
+END ELSE ACTION_IF GAME_IS ~bgt~ BEGIN
+  OUTER_SPRINT gorion_cre ~GORION~
+  OUTER_SPRINT Candlekeep_Ch6_BCS ~AR6526~
+END ELSE ACTION_IF GAME_IS ~tutu tutu_totsc~ BEGIN
+  OUTER_SPRINT gorion_cre ~_GORION~
+  OUTER_SPRINT Candlekeep_Ch6_BCS ~_AR2626~
 END
 
-
-ACTION_IF FILE_EXISTS ~OVERRIDE/FW2700.ARE~ THEN BEGIN			// Outside Candlekeep BG1-TUTU 
-
-EXTEND_TOP ~_AR2700.BCS~ ~WildMage/4/baf/wm_bgt.baf~
-COPY_EXISTING ~_GORION.CRE~ ~override~
+ACTION_IF GAME_IS ~bgee eet bgt tutu tutu_totsc~ BEGIN // Outside Candlekeep
+  EXTEND_TOP ~%Candlekeep_Ch6_BCS%.bcs~ ~WildMage/4/baf/wm_bgt.baf~
+  COPY_EXISTING ~%gorion_cre%.cre~ ~override~
 ADD_CRE_ITEM ~wm_scrl~ #0 #0 #0 ~IDENTIFIED~ ~INV16~
-
 END
 
-
-
-
