--- uniqueartifacts\core\macro.tph	Thu Sep 13 10:09:56 2007
+++ C:\BWP Patchstudio\patched files\uniqueartifacts\core\macro.tph	Fri May 09 02:32:54 2008
@@ -125,9 +125,9 @@
       SET index = (index + 1)
     END
     //correcting offsets
-    PATCH_FOR_EACH offset IN 0x54 0x5c 0x60 0x68 0x70 0x78 0x7c 0x84 0x88 0xa0 0xa8 0xb0 0xb8 0xbc 0xc0 0xc4 BEGIN
+    PATCH_FOR_EACH offset IN 0x54 0x5c 0x60 0x68 0x70 0x7c 0x84 0x88 0xa0 0xa8 0xb0 0xb8 0xbc 0xc0 0xc4 BEGIN
       READ_LONG offset current_off
-      PATCH_IF (current_off > itm_off) BEGIN
+      PATCH_IF (current_off >= itm_off) BEGIN
         WRITE_LONG offset (current_off + 0x14)
       END
     END
