--- AC_QUEST\Setup-AC_QUEST.TP2	Wed Feb 10 19:55:01 2016
+++ C:\BWP Patchstudio\patched files\AC_QUEST\Setup-AC_QUEST.TP2	Tue Nov 01 09:14:06 2016
@@ -1,7 +1,7 @@
 BACKUP ~AC_QUEST/backup~
 AUTHOR ~whiteagnus@baldursgatemods.de~ //This is Ascalon's Questpack, but any bugreports about this version should go to jastey
 
-VERSION ~2.00b_160119~
+VERSION ~2.00b_160119 BWP Fix~
 
 AUTO_TRA "AC_QUEST/TRA/autotra/%s"//%"
 
