--- AC_QUEST\tra\english\setup-AC_QUEST.tra	Wed Feb 10 19:55:01 2016
+++ C:\BWP Patchstudio\patched files\AC_QUEST\tra\english\setup-AC_QUEST.tra	Tue Nov 01 09:14:44 2016
@@ -364,9 +364,9 @@
 Thalantyr has enumerated several ways. He can mix the potion with Aiwell's blood, which is highly questionable. Or he can use Guril berries that do not always work. The best would be an elixir of belladonna. He also told us that he'll only be able to brew one of the potions.
 
 I could also bring him the corpse of Thonder so that he can better understand the werewolf disease.~
-@10035 = ~Von Wölfen und Männern
+@10035 = ~Of Wolves and Men
 
-Thalantyr hat den Leichnam von Thonder mit einem Flammenschwert erkauft. Auch nicht schlecht.~
+Thalantyr has bought the dead body of Thonder, giving in return a Flaming Sword. Not bad either.~
 @10036 = ~Of Wolves and Men
 
 Aiwell and Thonder were overjoyed that the man was freed from his curse.~
