--- Setup-NSCPortraits.TP2	Thu Mar 12 17:41:44 2015
+++ C:\BWP Patchstudio\patched files\Setup-NSCPortraits.TP2	Wed May 13 08:31:00 2015
@@ -482,7 +482,7 @@
 
 
 BEGIN ~Portraits for Tower of Deception~ 
-REQUIRE_PREDICATE FILE_EXISTS_IN_GAME ~VA#TIAN~ ~You haven't installed ToD, skip this component~ 
+REQUIRE_PREDICATE FILE_EXISTS_IN_GAME ~VA#TIAN.cre~ ~You haven't installed ToD, skip this component~ 
 DESIGNATED 1300
 
 OUTER_SPRINT ~folder~ ~TOD~
