--- bg1npc/bg1npc.tp2	Wed Oct 22 03:05:30 2014
+++ C:\BWP Patchstudio\patched files\bg1npc.tp2	Wed Oct 22 07:34:46 2014
@@ -15,7 +15,7 @@
 /* MODDER */
 
 /* WeiDU versions newer than v203 allow tp2 level version listing in the .log for troubleshooting  */
-VERSION ~v21_20150210~
+VERSION ~v21_20150210 BWP Fix~
 /* Launch the readme file immediately. */
 /* Isaya : display in native selected language if available, otherwise english */
 README ~bg1npc/readme-bg1npc-%LANGUAGE%.html~ ~bg1npc/readme-bg1npc.html~
