--- bg1npc\tra\german\bg1npc_tmp.tra	Sun Jun 14 09:06:31 2015
+++ C:\BWP Patchstudio\patched files\bg1npc\tra\german\bg1npc_tmp.tra	Sun Sep 20 23:36:04 2015
@@ -720,7 +720,8 @@
 
 In genau diesem Moment lacht die Gestalt auf, mit dem körperlichen, triumphierenden Lachen eines satten Raubtiers, einen Arm gen Boden deutend. "Bodhi", plötzlich fällt Euch ihr Name ein und dann seht Ihr es: Einen Körper in der hintersten Ecke des Raumes, in einem Haufen von Müll, zerschmettert und zerstört. Das elfische Blut in Euch kocht und friert gleichzeitig, flüstern in Euch von kalter Endgültigkeit: Es ist tot, jenseits aller Hoffnung auf die Auferstehung, die Seele verschwunden - verloren, und selbst die Seldarine können sie nicht mehr zurückbringen.
 
-Ihr wisst, von wem sie spricht, aber Ihr habt keine Zeit, den Namen laut auszusprechen. Stattdessen verschwindet der Raum in der Schwärze und Ihr stürzt mit ihm in die dunklen Tiefen des traumlosen Schlafes, die Vision verblasst langsam aus Eurem Gedächtnis.~@476  = ~[MAREK 1] Habt Ihr mal 'nen Moment Zeit für uns?~ [%tutu_var%MAREK01]
+Ihr wisst, von wem sie spricht, aber Ihr habt keine Zeit, den Namen laut auszusprechen. Stattdessen verschwindet der Raum in der Schwärze und Ihr stürzt mit ihm in die dunklen Tiefen des traumlosen Schlafes, die Vision verblasst langsam aus Eurem Gedächtnis.~
+@476  = ~[MAREK 1] Habt Ihr mal 'nen Moment Zeit für uns?~ [%tutu_var%MAREK01]
 @477  = ~Blumenanhänger~
 @478  = ~Verzauberter Blumenanhänger
 
