--- bg1npc\tra\polish\bg1npc_tmp.tra	Tue Mar 24 00:40:44 2015
+++ C:\BWP Patchstudio\patched files\bg1npc\tra\polish\bg1npc_tmp.tra	Sun Sep 20 23:36:04 2015
@@ -1090,7 +1090,8 @@
 @718 = ~[NEREID 08]~ [%tutu_var%NERED08]
 @719 = ~[NEREID 09]~ [%tutu_var%NERED09]
 @720  = ~Wódz Sahuaginów~
-@721  = ~Sahuaginów~@722 = ~~ [SAHAU01]
+@721  = ~Sahuaginów~
+@722 = ~~ [SAHAU01]
 @723 = ~~ [SAHAU02]
 @724 = ~~ [SAHAU03]
 @725 = ~~ [SAHAU04]
