--- bg1npc\tra\english\p#brlt.tra	Tue Nov 25 00:08:02 2014
+++ C:\BWP Patchstudio\patched files\bg1npc\tra\english\p#brlt.tra	Sun Sep 20 23:36:04 2015
@@ -83,7 +83,7 @@
 @82   = ~That comes dangerously close to the worst sort of heresy, <CHARNAME>. Weigh and ponder your words carefully, for no deity tolerates such cool disdain and ignorance, and Tempus above all.~
 @83   = ~'Tis flattering that you indulge my vanity, <CHARNAME>, but I will not allow you to lead me astray with your jesting. Women and wine soften your body outside the battle, while faith strengthens it.~
 @84   = ~Allow me to tell you of Tempus, and then you will see whether your heart listens to what your head does not.~
-@85   = ~I will not bother you with long and tedious recitals. Tempus sees true devotion, and does not favour smooth-tongued tricksters.~
+@85   = ~I will not bother you with long and tedious recitals. Tempus sees true devotion, and does not favor smooth-tongued tricksters.~
 @86   = ~War is a force of change, <CHARNAME>, fair and pure, and it should not be feared. Courage, decisiveness and skill is required of a warrior on the battlefield - Tempus does not require that you engage in a hopeless battle, but that if you do, you halt it quickly and resolutely, never underestimating your foe. The goal and challenge of every battle is personal betterment, and thus honorable demeanor is as important to the warrior as mother's milk is to the newborn babe.~
 @87   = ~Thank you, Branwen.~
 @88   = ~That is the way I live and fight, Tempus or not.~
