--- bg1npc\tra\polish\p#centeo.tra	Mon Apr 21 22:45:12 2014
+++ C:\BWP Patchstudio\patched files\bg1npc\tra\polish\p#centeo.tra	Sun Sep 20 23:36:04 2015
@@ -31,7 +31,7 @@
 @26   = ~Kobieto, czy nie rozumiesz, ¿e to wcale nie uczyni ciê szczêœliwsz¹?~
 @27   = ~Oczywiœcie, ¿e uczyni. Przywrócisz mi me dawne ¿ycie!~
 @28   = ~Tak, ale skoro to ¿ycie - och, o co ta sprzeczka? Odzyskaj swe dawne cia³o.~
-@29   = ~Moje cia³o... Och, znów jestem piêkna! Jestem wspania³a...~ [ELFFF03]
+@29   = ~Moje cia³o... Och, znów jestem piêkna! Jestem wspania³a...~
 @30   = ~Wzbiera burza mej furii... Niebo op³akuje me rany...~
 @31   = ~Zemsta, to wszystko co mam. Nie ma nic poza ni¹. Zemsta za to, co Jon ze mn¹ zrobi³.~
 @32   = ~I.. JA... JEJ... DOKONAM!~
