--- bg1npc\tra\german\p#centeo.tra	Tue Sep 02 08:17:04 2014
+++ C:\BWP Patchstudio\patched files\bg1npc\tra\german\p#centeo.tra	Sun Sep 20 23:36:04 2015
@@ -1,38 +1,39 @@
-@0    = ~Wait... I will not resist. Kill me, but first I want to tell you of my curse - to take it off my chest.~
-@1    = ~My curse - that is all I remember now. I used to be beautiful and powerful.~
-@2    = ~An exotic sorceress, with many powers at my command, and suitors at my door. But I only had eyes for one man, Jon Irenicus. He was a great and powerful wizard, the only man worthy of my affections, or so I thought. Though I lusted for Jon, he cared little for me, for he had another to whom he was married, lady Tanova.~
-@3    = ~Please, go on, that's quite a story!~
-@4    = ~Jon was married?! I... I think I heard about him. I thought he was in love with Ellesime, and then he lost his er... zest for women's charms...~
-@5    = ~So I plotted and schemed, and finally came up with a plan to rid the world of Tanova. When the deed was done, and Tanova lay dead, I was exultant, but not for long. Jon went mad with fury, and using his powerful magics, divined the identity of his wife's murderer. ~
-@6    = ~He arrived at my tower, and I allowed him entry, desirous to finally consummate our love. Jon disabled me with his spells, and then he cursed me to this body, and set spiders to feed me and keep me alive. So you see, if you were my friend, you would kill me. ~
-@7    = ~Centeol's Story
+@0  = ~Haltet ein... ich werde mich nicht wehren. Tötet mich, doch zuvor möchte ich euch von meinem Fluch erzählen - mein Gewissen erleichtern.~ 
+@1  = ~Mein Fluch - das ist alles an das ich mich noch erinnere. Ich war einst schön und mächtig.~
+@2  = ~Eine exotische Hexerin mit allerlei Kräften, sowie Verehrern, die an meiner Türe pochten. Ich aber habe nur Augen für einen Mann gehabt: Jon Irenicus, einen großartigen und machtvollen Zauberer. Er war der einzige Mann, der meiner Zuneigung würdig war, so dachte ich zumindest. Obwohl ich nach Jon lechzte, kümmerte er sich kaum um mich. Er war nämlich mit einer anderen Frau verheiratet,
+Fürstin Tanova.~ 
+@3  = ~Bitte fahrt fort, das ist eine spannende Geschichte!~
+@4  = ~Jon war verheiratet?! Ich... ich glaube, ich habe von ihm gehört. Ich dachte immer, er war in Ellesime verliebt, aber dann verlor er sein äh... Interesse an weiblichen Reizen...~ 
+@5  = ~Also plottete ich und schmiedete Ränke. Schließlich hatte ich eine Idee, um die Welt von Tanova zu befreien. Als die Tat vollbracht, und Tanova tot war, jubelte ich, jedoch nur kurze Zeit. Jon wurde tollwütig vor Zorn, und sah mit seiner mächtigen Magie die Identität des Mörders seiner Ehefrau voraus.~
+@6  = ~Er erreichte meinen Turm und ich ließ ihn ein, begierig darauf endlich unsere Liebe zu vollziehen. Jon machte mich mit seinen Zaubern kampfunfähig, danach verfluchte er mich in diesen Körper und befahl Spinnen mir als Futter zu dienen und mich am Leben zu erhalten. Ihr seht also, wenn ihr meine Freunde wärt, würdet ihr mich töten.~
+@7  = ~Centeol's Geschichte
 
-When we charmed the huge bloated creature that called herself Centeol, we were told an interesting story. It seems she was cursed to her current shape by a powerful mage called Jon Irenicus. The exact role of Tanova is unclear.~
-@8    = ~Jon lost his zest?! *she laughs bitterly* Please...~
-@9    = ~He had plenty and to spare... for Tanova. So I plotted and schemed, and finally came up with a plan to rid the world of Tanova. When the deed was done, and Tanova lay dead, I was exultant, but not for long. Jon went mad with fury, and using his powerful magics, divined the identity of his wife's murderer. ~
-@10   = ~He arrived at my tower, and I allowed him entry, desirous to finally consummate our love. Jon disabled me with his spells, and then he cursed me to this body, and set spiders to feed me and keep me alive. So you see, if you were my friend, you would kill me.~
-@11   = ~I... I see.~
-@12   = ~It's not what we were led to believe by a... uhm... different source.~
-@13   = ~Centeol's Story
+Als wir die riesige, aufgeblähte Kreatur, die sich selbst Centeol nannte, bezauberten, erfuhren wir eine interessante Geschichte. Anscheinend war ihre derzeitige Form, das Ergebnis des Fluchs eines mächtigen Magiers, der sich Jon Irenicus nennt. Die genaue Rolle von Tanova ist unklar.~ 
+@8  = ~Jon verlor seine Lust?! *Sie lachte bitter* Bitte...~
+@9  = ~Er hatte reichlich und teilte... mit Tanova. Also plottete ich und schmiedete Ränke. Schließlich hatte ich eine Idee, um die Welt von Tanova zu befreien. Als die Tat vollbracht, und Tanova tot war, jubelte ich, jedoch nur kurze Zeit. Jon wurde tollwütig vor Zorn, und sah mit seiner mächtigen Magie die Identität des Mörders seiner Ehefrau voraus.~ 
+@10 = ~Er erreichte meinen Turm und ich ließ ihn ein, begierig darauf endlich unsere Liebe zu vollziehen. Jon machte mich mit seinen Zaubern kampfunfähig, danach verfluchte er mich in diesen Körper und befahl Spinnen mir als Futter zu dienen und mich am Leben zu erhalten. Ihr seht also, wenn ihr meine Freunde wärt, würdet ihr mich töten.~
+@11 = ~Oh... Ich verstehe.~
+@12 = ~Es ist nicht das, was uns eine... ähm... andere Quelle glauben machen wollte.~ 
+@13 = ~Centeol's Geschichte
 
-When we charmed the huge bloated creature that called herself Centeol, we were told an interesting story. It seems she was cursed to her current shape by a powerful mage called Jon Irenicus. The exact role of Tanova is unclear.~
-@14   = ~The rain in this place... it tastes salty. The rain of tears... It would have consoled me that our death should not be unmourned, if it were not such a poor consolation...~
-@15   = ~The world is a haven of sorrows as is, but for some it is just not enough...~
-@16   = ~So I plotted and schemed, and finally came up with a plan to rid the world of Tanova. When the deed was done, and Tanova lay dead, I was exultant, but not for long.~
-@17   = ~The moments of joy are but fleeting fireflies in the darkness...~
-@18   = ~Jon went mad with fury, and using his powerful magics, divined the identity of his wife's murderer. ~
-@19   = ~I guessed that it might have been the case.~
-@20   = ~He arrived at my tower, and I allowed him entry, desirous to finally consummate our love.~
-@21   = ~What madness did take you, woman? O, that was love... Of course, of course. How could I have forgotten?~
-@22   = ~Jon disabled me with his spells, and then he cursed me to this body, and set spiders to feed me and keep me alive. So you see, if you were my friend, you would kill me. ~
-@23   = ~Be careful with what you wish for, my lady. For while we might not be in a position to deliver the mortal blow to a foe as powerful as you and your minions, someone else might just do it.~
-@24   = ~You wish to be dead because of the way you look and the spiders? What if I told you that the spell that is on you is a simple illusion in nature, and I could dispel it in an instant - would it make you... happy? Or would you perhaps realize that-~
-@25   = ~Could you... do that thing? O, please, my lord! Please...~
-@26   = ~Woman, do not you understand that it will not make you happier?~
-@27   = ~Of course it would. It will give me back my life!~
-@28   = ~Yes, and since the life is - oh, what's the point of arguing? Have your own body back.~
-@29   = ~My body... O, I am beautiful again! I am gorgeous...~ [ELFFF03]
-@30   = ~The storm is gathering to match my fury... The rain is crying for my hurts...~
-@31   = ~Revenge is all I have now. There is nothing else beyond it. Revenge for what Jon did to me.~
-@32   = ~And.. I... WILL... HAVE IT!~
-@33   = ~*sigh* Another good deed that did not go unpunished. I feel pity for this Jon, whoever he is, for there is no fury...~
+Als wir die riesige, aufgeblähte Kreatur, die sich selbst Centeol nannte, bezauberten, erfuhren wir eine interessante Geschichte. Anscheinend war ihre derzeitige Form, das Ergebnis des Fluchs eines mächtigen Magiers, der sich Jon Irenicus nennt. Die genaue Rolle von Tanova ist unklar.~
+@14 = ~Der Regen an diesem Platz... er schmeckt salzig. Ein Regen aus Tränen... Es hätte mich trösten können, dass unser Tod nicht betrauert wurde, wäre es nicht ein solch schwacher Trost...~ 
+@15 = ~Die Welt ist ein Hafen voller Sorgen, aber für manche ist das noch nicht genug...~
+@16 = ~Also plottete ich und schmiedete Ränke. Schließlich hatte ich eine Idee, um die Welt von Tanova zu befreien. Als die Tat vollbracht, und Tanova tot war, jubelte ich, jedoch nur kurze Zeit.~
+@17 = ~Die freudigen Momente sind wie flüchtige Glühwürmchen in der Dunkelheit...~
+@18 = ~Jon wurde tollwütig vor Zorn, und sah mit seiner mächtigen Magie die Identität des Mörders seiner Ehefrau voraus.~ 
+@19 = ~Ich vermutete, dass dies der Fall sein könnte.~
+@20 = ~Er erreichte meinen Turm und ich ließ ihn ein, begierig darauf endlich unsere Liebe zu vollziehen.~ 
+@21 = ~Welcher Wahnsinn hat dich getrieben, Weib? Oh, das war Liebe... Natürlich, ganz klar. Wie konnte ich das nur vergessen?~ 
+@22 = ~Jon machte mich mit seinen Zaubern kampfunfähig, danach verfluchte er mich in diesen Körper und befahl Spinnen mir als Futter zu dienen und mich am Leben zu erhalten. Ihr seht also, wenn ihr meine Freunde wärt, würdet ihr mich töten.~
+@23 = ~Seid vorsichtig was ihr euch wünscht, meine Dame. Auch wenn wir vielleicht nicht in der Lage sind den Todesstoß gegen einen so mächtigen Feind mit seinen Lakaien auszuführen, könnte es jemand anderes einfach tun.~ 
+@24 = ~Ihr wünscht nur wegen Eures Aussehens und das der Spinnen tot zu sein? Was, wenn ich euch sage, dass der Zauber auf Euch nur eine einfache Illusion der Natur ist, welche ich sofort brechen könnte - würde es Euch... glücklich machen? Oder würdet Ihr vielmehr realisieren, dass-~
+@25 = ~Könntet ihr... das tun? Oh, bitte mein Herr! Bitte...~
+@26 = ~Weib, könnt Ihr nicht verstehen, dass es euch nicht glücklicher machen würde?~
+@27 = ~Bestimmt würde es das. Es würde mir mein Leben zurück geben!~
+@28 = ~Ja, und weil das Leben - ach, was bringt es zu diskutieren? Hier habt Ihr Euren eigenen Körper zurück.~
+@29 = ~Mein Körper... Oh, ich bin wieder schön! Ich bin hinreißend...~
+@30 = ~Der Sturm schwillt an um meinem Zorn zu dienen... Der Regen weint um meine Wunden...~
+@31 = ~Rache ist alles was ich nun habe. Es gibt sonst nichts mehr. Rache, für das was Jon mir angetan hat.~
+@32 = ~Und... Ich... WERDE... SIE BEKOMMEN!~
+@33 = ~*seufz* Eine weitere gute Tat, die nicht ungestraft bleiben wird. Ich fühle Mitleid für diesen Jon, wer immer er auch ist. Weil, es gibt keinen Zorn...~
