--- bg1npc\tra\spanish\p#centeo.tra	Mon Mar 31 00:19:50 2014
+++ C:\BWP Patchstudio\patched files\bg1npc\tra\spanish\p#centeo.tra	Sun Sep 20 23:36:04 2015
@@ -31,7 +31,7 @@
 @26 = ~Mujer, ¿no entendéis que no os hará feliz?~
 @27 = ~¡Claro que lo hará, me devolverá mi vida!~
 @28 = ~Sí y como vuestra vida es - oh, ¿por qué discutir? Tomad vuestro cuerpo.~
-@29 = ~Mi cuerpo... ¡Soy hermosa otra vez! Preciosa...~ [ELFFF03]
+@29 = ~Mi cuerpo... ¡Soy hermosa otra vez! Preciosa...~
 @30 = ~La tormenta se forma para igualar mi furia... La lluvia llora por mi dolor...~
 @31 = ~Lo único que tengo ahora es venganza. No hay nada más. Venganza por lo que Jon me hizo.~
 @32 = ~Y... yo... ¡LA TENDRÉ!~
