--- bg1npc\tra\english\p#fw2700.tra	Sun Apr 13 08:26:41 2014
+++ C:\BWP Patchstudio\patched files\bg1npc\tra\english\p#fw2700.tra	Sun Sep 20 23:36:04 2015
@@ -1,3 +1,3 @@
-@0  = ~You hear the creaking of waggon wheels and move to the side of the road.~
+@0  = ~You hear the creaking of wagon wheels and move to the side of the road.~
 @1  = ~A black coach and team of horses passes you, bearing Gorion back to his resting place in Candlekeep.~
 @2  = ~You bury Gorion in a small stone cairn where he fell.~
