--- bg1npc\readme-bg1npc.html	Thu Jun 18 14:44:45 2015
+++ C:\BWP Patchstudio\patched files\bg1npc\readme-bg1npc.html	Sun Sep 20 23:36:04 2015
@@ -4,7 +4,7 @@
     <title>The BG1 NPC Project </title>
     <meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1"/>
     <link rel="stylesheet" href="style/g3readme.css" type="text/css"/>
-    <link href="style/g3icon.ico" rel="icon" type="image/bmp"/>
+    <link href="style/g3.ico" rel="icon" type="image/bmp"/>
   </head>
   <body>
     <h1>
@@ -14,21 +14,22 @@
       <strong><a href="http://www.gibberlings3.net/">A Gibberlings Three Mod</a></strong>
       <br/>
       <p>
-        <strong>Authors: <a href="http://forums.gibberlings3.net/?showtopic=1315">The <acronym title="Baldur's Gate">BG1</acronym><acronym title="Non-Player Character">NPC</acronym>Project team</a>. Please contact us on the Gibberlings3 forums for more information.</strong>
+        <strong>Authors: <a href="http://gibberlings3.net/forums/?showtopic=1315">The <acronym title="Baldur's Gate">BG1</acronym><acronym title="Non-Player Character">NPC</acronym>Project team</a>. Please contact us on the Gibberlings3 forums for more information.</strong>
       </p>
       <p>
         <strong>Project Coordinators (in temporal order):
         <br/>
-        <a href="http://forums.gibberlings3.net/?showuser=721">Blucher</a>, <a href="http://forums.gibberlings3.net/?showuser=62">Kish</a>, <a href="http://forums.gibberlings3.net/?showuser=30">Andyr</a>, <a href="http://forums.gibberlings3.net/?showuser=142">Domi</a>, currently: <a href="http://forums.gibberlings3.net/?showuser=844">cmorgan</a></strong>
+        <a href="http://gibberlings3.net/forums/?showuser=721">Blucher</a>, <a href="http://gibberlings3.net/forums/?showuser=62">Kish</a>, <a href="http://gibberlings3.net/forums/?showuser=30">Andyr</a>, <a href="http://gibberlings3.net/forums/?showuser=142">Domi</a>, <a href="http://gibberlings3.net/forums/?showuser=844">cmorgan</a>,
+        currently: <a href="http://gibberlings3.net/forums/index.php?showuser=6858">AstroBryGuy</a></strong>
       </p>
       <p>
-        <strong>On the web: <a href="http://www.gibberlings3.net/bg1npc/">Home page</a></strong> and <strong><a href="http://forums.gibberlings3.net/index.php?showforum=45">discussion forum</a></strong>
+        <strong>On the web: <a href="http://www.gibberlings3.net/bg1npc/">Home page</a></strong> and <strong><a href="http://gibberlings3.net/forums/index.php?showforum=45">discussion forum</a></strong>
         <br/>
         For support issues on a Tutu, EasyTutu, <acronym title="Baldur's Gate Trilogy">BGT</acronym>, or <acronym title="Baldur's Gate Enhanced Edition">BGEE</acronym> install, please contact <a href="http://www.gibberlings3.net/">www.gibberlings3.net</a>. For support issues on a <acronym title="Baldur's Gate Trilogy">BGT</acronym> install involving Big Picture (<acronym title="Big Picture">BP</acronym> ) mods or Mega Installs, please contact <a href="http://www.spellholdstudios.net/">www.spellholdstudios.net</a>.
         <br/>
       </p>
       <p>
-        <strong>Version 22 - <a href="http://forums.gibberlings3.net/index.php?app=downloads&showcat=11">Check for the most recent version</a></strong>
+        <strong>Version 22 - <a href="http://gibberlings3.net/forums/index.php?app=downloads&showcat=11">Check for the most recent version</a></strong>
         <br/>
         <strong>Languages:</strong> English, Spanish, French, Polish, German* (teilweise noch in englisch)
         <br/>
@@ -448,7 +449,7 @@
           <a href="bg1npc_docs/bg1npcvarindex.htm">Index of Variables used in The <acronym title="Baldur's Gate">BG1</acronym><acronym title="Non-Player Character">NPC</acronym>Project code, with file name and line number, sortable by column.</a>
         </li>
         <li>
-          Use of <a href="http://forums.gibberlings3.net/">Cross Platform Modding Variables</a> for single install code for multiple platforms
+          Use of <a href="http://gibberlings3.net/forums/">Cross Platform Modding Variables</a> for single install code for multiple platforms
         </li>
         <li>
           Use of Fluid State Variables for single install code for <acronym title="Baldur's Gate Trilogy">BGT</acronym> compatible coding of joinable NPCs who stack <acronym title="Baldur's Gate">BG</acronym> dialogue on top of <acronym title="Baldur's Gate II">BG2</acronym> dialogue. This will be most useful when used in conjunction with versions of <acronym title="Baldur's Gate Trilogy">BGT</acronym> v1.06 and above.
@@ -1171,7 +1172,7 @@
         <br/>
         <a href="http://www.crimsoneditor.com/">Crimson Editor</a> by Ingyu Kang
         <br/>
-        <a href="http://forums.gibberlings3.net/index.php?showforum=137"><acronym title="Dragonlance Total Conversion Editor Pro">DLTCEP</acronym></a> by Avenger
+        <a href="http://gibberlings3.net/forums/index.php?showforum=137"><acronym title="Dragonlance Total Conversion Editor Pro">DLTCEP</acronym></a> by Avenger
         <br/>
         <a href="http://iesdp.gibberlings3.net/"><acronym title="Infinity Engine Structures Description Project">IESDP</acronym></a> maintained by igi and Avenger
         <br/>
@@ -1211,7 +1212,7 @@
         (those not listed are unknown)
       </p>
       <p>
-        NOTE: To access this content, you will need to download and install the <em>The <acronym title="Baldur's Gate">BG</acronym>1<acronym title="Non-Player Characters">NPC</acronym>Music Pack</em>. You can find it on The Gibberlings Three download page, <a href="http://www.gibberlings3.net/downloads/#bg1npc">here.</a>
+        NOTE: To access this content, you will need to download and install the <em>The <acronym title="Baldur's Gate">BG</acronym>1<acronym title="Non-Player Characters">NPC</acronym>Music Pack</em>. You can find it on The Gibberlings Three download page, <a href="http://gibberlings3.net/forums/index.php?app=downloads&showcat=11">here.</a>
       </p>
       <ul>
         <li>
@@ -1278,7 +1279,7 @@
     </h2>
     <div class="section">
       <p>
-        A complete list of contributors is maintained at our forum. Visit the <a href="http://forums.gibberlings3.net/">Gibberlings Three forums</a> for information on this and any other Gibberlings Three mods on which we may be working. The current maintainer/coordinator is <a href="http://forums.gibberlings3.net/?showuser=844">cmorgan</a>
+        A complete list of contributors is maintained at our forum. Visit the <a href="http://gibberlings3.net/forums/">Gibberlings Three forums</a> for information on this and any other Gibberlings Three mods on which we may be working. The current maintainer/coordinator is <a href="http://gibberlings3.net/forums/?showuser=844">cmorgan</a>
       </p>
       <p>
         The <acronym title="Baldur's Gate">BG1</acronym><acronym title="Non-Player Character">NPC</acronym> Project is not developed, supported, or endorsed by BioWare&#8482; or Interplay/BlackIsle. The Tutu mod was created by Japheth, aided by Ghreyfain, SimDing0, and Kish. EasyTutu was created by Macready, based on a Baldurdashed (Kevin Dorner) version of Tutu v4 Fixpack 15. If there are any copyright issues or this statement needs revision then please contact one of us and advise us what to do about it. This mod was created to be freely enjoyed by all Baldur's Gate and Baldur's Gate II players. However, it should not be sold, published, compiled or redistributed in any form without the consent of the team leaders.
@@ -1394,7 +1395,6 @@
         <li>
           <font color="red">Added </font><strong>Traduction française</strong> : pour tout support, rendez-vous sur <a href="http://www.baldursgateworld.fr/lacouronne/forum.php">La Couronne de Cuivre</a>
         </li>
-		<a href="http://forums.gibberlings3.net/index.php?s=&showtopic=21446&view=findpost&p=182686"> ScuD's Yeslik repairs </a>
         <li> 
           <font color="green">Repaired </font> From Isaya, reporting translator findings: x#cosqu.tra, < PRO_MANWOAMN > repaired to < PRO_MANWOMAN >
         </li>
@@ -1408,7 +1408,7 @@
           <font color="green">Repaired </font>From Isaya, reporting translator findings: I_C_T ~%tutu_scripts%LAVLEAD~ 0 X#CoranRill repaired to correct line; I_C_T ~%tutu_scripts%LAVLEAD~ 2 X#CoranRill
         </li>
         <li>
-          <font color="green">Repaired </font> <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=21446&view=findpost&p=182686.php"> ScuD's Yeslik repairs </a>
+          <font color="green">Repaired </font> <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=21446&view=findpost&p=182686.php"> ScuD's Yeslik repairs </a>
         </li>
         <li>
           <font color="green">Repaired </font> <a href="https://github.com/cmorganbg/BG1NPC/commit/6bfbbdc8fcd1b62a844543998765f11799da1bc6"> BGT and TutTu Dream assignment repairs </a>
@@ -1425,67 +1425,67 @@
           WeiDU v230 for Windows, OS X, (Linux gets theirs from <a href="http://www.weidu.org">www.weidu.org</a>)
         </li>
         <li>
-          <font color="green">Repaired </font><a href="http://forums.gibberlings3.net/index.php?showtopic=22518">Repair Ajantis' Lovetalk sequence (1 branch)</a>
+          <font color="green">Repaired </font><a href="http://gibberlings3.net/forums/index.php?showtopic=22518">Repair Ajantis' Lovetalk sequence (1 branch)</a>
         </li>
         <li>
-          <font color="green">Repaired </font><a href="http://forums.gibberlings3.net/index.php?showtopic=21956">Repair Coran's Hanali Sequence initiation.</a>
+          <font color="green">Repaired </font><a href="http://gibberlings3.net/forums/index.php?showtopic=21956">Repair Coran's Hanali Sequence initiation.</a>
         </li>
         <li>
-          <font color="green">Repaired </font><a href="http://forums.gibberlings3.net/index.php?showtopic=21807">Repair swapped .tra lines in x#ict3.d</a>
+          <font color="green">Repaired </font><a href="http://gibberlings3.net/forums/index.php?showtopic=21807">Repair swapped .tra lines in x#ict3.d</a>
         </li>
         <li>
-          <font color="green">Repaired </font><a href="http://forums.gibberlings3.net/index.php?showtopic=21928">Recheck BGT and Tutu for possible mismatch in Skie's dialog, with utilization of CandleKeep BILL and GATEWA2 .dlg references.</a>
+          <font color="green">Repaired </font><a href="http://gibberlings3.net/forums/index.php?showtopic=21928">Recheck BGT and Tutu for possible mismatch in Skie's dialog, with utilization of CandleKeep BILL and GATEWA2 .dlg references.</a>
         </li>
         <li>
-          <font color="green">Repaired </font><a href="http://forums.gibberlings3.net/index.php?showtopic=21928">Recheck X#ImoenRaiken usage for x#imint.d; only one of the two are supposed to react, since she plays the same line in each.</a>
+          <font color="green">Repaired </font><a href="http://gibberlings3.net/forums/index.php?showtopic=21928">Recheck X#ImoenRaiken usage for x#imint.d; only one of the two are supposed to react, since she plays the same line in each.</a>
         </li>
         <li>
-          <font color="green">Repaired </font><a href="http://forums.gibberlings3.net/index.php?showtopic=19610">Repair OR() found by BWP</a>
+          <font color="green">Repaired </font><a href="http://gibberlings3.net/forums/index.php?showtopic=19610">Repair OR() found by BWP</a>
         </li>
         <li>
-          <font color="green">Repaired </font><a href="http://forums.gibberlings3.net/index.php?showtopic=19662">Include check-for-and-install-if-not-yet-done on Ascension64's Bere"ghost"fix </a>
+          <font color="green">Repaired </font><a href="http://gibberlings3.net/forums/index.php?showtopic=19662">Include check-for-and-install-if-not-yet-done on Ascension64's Bere"ghost"fix </a>
         </li>
         <li>
-          <font color="green">Repaired </font><a href="http://forums.gibberlings3.net/index.php?showtopic=19009">Recheck WTASIGHT script assignment on those pesky bandits (again)</a>
+          <font color="green">Repaired </font><a href="http://gibberlings3.net/forums/index.php?showtopic=19009">Recheck WTASIGHT script assignment on those pesky bandits (again)</a>
         </li>
         <li>
-          <font color="green">Repaired </font>Typos <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=18136&view=findpost&p=172090">reference post</a>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=18332&view=findpost&p=166167">reference post</a>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=18332&view=findpost&p=166168">reference post</a>
+          <font color="green">Repaired </font>Typos <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=18136&view=findpost&p=172090">reference post</a>
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=18332&view=findpost&p=166167">reference post</a>
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=18332&view=findpost&p=166168">reference post</a>
         </li>
         <li>
-          <font color="green">Repaired </font><a href="http://forums.gibberlings3.net/index.php?showtopic=18516">aVENGER_(RR) and wisp's replacement-of-regexp repair of "Bg2 Portrait" component</a>
+          <font color="green">Repaired </font><a href="http://gibberlings3.net/forums/index.php?showtopic=18516">aVENGER_(RR) and wisp's replacement-of-regexp repair of "Bg2 Portrait" component</a>
         </li>
         <li>
-          <font color="green">Repaired </font><a href="http://forums.gibberlings3.net/index.php?showtopic=17905">Repair dream responses cross-project on BGT, where DREAM = BG1DREAM</a>
+          <font color="green">Repaired </font><a href="http://gibberlings3.net/forums/index.php?showtopic=17905">Repair dream responses cross-project on BGT, where DREAM = BG1DREAM</a>
         </li>
         <li>
-          <font color="green">Repaired </font><a href="http://forums.gibberlings3.net/index.php?showtopic=20113">Kagain X#KAQUREWARDParty adjustment it NumInParty to avoid trigger failure if a party member is dead</a>
+          <font color="green">Repaired </font><a href="http://gibberlings3.net/forums/index.php?showtopic=20113">Kagain X#KAQUREWARDParty adjustment it NumInParty to avoid trigger failure if a party member is dead</a>
         </li>
         <li>
-          <font color="green">Repaired </font><a href="http://forums.gibberlings3.net/index.php?showtopic=20698">Xan's quest - Friendly Arm Inns owners described as Dwarves instead of Gnomes in one of the dialogs.</a>
+          <font color="green">Repaired </font><a href="http://gibberlings3.net/forums/index.php?showtopic=20698">Xan's quest - Friendly Arm Inns owners described as Dwarves instead of Gnomes in one of the dialogs.</a>
         </li>
         <li>
-          <font color="green">Repaired </font>Elminister doesn't leave after meeting in first BG city-area. Can talk to him again and get double Journal-entries (after this he leaves). Interjection with Minsc/Boo to blame? <a href="http://forums.gibberlings3.net/index.php?showtopic=20698">reference post</a>
-          <a href="http://forums.gibberlings3.net/index.php?showtopic=19941">the bigg suggests a repair for this</a>
+          <font color="green">Repaired </font>Elminister doesn't leave after meeting in first BG city-area. Can talk to him again and get double Journal-entries (after this he leaves). Interjection with Minsc/Boo to blame? <a href="http://gibberlings3.net/forums/index.php?showtopic=20698">reference post</a>
+          <a href="http://gibberlings3.net/forums/index.php?showtopic=19941">the bigg suggests a repair for this</a>
         </li>
         <li>
-          <font color="green">Repaired </font><a href="http://forums.gibberlings3.net/index.php?showtopic=20399">May have to adjust actions that break when interrupted at the end of dialog, like this one:</a>
+          <font color="green">Repaired </font><a href="http://gibberlings3.net/forums/index.php?showtopic=20399">May have to adjust actions that break when interrupted at the end of dialog, like this one:</a>
         </li>
         <li>
-          <font color="green">Repaired </font><a href="http://forums.gibberlings3.net/index.php?showtopic=20072">Recheck project for invalid actions sequences, like this one:</a>
+          <font color="green">Repaired </font><a href="http://gibberlings3.net/forums/index.php?showtopic=20072">Recheck project for invalid actions sequences, like this one:</a>
         </li>
         <li>
-          <font color="green">Repaired </font><a href="http://forums.gibberlings3.net/index.php?showtopic=20873">Internal workrooms - Ajantis Hostile Leave script adjustment block</a>
+          <font color="green">Repaired </font><a href="http://gibberlings3.net/forums/index.php?showtopic=20873">Internal workrooms - Ajantis Hostile Leave script adjustment block</a>
         </li>
         <li>
-          <font color="gold">Reported But Determined No Action Necessary:</font><a href="http://forums.gibberlings3.net/index.php?showtopic=18917">Internal workrooms -  raid Ascension64's run in ALWAYS block but don't bother doing it over and over again if it is already done during this install session code snippets from BGUB; </a>no action: his usage is cleaner, but there is no harm in leaving the old behavior.
+          <font color="gold">Reported But Determined No Action Necessary:</font><a href="http://gibberlings3.net/forums/index.php?showtopic=18917">Internal workrooms -  raid Ascension64's run in ALWAYS block but don't bother doing it over and over again if it is already done during this install session code snippets from BGUB; </a>no action: his usage is cleaner, but there is no harm in leaving the old behavior.
         </li>
         <li>
-          <font color="gold">Reported But Determined No Action Necessary:</font><a href="http://forums.gibberlings3.net/index.php?s=&showtopic=18754&view=findpost&p=175544">Replace regexp for temples with specific A_F EACH file IN list BEGIN A_I F_E_I_G to avoid catching mod-added temples or healing areas on a BGT install (or a Tutu, if anyone actually adds them there) and spamming Resurrection and greater Restoration, etc.; </a>no action: code already screens for actual cures and adds them - possibly overkill, but best to leave it as it is working for now.
+          <font color="gold">Reported But Determined No Action Necessary:</font><a href="http://gibberlings3.net/forums/index.php?s=&showtopic=18754&view=findpost&p=175544">Replace regexp for temples with specific A_F EACH file IN list BEGIN A_I F_E_I_G to avoid catching mod-added temples or healing areas on a BGT install (or a Tutu, if anyone actually adds them there) and spamming Resurrection and greater Restoration, etc.; </a>no action: code already screens for actual cures and adds them - possibly overkill, but best to leave it as it is working for now.
         </li>
         <li>
-          <font color="gold">Reported But Determined No Action Necessary:</font><a href="http://forums.gibberlings3.net/index.php?showtopic=20910">Recheck Xan dream for RestParty(); </a>no action: author intent. may need to strengthen docs when someone finally tackles that project.
+          <font color="gold">Reported But Determined No Action Necessary:</font><a href="http://gibberlings3.net/forums/index.php?showtopic=20910">Recheck Xan dream for RestParty(); </a>no action: author intent. may need to strengthen docs when someone finally tackles that project.
         </li>
       </ul>
       <p>
@@ -1496,55 +1496,55 @@
           WeiDU v211 for Windows, OS X, and Linux 09.18.2009
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?showtopic=17012&st=0&gopid=145893&#entry145893">Galathée's translation repairs to Garrick's dialog </a>03.24.2009
+          <a href="http://gibberlings3.net/forums/index.php?showtopic=17012&st=0&gopid=145893&#entry145893">Galathée's translation repairs to Garrick's dialog </a>03.24.2009
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?showtopic=17091&pid=146433&st=0&#entry146433">Erebusant's repairs to Coran's Rose Quest</a>04.05.2009
+          <a href="http://gibberlings3.net/forums/index.php?showtopic=17091&pid=146433&st=0&#entry146433">Erebusant's repairs to Coran's Rose Quest</a>04.05.2009
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?showtopic=17721">Xzar and Montaron Charming Scene: Montaron's attempting to speak while dead fixed.</a>07.15.2009
+          <a href="http://gibberlings3.net/forums/index.php?showtopic=17721">Xzar and Montaron Charming Scene: Montaron's attempting to speak while dead fixed.</a>07.15.2009
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=16855&view=findpost&p=144623">Temujin's mismapped resource references fixes </a>07.15.2009
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=16855&view=findpost&p=144623">Temujin's mismapped resource references fixes </a>07.15.2009
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=16855&view=findpost&p=144623">Druid dv mismatch in combat script</a>07.15.2009
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=16855&view=findpost&p=144623">Druid dv mismatch in combat script</a>07.15.2009
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?showtopic=16958">Mike1072's Winski Portrait Fix</a> 07.16.2009
+          <a href="http://gibberlings3.net/forums/index.php?showtopic=16958">Mike1072's Winski Portrait Fix</a> 07.16.2009
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=15740&view=findpost&p=152312">Player Comment on Shared Love when no longer romancing Branwen blocked. </a>07.16.2009
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=15740&view=findpost&p=152312">Player Comment on Shared Love when no longer romancing Branwen blocked. </a>07.16.2009
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=15740&view=findpost&p=1523152">Edwin's Dynaheir Romance Cutscene refusal option fixed.</a>07.16.2009
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=15740&view=findpost&p=1523152">Edwin's Dynaheir Romance Cutscene refusal option fixed.</a>07.16.2009
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=16403&view=findpost&p=146078">More robust patching of BIMOEN2 into interdia.2da for BGT </a>07.16.2009
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=16403&view=findpost&p=146078">More robust patching of BIMOEN2 into interdia.2da for BGT </a>07.16.2009
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=16588&view=findpost&p=152337">Faldorn/Izefia Quest loophole repaired </a>07.16.2009
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=16588&view=findpost&p=152337">Faldorn/Izefia Quest loophole repaired </a>07.16.2009
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=16588&view=findpost&p=152342">erebusant's "let Andarthe surrender, already!" fix </a>07.16.2009
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=16588&view=findpost&p=152342">erebusant's "let Andarthe surrender, already!" fix </a>07.16.2009
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=16855&view=findpost&p=152451">Temujin's Resource, DV, and typo repairs part II </a>07.17.2009
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=16855&view=findpost&p=152451">Temujin's Resource, DV, and typo repairs part II </a>07.17.2009
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=18332&view=findpost&p=156883">aVENGER_(RR)'s repairs to 'Use BG2 Portraits' component.</a>09.17.2009
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=18332&view=findpost&p=156883">aVENGER_(RR)'s repairs to 'Use BG2 Portraits' component.</a>09.17.2009
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=18104&view=findpost&p=156884">Wisp's repairs to Coran's Wyverns Address </a>09.17.2009
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=18104&view=findpost&p=156884">Wisp's repairs to Coran's Wyverns Address </a>09.17.2009
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=18332&view=findpost&p=156885">aVENGER_(RR)'s repairs to Jaheira's Quest items and scripts </a>09.17.2009
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=18332&view=findpost&p=156885">aVENGER_(RR)'s repairs to Jaheira's Quest items and scripts </a>09.17.2009
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=18332&view=findpost&p=156887">Item Revisions compatibility for the ring of Human Influence </a>09.17.2009
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=18332&view=findpost&p=156887">Item Revisions compatibility for the ring of Human Influence </a>09.17.2009
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=18136&view=findpost&p=155024">Daulmakan's typos and repairs </a>09.17.2009
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=18136&view=findpost&p=155024">Daulmakan's typos and repairs </a>09.17.2009
         </li>
       </ul>
       <p>
@@ -1597,67 +1597,67 @@
           Ajantis/Kivan banter will not play if Tazok has already been killed
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=15876&view=findpost&p=138061">Xan Multiple Interjections on Ice Island repaired </a>
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=15876&view=findpost&p=138061">Xan Multiple Interjections on Ice Island repaired </a>
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=16013&view=findpost&p=138076">Tutu:  Jaheira and Khalid sharing same coordinates in Elfsong, et al repaired</a>
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=16013&view=findpost&p=138076">Tutu:  Jaheira and Khalid sharing same coordinates in Elfsong, et al repaired</a>
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=15985&view=findpost&p=138080">SAY@ instances repaired to SAY @</a>
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=15985&view=findpost&p=138080">SAY@ instances repaired to SAY @</a>
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=15866&view=findpost&p=138084">Coran flirt sentence malformation repaired</a>
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=15866&view=findpost&p=138084">Coran flirt sentence malformation repaired</a>
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=15864&view=findpost&p=138087">Xan: loved -> laughed repaired</a>
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=15864&view=findpost&p=138087">Xan: loved -> laughed repaired</a>
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=15826&view=findpost&p=138089">Xan: token and grammar repair</a>
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=15826&view=findpost&p=138089">Xan: token and grammar repair</a>
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=15561&view=findpost&p=138097">Documentation: Dudleyville links updated to ForgottenWars, all instances, all documents</a>
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=15561&view=findpost&p=138097">Documentation: Dudleyville links updated to ForgottenWars, all instances, all documents</a>
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=15765&view=findpost&p=138114">Adjust Romance Challenge Areas for names of BGT versions of distributed .wed files on BGT, and Tutu ones on Tutu for areas X#CH[11,12,13,14] and X#AJR1</a>
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=15765&view=findpost&p=138114">Adjust Romance Challenge Areas for names of BGT versions of distributed .wed files on BGT, and Tutu ones on Tutu for areas X#CH[11,12,13,14] and X#AJR1</a>
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=15890&view=findpost&p=138322">Xan leaving due to reputation though you are bonded with him repaired</a>
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=15890&view=findpost&p=138322">Xan leaving due to reputation though you are bonded with him repaired</a>
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=15314&view=findpost&p=138316">Updated and clarified relationship of BG1NPC and UB Sarevok's Diary components in documentation</a>
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=15314&view=findpost&p=138316">Updated and clarified relationship of BG1NPC and UB Sarevok's Diary components in documentation</a>
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=16119&view=findpost&p=138325">Repaired inconsistent coding with SAY~ and DO~ missing spaces for easier code searches</a>
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=16119&view=findpost&p=138325">Repaired inconsistent coding with SAY~ and DO~ missing spaces for easier code searches</a>
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=15516&view=findpost&p=138335">-JW-'s repairs to the Edwin Ambush</a>
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=15516&view=findpost&p=138335">-JW-'s repairs to the Edwin Ambush</a>
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?showtopic=15480&st=0&gopid=138416&#entry138416">Documentation update, BGT compatibility and install order links at Spellhold Studios</a>
+          <a href="http://gibberlings3.net/forums/index.php?showtopic=15480&st=0&gopid=138416&#entry138416">Documentation update, BGT compatibility and install order links at Spellhold Studios</a>
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=15407&view=findpost&p=138418">Kagain Quest Journal entries reshuffled to remove any but the last seen as valid</a>
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=15407&view=findpost&p=138418">Kagain Quest Journal entries reshuffled to remove any but the last seen as valid</a>
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=15367&view=findpost&p=138423">Jaheira now no longer banters about Khalid with Shar-Teel and Montaron unless he is in the party</a>
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=15367&view=findpost&p=138423">Jaheira now no longer banters about Khalid with Shar-Teel and Montaron unless he is in the party</a>
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=15053&view=findpost&p=138538">Xan Checking for HaveSpellParty in Romance but player has an SoA install </a>
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=15053&view=findpost&p=138538">Xan Checking for HaveSpellParty in Romance but player has an SoA install </a>
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?s=&showtopic=15180&view=findpost&p=138549">Tiax Quest reweighting and scripts tweaks </a>
+          <a href="http://gibberlings3.net/forums/index.php?s=&showtopic=15180&view=findpost&p=138549">Tiax Quest reweighting and scripts tweaks </a>
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?showtopic=15525&st=0&gopid=138436&#entry138436">Coran Romance (Briel error) I_C_T moved to I_C_T2</a>
+          <a href="http://gibberlings3.net/forums/index.php?showtopic=15525&st=0&gopid=138436&#entry138436">Coran Romance (Briel error) I_C_T moved to I_C_T2</a>
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?showtopic=16588">maretha, beador, and andarthe, part 17. </a>
+          <a href="http://gibberlings3.net/forums/index.php?showtopic=16588">maretha, beador, and andarthe, part 17. </a>
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?showtopic=16141">BG1NPC, SCS, and SCSII: Killing Tazok given SPRITE_IS_DEAD resurrection in Undercity</a>
+          <a href="http://gibberlings3.net/forums/index.php?showtopic=16141">BG1NPC, SCS, and SCSII: Killing Tazok given SPRITE_IS_DEAD resurrection in Undercity</a>
         </li>
         <li>
-          <a href="http://forums.gibberlings3.net/index.php?showtopic=16710&pid=144499&st=0&#entry144499">Xan Romance talk missing movie call on EasyTutu repaired</a>
+          <a href="http://gibberlings3.net/forums/index.php?showtopic=16710&pid=144499&st=0&#entry144499">Xan Romance talk missing movie call on EasyTutu repaired</a>
         </li>
         <li>
           Updated to WeiDU v210
