--- bg1npc\tra\spanish\setup.tra	Sat Sep 06 15:35:54 2014
+++ C:\BWP Patchstudio\patched files\bg1npc\tra\spanish\setup.tra	Sun Sep 20 23:36:04 2015
@@ -130,17 +130,17 @@
 @1124 = ~Algunos ajustes del temporizador de banters ya fueron instalados anteriormente.~
 @1125 = "Por favor seleccione una de las siguientes opciones para el temporizador de banters:
 [1] Tiempo normal entre banters (~27 minutos entre banters)
-[2] Banters son mas r·pidos (ocurren un ~50% mas r·pido)
-[3] Banters muy r·pidos (ocurren al doble de la velocidad normal)
+[2] Banters son mas rapidos (ocurren un ~50% mas rapido)
+[3] Banters muy rapidos (ocurren al doble de la velocidad normal)
 [4] Banters son mas lentos (ocurren ~50% mas lento)
 [5] Banters muy lentos (ocurren a la mitad de la velocidad normal)"
 @1126 = ~Velocidad de los banters: Normal~
-@1127 = ~Velocidad de los banters: Mas r·pidos~
-@1128 = ~Velocidad de los banters: Muy r·pidos~
+@1127 = ~Velocidad de los banters: Mas rapidos~
+@1128 = ~Velocidad de los banters: Muy rapidos~
 @1129 = ~Velocidad de los banters: Mas lentos~
 @1130 = ~Velocidad de los banters: Muy lentos~
 @5000  = ~BG:EE detectado.~
-@5001  = ~Ce composant necessite d'avoir installe BGT ou TuTu.~
-@5002  = ~Ce composant necessite d'avoir installe BGEE.~
+@5001  = ~Este componente requiere BGT o TuTu.~
+@5002  = ~Este componente requiere BGEE.~
 @5003  = ~Incompatible con BGT y BGEE: 
 Por favor, usa el componente de Sword Coast Strategems para BGT y BGEE.~
