--- bg1npc\tra\english\x#addial_tmp.tra	Tue Nov 25 00:08:06 2014
+++ C:\BWP Patchstudio\patched files\bg1npc\tra\english\x#addial_tmp.tra	Sun Sep 20 23:36:04 2015
@@ -273,7 +273,7 @@
 @272  = ~Has Edwin turned discrete all of a sudden? Where are all the spicy details?~
 @273  = ~Oh, thank you for not sharing it, then. Gods forbid I should ever hear something trivial or inconsequential from you.~
 @274  = ~For once, I'd like to hear something constructive from you. Since I now know what kind of women you find repulsive, do enlighten me on what kind – if any – you actually like.~
-@275  = ~*pauses and smirks* Would you like an honest response, or the pompous line I serve debutantes at the Spring Solstice ball? It is quite charming and well conceived, as befitting a man who is as versed in the ways of the world as I am. However, you do seem to be past the age when young women believe that denying men sensual favors will result in death or permanent debilitating affliction. Or that they would do anything to win your favour.~
+@275  = ~*pauses and smirks* Would you like an honest response, or the pompous line I serve debutantes at the Spring Solstice ball? It is quite charming and well conceived, as befitting a man who is as versed in the ways of the world as I am. However, you do seem to be past the age when young women believe that denying men sensual favors will result in death or permanent debilitating affliction. Or that they would do anything to win your favor.~
 @276  = ~Who knows?~
 @277  = ~What a horrible and demeaning thing to say!~
 @278  = ~The honest answer will do, I should think...~
