--- bg1npc\tra\german\x#ajint.tra	Mon Jul 22 19:46:48 2013
+++ C:\BWP Patchstudio\patched files\bg1npc\tra\german\x#ajint.tra	Sun Sep 20 23:36:04 2015
@@ -1,7 +1,7 @@
 @0    = ~Bei Helm, die Lügen und Spielchen dieser Dämonen riechen nach dem Bösen. Lasst uns zu den Waffen greifen!~ 
 @1    = ~Dame <CHARNAME> hüllenlos, auf einem Bärenfell - ein entzückender Gedanke...~
 @2    = ~... ich meine natürlich, Ihr als Baby, meine Dame. *errötet*~ 
-@3    = ~Meine Dame, meiner Meinung nach seid Ihr immer noch so niedlich wie ein Käfer. Und das soll als Kompliment gemeint sein.~
+@3    = ~Meine Dame, meiner Meinung nach seid Ihr immer noch wirklich ganz niedlich. Und das soll als Kompliment gemeint sein.~
 @4    = ~Sie ist natürlich ein Baby...~ 
 @5    = ~Natürlich werden wir das machen, mein Freund. Diese Halboger werden bald keine Bedrohung mehr darstellen.~
 @6    = ~Jetzt noch nicht, mein Freund, aber ich bin sicher, dass uns <CHARNAME> bald dorthin führen wird.~
