--- bg1npc\tra\spanish\x#ajantis.tra	Mon Mar 31 00:19:50 2014
+++ C:\BWP Patchstudio\patched files\bg1npc\tra\spanish\x#ajantis.tra	Sun Sep 20 23:36:04 2015
@@ -238,3 +238,7 @@
 @237  = ~<CHARNAME>, es una lástima que no hubiéramos conservado el escudo de mi familia. Sólo espero que tengáis una razón sincera para su ausencia.~
 @238  = ~¡Oh, Ajantis, lo siento! El escudo dignifica mucho par vos. Lo recuperaremos cuanto antes.~
 @239  = ~Ajantis, he tenido mis razones. Lo siento, pero ahora tenéis una mejor armadura y necesitamos todo el espacio que podamos acumular para llevar otras cosas.~
+@240  = ~Oh dear. Please stop this whining. I'm sure your wealthy family won't even notice it's gone.~
+@241  = ~That was an unfriendly remark, <CHARNAME>. Let's hope that attitude is only temporary!~
+@242  = ~I don't insist on using it, since it might not provide the best protection against the enemies we face, but it is still an item of great value to me.~
+@243  = ~My family shield! <CHARNAME>, I'm very happy we have it back. Let's make sure it will be in a safe place if we ever have to put it away again.~
