--- bg1npc\tra\spanish\x#coran.tra	Mon Mar 31 00:19:50 2014
+++ C:\BWP Patchstudio\patched files\bg1npc\tra\spanish\x#coran.tra	Sun Sep 20 23:36:04 2015
@@ -209,7 +209,8 @@
 @181  = ~No perderé mi tiempo en semejante tontería. He nacido guerrera, no concubina.~
 @182  = ~Podría ayudaros a atraer la atención de vuestro salvaje dios.~
 @183  = ~¡Tempus ve mi furia y mi valor en la batalla! así es como hallaré su favor.~
-@184  = ~Pero vos podríais también darle una idea sobre por qué el clero femenino puede ser algo que él está perdiendo entre su arduo trabajo. Esa es vuestra meta, después de todo...~@185  = ~¡El Señor de la Batalla no está preocupado por mi aparición! El guardián de Tempus guarda respeto por las grandes hazañas.~
+@184  = ~Pero vos podríais también darle una idea sobre por qué el clero femenino puede ser algo que él está perdiendo entre su arduo trabajo. Esa es vuestra meta, después de todo...~
+@185  = ~¡El Señor de la Batalla no está preocupado por mi aparición! El guardián de Tempus guarda respeto por las grandes hazañas.~
 @186  = ~Oh, yo no estaría tan segura. Él sigue siendo un hombre, Branwen.~
 @187  = ~¿Alguien os dijo, Dynaheir, que tenéis un acento que puede hacer que el corazón de un hombre de vueltas por completo?~
 @188  = ~Me sorprendéis, Coran.~
