--- bg1npc\tra\german\x#faldorn.tra	Sat Jul 27 21:37:26 2013
+++ C:\BWP Patchstudio\patched files\bg1npc\tra\german\x#faldorn.tra	Sun Sep 20 23:36:04 2015
@@ -41,7 +41,8 @@
 @40 = ~*Faldoen schnappt sich das Trinkhorn von Branwen und leert es in einem Zug.* Die Natur wurde gerächt! Ruhm für Silvanus!~
 @41 = ~Tempus, der Fürst aller Schlachten, gewährt unseren Kriegern Stärke in den Armen und unseren Magiern Kampfgeist. Was hatte Silvanus mit unserem Kampf zu tun?~
 @42 = ~Ha! Der Eichenvater hat überall Feinde. In dem Haufen dort waren bestimmt ein oder zwei dabei!~
-@43 = ~Dann lasst uns darauf trinken! Ruhm den Siegreichen! Wehe den Besiegten!~@44   = ~Being turned to stone – how strange and wonderful that must be! To go from being a moving part of nature to the immobile, but one around which all motion happens.~
+@43 = ~Dann lasst uns darauf trinken! Ruhm den Siegreichen! Wehe den Besiegten!~
+@44   = ~Being turned to stone – how strange and wonderful that must be! To go from being a moving part of nature to the immobile, but one around which all motion happens.~
 @45   = ~Wonderful? ‘Tis like dying slowly and endlessly! ‘Tis worse than undeath!~
 @46   = ~Grrr! How can you compare! Undeath is an abomination of corrupted weaklings, who try to cheat the Great Mother and to shift the balance.~
 @47   = ~When you turned to stone, you were part of Great Mother herself. How can you find that repulsive, Branwen?~
