--- bg1npc\tra\english\x#ict3.tra	Tue Nov 25 00:08:26 2014
+++ C:\BWP Patchstudio\patched files\bg1npc\tra\english\x#ict3.tra	Sun Sep 20 23:36:04 2015
@@ -681,7 +681,7 @@
 @676  = ~And within a week they're partying like they never have before, right?~
 @677  = ~Sometimes sooner, miss! Say... have we met before?~
 @678  = ~Don't think so. Only Luckyfoot I ever met was old Wendal, back in Berdusk.~
-@679  = ~Ah, my favourite third cousin! How's he doing?~
+@679  = ~Ah, my favorite third cousin! How's he doing?~
 @680  = ~Just fine. He and the missus were expecting their seventh kid, when I left town. I bet she'll be as cute as the others...~
 @681  = ~And to think the pair'd only just had their third when I last saw 'em! I really must make the journey up there sometime, I heard it's a nice enough place, for a human built town... err, looks like your friends getting rather impatient with all this family stuff. What were we talking about? Ah yes, the a trip to the tower versus halfling hospitality...~
 @682  = ~Our business is our own to mind, and ye'd better be keepin' yer nose to yerself.~
