--- bg1npc\tra\spanish\x#imint.tra	Mon Mar 31 00:19:50 2014
+++ C:\BWP Patchstudio\patched files\bg1npc\tra\spanish\x#imint.tra	Sun Sep 20 23:36:04 2015
@@ -354,6 +354,7 @@
 @353  = ~¿Queréis que os explique quién es?~
 @354  = ~Sí, si no es muy largo. No quiero que me contéis una larga historia, sabéis, como esas viejas cotorras. Puede que seamos amigos y tal, pero tengo mis límites.~
 @355  = ~Muy bien, a ver...~
+@356  = ~Never mind, then. I don't think I can do it without boring you.~
 @357  = ~Malar es conocido como el Señor de las Bestias, ya que sus poderes incluyen los monstruos, las bestias y cosas así. No tiene mayores poderes sobre los humanos, pero intentó tenerlos en el pasado, cuando Bane, Bhaal y Myrkul fueron a conseguir los poderes del anterior gobernador del Fin de Todas las Cosas, Jergal.~
 @358  = ~Por supuesto, ellos no querían que pasara eso, así que realizaron otra prueba mientras él hacía los preparativos del juego inicial. Os aseguro que no quedó muy satisfecho con ese resultado.~
 @359  = ~Vaya. Me lo imagino.~
