--- bg1npc\tra\english\x#imint.tra	Tue Nov 25 00:08:46 2014
+++ C:\BWP Patchstudio\patched files\bg1npc\tra\english\x#imint.tra	Sun Sep 20 23:36:04 2015
@@ -194,7 +194,7 @@
 @193  = ~Of course! I wouldn't miss it for the world!~
 @194  = ~Boo! You're no fun! I thought we were s'posed to be adventuring!~
 @195  = ~Great! That means I'll have ta start making plans for it!~
-@196  = ~Wow! And Winthrop used ta tell me that diamonds don't grow on trees! Sure, in Candlekeep they don't! Hey, <CHARNAME>, cheer up! Tymora favours us, seems ta me.~
+@196  = ~Wow! And Winthrop used ta tell me that diamonds don't grow on trees! Sure, in Candlekeep they don't! Hey, <CHARNAME>, cheer up! Tymora favors us, seems ta me.~
 @197  = ~Is that... Gorion?~
 @198  = ~Yes. He died saving me from some... thing.~
 @199  = ~Yes, it *was* Gorion. He died while protecting me from an attack.~
