--- bg1npc\tra\german\x#imint.tra	Mon Jun 24 14:52:10 2013
+++ C:\BWP Patchstudio\patched files\bg1npc\tra\german\x#imint.tra	Sun Sep 20 23:36:04 2015
@@ -164,7 +164,6 @@
 @163  = ~ENDLICH! Hätte er auch nur noch ein Wort gesagt, ich hätte ihn um gebracht, das schwöre ich!~
 @164  = ~Warum sind alle immer böse auf mich? Ich versuche doch nur, freundlich zu sein!~
 @165  = ~Scheußlich. Hoffentlich werden wir niemals so gequält.~
-@166  = ~Halt mal! Was in aller Welt erzählt Ihr da? Welchen Ruf wollt Ihr mir da anhängen, he?~
 @166  = ~Wartet mal ne Sekunde! Was in aller Welt erzählt Ihr da? Welchen Ruf wollt Ihr mir da anhängen, he?~
 @167  = ~Genau! Haut ab, Ihr Betrüger!~
 @168  = ~Das kann man behaupten und es dann mit irgendwelchem Hokuspokus über Sternenkonstellationen und anderem Zeug in Beziehung setzen, um es gerade unbestimmt genug klingen zu lassen, dass es fast immer passt.~
