--- bg1npc\phase2\dlg\x#jaqu.d	Sun Jun 14 09:17:16 2015
+++ C:\BWP Patchstudio\patched files\bg1npc\phase2\dlg\x#jaqu.d	Sun Sep 20 23:36:04 2015
@@ -388,14 +388,25 @@
 == ~X#BEADOR~ @63
 END
 IF ~InParty("khalid") InMyArea("khalid") !StateCheck("khalid",CD_STATE_NOTVALID)~ DO ~SetGlobal("X#JaheiraHealsB","GLOBAL",5)~ EXTERN ~%KHALID_JOINED%~ KhalidBeadorTalk 
-IF ~OR(3) !InParty("khalid") !InMyArea("khalid") StateCheck("khalid",CD_STATE_NOTVALID)~ DO ~SetGlobal("X#JaheiraHealsB","GLOBAL",5)~ EXTERN ~%JAHEIRA_JOINED%~ JaheiraBeadorNoKhalidTalk
+IF ~OR(3) !InParty("khalid") !InMyArea("khalid") StateCheck("khalid",CD_STATE_NOTVALID)~ DO ~SetGlobal("X#JaheiraHealsB","GLOBAL",5)~ EXTERN ~%JAHEIRA_JOINED%~ JaheiraBeadorTalk
+IF ~Dead("khalid")~ DO ~SetGlobal("X#JaheiraHealsB","GLOBAL",5)~ EXTERN ~%JAHEIRA_JOINED%~ JaheiraBeadorNoKhalidTalk
 
 CHAIN ~%KHALID_JOINED%~ KhalidBeadorTalk
 @64
 == ~X#BEADOR~ IF ~InParty("khalid") InMyArea("khalid") !StateCheck("khalid",CD_STATE_NOTVALID)~ THEN @65
-== ~%JAHEIRA_JOINED%~ IF ~InParty("khalid") InMyArea("khalid") !StateCheck("khalid",CD_STATE_NOTVALID)~ THEN @66
-== ~X#BEADOR~ IF ~InParty("khalid") InMyArea("khalid") !StateCheck("khalid",CD_STATE_NOTVALID)~ THEN @67
-== ~%JAHEIRA_JOINED%~ IF ~InParty("khalid") InMyArea("khalid") !StateCheck("khalid",CD_STATE_NOTVALID)~ THEN @68
+END
+IF ~~ THEN EXTERN ~%JAHEIRA_JOINED%~ JaheiraBeadorTalk
+
+//== ~%JAHEIRA_JOINED%~ IF ~InParty("khalid") InMyArea("khalid") !StateCheck("khalid",CD_STATE_NOTVALID)~ THEN @66
+//== ~X#BEADOR~ IF ~InParty("khalid") InMyArea("khalid") !StateCheck("khalid",CD_STATE_NOTVALID)~ THEN @67
+//== ~%JAHEIRA_JOINED%~ IF ~InParty("khalid") InMyArea("khalid") !StateCheck("khalid",CD_STATE_NOTVALID)~ THEN @68
+//END
+//IF ~~ THEN DO ~SetGlobal("X#JaheiraQuest","GLOBAL",3) AddexperienceParty(500)~ EXIT
+
+CHAIN ~%JAHEIRA_JOINED%~ JaheiraBeadorTalk
+@66
+== ~X#BEADOR~ @67
+== ~%JAHEIRA_JOINED%~ @68
 END
 IF ~~ THEN DO ~SetGlobal("X#JaheiraQuest","GLOBAL",3) AddexperienceParty(500)~ EXIT
 
