--- bg1npc\phase2\dlg\x#jaqu.d	Wed Jun 15 10:06:36 2011
+++ C:\BWP Patchstudio\patched files\bg1npc\phase2\dlg\x#jaqu.d	Sun Sep 22 23:23:15 2013
@@ -388,7 +388,7 @@
 == ~X#BEADOR~ @63
 END
 IF ~InParty("khalid") InMyArea("khalid") !StateCheck("khalid",CD_STATE_NOTVALID)~ DO ~SetGlobal("X#JaheiraHealsB","GLOBAL",5)~ EXTERN ~%KHALID_JOINED%~ KhalidBeadorTalk 
-IF ~OR(2) !InParty("khalid") Dead("khalid")~ DO ~SetGlobal("X#JaheiraHealsB","GLOBAL",5)~ EXTERN ~%JAHEIRA_JOINED%~ JaheiraBeadorNoKhalidTalk
+IF ~OR(3) !InParty("khalid") !InMyArea("khalid") StateCheck("khalid",CD_STATE_NOTVALID)~ DO ~SetGlobal("X#JaheiraHealsB","GLOBAL",5)~ EXTERN ~%JAHEIRA_JOINED%~ JaheiraBeadorNoKhalidTalk
 
 CHAIN ~%KHALID_JOINED%~ KhalidBeadorTalk
 @64
