--- bg1npc\phase1\baf\x#jaheira1.baf	Mon Mar 31 00:19:42 2014
+++ C:\BWP Patchstudio\patched files\bg1npc\phase1\baf\x#jaheira1.baf	Sun Sep 20 23:36:04 2015
@@ -29,7 +29,7 @@
 /* Burying Gorion */
 IF %BGT_VAR%
 Global("X#GorionBuried","GLOBAL",0)
-Global("X#JaheiraGorBuryTalk","GLOBAL",0)
+Global("X#JaheiraGorBuryTimerStarted","GLOBAL",0)
 InParty(Myself)
 CombatCounter(0)
 !See([ENEMY])
@@ -37,6 +37,17 @@
 !StateCheck(Myself,CD_STATE_NOTVALID)
 See(Player1)
 CombatCounter(0)
+THEN
+RESPONSE #100
+SetGlobal("X#JaheiraGorBuryTimerStarted","GLOBAL",1)
+RealSetGlobalTimer("X#JaheiraGorBuryTimer","GLOBAL",EIGHT_HOURS)
+END
+
+IF %BGT_VAR%
+InParty(Myself)
+Global("X#GorionBuried","GLOBAL",0)
+RealGlobalTimerExpired("X#JaheiraGorBuryTimer","GLOBAL")
+Global("X#JaheiraGorBuryTalk","GLOBAL",0)
 THEN
 RESPONSE #100
 SetGlobal("X#JaheiraGorBuryTalk","GLOBAL",1)
