--- bg1npc\tra\spanish\x#lp2rep.tra	Mon Mar 31 00:19:50 2014
+++ C:\BWP Patchstudio\patched files\bg1npc\tra\spanish\x#lp2rep.tra	Sun Sep 20 23:36:04 2015
@@ -147,7 +147,8 @@
 @146  = ~Seguro que al menos hay unas cuantas personas que aún no han oído hablar de vuestras acciones. Yo las encontraré, y las convenceré de que no se debe jugar con vos. Para alguien con vuestra reputación, eso costará 2400 monedas de oro.~
 @147  = ~Me costará convencer a algunos de que podéis ser incluso más temible de lo que imaginan, pero mi talento no tiene límites. Aunque tendré que ser muy meticuloso, y un trabajo tan pesado cuesta. Para alguien con vuestra reputación, el precio será 3000 monedas de oro.~
 @148  = ~Todo un desafío, pero estoy a la altura de la tarea. Puedo afirmar con certeza que habéis venido al lugar adecuado. Para alguien con vuestra reputación, eso costará 4000 monedas de oro.~
-@149  = ~Será toda una hazaña, incluso para un bardo con tanto talento como yo, para alguien con vuestra reputación. Pocos os creerán con vuestras acciones. Os costará 6500 monedas de oro.~@150  = ~Será toda una hazaña, incluso para un bardo con tanto talento como yo, para alguien con vuestra reputación. Pocos os creerán con vuestras acciones. Os costará 2800 monedas de oro.~
+@149  = ~Será toda una hazaña, incluso para un bardo con tanto talento como yo, para alguien con vuestra reputación. Pocos os creerán con vuestras acciones. Os costará 6500 monedas de oro.~
+@150  = ~Será toda una hazaña, incluso para un bardo con tanto talento como yo, para alguien con vuestra reputación. Pocos os creerán con vuestras acciones. Os costará 2800 monedas de oro.~
 @151  = ~Me costará convencer a algunos, pero mi talento no tiene límites. Aunque tendré que ser muy meticuloso, y un trabajo tan pesado cuesta. Para alguien con vuestra reputación, el precio será 1500 monedas de oro.~
 @152  = ~Dejadme ver: tabernas y casas de reuniones por toda la Costa de la Espada, y otros gastos... para alguien con vuestra reputación, eso costará 700 monedas de oro.~
 @153  = ~Un juego de niños para un bardo de *mi* talento. Para alguien con vuestra reputación lo puedo hacer por la ínfima suma de 700 monedas de oro.~
