--- bg1npc\phase2\dlg\x#miint.d	Sat Oct 04 01:03:00 2014
+++ C:\BWP Patchstudio\patched files\bg1npc\phase2\dlg\x#miint.d	Sun Sep 20 23:36:04 2015
@@ -91,7 +91,7 @@
 DO ~SetGlobal("X#MinscGnarl","GLOBAL",1)~ EXTERN ~%MINSC_JOINED%~ X#MinscGnarlChain
 END
 
-CHAIN IF WEIGHT #-2 ~%BGT_VAR% Global("X#Minsc3600","GLOBAL",1) AreaCheck("%Lighthouse%")~ THEN ~%MINSC_JOINED%~ X#Minsc3600Talk
+CHAIN IF WEIGHT #-2 ~%BGT_VAR% Global("X#Minsc3600","GLOBAL",1)~ THEN ~%MINSC_JOINED%~ X#Minsc3600Talk
 @42
 DO ~SetGlobal("X#Minsc3600","GLOBAL",2)~
 == ~%MINSC_JOINED%~ @43 
