--- bg1npc\tra\german\x#npcwait.tra	Tue Sep 02 08:17:04 2014
+++ C:\BWP Patchstudio\patched files\bg1npc\tra\german\x#npcwait.tra	Sun Sep 20 23:36:04 2015
@@ -106,12 +106,12 @@
 @105 = ~There still be a few interestin' places left in Baldur's Gate. The Low Lantern is a cozy bordello disguised as a boat. Seek me out there. Ye won't be sorry.~
 @106 = ~Nashkel's Belching Dragon is our kind of a place. See if ye can find your way down there, before we get bored of waiting.~
 @107 = ~There still be a few interestin' places in Baldur's Gate. The Low Lantern is a cozy bordello disguised as a boat. Seek us out there. Ye won't be sorry.~
-@108 = ~Edwin, I do not want to part ways with you forever. Just wait for me here. I will return for you when I need your help again.~
-@109 = ~On second thought, Edwin, I want you to stay with the group.~
-@110 = ~Save your pleasantries. (I suffer the company of these cretins merely because they are useful.)~
-@111 = ~Good riddance. (I am not being cast out, I... I am merely biding my time, taking a well-deserved rest away from these... from these pesky... simians! Yes! That's the right word.)~
-@112 = ~You would cast me out as though I were a servant? (They will pay for their insolence!)~
-@113 = ~Do you take me for a complete fool, sim... er <CHARNAME>? (These monkeys could never appreciate a true genius, even if it was served to them on a silver plate with a golden spoon.) I would not stay in this disgusting cloaca, fit only for a bunch of diseased chimpanzees, not even for all the treasures of Thay. (And that is a *seriously* large pile of shiny, yellow coins, trust me on this one.)~
+@108 = ~Edwin, ich möchte, dass wir für einige Zeit trennen. Warte einfach hier auf mich. Ich werde zurückkehren, wenn ich Eure Hilfe benötige.~ 
+@109 = ~Ich habs mir anders überlegt, Edwin, ich möchte, dass Ihr in meiner Gruppe bleibt.~
+@110 = ~Spart Euch Eure Höflichkeiten. (Ich erdulde diese Gruppe voller Idioten nur, weil einigermaßen nützlich erscheinen.)~ 
+@111 = ~Gute Befreiung. (Und ich bin noch nicht verstoßen worden, ich... ich habe nur darauf gewartet, ich werde mir meine wohlverdiente Ruhe gönnen, abseits von diesen... diesen nervtötenden... Affen! Ja! Das ist das richtige Wort.)~
+@112 = ~Ihr verstößt mich, als ob ich ein einfacher Diener wäre? (Für diese Unverfrorenheit werden sie bezahlen!)~ 
+@113 = ~Haltet Ihr mich für einen Volltrottel, Aff... äh <CHARNAME>? (Diese Tölpel werden niemals ein wahres Genie zu schätzen wissen, selbst wenn er ihnen auf einem Silbertablett serviert werden würde.) Ich würde nicht in dieser ekelhaften Kloake bleiben, das ist nur etwas für einen Haufen kranker Schimpansen, nicht einmal für alle Schätze von Thay. (Und das sind *wirklich* viele goldene Münzen, das könnt Ihr mir glauben.)~
 @114 = ~Nashkel's Belching Dragon is a nauseous, filthy pigsty. But it will have to do. See if you can find your way down there, before I grow bored of waiting. (Let's see how quickly they will discover my irreplaceability. I bet they will run all the way down to Nashkel.)~
 @115 = ~There are yet some sumptuous places in the squalid collection of hovels that they call Baldur's Gate. The Low Lantern is a splendid Hell disguised as a boat. Seek me out there when you grow bored of your petty pursuits. (Not that these simians would appreciate true splendor if it came up to them and bit them.)~
 @116 = ~Shar-Teel, I do not want to part ways with you forever. Just wait for me here. I will return for you when I need your help again.~
