--- bg1npc\tra\english\x#quayl.tra	Tue Nov 25 00:08:54 2014
+++ C:\BWP Patchstudio\patched files\bg1npc\tra\english\x#quayl.tra	Sun Sep 20 23:36:04 2015
@@ -163,7 +163,7 @@
 @162  = ~I'm quite capable o' running mae own business, Quayle!~
 @163  = ~Granted, you haven't made any exceedingly stupid business decisions yet. But it's only a matter of time! A dwarven brain just can't cope with the intricacies of the modern financial world!~
 @164  = ~If I were ye, I'd shut yer mouth quick.~
-@165  = ~Kagain, I'm trying to do you a favour! Can't your tiny little mind even comprehend that most obvious of truths?~
+@165  = ~Kagain, I'm trying to do you a favor! Can't your tiny little mind even comprehend that most obvious of truths?~
 @166  = ~Ye're not at superior as ye think, fool.~
 @167  = ~Well, probably not. I do tend to underestimate my genius, after all. Ah, modesty, another one of my many admirable traits...~
 @168  = ~Methinks ye're the one missing something up top, gnome. Mebbe it be hidden under that ridiculous hat, har har!~
