--- bg1npc\tra\spanish\x#shrom.tra	Mon Mar 31 00:19:50 2014
+++ C:\BWP Patchstudio\patched files\bg1npc\tra\spanish\x#shrom.tra	Sun Sep 20 23:36:04 2015
@@ -682,7 +682,7 @@
 @681  = ~¿Qué tal si nos acostamos, osa?~
 @682  = ~¡No me estoy enfurruñando! Nunca me enfurruño. Mirad nuestra olla: la tapasteis demasiado fuerte, y el vapor acabó por reventarla, esparciendo vuestra comida por todo el lugar. Lo mismo pasa con el enfurruñamiento. No soy tan estúpido... Doy rienda suelta a mi ira perfectamente... en cada batalla.~
 @683  = ~*suspiro* O sobre mí...~
-@684  = ~No me recordéis lo de la olla. <Player2> no tiene talento para cocinar: el guiso de esta noche me aterroriza más que el Trono de Hierro.~
+@684  = ~No me recordéis lo de la olla. <PLAYER2> no tiene talento para cocinar: el guiso de esta noche me aterroriza más que el Trono de Hierro.~
 @685  = ~Estupendo, estupendo, entonces ¿cuál es el problema?~
 @686  = ~¡No me importa qué estúpida palabra prefiráis! Darle vueltas o enfurruñarse... todo es lo mismo.~
 @687  = ~*Se encoge de hombros.* Bueno, supongo que  nos podemos meter en el saco de dormir, mientras hierve la comida.~
@@ -690,7 +690,7 @@
 @689  = ~¿Shar-Teel? ¿Os... os estáis enfurruñando?~
 @690  = ~Dejad de mirarme... ¡así! No lo sé. Simplemente me apetecía hablar... pero no tenía mucho que decir.~
 @691  = ~Ya hemos recontado las muertes de hoy, y ordenado las joyas y monedas, y...~
-@692  = ~Y la comida aún no está lista, así que no podemos quejarnos sobre las habilidades culinarias de <Player2> o la carencia de ellas. Shar-Teel, ¿quizás os pudierais sentar más cerca de mí, y podríamos observar el fuego en silencio? Podría resultar... er... diferente.~
+@692  = ~Y la comida aún no está lista, así que no podemos quejarnos sobre las habilidades culinarias de <PLAYER2> o la carencia de ellas. Shar-Teel, ¿quizás os pudierais sentar más cerca de mí, y podríamos observar el fuego en silencio? Podría resultar... er... diferente.~
 @693  = ~Uhm... ¿Qué tal si nos acostamos, osa?~
 @694  = ~¿Es que sois estúpido? Ya os dije cómo. Maldito enfurruñado.~
 @695  = ~Bueno, mirad quién se ha animado ahora.~
