--- bg1npc\tra\english\x#shint.tra	Sun Apr 13 08:33:37 2014
+++ C:\BWP Patchstudio\patched files\bg1npc\tra\english\x#shint.tra	Sun Sep 20 23:36:04 2015
@@ -105,7 +105,7 @@
 @104  = ~Kill the *pesky* thing? With pleasure!~
 @105  = ~Petrine, your uncle wanted us to kill Angel. I reasoned that if we refused, tomorrow he'd find someone else. And I can not stand idle when people go around killing poor orphan girls' cats! I... I killed him.~
 @106  = ~I am going to make sure that you are transferred into the care of someone kinder than your rude uncle, Petrine. Trust me.~
-@107  = ~I will hunt Sarevok down, ma'am, but seeing that I could lose my life, I plead your Grace for one favour before I go. There is some unfinished business... a trivial thing. An orphan girl, Petrine and her cat, Angel need a proper guardian. I... I killed her uncle – a mean man who mistreated her, and yet she was left with no soul in the world to protect her. 'Tis a blood debt.~
+@107  = ~I will hunt Sarevok down, ma'am, but seeing that I could lose my life, I plead your Grace for one favor before I go. There is some unfinished business... a trivial thing. An orphan girl, Petrine and her cat, Angel need a proper guardian. I... I killed her uncle – a mean man who mistreated her, and yet she was left with no soul in the world to protect her. 'Tis a blood debt.~
 @108  = ~Rise, my good <PRO_LADYLORD>. You have my word that as long as I command power and authority in this city, the girl Petrine should be raised as my own... and Angel the Cat will find welcome in the palace as well. Now, do not tarry, Sarevok has to be destroyed for his crimes!~
 @109  = ~That goes to show that even among Chill, men are as pathetic as elsewhere.~
 @110  = ~We will fight for our coin, you dolt, unless <CHARNAME> is a worthless rag.~
