--- bg1npc\tra\polish\x#totsc.tra	Wed Aug 06 22:35:23 2014
+++ C:\BWP Patchstudio\patched files\bg1npc\tra\polish\x#totsc.tra	Sun Sep 20 23:36:04 2015
@@ -146,5 +146,3 @@
 @2005 = ~Wiem, ¿e wielu druidów wybiera tê œcie¿kê, aby jeszcze bardziej zbli¿yæ siê do Natury, ale znacznie wolê me zwyk³e umiejêtnoœci zmiany kszta³tu! Na Silvanusa, musimy odwróciæ tê kl¹twê, gdy¿ pewna jestem, ¿e nikt z nas nie chce spêdziæ reszty swych dni w skórze wilka. Jedynym sposobem jest... zabicie ich przywódczyni! Zabiæ Kaishasê! Szybko!~
 @2006 = ~Nazwa³bym to darem, gdybyœ uprzednio nas zapyta³a o zdanie. Wilki to dumne i silne bestie, i nie ma nic upokarzaj¹cego w posiadaniu wilczej krwi. Ale nie daj¹c nam wyboru, przeklê³aœ nas, Kaishaso.~
 @2007 = ~Myœl o tym co chcesz. Teraz nale¿ycie do rodziny.~
-@11   = ~To... to pu³apka? Ale... ale tu jest tak z-zimno...!~
-@116  = ~Wiêc oto osiedliœmy na tej lodowej krze poœrodku oceanu, bez jedzenia i w³aœciwego okrycia. Nie s¹dzisz, <CHARNAME>, ¿e to zabawne, ¿e akurat teraz jesteœmy bezpieczniejsi ni¿ kiedykolwiek? A mo¿e jesteœ zbyt zajêty planowaniem, w jaki sposób uciec z tego bezpiecznego schronienia prosto w rêce nieub³aganej zguby?~ ~Wiêc oto osiedliœmy na tej lodowej krze poœrodku oceanu, bez jedzenia i w³aœciwego okrycia. Nie s¹dzisz, <CHARNAME>, ¿e to zabawne, ¿e akurat teraz jesteœmy bezpieczniejsi ni¿ kiedykolwiek? A mo¿e jesteœ zbyt zajêta planowaniem, w jaki sposób uciec z tego bezpiecznego schronienia prosto w rêce nieub³aganej zguby?~
