--- bg1npc\tra\spanish\x#xan.tra	Mon Mar 31 00:19:50 2014
+++ C:\BWP Patchstudio\patched files\bg1npc\tra\spanish\x#xan.tra	Sun Sep 20 23:36:04 2015
@@ -37,7 +37,8 @@
 @36   = ~Vamos, Xan. Habladme de vuestra espada.~
 @37   = ~Nunca me he podido decidir por si un conocimiento mayor aumenta o reduce nuestras posibilidades de sobrevivir, <CHARNAME>. Por desgracia,  ahora mismo no tiene la menor importancia, así que da lo mismo.~
 @38   = ~¿Augurios nefastos? Así que preferís ignorar el peligro extremo en que nos encontramos. ¿Os puedo aconsejar un par de parches para los ojos?~
-@39   = ~Es... es casi alentador que me reconozcáis como... como hermano.~ ~Es... es casi alentador que me reconozcáis como... como hermana.~ @40   = ~La espada lunar escoge a héroes que merezcan empuñarla, Xan. Puede  que seáis una colección andante y parlante de lamentaciones, pero vuestro  corazón es noble y puro. Es un honor viajar a vuestro lado.~
+@39   = ~Es... es casi alentador que me reconozcáis como... como hermano.~ ~Es... es casi alentador que me reconozcáis como... como hermana.~
+@40   = ~La espada lunar escoge a héroes que merezcan empuñarla, Xan. Puede  que seáis una colección andante y parlante de lamentaciones, pero vuestro  corazón es noble y puro. Es un honor viajar a vuestro lado.~
 @41   = ~Tengo una teoría, Xan. Las espadas lunares están encantadas para matar a sus portadores por actos de gran cobardía, malicia, brutalidad y  deshonor. Conociéndoos, está claro que la espada os eligió porque vio que no  teníais agallas para hacer algo que pusiera vuestra vida en peligro. Así que  sois un héroe en contra de vuestra voluntad, cometiendo actos de valentía y  abnegación, pero lloriqueando de terror.~
 @42   = ~Creedme, Xan, conoceros es una experiencia inolvidable.~
 @43   = ~¿Así que una vez leísteis un libro? No me hagáis reír, las costumbres de los míos no se pueden aprender en una biblioteca.~
