--- bg1npc\tra\english\x#xanlt.tra	Sun Apr 13 08:33:57 2014
+++ C:\BWP Patchstudio\patched files\bg1npc\tra\english\x#xanlt.tra	Sun Sep 20 23:36:04 2015
@@ -126,7 +126,7 @@
 @125  = ~Yes, you are quite safe with me, Xan. Let's move on, shall we?~
 @126  = ~But you are. It is not something to be ashamed of, <CHARNAME>. Believe me, it will pass away faster than you think... if you live that long, of course. And perhaps you will.~
 @127  = ~As you wish. I may have been mistaken, and our recent progress is simply a rare stroke of luck. I shall try to enjoy my last moments of peace and quiet, in any case.~
-@128  = ~I was sent to the Sword Coast to accomplish a mission I thought to be impossible. I still do, for the odds are not at all in our favour. But we go on, and that gives me a gleam of hope. It might happen that I will finish my task and live to see my home at Evereska once again.~
+@128  = ~I was sent to the Sword Coast to accomplish a mission I thought to be impossible. I still do, for the odds are not at all in our favor. But we go on, and that gives me a gleam of hope. It might happen that I will finish my task and live to see my home at Evereska once again.~
 @129  = ~I am glad you think so.~
 @130  = ~Are you returning to Evereska once it is over?~
 @131  = ~What task are you speaking of?~
@@ -366,7 +366,7 @@
 @365  = ~I did not love? And how, pray, did you come to this conclusion?~
 @366  = ~Fine. I'll leave you alone, if you wish.~
 @367  = ~Ah, yes. Now you're rational and reasonable, and I am out of my mind. Fair enough.~
-@368  = ~And you can help, by keeping a distance. Or are you a child who won't be parted from her favourite toy?~
+@368  = ~And you can help, by keeping a distance. Or are you a child who won't be parted from her favorite toy?~
 @369  = ~Bravo, <CHARNAME>! I believe in a few months you will be able to enter Shar's priesthood; you sense hidden pains well enough to qualify.~
 @370  = ~If you did, you would have... recognized it. I repeat my request: leave me alone.~
 @371  = ~I've had enough of these illusions of safety and power. I am tired from this endless torture of guessing whether we will be alive in the morning, whether I will be able to... no. There is no point in further discussion. Just leave me be.~
@@ -1148,7 +1148,7 @@
 @1140 = ~Boys, could you please be a bit more friendly with each other?~
 @1141 = ~Leave me out of this.~
 @1142 = ~As do I, at times like these.~
-@1143 = ~Cheer me up? Bring me to my grave, most likely. *sigh* Eldoth, you could certainly do me a favour: go, and be killed in the next battle.~
+@1143 = ~Cheer me up? Bring me to my grave, most likely. *sigh* Eldoth, you could certainly do me a favor: go, and be killed in the next battle.~
 @1144 = ~Ah, in that *particular* case, I hope to disappoint you.~
 @1145 = ~In your case, my hopes are ever golden.~
 @1146 = ~*sigh* <CHARNAME>, you are ever asking for miracles. But this one will not come to be. I would rather relieve you of the questionable benefit of my company than shake this scoundrel's hand - and I am unable to do either.~
@@ -1258,7 +1258,7 @@
 @1250 = ~Think it's all just the echo of a belch in a empty jug?~
 @1251 = ~*sigh* The fact that I am still able to think proves it well enough, I suppose.~
 @1252 = ~Considerin' the nonsense ye *do* think on, there's naught t'be lost if ye were stopped.~
-@1253 = ~Do you intend to do me a favour, and end my miserable existence?~
+@1253 = ~Do you intend to do me a favor, and end my miserable existence?~
 @1254 = ~*shrug* Fer a pal like yerself? Sure!~
 @1255 = ~*sigh* Proceed, then.~
 @1256 = ~You two, break it up.~
