--- imoenfriendship/scripts/imoen2.baf	2013-02-07 10:46:04.000000000 -0500
+++ Fixpack/imoenfriendship/scripts/imoen2.baf	2016-11-21 10:40:24.204656000 -0500
@@ -2,7 +2,7 @@
 IF
     Global("I#ImoenFriendshipActive","GLOBAL",0)
     InParty(Myself)
-    GlobalGT("chapter","GLOBAL",3)
+    GlobalGT("chapter","GLOBAL",%bg2_chapter_3%)
 THEN
   RESPONSE #100
     RealSetGlobalTimer("I#ImoenFriendshipTalksTimer","GLOBAL",1200)
