--- SOS\BAF\AR4203.BAF	Wed Aug 30 18:06:32 2006
+++ C:\BWP Patchstudio\patched files\SOS\BAF\AR4203.BAF	Wed Sep 21 16:30:47 2016
@@ -1,7 +1,7 @@
 IF
   OnCreation()
-  TimeGT(DAWN_END)
-  TimeLT(DUSK_START)
+  TimeGT(7)
+  TimeLT(21)
   !Dead("CBBUCKMO")  // ~Master Werewolf~
 THEN
   RESPONSE #100
@@ -17,8 +17,8 @@
 
 IF
   OnCreation()
-  TimeLT(DAWN_END)
-  TimeGT(DUSK_START)
+  TimeLT(7)
+  TimeGT(21)
   !Dead("CBBUCKMO")  // ~Master Werewolf~
 THEN
   RESPONSE #100
