--- SOS\Language\English\CBDRAGDP.tra	Thu Feb 02 12:47:04 2006
+++ C:\BWP Patchstudio\patched files\SOS\Language\English\CBDRAGDP.tra	Fri Nov 12 16:50:24 2010
@@ -3,7 +3,7 @@
 @2    = ~Oh. I'm sorry.  I thought I said 'what'.~
 @3    = ~You die now, wrym!~
 @4    = ~Um... actually... I don't, um... think that I want... I want... that is... er...~
-@5    = ~No, you need to actually open your <RACE> mouth to speak.  Something your mother *should* have taught you in your <BOYGIRL>hood.~
+@5    = ~No, you need to actually open your <RACE> mouth to speak.  Something your mother *should* have taught you in your <GIRLBOY>hood.~
 @6    = ~Now that we've got that cleared up, dear adventurer, what is it that you want?~
 @7    = ~Spit it out, <RACE>.  I don't have all year.~
 @8    = ~Umm... I'm going to... um... go away.~
