--- SOS\Language\English\CBISSON1.tra	Thu Feb 02 12:47:06 2006
+++ C:\BWP Patchstudio\patched files\SOS\Language\English\CBISSON1.tra	Fri Nov 12 16:50:23 2010
@@ -21,7 +21,7 @@
 @20   = ~Thanks.  I'll keep that in mind.~
 @21   = ~Whatever these men want from me, I am sure I can handle it.~
 @22   = ~I shall find these men immediately, and find out what they want.~
-@23   = ~I am Ceeb.  I have followed your progress in this world for as long as I have been aware that you exist.  Which, I must say, has been nearly four years since you first left your <PRO_BOYGIRL>hood home.~ [CB_77319]
+@23   = ~I am Ceeb.  I have followed your progress in this world for as long as I have been aware that you exist.  Which, I must say, has been nearly four years since you first left your <PRO_GIRLBOY>hood home.~ [CB_77319]
 @24   = ~Why would you be interested in my life?  My doings?  I don't really think its any of your concern what I do.~
 @25   = ~You are most definately not simple.  And you lie to me.  You do know what I speak about.  So until you admit it to yourself, you cannot admit it to me.~ [CB_77320]
 @26   = ~You seem to know much.  But why do you know so much about me?  Who are you to know so much about me?~
