--- SOS\Language\English\CBSOUMJ3.tra	Thu Feb 02 12:54:48 2006
+++ C:\BWP Patchstudio\patched files\SOS\Language\English\CBSOUMJ3.tra	Fri Nov 12 16:50:25 2010
@@ -63,7 +63,7 @@
 @62   = ~No, Major, I have not.~
 @63   = ~I've been thinking about it.  I think I would be a good Mayor for Soubar.  I'll do it.~
 @64   = ~Ah well.  If you change your mind, I'll still be here in the barracks.  With tremendous amounts of work to do.~
-@65   = ~<PRO_LORDLADY> Mayor <CHARNAME>.  Soubar will be lead quite well with you in charge.  Don't worry about the defense of the town;  I'll care for that.~
+@65   = ~<PRO_LADYLORD> Mayor <CHARNAME>.  Soubar will be lead quite well with you in charge.  Don't worry about the defense of the town;  I'll care for that.~
 @66   = ~That will be all, Captain.  Dismissed.~
 @67   = ~If you need anything before you make your assault on Lyrar's, let me know, Chief Lieutanant.~
 @68   = ~Chief Lieutanant, if you are having difficulties gaining entry to Lyrar's, perhaps you should try one of the other routes.~
