--- SOS\Language\German\CBSOUMJ3.tra	Thu Aug 24 09:16:22 2006
+++ C:\BWP Patchstudio\patched files\SOS\Language\German\CBSOUMJ3.tra	Wed Nov 11 13:51:33 2015
@@ -63,7 +63,7 @@
 @62 = ~Nein, Major, bin ich nicht. ~
 @63 = ~Ich habe darüber nachgedacht. Ich denke, dass ich ein guter Bürgermeister für Soubar wäre. Ich werde es tun. ~ ~Ich habe darüber nachgedacht. Ich denke, dass ich eine gute Bürgermeisterin für Soubar wäre. Ich werde es tun. ~
 @64 = ~Ah gut. Wenn Ihr Eure Meinung ändert, ich bin noch hier in den Baracken. Mit enormer Menge an Arbeit, die zu tun ist. ~
-@65 = ~<PRO_LORDLADY> Bürgermeister <CHARNAME>. Soubar wird unter Eurer Leitung ganz gut geführt werden. Kümmert Euch nicht um die Verteidigung der Stadt; ich erledige das. ~ ~<PRO_LORDLADY> Bürgermeisterin <CHARNAME>. Soubar wird unter Eurer Leitung ganz gut geführt werden. Kümmert Euch nicht um die Verteidigung der Stadt; ich erledige das. ~
+@65 = ~<PRO_LADYLORD> Bürgermeister <CHARNAME>. Soubar wird unter Eurer Leitung ganz gut geführt werden. Kümmert Euch nicht um die Verteidigung der Stadt; ich erledige das. ~ ~<PRO_LADYLORD> Bürgermeisterin <CHARNAME>. Soubar wird unter Eurer Leitung ganz gut geführt werden. Kümmert Euch nicht um die Verteidigung der Stadt; ich erledige das. ~
 @66 = ~Das ist alles, Hauptmann. Wegtreten. ~
 @67 = ~Wenn Ihr irgendetwas braucht, bevor Ihr Euren Angriff auf Lyrar macht, lasst es mich wissen, Oberleutnant. ~
 @68 = ~ Oberleutnant, wenn Ihr Schwierigkeiten habt, den Zugang zu Lyrar zu erlangen, solltet Ihr vielleicht einen der anderen Wege versuchen. ~
