--- SOS\Language\Italian\CBSOUMJR.tra	Fri Sep 12 22:01:34 2008
+++ C:\BWP Patchstudio\patched files\SOS\Language\Italian\CBSOUMJR.tra	Wed Nov 11 13:51:33 2015
@@ -221,7 +221,7 @@
 @220  = ~No, Maggiore.~
 @221  = ~Ci ho riflettuto molto e credo che in effetti potrei essere un buon Sindaco per Soubar. Accetto.~
 @222  = ~D'accordo. Se doveste cambiare idea, sapete dove trovarmi. Indaffarato come sempre, naturalmente.~
-@223  = ~<PRO_LORDLADY> Sindaco <CHARNAME>. Soubar avrà in voi un'ottima guida. Non dovete preoccuparvi della difesa della città: quella è compito mio.~
+@223  = ~<PRO_LADYLORD> Sindaco <CHARNAME>. Soubar avrà in voi un'ottima guida. Non dovete preoccuparvi della difesa della città: quella è compito mio.~
 @224  = ~Sergente, devo dissentire. Ci sono troppe prove a sostegno della tesi opposta. Vi prego di continuare le vostre indagini.~
 @225  = ~E' tutto, Sergente.~
 @226  = ~Molto bene, Sergente. Andate avanti.~
