--- SOS\Language\German\CBSOUMJR.tra	Fri Sep 08 20:07:44 2006
+++ C:\BWP Patchstudio\patched files\SOS\Language\German\CBSOUMJR.tra	Wed Nov 11 13:51:33 2015
@@ -221,7 +221,7 @@
 @220 = ~Nein, Major, bin ich nicht. ~
 @221 = ~Ich habe darüber nachgedacht. Ich denke, dass ich ein guter Bürgermeister für Soubar wäre. Ich werde es tun. ~ ~Ich habe darüber nachgedacht. Ich denke, dass ich eine gute Bürgermeisterin für Soubar wäre. Ich werde es tun. ~
 @222 = ~Ah gut. Wenn Ihr Eure Meinung ändert, ich bin noch hier in den Baracken. Mit enormer Menge an Arbeit, die zu tun ist. ~
-@223 = ~<PRO_LORDLADY> Bürgermeister <CHARNAME>. Soubar wird unter Eurer Leitung ganz gut geführt werden. Kümmert Euch nicht um die Verteidigung der Stadt; ich erledige das. ~ ~<PRO_LORDLADY> Bürgermeisterin <CHARNAME>. Soubar wird unter Eurer Leitung ganz gut geführt werden. Kümmert Euch nicht um die Verteidigung der Stadt; ich erledige das. ~
+@223 = ~<PRO_LADYLORD> Bürgermeister <CHARNAME>. Soubar wird unter Eurer Leitung ganz gut geführt werden. Kümmert Euch nicht um die Verteidigung der Stadt; ich erledige das. ~ ~<PRO_LADYLORD> Bürgermeisterin <CHARNAME>. Soubar wird unter Eurer Leitung ganz gut geführt werden. Kümmert Euch nicht um die Verteidigung der Stadt; ich erledige das. ~
 @224 = ~Feldwebel, ich muss nicht mit Ihnen übereinstimmen. Es gibt zu viel Beweise für das Gegenteil. Setzt Eure Untersuchungen fort. ~
 @225 = ~Wegtreten, Feldwebel. ~
 @226 = ~Sehr gut, Feldwebel. Weitermachen. ~
