--- SOS\Language\English\CBWLSERV.tra	Thu Feb 02 12:47:16 2006
+++ C:\BWP Patchstudio\patched files\SOS\Language\English\CBWLSERV.tra	Fri Nov 12 16:12:57 2010
@@ -50,7 +50,7 @@
 @49   = ~Thank you, that will be quite a time-saver.~
 @50   = ~Where can I find Sir William then?~
 @51   = ~I know that he was conducting business in Waterdeep but recently sent word to Bylanna that he would be conducting some affairs on Brynnlaw.  Perhaps you should search there first.~
-@52   = ~Sir William?  Host a dinner party?  He had not informed me he was returning from Brynnlaw so quickly!  I must prepare the _hor'derves_.  Oh, there is so much to do!  Excuse me.~
+@52   = ~Sir William?  Host a dinner party?  He had not informed me he was returning from Brynnlaw so quickly!  I must prepare the hors d'oeuvre.  Oh, there is so much to do!  Excuse me.~
 @53   = ~Rob?  Steal?  You cannot!  I will not tell my master's secrets.~
 @54   = ~If you do not leave, I will be forced to summon the authorities.  Are you going to leave quietly?~
 @55   = ~Of course not.  Tell me where the safe is or I will kill you and find it myself.~
