--- SOS\SNIP\tBALDUR.BAF	Fri Jun 17 15:19:46 2005
+++ C:\BWP Patchstudio\patched files\SOS\SNIP\tBALDUR.BAF	Fri Apr 02 04:52:35 2010
@@ -10,7 +10,7 @@
 	AreaType(OUTDOOR)
 	!StateCheck("CHARLOTT",STATE_SLEEPING)
 	!StateCheck("NALIA",STATE_SLEEPING)
-	!StateCheck("Player1",STATE_SLEEPING)
+	!StateCheck(Player1,STATE_SLEEPING)
 THEN
 	RESPONSE #100
 		ActionOverride("NALIA",StartDialogueNoSet("CHARLOTT"))
