--- laviconia/setup-laviconia.tp2	2015-02-01 00:35:04.000000000 -0500
+++ Fixpack/laviconia/setup-laviconia.tp2	2016-11-26 02:49:17.714894900 -0500
@@ -1,6 +1,6 @@
 BACKUP ~laviconia/backup~
 AUTHOR ~www.shsforums.net/topic/44027-laviconia-tweak/~
-VERSION ~v4~
+VERSION ~v5 BWP Fix~
 README ~laviconia/laviconia-readme.txt~
 
 AUTO_TRA ~laviconia/translations/%s~
@@ -12,16 +12,14 @@
   // No exception, everything needs to be converted
   ACTION_DEFINE_ARRAY VaVinoconvert BEGIN END
 
-  // Only setup-Innershade.tra needs to be reloaded
-  ACTION_DEFINE_ARRAY VaVireload BEGIN Setup END
+  // Only setup.tra needs to be reloaded
+  ACTION_DEFINE_ARRAY VaVireload BEGIN setup END
 
-  // There is no "language" folder so the root of all languages in %MOD_FOLDER%
   LAF HANDLE_CHARSETS
     INT_VAR
       infer_charset = 1
     STR_VAR
       tra_path = EVAL ~%MOD_FOLDER%/translations~
-      // charset_table = VaVicharsets // Included for illustrative purposes.
       noconvert_array = VaVinoconvert
       reload_array = VaVireload
   END
@@ -60,7 +58,7 @@
 OUTER_SET ladual = 1
 INCLUDE ~laviconia/lib/lav-init.tpa~
 
-ACTION_IF GAME_IS ~tutu tutu_totsc bgt~ BEGIN
+ACTION_IF GAME_IS ~tutu tutu_totsc bgt eet~ BEGIN
   COPY_EXISTING ~%tutu_var%viconi.cre~ ~override~
     LAUNCH_PATCH_MACRO ~LaViconiaPatch~
     WRITE_BYTE 0x54 13 //Save vs death (was 10)
@@ -128,7 +126,7 @@
   BUT_ONLY
 END
 
-ACTION_IF GAME_IS ~bg2 tob bg2ee~ BEGIN
+ACTION_IF GAME_IS ~bg2 tob bg2ee eet~ BEGIN
   EXTEND_BOTTOM ~ar1000.bcs~ ~laviconia/scripts/ar1000.baf~
   EXTEND_TOP ~viconia.bcs~ ~laviconia/scripts/viconia.baf~
 
@@ -246,7 +244,7 @@
   BUT_ONLY
 END
 
-ACTION_IF GAME_IS ~tob bg2ee~ BEGIN
+ACTION_IF GAME_IS ~tob bg2ee eet~ BEGIN
   EXTEND_TOP ~vico25.bcs~ ~laviconia\scripts\viconia.baf~
 
   COPY_EXISTING ~viconi16.cre~ ~override~
@@ -311,7 +309,7 @@
 OUTER_SET ladual = 0
 INCLUDE ~laviconia/lib/lav-init.tpa~
 
-ACTION_IF GAME_IS ~tutu tutu_totsc bgt~ BEGIN
+ACTION_IF GAME_IS ~tutu tutu_totsc bgt eet~ BEGIN
   COPY_EXISTING ~%tutu_var%viconi.cre~ ~override~
     LAUNCH_PATCH_MACRO ~LaViconiaPatch~
     WRITE_SHORT 0x24 7 //Current HP (was 12)
@@ -386,7 +384,7 @@
   BUT_ONLY
 END
 
-ACTION_IF GAME_IS ~bg2 tob bg2ee~ BEGIN
+ACTION_IF GAME_IS ~bg2 tob bg2ee eet~ BEGIN
   EXTEND_BOTTOM ~ar1000.bcs~ ~laviconia/scripts/ar1000.baf~
   EXTEND_TOP ~viconia.bcs~ ~laviconia/scripts/viconia.baf~
 
@@ -562,7 +560,7 @@
   BUT_ONLY
 END
 
-ACTION_IF GAME_IS ~tob bg2ee~ BEGIN
+ACTION_IF GAME_IS ~tob bg2ee eet~ BEGIN
   EXTEND_TOP ~vico25.bcs~ ~laviconia\scripts\viconia.baf~
 
   COPY_EXISTING ~viconi16.cre~ ~override~
@@ -626,7 +624,7 @@
 
 COPY ~laviconia/portrait~ ~override~
 
-ACTION_IF GAME_IS ~tutu tutu_totsc bgt~ BEGIN
+ACTION_IF GAME_IS ~tutu tutu_totsc bgt eet~ BEGIN
   COPY ~laviconia/portrait/nviconl.bmp~ ~override/viconial.bmp~
   COPY ~laviconia/portrait/nviconm.bmp~ ~override/viconiam.bmp~
   COPY ~laviconia/portrait/nvicons.bmp~ ~override/viconias.bmp~
