--- ACBre\baf\bg2\compile\acbre25.baf	Thu Feb 12 00:07:26 2015
+++ C:\BWP Patchstudio\patched files\ACBre\baf\bg2\compile\acbre25.baf	Tue Nov 01 11:00:00 2016
@@ -395,14 +395,11 @@
 IF
 	InParty(Myself) 
 	See(Player1)
-OR(7)
+OR(4)
 	Global("ACCOMPON01","LOCALS",1)
 	Global("ACCOMPON08","LOCALS",1)
 	Global("ACCOMPON09","LOCALS",1)
 	Global("ACHGHEART","LOCALS",1)
-	Global("ACDRAGONSCALES","LOCALS",1)
-	Global("ACDRAGONSCALES","LOCALS",3)
-	Global("ACDRAGONSCALES","LOCALS",5)
 THEN
 	RESPONSE #100
 		StartDialogNoSet(Player1)
