--- ACBre\d\bg1\ACBREB.D	Thu Feb 12 04:37:26 2015
+++ C:\BWP Patchstudio\patched files\ACBre\d\bg1\ACBREB.D	Mon Aug 31 21:07:34 2015
@@ -310,7 +310,7 @@
 @178 = @179
 ==%tutu_var%MENDA1 @180
 END
-IF ~~ THEN DO ~SetGlobal("ReadyToSail","GLOBAL",1)ActionOverride("MENDA1",EscapeAreaDestroy(90))~ EXIT
+IF ~~ THEN DO ~SetGlobal("ReadyToSail","GLOBAL",1)ActionOverride("MENDAS",EscapeAreaDestroy(90))~ EXIT
 
 INTERJECT %tutu_var%MULAHE 3 ACMULAHE3
 ==ACBREB IF ~InParty("ACBRE")InMyArea("ACBRE")!StateCheck("ACBRE",CD_STATE_NOTVALID)~ THEN 
@@ -738,7 +738,7 @@
 END
 IF ~~ THEN EXTERN %tutu_var%OGRECO 1
 
-APPEND %tutu_var%JARED
+APPEND JARED
 IF ~~ THEN BEGIN ACBRE1
 SAY @376
 IF ~~ THEN DO ~AddexperienceParty(150)
@@ -869,7 +869,7 @@
 @57
 END
 
-INTERJECT_COPY_TRANS %tutu_var%EDWIN %EDWIN_STATE_0% ACEDWIN74
+INTERJECT_COPY_TRANS %tutu_var%EDWIN 0 ACEDWIN74
 ==ACBREB IF ~InParty("ACBRE")See("ACBRE")!StateCheck("ACBRE",CD_STATE_NOTVALID)~ THEN 
 @88
 ==%tutu_var%EDWIN IF ~InParty("ACBRE")See("ACBRE")!StateCheck("ACBRE",CD_STATE_NOTVALID)~ THEN 
@@ -884,7 +884,7 @@
 @93
 END
 
-INTERJECT_COPY_TRANS %tutu_var%EDWIN %EDWIN_STATE_1% ACEDWIN75
+INTERJECT_COPY_TRANS %tutu_var%EDWIN 1 ACEDWIN75
 ==ACBREB IF ~InParty("ACBRE")See("ACBRE")!StateCheck("ACBRE",CD_STATE_NOTVALID)~ THEN 
 @88
 ==%tutu_var%EDWIN IF ~InParty("ACBRE")See("ACBRE")!StateCheck("ACBRE",CD_STATE_NOTVALID)~ THEN 
@@ -899,7 +899,7 @@
 @93
 END
 
-INTERJECT_COPY_TRANS %tutu_var%EDWIN %EDWIN_STATE_2% ACEDWIN76
+INTERJECT_COPY_TRANS %tutu_var%EDWIN 2 ACEDWIN76
 ==ACBREB IF ~InParty("ACBRE")See("ACBRE")!StateCheck("ACBRE",CD_STATE_NOTVALID)~ THEN 
 @88
 ==%tutu_var%EDWIN IF ~InParty("ACBRE")See("ACBRE")!StateCheck("ACBRE",CD_STATE_NOTVALID)~ THEN 
