--- JondalarFix\Setup-JondalarFix.tp2	Mon Nov 22 16:52:08 2010
+++ C:\BWP Patchstudio\patched files\JondalarFix\Setup-JondalarFix.tp2	Mon Oct 07 08:47:43 2013
@@ -1,7 +1,7 @@
 BACKUP ~JondalarFix/backup~
 AUTHOR ~adul@adul.net~
+VERSION ~BWP Patching version~
 
 BEGIN ~Jondalar Fix for BGT~
 
-COPY ~JondalarFix/JONDAL.DLG~ ~override~
-  SAY 0x074 ~I was going to teach you a thing or two about fighting, but I see you are rather handy with that weapon already.  You should go talk to the Gatewarden if you want some experience in fighting with a group of people.  You can find him just south of here.  Out there it can be pretty important to know how to fight with allies at your side.~
\ No newline at end of file
+COMPILE ~JondalarFix/JONDAL.d~
\ No newline at end of file
