--- wheels\sfo\general\lib_iterate.tpa	Tue Feb  9 20:03:14 2016
+++ wheels\sfo\general\lib_iterate2.tpa	Sat Apr  2 22:04:26 2016
@@ -626,9 +626,11 @@
       COUNT_2DA_ROWS 2 rowcount
       FOR (i=0;i<rowcount;i+=1) BEGIN
           READ_2DA_ENTRY i 0 2 idscode
-          LPF idscode_to_spell INT_VAR idscode RET resref END
-          SPRINT list "%list% %resref%.spl"
-      END
+          PATCH_IF IS_AN_INT idscode BEGIN
+            LPF idscode_to_spell INT_VAR idscode RET resref END
+            SPRINT list "%list% %resref%.spl"
+          END 
+       END
    BUT_ONLY
    COPY_EXISTING_REGEXP "\(clab\|lu\).*\.2da" "%workspace%"
        COUNT_2DA_COLS colcount
