--- wheels\setup-wheels.tp2	Mon Nov 11 00:00:42 2013
+++ C:\BWP Patchstudio\patched files\wheels\setup-wheels.tp2	Fri Dec 18 23:51:43 2015
@@ -1,6 +1,6 @@
 BACKUP ~stratagems_external/backup/wheels~
 AUTHOR ~For help troubleshooting installation problems, go to the Wheels of Prophecy forum at forums.gibberlings3.net.~
-VERSION ~v5~
+VERSION ~v5 BWP Fix~
 README ~wheels/readme-wheels.html~
 AUTO_EVAL_STRINGS
 AUTO_TRA ~%s~
