--- JA#BGT_AdvPack\BAF\AR7240.BAF	Tue Mar 20 15:03:38 2012
+++ C:\BWP Patchstudio\patched files\JA#BGT_AdvPack\BAF\AR7240.BAF	Mon Aug 24 21:17:10 2015
@@ -11,7 +11,7 @@
 		CreateCreature("MEPSAL01",[424.286],1)
 		CreateCreature("RAMAZI",[422.293],6)
 		ActionOverride("Ramazith",SetNumTimesTalkedTo(1))
-		ActionOverride("RAMAZITH",Enemy())
+//		ActionOverride("RAMAZITH",Enemy())
 		Wait(1)
 		ActionOverride("RAMAZITH",StartDialogueNoSet([PC]))
 END
\ No newline at end of file
