--- JA#BGT_AdvPack\DLG\ja#law.d	Sun Apr 01 11:15:37 2012
+++ C:\BWP Patchstudio\patched files\JA#BGT_AdvPack\DLG\ja#law.d	Wed May 01 15:11:35 2013
@@ -36,7 +36,7 @@
 IF ~Global("JA#FRIEND_RAUS","GLOBAL",0)~ THEN BEGIN 3
   SAY ~Halt! Stehen bleiben! Ihr wurdet beim Stehlen im Freundlichen Arm erwischt und seid verhaftet! Nach dem Gesetz der Grenze wäre es unser gutes Recht, Euren Kopf zu fordern.~
   IF ~ReactionLT(LastTalkedToBy,NEUTRAL_LOWER)~ THEN GOTO 4
-  IF ~ReactionGT(LastTalkedToBy,HOSTILE_UPPER)~ THEN GOTO 5
+  IF ~ReactionGT(LastTalkedToBy,HOSTILE_UPPER)~ THEN GOTO 5
 END
 
 IF ~~ THEN BEGIN 4
