--- Foundling/Scripts/BOTSMITH.baf	2013-06-02 18:40:02.000000000 -0400
+++ Fixpack/Foundling/Scripts/BOTSMITH.baf	2016-11-30 17:15:33.993793700 -0500
@@ -1,8 +1,9 @@
 IF
-  Global("ItemMaker","GLOBAL",178)
+  Global("L#FouItemMaker","GLOBAL",178)
   Global("ImpForgeStuff","GLOBAL",1)
 THEN
   RESPONSE #100
+    SetGlobal("L#FouItemMaker","GLOBAL",0)
     SetGlobal("ImpForgeStuff","GLOBAL",0)
     GiveItemCreate("L#FGLVF",Player1,0,0,0)
     StartDialogNoSet(Player1)
