--- Foundling\Dialogues\L#Fou25J.d	Wed May 20 23:24:46 2015
+++ C:\BWP Patchstudio\patched files\Foundling\Dialogues\L#Fou25J.d	Sat Oct 29 17:41:50 2016
@@ -407,24 +407,24 @@
 
 CHAIN L#Fou25J Love012
 @188 /* I need to think about it. */
-DO ~SetGlobal("L#FouToBTimeredTalk","GLOBAL",11)~ EXIT
+DO ~SetGlobal("L#FouToBTimeredTalk","GLOBAL",12)~ EXIT
 
 CHAIN L#Fou25J Love013
 @189 /* I'm not sure if that's for me, but I may think about it. */
-DO ~SetGlobal("L#FouToBTimeredTalk","GLOBAL",11)~ EXIT
+DO ~SetGlobal("L#FouToBTimeredTalk","GLOBAL",12)~ EXIT
 
 
 CHAIN L#Fou25J Love014
 @190 /* I'm not everyone, though. */
-DO ~SetGlobal("L#FouToBTimeredTalk","GLOBAL",11)~ EXIT
+DO ~SetGlobal("L#FouToBTimeredTalk","GLOBAL",12)~ EXIT
 
 CHAIN L#Fou25J Love015
 @191 /* Maybe. */
-DO ~SetGlobal("L#FouToBTimeredTalk","GLOBAL",11)~ EXIT
+DO ~SetGlobal("L#FouToBTimeredTalk","GLOBAL",12)~ EXIT
 
 CHAIN L#Fou25J Love002
 @192 /* Fine.  I won't bother you. */
-DO ~SetGlobal("L#FouToBTimeredTalk","GLOBAL",11)~ EXIT
+DO ~SetGlobal("L#FouToBTimeredTalk","GLOBAL",12)~ EXIT
  
 //////////////////////////
 ///////SARADUSH///////////
