--- Foundling/translations/english/L#FOU.TRA	2015-05-22 01:36:16.000000000 -0400
+++ Fixpack/Foundling/translations/english/L#FOU.TRA	2016-11-26 20:34:47.884526200 -0500
@@ -1,5 +1,5 @@
 @0    = ~I won't let you continue. You should go back--take the same route you used to get here and just forget this place. Everything happening here is beyond your comprehension.~ [L#Fod01]
-@1    = ~We know exactly what is going on here, creature, so I'd rather you stop telling us what you ìthinkî we know and expect.~
+@1    = ~We know exactly what is going on here, creature, so I'd rather you stop telling us what you ‚Äúthink‚Äù we know and expect.~
 @2    = ~You're the one who was kept in the western cells!  It seems you forced the lock and killed his minions...~
 @3    = ~I was offered aid.  We slew many on our way here.  We have the support of Arvoreen and Amaunator: you and your master stand no chance.~
 @4    = ~I'm surprised we've met someone who's telling us to go back instead of trying to kill us where we stand.  But somehow I doubt this creature's good intentions.~
@@ -24,7 +24,7 @@
 @23   = ~I did nothing to them.  It was someone else.  But not me.~
 @24   = ~Forgive me for interrupting, my friend, but I am not sure we should believe in these words.~
 @25   = ~I didn't expect those of your plane to be so blind with self-confidence.~
-@26   = ~Boo sees through you, shadowy-man! ÖAlthough I suppose we can all see through you, but it doesn't matter!  We know there be evil here needing a good butt-kicking!  Out of our way, shady one, that we may deliver it like all good-doers deliver the milk to the righteous families! Right, Boo?~
+@26   = ~Boo sees through you, shadowy-man! ‚Ä¶Although I suppose we can all see through you, but it doesn't matter!  We know there be evil here needing a good butt-kicking!  Out of our way, shady one, that we may deliver it like all good-doers deliver the milk to the righteous families! Right, Boo?~
 @27   = ~(Boo) *Squeak!*~ [GAM_48]
 @28   = ~Easy, Minsc my big friend.  If you shout, you may lure some more of his kind and we should deal with this one, first.~
 @29   = ~This is leading us nowhere.  Go back or you'll diminish right here.~
@@ -51,7 +51,7 @@
 @50   = ~Up there you can find my master--Thaxll'ssillyia--and his master, the Shade Lord.  Their power is far too formidable. You cannot hope to win. You just can't. It's not possible. Save yourselves and walk away.~
 @51   = ~Where is Patrick?  Did you see him?~
 @52   = ~The other short man?  The one of your height?~
-@53   = ~Yes.  I suppose I'll take that as a ìyesî.  <CHARNAME>, I cannot let him down, so no matter what will happen, we need to go there.~
+@53   = ~Yes.  I suppose I'll take that as a ‚Äúyes.‚Äù  <CHARNAME>, I cannot let him down, so no matter what will happen, we need to go there.~
 @54   = ~I'm not sure if he can be saved.~
 @55   = ~What do you mean?~
 @56   = ~He's with the Shade Lord.  That's all I know.~
@@ -77,7 +77,7 @@
 @76   = ~I see...~
 @77   = ~You should know that I am nothing more than a prisoner here.  I wish I could support you and make my master disappear on your way to the Shade Lord, but I can't oppose him openly.~
 @78   = ~I was sent to try to stop you. I did so. I'm not going to stand in your way or fight you as I was never ordered to.~
-@79   = ~Here, take these. They are the belongings of--(he seems troubled again)-ñthey won't be needing them anymore.~
+@79   = ~Here, take these. They are the belongings of--(he seems troubled again)--they won't be needing them anymore.~
 @80   = ~I am sure you can make proper use of these. That's all I can do.~
 @81   = ~You should examine these "gifts" before you use them. They may be cursed.~
 @82   = ~You've done enough.~
