--- RoTerror\SNIP\BAF\AR2601.BAF	Tue Aug 08 18:53:26 2006
+++ C:\BWP Patchstudio\patched files\RoTerror\SNIP\BAF\AR2601.BAF	Sun Oct 04 16:18:08 2009
@@ -1,3 +1,4 @@
+/*
 IF
 	OnCreation()
 	Global("DrizztEncounter","AR2601",0)
@@ -6,5 +7,27 @@
 		SetGlobal("DrizztEncounter","AR2601",1)
 		Continue()
 END
+*/
 
-
+IF
+	OnCreation()
+	OR(4)
+		InParty("Driz2")
+		InParty("Wulfgi2")
+		InParty("Reg2")
+		InParty("Brue2")
+	Global("DrizztEncounter","GLOBAL",0)
+THEN
+	RESPONSE #100
+		Explore()
+		SetEncounterProbability("AR2500","AR0020",0)
+		SetEncounterProbability("AR2500","AR1300",0)
+		SetEncounterProbability("AR2500","AR1304",0)
+		SetEncounterProbability("AR2500","AR1700",0)
+		SetGlobal("DrizztEncounter","GLOBAL",1)
+		CreateCreature("TROLL01",[692.332],2) // Troll
+		CreateCreature("TROLL01",[673.256],2) // Troll
+		CreateCreature("TROLL01",[752.294],2) // Troll
+		CreateCreature("TROLL01",[790.337],2) // Troll
+		CreateCreature("TROLL01",[741.225],2) // Troll
+END
\ No newline at end of file
