--- RoTerror\SNIP\BAF\Mage\MAGE14C.BAF	Wed Mar 15 10:57:10 2006
+++ C:\BWP Patchstudio\patched files\RoTerror\SNIP\BAF\Mage\MAGE14C.BAF	Thu Apr 01 16:55:19 2010
@@ -33,7 +33,7 @@
 	See(NearestEnemyOf(Myself))
 	!HasBounceEffects(Myself)
 	!HasImmunityEffects(Myself)
-	!Dead("yself)")
+	!StateCheck(Myself,STATE_REALLY_DEAD)
 	HaveSpell(WIZARD_SPELL_TURNING)
 THEN
 	RESPONSE #100
@@ -44,7 +44,7 @@
 	See(NearestEnemyOf(Myself))
 	HaveSpell(WIZARD_STONE_SKIN)
 	CheckStatLT(Myself,1,STONESKINS)
-	!Dead("yself)")
+	!StateCheck(Myself,STATE_REALLY_DEAD)
 THEN
 	RESPONSE #100
 		Spell(Myself,WIZARD_STONE_SKIN)
