--- RoTerror\SNIP\BAF\Mage\MAGE20B.BAF	Fri Feb 24 19:35:02 2006
+++ C:\BWP Patchstudio\patched files\RoTerror\SNIP\BAF\Mage\MAGE20B.BAF	Thu Apr 01 16:55:19 2010
@@ -1,7 +1,7 @@
 IF
 	See(NearestEnemyOf(Myself))
 	Global("Prep","LOCALS",0)
-	!Dead("yself)")
+	!StateCheck(Myself,STATE_REALLY_DEAD)
 THEN
 	RESPONSE #100
 		ApplySpell(Myself,WIZARD_STONE_SKIN)
@@ -13,7 +13,7 @@
 IF
 	See(NearestEnemyOf(Myself))
 	Global("SpellTrigger","LOCALS",0)
-	!Dead("yself)")
+	!StateCheck(Myself,STATE_REALLY_DEAD)
 THEN
 	RESPONSE #100
 		DisplayString(Myself,39968) // Spell Trigger - Fired
@@ -102,7 +102,7 @@
 IF
 	See(NearestEnemyOf(Myself))
 	Global("Prep","LOCALS",1)
-	!Dead("yself)")
+	!StateCheck(Myself,STATE_REALLY_DEAD)
 THEN
 	RESPONSE #100
 		Spell(Myself,WIZARD_SPELL_TURNING)
@@ -123,7 +123,7 @@
 	See(NearestEnemyOf(Myself))
 	TookDamage()
 	Global("Prep2","LOCALS",0)
-	!Dead("yself)")
+	!StateCheck(Myself,STATE_REALLY_DEAD)
 THEN
 	RESPONSE #100
 		DisplayString(Myself,43050) // Chain Contingency - Improved Mantle
@@ -139,7 +139,7 @@
 	See(NearestEnemyOf(Myself))
 	TookDamage()
 	Global("Prep2","LOCALS",1)
-	!Dead("yself)")
+	!StateCheck(Myself,STATE_REALLY_DEAD)
 THEN
 	RESPONSE #100
 		DisplayString(Myself,40252) // Contingency - Protection from Magical Weapons
@@ -160,7 +160,7 @@
 IF
 	See(NearestEnemyOf(Myself))
 	Global("SpellTrigger","LOCALS",0)
-	!Dead("yself)")
+	!StateCheck(Myself,STATE_REALLY_DEAD)
 THEN
 	RESPONSE #100
 		DisplayString(Myself,39969) // Spell Sequencer - Fired
