--- RoTerror\BAF\RA5100.BAF	Fri Feb 24 20:40:28 2006
+++ C:\BWP Patchstudio\patched files\RoTerror\BAF\RA5100.BAF	Tue Nov 01 08:29:39 2016
@@ -78,7 +78,7 @@
 		ActionOverride("EgMage2",DestroySelf())
 		Wait(1)
 		IncrementGlobal("BardPlot1","GLOBAL",-80)
-		IncrementGlobal("JosterLeave","GLOBAL",-80)
+		IncrementGlobal("JosterReturn","GLOBAL",-80)
 		IncrementGlobal("NotifyThiefHead","GLOBAL",-80)
 		IncrementGlobal("KeepPlot","GLOBAL",-80)
 		IncrementGlobal("KPRunnerSpawn","GLOBAL",-80)
@@ -177,4 +177,18 @@
 		SetGlobal("jkasdflk","GLOBAL",0)
 END
 
-
+
+IF
+	PartyRested()
+	//AreaCheck("RA5100")
+THEN
+	RESPONSE #100
+		ApplyDamage(Player2,50,COLD)
+		ApplyDamage(Player3,50,COLD)
+		ApplyDamage(Player4,50,COLD)
+		ApplyDamage(Player5,50,COLD)
+		ApplyDamage(Player6,50,COLD)
+		ApplyDamage(Myself,50,COLD)
+		Wait(1)
+		DisplayStringHead(Myself,@573848) // Icecold wind blows from the lake, chilling your bones. It is not wise to sleep outside.
+END
