--- BG2eeAR\translations\polish\setup-bgeear.tra	Mon Aug 18 11:21:49 2014
+++ C:\BWP Patchstudio\patched files\BG2eeAR\translations\polish\setup-bgeear.tra	Fri Mar 06 14:07:30 2015
@@ -1,8 +1,4 @@
-@0    = ~Wykryto zainstalowane BG:EE~
-@1    = ~Wykryto zainstalowane Tutu~
-@2    = ~Wykryto zainstalowane BGT~
-@3    = ~Zainstaluj na BG:EE, Tutu lub BGT~
-@4    = ~TeamBG's Armors dla BG:EE, BGT i TuTu Wersja 1.05~
+@4    = ~TeamBG's Armor Pack for BG2EE~
 @5    = ~Buty~
 @6    = ~Buty Uników~
 @7    = ~Buty są zwykle ręcznie wyrabiane przez szewców. Zwykłe buty są robione z użyciem jakiegoś ogólnego modelu stopy, ale te dobre wykonywane są na zamówienie i dopasowywane do stóp zamawiającego.
