--- Bonehillv275\bafs\BH2000.BAF	Sat Aug 04 14:32:02 2007
+++ C:\BWP Patchstudio\patched files\Bonehillv275\bafs\BH2000.BAF	Mon Feb 01 17:00:38 2016
@@ -31,7 +31,7 @@
     CreateCreature("BHOscar",[4280.2240],0)
     CreateCreature("BHPhilma",[4535.2531],5)
     CreateCreature("BHBasmar",[4210.2490],13)
-    ActionOverride("BHBasmar",StartDialogue("BHBasmar",[PC]))
+    ActionOverride("BHBasmar",StartDialog("BHBasmar",[PC]))
 END
 
 IF
@@ -53,7 +53,7 @@
   RESPONSE #100
     ActionOverride("BHASCUE",DestroySelf())
     ActionOverride("BHOGUARD",DestroySelf())
-    SetGlobal("BHEscortSpawn","GLOBAL",3)
+    SetGlobal("BHEscortedParty","GLOBAL",3)
 END
 
 IF
