--- Bonehillv275\BHAreas.bat	Sun Apr 04 11:32:07 2010
+++ C:\BWP Patchstudio\patched files\Bonehillv275\BHAreas.bat	Sat Apr 17 12:24:02 2010
@@ -51,7 +51,7 @@
 tisunpack -s -h -o BH-TIS2\BH2301.tis BoneHillv275\TIZ\BH2301.tiz
 del tisunpack.exe
 
-xcopy /q BoneHillv275\sounds\*.ogg BH-REST\*.*
+xcopy /q /y BoneHillv275\sounds\*.ogg BH-REST\*.*
 copy BoneHillv275\oggdec.exe BH-REST\*.*
 cd BH-REST
 oggdec *.ogg
@@ -66,196 +66,196 @@
 weidu --out BH-REST --biff-get-rest GAFIREP1.WAV GAHOSMAG.WAV GASMITH.WAV
 
 
-move override\BH*.* BH-REST
-move override\1003MS1.BAM BH-REST
-move override\1007MS1.BAM BH-REST
-move override\1008MS1.BAM BH-REST
-move override\25SPELL.dlg BH-REST
-move override\9101MS1.bam BH-REST
-move override\9101MS2.bam BH-REST
-move override\9105MS1.BAM BH-REST
-move override\9106MS1.BAM BH-REST
-move override\9108MS1.BAM BH-REST
-move override\9711MS1.BAM BH-REST
-move override\9714MS1.BAM BH-REST
-move override\9714MS2.BAM BH-REST
-move override\9714MS3.BAM BH-REST
-move override\9714MS4.BAM BH-REST
-move override\9717MS1.BAM BH-REST
-move override\9717MS2.BAM BH-REST
-move override\AR6526.BCS  BH-REST 2> Null
-move override\AR7223.bcs  BH-REST 2> Null
-move override\AR7800.bcs  BH-REST 2> Null
-move override\AR9300.are  BH-REST 2> Null
-move override\AR9300.bcs  BH-REST 2> Null
-move override\AR9302.bcs  BH-REST 2> Null
-move override\ARAMIS.BAM  BH-REST
-move override\BALDUR.BCS  BH-REST
-move override\BERSERKR.ITM BH-REST
-move override\BFABIO.dlg  BH-REST 2> Null
-move override\BGARRI.dlg  BH-REST 2> Null
-move override\BMINSC.dlg  BH-REST 2> Null
-move override\BOULDER.ITM BH-REST
-move override\BOULDER.PRO BH-REST
-move override\BRANWJ.dlg  BH-REST 2> Null
-move override\cbhghan1.bam BH-REST
-move override\CBHGHAND.BAM BH-REST
-move override\COSPREM.BAM BH-REST
-move override\DUDOORE.bcs BH-REST
-move override\DUDOORW.bcs BH-REST
-move override\DUTRIGE.bcs BH-REST
-move override\DUTRIGW.bcs BH-REST
-move override\DYNAJ.dlg   BH-REST 2> Null
-move override\EDWINJ.dlg  BH-REST 2> Null
-move override\GARRIJ.dlg  BH-REST 2> Null
-move override\GIANTSTN.BAM BH-REST
-move override\HOBGO2.CRE   BH-REST
-move override\ibhghand.bam BH-REST
-move override\IBOULDER.BAM BH-REST
-move override\ICBCL039.BAM BH-REST
-move override\ICLCKBS.BAM  BH-REST
-move override\IKEYDUNG.BAM BH-REST
-move override\IMISC4F.bam  BH-REST
-move override\IMOEN2J.dlg  BH-REST 2> Null
-move override\INTERDIA.2DA BH-REST
-move override\IOSPREM.BAM  BH-REST
-move override\itemdial.2da BH-REST
-move override\JAHEIRAJ.dlg BH-REST 2> Null
-move override\JELLYCO.CRE  BH-REST
-move override\KAGAIJ.dlg   BH-REST 2> Null
-move override\KHALIJ.dlg   BH-REST 2> Null
-move override\KIVANJ.dlg   BH-REST 2> Null
-move override\mastarea.2da BH-REST
-move override\MINSCJ.dlg   BH-REST 2> Null
-move override\MLI3A1.BAM   BH-REST
-move override\MLI3A1E.BAM  BH-REST
-move override\MLI3A2.BAM   BH-REST
-move override\MLI3A2E.BAM  BH-REST
-move override\MLI3CA.BAM   BH-REST
-move override\MLI3CAE.BAM  BH-REST
-move override\MLI3DE.BAM   BH-REST
-move override\MLI3DEE.BAM  BH-REST
-move override\MLI3GH.BAM   BH-REST
-move override\MLI3GHE.BAM  BH-REST
-move override\MLI3GU.BAM   BH-REST
-move override\MLI3GUE.BAM  BH-REST
-move override\MLI3SC.BAM   BH-REST
-move override\MLI3SCE.BAM  BH-REST
-move override\MLI3SD.BAM   BH-REST
-move override\MLI3SDE.BAM  BH-REST
-move override\MLI3SL.BAM   BH-REST
-move override\MLI3SLE.BAM  BH-REST
-move override\MLI3SP.BAM   BH-REST
-move override\MLI3SPE.BAM  BH-REST
-move override\MLI3TW.BAM   BH-REST
-move override\MLI3TWE.BAM  BH-REST
-move override\MLI3WK.BAM   BH-REST
-move override\MLI3WKE.BAM  BH-REST
-move override\MLIZCA1.BAM  BH-REST
-move override\MLIZCA1E.BAM BH-REST
-move override\MLIZCA2.BAM  BH-REST
-move override\MLIZCA2E.BAM BH-REST
-move override\MLIZCCA.BAM  BH-REST
-move override\MLIZCCAE.BAM BH-REST
-move override\MLIZCGH.BAM  BH-REST
-move override\MLIZCGHE.BAM BH-REST
-move override\MLIZCGU.BAM  BH-REST
-move override\MLIZCGUE.BAM BH-REST
-move override\MLIZCSC.BAM  BH-REST
-move override\MLIZCSCE.BAM BH-REST
-move override\MLIZCSD.BAM  BH-REST
-move override\MLIZCSDE.BAM BH-REST
-move override\MLIZCSL.BAM  BH-REST
-move override\MLIZCSLE.BAM BH-REST
-move override\MLIZCSP.BAM  BH-REST
-move override\MLIZCSPE.BAM BH-REST
-move override\MLIZCWK.BAM  BH-REST
-move override\MLIZCWKE.BAM BH-REST
-move override\MLIZHA1.BAM  BH-REST
-move override\MLIZHA1E.BAM BH-REST
-move override\MLIZHA2.BAM  BH-REST
-move override\MLIZHA2E.BAM BH-REST
-move override\MLIZHCA.BAM  BH-REST
-move override\MLIZHCAE.BAM BH-REST
-move override\MLIZHGH.BAM  BH-REST
-move override\MLIZHGHE.BAM BH-REST
-move override\MLIZHGU.BAM  BH-REST
-move override\MLIZHGUE.BAM BH-REST
-move override\MLIZHSC.BAM  BH-REST
-move override\MLIZHSCE.BAM BH-REST
-move override\MLIZHSD.BAM  BH-REST
-move override\MLIZHSDE.BAM BH-REST
-move override\MLIZHSL.BAM  BH-REST
-move override\MLIZHSLE.BAM BH-REST
-move override\MLIZHSP.BAM  BH-REST
-move override\MLIZHSPE.BAM BH-REST
-move override\MLIZHWK.BAM  BH-REST
-move override\MLIZHWKE.BAM BH-REST
-move override\MLIZQA1.BAM  BH-REST
-move override\MLIZQA1E.BAM BH-REST
-move override\MLIZQA2.BAM  BH-REST
-move override\MLIZQA2E.BAM BH-REST
-move override\MLIZQCA.BAM  BH-REST
-move override\MLIZQCAE.BAM BH-REST
-move override\MLIZQGH.BAM  BH-REST
-move override\MLIZQGHE.BAM BH-REST
-move override\MLIZQGU.BAM  BH-REST
-move override\MLIZQGUE.BAM BH-REST
-move override\MLIZQSC.BAM  BH-REST
-move override\MLIZQSCE.BAM BH-REST
-move override\MLIZQSD.BAM  BH-REST
-move override\MLIZQSDE.BAM BH-REST
-move override\MLIZQSL.BAM  BH-REST
-move override\MLIZQSLE.BAM BH-REST
-move override\MLIZQSP.BAM  BH-REST
-move override\MLIZQSPE.BAM BH-REST
-move override\MLIZQWK.BAM  BH-REST
-move override\MLIZQWKE.BAM BH-REST
-move override\MONTAJ.dlg   BH-REST 2> Null
-move override\NETRIG1.bcs  BH-REST
-move override\NPCLEVEL.2DA BH-REST
-move override\NWTRIG1.bcs  BH-REST
-move override\PDIALOG.2DA  BH-REST
-move override\PROJECTL.IDS BH-REST
-move override\SAFANJ.dlg   BH-REST 2> Null
-move override\SETRIG1.bcs  BH-REST
-move override\SHARTJ.dlg   BH-REST 2> Null
-move override\SWTRIG1.bcs  BH-REST
-move override\TOOLTIP.2DA  BH-REST
-move override\XANJ.dlg     BH-REST 2> Null
-move override\xhlm015.bam  BH-REST
-move override\XZARJ.dlg    BH-REST 2> Null
-
-move override\GNOLL02.CRE    BH-REST
-move override\GNOLL03.CRE    BH-REST
-move override\GNOLL3.CRE    BH-REST
-move override\SKELET02.CRE    BH-REST
-
-
-move override\FW3402.ARE   BH-REST 2> Null
-move override\FW3400.ARE   BH-REST 2> Null
-move override\_XZARJ.DLG   BH-REST 2> Null
-move override\_xANJ.DLG    BH-REST 2> Null
-move override\_SHARTJ.DLG  BH-REST 2> Null
-move override\_SAFANJ.DLG  BH-REST 2> Null
-move override\_MONTAJ.DLG  BH-REST 2> Null
-move override\_MINSCJ.DLG  BH-REST 2> Null
-move override\_KIVANJ.DLG  BH-REST 2> Null
-move override\_KHALIJ.DLG  BH-REST 2> Null
-move override\_KAGAIJ.DLG  BH-REST 2> Null
-move override\_JAHEIJ.DLG  BH-REST 2> Null
-move override\_IMOEN2.DLG  BH-REST 2> Null
-move override\_GARRIJ.DLG  BH-REST 2> Null
-move override\_EDWINJ.DLG  BH-REST 2> Null
-move override\_DYNAHJ.DLG  BH-REST 2> Null
-move override\_BRANWJ.DLG  BH-REST 2> Null
-move override\_BMINSC.DLG  BH-REST 2> Null
-move override\_BGARRI.DLG  BH-REST 2> Null
-move override\_AR3402.BCS  BH-REST 2> Null
-move override\_AR3400.BCS  BH-REST 2> Null
-move override\_AR2626.BCS  BH-REST 2> Null
-move override\_AR0800.BCS  BH-REST 2> Null
-move override\_AR0123.BCS  BH-REST 2> Null
+:: move override\BH*.* BH-REST
+:: move override\1003MS1.BAM BH-REST
+:: move override\1007MS1.BAM BH-REST
+:: move override\1008MS1.BAM BH-REST
+:: move override\25SPELL.dlg BH-REST
+:: move override\9101MS1.bam BH-REST
+:: move override\9101MS2.bam BH-REST
+:: move override\9105MS1.BAM BH-REST
+:: move override\9106MS1.BAM BH-REST
+:: move override\9108MS1.BAM BH-REST
+:: move override\9711MS1.BAM BH-REST
+:: move override\9714MS1.BAM BH-REST
+:: move override\9714MS2.BAM BH-REST
+:: move override\9714MS3.BAM BH-REST
+:: move override\9714MS4.BAM BH-REST
+:: move override\9717MS1.BAM BH-REST
+:: move override\9717MS2.BAM BH-REST
+:: move override\AR6526.BCS  BH-REST 2> Null
+:: move override\AR7223.bcs  BH-REST 2> Null
+:: move override\AR7800.bcs  BH-REST 2> Null
+:: move override\AR9300.are  BH-REST 2> Null
+:: move override\AR9300.bcs  BH-REST 2> Null
+:: move override\AR9302.bcs  BH-REST 2> Null
+:: move override\ARAMIS.BAM  BH-REST
+:: move override\BALDUR.BCS  BH-REST
+:: move override\BERSERKR.ITM BH-REST
+:: move override\BFABIO.dlg  BH-REST 2> Null
+:: move override\BGARRI.dlg  BH-REST 2> Null
+:: move override\BMINSC.dlg  BH-REST 2> Null
+:: move override\BOULDER.ITM BH-REST
+:: move override\BOULDER.PRO BH-REST
+:: move override\BRANWJ.dlg  BH-REST 2> Null
+:: move override\cbhghan1.bam BH-REST
+:: move override\CBHGHAND.BAM BH-REST
+:: move override\COSPREM.BAM BH-REST
+:: move override\DUDOORE.bcs BH-REST
+:: move override\DUDOORW.bcs BH-REST
+:: move override\DUTRIGE.bcs BH-REST
+:: move override\DUTRIGW.bcs BH-REST
+:: move override\DYNAJ.dlg   BH-REST 2> Null
+:: move override\EDWINJ.dlg  BH-REST 2> Null
+:: move override\GARRIJ.dlg  BH-REST 2> Null
+:: move override\GIANTSTN.BAM BH-REST
+:: move override\HOBGO2.CRE   BH-REST
+:: move override\ibhghand.bam BH-REST
+:: move override\IBOULDER.BAM BH-REST
+:: move override\ICBCL039.BAM BH-REST
+:: move override\ICLCKBS.BAM  BH-REST
+:: move override\IKEYDUNG.BAM BH-REST
+:: move override\IMISC4F.bam  BH-REST
+:: move override\IMOEN2J.dlg  BH-REST 2> Null
+:: move override\INTERDIA.2DA BH-REST
+:: move override\IOSPREM.BAM  BH-REST
+:: move override\itemdial.2da BH-REST
+:: move override\JAHEIRAJ.dlg BH-REST 2> Null
+:: move override\JELLYCO.CRE  BH-REST
+:: move override\KAGAIJ.dlg   BH-REST 2> Null
+:: move override\KHALIJ.dlg   BH-REST 2> Null
+:: move override\KIVANJ.dlg   BH-REST 2> Null
+:: move override\mastarea.2da BH-REST
+:: move override\MINSCJ.dlg   BH-REST 2> Null
+:: move override\MLI3A1.BAM   BH-REST
+:: move override\MLI3A1E.BAM  BH-REST
+:: move override\MLI3A2.BAM   BH-REST
+:: move override\MLI3A2E.BAM  BH-REST
+:: move override\MLI3CA.BAM   BH-REST
+:: move override\MLI3CAE.BAM  BH-REST
+:: move override\MLI3DE.BAM   BH-REST
+:: move override\MLI3DEE.BAM  BH-REST
+:: move override\MLI3GH.BAM   BH-REST
+:: move override\MLI3GHE.BAM  BH-REST
+:: move override\MLI3GU.BAM   BH-REST
+:: move override\MLI3GUE.BAM  BH-REST
+:: move override\MLI3SC.BAM   BH-REST
+:: move override\MLI3SCE.BAM  BH-REST
+:: move override\MLI3SD.BAM   BH-REST
+:: move override\MLI3SDE.BAM  BH-REST
+:: move override\MLI3SL.BAM   BH-REST
+:: move override\MLI3SLE.BAM  BH-REST
+:: move override\MLI3SP.BAM   BH-REST
+:: move override\MLI3SPE.BAM  BH-REST
+:: move override\MLI3TW.BAM   BH-REST
+:: move override\MLI3TWE.BAM  BH-REST
+:: move override\MLI3WK.BAM   BH-REST
+:: move override\MLI3WKE.BAM  BH-REST
+:: move override\MLIZCA1.BAM  BH-REST
+:: move override\MLIZCA1E.BAM BH-REST
+:: move override\MLIZCA2.BAM  BH-REST
+:: move override\MLIZCA2E.BAM BH-REST
+:: move override\MLIZCCA.BAM  BH-REST
+:: move override\MLIZCCAE.BAM BH-REST
+:: move override\MLIZCGH.BAM  BH-REST
+:: move override\MLIZCGHE.BAM BH-REST
+:: move override\MLIZCGU.BAM  BH-REST
+:: move override\MLIZCGUE.BAM BH-REST
+:: move override\MLIZCSC.BAM  BH-REST
+:: move override\MLIZCSCE.BAM BH-REST
+:: move override\MLIZCSD.BAM  BH-REST
+:: move override\MLIZCSDE.BAM BH-REST
+:: move override\MLIZCSL.BAM  BH-REST
+:: move override\MLIZCSLE.BAM BH-REST
+:: move override\MLIZCSP.BAM  BH-REST
+:: move override\MLIZCSPE.BAM BH-REST
+:: move override\MLIZCWK.BAM  BH-REST
+:: move override\MLIZCWKE.BAM BH-REST
+:: move override\MLIZHA1.BAM  BH-REST
+:: move override\MLIZHA1E.BAM BH-REST
+:: move override\MLIZHA2.BAM  BH-REST
+:: move override\MLIZHA2E.BAM BH-REST
+:: move override\MLIZHCA.BAM  BH-REST
+:: move override\MLIZHCAE.BAM BH-REST
+:: move override\MLIZHGH.BAM  BH-REST
+:: move override\MLIZHGHE.BAM BH-REST
+:: move override\MLIZHGU.BAM  BH-REST
+:: move override\MLIZHGUE.BAM BH-REST
+:: move override\MLIZHSC.BAM  BH-REST
+:: move override\MLIZHSCE.BAM BH-REST
+:: move override\MLIZHSD.BAM  BH-REST
+:: move override\MLIZHSDE.BAM BH-REST
+:: move override\MLIZHSL.BAM  BH-REST
+:: move override\MLIZHSLE.BAM BH-REST
+:: move override\MLIZHSP.BAM  BH-REST
+:: move override\MLIZHSPE.BAM BH-REST
+:: move override\MLIZHWK.BAM  BH-REST
+:: move override\MLIZHWKE.BAM BH-REST
+:: move override\MLIZQA1.BAM  BH-REST
+:: move override\MLIZQA1E.BAM BH-REST
+:: move override\MLIZQA2.BAM  BH-REST
+:: move override\MLIZQA2E.BAM BH-REST
+:: move override\MLIZQCA.BAM  BH-REST
+:: move override\MLIZQCAE.BAM BH-REST
+:: move override\MLIZQGH.BAM  BH-REST
+:: move override\MLIZQGHE.BAM BH-REST
+:: move override\MLIZQGU.BAM  BH-REST
+:: move override\MLIZQGUE.BAM BH-REST
+:: move override\MLIZQSC.BAM  BH-REST
+:: move override\MLIZQSCE.BAM BH-REST
+:: move override\MLIZQSD.BAM  BH-REST
+:: move override\MLIZQSDE.BAM BH-REST
+:: move override\MLIZQSL.BAM  BH-REST
+:: move override\MLIZQSLE.BAM BH-REST
+:: move override\MLIZQSP.BAM  BH-REST
+:: move override\MLIZQSPE.BAM BH-REST
+:: move override\MLIZQWK.BAM  BH-REST
+:: move override\MLIZQWKE.BAM BH-REST
+:: move override\MONTAJ.dlg   BH-REST 2> Null
+:: move override\NETRIG1.bcs  BH-REST
+:: move override\NPCLEVEL.2DA BH-REST
+:: move override\NWTRIG1.bcs  BH-REST
+:: move override\PDIALOG.2DA  BH-REST
+:: move override\PROJECTL.IDS BH-REST
+:: move override\SAFANJ.dlg   BH-REST 2> Null
+:: move override\SETRIG1.bcs  BH-REST
+:: move override\SHARTJ.dlg   BH-REST 2> Null
+:: move override\SWTRIG1.bcs  BH-REST
+:: move override\TOOLTIP.2DA  BH-REST
+:: move override\XANJ.dlg     BH-REST 2> Null
+:: move override\xhlm015.bam  BH-REST
+:: move override\XZARJ.dlg    BH-REST 2> Null
+
+:: move override\GNOLL02.CRE    BH-REST
+:: move override\GNOLL03.CRE    BH-REST
+:: move override\GNOLL3.CRE    BH-REST
+:: move override\SKELET02.CRE    BH-REST
+
+
+:: move override\FW3402.ARE   BH-REST 2> Null
+:: move override\FW3400.ARE   BH-REST 2> Null
+:: move override\_XZARJ.DLG   BH-REST 2> Null
+:: move override\_xANJ.DLG    BH-REST 2> Null
+:: move override\_SHARTJ.DLG  BH-REST 2> Null
+:: move override\_SAFANJ.DLG  BH-REST 2> Null
+:: move override\_MONTAJ.DLG  BH-REST 2> Null
+:: move override\_MINSCJ.DLG  BH-REST 2> Null
+:: move override\_KIVANJ.DLG  BH-REST 2> Null
+:: move override\_KHALIJ.DLG  BH-REST 2> Null
+:: move override\_KAGAIJ.DLG  BH-REST 2> Null
+:: move override\_JAHEIJ.DLG  BH-REST 2> Null
+:: move override\_IMOEN2.DLG  BH-REST 2> Null
+:: move override\_GARRIJ.DLG  BH-REST 2> Null
+:: move override\_EDWINJ.DLG  BH-REST 2> Null
+:: move override\_DYNAHJ.DLG  BH-REST 2> Null
+:: move override\_BRANWJ.DLG  BH-REST 2> Null
+:: move override\_BMINSC.DLG  BH-REST 2> Null
+:: move override\_BGARRI.DLG  BH-REST 2> Null
+:: move override\_AR3402.BCS  BH-REST 2> Null
+:: move override\_AR3400.BCS  BH-REST 2> Null
+:: move override\_AR2626.BCS  BH-REST 2> Null
+:: move override\_AR0800.BCS  BH-REST 2> Null
+:: move override\_AR0123.BCS  BH-REST 2> Null
 
 weidu --make-biff BH-TIS1
 weidu --make-biff BH-TIS2
@@ -264,3 +264,10 @@
 call BoneHillv275\deldir BH-TIS1
 call BoneHillv275\deldir BH-TIS2
 call BoneHillv275\deldir BH-REST
+
+FOR %%s IN (
+override\*.baf
+override\*.d
+override\*.ogg
+override\*.tiz
+) DO del %%s
