--- Bonehillv275\d\BHBALMOR.d	Thu Feb 25 14:19:26 2010
+++ C:\BWP Patchstudio\patched files\Bonehillv275\d\BHBALMOR.d	Sun Jan 24 18:46:52 2016
@@ -3,6 +3,7 @@
 IF ~IfValidForPartyDialogue("Fabio")
 NumberOfTimesTalkedTo(0)
 AreaCheck("BH2018")
+Global("BHBardFree","GLOBAL",0)
 ~ THEN BEGIN 0
   SAY @1
   IF ~~ THEN EXTERN ~BFABIO~ 12
@@ -37,6 +38,7 @@
 IF ~!IfValidForPartyDialogue("Fabio")
 NumberOfTimesTalkedTo(0)
 AreaCheck("BH2018")
+Global("BHBardFree","GLOBAL",0)
 ~ THEN BEGIN 6
   SAY @7
   IF ~~ THEN REPLY @8 GOTO 7
