--- Setup-BoneHillv275.tp2	Sun Oct 02 04:24:17 2011
+++ C:\BWP Patchstudio\patched files\Setup-BoneHillv275.tp2	Thu Oct 27 06:49:23 2011
@@ -1508,7 +1508,10 @@
     SAY 0x1370 @428 
     SAY 0x4054 @406
   COPY ~bonehillv275/BGT/BH2103.are~ ~override/BH2103.are~         
-    SAY 0x244 @429 
+    SAY 0x574 @429
+    READ_LONG 0xc0 "rest_off"
+    WRITE_SHORT  ("%rest_off%" + 0x98) 0
+  BUT_ONLY_IF_IT_CHANGES
   COPY ~bonehillv275/BGT/BH2200.are~ ~override/BH2200.are~         
     SAY 0x1344 @430 
   COPY ~bonehillv275/BGT/BH2201.are~ ~override/BH2201.are~         
