--- TyrisFlare\Dialogue\FinalSolarChoice.d	Tue Jan 20 19:23:00 2015
+++ C:\BWP Patchstudio\patched files\TyrisFlare\Dialogue\FinalSolarChoice.d	Sun Jul 19 09:57:31 2015
@@ -192,39 +192,57 @@
 // Romantic consequences of ascension
 
 EXTEND_TOP FINSOL01 29 30 31 #1 //FinalSolarConsequences#1.1
-  IF ~Global("G#TF.TyrisRomanceActive","GLOBAL",2)
+  IF ~InParty("G#Tyris")
+      !StateCheck("G#Tyris",CD_STATE_NOTVALID)
+      Global("G#TF.TyrisRomanceActive","GLOBAL",2)
       OR(2)
         Global("G#TF.DestinyAscendGood","LOCALS",2)
         Global("G#TF.DestinyAscendEvil","LOCALS",2)~ EXTERN G#TRS25J FinalSolarConsequences#1.4
-  IF ~Global("G#TF.TyrisRomanceActive","GLOBAL",2)
+  IF ~InParty("G#Tyris")
+      !StateCheck("G#Tyris",CD_STATE_NOTVALID)
+      Global("G#TF.TyrisRomanceActive","GLOBAL",2)
       Global("G#TF.DestinyMortal","LOCALS",2)~ EXTERN G#TRS25J FinalSolarConsequences#1.5
-  IF ~Global("G#TF.TyrisRomanceActive","GLOBAL",2)
+  IF ~InParty("G#Tyris")
+      !StateCheck("G#Tyris",CD_STATE_NOTVALID)
+      Global("G#TF.TyrisRomanceActive","GLOBAL",2)
       !Global("G#TF.DestinyMortal","LOCALS",2)
       !Global("G#TF.DestinyAscendGood","LOCALS",2)
       !Global("G#TF.DestinyAscendEvil","LOCALS",2)~ EXTERN G#TRS25J FinalSolarConsequences#1.6
 END
 
 EXTEND_TOP IMOEN25J 18 #1 //FinalSolarConsequences#1.2
-  IF ~Global("G#TF.TyrisRomanceActive","GLOBAL",2)
+  IF ~InParty("G#Tyris")
+      !StateCheck("G#Tyris",CD_STATE_NOTVALID)
+      Global("G#TF.TyrisRomanceActive","GLOBAL",2)
       OR(2)
         Global("G#TF.DestinyAscendGood","LOCALS",2)
         Global("G#TF.DestinyAscendEvil","LOCALS",2)~ EXTERN G#TRS25J FinalSolarConsequences#1.4
-  IF ~Global("G#TF.TyrisRomanceActive","GLOBAL",2)
+  IF ~InParty("G#Tyris")
+      !StateCheck("G#Tyris",CD_STATE_NOTVALID)
+      Global("G#TF.TyrisRomanceActive","GLOBAL",2)
       Global("G#TF.DestinyMortal","LOCALS",2)~ EXTERN G#TRS25J FinalSolarConsequences#1.5
-  IF ~Global("G#TF.TyrisRomanceActive","GLOBAL",2)
+  IF ~InParty("G#Tyris")
+      !StateCheck("G#Tyris",CD_STATE_NOTVALID)
+      Global("G#TF.TyrisRomanceActive","GLOBAL",2)
       !Global("G#TF.DestinyMortal","LOCALS",2)
       !Global("G#TF.DestinyAscendGood","LOCALS",2)
       !Global("G#TF.DestinyAscendEvil","LOCALS",2)~ EXTERN G#TRS25J FinalSolarConsequences#1.6
 END
 
 EXTEND_TOP SAREV25J 4 6 #1 //FinalSolarConsequences#1.3
-  IF ~Global("G#TF.TyrisRomanceActive","GLOBAL",2)
+  IF ~InParty("G#Tyris")
+      !StateCheck("G#Tyris",CD_STATE_NOTVALID)
+      Global("G#TF.TyrisRomanceActive","GLOBAL",2)
       OR(2)
         Global("G#TF.DestinyAscendGood","LOCALS",2)
         Global("G#TF.DestinyAscendEvil","LOCALS",2)~ EXTERN G#TRS25J FinalSolarConsequences#1.4
-  IF ~Global("G#TF.TyrisRomanceActive","GLOBAL",2)
+  IF ~InParty("G#Tyris")
+      !StateCheck("G#Tyris",CD_STATE_NOTVALID)
+      Global("G#TF.TyrisRomanceActive","GLOBAL",2)
       Global("G#TF.DestinyMortal","LOCALS",2)~ EXTERN G#TRS25J FinalSolarConsequences#1.5
-  IF ~Global("G#TF.TyrisRomanceActive","GLOBAL",2)
+  IF ~InParty("G#Tyris")
+      !StateCheck("G#Tyris",CD_STATE_NOTVALID)
+      Global("G#TF.TyrisRomanceActive","GLOBAL",2)
       !Global("G#TF.DestinyMortal","LOCALS",2)
       !Global("G#TF.DestinyAscendGood","LOCALS",2)
       !Global("G#TF.DestinyAscendEvil","LOCALS",2)~ EXTERN G#TRS25J FinalSolarConsequences#1.6
@@ -299,7 +317,9 @@
 // Romantic consequences of remaining mortal
 
 EXTEND_TOP FINSOL01 32 #1 //FinalSolarConsequences#2.1
-  IF ~Global("G#TF.TyrisRomanceActive","GLOBAL",2)
+  IF ~InParty("G#Tyris")
+      !StateCheck("G#Tyris",CD_STATE_NOTVALID)
+      Global("G#TF.TyrisRomanceActive","GLOBAL",2)
       OR(2)
         Global("G#TF.DestinyAscendGood","LOCALS",2)
         Global("G#TF.DestinyAscendEvil","LOCALS",2)~ EXTERN G#TRS25J FinalSolarConsequences#2.4
@@ -312,26 +332,38 @@
 END
 
 EXTEND_TOP IMOEN25J 20 #1 //FinalSolarConsequences#2.2
-  IF ~Global("G#TF.TyrisRomanceActive","GLOBAL",2)
+  IF ~InParty("G#Tyris")
+      !StateCheck("G#Tyris",CD_STATE_NOTVALID)
+      Global("G#TF.TyrisRomanceActive","GLOBAL",2)
       OR(2)
         Global("G#TF.DestinyAscendGood","LOCALS",2)
         Global("G#TF.DestinyAscendEvil","LOCALS",2)~ EXTERN G#TRS25J FinalSolarConsequences#2.4
-  IF ~Global("G#TF.TyrisRomanceActive","GLOBAL",2)
+  IF ~InParty("G#Tyris")
+      !StateCheck("G#Tyris",CD_STATE_NOTVALID)
+      Global("G#TF.TyrisRomanceActive","GLOBAL",2)
       Global("G#TF.DestinyMortal","LOCALS",2)~ EXTERN G#TRS25J FinalSolarConsequences#2.5
-  IF ~Global("G#TF.TyrisRomanceActive","GLOBAL",2)
+  IF ~InParty("G#Tyris")
+      !StateCheck("G#Tyris",CD_STATE_NOTVALID)
+      Global("G#TF.TyrisRomanceActive","GLOBAL",2)
       !Global("G#TF.DestinyMortal","LOCALS",2)
       !Global("G#TF.DestinyAscendGood","LOCALS",2)
       !Global("G#TF.DestinyAscendEvil","LOCALS",2)~ EXTERN G#TRS25J FinalSolarConsequences#2.6
 END
 
 EXTEND_TOP SAREV25J 5 7 #1 //FinalSolarConsequences#2.3
-  IF ~Global("G#TF.TyrisRomanceActive","GLOBAL",2)
+  IF ~InParty("G#Tyris")
+      !StateCheck("G#Tyris",CD_STATE_NOTVALID)
+      Global("G#TF.TyrisRomanceActive","GLOBAL",2)
       OR(2)
         Global("G#TF.DestinyAscendGood","LOCALS",2)
         Global("G#TF.DestinyAscendEvil","LOCALS",2)~ EXTERN G#TRS25J FinalSolarConsequences#2.4
-  IF ~Global("G#TF.TyrisRomanceActive","GLOBAL",2)
+  IF ~InParty("G#Tyris")
+      !StateCheck("G#Tyris",CD_STATE_NOTVALID)
+      Global("G#TF.TyrisRomanceActive","GLOBAL",2)
       Global("G#TF.DestinyMortal","LOCALS",2)~ EXTERN G#TRS25J FinalSolarConsequences#2.5
-  IF ~Global("G#TF.TyrisRomanceActive","GLOBAL",2)
+  IF ~InParty("G#Tyris")
+      !StateCheck("G#Tyris",CD_STATE_NOTVALID)
+      Global("G#TF.TyrisRomanceActive","GLOBAL",2)
       !Global("G#TF.DestinyMortal","LOCALS",2)
       !Global("G#TF.DestinyAscendGood","LOCALS",2)
       !Global("G#TF.DestinyAscendEvil","LOCALS",2)~ EXTERN G#TRS25J FinalSolarConsequences#2.6
