--- w_packmule/area_ext.baf	2013-11-16 03:14:34.000000000 -0500
+++ Fixpack/w_packmule/area_ext.baf	2016-11-26 02:07:18.492346200 -0500
@@ -3,11 +3,11 @@
   !Global("W_MWAIT","GLOBAL",3)
 //  !InActiveArea("W_MULE")
   !InActiveArea("W_MSUMA")  
-  InActiveArea(Protagonist)
+  InActiveArea(Player1)
 THEN
   RESPONSE #100
     SetGlobal("W_MuleMo","GLOBAL",1)
-    MoveGlobalObjectOffScreen("W_MSUMA",Protagonist)
+    MoveGlobalObjectOffScreen("W_MSUMA",Player1)
     ActionOverride("W_MSUMA",JumpToPoint([1.1]))
 END
 IF
