--- w_packmule/w_msum.baf	2013-11-16 03:14:34.000000000 -0500
+++ Fixpack/w_packmule/w_msum.baf	2016-11-26 02:07:18.631359600 -0500
@@ -4,8 +4,8 @@
 THEN
   RESPONSE #100
     ApplySpellRES("W_MFREE","W_MULE")
-    MoveGlobalObjectOffScreen("W_MULE",Protagonist)
-    ActionOverride("W_MULE",MoveToObject(Protagonist))
+    MoveGlobalObjectOffScreen("W_MULE",Player1)
+    ActionOverride("W_MULE",MoveToObject(Player1))
     DestroySelf()
 END
 
@@ -17,9 +17,9 @@
   RESPONSE #100
     Wait(2)
 //    SetGlobal("W_MuleMo","GLOBAL",4)
-    MoveGlobalObjectOffScreen("W_MULE",Protagonist)
+    MoveGlobalObjectOffScreen("W_MULE",Player1)
     ApplySpellRES("W_MFREE","W_MULE")
-    ActionOverride("W_MULE",MoveToObject(Protagonist))
+    ActionOverride("W_MULE",MoveToObject(Player1))
     DestroySelf()
 END
 
@@ -31,7 +31,7 @@
   RESPONSE #100
     Wait(2)
     SetGlobal("W_MuleMo","GLOBAL",4)
-    MoveGlobalObject("W_MULE",Protagonist)
+    MoveGlobalObject("W_MULE",Player1)
     ApplySpellRES("W_MFREE","W_MULE")
     ActionOverride("W_MULE",MoveToOffset([2.3]))
     DestroySelf()
@@ -42,6 +42,6 @@
   True()
 THEN
   RESPONSE #100
-    ActionOverride("W_MULE",MoveToObject(Protagonist))
+    ActionOverride("W_MULE",MoveToObject(Player1))
     DestroySelf()
 END
