--- w_packmule/w_mule.baf	2013-12-03 17:06:12.000000000 -0500
+++ Fixpack/w_packmule/w_mule.baf	2016-11-26 02:07:18.770538500 -0500
@@ -84,8 +84,8 @@
 
 IF
   Or(2)
-    CheckStat(Protagonist,1,185)
-    HasItemEquiped("BOOT01",Protagonist)
+    CheckStat(Player1,1,185)
+    HasItemEquiped("BOOT01",Player1)
   Global("W_MSPEED","LOCALS",0)
 THEN
   RESPONSE #100
@@ -94,8 +94,8 @@
 END
 
 IF
-  CheckStat(Protagonist,0,185)
-  !HasItemEquiped("BOOT01",Protagonist)
+  CheckStat(Player1,0,185)
+  !HasItemEquiped("BOOT01",Player1)
   Global("W_MSPEED","LOCALS",1)
 THEN
   RESPONSE #100
