--- planarspheremod\baf\psbaatl.baf	Sun May 28 15:26:04 2006
+++ C:\BWP Patchstudio\patched files\planarspheremod\baf\psbaatl.baf	Sat Jun 23 18:13:24 2012
@@ -4,6 +4,7 @@
 	See(NearestEnemyOf(Myself))
 Then
 	Response #100
+		ReallyForceSpell(Myself, CLERIC_HEAL)
 		IncrementGlobal("psBaatSummonedCounter", "LOCALS", 1)
 		CreateCreatureObject("psBaat01", NearestEnemyOf(Myself), 0, 0, 0)
 		CreateCreatureObject("psBaat02", NearestEnemyOf(Myself), 0, 0, 0)
