--- planarspheremod\baf\psglobe1.baf	Mon Jun 12 16:06:12 2006
+++ C:\BWP Patchstudio\patched files\planarspheremod\baf\psglobe1.baf	Sat Jun 23 04:40:15 2012
@@ -1,4 +1,5 @@
 IF
+	!GlobalTimerNotExpired("Globe1Timer","LOCALS")
 	Global("psGlobeSpellCounter", "LOCALS", 8)
 THEN
 	RESPONSE #100
@@ -7,9 +8,11 @@
 END
 
 IF
-	RandomNum(5, 1)
+	!GlobalTimerNotExpired("Globe1Timer","LOCALS")
+	RandomNumLT(43, 7)
 THEN
 	RESPONSE #100
+		SetGlobalTimer("Globe1Timer","LOCALS",6)
 		IncrementGlobal("psGlobeSpellCounter", "LOCALS", 1)
 		Wait(4)
 		ReallyForceSpell(Player1,WIZARD_SKULL_TRAP)
@@ -21,9 +24,12 @@
 END
 
 IF
-	RandomNum(5, 1)
+	!GlobalTimerNotExpired("Globe1Timer","LOCALS")
+	RandomNumGT(43, 6)
+	RandomNumLT(43, 13)
 THEN
 	RESPONSE #100
+		SetGlobalTimer("Globe1Timer","LOCALS",6)
 		IncrementGlobal("psGlobeSpellCounter", "LOCALS", 1)
 		Wait(1)
 		ReallyForceSpell(Player1,WIZARD_AGANNAZAR_SCORCHER)
@@ -35,9 +41,12 @@
 END
 
 IF
-	RandomNum(5, 1)
+	!GlobalTimerNotExpired("Globe1Timer","LOCALS")
+	RandomNumGT(43, 12)
+	RandomNumLT(43, 19)
 THEN
 	RESPONSE #100
+		SetGlobalTimer("Globe1Timer","LOCALS",6)
 		IncrementGlobal("psGlobeSpellCounter", "LOCALS", 1)
 		Wait(1)
 		ReallyForceSpell(Player1,WIZARD_MELF_ACID_ARROW)
@@ -49,9 +58,12 @@
 END
 
 IF
-	RandomNum(5, 1)
+	!GlobalTimerNotExpired("Globe1Timer","LOCALS")
+	RandomNumGT(43, 18)
+	RandomNumLT(43, 25)
 THEN
 	RESPONSE #100
+		SetGlobalTimer("Globe1Timer","LOCALS",6)
 		IncrementGlobal("psGlobeSpellCounter", "LOCALS", 1)
 		Wait(3)
 		ReallyForceSpell(Player1,WIZARD_LIGHTNING_BOLT)
@@ -63,9 +75,12 @@
 END
 
 IF
-	RandomNum(3, 1)
+	!GlobalTimerNotExpired("Globe1Timer","LOCALS")
+	RandomNumGT(43, 24)
+	RandomNumLT(43, 35)
 THEN
 	RESPONSE #100
+		SetGlobalTimer("Globe1Timer","LOCALS",6)
 		IncrementGlobal("psGlobeSpellCounter", "LOCALS", 1)
 		Wait(2)
 		ReallyForceSpell(Player1,WIZARD_FLAME_ARROW)
@@ -78,9 +93,12 @@
 
 
 IF
-	RandomNum(10, 1)
+	!GlobalTimerNotExpired("Globe1Timer","LOCALS")
+	RandomNumGT(43, 34)
+	RandomNumLT(43, 38)
 THEN
 	RESPONSE #100
+		SetGlobalTimer("Globe1Timer","LOCALS",6)
 		IncrementGlobal("psGlobeSpellCounter", "LOCALS", 1)
 		Wait(5)
 		ReallyForceSpell(Player1,WIZARD_FIREBALL)
@@ -92,9 +110,11 @@
 END
 
 IF
-	RandomNum(5, 1)
+	!GlobalTimerNotExpired("Globe1Timer","LOCALS")
+	RandomNumGT(43, 37)
 THEN
 	RESPONSE #100
+		SetGlobalTimer("Globe1Timer","LOCALS",6)
 		IncrementGlobal("psGlobeSpellCounter", "LOCALS", 1)
 		ReallyForceSpell(Player1,WIZARD_MAGIC_MISSILE)
 		ReallyForceSpell(Player2,WIZARD_MAGIC_MISSILE)
