--- planarspheremod\baf\psglobe2.baf	Mon Jun 12 16:06:06 2006
+++ C:\BWP Patchstudio\patched files\planarspheremod\baf\psglobe2.baf	Sat Jun 23 04:40:47 2012
@@ -1,4 +1,5 @@
 IF
+	!GlobalTimerNotExpired("Globe2Timer","LOCALS")
 	Global("psGlobeSpellCounter", "LOCALS", 8)
 THEN
 	RESPONSE #100
@@ -7,9 +8,11 @@
 END
 
 IF
-	RandomNum(5, 1)
+	!GlobalTimerNotExpired("Globe2Timer","LOCALS")
+	RandomNum(6, 1)
 THEN
 	RESPONSE #100
+		SetGlobalTimer("Globe2Timer","LOCALS",6)
 		IncrementGlobal("psGlobeSpellCounter", "LOCALS", 1)
 		Wait(1)
 		ReallyForceSpell(Player1,WIZARD_CONFUSION)
@@ -21,9 +24,11 @@
 END
 
 IF
-	RandomNum(5, 1)
+	!GlobalTimerNotExpired("Globe2Timer","LOCALS")
+	RandomNum(6, 2)
 THEN
 	RESPONSE #100
+		SetGlobalTimer("Globe2Timer","LOCALS",6)
 		IncrementGlobal("psGlobeSpellCounter", "LOCALS", 1)
 		Wait(2)
 		ReallyForceSpell(Player1,WIZARD_GREATER_MALISON)
@@ -35,9 +40,11 @@
 END
 
 IF
-	RandomNum(5, 1)
+	!GlobalTimerNotExpired("Globe2Timer","LOCALS")
+	RandomNum(6, 3)
 THEN
 	RESPONSE #100
+		SetGlobalTimer("Globe2Timer","LOCALS",6)
 		IncrementGlobal("psGlobeSpellCounter", "LOCALS", 1)
 		Wait(2)
 		ReallyForceSpell(Player1,WIZARD_HOLD_PERSON)
@@ -49,9 +56,11 @@
 END
 
 IF
-	RandomNum(5, 1)
+	!GlobalTimerNotExpired("Globe2Timer","LOCALS")
+	RandomNum(6, 4)
 THEN
 	RESPONSE #100
+		SetGlobalTimer("Globe2Timer","LOCALS",6)
 		IncrementGlobal("psGlobeSpellCounter", "LOCALS", 1)
 		Wait(2)
 		ReallyForceSpell(Player1,WIZARD_CONTAGION)
@@ -63,9 +72,11 @@
 END
 
 IF
-	RandomNum(5, 1)
+	!GlobalTimerNotExpired("Globe2Timer","LOCALS")
+	RandomNum(6, 5)
 THEN
 	RESPONSE #100
+		SetGlobalTimer("Globe2Timer","LOCALS",6)
 		IncrementGlobal("psGlobeSpellCounter", "LOCALS", 1)
 		Wait(1)
 		ReallyForceSpell(Player1,WIZARD_SLOW)
@@ -78,9 +89,11 @@
 
 
 IF
-	RandomNum(5, 1)
+	!GlobalTimerNotExpired("Globe2Timer","LOCALS")
+	RandomNum(6, 6)
 THEN
 	RESPONSE #100
+		SetGlobalTimer("Globe2Timer","LOCALS",6)
 		IncrementGlobal("psGlobeSpellCounter", "LOCALS", 1)
 		Wait(1)
 		ReallyForceSpell(Player1,WIZARD_STINKING_CLOUD)
