--- setup-planarspheremod.tp2	Sun May 01 14:04:17 2011
+++ C:\BWP Patchstudio\patched files\setup-planarspheremod.tp2	Sun Feb 10 10:35:37 2013
@@ -8,7 +8,7 @@
 //MODDER
 
 /* WeiDU v204 allows tp2 level version listing in the .log, so no more tra changes  */
-VERSION ~v2.6c~
+VERSION ~v2.6e BWP Fix~
 
 /* launch the readme file immediately. If you want to disable the ReadMe, place two slashes before it, like //README  */
 README ~PlanarSphereMod/readme-planarsphere.html~
@@ -1021,7 +1021,7 @@
     SAY NAME2 @115
     SAY UNIDENTIFIED_DESC @116
     SAY DESC @117
-    SAY 0x1ce @118
+//  SAY 0x1ce @118
 
 COPY ~PlanarSphereMod/itm/psBolt02.itm~ ~override/psBolt02.itm~
     SAY NAME1 @114
