--- setup-planarspheremod.tp2	Tue Feb 02 17:01:34 2016
+++ setup-planarspheremod.tp2	Tue Feb 02 17:03:38 2016
@@ -5,44 +5,64 @@
 AUTHOR ~See_The_Readme (Org: Inferno & M, Updates: Hlid, Duality, Malbolgia)~
 
 /* enable all error messages; nothing suppressed. comment this out for release version - cmorgan */
 //MODDER
 
 /* WeiDU v204 allows tp2 level version listing in the .log, so no more tra changes  */
-VERSION ~v2.6c~
+VERSION ~v2.6e BWP Fix 3~
 
 /* launch the readme file immediately. If you want to disable the ReadMe, place two slashes before it, like //README  */
 README ~PlanarSphereMod/readme-planarsphere.html~
+
+/* Ineth - make sure strings are correctly encoded for both EE and non-EE games */
+ALWAYS
+  ACTION_IF "%WEIDU_OS%" STRING_EQUAL_CASE ~win32~ THEN BEGIN
+    AT_NOW ~if not exist "iconv" mkdir iconv~
+    AT_NOW ~if not exist "iconv\iconv.exe" copy "BiG World Fixpack\_utils\iconv" "iconv"~
+  END
+  ACTION_DEFINE_ARRAY tra_reload BEGIN setup END
+  LAF HANDLE_CHARSETS
+    INT_VAR
+      infer_charset = 1
+    STR_VAR
+      tra_path      = EVAL ~%MOD_FOLDER%~
+      iconv_path    = ~iconv~
+      reload_array  = tra_reload
+  END
+END
 
 AUTO_TRA ~PlanarSphereMod/%s~
 
 LANGUAGE ~English~
 ~english~
 //~PlanarSphereMod/english~
 ~PlanarSphereMod/english/setup.tra~
 
 LANGUAGE ~Russian~
 ~russian~
 //~PlanarSphereMod/russian~
 ~PlanarSphereMod/russian/setup.tra~
 
 BEGIN @0
 
+/* BWP Fix - this section will only match if Trimpack was not applied first
 COPY + ~Setup-PlanarSphereMod.exe~ ~WeiDU.exe~
 
 COPY + ~chitin.key~ ~PlanarSphereMod/chitin.psm~
 COPY_LARGE + ~dialog.tlk~ ~PlanarSphereMod/dialog.psm~
 MKDIR ~PlanarSphereMod/over.bak~
 COPY_LARGE + ~override~ ~PlanarSphereMod/over.bak~
+*/
 
 
 // TOB Check prior to compiling.
-REQUIRE_FILE ~Data/25Dialog.bif~ @1 // ToB  check// Base Copy Overs
+REQUIRE_PREDICATE GAME_INCLUDES ~TOB~ @1 // ToB check
 
 
+// Base Copy Overs
 COPY ~PlanarSphereMod/are~   ~override~ // Area Files (new area files not previously installed in game)
      ~PlanarSphereMod/Pics~   ~override~ // Mos and Bmp files including additional search regions in AR0412SR.bmp
      ~PlanarSphereMod/eff~   ~override~ // Eff Files
      ~PlanarSphereMod/misc/SPPSQ22.2da~   ~override/SPPSQ22.2da~ // Non-.pro file copy
      ~PlanarSphereMod/bam~   ~override~ // Bam files
 
 ACTION_IF NOT FILE_EXISTS_IN_GAME	~MSPLG1.BAM~	THEN BEGIN
