--- setup-planarspheremod.tp2	Sun May 01 17:34:17 2011
+++ C:\BWP Patchstudio\patched files\setup-planarspheremod.tp2	Wed Jan 13 23:14:13 2016
@@ -8,11 +8,24 @@
 //MODDER
 
 /* WeiDU v204 allows tp2 level version listing in the .log, so no more tra changes  */
-VERSION ~v2.6c~
+VERSION ~v2.6e BWP Fix~
 
 /* launch the readme file immediately. If you want to disable the ReadMe, place two slashes before it, like //README  */
 README ~PlanarSphereMod/readme-planarsphere.html~
 
+/* Ineth - make sure strings are correctly encoded for both EE and non-EE games */
+ALWAYS
+  ACTION_DEFINE_ARRAY tra_reload BEGIN setup END
+  LAF HANDLE_CHARSETS
+    INT_VAR
+      infer_charset = 1
+    STR_VAR
+      tra_path      = EVAL ~%MOD_FOLDER%~
+      iconv_path    = EVAL ~%MOD_FOLDER%/tools/iconv~
+      reload_array  = tra_reload
+  END
+END
+
 AUTO_TRA ~PlanarSphereMod/%s~
 
 LANGUAGE ~English~
@@ -36,7 +48,7 @@
 
 
 // TOB Check prior to compiling.
-REQUIRE_FILE ~Data/25Dialog.bif~ @1 // ToB  check// Base Copy Overs
+REQUIRE_PREDICATE GAME_INCLUDES ~tob~ @1 // ToB  check// Base Copy Overs
 
 
 COPY ~PlanarSphereMod/are~   ~override~ // Area Files (new area files not previously installed in game)
@@ -1021,7 +1033,7 @@
     SAY NAME2 @115
     SAY UNIDENTIFIED_DESC @116
     SAY DESC @117
-    SAY 0x1ce @118
+//  SAY 0x1ce @118
 
 COPY ~PlanarSphereMod/itm/psBolt02.itm~ ~override/psBolt02.itm~
     SAY NAME1 @114
