--- Skie/DIALOGUES/1xSkieJ.d	2012-08-02 00:44:02.000000000 -0400
+++ Fixpack/Skie/DIALOGUES/1xSkieJ.d	2016-11-21 21:53:43.686935200 -0500
@@ -1483,7 +1483,7 @@
 
 ///////////////////////////////////////23
 
-IF~Global("CHAPTER","GLOBAL",6) Global("SkieRomanceActive","GLOBAL",2) Global("SkieRomanceC6","GLOBAL",1) RealGlobalTimerExpired("SkieTimerC6","GLOBAL")~THEN BEGIN Skie23x00
+IF~Global("CHAPTER","GLOBAL",%bg2_chapter_6%) Global("SkieRomanceActive","GLOBAL",2) Global("SkieRomanceC6","GLOBAL",1) RealGlobalTimerExpired("SkieTimerC6","GLOBAL")~THEN BEGIN Skie23x00
 SAY @535 /* Czy nie zdaje ci siê, ¿e ostatnio przyty³am, <CHARNAME>? Ale ja nie jad³am jakoœ du¿o... Dziwne... A w ogóle to mam ochotê na szparagi... Nie wiesz czy nie mogê gdzieœ niedaleko kupiæ kilka...? */
 IF~~THEN REPLY @536 /* Skie, czy ty... ostatnio nie czu³aœ siê niedobrze z rana...? */ GOTO Skie23x01
 IF~~THEN REPLY @537 /* Nawet tak nie ¿artuj... Czy ty... Czy ty chcesz powiedzieæ, ¿e... */ GOTO Skie23x01
