--- Skie/Setup-Skie.tp2	2015-02-02 00:15:14.000000000 -0500
+++ Fixpack/Skie/Setup-Skie.tp2	2016-11-26 23:38:49.935104100 -0500
@@ -1,25 +1,31 @@
 BACKUP ~Skie\backup~
 AUTHOR ~Lava Del'Vortel~
-
+VERSION ~3.0 BWP Fix~
 AUTO_TRA ~Skie/translations/%s~
 
 ALWAYS
-  // Borrowed from Edwin Romance: convert strings to UTF-8 for BGEE/BG2EE
-  INCLUDE ~%MOD_FOLDER%/handle_charsets.tpa~
+  ACTION_IF GAME_IS ~eet~ BEGIN
+    OUTER_SET bg2_chapter = 12
+  END ELSE BEGIN
+    OUTER_SET bg2_chapter = 0
+  END
+  OUTER_FOR (i=1; i<=10; i=i+1) BEGIN
+    OUTER_SET bg2_chapter = bg2_chapter + 1
+    OUTER_SPRINT name_source ~bg2_chapter_%i%~
+    OUTER_SET EVAL ~%name_source%~ = bg2_chapter
+  END
 
   // No exception, everything needs to be converted
   ACTION_DEFINE_ARRAY 1xSnoconvert BEGIN END
 
-  // Only setup-Innershade.tra needs to be reloaded
-  ACTION_DEFINE_ARRAY 1xSreload BEGIN setup-Skie END
+  // Only setup-skie.tra needs to be reloaded
+  ACTION_DEFINE_ARRAY 1xSreload BEGIN setup-skie END
 
-  // There is no "language" folder so the root of all languages in %MOD_FOLDER%
   LAF HANDLE_CHARSETS
     INT_VAR
-      infer_charset = 1
+      infer_charsets = 1
     STR_VAR
       tra_path = EVAL ~%MOD_FOLDER%/translations~
-      // charset_table = 1xScharsets // Included for illustrative purposes.
       noconvert_array = 1xSnoconvert
       reload_array = 1xSreload
   END
@@ -31,9 +37,12 @@
 BEGIN @0 /* Skie NPC - BG2 */ 
 
 // Fixing Imoen's dialogue file
-ACTION_IF FILE_EXISTS_IN_GAME ~mel01.cre~ THEN BEGIN
+ACTION_IF GAME_INCLUDES ~tob~ THEN BEGIN
   COPY_EXISTING ~interdia.2da~ ~override~
+    READ_2DA_ENTRY 17 0 2 scriptname
+    PATCH_IF (~%scriptname%~ STR_EQ ~IMOEN~) THEN BEGIN
     SET_2DA_ENTRY 17 1 2 ~BIMOEN2~ // fixes ToB version
+    END
     BUT_ONLY_IF_IT_CHANGES
 END ELSE BEGIN
   APPEND ~interdia.2da~ ~IMOEN       BIMOEN2~ // fixes SoA version
@@ -45,11 +54,11 @@
 
 ACTION_IF ("%LANGUAGE%" STRING_COMPARE_CASE "English" =0) BEGIN
  
-COPY ~Skie\AUDIOEng~ ~override\~
+  LAF HANDLE_AUDIO STR_VAR audio_path = ~Skie/AudioEng~ oggdec_path = ~Skie/Music~ END // convert ogg to wav
 
    END ELSE ACTION_IF  ("%LANGUAGE%" STRING_COMPARE_CASE "English" =0) BEGIN
 
-COPY ~Skie\AUDIO~ ~override\~
+  COPY ~Skie\AUDIO~ ~override~ // wav files
 
 END
 
@@ -115,7 +124,7 @@
 COMPILE ~Skie\DIALOGUES\1xSPR.d~
 COMPILE ~Skie\DIALOGUES\1xSkie.d~
 COMPILE ~Skie\DIALOGUES\1xSkieB.d~
-COMPILE ~Skie\DIALOGUES\1xSkieJ.d~
+COMPILE ~Skie\DIALOGUES\1xSkieJ.d~ EVALUATE_BUFFER
 COMPILE ~Skie\DIALOGUES\1xSkii1.d~
 COMPILE ~Skie\DIALOGUES\1xSkii2.d~
 COMPILE ~Skie\DIALOGUES\1xSkieVa.d~
@@ -124,7 +133,7 @@
 
 COMPILE ~Skie\BAF\1xSkii2.baf~
 COMPILE ~Skie\BAF\1xSkii1.baf~
-COMPILE ~Skie\BAF\1xSkie.baf~
+COMPILE ~Skie\BAF\1xSkie.baf~ EVALUATE_BUFFER
 EXTEND_TOP ~Aerie.bcs~ ~Skie\BAF\Aerie.baf~
 EXTEND_TOP ~Jaheira.bcs~ ~Skie\BAF\Jaheira.baf~
 EXTEND_TOP ~Viconia.bcs~ ~Skie\BAF\Viconia.baf~
@@ -294,13 +303,29 @@
 COPY ~Skie\MUSIC\SkieBlank.mus~ ~music/SkieBlank.mus~
 
 COPY ~Skie\MUSIC\SKIER.ogg~ ~override/SKIER.ogg~
-COPY ~Skie\MUSIC\oggdec.exe~ ~override/oggdec.exe~
 
-COPY ~Skie\SKIEMUSICI.bat~ ~SKIEMUSICI.bat~
-COPY ~Skie\SKIEMUSICU.bat~ ~SKIEMUSICU.bat~
+LAF HANDLE_AUDIO STR_VAR audio_path = ~Skie/MUSIC~ END
+
+ACTION_IF GAME_IS ~eet~ BEGIN
+  INCLUDE ~EET/other/EET_functions.tph~
+  LAF ~EET_NPC_TRANSITION~
+    INT_VAR
+      type = 2
+      clean_ToB = 1
+    STR_VAR
+      dv = "SKIEDV"
+      override_SoA = "1XSKIE"
+      override_ToB = "1XSKI25"
+      dialog_ToB = "1XSKI25"
+      traFile = EVAL "Skie/Translations/%LANGUAGE%/1XSKI25J.TRA"
+      string = "@132" // Bring Skie, the constantly-complaining thief.
+      stringPosDV = "Tiax"
+      clean_ToB_DV = "Skie"
+  END
+END ELSE BEGIN
+  COMPILE ~Skie/DIALOGUES/FATESP.d~ USING ~Skie/Translations/%LANGUAGE%/1XSKI25J.TRA~
+END
 
-AT_INTERACTIVE_EXIT ~SKIEMUSICI.bat~
-AT_UNINSTALL ~SKIEMUSICU.bat~
 
 
 
@@ -373,23 +398,21 @@
   SAY SELECT_RARE1 @27 /* Dlaczego zawsze siê ze mnie wyœmiewasz?! Nie cierpiê ciê! IdŸ precz! */ 
   SAY SELECT_RARE2 @28 /* Przykro mi, czy¿bym zrobi³a coœ z³ego? */ 
 
-/*
 
-BEGIN @50 /* Alternatywny portret Skie wykonany rêk¹ Plasmocat */
-SUBCOMPONENT @51
-REQUIRE_COMPONENT ~Setup-Skie.tp2~ ~0~ @49 /* Wymaga komponentu g³ównego */
-
-COPY ~Skie\PORTRAIT\ALTER\1xSkieG.bmp~ ~override\1xSkieG.bmp~
-COPY ~Skie\PORTRAIT\ALTER\1xSkieM.bmp~ ~override\1xSkieM.bmp~
-COPY ~Skie\PORTRAIT\ALTER\1xSkieS.bmp~ ~override\1xSkieS.bmp~
+// BEGIN @50 /* Alternatywny portret Skie wykonany rêk¹ Plasmocat */
+// SUBCOMPONENT @51
+// REQUIRE_COMPONENT ~Setup-Skie.tp2~ ~0~ @49 /* Wymaga komponentu g³ównego */
+
+// COPY ~Skie\PORTRAIT\ALTER\1xSkieG.bmp~ ~override\1xSkieG.bmp~
+// COPY ~Skie\PORTRAIT\ALTER\1xSkieM.bmp~ ~override\1xSkieM.bmp~
+// COPY ~Skie\PORTRAIT\ALTER\1xSkieS.bmp~ ~override\1xSkieS.bmp~
+
+
+// BEGIN @52
+// SUBCOMPONENT @51
+// REQUIRE_COMPONENT ~Setup-Skie.tp2~ ~0~ @49 /* Wymaga komponentu g³ównego */
+
+// COPY ~Skie\PORTRAIT\BrandNew\1xSkieG.bmp~ ~override\1xSkieG.bmp~
+// COPY ~Skie\PORTRAIT\BrandNew\1xSkieM.bmp~ ~override\1xSkieM.bmp~
+// COPY ~Skie\PORTRAIT\BrandNew\1xSkieS.bmp~ ~override\1xSkieS.bmp~
 
-
-BEGIN @52
-SUBCOMPONENT @51
-REQUIRE_COMPONENT ~Setup-Skie.tp2~ ~0~ @49 /* Wymaga komponentu g³ównego */
-
-COPY ~Skie\PORTRAIT\BrandNew\1xSkieG.bmp~ ~override\1xSkieG.bmp~
-COPY ~Skie\PORTRAIT\BrandNew\1xSkieM.bmp~ ~override\1xSkieM.bmp~
-COPY ~Skie\PORTRAIT\BrandNew\1xSkieS.bmp~ ~override\1xSkieS.bmp~
-
-*/
