--- setup-refinements.tp2	Tue Jan 03 00:19:55 2012
+++ C:\BWP Patchstudio\patched files\setup-refinements.tp2	Mon Dec 09 22:00:34 2013
@@ -16,7 +16,7 @@
 AUTO_TRA ~refinements/%s~
 
 README ~Refinements/README-Refinements.htm~
-VERSION ~Refinements v 3.31~
+VERSION ~Refinements v 3.31 BWP Fix~
 
 
 ALWAYS
@@ -248,8 +248,8 @@
 			  ~imoen211.cre~ ~override~
 			  ~imoen213.cre~ ~override~
 	WRITE_SHORT   0x244 0    //needed to change the the kit
-	WRITE_BYTE    0x246 12   //swashbuckler
-	WRITE_BYTE    0x247 0x40 //needed to change the kit
+	WRITE_SHORT   0x246 0x4012    //swashbuckler
+//	WRITE_BYTE    0x247 0x40 //needed to change the kit
 
 
 
