--- song_and_silence\setup-song_and_silence.tp2	Thu Sep 25 22:18:56 2014
+++ C:\BWP Patchstudio\patched files\song_and_silence\setup-song_and_silence.tp2	Tue Oct 21 18:09:27 2014
@@ -1,7 +1,7 @@
 BACKUP ~song_and_silence/backup~
 AUTHOR ~andyr@gibberlings3.net~
 
-VERSION ~v6~
+VERSION ~v6 BWP Fix~
 README ~song_and_silence/readme-song_and_silence.html~
 
 ALWAYS
@@ -760,14 +760,14 @@
       PATCH_IF (("%dv%" STRING_MATCHES_REGEXP ~^imoen2?$~) == 0) && (MOD_IS_INSTALLED npckit.tp2 1400) BEGIN
         PATCH_PRINT @80
         WRITE_SHORT 0x244 0x0000
-        WRITE_BYTE  0x246 %A!ADVENTURER%
-        WRITE_BYTE  0x247 0x40
+        WRITE_SHORT 0x246 0x4000 + %A!ADVENTURER%
+//      WRITE_BYTE  0x247 0x40
       END ELSE 
 	PATCH_IF (("%dv%" STRING_MATCHES_REGEXP ~^nalia$~) == 0) && (MOD_IS_INSTALLED npckit.tp2 2500) BEGIN
         PATCH_PRINT @81
         WRITE_SHORT 0x244 0x0000
-        WRITE_BYTE  0x246 %A!ADVENTURER%
-        WRITE_BYTE  0x247 0x40
+        WRITE_SHORT 0x246 0x4000 + %A!ADVENTURER%
+//      WRITE_BYTE  0x247 0x40
       END
     END
   BUT_ONLY_IF_IT_CHANGES
