--- Setup-RevisedBattles.tp2	Sat Aug 23 00:00:58 2008
+++ C:\BWP Patchstudio\patched files\Setup-RevisedBattles.tp2	Sat Jan 09 10:57:36 2016
@@ -4,7 +4,7 @@
 
 BACKUP ~RevisedBattles/backup~
 AUTHOR ~erebusant@verizon.net~
-
+VERSION ~v6.3 BWP Fix~
 // Modified to Patch vs. Overwrite by erebusant based entirely upon the original ideas and concepts
 // of moonfruit & tyrus
 
@@ -1108,7 +1108,7 @@
 REQUIRE_COMPONENT ~Setup-RevisedBattles.TP2~ ~0~ ~Requires Improved Battles - "erebusant's Patching Version" v6~
 REQUIRE_COMPONENT ~Setup-NEJ2V691.TP2~ ~0~ ~Never Ending Journey Second Edition v6.91~
 
-COPY - ~WeiDU.log~ ~WeiDU.log~
+/* COPY - ~WeiDU.log~ ~WeiDU.log~
   READ_2DA_ENTRIES_NOW ~_#_#_#weidu~ 1
   SET good_install = 0
   SET row = 0
@@ -1134,7 +1134,7 @@
   END
 ACTION_IF (good_install = 0) BEGIN
   FAIL ~NeJ2 needs to be installed directly after BGT and/or BGTMusic.~
-END
+END */
 
 COMPILE ~RevisedBattles/firkraag/dlg/NEJv691 Compat~
 
@@ -1936,7 +1936,7 @@
   END
 BUT_ONLY_IF_IT_CHANGES
 
-ACTION_IF NOT FILE_EXISTS_IN_GAME ~deitm011~ THEN BEGIN
+ACTION_IF NOT FILE_EXISTS_IN_GAME ~deitm011.itm~ THEN BEGIN
 COPY_EXISTING ~gall.CRE~	~override~
   PATCH_IF (SOURCE_SIZE > 0x2d3) THEN BEGIN // protects against invalid files
     ADD_CRE_ITEM			~BOW17~	#0 #0 #0	~NONE~	~WEAPON1~	EQUIP
