--- setup-npcflirt.tp2	2013-11-15 11:56:00.000000000 -0500
+++ Fixpack/setup-npcflirt.tp2	2016-11-22 12:40:55.115058000 -0500
@@ -1,6 +1,8 @@
 BACKUP ~npcflirt\Backup~
 AUTHOR ~jcompton@forgottenwars.net~
-VERSION ~v1.04~
+
+VERSION ~v1.04 BWP Fix~
+
 README ~npcflirt/npcflirt-readme.txt~
 
 // ASK_EVERY_COMPONENT
@@ -50,7 +52,7 @@
 // Aerie TOB flirts
 
 BEGIN @3
-REQUIRE_PREDICATE GAME_IS ~tob bg2ee bgt~ @100
+REQUIRE_PREDICATE GAME_IS ~tob bg2ee eet bgt~ @100
 
 COMPILE ~npcflirt\aerietobflirt.d~
   USING ~npcflirt\english\aerietobflirt.tra~
@@ -71,7 +73,7 @@
 // Aerie/Solaufein TOB
 
 BEGIN @4
-REQUIRE_PREDICATE GAME_IS ~tob bg2ee bgt~ @100
+REQUIRE_PREDICATE GAME_IS ~tob bg2ee eet bgt~ @100
 
 COMPILE ~npcflirt\aeriesolaconflicttob.d~
 USING ~npcflirt\english\aeriesolaconflictsoa.tra~
@@ -106,7 +108,7 @@
 // Jaheira TOB
 
 BEGIN @7
-REQUIRE_PREDICATE GAME_IS ~tob bg2ee bgt~ @100
+REQUIRE_PREDICATE GAME_IS ~tob bg2ee eet bgt~ @100
 
 COMPILE ~npcflirt\jaheiratobflirt.d~
 USING ~npcflirt\english\jaheiraflirt.tra~
@@ -129,7 +131,7 @@
 // Jaheira/Solaufein TOB
 
 BEGIN @8
-REQUIRE_PREDICATE GAME_IS ~tob bg2ee bgt~ @100
+REQUIRE_PREDICATE GAME_IS ~tob bg2ee eet bgt~ @100
 
 COMPILE ~npcflirt\jsolaconflicttob.d~
 USING ~npcflirt\english\jsolaconflictsoa.tra~
@@ -163,7 +165,7 @@
 // Viconia TOB
 
 BEGIN @11
-REQUIRE_PREDICATE GAME_IS ~tob bg2ee bgt~ @100
+REQUIRE_PREDICATE GAME_IS ~tob bg2ee eet bgt~ @100
 
 COMPILE ~npcflirt\viconiatobflirt.d~
 USING ~npcflirt\english\viconiaflirt.tra~
@@ -180,7 +182,7 @@
 // Viconia/Solaufein TOB
 
 BEGIN @12
-REQUIRE_PREDICATE GAME_IS ~tob bg2ee bgt~ @100
+REQUIRE_PREDICATE GAME_IS ~tob bg2ee eet bgt~ @100
 
 COMPILE ~npcflirt\vsolaconflicttob.d~
 USING ~npcflirt\english\vsolaconflictsoa.tra~
@@ -214,7 +216,7 @@
 USING ~npcflirt\english\anomenflirt.tra~
 ~npcflirt\%s\anomenflirt.tra~
 
-ACTION_IF GAME_IS ~bg2ee~ BEGIN
+ACTION_IF GAME_IS ~bg2ee eet~ BEGIN
 COPY ~npcflirt\anobmp\ee~ ~override~
 END ELSE BEGIN
 COPY ~npcflirt\anobmp\bg2~ ~override~
@@ -239,7 +241,7 @@
 // Anomen TOB
 
 BEGIN @15
-REQUIRE_PREDICATE GAME_IS ~tob bg2ee bgt~ @100
+REQUIRE_PREDICATE GAME_IS ~tob bg2ee eet bgt~ @100
 
 COMPILE ~npcflirt\anomentobflirt.d~
 USING ~npcflirt\english\anomenflirt.tra~
@@ -257,7 +259,7 @@
 // Anomen/Solaufein TOB
 
 BEGIN @16
-REQUIRE_PREDICATE GAME_IS ~tob bg2ee bgt~ @100
+REQUIRE_PREDICATE GAME_IS ~tob bg2ee eet bgt~ @100
 
 COMPILE ~npcflirt\anosolaconflicttob.d~
 USING ~npcflirt\english\anosolaconflictsoa.tra~
