--- aurora\lib\t-arecre.tpa	Mon Feb 01 10:47:18 2010
+++ C:\BWP Patchstudio\patched files\aurora\lib\t-arecre.tpa	Sun Dec 12 15:34:12 2010
@@ -56,10 +56,10 @@
               VERBOSE
             END
           END
-          PATCH_IF (~%t-name%~ STRING_EQUAL ~%t-crnm%~ = 0) BEGIN
+/*        PATCH_IF (~%t-name%~ STRING_EQUAL ~%t-crnm%~ = 0) BEGIN
             t-diff += 1
             WRITE_ASCIIE (t * 0x110 + t-act_off) ~%t-crnm%~ #32
-          END
+          END */
           PATCH_IF t-old != t-crean BEGIN
             LOOKUP_IDS_SYMBOL_OF_INT t-creant animate %t-crean%
             t-anm = t-crean
