--- aurora\lib\t-goblin.tpa	Fri Jan 29 15:13:32 2010
+++ C:\BWP Patchstudio\patched files\aurora\lib\t-goblin.tpa	Sun Mar 14 08:38:20 2010
@@ -1,6 +1,6 @@
 ACTION_FOR_EACH bm IN ~mgo4a2~ ~mgo4a2e~ ~mgo4ca~ ~mgo4cae~ ~mgo4sp~ ~mgo4spe~ BEGIN
   ACTION_IF FILE_EXISTS ~override/%bm%.bam~ BEGIN
-    MOVE ~override/%bm%.bam~ ~aurora/backup~
+    MOVE ~override/%bm%.bam~ ~aurora/backup/%bm%.bam~
   END
 END
 
