--- White\white.tp2	Sun Apr 14 14:47:34 2013
+++ C:\BWP Patchstudio\patched files\White\white.tp2	Wed Oct 19 15:23:51 2016
@@ -2,6 +2,7 @@
 // Basics
 BACKUP ~white/backup~
 AUTHOR ~Lava~
+VERSION ~v1.4 BWP Fix~
 
 // README ~white/readme.txt~
 
@@ -417,14 +418,18 @@
   SAY SET_A_TRAP ~~ []
   SAY HIDDEN_IN_SHADOWS ~~ []
   SAY PICKED_POCKET ~~ []
+  WRITE_ASCII 0x248 ~V#1WHI~ #8  // override script where you put all the commands for lovetalks and such.
+  WRITE_ASCII 0x2cc ~V#1WHI~  #8  // dialogue file where we'll put Branwen's greeting dialogue.
+  WRITE_ASCII 0x280 ~V#1WHI~  #32 // death variable - when we want to refer to Branwen in the game, we'll call her O#Bran. For example, InParty("O#Bran") means that Branwen is in party.
+  
   
   
 // Appends
 APPEND ~pdialog.2da~ ~V#1whi V#1whiP V#1whiJ V#1whiD~
-  UNLESS ~rh#Isra~
+  UNLESS ~V#1whi~
 
 APPEND ~interdia.2da~ ~V#1whi V#1whiB~
-  UNLESS ~rh#Isra~
+  UNLESS ~V#1whi~
   
 //music
 
