--- AbyStore\readme-AbyStore.html	Mon Apr 24 20:24:50 2006
+++ C:\BWP Patchstudio\patched files\AbyStore\readme-AbyStore.html	Thu Sep 26 21:57:57 2013
@@ -786,8 +786,8 @@
 <p align=center style='text-align:center'><span class=style31><span
 style='font-size:14.0pt'>The Unusual Oddities Shop <span class=SpellE>Readme</span></span></span><br>
 <strong>Author:</strong> &nbsp;<span class=SpellE>Rastor</span><br>
-<b>Current Version:</b>&nbsp; v2.1<br>
-<b>Current Release Date:</b>&nbsp; January 7, 2006</p>
+<b>Current Version:</b>&nbsp; v3.0<br>
+<b>Current Release Date:</b>&nbsp; October 9, 2006</p>
 
 
 <p class=style4>Table of Contents</p>
