--- Angelo/Dialogue/ADAngelJ.d	2013-11-11 03:30:24.000000000 -0500
+++ Fixpack/Angelo/Dialogue/ADAngelJ.d	2016-11-04 14:55:56.824567000 -0400
@@ -1672,7 +1672,7 @@
 END
 IF ~~ EXTERN ADSAWARA s11.duel
 
-CHAIN IF ~Dead("ADSawara") GlobalLT("Chapter","GLOBAL",8)~ THEN ADSUU afterwards
+CHAIN IF ~Dead("ADSawara") GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN ADSUU afterwards
 @592
 END
 ++ @593 EXTERN ADSUU a1.1
@@ -4382,19 +4382,19 @@
 
 + ~Global("ADAngelPIDSembia","GLOBAL",0) GlobalGT("ADAngelPIDMore","GLOBAL",0)~ + @1566 DO ~SetGlobal("ADAngelPIDSembia","GLOBAL",1)~ + a6.6b
 
-+ ~GlobalGT("Chapter","GLOBAL",4) Global("ADAngelPIDMore","GLOBAL",1)~ + @1567 DO ~SetGlobal("ADAngelPIDMore","GLOBAL",2)~ + a0.story
++ ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_4%) Global("ADAngelPIDMore","GLOBAL",1)~ + @1567 DO ~SetGlobal("ADAngelPIDMore","GLOBAL",2)~ + a0.story
 
-+ ~GlobalGT("Chapter","GLOBAL",5) Global("ADAngelPIDMore","GLOBAL",2)~ + @1568 DO ~SetGlobal("ADAngelPIDMore","GLOBAL",3)~ + a0.story2
++ ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%) Global("ADAngelPIDMore","GLOBAL",2)~ + @1568 DO ~SetGlobal("ADAngelPIDMore","GLOBAL",3)~ + a0.story2
 
 + ~GlobalGT("ADAngelPIDMore","GLOBAL",2) Global("ADAngelPIDMoreTalk","GLOBAL",6)~ + @1569 DO ~SetGlobal("ADAngelPIDMoreTalk","GLOBAL",7)~ + a6.7
 + ~GlobalGT("ADAngelPIDMore","GLOBAL",2) Global("ADAngelPIDMoreTalk","GLOBAL",7)~ + @1570 DO ~SetGlobal("ADAngelPIDMoreTalk","GLOBAL",8)~ + a6.8
 
-+ ~Global("Chapter","GLOBAL",2)~ + @1571 + a7.2
-+ ~Global("Chapter","GLOBAL",3)~ + @1571 + a7.3
-+ ~Global("Chapter","GLOBAL",4)~ + @1571 + a7.4
-+ ~Global("Chapter","GLOBAL",5)~ + @1571 + a7.5
-+ ~Global("Chapter","GLOBAL",6) !Dead("C6Bodhi")~ + @1571 + a7.6
-+ ~Global("Chapter","GLOBAL",7)~ + @1571 + a7.7
++ ~Global("Chapter","GLOBAL",%bg2_chapter_2%)~ + @1571 + a7.2
++ ~Global("Chapter","GLOBAL",%bg2_chapter_3%)~ + @1571 + a7.3
++ ~Global("Chapter","GLOBAL",%bg2_chapter_4%)~ + @1571 + a7.4
++ ~Global("Chapter","GLOBAL",%bg2_chapter_5%)~ + @1571 + a7.5
++ ~Global("Chapter","GLOBAL",%bg2_chapter_6%) !Dead("C6Bodhi")~ + @1571 + a7.6
++ ~Global("Chapter","GLOBAL",%bg2_chapter_7%)~ + @1571 + a7.7
 
 + ~Dead("ADSawara") Global("ADSawaraTalk","GLOBAL",0)~ + @1572 DO ~SetGlobal("ADSawaraTalk","GLOBAL",1)~ + saw1
 
