--- bgt\compat\bgt\cut01bgt.baf	Sat Dec 17 20:23:22 2005
+++ C:\BWP Patchstudio\patched files\bgt\compat\bgt\cut01bgt.baf	Tue Sep 24 08:43:07 2013
@@ -23,6 +23,11 @@
     ActionOverride("Imoen2",ChangeAIScript("IMOEN2",OVERRIDE))
     ActionOverride("Imoen2",GivePartyAllEquipment())
     ActionOverride("Imoen2",MoveGlobal("AR1512",Myself,[1857.1520]))
+    ActionOverride("Imoen2",SetPlayerSound(Myself,30759,MORALE))
+    ActionOverride("Imoen2",SetPlayerSound(Myself,30760,BORED))
+    ActionOverride("Imoen2",SetPlayerSound(Myself,30761,BATTLE_CRY2))
+    ActionOverride("Imoen2",SetPlayerSound(Myself,30762,BATTLE_CRY3))
+    ActionOverride("Imoen2",SetPlayerSound(Myself,30764,HURT))
     CreateCreature("CSIMOEN",[2732.831],2)
     CreateCreature("CSIREN",[2557.935],0)
     CreateCreature("CSTHIEF1",[2740.1009],6)
