--- Yasraena\Setup-Yasraena.tp2	Wed Apr 08 00:35:09 2015
+++ C:\BWP Patchstudio\patched files\Yasraena\Setup-Yasraena.tp2	Fri Jun 26 22:48:57 2015
@@ -1,7 +1,7 @@
 BACKUP ~Yasraena/backup~
 AUTHOR ~gilalion@gmail.com~
 
-VERSION ~v16~
+VERSION ~v16 BWP Fix~
 
 README ~Yasraena/README-Yasraena.html~
 
@@ -190,12 +190,12 @@
 
 /* compile .d files into .dlg files which get stored in the Overide directory */
 COMPILE ~Yasraena/Dialogues/YASRAENP.d~ ~Yasraena/Dialogues/YASRA25P.d~
-USING ~Yasraena/English/JOURNAL.tra~
+USING ~Yasraena/%s/JOURNAL.tra~
 COMPILE ~Yasraena/Dialogues/BYASRAEN.d~ ~Yasraena/Dialogues/BYASRA25.d~
 COMPILE ~Yasraena/Dialogues/YASRAENJ.d~ ~Yasraena/Dialogues/YASRA25J.d~
-USING ~Yasraena/English/JOURNAL.tra~
+USING ~Yasraena/%s/JOURNAL.tra~
 COMPILE ~Yasraena/Dialogues/YASRAENA.d~ ~Yasraena/Dialogues/DIVALIR.d~ ~Yasraena/Dialogues/DIVALIR1.d~ ~Yasraena/Dialogues/YSCPTN.d~ ~Yasraena/Dialogues/YSTMCPTN.d~ ~Yasraena/Dialogues/YSNETHR.d~ ~Yasraena/Dialogues/YSMOM.d~
-USING ~Yasraena/English/JOURNAL.tra~
+USING ~Yasraena/%s/JOURNAL.tra~
 
 /* compile .baf files into .bcs files which get stored in the Override directory */
 COMPILE ~Yasraena/Scripts/YSAR0603.baf~ ~Yasraena/Scripts/YSAR2200.baf~ ~Yasraena/Scripts/YSAR2800.baf~ ~Yasraena/Scripts/YSAR0300.baf~ ~Yasraena/Scripts/YSAR0704.baf~ ~Yasraena/Scripts/YSAR5202.baf~ ~Yasraena/Scripts/YSAR6100.baf~
