--- stivan\core.tpa	Sat Aug 20 19:20:54 2011
+++ C:\BWP Patchstudio\patched files\stivan\core.tpa	Fri Jul 03 22:37:03 2015
@@ -91,7 +91,7 @@
 	WRITE_ASCII 0x2cc ~tb#svul~ #8 // dialogue
 
 COPY_EXISTING ~vulfgt01.cre~ ~override/tb#sv01.cre~
-			  ~vulfgt01.cre~ ~override/tb#sv01.cre~
+			  ~vulfgt01.cre~ ~override/tb#sv02.cre~
 	WRITE_ASCII 0x248 ~SHOUT~ #8 // override script
 	WRITE_EVALUATED_ASCII 0x280 ~%DEST_RES%~ #32 // name
 	WRITE_ASCII 0x2cc ~~ #8 // dialogue
