--- sarevokromance/sarevokromance.tp2	2013-11-11 16:30:32.489047800 -0500
+++ Fixpack/sarevokromance/sarevokromance.tp2	2016-11-21 22:45:56.099638800 -0500
@@ -14,1 +14,1 @@
-VERSION ~1.2~
+VERSION ~1.2 BWP Fix~
@@ -25,7 +25,7 @@
 
 // Component
 BEGIN @1
-  REQUIRE_PREDICATE GAME_IS ~tob bgt bg2ee~ @2
+  REQUIRE_PREDICATE GAME_IS ~tob bgt bg2ee eet~ @2
 
 
 // --------------------------------------
