--- setup-RotBArmorSet.tp2	Wed Jun 22 18:07:24 2005
+++ C:\BWP Patchstudio\patched files\setup-RotBArmorSet.tp2	Thu Jun 11 04:02:34 2015
@@ -1,13 +1,16 @@
 BACKUP ~RotBArmorSet\backup~
 
 AUTHOR ~lordjyssev@yahoo.com~
+
+VERSION ~v1.0 BWP Fix~
+
 BEGIN ~Realm of the Bhaalspawn Armor Set~
 
 COPY ~RotBArmorSet\Store\rbarmor.sto~ ~override\rbarmor.sto~
 SAY NAME2 ~Jyssev's Heirlooms~
 
 COPY ~RotBArmorSet\Items\rbplat01.itm~ ~override\rbplat01.itm~
-SAY NAME2~  Jyssev Plate +5~
+SAY NAME2 ~Jyssev Plate +5~
 SAY UNIDENTIFIED_DESC ~James Issiac Jyssev, one of the wealthy nobles of Waterdeep, had this armor forged for him when he was in the service of the army. The armor was used in many battles before it was put in display in one of James's estates. James's wife bore a son, Issiac, and when Issiac grew older, the armor was passed down to him, when he, like his father, joined the army. The Jyssev plate had been handed down for many years until it got to William Jyssev, who is now a traveling merchant.
 
 STATISTICS:
@@ -24,7 +27,7 @@
 
 
 COPY ~RotBArmorSet\Items\rbplat02.itm~ ~override\rbplat02.itm~
-SAY NAME1 ~Blessed of the Triad~ 
+SAY NAME2 ~Blessed of the Triad~ 
 SAY UNIDENTIFIED_DESC ~This armor is the first of a set of armors that were created by three syblings. This plate, Blessed, was made by Godric Gladivir, a powerful mage who believed in the god known as Torm. It was said that if a person ever wore this plate, then they would be blessed and protected from many things.
 
 STATISTICS:
