--- divine_remix/lib/a#koss.tph	2014-11-01 18:30:52.000000000 -0400
+++ Fixpack/divine_remix/lib/a#koss.tph	2016-11-21 16:21:30.556272100 -0500
@@ -31,7 +31,7 @@
   LAUNCH_PATCH_MACRO ~spell_to_innate~ // change to innate spell
   BUT_ONLY_IF_IT_CHANGES
 
-ACTION_IF GAME_IS ~tob bg2ee bgt~ THEN BEGIN
+ACTION_IF GAME_IS ~tob bg2ee eet bgt~ THEN BEGIN
 
   COPY ~Divine_Remix/spl/a#koss20.spl~ ~override~ //Kossuth's Purifying Touch
     SAY NAME1 @11505
