--- Divine_Remix\lib\macro_spellbook_oghma.tph	Sun Jul 29 14:28:44 2012
+++ C:\BWP Patchstudio\patched files\Divine_Remix\lib\macro_spellbook_oghma.tph	Tue Oct 15 07:49:23 2013
@@ -1,9 +1,11 @@
 // kit stormlords, adjust spellbooks
 DEFINE_PATCH_MACRO ~spellbook_oghma~ BEGIN
 
-  WRITE_SHORT   0x244 0
-  WRITE_BYTE    0x246 "%A#OGMA%"
-  WRITE_BYTE    0x247 0x40
+  SET x = IDS_OF_SYMBOL(~kit~ ~A#OGMA~)
+  SET y = x / 0x10000
+  SET z = x & 0xFFFF
+  WRITE_SHORT 0x244 ~%y%~
+  WRITE_SHORT 0x246 ~%z%~
   READ_LONG 0x2A0 "sb_off"
   READ_LONG 0x2A4 "sb_num"
   READ_LONG 0x2A8 "meminfo_off"
