--- Divine_Remix\lib\macro_spellbook_cyric.tph	Sun Jul 29 14:28:44 2012
+++ C:\BWP Patchstudio\patched files\Divine_Remix\lib\macro_spellbook_cyric.tph	Tue Oct 15 07:52:30 2013
@@ -1,9 +1,12 @@
 // kit lorekeepers, adjust spellbooks
 DEFINE_PATCH_MACRO ~spellbook_cyric~ BEGIN
 
-  WRITE_SHORT   0x244 0
-  WRITE_BYTE    0x246 "%A#CYRIC%"
-  WRITE_BYTE    0x247 0x40
+
+  SET x = IDS_OF_SYMBOL(~kit~ ~A#CYRIC~)
+  SET y = x / 0x10000
+  SET z = x & 0xFFFF
+  WRITE_SHORT 0x244 ~%y%~
+  WRITE_SHORT 0x246 ~%z%~
   READ_LONG 0x2A0 "sb_off"
   READ_LONG 0x2A4 "sb_num"
   READ_LONG 0x2A8 "meminfo_off"
