--- divine_remix\setup-divine_remix.tp2	Tue Mar 17 19:20:30 2015
+++ C:\BWP Patchstudio\patched files\divine_remix\setup-divine_remix.tp2	Sun Jun 28 20:58:23 2015
@@ -1797,7 +1797,7 @@
 
 //Alignment changes
 COPY_EXISTING ~alignmnt.2da~ ~override~
-   REPLACE_TEXTUALLY ~^\(DRUID\|FIGHTER_DRUID\|TOTEMIC_DRUID\|SHAPESHIFTER\|BEAST_FRIEND\)[ %TAB%]+.+~ ~\1 0 1 0 1 1 1 0 1 0~
+  REPLACE_TEXTUALLY ~^\(DRUID\|FIGHTER_DRUID\|TOTEMIC_DRUID\|SHAPESHIFTER\|BEAST_FRIEND\)[ %TAB%]~ ~\1 0 1 0 1 1 1 0 1 0~
   BUT_ONLY_IF_IT_CHANGES
 
 PRINT @3

@@ -31,7 +31,7 @@
 
 END
 
-VERSION  ~v8~
+VERSION  ~v8 BWP fix 2~
 
 README   ~divine_remix/readme-divine_remix.html~
 
@@ -354,8 +354,8 @@
 
     COPY_EXISTING ~%creature%.cre~ ~override~  
       WRITE_SHORT   0x244 0
-      WRITE_BYTE    0x246 "%kit%"
-      WRITE_BYTE    0x247 0x40
+      WRITE_SHORT   0x246 0x4000 + "%kit%"
+//    WRITE_BYTE    0x247 0x40
       BUT_ONLY
 
   END
@@ -369,8 +369,8 @@
 
     COPY_EXISTING ~%creature%.cre~ ~override~  
       WRITE_SHORT   0x244 0
-      WRITE_BYTE    0x246 "%kit%"
-      WRITE_BYTE    0x247 0x40
+      WRITE_SHORT   0x246 0x4000 + "%kit%"
+//    WRITE_BYTE    0x247 0x40
       BUT_ONLY
 
   END
@@ -384,8 +384,8 @@
 
     COPY_EXISTING ~%creature%.cre~ ~override~  
       WRITE_SHORT   0x244 0
-      WRITE_BYTE    0x246 "%kit%"
-      WRITE_BYTE    0x247 0x40
+      WRITE_SHORT   0x246 0x4000 + "%kit%"
+//    WRITE_BYTE    0x247 0x40
       BUT_ONLY
 
   END
@@ -401,8 +401,8 @@
 
       COPY_EXISTING ~%creature%.cre~ ~override~
         WRITE_SHORT   0x244 0
-        WRITE_BYTE    0x246 "%kit%"
-        WRITE_BYTE    0x247 0x40
+        WRITE_SHORT   0x246 0x4000 + "%kit%"
+//      WRITE_BYTE    0x247 0x40
         BUT_ONLY
 
     END
@@ -420,8 +420,8 @@
 
       COPY_EXISTING ~%creature%.cre~ ~override~
         WRITE_SHORT   0x244 0
-        WRITE_BYTE    0x246 "%kit%"
-        WRITE_BYTE    0x247 0x40
+        WRITE_SHORT   0x246 0x4000 + "%kit%"
+//      WRITE_BYTE    0x247 0x40
         BUT_ONLY
 
     END
@@ -890,8 +890,8 @@
 
     COPY_EXISTING ~%creature%.cre~ ~override~
       WRITE_SHORT   0x244 0
-      WRITE_BYTE    0x246 "%kit%"
-      WRITE_BYTE    0x247 0x40
+      WRITE_SHORT   0x246 0x4000 + "%kit%"
+//    WRITE_BYTE    0x247 0x40
       BUT_ONLY
 
   END
@@ -1022,8 +1022,8 @@
 
     COPY_EXISTING ~%creature%.cre~ ~override~
       WRITE_SHORT   0x244 0
-      WRITE_BYTE    0x246 "%kit%"
-      WRITE_BYTE    0x247 0x40
+      WRITE_SHORT   0x246 0x4000 + "%kit%"
+//    WRITE_BYTE    0x247 0x40
       BUT_ONLY
 
   END
@@ -1391,8 +1391,8 @@
 
     COPY_EXISTING ~%creature%.cre~ ~override~
       WRITE_SHORT   0x244 0
-      WRITE_BYTE    0x246 "%kit%"
-      WRITE_BYTE    0x247 0x40
+      WRITE_SHORT   0x246 0x4000 + "%kit%"
+//    WRITE_BYTE    0x247 0x40
       BUT_ONLY
 
   END
@@ -1750,8 +1750,8 @@
 
     COPY_EXISTING ~%creature%.cre~ ~override~
       WRITE_SHORT   0x244 0
-      WRITE_BYTE    0x246 "%kit%"
-      WRITE_BYTE    0x247 0x40
+      WRITE_SHORT   0x246 0x4000 + "%kit%"
+//    WRITE_BYTE    0x247 0x40
       BUT_ONLY
 
   END
@@ -2211,8 +2211,8 @@
 
     COPY_EXISTING ~%creature%.cre~ ~override~
       WRITE_SHORT   0x244 0
-      WRITE_BYTE    0x246 "%kit%"
-      WRITE_BYTE    0x247 0x40
+      WRITE_SHORT   0x246 0x4000 + "%kit%"
+//    WRITE_BYTE    0x247 0x40
       BUT_ONLY
 
   END
@@ -2289,8 +2289,8 @@
 
     COPY_EXISTING ~%creature%.cre~ ~override~
       WRITE_SHORT   0x244 0
-      WRITE_BYTE    0x246 "%kit%"
-      WRITE_BYTE    0x247 0x40
+      WRITE_SHORT   0x246 0x4000 + "%kit%"
+//    WRITE_BYTE    0x247 0x40
       BUT_ONLY
 
   END
