--- Assassinations/Setup-Assassinations.tp2	2014-11-10 15:52:22.000000000 -0500
+++ Fixpack/Assassinations/Setup-Assassinations.tp2	2016-11-22 15:25:43.709557900 -0500
@@ -1,26 +1,34 @@
 BACKUP ~Assassinations/Backup~
 AUTHOR ~http://forums.pocketplane.net~
-VERSION ~v11~
+
+VERSION ~v11 BWP Fix~
 
 README ~Assassinations/%LANGUAGE%/Readme-Assassinations.txt~ ~Assassinations/Readme-Assassinations.txt~ 
 
 ALWAYS
-  // Borrowed from Edwin Romance: convert strings to UTF-8 for BGEE/BG2EE
-  INCLUDE ~%MOD_FOLDER%/handle_charsets.tpa~
+  ACTION_IF GAME_IS ~eet~ BEGIN
+    OUTER_SET bg2_chapter = 12
+  END ELSE BEGIN
+    OUTER_SET bg2_chapter = 0
+  END
+  OUTER_FOR (i=1; i<=10; i=i+1) BEGIN
+    OUTER_SET bg2_chapter = bg2_chapter + 1
+    OUTER_SPRINT name_source ~bg2_chapter_%i%~
+    OUTER_SET EVAL ~%name_source%~ = bg2_chapter
+  END
 
   // No exception, everything needs to be converted
   ACTION_DEFINE_ARRAY O#noconvert BEGIN END
 
-  // Only setup-Mod.tra needs to be reloaded
+  // any tra files used in LANGUAGE need to be reloaded after UTF-8 conversion
   ACTION_DEFINE_ARRAY O#reload BEGIN Setup-Assassinations END
 
-  // There is no "language" folder so the root of all languages in %MOD_FOLDER%
+  // There is no "language" folder so the root of all languages is %MOD_FOLDER%
   LAF HANDLE_CHARSETS
     INT_VAR
-      infer_charset = 1
+      infer_charsets = 1
     STR_VAR
       tra_path = EVAL ~%MOD_FOLDER%~
-      // charset_table = O#charsets // Included for illustrative purposes.
       noconvert_array = O#noconvert
       reload_array = O#reload
   END
@@ -38,7 +46,7 @@
 
 BEGIN @0
 
-ACTION_IF GAME_IS ~BG2EE~ BEGIN
+ACTION_IF GAME_IS ~bg2ee eet~ BEGIN
 ADD_JOURNAL @10000 @10010 @10020 @10030 @10040 @10050 @10060 @10070 @10080 @10090 @10100 @10110 @10120 @10130 @10140 @10150 @10160 @10170 @10180 @10190 @10200 @10210 @10220 @10230 @10240 @10250 @10260 @10270 @10280 @10290 @10300 @10310 @10320 @10330 @10340 @10350 @10360 @10370 @10380 @10390 @10400 @10410 @10420 @10430 @10440 @10450 @10460 @10470 @10480 @10490 @10500 @10510 @10520 @10530 @10540 @10550 @10560
 END
 
@@ -311,7 +319,7 @@
 
 // Dialogue and scripts
 
-COMPILE ~Assassinations/Dialogue/Assassinations.d~
+COMPILE EVALUATE_BUFFER ~Assassinations/Dialogue/Assassinations.d~
 
 	~Assassinations/Scripts/O#CrCut1.baf~
 	~Assassinations/Scripts/O#CrCut2.baf~
