--- vanim\M#Verhaf.baf	Sat May 03 19:41:12 2003
+++ C:\BWP Patchstudio\patched files\vanim\M#Verhaf.baf	Sat Jan 16 11:47:56 2016
@@ -11,7 +11,7 @@
     Wait(3)
     DisplayStringHead("M#Verhaf",@2)
     Wait(1)
-    EscapeAreaObject("M#Verhaf")
-    EscapeAreaObject("M#Gonla")
+    ActionOverride("M#Verhaf",DestroySelf())
+    ActionOverride("M#Gonla",DestroySelf())
     EndCutSceneMode()
 END
