--- iwdification\setup-iwdification.tp2	Sun Aug 18 21:02:20 2013
+++ C:\BWP Patchstudio\patched files\iwdification\setup-iwdification.tp2	Sun Jan 03 13:04:15 2016
@@ -5,7 +5,7 @@
 
 AUTHOR ~pcamagna@yahoo.com~ // email address displayed if install fails
 
-VERSION ~Beta 2~
+VERSION ~Beta 3 BWP Fix~
 
 README ~iwdification/readme-iwdification.html~
 
@@ -741,7 +741,7 @@
              (FILE_EXISTS_IN_GAME ~µDAa1.bam~)) THEN BEGIN
   
     COPY ~iwdification/cre/cdm4yeti.cre~ ~override~
-      SAY 0x0c @3051
+      SAY 0x08 @3051
       SAY 0x0c @3051
       WRITE_LONG 0x28 IDS_OF_SYMBOL (~animate~ ~TUNDRA_YETI~)
   
@@ -1889,12 +1889,12 @@
          ~iwdification/wav/cdeffm47.wav~ ~override~
   
     COPY ~iwdification/cre/cdgibomb.cre~ ~override~
-      SAY 0x0c @4076
+      SAY 0x08 @4076
       SAY 0x0c @4076
       WRITE_LONG 0x28 IDS_OF_SYMBOL (~animate~ ~BEETLE_BLACK~)
   
     COPY ~iwdification/cre/cdgiborb.cre~ ~override~
-      SAY 0x0c @4077
+      SAY 0x08 @4077
       SAY 0x0c @4077
       WRITE_LONG 0x28 IDS_OF_SYMBOL (~animate~ ~BEETLE_BLACK~)
   
@@ -3134,7 +3134,9 @@
 ACTION_IF FILE_EXISTS_IN_GAME ~ar0602.bcs~ THEN BEGIN // soa
 
   EXTEND_BOTTOM ~ar0602.bcs~ ~iwdification/baf/ar0602.baf~ // add generic 2H axe to CI
-  
+END
+
+ACTION_IF FILE_EXISTS_IN_GAME ~hlolaf.cre~ THEN BEGIN // Olaf Rasmussen  
   COPY_EXISTING ~hlolaf.cre~ ~override~
     REPLACE_CRE_ITEM ~cdax2h3~ #0 #0 #0 ~NONE~ ~WEAPON1~ EQUIP TWOHANDED // add +2 axe to olaf at guarded compound
 
