--- coranbgfriend\tra\english\p#coranft.tra	Sat Apr 26 11:02:38 2008
+++ C:\BWP Patchstudio\patched files\coranbgfriend\tra\english\p#coranft.tra	Fri Nov 12 16:23:32 2010
@@ -181,7 +181,7 @@
 @180  = ~With those creatures, all the fun is not in capturing them, but in letting them capture you. *Wink.* ~
 @181  = ~Hmm, that starts to sound like a job, but for you - anything, my friend. Let it be a bear.~
 @182  = ~Well, yes, but if you prefer porcupines... It matters not!~
-@183  = ~And that is exactly why you need to come with me on a hunting expedition. Why, when a <PRO_MANWOMAN> starts jumping when someone talks to <PROHIMHER>, it is a sign of approaching doom.~
+@183  = ~And that is exactly why you need to come with me on a hunting expedition. Why, when a <PRO_MANWOMAN> starts jumping when someone talks to <PRO_HIMHER>, it is a sign of approaching doom.~
 @184  = ~If we drop everything and go hunting, *then* we'll meet our doom really quick. This is the worst idea you've ever had.~
 @185  = ~Er... Maybe you are right, but I'd rather go fishing.~
 @186  = ~Hmm, now that it has actually sunk in, I like your idea. Where do you want to go? ~
