--- crossmodbg2/setup-crossmodbg2.tp2	2016-11-27 13:18:35.115510300 -0500
+++ Fixpack/crossmodbg2/setup-crossmodbg2.tp2	2016-11-30 11:40:32.147399800 -0500
@@ -7,1 +7,1 @@
-VERSION @0 /* <--- current version number from setup.tra here */
+VERSION @0 /* <--- current version number from setup.tra here (BWP Fix) */
@@ -10,18 +10,36 @@
 
 
 ALWAYS
+	ACTION_IF GAME_IS ~eet~ BEGIN
+		OUTER_SET bg2_chapter = 12
+	END ELSE BEGIN
+		OUTER_SET bg2_chapter = 0
+	END
+	OUTER_FOR (i=1; i<=10; i=i+1) BEGIN
+		OUTER_SET bg2_chapter = bg2_chapter + 1
+		OUTER_SPRINT name_source ~bg2_chapter_%i%~
+		OUTER_SET EVAL ~%name_source%~ = bg2_chapter
+	END
+
 	ACTION_IF GAME_IS ~bgee tutu bg1~ THEN BEGIN
 		PRINT @1 /* WARNING. THIS MOD IS FOR BG2 OR BGT. DO NOT INSTALL THIS MOD. */
 	END
 	ACTION_IF GAME_IS ~bg2ee eet~ THEN BEGIN
 		PRINT @2 /* BG2:EE detected... */
+		
+		ACTION_IF "%WEIDU_OS%" STRING_EQUAL_CASE ~win32~ BEGIN // HANDLE_CHARSETS doesn't work with iconv_path containing spaces
+			AT_NOW ~if not exist "iconv" move "BiG World Fixpack\_utils\iconv" "iconv"~
+		END
+		ACTION_DEFINE_ARRAY fl#noconvert BEGIN END
+		ACTION_DEFINE_ARRAY fl#reload BEGIN setup END
 		LAF HANDLE_CHARSETS
 			INT_VAR
-				infer_charset = 1
+				infer_charsets = 1
 			STR_VAR
 				tra_path = EVAL ~crossmodbg2/tra~
 				noconvert_array = fl#noconvert
 				reload_array = fl#reload
+				iconv_path = ~iconv~
 		END
 	END
 	ACTION_IF GAME_IS ~soa~ THEN BEGIN
@@ -243,7 +261,7 @@
 	PRINT @26 /* Adding SoA banters between Amber and Kulyok's Xan... */
 	APPEND_OUTER ~crossmodbg2/crossmod_0_debug.log~ ~Adding SoA banters between Amber and Kulyok's Xan...~
 	COMPILE ~crossmodbg2/soa/banters/soa_Amber-kulyokxan/soa_Amber-kulyokxan_banters.d~
-	EXTEND_BOTTOM ~m#amber.bcs~ ~crossmodbg2/soa/banters/soa_Amber-kulyokxan/Amber-kulyokxan_M#Amber.baf~
+	EXTEND_BOTTOM ~m#amber.bcs~ ~crossmodbg2/soa/banters/soa_Amber-kulyokxan/Amber-kulyokxan_M#Amber.baf~ EVALUATE_BUFFER
 END ELSE BEGIN
 	PRINT @27 /* bm#amber.dlg and/or o#xan.dlg not detected. Skipping these particular SoA banters. */
 	APPEND_OUTER ~crossmodbg2/crossmod_0_debug.log~ ~bm#amber.dlg and/or o#xan.dlg not detected. Skipping these particular SoA banters.~
@@ -301,7 +319,7 @@
 THEN BEGIN
 	PRINT @36 /* Adding SoA banters between Auren and Kivan of Shilmista... */
 	APPEND_OUTER ~crossmodbg2/crossmod_0_debug.log~ ~Adding SoA banters between Auren and Kivan of Shilmista...~
-	COMPILE ~crossmodbg2/soa/banters/soa_Auren-KivanOfShilmista/soa_Auren-KivanOfShilmista_banters.d~
+	COMPILE EVALUATE_BUFFER ~crossmodbg2/soa/banters/soa_Auren-KivanOfShilmista/soa_Auren-KivanOfShilmista_banters.d~
 END ELSE BEGIN
 	PRINT @37 /* k#aurenb.dlg and/or bp#kiv.dlg not detected. Skipping these particular SoA banters. */
 	APPEND_OUTER ~crossmodbg2/crossmod_0_debug.log~ ~k#aurenb.dlg and/or bp#kiv.dlg not detected. Skipping these particular SoA banters.~
@@ -606,7 +624,7 @@
 THEN BEGIN
 	PRINT @90 /* Adding SoA banters between Kivan of Shilmista and Kulyok's Coran... */
 	APPEND_OUTER ~crossmodbg2/crossmod_0_debug.log~ ~Adding SoA banters between Kivan of Shilmista and Kulyok's Coran...~
-	COMPILE ~crossmodbg2/soa/banters/soa_kivanofshilmista-kulyokcoran/soa_kivanofshilmista-kulyokcoran_banters.d~
+	COMPILE EVALUATE_BUFFER ~crossmodbg2/soa/banters/soa_kivanofshilmista-kulyokcoran/soa_kivanofshilmista-kulyokcoran_banters.d~
 END ELSE BEGIN
 	PRINT @91 /* p#kivan.dlg and/or o#coran.dlg not detected. Skipping these particular SoA banters. */
 	APPEND_OUTER ~crossmodbg2/crossmod_0_debug.log~ ~p#kivan.dlg and/or o#coran.dlg not detected. Skipping these particular SoA banters.~
@@ -768,7 +786,7 @@
 THEN BEGIN
 	PRINT @118 /* Adding SoA banters between Tsujatha and Yasraena... */
 	APPEND_OUTER ~crossmodbg2/crossmod_0_debug.log~ ~Adding SoA banters between Tsujatha and Yasraena...~
-	COMPILE ~crossmodbg2/soa/banters/soa_Tsujatha-Yasraena/soa_Tsujatha-Yasraena_banters.d~
+	COMPILE EVALUATE_BUFFER ~crossmodbg2/soa/banters/soa_Tsujatha-Yasraena/soa_Tsujatha-Yasraena_banters.d~
 END ELSE BEGIN
 	PRINT @119 /* btsujat.dlg and/or byasraen.dlg not detected. Skipping these particular SoA banters. */
 	APPEND_OUTER ~crossmodbg2/crossmod_0_debug.log~ ~btsujat.dlg and/or byasraen.dlg not detected. Skipping these particular SoA banters.~
@@ -889,7 +907,7 @@
 THEN BEGIN
 	PRINT @140 /* Adding SoA banters between Isra and Kulyok's Coran... */
 	APPEND_OUTER ~crossmodbg2/crossmod_0_debug.log~ ~Adding SoA banters between Isra and Kulyok's Coran...~
-	COMPILE ~crossmodbg2/soa/banters/soa_Isra-kulyokcoran/soa_Isra-kulyokcoran_banters.d~
+	COMPILE EVALUATE_BUFFER ~crossmodbg2/soa/banters/soa_Isra-kulyokcoran/soa_Isra-kulyokcoran_banters.d~
 END ELSE BEGIN
 	PRINT @141 /* brh#isr.dlg and/or bo#coran.dlg not detected. Skipping these particular SoA banters. */
 	APPEND_OUTER ~crossmodbg2/crossmod_0_debug.log~ ~brh#isr.dlg and/or bo#coran.dlg not detected. Skipping these particular SoA banters.~
@@ -1121,7 +1139,7 @@
 THEN BEGIN
 	PRINT @182 /* Adding SoA banters between Adrian and Arath... */
 	APPEND_OUTER ~crossmodbg2/crossmod_0_debug.log~ ~Adding SoA banters between Adrian and Arath...~
-	COMPILE ~crossmodbg2/soa/banters/soa_Adrian-Arath/soa_Adrian-Arath_banters.d~
+	COMPILE EVALUATE_BUFFER ~crossmodbg2/soa/banters/soa_Adrian-Arath/soa_Adrian-Arath_banters.d~
 END ELSE BEGIN
 	PRINT @183 /* badangel.dlg and/or lk#arath.dlg not detected. Skipping these particular SoA banters. */
 	APPEND_OUTER ~crossmodbg2/crossmod_0_debug.log~ ~badangel.dlg and/or lk#arath.dlg not detected. Skipping these particular SoA banters.~
@@ -1167,7 +1185,7 @@
 THEN BEGIN
 	PRINT @190 /* Adding SoA banters between Adrian and Kivan of Shilmista... */
 	APPEND_OUTER ~crossmodbg2/crossmod_0_debug.log~ ~Adding SoA banters between Adrian and Kivan of Shilmista...~
-	COMPILE ~crossmodbg2/soa/banters/soa_Adrian-KivanOfShilmista/soa_Adrian-KivanOfShilmista_banters.d~
+	COMPILE EVALUATE_BUFFER ~crossmodbg2/soa/banters/soa_Adrian-KivanOfShilmista/soa_Adrian-KivanOfShilmista_banters.d~
 	EXTEND_BOTTOM ~rh#adrd.bcs~ ~crossmodbg2/soa/banters/soa_Adrian-KivanOfShilmista/Adrian-KivanOfShilmista_rh#adrd.baf~
 END ELSE BEGIN
 	PRINT @191 /* rh#adr.dlg and/or p#kivan.dlg not detected. Skipping these particular SoA banters. */
@@ -1672,7 +1690,7 @@
 
 BEGIN @223 /* Crossmod Banter Pack for Throne of Bhaal */
 REQUIRE_PREDICATE NOT FILE_EXISTS_IN_GAME ~fw0100.are~ @7 /* You can't have BG1Tutu installed to install this component. */
-REQUIRE_PREDICATE FILE_EXISTS_IN_GAME ~baldur25.bcs~ @224 /* You must have Throne of Bhaal installed to install this component. */
+REQUIRE_PREDICATE GAME_INCLUDES ~tob~ @224 /* You must have Throne of Bhaal installed to install this component. */
 
 COPY ~crossmodbg2/media/blank.log~ ~crossmodbg2/crossmod_1_debug.log~
 APPEND_OUTER ~crossmodbg2/crossmod_1_debug.log~ ~//\\//\\//\\//\\//\\//\\//\\//\\//\\//\\//\\//~
