--- crossmodbg2/soa/banters/soa_kivanofshilmista-kulyokcoran/soa_kivanofshilmista-kulyokcoran_banters.d	2016-11-27 13:18:37.410981300 -0500
+++ Fixpack/crossmodbg2/soa/banters/soa_kivanofshilmista-kulyokcoran/soa_kivanofshilmista-kulyokcoran_banters.d	2016-11-07 03:31:14.000000000 -0500
@@ -51,7 +51,7 @@
 !StateCheck("O#Coran",CD_STATE_NOTVALID)
 CombatCounter(0)
 !See([ENEMY])
-GlobalLT("Chapter","GLOBAL",4)
+GlobalLT("Chapter","GLOBAL",%bg2_chapter_4%)
 Global("G#XB.CoranKivanBanter3","GLOBAL",0)~ THEN BO#CORAN CoranKivanBanter#3.1
 @18 /* Have you seen that mysterious enemy of <CHARNAME>'s? */
 DO ~SetGlobal("G#XB.CoranKivanBanter3","GLOBAL",1)~
