--- crossmodbg2/soa/banters/soa_adrian-arath/soa_adrian-arath_banters.d	2016-11-27 13:18:35.483140700 -0500
+++ Fixpack/crossmodbg2/soa/banters/soa_adrian-arath/soa_adrian-arath_banters.d	2016-11-07 03:31:14.000000000 -0500
@@ -66,7 +66,7 @@
 !StateCheck("rh#Adrian",CD_STATE_NOTVALID)
 CombatCounter(0)
 !See([ENEMY])
-GlobalGT("Chapter","GLOBAL",5)
+GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("G#XB.AdrianArathBanter2","GLOBAL",1)
 Global("G#XB.AdrianArathBanter4","GLOBAL",0)~ THEN BRH#ADR AdrianArathBanter#4.1
 @24 /* I didn't realise I was quite so confusing, druid. */
