--- stratagems/lib/always.tph	Sat Jan 17 15:04:43 2015
+++ Fixpack/stratagems/lib/always.tph	Tue Dec 20 16:47:08 2016
@@ -9,12 +9,15 @@
   OUTER_SET initialise=1
 
 ////////////////////////////////////////////////////////
-//// load languages for BGEE
+//// load language files for Enhanced Edition
 ////////////////////////////////////////////////////////
 
-  ACTION_IF GAME_IS "bgee bg2ee" BEGIN
+  ACTION_IF GAME_IS ~bgee bg2ee eet~ BEGIN
     LOAD_TRA ~%scsroot%/tra/ee/english/english.tra~
     LOAD_TRA ~%scsroot%/tra/ee/%LANGUAGE%/%LANGUAGE%.tra~
+	OUTER_SPRINT eetra ~ee/~
+  END ELSE BEGIN
+	OUTER_SPRINT eetra ~~  
   END
 
 
@@ -126,6 +129,10 @@
 	SPRINT $script_name_pdialog(~deder~) ~~
 	SPRINT $script_name_pdialog(~mordaine~) ~~
 	SPRINT $script_name_pdialog(~osprey~) ~~
+	// SoD Joinable NPCs whose DVs do not match their PDIALOG.2DA entries
+	SPRINT $script_name_pdialog(~glint~) ~~
+	SPRINT $script_name_pdialog(~mkhiin~) ~~
+	SPRINT $script_name_pdialog(~voghiln~) ~~
   BUT_ONLY_IF_IT_CHANGES
 
 ///////////////////////////////////////////////////////////////
@@ -153,7 +160,7 @@
 // String localisation
 ///////////////////////////////////////////////////
 
-ACTION_IF GAME_IS BGEE BEGIN
+ACTION_IF GAME_IS ~bgee~ BEGIN
    OUTER_SPRINT trigger_string 25101
    OUTER_SPRINT minor_sequencer_string 26469
    OUTER_SPRINT sequencer_string 24618
@@ -300,7 +307,7 @@
 // detect whether Truesight cures blindness
 ///////////////////////////////////////////////////
 
-    ACTION_IF FILE_EXISTS_IN_GAME ~dw#truesight_blindness.mrk~ BEGIN
+    ACTION_IF FILE_EXISTS_IN_GAME ~dw#truesight_blindness.mrk~ OR FILE_EXISTS_IN_GAME ~dvimhere.mrk~ BEGIN
            OUTER_SPRINT ~sslvariables~ ~%sslvariables%~^~&TruesightCuresBlindness=True~
     END
     
