--- stratagems/tactical_bg1/c2end.tpa	2013-11-14 08:28:46.000000000 -0500
+++ Fixpack/stratagems/tactical_bg1/c2end.tpa	2016-11-20 18:06:50.572288200 -0500
@@ -73,7 +73,7 @@
 BEGIN 2 6 8 END
 BEGIN 0 END
 BEGIN
-"ACTION" ~Enemy() SetGlobal("CDMulahey","AR5405",3) SetGlobal("DMWWMulaheyFight","MYAREA",1)SetGlobal("DMWWMulaheyFlunky","MYAREA",1)~  // the CDMulahey is relevant only on BGEE
+"ACTION" ~Enemy() SetGlobal("CDMulahey","%NashkelMines_MulaheysCave%",3) SetGlobal("DMWWMulaheyFight","MYAREA",1)SetGlobal("DMWWMulaheyFlunky","MYAREA",1)~  // the CDMulahey is relevant only on BGEE
 END
 >>>>>>>>
      LAF install STR_VAR files=mulreplace.d inline=yes END
