--- stratagems\item\dispelling_arrows.tpa	Sun Oct  6 15:43:34 2013
+++ stratagems\item\dispelling_arrows.tpa	Sun Jul 24 17:16:01 2016
@@ -2,7 +2,8 @@
   STR_VAR version="none"
 BEGIN
    OUTER_SPRINT sells_arrows ""
-   COPY_EXISTING_REGEXP ".*\sto" "%workspace%"
+   //COPY_EXISTING_REGEXP ".*\sto" "%workspace%"
+   COPY_EXISTING_REGEXP ".*\.\sto" "%workspace%"
         PATCH_IF FILE_CONTAINS_EVALUATED ("%SOURCE_FILE%" "AROW07") BEGIN
            PUSH sells_arrows "%SOURCE_RES%"
         END
