--- stratagems/genai/genai.tpa	2015-01-20 09:15:12.322490100 -0500
+++ Fixpack/stratagems/genai/genai.tpa	2016-11-20 18:06:50.547287900 -0500
@@ -474,7 +474,7 @@
   COPY_EXISTING_REGEXP GLOB ~.*\.cre~ ~override~
       PATCH_SILENT
       LPF FJ_CRE_VALIDITY END
-      PATCH_IF GAME_IS bg2ee BEGIN
+      PATCH_IF GAME_IS ~bg2ee eet~ BEGIN
          LPF FJ_CRE_EFF_V2 END //work around the fact that as of v235 of WEIDU, FJ_CRE_VALIDITY doesn't work on BGIIEE
       END
       SPRINT filename ~%SOURCE_RES%~
