--- stratagems/initial/initial.tpa	2014-12-27 21:22:55.000000000 -0500
+++ Fixpack/stratagems/initial/initial.tpa	2016-11-20 18:06:50.548284300 -0500
@@ -465,13 +465,14 @@
       PATCH_IF !(FILE_CONTAINS ~override/player1.dlg~ ~slayer3~) BEGIN
           INNER_ACTION BEGIN
                 COMPILE ~%scsroot%/%component_loc%/dlg/player1.d~
-                   USING  ~%scsroot%/tra/english/player1.tra~
-                          ~%s/player1.tra~
+                   USING  ~%scsroot%/tra/%eetra%english/player1.tra~
+                          ~%scsroot%/tra/%eetra%%s/player1.tra~
           END
       END
    END
    
-   ADD_JOURNAL EXISTING @314 @315 @316
+   ADD_JOURNAL EXISTING @314 @315 @316 USING ~%scsroot%/tra/%eetra%english/player1.tra~
+											  ~%scsroot%/tra/%eetra%%s/player1.tra~
 END
 
 // rudimentary scripting shift to make enemy-summoned Planetars and Devas avoid healing the party
