--- stratagems/lib/install.tph	2013-11-14 08:42:13.000000000 -0500
+++ Fixpack/stratagems/lib/install.tph	2016-11-20 18:06:50.554276300 -0500
@@ -65,12 +65,12 @@
    ACTION_IF quit=0 BEGIN
     OUTER_SET skip_this=0
     ACTION_IF component=16041 BEGIN
-       ACTION_IF (GAME_IS ~tutu tutu_totsc bgee~) BEGIN
+       ACTION_IF (GAME_IS ~tutu tutu_totsc bgee~) BEGIN
           OUTER_SET skip_this=1
        END
     END
     ACTION_IF component=16040 BEGIN
-       ACTION_IF !(GAME_IS ~tutu tutu_totsc bgt bgee~) BEGIN
+       ACTION_IF !(GAME_IS ~tutu tutu_totsc bgt eet bgee~) BEGIN
           OUTER_SET skip_this=1
        END
     END
