--- stratagems/sfo/general/lib_iterate.tpa	2013-10-28 11:03:46.000000000 -0400
+++ Fixpack/stratagems/sfo/general/lib_iterate.tpa	2016-11-20 18:06:50.558292700 -0500
@@ -27,7 +27,7 @@
 
     RET to_check
 BEGIN
- PATCH_IF GAME_IS ~bg2 tob tutu tutu_totsc bgee bg2ee~ BEGIN
+ PATCH_IF GAME_IS ~bg2 tob tutu tutu_totsc bgee bg2ee eet~ BEGIN
    PATCH_MATCH "%opcode%" WITH
        67 135 151 BEGIN SPRINT to_check "%to_check% %resref%.cre"
        END
@@ -626,8 +626,10 @@
       COUNT_2DA_ROWS 2 rowcount
       FOR (i=0;i<rowcount;i+=1) BEGIN
           READ_2DA_ENTRY i 0 2 idscode
-          LPF idscode_to_spell INT_VAR idscode RET resref END
-          SPRINT list "%list% %resref%.spl"
+          PATCH_IF IS_AN_INT idscode BEGIN
+              LPF idscode_to_spell INT_VAR idscode RET resref END
+              SPRINT list "%list% %resref%.spl"
+          END
       END
    BUT_ONLY
    COPY_EXISTING_REGEXP "\(clab\|lu\).*\.2da" "%workspace%"
