--- stratagems\sfo\general\lib_iterate.tpa	Thu Mar 27 17:55:28 2016
+++ C:\BWP Patchstudio\patched files\stratagems\sfo\general\lib_iterate.tpa	Thu Mar 27 17:55:28 2016
@@ -626,8 +626,10 @@
       COUNT_2DA_ROWS 2 rowcount
       FOR (i=0;i<rowcount;i+=1) BEGIN
           READ_2DA_ENTRY i 0 2 idscode
-          LPF idscode_to_spell INT_VAR idscode RET resref END
-          SPRINT list "%list% %resref%.spl"
+          PATCH_IF IS_AN_INT idscode BEGIN
+            LPF idscode_to_spell INT_VAR idscode RET resref END
+            SPRINT list "%list% %resref%.spl"
+          END 
       END
    BUT_ONLY
    COPY_EXISTING_REGEXP "\(clab\|lu\).*\.2da" "%workspace%"
