--- stratagems/gameplay/move_npcs.tpa	Mon Oct 28 09:34:14 2013
+++ Fixpack/stratagems/gameplay/move_npcs.tpa	Wed Jan 25 22:52:14 2017
@@ -9,7 +9,7 @@
       LAF move_npc_core INT_VAR xloc=753 yloc=872 STR_VAR crename=eldoth area= ~%CoastWay%~ check_for=X#EldothMove END
       LAF move_npc_core INT_VAR xloc=2115 yloc=2203 STR_VAR crename=tiax area= ~%Beregost%~ check_for=X#TiaxMove END
       LAF move_npc_core INT_VAR xloc=1870 yloc=3081 STR_VAR crename=quayle area= ~%NashkelCarnival%~ check_for=X#QuayleMove END
-      LAF swap_text STR_VAR files=~%tutu_var%viconi.dlg~ swaps=~605.901 => 1620.420 507.806=> 1520.320~ END
+      LAF swap_text STR_VAR files=~%tutu_var%viconi%eet_var%.dlg~ swaps=~605.901 => 1620.420 507.806=> 1520.320~ END
    END
    ACTION_IF enhanced_edition BEGIN
       LAF move_npc_sharteel_ee END
