--- stratagems/sarevok/sarevok.tpa	2013-10-27 14:21:59.000000000 -0400
+++ Fixpack/stratagems/sarevok/sarevok.tpa	2016-11-20 18:06:50.555276000 -0500
@@ -52,15 +52,17 @@
 
      LAF extend_area_script STR_VAR area=~%Undercity%~ top=undercityadd location=resource END
 
-//end-of-game script - not needed for BGT
+//end-of-game script - not needed for BGT and EET
 
-     ACTION_IF !(GAME_IS bgt) BEGIN
+     ACTION_IF !(GAME_IS ~bgt eet~) BEGIN
            LAF extend_area_script STR_VAR area=~%Undercity_TempleofBhaal%~ top=dw#end location=resource END
      END
 
 // Sarevok's script
      ACTION_IF GAME_IS bgt BEGIN
         OUTER_SPRINT sarevok_script bgsarvok
+     END ELSE ACTION_IF GAME_IS eet BEGIN
+        OUTER_SPRINT sarevok_script sarevok_
      END ELSE BEGIN
         OUTER_SPRINT sarevok_script ~%tutu_var%sarevok~
      END
@@ -83,22 +85,22 @@
          add_items=>~dw#angar(x20) %tutu_var%bow01~
          enforce_charclass=>is_bg1
       END
-      LAF clone_creature STR_VAR creature= ~%tutu_scriptbg%skelwa=>dw#angsk~ END
+      LAF clone_creature STR_VAR creature= ~%tutu_scriptbg%skelwa%eet_var%=>dw#angsk~ END
       MAKE_PATCH
          add_items=>~%tutu_var%arow04(x20) %tutu_var%bow01~
          enforce_charclass=>is_bg1
       END
-      LAF clone_creature STR_VAR creature=~%tutu_scriptbg%skelwa=>dw#diask~ END
+      LAF clone_creature STR_VAR creature=~%tutu_scriptbg%skelwa%eet_var%=>dw#diask~ END
       MAKE_PATCH
          replace_items=>~%tutu_var%staf01~
          enforce_charclass=>is_bg1
       END
-      LAF clone_creature STR_VAR creature= ~%tutu_scriptbg%skelwa=>dw#semsk~ END
+      LAF clone_creature STR_VAR creature= ~%tutu_scriptbg%skelwa%eet_var%=>dw#semsk~ END
 
       //Skeleton-warrior-creating spells [slightly clunky code]
 
       LAF install STR_VAR file=~dw#tazsk.spl~ location=resource END
-      COPY_EXISTING ~dw#tazsk.spl~ override WRITE_EVALUATED_ASCII 0xae ~%tutu_scriptbg%skelwa~ #8
+      COPY_EXISTING ~dw#tazsk.spl~ override WRITE_EVALUATED_ASCII 0xae ~%tutu_scriptbg%skelwa%eet_var%~ #8
       ACTION_FOR_EACH skeleton IN dw#angsk dw#semsk dw#diask BEGIN
          COPY_EXISTING ~dw#tazsk.spl~ ~override/%skeleton%.spl~ WRITE_EVALUATED_ASCII 0xae ~%skeleton%~ #8
       END
