--- stratagems/setup-stratagems_autoinstall.tp2	2013-11-24 17:53:08.000000000 -0500
+++ Fixpack/stratagems/setup-stratagems_autoinstall.tp2	2016-11-20 18:06:50.557292200 -0500
@@ -5,6 +5,19 @@
 ////	Language commands (direct from stratagems)
 //////////////////////////////////////////////////////////////////////
 
+ALWAYS
+  ACTION_IF GAME_IS ~eet~ BEGIN
+    OUTER_SET bg2_chapter = 12
+  END ELSE BEGIN
+    OUTER_SET bg2_chapter = 0
+  END
+  OUTER_FOR (i=1; i<=10; i=i+1) BEGIN
+    OUTER_SET bg2_chapter = bg2_chapter + 1
+    OUTER_SPRINT name_source ~bg2_chapter_%i%~
+    OUTER_SET EVAL ~%name_source%~ = bg2_chapter
+  END
+END
+
 LANGUAGE
 "English"
 ENGLISH
