--- stratagems/setup-stratagems.tp2	2015-01-20 10:27:56.406546400 -0500
+++ Fixpack/stratagems/setup-stratagems.tp2	2016-11-20 18:06:50.557292200 -0500
@@ -1,12 +1,23 @@
 BACKUP ~stratagems_external/backup/stratagems~
 AUTHOR ~For help troubleshooting installation problems, go to the Sword Coast Stratagems forum at forums.gibberlings3.net.~
-VERSION ~v30~
+VERSION ~v30 BWP fix + K4thos' EET compatibility~
 README ~stratagems/readme-stratagems.html~
 AUTO_EVAL_STRINGS
 
 ///---/// ASK_EVERY_COMPONENT
 
 ALWAYS
+  ACTION_IF GAME_IS ~eet~ BEGIN
+    OUTER_SET bg2_chapter = 12
+  END ELSE BEGIN
+    OUTER_SET bg2_chapter = 0
+  END
+  OUTER_FOR (i=1; i<=10; i=i+1) BEGIN
+    OUTER_SET bg2_chapter = bg2_chapter + 1
+    OUTER_SPRINT name_source ~bg2_chapter_%i%~
+    OUTER_SET EVAL ~%name_source%~ = bg2_chapter
+  END
+
    OUTER_SET debug_variable=0
    OUTER_SPRINT scsroot stratagems
    OUTER_SPRINT workspace ~stratagems_external/workspace~
@@ -143,7 +154,7 @@
 BEGIN @11 DESIGNATED 2000 GROUP @2
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE !FILE_EXISTS ~override/dvimhere.mrk~ @3352
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#spellstrikevsPM END
 
@@ -197,7 +208,7 @@
 
 REQUIRE_PREDICATE !FILE_EXISTS_IN_GAME ~dvimhere.mrk~ @3352
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#harm1 END
 
@@ -221,7 +232,7 @@
 BEGIN @18 DESIGNATED 2050 GROUP @2
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE !FILE_EXISTS ~tobex.dll~ @16045
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 
 LAF make_label STR_VAR label=dw#spellimm END
@@ -236,8 +247,8 @@
 BEGIN @19 DESIGNATED 2060 GROUP @2
 REQUIRE_PREDICATE !FILE_EXISTS_IN_GAME ~dvimhere.mrk~ @3352
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
-REQUIRE_PREDICATE !GAME_IS bg2ee @15999
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
+REQUIRE_PREDICATE !GAME_IS ~bg2ee eet~ @15999
 
 
 LAF make_label STR_VAR label=dw#restoration END
@@ -252,7 +263,7 @@
 BEGIN @20 DESIGNATED 2070 GROUP @2
 REQUIRE_PREDICATE !FILE_EXISTS_IN_GAME ~dvimhere.mrk~ @3352
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#blades END
 
@@ -266,7 +277,7 @@
 BEGIN @21 DESIGNATED 2080 GROUP @2
 REQUIRE_PREDICATE !FILE_EXISTS_IN_GAME ~dvimhere.mrk~ @3352
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#skulltrap END
 
@@ -280,7 +291,7 @@
 BEGIN @23  DESIGNATED 2090 GROUP @2
 REQUIRE_PREDICATE !FILE_EXISTS_IN_GAME ~dvimhere.mrk~ @3352
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#PWblind END
 
@@ -294,7 +305,7 @@
 BEGIN @22 DESIGNATED 2100 GROUP @2
 REQUIRE_PREDICATE !FILE_EXISTS_IN_GAME ~dvimhere.mrk~ @3352
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#meteor END
 
@@ -359,7 +370,7 @@
 BEGIN @3353 DESIGNATED 2140 GROUP @2
 REQUIRE_PREDICATE !FILE_EXISTS_IN_GAME ~dvimhere.mrk~ @3352
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#mantle  END
 
@@ -386,7 +397,7 @@
 
 BEGIN @3387 DESIGNATED 2160 GROUP @2
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 
 LAF make_label STR_VAR label=dw#extra_scrolls  END
@@ -400,7 +411,7 @@
 
 BEGIN @3392 DESIGNATED 2170 GROUP @2
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#truesight_blindness  END
 
@@ -414,7 +425,7 @@
 BEGIN @220 DESIGNATED 2180 GROUP @2
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE !FILE_EXISTS ~override/dvimhere.mrk~ @3352
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#clone_slots  END
 
@@ -448,7 +459,7 @@
 
 BEGIN @16003 DESIGNATED 3000 GROUP @3
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE (GAME_IS ~tutu tutu_totsc bgee~) @16002
+REQUIRE_PREDICATE (GAME_IS ~tutu tutu_totsc bgee sod~) @16002
 
 LAF make_label STR_VAR label=dw#elemental_arrows  END
 
@@ -474,7 +485,7 @@
 BEGIN @35 DESIGNATED 3020 GROUP @3 // BG2 only
 
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE !GAME_IS ~bgt tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~bgt eet tutu tutu_totsc bgee sod~ @16000
 REQUIRE_PREDICATE !(MOD_IS_INSTALLED ~item_rev/item_rev.tp2~ 0) @3376
 
 
@@ -487,7 +498,7 @@
 
 BEGIN @20073 DESIGNATED 3021 GROUP @3 SUBCOMPONENT @36 // no-shatter
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE GAME_IS ~bgt tutu tutu_totsc bgee~ @16001
+REQUIRE_PREDICATE GAME_IS ~bgt eet tutu tutu_totsc bgee sod~ @16001
 REQUIRE_PREDICATE !(MOD_IS_INSTALLED ~item_rev/item_rev.tp2~ 0) @3376
 
 LAF make_label STR_VAR label=dw#fine_weapons_no_shatter  END
@@ -510,7 +521,7 @@
 
 BEGIN @20008 DESIGNATED 3030 GROUP @3 // BG2 only
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc~  || FILE_EXISTS_IN_GAME ~dorn.cre~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#extra_healing  END
 
@@ -545,7 +556,7 @@
 BEGIN @31 DESIGNATED 3050 GROUP @3
 REQUIRE_PREDICATE (MOD_IS_INSTALLED ~item_rev/item_rev.tp2~ 0)=0 @3377
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#balduran END
 
@@ -559,7 +570,7 @@
 BEGIN @32 DESIGNATED 3060 GROUP @3
 REQUIRE_PREDICATE (MOD_IS_INSTALLED ~item_rev/item_rev.tp2~ 0)=0 @3376
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#magistaff END
 
@@ -573,7 +584,7 @@
 BEGIN @33 DESIGNATED 3070 GROUP @3
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE (MOD_IS_INSTALLED ~item_rev/item_rev.tp2~ 0)=0 @3376
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#vhailor END
 
@@ -587,7 +598,7 @@
 BEGIN @198 DESIGNATED 3080 GROUP @3
 REQUIRE_PREDICATE (MOD_IS_INSTALLED ~item_rev/item_rev.tp2~ 0)=0 @3376
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#mirror END
 
@@ -601,7 +612,7 @@
 BEGIN @34 DESIGNATED 3090 GROUP @3
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE (MOD_IS_INSTALLED ~item_rev/item_rev.tp2~ 0)=0 @3376
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#vecna END
 
@@ -615,7 +626,7 @@
 BEGIN @3354 DESIGNATED 3100 GROUP @3
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE (MOD_IS_INSTALLED ~item_rev/item_rev.tp2~ 0)=0 @3376
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#resurrectrod END
 
@@ -629,7 +640,7 @@
 BEGIN @3362 DESIGNATED 3110 GROUP @3
 REQUIRE_PREDICATE (MOD_IS_INSTALLED ~item_rev/item_rev.tp2~ 0)=0 @3376
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#carsomyr END
 
@@ -673,7 +684,7 @@
 ////////////////////////////////////////////////////////////////////////////////////////
 BEGIN @24118 DESIGNATED 4020 GROUP @4
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#wolves END
 
@@ -757,8 +768,8 @@
 
 BEGIN @20010 DESIGNATED 4100 GROUP @4
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
-REQUIRE_PREDICATE !GAME_IS bgee ~This component isn't quite ready for BG:EE yet~
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
+REQUIRE_PREDICATE !GAME_IS ~bgee sod eet~ ~This component isn't quite ready for BG:EE or EET yet~
 
 LAF make_label STR_VAR label=dw#npc_profs END
 
@@ -771,7 +782,7 @@
 
 BEGIN @20011 DESIGNATED 4110 GROUP @4
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#npc_profs END
 
@@ -785,7 +796,7 @@
 BEGIN @20012 DESIGNATED 4120 GROUP @4
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE !FILE_EXISTS ~override/X#WaitAtInn.G3~ @20066
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#npc_profs END
 
@@ -797,7 +808,7 @@
 ///////////////////////////////////////////////////////////////////
 BEGIN @20016 DESIGNATED 4130 GROUP @4
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 
 LAF make_label STR_VAR label=dw#npc_start_loc END
@@ -811,7 +822,7 @@
 
 BEGIN @20014 DESIGNATED 4140 GROUP @4
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#yeslick_axe END
 
@@ -824,7 +835,7 @@
 
 BEGIN @16029 DESIGNATED 4145  GROUP @4  SUBCOMPONENT  @16028
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#candleskip END
 
@@ -847,7 +858,7 @@
 
 BEGIN @3372 DESIGNATED 4150 GROUP @4
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#licenseextension END
 
@@ -860,7 +871,7 @@
 
 BEGIN @3367  DESIGNATED 4160 GROUP @4 SUBCOMPONENT @3366
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#licensecost END
 
@@ -910,7 +921,7 @@
 
 BEGIN @42 DESIGNATED 4170 GROUP @4 SUBCOMPONENT @41
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 LAF make_label STR_VAR label=dw#spellholdcost END
 
 OUTER_SPRINT component_loc gameplay
@@ -954,7 +965,7 @@
 
 BEGIN @196 DESIGNATED 4180 GROUP @4
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#freedomscroll END
 
@@ -968,8 +979,8 @@
 BEGIN @49 DESIGNATED 4190 GROUP @4
 
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
-REQUIRE_PREDICATE !GAME_IS bg2ee @15999
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
+REQUIRE_PREDICATE !GAME_IS ~bg2ee eet~ @15999
 REQUIRE_PREDICATE !FILE_EXISTS_IN_GAME ~dw#hellloot.mrk~ @219
 
 LAF make_label STR_VAR label=dw#wkearly END
@@ -984,8 +995,8 @@
 
 BEGIN @193 DESIGNATED 4200 GROUP @4
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
-REQUIRE_PREDICATE !GAME_IS bg2ee @15999
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
+REQUIRE_PREDICATE !GAME_IS ~bg2ee eet~ @15999
 
 LAF make_label STR_VAR label=dw#hellloot END
 
@@ -998,7 +1009,7 @@
 
 BEGIN @3365 DESIGNATED 4210 GROUP @4
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#maze END
 
@@ -1011,7 +1022,7 @@
 
 BEGIN @47 DESIGNATED 4215 GROUP @4
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 LAF make_label STR_VAR label=dw#nochimp END
 
 OUTER_SPRINT component_loc gameplay
@@ -1023,7 +1034,7 @@
 
 BEGIN @3321 DESIGNATED 4216 SUBCOMPONENT @48 GROUP @4
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#noammo END
 
@@ -1057,7 +1068,7 @@
 
 BEGIN @3373 DESIGNATED 4230 GROUP @4
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#bonus END
 
@@ -1070,7 +1081,7 @@
 
 BEGIN @3397 DESIGNATED 4240 GROUP @4
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#hla_innate END
 
@@ -1125,7 +1136,7 @@
 
 BEGIN @50 DESIGNATED 5030 GROUP @5
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#mirrorcloak END
 
@@ -1140,7 +1151,7 @@
 
 BEGIN @20007 DESIGNATED 5050  GROUP @5
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#ankhegstack END
 
@@ -1154,7 +1165,7 @@
 BEGIN @20071 DESIGNATED 5060  GROUP @5
 /// --- /// DEFAULT_SKIP
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#protectshar END
 
@@ -1214,7 +1225,7 @@
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#genai.mrk~ @3350
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 
 LAF make_label STR_VAR label=dw#hla END
@@ -1452,7 +1463,7 @@
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#genai.mrk~ @3350
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#dryad END
 
@@ -1467,7 +1478,7 @@
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#genai.mrk~ @3350
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#carrion END
 
@@ -1482,7 +1493,7 @@
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#genai.mrk~ @3350
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#basilisk END
 
@@ -1504,7 +1515,7 @@
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#genai.mrk~ @3350
-REQUIRE_PREDICATE !(GAME_IS ~tutu tutu_totsc bgee~) @16000
+REQUIRE_PREDICATE !(GAME_IS ~tutu tutu_totsc bgee sod~) @16000
 
 LAF make_label STR_VAR label=dw#golem END
 
@@ -1519,7 +1530,7 @@
 BEGIN @16032 DESIGNATED 6510 GROUP @6 SUBCOMPONENT @63
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#fiend END
 
@@ -1542,7 +1553,7 @@
 BEGIN @16051 DESIGNATED 6520 GROUP @6 SUBCOMPONENT @69
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#genie END
 
@@ -1568,7 +1579,7 @@
 BEGIN @16034 DESIGNATED 6530 GROUP @6 SUBCOMPONENT @66
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#celestial END
 
@@ -1591,7 +1602,7 @@
 BEGIN @16036 DESIGNATED 6540 GROUP @6 SUBCOMPONENT @72
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#dragon END
 
@@ -1614,7 +1625,7 @@
 BEGIN @3304 DESIGNATED 6550 GROUP @6 SUBCOMPONENT @75
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#beholder END
 
@@ -1652,7 +1663,7 @@
 BEGIN @175 DESIGNATED 6560 GROUP @6 SUBCOMPONENT @78  // inv-by-script
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#flayer END
 
@@ -1693,7 +1704,7 @@
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#genai.mrk~ @3350
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#githyanki END
 
@@ -1707,7 +1718,7 @@
 BEGIN @187 DESIGNATED 6580 GROUP @6
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#vampire END
 
@@ -1722,7 +1733,7 @@
 BEGIN @3209 DESIGNATED 6590 GROUP @6
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#melissan END
 
@@ -1744,7 +1755,7 @@
 BEGIN @3306 DESIGNATED 6800 GROUP @6
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc~ @16000 REQUIRE_PREDICATE !FILE_EXISTS_IN_GAME ~dorn.cre~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 REQUIRE_PREDICATE FILE_EXISTS_IN_GAME ~illas01.itm~ @3309
 
 
@@ -1760,7 +1771,7 @@
 BEGIN @3312 DESIGNATED 6810 GROUP @6
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc~ @16000 REQUIRE_PREDICATE !FILE_EXISTS_IN_GAME ~dorn.cre~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 REQUIRE_PREDICATE (FILE_EXISTS_IN_GAME ~jumjum.bcs~ || FILE_EXISTS_IN_GAME ~fingrom.cre~) @3313
 
 
@@ -1777,7 +1788,7 @@
 BEGIN @3307 DESIGNATED 6820 GROUP @6
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc~ @16000 REQUIRE_PREDICATE !FILE_EXISTS_IN_GAME ~dorn.cre~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 REQUIRE_PREDICATE (FILE_EXISTS_IN_GAME ~yagaft.bcs~ || FILE_EXISTS_IN_GAME finyaga.cre) @3310
 
 
@@ -1793,7 +1804,7 @@
 BEGIN @3308 DESIGNATED 6830 GROUP @6
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc~ @16000 REQUIRE_PREDICATE !FILE_EXISTS_IN_GAME ~dorn.cre~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 REQUIRE_PREDICATE (FILE_EXISTS_IN_GAME ~abazdg02.cre~ || FILE_EXISTS_IN_GAME ~finabaz.cre~) @3314
 
 
@@ -1809,7 +1820,7 @@
 BEGIN @3324 DESIGNATED 6840 GROUP @6
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc~ @16000 REQUIRE_PREDICATE !FILE_EXISTS_IN_GAME ~dorn.cre~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 REQUIRE_PREDICATE FILE_EXISTS_IN_GAME ~fingrom.cre~ @3314
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#mage.mrk~ @16046
 
@@ -1826,7 +1837,7 @@
 BEGIN @3315 DESIGNATED 6850 GROUP @6
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc~ @16000 REQUIRE_PREDICATE !FILE_EXISTS_IN_GAME ~dorn.cre~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 REQUIRE_PREDICATE FILE_EXISTS_IN_GAME ~fingrom.cre~ @3314
 
 
@@ -1849,7 +1860,7 @@
 BEGIN @20031 DESIGNATED 7000 GROUP @16040
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#doppelganger END
 
@@ -1863,7 +1874,7 @@
 BEGIN @20033 DESIGNATED 7010 GROUP @16040
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#blacktalon END
 
@@ -1877,7 +1888,7 @@
 BEGIN @20070 DESIGNATED 7020 GROUP @16040
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#c3assassin END
 
@@ -1891,7 +1902,7 @@
 BEGIN @20027 DESIGNATED 7030 GROUP @16040
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#kobold END
 
@@ -1905,7 +1916,7 @@
 BEGIN @20034 DESIGNATED 7040 GROUP @16040
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#ogreassassin END
 
@@ -1919,7 +1930,7 @@
 BEGIN @20035 DESIGNATED 7050 GROUP @16040
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#ulcaster END
 
@@ -1933,7 +1944,7 @@
 BEGIN @20036 DESIGNATED 7060 GROUP @16040
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#werewolf END
 
@@ -1947,7 +1958,7 @@
 BEGIN @20037 DESIGNATED 7070 GROUP @16040
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#durlag END
 
@@ -1961,7 +1972,7 @@
 BEGIN @20038 DESIGNATED 7080 GROUP @16040
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#demoncult END
 
@@ -1976,7 +1987,7 @@
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#priest.mrk~ @16047
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#druid END
 
@@ -1990,7 +2001,7 @@
 BEGIN @20090 DESIGNATED 7100 GROUP @16040
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#bassilus END
 
@@ -2004,7 +2015,7 @@
 BEGIN @20091 DESIGNATED 7110 GROUP @16040
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#drasus END
 
@@ -2019,7 +2030,7 @@
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#mage.mrk~ @16046
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#redwizard END
 
@@ -2033,7 +2044,7 @@
 BEGIN @20093 DESIGNATED 7140 GROUP @16040
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#c7throne END
 
@@ -2047,7 +2058,7 @@
 BEGIN @20042 DESIGNATED 7200 GROUP @16040
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#c2end END
 
@@ -2061,7 +2072,7 @@
 BEGIN @20043 DESIGNATED 7210 GROUP @16040
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#c3end END
 
@@ -2075,7 +2086,7 @@
 BEGIN @20044 DESIGNATED 7220 GROUP @16040
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#c4end END
 
@@ -2089,7 +2100,7 @@
 BEGIN @20045 DESIGNATED 7230 GROUP @16040
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#c5end END
 
@@ -2103,7 +2114,7 @@
 BEGIN @20047 DESIGNATED 7250 GROUP @16040
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#c7end END
 
@@ -2117,7 +2128,7 @@
 BEGIN @20039 DESIGNATED 7900 GROUP @16040
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE (GAME_IS ~bgt tutu tutu_totsc bgee~) @16001
+REQUIRE_PREDICATE (GAME_IS ~bgt eet tutu tutu_totsc bgee sod~) @16001
 
 LAF make_label STR_VAR label=dw#bg1minor END
 
@@ -2132,7 +2143,7 @@
 BEGIN @3334 DESIGNATED 8000 GROUP @16041
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#startingdungeon END
 
@@ -2146,7 +2157,7 @@
 BEGIN @172 DESIGNATED 8010 GROUP @16041
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#shadelord END
 
@@ -2162,7 +2173,7 @@
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#mage.mrk~ @16046
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#demilich END
 
@@ -2176,7 +2187,7 @@
 BEGIN @182 DESIGNATED 8030 GROUP @16041
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#resilienttrolls END
 
@@ -2190,7 +2201,7 @@
 BEGIN @155 DESIGNATED 8040 GROUP @16041 SUBCOMPONENT @154
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#spawns END
 
@@ -2231,7 +2242,7 @@
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
 
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#random END
 
@@ -2245,7 +2256,7 @@
 BEGIN @178 DESIGNATED 8060 GROUP @16041 SUBCOMPONENT @158
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#darnise END
 
@@ -2269,7 +2280,7 @@
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#beholder.mrk~ @3399
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#unseeing END
 
@@ -2284,7 +2295,7 @@
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#vampire.mrk~ @16049
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#bodhi END
 
@@ -2305,7 +2316,7 @@
 
 BEGIN @181 DESIGNATED 8090 GROUP @16041
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#spellholditems END
 
@@ -2319,7 +2330,7 @@
 BEGIN @3357 DESIGNATED 8100 GROUP @16041
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#spellholdirenicus END
 
@@ -2333,7 +2344,7 @@
 BEGIN @194 DESIGNATED 8110 GROUP @16041
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#sahuagin END
 
@@ -2347,7 +2358,7 @@
 BEGIN @3317 DESIGNATED 8120 GROUP @16041
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#beholderhive END
 
@@ -2361,7 +2372,7 @@
 BEGIN @3318 DESIGNATED 8130 GROUP @16041
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#illithidrest END
 
@@ -2375,7 +2386,7 @@
 BEGIN @3319 DESIGNATED 8140 GROUP @16041 SUBCOMPONENT @199
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#drow END
 
@@ -2398,7 +2409,7 @@
 BEGIN @200 DESIGNATED 8150 GROUP @16041
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#wk END
 
@@ -2412,7 +2423,7 @@
 BEGIN @180 DESIGNATED 8160 GROUP @16041
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#firegiant END
 
@@ -2426,7 +2437,7 @@
 BEGIN @161 DESIGNATED 8170 GROUP @16041
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#sendai END
 OUTER_SPRINT component_loc sendai
@@ -2439,7 +2450,7 @@
 BEGIN @173 DESIGNATED 8180 GROUP @16041
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#abazigal END
 
@@ -2453,7 +2464,7 @@
 BEGIN @3203 DESIGNATED 8190 GROUP @16041
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#AIsetup.mrk~ @16025
-REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
+REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee sod~ @16000
 
 LAF make_label STR_VAR label=dw#bg2minor END
 
