--- stratagems\setup-stratagems.tp2	Tue Jan 20 10:27:56 2015
+++ C:\BWP Patchstudio\patched files\stratagems\setup-stratagems.tp2	Fri Dec 18 23:48:38 2015
@@ -1,6 +1,6 @@
 BACKUP ~stratagems_external/backup/stratagems~
 AUTHOR ~For help troubleshooting installation problems, go to the Sword Coast Stratagems forum at forums.gibberlings3.net.~
-VERSION ~v30~
+VERSION ~v30 BWP Fix~
 README ~stratagems/readme-stratagems.html~
 AUTO_EVAL_STRINGS
 
