--- stratagems/help/baf/shout.baf	2013-08-15 12:05:29.000000000 -0400
+++ Fixpack/stratagems/help/baf/shout.baf	2016-11-20 18:06:50.548284300 -0500
@@ -63,6 +63,33 @@
 
 IF
 	Class(Myself,INNOCENT)
+	Heard([NOTEVIL],99)
+	OR(6)
+		Alignment(Myself,LAWFUL_GOOD)
+		Alignment(Myself,CHAOTIC_GOOD)
+		Alignment(Myself,NEUTRAL_GOOD)
+		Alignment(Myself,LAWFUL_NEUTRAL)
+		Alignment(Myself,CHAOTIC_NEUTRAL)
+		Alignment(Myself,NEUTRAL)
+THEN
+	RESPONSE #100
+		Enemy()
+END
+
+IF
+	Class(Myself,INNOCENT)
+	Heard([NOTGOOD],99)
+	OR(3)
+		Alignment(Myself,CHAOTIC_EVIL)
+		Alignment(Myself,NEUTRAL_EVIL)
+		Alignment(Myself,LAWFUL_EVIL)
+THEN
+	RESPONSE #100
+		Enemy()
+END
+
+IF
+	Class(Myself,INNOCENT)
 	Heard([ANYONE],99)
 THEN
 	RESPONSE #100
