--- stratagems/tactical_bg2/spawn.tpa	2013-10-21 01:23:37.000000000 -0400
+++ Fixpack/stratagems/tactical_bg2/spawn.tpa	2016-11-20 18:06:50.587299100 -0500
@@ -8,11 +8,11 @@
      ACTION_MATCH ~%version%~ WITH
      1 BEGIN  // slight increase
          LAF spawn_xp INT_VAR one_million=400002 two_million=1000002 STR_VAR files=~%general_spawns%~  END
-         LAF spawn_c6 STR_VAR spawn_chapter_level=~OR(2)Global("Chapter","GLOBAL",6)XPGT(Player1,800000)~ END
+         LAF spawn_c6 STR_VAR spawn_chapter_level= EVAL ~OR(2)Global("Chapter","GLOBAL",%bg2_chapter_6%)XPGT(Player1,800000)~ END
      END
      2 BEGIN // significant increase
          LAF spawn_xp INT_VAR one_million=1 two_million=400002 STR_VAR files=~%general_spawns%~  END
-         LAF spawn_c6 STR_VAR spawn_chapter_level=~OR(2)Global("Chapter","GLOBAL",6)XPGT(Player1,400000)~ END
+         LAF spawn_c6 STR_VAR spawn_chapter_level= EVAL ~OR(2)Global("Chapter","GLOBAL",%bg2_chapter_6%)XPGT(Player1,400000)~ END
      END
      3 BEGIN  // almost-maximum increase
          LAF spawn_xp INT_VAR one_million=1 two_million=3 STR_VAR files=~%general_spawns%~  END
@@ -71,6 +71,6 @@
       LAF swap_text
           STR_VAR files=~c6behold c6goljug c6golsto c6mindul c6rakraj c6vamp c6vamver c6weregr~
                   postfix=bcs
-                  swaps=~Global("Chapter","GLOBAL",6)=>%spawn_chapter_level%~
+                  swaps= EVAL ~Global("Chapter","GLOBAL",%bg2_chapter_6%)=>%spawn_chapter_level%~
       END
 END
