--- stratagems/gameplay/wk_early.tpa	2014-12-25 23:08:35.000000000 -0500
+++ Fixpack/stratagems/gameplay/wk_early.tpa	2016-11-20 18:06:50.546287000 -0500
@@ -169,17 +169,17 @@
 	
 		IF
 			GlobalGT("TearDoor","GLOBAL",0) // been in Hell
-			!GlobalGT("Chapter","GLOBAL",7)
+			!GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)
 		THEN
 			RESPONSE #100
-				SetGlobal("Chapter","GLOBAL",8)
+				SetGlobal("Chapter","GLOBAL",%bg2_chapter_8%)
 				EraseJournalEntry(4352) // Finding Irenicus in Hell  Irenicus' death did not have the same effect as Bodhi's did, and my soul has not returned to me.  Perhaps it was because of the power he gathered from the Tree of Life, or because Bodhi was undead. Regardless, Irenicus has fallen, dragging both my soul and myself inexorably behind.  I am now alone in what must be the Nine Hells, but I am certain Irenicus is here somewhere... and somehow, in this, I must find the mage and battle him once and for all to get my soul back.
 				Continue()
 		END
 	
 		>>>>>>>>
 	
-		EXTEND_TOP ~ar3000.bcs~ ~.../stratagems-inline/ar3000add.baf~			
+		EXTEND_TOP ~ar3000.bcs~ ~.../stratagems-inline/ar3000add.baf~ EVALUATE_BUFFER			
 	
 		COPY_EXISTING 
 			~cut100a.bcs~ ~override~
