--- setup-d0tweak.tp2	Tue Dec 25 01:12:10 2007
+++ C:\BWP Patchstudio\patched files\setup-d0tweak.tp2	Wed Oct 24 04:07:11 2012
@@ -124,6 +124,8 @@
   ~WORGSU.CRE~ 
   ~XVARTSU.CRE~
 
+VERSION ~v20 BWP Fix~
+
 ALWAYS
 
   COPY_EXISTING_REGEXP GLOB ~^anomen[0-9]*\.cre~ ~override~
@@ -169,7 +171,7 @@
     REPLACE_TEXTUALLY ~.*\bMAGESCHOOL_INVOKER\b~     ~0x0800 MAGESCHOOL_INVOKER~
     REPLACE_TEXTUALLY ~.*\bMAGESCHOOL_NECROMANCER\b~ ~0x1000 MAGESCHOOL_NECROMANCER~
     REPLACE_TEXTUALLY ~.*\bMAGESCHOOL_TRANSMUTER\b~  ~0x2000 MAGESCHOOL_TRANSMUTER~
-    REPLACE_TEXTUALLY ~.*\bMAGESCHOOL_GENERALIST\b~  ~0x0000 WILDMAGE~
+    REPLACE_TEXTUALLY ~.*\bMAGESCHOOL_GENERALIST\b~  ~0x4000 MAGESCHOOL_GENERALIST~
     REPLACE_TEXTUALLY ~.*\bFERALAN\b~                ~0x4007 FERALAN~
     REPLACE_TEXTUALLY ~.*\bSTALKER\b~                ~0x4008 STALKER~
     REPLACE_TEXTUALLY ~.*\bBEASTMASTER\b~            ~0x4009 BEASTMASTER~
@@ -185,15 +187,15 @@
     REPLACE_TEXTUALLY ~.*\bTOTEMIC\b~                ~0x4010 TOTEMIC~
     REPLACE_TEXTUALLY ~.*\bSHAPESHIFTER\b~           ~0x4011 SHAPESHIFTER~
     REPLACE_TEXTUALLY ~.*\bBEASTFRIEND\b~            ~0x4012 BEASTFRIEND~
-    REPLACE_TEXTUALLY ~.*\bBARBARIAN\b~              ~0x0000 BARBARIAN~
-    REPLACE_TEXTUALLY ~.*\bWILDMAGE\b~               ~0x0000 WILDMAGE~
+    REPLACE_TEXTUALLY ~.*\bBARBARIAN\b~              ~0x40000000 BARBARIAN~
+    REPLACE_TEXTUALLY ~.*\bWILDMAGE\b~               ~0x80000000 WILDMAGE~
   BUT_ONLY_IF_IT_CHANGES
 
   APPEND ~kit.ids~ ~0x4000 TRUECLASS~
     UNLESS ~\bTRUECLASS\b~
-  APPEND ~kit.ids~ ~0x0000 BARBARIAN~
+  APPEND ~kit.ids~ ~0x40000000 BARBARIAN~
     UNLESS ~\bBARBARIAN\b~
-  APPEND ~kit.ids~ ~0x0000 WILDMAGE~
+  APPEND ~kit.ids~ ~0x80000000 WILDMAGE~
     UNLESS ~\bWILDMAGE\b~
 
   APPEND ~state.ids~ ~0x00000FC0 STATE_REALLY_DEAD~ UNLESS ~STATE_REALLY_DEAD~
@@ -453,10 +455,10 @@
     END ELSE
 	PATCH_IF ("%opcode%" = 71)	// Sex change
 	THEN BEGIN
-	  READ_LONG ("%fx_off%" + 0x04 + ("%index2%" * 0x30)) "IDS_entry"
+	  READ_LONG ("%fx_off%" + 0x04 + ("%index%" * 0x30)) "IDS_entry"
 	  PATCH_IF ("%IDS_entry%" = 6)
 	  THEN BEGIN
-	    WRITE_LONG ("%fx_off%" + 0x04 + ("%index2%" * 0x30)) 20
+	    WRITE_LONG ("%fx_off%" + 0x04 + ("%index%" * 0x30)) 20
 	  END
 	END
   END
@@ -667,7 +669,7 @@
   END
   WHILE (("%ddoor_added%" = 0) AND ("%ddoor_insert%" > 0)) BEGIN
     INSERT_BYTES ("%sale_offset%" + ("%ddoor_insert%" * 0x1C)) 28
-    WRITE_ASCII ("%sale_offset%" +        ("%ddoor_insert%" * 0x1C)) ~D0SCRLDD~
+    WRITE_ASCII ("%sale_offset%" +        ("%ddoor_insert%" * 0x1C)) ~D0SCRLDD~ #8
     WRITE_LONG  ("%sale_offset%" + 0xa + ("%ddoor_insert%" * 0x1C)) 1 // stacks of 1
     WRITE_LONG  ("%sale_offset%" + 0x10 + ("%ddoor_insert%" * 0x1C)) 1 // identified
     WRITE_LONG  ("%sale_offset%" + 0x14 + ("%ddoor_insert%" * 0x1C)) 3 // quantity
@@ -713,7 +715,7 @@
   END
   WHILE (("%ddoor_added%" = 0) AND ("%ddoor_insert%" > 0)) BEGIN
     INSERT_BYTES ("%sale_offset%" + ("%ddoor_insert%" * 0x1C)) 28
-    WRITE_ASCII ("%sale_offset%" +        ("%ddoor_insert%" * 0x1C)) ~D0SCRLDD~
+    WRITE_ASCII ("%sale_offset%" +        ("%ddoor_insert%" * 0x1C)) ~D0SCRLDD~ #8
     WRITE_LONG  ("%sale_offset%" + 0xa + ("%ddoor_insert%" * 0x1C)) 1 // stacks of 1
     WRITE_LONG  ("%sale_offset%" + 0x10 + ("%ddoor_insert%" * 0x1C)) 1 // identified
     WRITE_LONG  ("%sale_offset%" + 0x14 + ("%ddoor_insert%" * 0x1C)) 4 // quantity
@@ -904,24 +906,24 @@
      ~d0tweak/ioun/d0io9.bam~ ~override~
 
 COPY ~d0tweak/ioun/sporbit.vvc~ ~override/d0io1.vvc~
-WRITE_ASCII 0x8 "D0IO1"
+WRITE_ASCII 0x8 "D0IO1" #8
 WRITE_BYTE 0x1a 0x8
 COPY ~d0tweak/ioun/sporbit.vvc~ ~override/d0io2.vvc~
-WRITE_ASCII 0x8 "D0IO2"
+WRITE_ASCII 0x8 "D0IO2" #8
 COPY ~d0tweak/ioun/sporbit.vvc~ ~override/d0io3.vvc~
-WRITE_ASCII 0x8 "D0IO3"
+WRITE_ASCII 0x8 "D0IO3" #8
 COPY ~d0tweak/ioun/sporbit.vvc~ ~override/d0io4.vvc~
-WRITE_ASCII 0x8 "D0IO4"
+WRITE_ASCII 0x8 "D0IO4" #8
 COPY ~d0tweak/ioun/sporbit.vvc~ ~override/d0io5.vvc~
-WRITE_ASCII 0x8 "D0IO5"
+WRITE_ASCII 0x8 "D0IO5" #8
 COPY ~d0tweak/ioun/sporbit.vvc~ ~override/d0io6.vvc~
-WRITE_ASCII 0x8 "D0IO6"
+WRITE_ASCII 0x8 "D0IO6" #8
 COPY ~d0tweak/ioun/sporbit.vvc~ ~override/d0io7.vvc~
-WRITE_ASCII 0x8 "D0IO7"
+WRITE_ASCII 0x8 "D0IO7" #8
 COPY ~d0tweak/ioun/sporbit.vvc~ ~override/d0io8.vvc~
-WRITE_ASCII 0x8 "D0IO8"
+WRITE_ASCII 0x8 "D0IO8" #8
 COPY ~d0tweak/ioun/sporbit.vvc~ ~override/d0io9.vvc~
-WRITE_ASCII 0x8 "D0IO9"
+WRITE_ASCII 0x8 "D0IO9" #8
 
 COPY_EXISTING ~helm18.itm~ ~override~
               ~helm25.itm~ ~override~
@@ -934,7 +936,7 @@
 WRITE_BYTE ("%effectsoffset%" + 8) 1
 WRITE_BYTE ("%effectsoffset%" + 12) 2
 WRITE_BYTE ("%effectsoffset%" + 18) 100
-WRITE_ASCII ("%effectsoffset%" + 20) "D0IO6"
+WRITE_ASCII ("%effectsoffset%" + 20) "D0IO6" #8
 
 COPY_EXISTING ~helm20.itm~ ~override~
               ~helm34.itm~ ~override~
@@ -947,7 +949,7 @@
 WRITE_BYTE ("%effectsoffset%" + 8) 1
 WRITE_BYTE ("%effectsoffset%" + 12) 2
 WRITE_BYTE ("%effectsoffset%" + 18) 100
-WRITE_ASCII ("%effectsoffset%" + 20) "D0IO4"
+WRITE_ASCII ("%effectsoffset%" + 20) "D0IO4" #8
 
 COPY_EXISTING ~helm23.itm~ ~override~
               ~helm27.itm~ ~override~
@@ -960,7 +962,7 @@
 WRITE_BYTE ("%effectsoffset%" + 8) 1
 WRITE_BYTE ("%effectsoffset%" + 12) 2
 WRITE_BYTE ("%effectsoffset%" + 18) 100
-WRITE_ASCII ("%effectsoffset%" + 20) "D0IO3"
+WRITE_ASCII ("%effectsoffset%" + 20) "D0IO3" #8
 
 COPY_EXISTING ~helm26.itm~ ~override~
 READ_LONG 0x6a "effectsoffset"
@@ -972,7 +974,7 @@
 WRITE_BYTE ("%effectsoffset%" + 8) 1
 WRITE_BYTE ("%effectsoffset%" + 12) 2
 WRITE_BYTE ("%effectsoffset%" + 18) 100
-WRITE_ASCII ("%effectsoffset%" + 20) "D0IO5"
+WRITE_ASCII ("%effectsoffset%" + 20) "D0IO5" #8
 
 COPY_EXISTING ~helm24.itm~ ~override~
 READ_LONG 0x6a "effectsoffset"
@@ -984,7 +986,7 @@
 WRITE_BYTE ("%effectsoffset%" + 8) 1
 WRITE_BYTE ("%effectsoffset%" + 12) 2
 WRITE_BYTE ("%effectsoffset%" + 18) 100
-WRITE_ASCII ("%effectsoffset%" + 20) "D0IO1"
+WRITE_ASCII ("%effectsoffset%" + 20) "D0IO1" #8
 
 COPY_EXISTING ~helm19.itm~ ~override~
 READ_LONG 0x6a "effectsoffset"
@@ -996,7 +998,7 @@
 WRITE_BYTE ("%effectsoffset%" + 8) 1
 WRITE_BYTE ("%effectsoffset%" + 12) 2
 WRITE_BYTE ("%effectsoffset%" + 18) 100
-WRITE_ASCII ("%effectsoffset%" + 20) "D0IO7"
+WRITE_ASCII ("%effectsoffset%" + 20) "D0IO7" #8
 
 ACTION_IF FILE_EXISTS ~data/25Items.bif~
 THEN BEGIN
@@ -1010,7 +1012,7 @@
 WRITE_BYTE ("%effectsoffset%" + 8) 1
 WRITE_BYTE ("%effectsoffset%" + 12) 2
 WRITE_BYTE ("%effectsoffset%" + 18) 100
-WRITE_ASCII ("%effectsoffset%" + 20) "D0IO2"
+WRITE_ASCII ("%effectsoffset%" + 20) "D0IO2" #8
 END
 
 ACTION_IF FILE_EXISTS ~override/CDIOUN1.itm~
@@ -1025,7 +1027,7 @@
 WRITE_BYTE ("%effectsoffset%" + 8) 1
 WRITE_BYTE ("%effectsoffset%" + 12) 2
 WRITE_BYTE ("%effectsoffset%" + 18) 100
-WRITE_ASCII ("%effectsoffset%" + 20) "D0IO8"
+WRITE_ASCII ("%effectsoffset%" + 20) "D0IO8" #8
 END
 
 ACTION_IF FILE_EXISTS ~override/CDIOUN2.itm~
@@ -1040,7 +1042,7 @@
 WRITE_BYTE ("%effectsoffset%" + 8) 1
 WRITE_BYTE ("%effectsoffset%" + 12) 2
 WRITE_BYTE ("%effectsoffset%" + 18) 100
-WRITE_ASCII ("%effectsoffset%" + 20) "D0IO9"
+WRITE_ASCII ("%effectsoffset%" + 20) "D0IO9" #8
 END
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////
@@ -1108,8 +1110,8 @@
 REQUIRE_PREDICATE NOT FILE_EXISTS ~override/fw2600.are~ ~Skipping Tutu-incompatible component.~
 
 COPY_EXISTING ~QUAYLEM.CRE~ ~override/QUAYLEM.CRE~
-WRITE_ASCII ~0x34~ ~D0Nothin~
-WRITE_ASCII ~0x3c~ ~D0Nothin~
+WRITE_ASCII ~0x34~ ~D0Nothin~ #8
+WRITE_ASCII ~0x3c~ ~D0Nothin~ #8
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////
 ////////////////////////////////////////////////////////////////////////////////////////////////////////
@@ -1271,7 +1273,7 @@
 COPY_EXISTING ~SKELWASU.CRE~ ~override~
 WRITE_LONG 0x14 1150
 WRITE_BYTE 0x272 115
-WRITE_ASCII 0x248 "GMINIONM"
+WRITE_ASCII 0x248 "GMINIONM" #8
 
 COPY ~d0tweak/summons/newcres~ ~override~
 
