--- JklHel\scripts\AR2300.baf	Mon Mar 04 14:34:34 2013
+++ D:\EE_Patchstudio\patched files\JklHel\scripts\AR2300.baf	Tue Jan 19 21:46:39 2016
@@ -1,12 +1,12 @@
 IF
  
-Global("JKLHELExists","AR2300",0)
+Global("JKLHELExists","%FriendlyArmInn%",0)
  
 THEN
  
 RESPONSE #100
  
-SetGlobal("JKLHELExists","AR2300",1)
+SetGlobal("JKLHELExists","%FriendlyArmInn%",1)
  
 CreateCreature("JKLHEL",[1896.2333],3)
  
