--- bg1re\ladylowlantern\summerumbrella.d	Sat Mar 15 11:26:26 2014
+++ C:\BWP Patchstudio\patched files\bg1re\ladylowlantern\summerumbrella.d	Sat Nov 28 20:19:36 2015
@@ -63,7 +63,7 @@
 + ~Global("RE1_SumUmTalk","GLOBAL",3)~ + @9 DO ~SetGlobal("RE1_SumUmTalk","GLOBAL",4)~ + suchild
 + ~Global("RE1_SumUmTalk","GLOBAL",4)~ + @10 DO ~SetGlobal("RE1_SumUmTalk","GLOBAL",5)~ + suapprentice
 + ~Global("RE1_SumUmTalk","GLOBAL",5)~ + @11 DO ~SetGlobal("RE1_SumUmTalk","GLOBAL",6)~ + sudance
-+ ~Global("RE1_SumUmTalk","GLOBAL",6)~ + @12 DO ~SetGlobal("RE1_SumUmTalk","GLOBAL",7)~ DO ~SetGlobal("RE1_SumUmTalk","GLOBAL",7)~ + suregrets
++ ~Global("RE1_SumUmTalk","GLOBAL",6)~ + @12 DO ~SetGlobal("RE1_SumUmTalk","GLOBAL",7)~ + suregrets
 + ~Global("RE1_SumUmTalk","GLOBAL",7) RandomNum(3,1)~ + @13 + sudroppingby1
 + ~Global("RE1_SumUmTalk","GLOBAL",7) RandomNum(3,2)~ + @13 + sudroppingby2
 + ~Global("RE1_SumUmTalk","GLOBAL",7) RandomNum(3,3)~ + @13 + sudroppingby3
