--- ub_iwd\baf\tp8009.baf	Thu Feb 21 21:58:40 2008
+++ D:\EE_Patchstudio\patched files\ub_iwd\baf\tp8009.baf	Sat Jan 16 18:33:15 2016
@@ -1,27 +1,27 @@
 IF
-  Global("CDMarkHere","AR8009",2)
+  Global("CDMarkHere","%LowerDornsDeep_ArtisansDistrict%",2)
 THEN
   RESPONSE #25
     SetGlobal("CDMarkethTeleporting","GLOBAL",1)
-    SetGlobal("CDMarkHere","AR8009",0)
+    SetGlobal("CDMarkHere","%LowerDornsDeep_ArtisansDistrict%",0)
     ActionOverride("cdmark",ForceSpellPoint([2926.1784],WIZARD_DIMENSION_DOOR))
     Wait(1)
     ActionOverride("cdmark",DestroySelf())
   RESPONSE #25
     SetGlobal("CDMarkethTeleporting","GLOBAL",2)
-    SetGlobal("CDMarkHere","AR8009",0)
+    SetGlobal("CDMarkHere","%LowerDornsDeep_ArtisansDistrict%",0)
     ActionOverride("cdmark",ForceSpellPoint([2926.1784],WIZARD_DIMENSION_DOOR))
     Wait(1)
     ActionOverride("cdmark",DestroySelf())
   RESPONSE #25
     SetGlobal("CDMarkethTeleporting","GLOBAL",3)
-    SetGlobal("CDMarkHere","AR8009",0)
+    SetGlobal("CDMarkHere","%LowerDornsDeep_ArtisansDistrict%",0)
     ActionOverride("cdmark",ForceSpellPoint([2926.1784],WIZARD_DIMENSION_DOOR))
     Wait(1)
     ActionOverride("cdmark",DestroySelf())
   RESPONSE #25
     SetGlobal("CDMarkethTeleporting","GLOBAL",5)
-    SetGlobal("CDMarkHere","AR8009",0)
+    SetGlobal("CDMarkHere","%LowerDornsDeep_ArtisansDistrict%",0)
     ActionOverride("cdmark",ForceSpellPoint([2926.1784],WIZARD_DIMENSION_DOOR))
     Wait(1)
     ActionOverride("cdmark",DestroySelf())
@@ -29,11 +29,11 @@
 
 IF
   Global("CDMarkethTeleporting","GLOBAL",4)
-  Global("CDMarkHere","AR8009",0)
+  Global("CDMarkHere","%LowerDornsDeep_ArtisansDistrict%",0)
   !Global("Marketh_Ring","GLOBAL",6)
 THEN
   RESPONSE #100
-    SetGlobal("CDMarkHere","AR8009",1)
+    SetGlobal("CDMarkHere","%LowerDornsDeep_ArtisansDistrict%",1)
     CreateCreature("cdmark",[2926.1784],5)
     ActionOverride("cdmark",ForceSpellPoint([2926.1784],WIZARD_DIMENSION_DOOR))
 END
