--- c#solaufein/dialogues/C#SOLAUB.D	2014-05-22 17:42:26.000000000 -0400
+++ Fixpack/c#solaufein/dialogues/C#SOLAUB.D	2016-11-21 22:37:15.275135600 -0500
@@ -15,14 +15,14 @@
 Global("C#Solaufein_ImoenBanter","GLOBAL",0)
 InParty("IMOEN2") InMyArea("IMOEN2") !StateCheck("IMOEN2",CD_STATE_NOTVALID)~ THEN C#SOLAUB imoen_01
 @908 DO ~SetGlobal("C#Solaufein_ImoenBanter","GLOBAL",1)~
-== BIMOEN2 IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @909
-== BIMOEN25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @909
+== BIMOEN2 IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @909
+== BIMOEN25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @909
 == C#SOLAUB @910
-== BIMOEN2 IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @911
-== BIMOEN25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @911
+== BIMOEN2 IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @911
+== BIMOEN25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @911
 == C#SOLAUB @912
-== BIMOEN2 IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @913
-== BIMOEN25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @913
+== BIMOEN2 IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @913
+== BIMOEN25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @913
 EXIT
 
 
@@ -37,18 +37,18 @@
 InParty("ANOMEN") InMyArea("ANOMEN") !StateCheck("ANOMEN",CD_STATE_NOTVALID)
 Global("AnomenIsNotKnight","GLOBAL",0)~ THEN C#SOLAUB anomen_01
 @914 DO ~SetGlobal("C#Solaufein_AnomenBanter","GLOBAL",1)~
-== BANOMEN IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @915
-== BANOME25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @915
-== BANOMEN IF ~GlobalLT("Chapter","GLOBAL",8) InParty("C#Ajantis")~ THEN @916
-== BANOME25 IF ~GlobalGT("Chapter","GLOBAL",7) InParty("C#Ajantis")~ THEN @916
+== BANOMEN IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @915
+== BANOME25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @915
+== BANOMEN IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%) InParty("C#Ajantis")~ THEN @916
+== BANOME25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%) InParty("C#Ajantis")~ THEN @916
 == C#SOLAUB IF ~InParty("C#Ajantis")~ THEN @917
-== BANOMEN IF ~GlobalLT("Chapter","GLOBAL",8) !InParty("C#Ajantis")~ THEN @918
-== BANOME25 IF ~GlobalGT("Chapter","GLOBAL",7) InParty("C#Ajantis")~ THEN @918 
-== BANOMEN IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @919
-== BANOME25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @919
+== BANOMEN IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%) !InParty("C#Ajantis")~ THEN @918
+== BANOME25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%) InParty("C#Ajantis")~ THEN @918 
+== BANOMEN IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @919
+== BANOME25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @919
 == C#SOLAUB @920
-== BANOMEN IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @921
-== BANOME25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @921
+== BANOMEN IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @921
+== BANOME25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @921
 == C#SOLAUB @922
 EXIT
 
@@ -65,14 +65,14 @@
 Global("C#Solaufein_MinscBanter","GLOBAL",0)
 InParty("MINSC") InMyArea("MINSC") !StateCheck("MINSC",CD_STATE_NOTVALID)~ THEN C#SOLAUB minsc_01
 @923 DO ~SetGlobal("C#Solaufein_MinscBanter","GLOBAL",1)~
-== BMINSC IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @924
-== BMINSC25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @924
+== BMINSC IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @924
+== BMINSC25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @924
 == C#SOLAUB @925
-== BMINSC IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @926
-== BMINSC25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @926
+== BMINSC IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @926
+== BMINSC25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @926
 == C#SOLAUB @927
-== BMINSC IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @928
-== BMINSC25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @928
+== BMINSC IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @928
+== BMINSC25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @928
 == C#SOLAUB @929
 EXIT
 
@@ -88,11 +88,11 @@
 Global("C#Solaufein_JaheiraBanter","GLOBAL",0)
 InParty("Jaheira") InMyArea("Jaheira") !StateCheck("Jaheira",CD_STATE_NOTVALID)~ THEN C#SOLAUB jaheira_01
 @930 DO ~SetGlobal("C#Solaufein_JaheiraBanter","GLOBAL",1)~
-== BJAHEIR IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @931
-== BJAHEI25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @931
+== BJAHEIR IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @931
+== BJAHEI25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @931
 == C#SOLAUB @932
-== BJAHEIR IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @933
-== BJAHEI25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @933
+== BJAHEIR IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @933
+== BJAHEI25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @933
 EXIT 
 
 APPEND BJAHEIR
@@ -133,14 +133,14 @@
 CHAIN
 IF ~~ THEN C#SOLAUB jaheira_02
 @935
-== BJAHEIR IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @936
-== BJAHEI25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @936
+== BJAHEIR IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @936
+== BJAHEI25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @936
 == C#SOLAUB @937
-== BJAHEIR IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @938
-== BJAHEI25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @938
+== BJAHEIR IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @938
+== BJAHEI25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @938
 == C#SOLAUB @939
-== BJAHEIR IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @940
-== BJAHEI25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @940
+== BJAHEIR IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @940
+== BJAHEI25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @940
 EXIT
 
 /* Solaufein - Mazzy */
@@ -183,11 +183,11 @@
 CHAIN
 IF ~~ THEN C#SOLAUB mazzy_01
 @942
-== BMAZZY IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @943
-== BMAZZY25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @943
+== BMAZZY IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @943
+== BMAZZY25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @943
 == C#SOLAUB @944
-== BMAZZY IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @945
-== BMAZZY25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @945
+== BMAZZY IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @945
+== BMAZZY25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @945
 EXIT
 
 /* Solaufein - Mazzy */
@@ -200,14 +200,14 @@
 Global("C#Solaufein_MazzyBanter","GLOBAL",1)
 InParty("MAZZY") InMyArea("MAZZY") !StateCheck("MAZZY",CD_STATE_NOTVALID)~ THEN C#SOLAUB mazzy_02
 @946 DO ~SetGlobal("C#Solaufein_MazzyBanter","GLOBAL",2)~
-== BMAZZY IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @947
-== BMAZZY25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @947
+== BMAZZY IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @947
+== BMAZZY25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @947
 == C#SOLAUB @948
-== BMAZZY IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @949
-== BMAZZY25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @949
+== BMAZZY IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @949
+== BMAZZY25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @949
 == C#SOLAUB @950
-== BMAZZY IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @951
-== BMAZZY25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @951
+== BMAZZY IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @951
+== BMAZZY25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @951
 EXIT
 
 
@@ -223,14 +223,14 @@
 Global("C#Solaufein_JanBanter","GLOBAL",0)
 InParty("JAN") InMyArea("JAN") !StateCheck("JAN",CD_STATE_NOTVALID)~ THEN C#SOLAUB jan_01
 @952 DO ~SetGlobal("C#Solaufein_JanBanter","GLOBAL",1)~
-== BJAN IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @953
-== BJAN25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @953
+== BJAN IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @953
+== BJAN25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @953
 == C#SOLAUB @954
-== BJAN IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @955
-== BJAN25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @955
+== BJAN IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @955
+== BJAN25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @955
 == C#SOLAUB @956
-== BJAN IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @957
-== BJAN25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @957
+== BJAN IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @957
+== BJAN25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @957
 == C#SOLAUB @958
 EXIT
 
@@ -276,11 +276,11 @@
 CHAIN
 IF ~~ THEN C#SOLAUB minsc_02
 @960
-== BMINSC IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @961
-== BMINSC25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @961
+== BMINSC IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @961
+== BMINSC25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @961
 == C#SOLAUB @962
-== BMINSC IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @963
-== BMINSC25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @963
+== BMINSC IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @963
+== BMINSC25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @963
 == C#SOLAUB @964
 EXIT
 
@@ -347,11 +347,11 @@
 CHAIN
 IF ~~ THEN C#SOLAUB korgan_01
 @971
-== BKORGAN IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @972
-== BKORGA25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @972
+== BKORGAN IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @972
+== BKORGA25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @972
 == C#SOLAUB @973
-== BKORGAN IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @974
-== BKORGA25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @974
+== BKORGAN IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @974
+== BKORGA25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @974
 EXIT
 
 
@@ -369,14 +369,14 @@
 InParty("aerie") InMyArea("aerie") !StateCheck("aerie",CD_STATE_NOTVALID)~ THEN C#SOLAUB aerie_01
 @975 DO ~SetGlobal("C#Solaufein_AerieBanter","GLOBAL",1)~ 
 = @976
-== BAERIE IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @977 
-== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @977
+== BAERIE IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @977 
+== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @977
 == C#SOLAUB @978
-== BAERIE IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @979
-== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @979
+== BAERIE IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @979
+== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @979
 == C#SOLAUB @980
-== BAERIE IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @981
-== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @981
+== BAERIE IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @981
+== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @981
 EXIT
 
 /* Solaufein - Mazzy */
@@ -389,12 +389,12 @@
 Global("C#Solaufein_MazzyBanter","GLOBAL",2)
 InParty("MAZZY") InMyArea("MAZZY") !StateCheck("MAZZY",CD_STATE_NOTVALID)~ THEN C#SOLAUB mazzy_03
 @982 DO ~SetGlobal("C#Solaufein_MazzyBanter","GLOBAL",3)~
-== BMAZZY IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @983
-== BMAZZY25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @983
+== BMAZZY IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @983
+== BMAZZY25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @983
 == C#SOLAUB @984
 = @985
-== BMAZZY IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @986
-== BMAZZY25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @986
+== BMAZZY IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @986
+== BMAZZY25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @986
 EXIT
 
 
@@ -410,8 +410,8 @@
 Global("C#Solaufein_ValygarBanter","GLOBAL",0)
 InParty("Valygar") InMyArea("Valygar") !StateCheck("Valygar",CD_STATE_NOTVALID)~ THEN C#SOLAUB valygar_01
 @987 DO ~SetGlobal("C#Solaufein_ValygarBanter","GLOBAL",1)~
-== BVALYGA IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @988
-== BVALYG25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @988
+== BVALYGA IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @988
+== BVALYG25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @988
 == C#SOLAUB @989
 EXIT
 
@@ -431,12 +431,12 @@
 Global("C#Solaufein_KeldornBanter","GLOBAL",0)
 InParty("Keldorn") InMyArea("Keldorn") !StateCheck("Keldorn",CD_STATE_NOTVALID)~ THEN C#SOLAUB Keldorn_01
 @990 DO ~SetGlobal("C#Solaufein_KeldornBanter","GLOBAL",1)~
-== BKELDOR  IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @991
-== BKELDO25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @991
+== BKELDOR  IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @991
+== BKELDO25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @991
 == C#SOLAUB @992
 = @993
-== BKELDOR  IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @994
-== BKELDO25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @994
+== BKELDOR  IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @994
+== BKELDO25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @994
 == C#SOLAUB @995
 EXIT
 
@@ -452,11 +452,11 @@
 Global("C#Solaufein_CerndBanter","GLOBAL",0)
 InParty("CERND") InMyArea("CERND") !StateCheck("CERND",CD_STATE_NOTVALID)~ THEN C#SOLAUB cernd_01
 @996 DO ~SetGlobal("C#Solaufein_CerndBanter","GLOBAL",1)~
-== BCERND IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @997
-== BCERND25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @997
+== BCERND IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @997
+== BCERND25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @997
 == C#SOLAUB @998
-== BCERND IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @999
-== BCERND25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @999
+== BCERND IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @999
+== BCERND25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @999
 EXIT
 
 
@@ -475,11 +475,11 @@
 Global("C#Solaufein_EdwinABanter","GLOBAL",0)
 InParty("Edwin") InMyArea("Edwin") !StateCheck("Edwin",CD_STATE_NOTVALID)~ THEN C#SOLAUB Edwina_01
 @1000 DO ~SetGlobal("C#Solaufein_EdwinABanter","GLOBAL",1)~
-== BEDWIN IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1001
-== BEDWIN25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1001
+== BEDWIN IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1001
+== BEDWIN25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1001
 == C#SOLAUB @1002
-== BEDWIN IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1003
-== BEDWIN25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1003
+== BEDWIN IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1003
+== BEDWIN25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1003
 == C#SOLAUB @1004
 EXIT
 
@@ -495,7 +495,7 @@
 !See([ENEMY])
 Global("C#Solaufein_EdwinBanterToB","GLOBAL",0)
 InParty("Edwin") InMyArea("Edwin") !StateCheck("Edwin",CD_STATE_NOTVALID)
-GlobalGT("Chapter","GLOBAL",7)~ THEN C#SOLAUB edwin_tob
+GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN C#SOLAUB edwin_tob
 @1005 DO ~SetGlobal("C#Solaufein_EdwinBanterToB","GLOBAL",1)~
 == BEDWIN25 @1006
 == C#SOLAUB @1007
@@ -519,11 +519,11 @@
 Global("C#Solaufein_ViconiaBanter","GLOBAL",0)
 InParty("viconia") InMyArea("viconia") !StateCheck("viconia",CD_STATE_NOTVALID)~ THEN C#SOLAUB viconia_01
 @1013 DO ~SetGlobal("C#Solaufein_ViconiaBanter","GLOBAL",1)~
-== BVICONI IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1014
-== BVICON25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1014
+== BVICONI IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1014
+== BVICON25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1014
 == C#SOLAUB @1015
-== BVICONI IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1016
-== BVICON25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1016
+== BVICONI IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1016
+== BVICON25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1016
 EXIT
 
 
@@ -539,11 +539,11 @@
 InParty("haerdalis") InMyArea("haerdalis") !StateCheck("haerdalis",CD_STATE_NOTVALID)~ THEN C#SOLAUB haerdalis_01
 @1017
 = @1018 DO ~SetGlobal("C#Solaufein_HaerDalisBanter","GLOBAL",1)~
-== BHAERDA IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1019
-== BHAERD25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1019
+== BHAERDA IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1019
+== BHAERD25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1019
 == C#SOLAUB @1020
-== BHAERDA IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1021
-== BHAERD25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1021
+== BHAERDA IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1021
+== BHAERD25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1021
 EXIT
 
 
@@ -591,7 +591,7 @@
 !StateCheck("C#Solaufein",CD_STATE_NOTVALID)
 !Dead("C#Solaufein")
 Global("C#Solaufein_HaerDalisBanter","GLOBAL",2)
-GlobalGT("Chapter","GLOBAL",7)~ THEN haerdalis_03
+GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN haerdalis_03
 SAY @1023
 IF ~~ THEN DO ~SetGlobal("C#Solaufein_HaerDalisBanter","GLOBAL",3)~ EXTERN C#SOLAUB haerdalis_03
 END
@@ -601,8 +601,8 @@
 CHAIN
 IF ~~ THEN C#SOLAUB haerdalis_02
 @1024
-== BHAERDA IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1025
-== BHAERD25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1025
+== BHAERDA IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1025
+== BHAERD25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1025
 == C#SOLAUB @1026
 EXIT
 
@@ -627,10 +627,10 @@
 Global("C#Solaufein_ImoenBanter","GLOBAL",1)
 InParty("IMOEN2") InMyArea("IMOEN2") !StateCheck("IMOEN2",CD_STATE_NOTVALID)~ THEN C#SOLAUB imoen_02
 @1030 DO ~SetGlobal("C#Solaufein_ImoenBanter","GLOBAL",2)~
-== BIMOEN2 IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1031
+== BIMOEN2 IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1031
 = @1032
 = @1033
-== BIMOEN25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1031
+== BIMOEN25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1031
 = @1032
 = @1033
 EXIT
@@ -646,7 +646,7 @@
 !See([ENEMY])
 Global("C#Solaufein_MazzyBanter","GLOBAL",3)
 InParty("MAZZY") InMyArea("MAZZY") !StateCheck("MAZZY",CD_STATE_NOTVALID)
-GlobalGT("Chapter","GLOBAL",7)~ THEN C#SOLAUB mazzy_04
+GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN C#SOLAUB mazzy_04
 @1034 DO ~SetGlobal("C#Solaufein_MazzyBanter","GLOBAL",4)~
 == BMAZZY25 @1035
 == C#SOLAUB @1036
@@ -667,14 +667,14 @@
 InParty("ANOMEN") InMyArea("ANOMEN") !StateCheck("ANOMEN",CD_STATE_NOTVALID)
 Global("AnomenIsNotKnight","GLOBAL",0)~ THEN C#SOLAUB anomen_02
 @1040 DO ~SetGlobal("C#Solaufein_AnomenBanter","GLOBAL",2)~
-== BANOMEN IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @247
-== BANOME25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @247
+== BANOMEN IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @247
+== BANOME25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @247
 == C#SOLAUB @1041
-== BANOMEN IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1042
-== BANOME25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1042
+== BANOMEN IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1042
+== BANOME25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1042
 == C#SOLAUB @1043
-== BANOMEN IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1044
-== BANOME25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1044
+== BANOMEN IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1044
+== BANOME25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1044
 == C#SOLAUB @1045
 EXIT
 
@@ -745,22 +745,22 @@
 CHAIN
 IF ~~ THEN C#SOLAUB anomen_03
 @1048
-== BANOMEN IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1049
-== BANOME25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1049
+== BANOMEN IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1049
+== BANOME25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1049
 == C#SOLAUB @1050
 EXIT
 
 CHAIN
 IF ~~ THEN C#SOLAUB anomen_04
 @1051
-== BANOMEN IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1052
-== BANOME25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1052
+== BANOMEN IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1052
+== BANOME25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1052
 == C#SOLAUB @1053
-== BANOMEN IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1054
-== BANOME25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1054
+== BANOMEN IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1054
+== BANOME25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1054
 == C#SOLAUB @1055
-== BANOMEN IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1056
-== BANOME25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1056
+== BANOMEN IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1056
+== BANOME25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1056
 == C#SOLAUB @1057
 EXIT
 
@@ -882,14 +882,14 @@
 CHAIN
 IF ~~ THEN C#SOLAUB nalia_01
 @1063
-== BNALIA IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1064
-== BNALIA25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1064
+== BNALIA IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1064
+== BNALIA25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1064
 == C#SOLAUB @1065
-== BNALIA IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1066
-== BNALIA25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1066
+== BNALIA IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1066
+== BNALIA25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1066
 == C#SOLAUB @1067
-== BNALIA IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1068
-== BNALIA25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1068
+== BNALIA IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1068
+== BNALIA25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1068
 == C#SOLAUB @1069
 EXIT
 
@@ -904,13 +904,13 @@
 InParty("nalia") InMyArea("nalia") !StateCheck("nalia",CD_STATE_NOTVALID)~ THEN C#SOLAUB nalia_02
 @1070
 = @1071 DO ~SetGlobal("C#Solaufein_NaliaBanter","GLOBAL",2)~
-== BNALIA IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1072
+== BNALIA IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1072
 = @1073
-== BNALIA25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1072
+== BNALIA25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1072
 = @1073
 == C#SOLAUB @1074
-== BNALIA IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1075
-== BNALIA25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1075
+== BNALIA IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1075
+== BNALIA25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1075
 == C#SOLAUB @1076
 EXIT
 
@@ -924,8 +924,8 @@
 CHAIN
 IF ~~ THEN C#SOLAUB nalia_04
 @1078
-== BNALIA IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1079
-== BNALIA25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1079
+== BNALIA IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1079
+== BNALIA25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1079
 == C#SOLAUB @1080
 EXIT
 
@@ -957,11 +957,11 @@
 Global("C#Solaufein_JaheiraBanter","GLOBAL",2)
 InParty("Jaheira") InMyArea("Jaheira") !StateCheck("Jaheira",CD_STATE_NOTVALID)~ THEN C#SOLAUB jaheira_03
 @1087 DO ~SetGlobal("C#Solaufein_JaheiraBanter","GLOBAL",3)~
-== BJAHEIR IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1088
-== BJAHEI25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1088 
+== BJAHEIR IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1088
+== BJAHEI25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1088 
 == C#SOLAUB @1089 DO ~SetGlobal("C#Solaufein_JaheiraBanter","GLOBAL",2)~
-== BJAHEIR IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1090
-== BJAHEI25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1090
+== BJAHEIR IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1090
+== BJAHEI25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1090
 == C#SOLAUB @1091
 EXIT
 
@@ -976,11 +976,11 @@
 Global("C#Solaufein_JanBanter","GLOBAL",1)
 InParty("JAN") InMyArea("JAN") !StateCheck("JAN",CD_STATE_NOTVALID)~ THEN C#SOLAUB jan_02
 @1092 DO ~SetGlobal("C#Solaufein_JanBanter","GLOBAL",2)~
-== BJAN IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1093
-== BJAN25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1093
+== BJAN IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1093
+== BJAN25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1093
 == C#SOLAUB @1094
-== BJAN IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1095
-== BJAN25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1095
+== BJAN IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1095
+== BJAN25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1095
 EXIT
 
 /* Solaufein - Valygar */
@@ -993,9 +993,9 @@
 Global("C#Solaufein_ValygarBanter","GLOBAL",1)
 InParty("Valygar") InMyArea("Valygar") !StateCheck("Valygar",CD_STATE_NOTVALID)~ THEN C#SOLAUB valygar_02
 @1096 DO ~SetGlobal("C#Solaufein_ValygarBanter","GLOBAL",2)~
-== BVALYGA IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1097
+== BVALYGA IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1097
 = @1098
-== BVALYG25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1097
+== BVALYG25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1097
 = @1098
 == C#SOLAUB @1099
 EXIT
@@ -1012,18 +1012,18 @@
 Global("C#Solaufein_MinscBanter","GLOBAL",2)
 InParty("MINSC") InMyArea("MINSC") !StateCheck("MINSC",CD_STATE_NOTVALID)~ THEN C#SOLAUB minsc_03
 @1100 DO ~SetGlobal("C#Solaufein_MinscBanterToB","GLOBAL",3)~
-== BMINSC IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1101
-== BMINSC25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1101
+== BMINSC IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1101
+== BMINSC25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1101
 == C#SOLAUB @1102
-== BMINSC IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1103
-== BMINSC25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1103
+== BMINSC IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1103
+== BMINSC25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1103
 == C#SOLAUB @1104
-== BMINSC IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1105
+== BMINSC IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1105
 == C#SOLAUB @1106
-== BMINSC25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1105
+== BMINSC25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1105
 == C#SOLAUB @1106
-== BMINSC IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1101
-== BMINSC25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1101
+== BMINSC IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1101
+== BMINSC25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1101
 == C#SOLAUB @1107
 EXIT
 
@@ -1055,7 +1055,7 @@
 !See([ENEMY])
 Global("C#Solaufein_ImoenBanterToB","GLOBAL",0)
 InParty("IMOEN2") InMyArea("IMOEN2") !StateCheck("IMOEN2",CD_STATE_NOTVALID)
-GlobalGT("Chapter","GLOBAL",7)~ THEN C#SOLAUB imoen_tob
+GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN C#SOLAUB imoen_tob
 @1114 DO ~SetGlobal("C#Solaufein_ImoenBanterToB","GLOBAL",1)~
 == BIMOEN25 @1115
 = @1116
@@ -1072,7 +1072,7 @@
 !See([ENEMY])
 Global("C#Solaufein_ImoenBanterToB","GLOBAL",1)
 InParty("IMOEN2") InMyArea("IMOEN2") !StateCheck("IMOEN2",CD_STATE_NOTVALID)
-GlobalGT("Chapter","GLOBAL",7)~ THEN C#SOLAUB imoen_tob_2
+GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN C#SOLAUB imoen_tob_2
 @1117 DO ~SetGlobal("C#Solaufein_ImoenBanterToB","GLOBAL",2)~
 == BIMOEN25 @1118
 = @1119
@@ -1091,8 +1091,8 @@
 Global("C#Solaufein_KorganBanter","GLOBAL",1)
 InParty("KORGAN") InMyArea("KORGAN") !StateCheck("KORGAN",CD_STATE_NOTVALID)~ THEN C#SOLAUB korgan_02
 @1120 DO ~SetGlobal("C#Solaufein_KorganBanter","GLOBAL",2)~
-== BKORGAN IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1121
-== BKORGA25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1121
+== BKORGAN IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1121
+== BKORGA25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1121
 == C#SOLAUB @1122
 = @1123
 = @1124
@@ -1112,15 +1112,15 @@
 Global("C#Solaufein_AerieBanter","GLOBAL",1)
 InParty("aerie") InMyArea("aerie") !StateCheck("aerie",CD_STATE_NOTVALID)~ THEN C#SOLAUB aerie_02
 @1125 DO ~SetGlobal("C#Solaufein_AerieBanter","GLOBAL",2)~
-== BAERIE IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1126
-== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1126
+== BAERIE IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1126
+== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1126
 == C#SOLAUB @1127
 = @1128
-== BAERIE IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1129
-== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1129
+== BAERIE IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1129
+== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1129
 == C#SOLAUB @1130
-== BAERIE IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1131
-== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1131
+== BAERIE IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1131
+== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1131
 EXIT
 
 
@@ -1134,7 +1134,7 @@
 !See([ENEMY])
 Global("C#Solaufein_MinscBanter","GLOBAL",3)
 InParty("MINSC") InMyArea("MINSC") !StateCheck("MINSC",CD_STATE_NOTVALID)
-GlobalGT("Chapter","GLOBAL",7)~ THEN C#SOLAUB minsc_04
+GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN C#SOLAUB minsc_04
 @1100 DO ~SetGlobal("C#Solaufein_MinscBanterToB","GLOBAL",4)~
 == BMINSC25 @1101
 == C#SOLAUB @1132
@@ -1159,7 +1159,7 @@
 !See([ENEMY])
 Global("C#Solaufein_ValygarBanter","GLOBAL",2)
 InParty("Valygar") InMyArea("Valygar") !StateCheck("Valygar",CD_STATE_NOTVALID)
-GlobalGT("Chapter","GLOBAL",7)~ THEN C#SOLAUB valygar_03
+GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN C#SOLAUB valygar_03
 @1140 DO ~SetGlobal("C#Solaufein_ValygarBanter","GLOBAL",3)~
 == BVALYG25 @1141
 == C#SOLAUB @1142
@@ -1182,12 +1182,12 @@
 Global("C#Solaufein_KeldornBanter","GLOBAL",1)
 InParty("Keldorn") InMyArea("Keldorn") !StateCheck("Keldorn",CD_STATE_NOTVALID)~ THEN C#SOLAUB Keldorn_02
 @1145 DO ~SetGlobal("C#Solaufein_KeldornBanter","GLOBAL",2)~
-== BKELDOR  IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1146
-== BKELDO25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1146
+== BKELDOR  IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1146
+== BKELDO25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1146
 == C#SOLAUB @1147
 = @1148
-== BKELDOR  IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1149
-== BKELDO25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1149
+== BKELDOR  IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1149
+== BKELDO25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1149
 == C#SOLAUB @1150
 EXIT
 
@@ -1202,14 +1202,14 @@
 Global("C#Solaufein_CerndBanter","GLOBAL",1)
 InParty("CERND") InMyArea("CERND") !StateCheck("CERND",CD_STATE_NOTVALID)~ THEN C#SOLAUB cernd_02
 @1151 DO ~SetGlobal("C#Solaufein_CerndBanter","GLOBAL",2)~
-== BCERND IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1152
-== BCERND25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1152
+== BCERND IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1152
+== BCERND25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1152
 == C#SOLAUB @1153
-== BCERND IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1154
-== BCERND25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1154
+== BCERND IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1154
+== BCERND25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1154
 == C#SOLAUB @1155
-== BCERND IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1156
-== BCERND25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1156
+== BCERND IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1156
+== BCERND25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1156
 EXIT
 
 
@@ -1251,11 +1251,11 @@
 CHAIN
 IF ~~ THEN C#SOLAUB cernd_03
 @1158
-== BCERND IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1159
-== BCERND25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1159
+== BCERND IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1159
+== BCERND25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1159
 == C#SOLAUB @1160
-== BCERND IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1161
-== BCERND25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1161
+== BCERND IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1161
+== BCERND25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1161
 EXIT
 
 
@@ -1270,7 +1270,7 @@
 Global("C#Solaufein_AnomenBanterToB","GLOBAL",0)
 InParty("ANOMEN") InMyArea("ANOMEN") !StateCheck("ANOMEN",CD_STATE_NOTVALID)
 Global("AnomenIsNotKnight","GLOBAL",0) ReputationGT(Player1,10)
-GlobalGT("Chapter","GLOBAL",7)~ THEN C#SOLAUB anomen_05_isknight
+GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN C#SOLAUB anomen_05_isknight
 @1162 DO ~SetGlobal("C#Solaufein_AnomenBanterToB","GLOBAL",1)~
 == BANOME25 @1163
 == C#SOLAUB @1164
@@ -1291,7 +1291,7 @@
 Global("C#Solaufein_AnomenBanterToB","GLOBAL",0)
 InParty("ANOMEN") InMyArea("ANOMEN") !StateCheck("ANOMEN",CD_STATE_NOTVALID)
 Global("AnomenIsNotKnight","GLOBAL",1) ReputationLT(Player1,9)
-GlobalGT("Chapter","GLOBAL",7)~ THEN C#SOLAUB anomen_05_isnotknight
+GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN C#SOLAUB anomen_05_isnotknight
 @1169 DO ~SetGlobal("C#Solaufein_AnomenBanterToB","GLOBAL",1)~
 == BANOME25 @1163
 == C#SOLAUB @1170
@@ -1343,11 +1343,11 @@
 CHAIN
 IF ~~ THEN C#SOLAUB edwina_02
 @1175 
-== BEDWIN IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1176
-== BEDWIN25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1176
+== BEDWIN IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1176
+== BEDWIN25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1176
 == C#SOLAUB @1177
-== BEDWIN IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1178
-== BEDWIN25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1178
+== BEDWIN IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1178
+== BEDWIN25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1178
 EXIT
 
 
@@ -1362,12 +1362,12 @@
 Global("C#Solaufein_ViconiaBanter","GLOBAL",1)
 InParty("viconia") InMyArea("viconia") !StateCheck("viconia",CD_STATE_NOTVALID)~ THEN C#SOLAUB viconia_02
 @1179 DO ~SetGlobal("C#Solaufein_ViconiaBanter","GLOBAL",2)~
-== BVICONI IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1180
-== BVICON25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1180
+== BVICONI IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1180
+== BVICON25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1180
 == C#SOLAUB @1181
 = @1182
-== BVICONI IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1183
-== BVICON25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1183
+== BVICONI IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1183
+== BVICON25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1183
 EXIT
 
 
@@ -1382,16 +1382,16 @@
 Global("C#Solaufein_JaheiraBanter","GLOBAL",3)
 InParty("Jaheira") InMyArea("Jaheira") !StateCheck("Jaheira",CD_STATE_NOTVALID)~ THEN C#SOLAUB jaheira_04
 @1184 DO ~SetGlobal("C#Solaufein_JaheiraBanter","GLOBAL",4)~
-== BJAHEIR IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1185
-== BJAHEI25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1185 
+== BJAHEIR IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1185
+== BJAHEI25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1185 
 == C#SOLAUB @1186
-== BJAHEIR IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1187
+== BJAHEIR IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1187
 = @1188
-== BJAHEI25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1187
+== BJAHEI25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1187
 = @1188 
 == C#SOLAUB @1189
-== BJAHEIR IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1190
-== BJAHEI25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1190 
+== BJAHEIR IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1190
+== BJAHEI25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1190 
 EXIT
 
 
@@ -1407,12 +1407,12 @@
 Global("C#Solaufein_JanBanter","GLOBAL",2)
 InParty("JAN") InMyArea("JAN") !StateCheck("JAN",CD_STATE_NOTVALID)~ THEN C#SOLAUB jan_03
 @1191 DO ~SetGlobal("C#Solaufein_JanBanter","GLOBAL",3)~
-== BJAN IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1192
-== BJAN25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1192
+== BJAN IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1192
+== BJAN25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1192
 == C#SOLAUB @1193
-== BJAN IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1194
+== BJAN IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1194
 = @1195
-== BJAN25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1194
+== BJAN25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1194
 = @1195
 EXIT
 
@@ -1447,16 +1447,16 @@
 InParty("aerie") InMyArea("aerie") !StateCheck("aerie",CD_STATE_NOTVALID)~ THEN C#SOLAUB aerie_03
 @1200 DO ~SetGlobal("C#Solaufein_AerieBanter","GLOBAL",3)~
 = @1201
-== BAERIE IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1202
-== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1202
+== BAERIE IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1202
+== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1202
 == C#SOLAUB @1203
 = @1204
-== BAERIE IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1205
-== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1205
+== BAERIE IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1205
+== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1205
 == C#SOLAUB @1206
 = @1207
-== BAERIE IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1208
-== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1208
+== BAERIE IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1208
+== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1208
 EXIT
 
 
@@ -1473,11 +1473,11 @@
 InParty("Keldorn") InMyArea("Keldorn") !StateCheck("Keldorn",CD_STATE_NOTVALID)~ THEN C#SOLAUB Keldorn_03
 @1209
 = @1210 DO ~SetGlobal("C#Solaufein_KeldornBanter","GLOBAL",3)~
-== BKELDOR  IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1211
-== BKELDO25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1211
+== BKELDOR  IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1211
+== BKELDO25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1211
 == C#SOLAUB @1212
-== BKELDOR  IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1213
-== BKELDO25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1213
+== BKELDOR  IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1213
+== BKELDO25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1213
 EXIT
 
 
@@ -1490,7 +1490,7 @@
 !See([ENEMY])
 Global("C#Solaufein_CerndBanter","GLOBAL",3)
 InParty("CERND") InMyArea("CERND") !StateCheck("CERND",CD_STATE_NOTVALID)
-GlobalGT("Chapter","GLOBAL",7)~ THEN C#SOLAUB cernd_04
+GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN C#SOLAUB cernd_04
 @1214 DO ~SetGlobal("C#Solaufein_CerndBanter","GLOBAL",4)~
 == BCERND25 @1215
 == C#SOLAUB @1216
@@ -1543,8 +1543,8 @@
 CHAIN
 IF ~~ THEN C#SOLAUB viconia_03
 @1223
-== BVICONI IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1224
-== BVICON25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1224
+== BVICONI IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1224
+== BVICON25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1224
 == C#SOLAUB @1225
 = @1226 
 EXIT
@@ -1560,7 +1560,7 @@
 !See([ENEMY])
 Global("C#Solaufein_JaheiraBanter","GLOBAL",4)
 InParty("Jaheira") InMyArea("Jaheira") !StateCheck("Jaheira",CD_STATE_NOTVALID)
-GlobalGT("Chapter","GLOBAL",7)~ THEN C#SOLAUB jaheira_05
+GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN C#SOLAUB jaheira_05
 @1227 DO ~SetGlobal("C#Solaufein_JaheiraBanter","GLOBAL",5)~
 == BJAHEI25 @1228
 == C#SOLAUB @1229
@@ -1580,7 +1580,7 @@
 !See([ENEMY])
 Global("C#Solaufein_JanBanter","GLOBAL",3)
 InParty("JAN") InMyArea("JAN") !StateCheck("JAN",CD_STATE_NOTVALID)
-GlobalGT("Chapter","GLOBAL",7)~ THEN C#SOLAUB jan_04
+GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN C#SOLAUB jan_04
 @1233 DO ~SetGlobal("C#Solaufein_JanBanter","GLOBAL",4)~
 == BJAN25 @1234
 = @1235
@@ -1606,11 +1606,11 @@
 InParty("aerie") InMyArea("aerie") !StateCheck("aerie",CD_STATE_NOTVALID)~ THEN C#SOLAUB aerie_04
 @1242 DO ~SetGlobal("C#Solaufein_AerieBanter","GLOBAL",4)~
 = @1243
-== BAERIE IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1244
-== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1244
+== BAERIE IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1244
+== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1244
 == C#SOLAUB @1245
-== BAERIE IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1246
-== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1246
+== BAERIE IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1246
+== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1246
 == C#SOLAUB @1247
 EXIT
 
@@ -1629,14 +1629,14 @@
 Global("C#Solaufein_KeldornBanter","GLOBAL",3)
 InParty("Keldorn") InMyArea("Keldorn") !StateCheck("Keldorn",CD_STATE_NOTVALID)~ THEN C#SOLAUB Keldorn_04
 @1248 DO ~SetGlobal("C#Solaufein_KeldornBanter","GLOBAL",4)~
-== BKELDOR  IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1249
-== BKELDO25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1249
+== BKELDOR  IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1249
+== BKELDO25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1249
 == C#SOLAUB @1250
-== BKELDOR  IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1251
-== BKELDO25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1251
+== BKELDOR  IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1251
+== BKELDO25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1251
 == C#SOLAUB @1252
-== BKELDOR  IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1253
-== BKELDO25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1253
+== BKELDOR  IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1253
+== BKELDO25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1253
 == C#SOLAUB @1254
 EXIT
 
@@ -1653,12 +1653,12 @@
 Global("C#Solaufein_ViconiaBanter","GLOBAL",3)
 InParty("viconia") InMyArea("viconia") !StateCheck("viconia",CD_STATE_NOTVALID)~ THEN C#SOLAUB viconia_04
 @1255 DO ~SetGlobal("C#Solaufein_ViconiaBanter","GLOBAL",4)~
-== BVICONI IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1256
-== BVICON25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1256
+== BVICONI IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1256
+== BVICON25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1256
 == C#SOLAUB @1257
 = @1258
-== BVICONI IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1259
-== BVICON25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1259
+== BVICONI IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1259
+== BVICON25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1259
 EXIT
 
 
@@ -1673,14 +1673,14 @@
 Global("C#Solaufein_AerieBanter","GLOBAL",4)
 InParty("aerie") InMyArea("aerie") !StateCheck("aerie",CD_STATE_NOTVALID)~ THEN C#SOLAUB aerie_05
 @1260 DO ~SetGlobal("C#Solaufein_AerieBanter","GLOBAL",5)~
-== BAERIE IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1261
-== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1261
+== BAERIE IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1261
+== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1261
 == C#SOLAUB @1262
-== BAERIE IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1263
-== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1263
+== BAERIE IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1263
+== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1263
 == C#SOLAUB @86
-== BAERIE IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1264
-== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1264
+== BAERIE IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1264
+== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1264
 == C#SOLAUB @1265
 EXIT
 
@@ -1694,7 +1694,7 @@
 !See([ENEMY])
 Global("C#Solaufein_KeldornBanter","GLOBAL",4)
 InParty("Keldorn") InMyArea("Keldorn") !StateCheck("Keldorn",CD_STATE_NOTVALID)
-GlobalGT("Chapter","GLOBAL",7)~ THEN C#SOLAUB Keldorn_05
+GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN C#SOLAUB Keldorn_05
 @1266 DO ~SetGlobal("C#Solaufein_KeldornBanter","GLOBAL",5)~
 == BKELDO25 @1267
 == C#SOLAUB @1268
@@ -1716,7 +1716,7 @@
 !See([ENEMY])
 Global("C#Solaufein_KorganBanter","GLOBAL",2)
 InParty("KORGAN") InMyArea("KORGAN") !StateCheck("KORGAN",CD_STATE_NOTVALID)
-GlobalGT("Chapter","GLOBAL",7)~ THEN C#SOLAUB korgan_03
+GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN C#SOLAUB korgan_03
 @1274 DO ~SetGlobal("C#Solaufein_KorganBanter","GLOBAL",3)~
 == BKORGA25 @1275
 == C#SOLAUB @1276
@@ -1734,18 +1734,18 @@
 Global("C#Solaufein_ViconiaBanter","GLOBAL",4)
 InParty("viconia") InMyArea("viconia") !StateCheck("viconia",CD_STATE_NOTVALID)~ THEN C#SOLAUB viconia_05
 @1277 DO ~SetGlobal("C#Solaufein_ViconiaBanter","GLOBAL",5)~
-== BVICONI IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1278
-== BVICON25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1278
+== BVICONI IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1278
+== BVICON25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1278
 == C#SOLAUB @1279
-== BVICONI IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1280
-== BVICON25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1280
+== BVICONI IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1280
+== BVICON25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1280
 == C#SOLAUB @1281
 END
 ++ @1282 EXTERN C#SOLAUB viconia_05_2
 ++ @1283 EXTERN C#SOLAUB viconia_05_2 
 ++ @1284 EXTERN C#SOLAUB viconia_05_2 
-+ ~GlobalLT("Chapter","GLOBAL",8)~ + @1285 EXTERN BVICONI viconia_05_2
-+ ~GlobalGT("Chapter","GLOBAL",7)~ + @1285 EXTERN BVICON25 viconia_05_2
++ ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ + @1285 EXTERN BVICONI viconia_05_2
++ ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ + @1285 EXTERN BVICON25 viconia_05_2
 
 
 
@@ -1773,12 +1773,12 @@
 CHAIN  
 IF ~~ THEN C#SOLAUB viconia_05_2
 @1288
-== BVICONI IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1289
-== BVICON25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1289
+== BVICONI IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1289
+== BVICON25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1289
 == C#SOLAUB @1290
 = @1291
-== BVICONI IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1292
-== BVICON25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1292
+== BVICONI IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1292
+== BVICON25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1292
 == C#SOLAUB @1293
 EXIT
 
@@ -1797,8 +1797,8 @@
 InParty("aerie") InMyArea("aerie") !StateCheck("aerie",CD_STATE_NOTVALID)~ THEN C#SOLAUB aerie_06
 @1294 DO ~SetGlobal("C#Solaufein_AerieBanter","GLOBAL",6)~
 = @1295
-== BAERIE IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @1296
-== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1296
+== BAERIE IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @1296
+== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1296
 == C#SOLAUB @1297
 = @1298
 = @1299
@@ -1816,7 +1816,7 @@
 !See([ENEMY])
 Global("C#Solaufein_ViconiaBanter","GLOBAL",5)
 InParty("viconia") InMyArea("viconia") !StateCheck("viconia",CD_STATE_NOTVALID)
-GlobalGT("Chapter","GLOBAL",7)~ THEN C#SOLAUB viconia_06
+GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN C#SOLAUB viconia_06
 @1300 DO ~SetGlobal("C#Solaufein_ViconiaBanter","GLOBAL",6)~
 == BVICON25 @1301
 == C#SOLAUB @1302
@@ -1838,7 +1838,7 @@
 !StateCheck("C#Solaufein",CD_STATE_NOTVALID)
 !Dead("C#Solaufein")
 Global("C#Solaufein_AerieBanter","GLOBAL",6)
-GlobalGT("Chapter","GLOBAL",7)~ THEN aerie_07
+GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN aerie_07
 SAY @1303 
 IF ~~ THEN DO ~SetGlobal("C#Solaufein_AerieBanter","GLOBAL",7)~ EXTERN C#SOLAUB aerie_07
 END
@@ -1853,7 +1853,7 @@
 !StateCheck("C#Solaufein",CD_STATE_NOTVALID)
 !Dead("C#Solaufein")
 Global("C#Solaufein_AerieBanter","GLOBAL",7)
-GlobalGT("Chapter","GLOBAL",7)~ THEN aerie_08
+GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN aerie_08
 SAY @1304
 = @1305
 = @1306 
@@ -1867,17 +1867,17 @@
 CHAIN
 IF ~~ THEN C#SOLAUB aerie_07
 @1307
-== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1308
+== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1308
 == C#SOLAUB @1309
 = @1310
 = @1311
-== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1312 
+== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1312 
 == C#SOLAUB @1313
-== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1314
+== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1314
 == C#SOLAUB @1315
-== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1316
+== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1316
 == C#SOLAUB @1317
-== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1318
+== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1318
 == C#SOLAUB @1319
 EXIT
 
@@ -1885,11 +1885,11 @@
 CHAIN
 IF ~~ THEN C#SOLAUB aerie_08
 @1320
-== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1321
+== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1321
 == C#SOLAUB @1322
-== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1323
+== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1323
 == C#SOLAUB @1324
-== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @1325
+== BAERIE25 IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @1325
 == C#SOLAUB @1326
 = @1327
 EXIT
