--- c#solaufein/dialogues/c#solauj.D	2014-02-15 16:46:54.000000000 -0500
+++ Fixpack/c#solaufein/dialogues/c#solauj.D	2016-11-21 22:36:50.988371000 -0500
@@ -186,8 +186,8 @@
 
 IF ~~ THEN bhaal_heritage_01
 SAY @203
-IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN + bhaal_heritage_tob
-IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN + bhaal_heritage_01a
+IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN + bhaal_heritage_tob
+IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN + bhaal_heritage_01a
 END
 
 IF ~~ THEN bhaal_heritage_01a
@@ -216,8 +216,8 @@
 IF ~~ THEN C#SOLAUJ bhaal_heritage_03
 @216
 = @217
-== C#SOLAUJ IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN @218
-== C#SOLAUJ IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN @219
+== C#SOLAUJ IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @218
+== C#SOLAUJ IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @219
 END
 IF ~~ THEN DO ~IncrementGlobal("C#Solaufein_Dialog","GLOBAL",1)~ EXIT
 
@@ -856,13 +856,13 @@
 
 IF ~Global("C#Solaufein_Dialog","GLOBAL",28)~ THEN ansage
 SAY @422
-IF ~GlobalLT("Chapter","GLOBAL",8) ReputationGT(Player1,14)~ + rep_high
-IF ~GlobalLT("Chapter","GLOBAL",8) ReputationLT(Player1,15) GlobalGT("C#SolauFriendshipCounter","LOCALS",0)~ THEN + rep_low
-IF ~GlobalLT("Chapter","GLOBAL",8) ReputationLT(Player1,15) GlobalLT("C#SolauFriendshipCounter","LOCALS",1)~ THEN + both_low
-
-IF ~GlobalGT("Chapter","GLOBAL",7) ReputationGT(Player1,9)~ + rep_high
-IF ~GlobalGT("Chapter","GLOBAL",7) ReputationLT(Player1,10) GlobalGT("C#SolauFriendshipCounter","LOCALS",0)~ THEN + rep_low
-IF ~GlobalGT("Chapter","GLOBAL",7) ReputationLT(Player1,10) GlobalLT("C#SolauFriendshipCounter","LOCALS",1)~ THEN + both_low
+IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%) ReputationGT(Player1,14)~ + rep_high
+IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%) ReputationLT(Player1,15) GlobalGT("C#SolauFriendshipCounter","LOCALS",0)~ THEN + rep_low
+IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%) ReputationLT(Player1,15) GlobalLT("C#SolauFriendshipCounter","LOCALS",1)~ THEN + both_low
+
+IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%) ReputationGT(Player1,9)~ + rep_high
+IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%) ReputationLT(Player1,10) GlobalGT("C#SolauFriendshipCounter","LOCALS",0)~ THEN + rep_low
+IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%) ReputationLT(Player1,10) GlobalLT("C#SolauFriendshipCounter","LOCALS",1)~ THEN + both_low
 END
 
 IF ~~ THEN rep_high
