--- c#solaufein/dialogues/C#SOLAU_PID.D	2014-12-08 22:34:46.000000000 -0500
+++ Fixpack/c#solaufein/dialogues/C#SOLAU_PID.D	2016-11-21 22:36:53.840065300 -0500
@@ -178,16 +178,16 @@
 
 
 
-/* ToB - GlobalGT("Chapter","GLOBAL",7) */
+/* ToB - GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%) */
 
-+ ~GlobalGT("Chapter","GLOBAL",7) Global("ToBPID_unterwegs","LOCALS",0)
++ ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%) Global("ToBPID_unterwegs","LOCALS",0)
 GlobalGT("C#Solau_ElhanGeas","GLOBAL",0)~ + @1415 DO ~SetGlobal("ToBPID_unterwegs","LOCALS",1)~ + unterwegs
 
-+ ~GlobalGT("Chapter","GLOBAL",7) Global("ToBPID_unterwegs","LOCALS",0)
++ ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%) Global("ToBPID_unterwegs","LOCALS",0)
 Global("C#Solau_ElhanGeas","GLOBAL",0)~ + @1416 DO ~SetGlobal("ToBPID_unterwegs","LOCALS",1)~ + unterwegs_tob
 
 
-+ ~GlobalGT("Chapter","GLOBAL",7) Global("ToBPID_hoelle","LOCALS",0)~ + @1417 DO ~SetGlobal("ToBPID_hoelle","LOCALS",1)~ + dunkle_hoelle
++ ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%) Global("ToBPID_hoelle","LOCALS",0)~ + @1417 DO ~SetGlobal("ToBPID_hoelle","LOCALS",1)~ + dunkle_hoelle
 
 + ~GlobalGT("SarevokAlive","GLOBAL",0) Global("C#SolauPIDToB","LOCALS",0)
 !AreaCheck("AR5500")~ + @1418 DO ~IncrementGlobal("C#SolauPIDToB","LOCALS",1)~ + sarevok_comment
@@ -248,9 +248,9 @@
 Global("C#SolauNoScenery","GLOBAL",1)~ THEN BEGIN anrede_tob // from:
   SAY @1397
 
-+ ~GlobalGT("Chapter","GLOBAL",7) Global("ToBPID_hoelle","LOCALS",0)~ + @1417 DO ~SetGlobal("ToBPID_hoelle","LOCALS",1)~ + dunkle_hoelle
++ ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%) Global("ToBPID_hoelle","LOCALS",0)~ + @1417 DO ~SetGlobal("ToBPID_hoelle","LOCALS",1)~ + dunkle_hoelle
 
-+ ~GlobalGT("Chapter","GLOBAL",7) Global("ToBPID_unterwegs","LOCALS",0)
++ ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%) Global("ToBPID_unterwegs","LOCALS",0)
 Global("C#Solau_ElhanGeas","GLOBAL",0)~ + @1416 DO ~SetGlobal("ToBPID_unterwegs","LOCALS",1)~ + unterwegs_tob
 
 + ~GlobalGT("SarevokAlive","GLOBAL",0) Global("C#SolauPIDToB","LOCALS",0)
@@ -873,8 +873,8 @@
 
 IF ~~ THEN suche
 SAY @1626
-IF ~GlobalLT("Chapter","GLOBAL",8)~ THEN + suche_01
-IF ~GlobalGT("Chapter","GLOBAL",7)~ THEN + suche_02
+IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN + suche_01
+IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN + suche_02
 END
 
 IF ~~ THEN suche_01
