--- c#solaufein/setup-c#solaufein.tp2	2016-01-29 00:13:24.000000000 -0500
+++ Fixpack/c#solaufein/setup-c#solaufein.tp2	2016-11-22 19:30:49.771711100 -0500
@@ -5,16 +5,21 @@
 BACKUP ~C#Solaufein\Install\backup~
 AUTHOR ~jastey@web.de~ 
 
-
-VERSION ~Alpha_160128~
+VERSION ~Alpha_160128 BWP Fix~
 
 README ~C#Solaufein/readme.%LANGUAGE%.txt~ ~C#Solaufein/readme.german.txt~
 
-AUTO_TRA ~C#Solaufein\Translations\autotra/%s~
-
-
 ALWAYS
-
+  ACTION_IF GAME_IS ~eet~ BEGIN
+    OUTER_SET bg2_chapter = 12
+  END ELSE BEGIN
+    OUTER_SET bg2_chapter = 0
+  END
+  OUTER_FOR (i=1; i<=10; i=i+1) BEGIN
+    OUTER_SET bg2_chapter = bg2_chapter + 1
+    OUTER_SPRINT name_source ~bg2_chapter_%i%~
+    OUTER_SET EVAL ~%name_source%~ = bg2_chapter
+  END
 
     OUTER_SET SOLAUFEIN_TIMER = 1200
     OUTER_SET SOLAUFEIN_TIMERTOB = 900
@@ -29,30 +33,23 @@
     OUTER_SPRINT ~Drideranimation_female~ ~IC2_DRIDER_FEMALE~
 END
 
-
 /* all following actions are only processed ONCE for the whole mod, independent of un- and reinstalling of single components */
 
-ACTION_IF !FILE_EXISTS ~C#Solaufein\Install/c#solaufeinreinstall.mrk~ BEGIN
-
+  ACTION_IF !FILE_EXISTS ~C#Solaufein/Install/c#solaufeinreinstall.mrk~ BEGIN
   /*
    * Copies tra files into the autotra-folder (to leave the originals untouched)
    */
-  DEFINE_ACTION_FUNCTION autotra_workaround BEGIN
-    COPY ~C#Solaufein\Translations/%LANGUAGE%~  ~C#Solaufein\Translations/autotra/%LANGUAGE%~
-  END
-
-  LAF autotra_workaround END
-
-
-ACTION_DEFINE_ARRAY fl#noconvert BEGIN setup END
-
-ACTION_DEFINE_ARRAY fl#reload BEGIN game END
+    COPY ~C#Solaufein/Translations/%LANGUAGE%~  ~C#Solaufein/Translations/autotra/%LANGUAGE%~
 
+    // TRAs declared in LANGUAGE below must be reloaded after UTF-8 conversion
+    ACTION_DEFINE_ARRAY fl#reload BEGIN game setup END
+    ACTION_DEFINE_ARRAY fl#noconvert BEGIN END
+	// convert tra files to UTF-8 encoding - does nothing if game is not Enhanced Edition 
    LAF HANDLE_CHARSETS
     INT_VAR
-      infer_charset = 1
+        infer_charsets = 1
     STR_VAR
-      tra_path = EVAL ~C#Solaufein\Translations/autotra~
+        tra_path = EVAL ~C#Solaufein/Translations/autotra~
       noconvert_array = fl#noconvert
       reload_array = fl#reload
   END
@@ -57,30 +54,22 @@
       reload_array = fl#reload
   END
 
-
-
-
 COPY ~C#Solaufein/install/component.xx~ ~C#Solaufein\Install/c#solaufeinreinstall.mrk~
-
 END //c#solaufeinreinstall.mrk
 
-
-    //TRAs declared in LANGUAGE must be reloaded
-  LOAD_TRA ~C#Solaufein\Translations/autotra/%LANGUAGE%/game.tra~
-
-
 END //ALWAYS
 
+AUTO_TRA ~C#Solaufein/Translations/%s~
 
 LANGUAGE ~Deutsch~
          ~german~   
-         ~C#Solaufein\Translations\autotra/german\setup.tra~
-         ~C#Solaufein\Translations\autotra/german\game.tra~
+         ~C#Solaufein/Translations/german/setup.tra~
+         ~C#Solaufein/Translations/german/game.tra~
 
 LANGUAGE ~English (still great parts in German)~
          ~english~   
-         ~C#Solaufein\Translations\autotra/english\setup.tra~
-         ~C#Solaufein\Translations\autotra/english\game.tra~
+         ~C#Solaufein/Translations/english/setup.tra~
+         ~C#Solaufein/Translations/english/game.tra~
 
 
 BEGIN @90000
@@ -95,7 +84,7 @@
   COPY ~C#Solaufein/Install/component.xx~ ~override/C#SolaufeinCore.RR~
 
 
-ADD_JOURNAL TITLE (@1799) @67 @1795 @1796 @1797 @1798 USING ~C#Solaufein\Translations\autotra/%LANGUAGE%/C#SOLATRA.TRA~
+ADD_JOURNAL TITLE (@1799) @67 @1795 @1796 @1797 @1798 USING ~C#Solaufein/Translations/autotra/%LANGUAGE%/C#SOLATRA.TRA~
 
 
 
@@ -216,22 +205,22 @@
 END
 
 
-COMPILE ~C#Solaufein\Dialogues\C#SOLAU.d~
-USING ~C#Solaufein\Translations\autotra/%s\c#solatra.tra~ 
-COMPILE ~C#Solaufein\Dialogues\C#SOLAUB.d~
-USING ~C#Solaufein\Translations\autotra/%s\c#solatra.tra~ 
+COMPILE ~C#Solaufein/Dialogues/C#SOLAU.d~
+USING ~C#Solaufein/Translations/autotra/%LANGUAGE%/c#solatra.tra~ 
+COMPILE EVALUATE_BUFFER ~C#Solaufein/Dialogues/C#SOLAUB.d~
+USING ~C#Solaufein/Translations/autotra/%LANGUAGE%/c#solatra.tra~ 
 
 ACTION_IF GAME_IS ~bg2ee eet~ THEN BEGIN
-COMPILE ~C#Solaufein\Dialogues\C#SOLAUB_ee.d~
-USING ~C#Solaufein\Translations\autotra/%s\c#solatra.tra~ 
+COMPILE EVALUATE_BUFFER ~C#Solaufein/Dialogues/C#SOLAUB_ee.d~
+USING ~C#Solaufein/Translations/autotra/%LANGUAGE%/c#solatra.tra~ 
 END
 
-COMPILE ~C#Solaufein\Dialogues\C#SOLAUJ.d~
-USING ~C#Solaufein\Translations\autotra/%s\c#solatra.tra~ 
-COMPILE ~C#Solaufein\Dialogues\c#solaufein_interjections.d~
-USING ~C#Solaufein\Translations\autotra/%s\c#solatra.tra~ 
-COMPILE ~C#Solaufein\Dialogues\scenery_dialogues_soa.d~
-USING ~C#Solaufein\Translations\autotra/%s\c#solatra.tra~ 
+COMPILE EVALUATE_BUFFER ~C#Solaufein/Dialogues/C#SOLAUJ.d~
+USING ~C#Solaufein/Translations/autotra/%LANGUAGE%/c#solatra.tra~ 
+COMPILE ~C#Solaufein/Dialogues/c#solaufein_interjections.d~
+USING ~C#Solaufein/Translations/autotra/%LANGUAGE%/c#solatra.tra~ 
+COMPILE ~C#Solaufein/Dialogues/scenery_dialogues_soa.d~
+USING ~C#Solaufein/Translations/autotra/%LANGUAGE%/c#solatra.tra~ 
 
 
 /* ------------  tob ------------------ - */
@@ -239,18 +228,18 @@
 
 ACTION_IF GAME_IS ~tob bg2ee eet~ THEN BEGIN 
 
-  COMPILE ~C#Solaufein\Dialogues\C#SOLAU_ToB.d~
-USING ~C#Solaufein\Translations\autotra/%s\c#solatra.tra~ 
+  COMPILE ~C#Solaufein/Dialogues/C#SOLAU_ToB.d~
+    USING ~C#Solaufein/Translations/autotra/%LANGUAGE%/c#solatra.tra~ 
 
   //  epilogue: ##
-  EXTEND_TOP ~AR6200.bcs~ ~c#solaufein\scripts\solau_epilog_AR6200.baf~
+  EXTEND_TOP ~AR6200.bcs~ ~c#solaufein/scripts/solau_epilog_AR6200.baf~
 
   /* no friendship */
-  COPY ~c#solaufein\2da\c#soepxx.2da~ ~override/c#soep01.2da~
+  COPY ~c#solaufein/2da/c#soepxx.2da~ ~override/c#soep01.2da~
   REPLACE ~99999~ @3
 
   /* friendship */
-  COPY ~c#solaufein\2da\c#soepxx.2da~ ~override/c#soep02.2da~
+  COPY ~c#solaufein/2da/c#soepxx.2da~ ~override/c#soep02.2da~
   REPLACE ~99999~ @4
 
 END
@@ -259,47 +248,47 @@
 
 
 //The PID, to be compiled as last dialogue file
- COMPILE ~C#Solaufein\Dialogues\C#SOLAU_PID.d~ 
-USING ~C#Solaufein\Translations\autotra/%s\c#solatra.tra~ 
+COMPILE EVALUATE_BUFFER ~C#Solaufein/Dialogues/C#SOLAU_PID.d~ 
+  USING ~C#Solaufein/Translations/autotra/%LANGUAGE%/c#solatra.tra~ 
 
 
-COMPILE EVALUATE_BUFFER ~C#Solaufein\Scripts\C#SOLAU.baf~
+COMPILE EVALUATE_BUFFER ~C#Solaufein/scripts/C#SOLAU.baf~
 
 
-COMPILE ~C#Solaufein\Scripts\C#SOLAU1.baf~
-COMPILE ~C#Solaufein\Scripts\c#socut1.baf~
-COMPILE ~C#Solaufein\Scripts\c#socut2.baf~
+COMPILE ~C#Solaufein/scripts/C#SOLAU1.baf~
+COMPILE ~C#Solaufein/scripts/c#socut1.baf~
+COMPILE ~C#Solaufein/scripts/c#socut2.baf~
 
 
-COMPILE ~C#Solaufein\Scripts\c#sodrsc.baf~
-COMPILE ~C#Solaufein\Scripts\c#sodrf0.baf~
-COMPILE ~C#Solaufein\Scripts\c#sodrm1.baf~
+COMPILE ~C#Solaufein/scripts/c#sodrsc.baf~
+COMPILE ~C#Solaufein/scripts/c#sodrf0.baf~
+COMPILE ~C#Solaufein/scripts/c#sodrm1.baf~
 
-COMPILE ~C#Solaufein\Scripts\c#so01.baf~
+COMPILE ~C#Solaufein/scripts/c#so01.baf~
 
-COMPILE ~C#Solaufein\Scripts\c#sopass.baf~
-USING ~C#Solaufein\Translations\autotra/%s\c#solatra.tra~ 
-COMPILE ~C#Solaufein\Scripts\c#sococ1.baf~
-USING ~C#Solaufein\Translations\autotra/%s\c#solatra.tra~ 
-COMPILE ~C#Solaufein\Scripts\c#sococ2.baf~
-USING ~C#Solaufein\Translations\autotra/%s\c#solatra.tra~ 
-COMPILE ~C#Solaufein\Scripts\c#sococ3.baf~
-USING ~C#Solaufein\Translations\autotra/%s\c#solatra.tra~ 
-COMPILE ~C#Solaufein\Scripts\c#sococ4.baf~
-USING ~C#Solaufein\Translations\autotra/%s\c#solatra.tra~ 
-COMPILE ~C#Solaufein\Scripts\c#sococ5.baf~
-USING ~C#Solaufein\Translations\autotra/%s\c#solatra.tra~ 
-COMPILE ~C#Solaufein\Scripts\c#sococ6.baf~
-USING ~C#Solaufein\Translations\autotra/%s\c#solatra.tra~ 
-COMPILE ~C#Solaufein\Scripts\c#sobrut.baf~
-USING ~C#Solaufein\Translations\autotra/%s\c#solatra.tra~ 
+COMPILE ~C#Solaufein/scripts/c#sopass.baf~
+  USING ~C#Solaufein/Translations/autotra/%LANGUAGE%/c#solatra.tra~ 
+COMPILE ~C#Solaufein/scripts/c#sococ1.baf~
+  USING ~C#Solaufein/Translations/autotra/%LANGUAGE%/c#solatra.tra~ 
+COMPILE ~C#Solaufein/scripts/c#sococ2.baf~
+  USING ~C#Solaufein/Translations/autotra/%LANGUAGE%/c#solatra.tra~ 
+COMPILE ~C#Solaufein/scripts/c#sococ3.baf~
+  USING ~C#Solaufein/Translations/autotra/%LANGUAGE%/c#solatra.tra~ 
+COMPILE ~C#Solaufein/scripts/c#sococ4.baf~
+  USING ~C#Solaufein/Translations/autotra/%LANGUAGE%/c#solatra.tra~ 
+COMPILE ~C#Solaufein/scripts/c#sococ5.baf~
+  USING ~C#Solaufein/Translations/autotra/%LANGUAGE%/c#solatra.tra~ 
+COMPILE ~C#Solaufein/scripts/c#sococ6.baf~
+  USING ~C#Solaufein/Translations/autotra/%LANGUAGE%/c#solatra.tra~ 
+COMPILE ~C#Solaufein/scripts/c#sobrut.baf~
+  USING ~C#Solaufein/Translations/autotra/%LANGUAGE%/c#solatra.tra~ 
 
 
-EXTEND_TOP ~AR2100.bcs~ ~C#Solaufein\Scripts\C#Solau_AR2100.baf~
+EXTEND_TOP ~AR2100.bcs~ ~C#Solaufein/scripts/C#Solau_AR2100.baf~
 
-EXTEND_TOP ~AR2401.bcs~ ~C#Solaufein\Scripts\C#Solau_AR2401.baf~
+EXTEND_TOP ~AR2401.bcs~ ~C#Solaufein/scripts/C#Solau_AR2401.baf~
 
-EXTEND_TOP ~AR2500.bcs~ ~C#Solaufein\Scripts\C#Solau_AR2500.baf~
+EXTEND_TOP ~AR2500.bcs~ ~C#Solaufein/scripts/C#Solau_AR2500.baf~
 
 
 
@@ -307,7 +296,7 @@
 
 /* Solaufein kit */
 
-INCLUDE ~C#Solaufein\lib/fl#add_kit_ee.tpa~
+INCLUDE ~C#Solaufein/lib/fl#add_kit_ee.tpa~
 
 ADD_KIT ~C#SoDrow~ ///Solaufein's drow kit
   // appended to CLASWEAP.2da
@@ -384,9 +373,7 @@
 COPY ~C#Solaufein/Portraits/C#SolauM.bmp~ ~override/C#SolauM.bmp~
 COPY ~C#Solaufein/Portraits/C#SolauS.bmp~ ~override/C#SolauS.bmp~
 
-END
-
-ACTION_IF GAME_IS ~bg2ee eet~ BEGIN
+END ELSE ACTION_IF GAME_IS ~bg2ee eet~ BEGIN
 
 COPY ~C#Solaufein/Portraits/C#SolauL.bmp~ ~override/C#SolauL.bmp~
 COPY ~C#Solaufein/Portraits/Solaufein_EE/C#SolauM.bmp~ ~override/C#SolauM.bmp~
@@ -620,7 +607,7 @@
 COPY ~C#Solaufein/animations/ee/sound~ ~override~
 COPY ~C#Solaufein/animations/ee/bam~ ~override~
 
-EXTEND_BOTTOM ~C#SO01.bcs~ ~c#solaufein\scripts\C#So01_driderchange.baf~ EVALUATE_BUFFER
+EXTEND_BOTTOM ~C#SO01.bcs~ ~c#solaufein/scripts/C#So01_driderchange.baf~ EVALUATE_BUFFER
 
 COPY_EXISTING ~C#sococ2.bcs~ ~override~
 DECOMPILE_AND_PATCH BEGIN
@@ -643,12 +630,27 @@
 END
 BUT_ONLY_IF_IT_CHANGES
 
+END // GAME_IS ~bg2ee eet~
 
 
 
-
+ACTION_IF GAME_IS ~eet~ BEGIN
+  INCLUDE ~EET/other/EET_functions.tph~
+  LAF ~EET_NPC_TRANSITION~
+    INT_VAR
+      type = 2
+    STR_VAR
+      dv = "C#Solaufein"
+      override_SoA = "C#SOLAU"
+      override_ToB = "C#SOLAU"
+      dialog_ToB = "C#SOLA25"
+      traFile = EVAL "C#Solaufein/Translations/autotra/%LANGUAGE%/c#solatra.tra"
+      string = "@1328" //Bring me Solaufein, the drow fighter, who I saved from the drider cocoons.
+      stringPosDV = "Tiax"
+  END
+END ELSE BEGIN
+  COMPILE ~c#solaufein/Dialogues/FATESP.d~ USING ~C#Solaufein/Translations/autotra/%LANGUAGE%/c#solatra.tra~
 END
-
 
 
 /*
@@ -692,7 +694,7 @@
 /* BGII / BGT version: IA needs to be installed for the drider animations */
 
 
-EXTEND_BOTTOM ~C#SO01.bcs~ ~c#solaufein\scripts\C#So01_driderchange.baf~ EVALUATE_BUFFER
+EXTEND_BOTTOM ~C#SO01.bcs~ ~c#solaufein/scripts/C#So01_driderchange.baf~ EVALUATE_BUFFER
 
 COPY_EXISTING ~C#sococ2.bcs~ ~override~
 DECOMPILE_AND_PATCH BEGIN
