--- NEJ\toda\Animate.ids	Wed Feb 02 11:31:04 2005
+++ C:\BWP Patchstudio\patched files\NEJ\toda\Animate.ids	Fri Nov 21 06:43:56 2008
@@ -297,8 +297,8 @@
 0xE090 IC_MERILITH
 0xE0A0 IC_ICE_TROLL
 0xE0B0 IC_TROLL1
-0XE0C0 IC_SNOW_TROLL
-0XE0D0 IC_UMBERHULK     
+0xE0C0 IC_SNOW_TROLL
+0xE0D0 IC_UMBERHULK     
 0xE0E0 IC_CORNUGONSKI                       
 0xE0F0 IC_ICE_GOLEM
 0xE0F1 IC_GLAB
