--- TotDG/Setup-TotDG.tp2	2015-01-28 00:53:58.000000000 -0500
+++ Fixpack/TotDG/Setup-TotDG.tp2	2016-11-30 17:30:56.072744400 -0500
@@ -2,11 +2,18 @@
 AUTHOR ~Lava Del'Vortel~
 README ~TotDG/totdg_readme.html~
 
+VERSION ~10.1 BWP Fix~
+
 AUTO_TRA ~TotDG/Translations/%s~
 
 ALWAYS
-  // Borrowed from Edwin Romance: convert strings to UTF-8 for BGEE/BG2EE
-  INCLUDE ~%MOD_FOLDER%/handle_charsets.tpa~
+  ACTION_IF GAME_IS ~eet~ BEGIN
+    OUTER_SET wmp_xCoordAdd_SoA = 0
+    OUTER_SET wmp_yCoordAdd_SoA = 2283
+  END ELSE BEGIN
+    OUTER_SET wmp_xCoordAdd_SoA = 0
+    OUTER_SET wmp_yCoordAdd_SoA = 0
+  END
 
   // No exception, everything needs to be converted
   ACTION_DEFINE_ARRAY 0xnoconvert BEGIN END
@@ -14,10 +21,9 @@
   // Only setup.tra needs to be reloaded
   ACTION_DEFINE_ARRAY 0xreload BEGIN Setup END
 
-  // There is no "language" folder so the root of all languages in %MOD_FOLDER%
   LAF HANDLE_CHARSETS
     INT_VAR
-      infer_charset = 1
+      infer_charsets = 1
     STR_VAR
       tra_path = EVAL ~%MOD_FOLDER%/translations~
       // charset_table = 0xcharsets // Included for illustrative purposes.
@@ -405,12 +411,13 @@
     
 /////////////////////////////////SOUNDS
 
-COPY ~TotDG/SOUNDS~ ~override/~
-
-ACTION_IF ("%LANGUAGE%" STRING_COMPARE_CASE "Polish" =0) BEGIN
  
-COPY ~TotDG\DeepAudio\Polish~ ~override\~
+ACTION_IF ("%LANGUAGE%" STR_EQ "Polish") THEN BEGIN
+  COPY ~TotDG/DeepAudio/Polish~ ~TotDG/SOUNDS~
+END
 
+LAF HANDLE_AUDIO
+  STR_VAR audio_path = ~TotDG/SOUNDS~
 END
 
 /////////////////////////////////SPELLS
@@ -808,70 +815,42 @@
 COPY_EXISTING ~merchant.sto~ ~override/merchant.sto~
 ADD_STORE_ITEM ~DGxQ1~ #10 #0 #0 ~IDENTIFIED~ #5
 
-
-//////////////////////////////////AUDIO INSTALLATION
-
-COPY   ~TotDG/TotDGIA.bat~          ~TotDGIA.bat~
-COPY   ~TotDG/TotDGUA.bat~         ~TotDGUA.bat~
-    
-AT_INTERACTIVE_EXIT ~TotDGIA.bat~
-AT_UNINSTALL        ~TotDGUA.bat~
-
-//////////////////////////////////Possible Content
-
-BEGIN @180
-REQUIRE_COMPONENT ~TotDG/Setup-TotDG.tp2~ ~0~ @181
-
-COMPILE ~TotDG/DIALOGUES/DialApp.d~
-EXTEND_BOTTOM   ~ARLDV1.bcs~ ~TotDG/SCRIPTS/ARLDV1AddOn.baf~
-EXTEND_BOTTOM   ~AR1000.bcs~ ~TotDG/SCRIPTS/AR1000AddOn.baf~
-EXTEND_BOTTOM   ~AR0900.bcs~ ~TotDG/SCRIPTS/AR0900AddOn.baf~
-
-BEGIN @182
-REQUIRE_COMPONENT ~TotDG/Setup-TotDG.tp2~ ~0~ @181
-
 //////////////////////////////////ADD AREA TO MAP
 
-MKDIR save
-
-INCLUDE ~TotDG/wmp/worldmap.tph~
+// Append mastarea.2da with the new area
+APPEND ~MASTAREA.2da~ ~ARLDV1 value~ UNLESS ~ARLDV1~ //Deep Gardens
 
+// BP-BGT-Worldmap compatibility always (these files will only be used if BP-BGT-Worldmap is installed)
+ACTION_IF FILE_EXISTS ~Worldmap/map_mods_areas.tbl~ THEN BEGIN
+  COPY   ~Worldmap/map_mods_areas.tbl~   ~Worldmap~   APPEND_FILE ~TotDG/Worldmap/areas.tbl~
+  COPY   ~Worldmap/map_mods_links.tbl~   ~Worldmap~   APPEND_FILE ~TotDG/Worldmap/links.tbl~
+  COPY   ~Worldmap/map_mods_trans.tra~   ~Worldmap~   APPEND_FILE ~TotDG/Worldmap/worldmap.tra~
+END ELSE BEGIN
+  MKDIR  ~Worldmap~
+  COPY   ~TotDG/Worldmap/areas.tbl~      ~Worldmap/map_mods_areas.tbl~
+  COPY   ~TotDG/Worldmap/links.tbl~      ~Worldmap/map_mods_links.tbl~
+  COPY   ~TotDG/Worldmap/worldmap.tra~   ~Worldmap/map_mods_trans.tra~
+END
 
-//**************************************************
-// Worldmap patching - do this last so we can patch the new areas too
-//**************************************************
-// Append mastarea.2da with the new areas
-APPEND ~MASTAREA.2da~ ~ARLDV1 value~ UNLESS ~ARLDV1~ //DG
-
+// Always patch regular worldmap (no conflict with BP-BGT-Worldmap)
 //~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-// Deep Gardens abandoned
-  OUTER_SPRINT areName "ARLDV1"   // area reference, like "AR0700"
-  OUTER_SPRINT strName "Deep Gardens" // area name, like "Waukeen's Promenade"
-  OUTER_SPRINT strDesc "Deep Gardens" // area description which will show up when hovering the area on the worldmap
-
-  OUTER_SET mapIcon = 9              // map icon
-  OUTER_SET xCoord  = 490            // x coordinate
-  OUTER_SET yCoord  = 40             // y coordinate
-  OUTER_SET tTime   = 2              // travel time, *4, so two means eight hours
-  OUTER_SET inclSv  = 1              // include save games bool, 1 means yes and 0 means no
-
 // set up link variables
 
   // add links from these area(s) to the new area
   ACTION_DEFINE_ASSOCIATIVE_ARRAY toNewArea BEGIN
-    "AR0020" => "Exitne"       // City Gates
-    "AR0300" => "Exitne"       // Docks
-    "AR0400" => "Exitne"       // Slums
-    "AR0500" => "Exitne"       // Bridge District
-    "AR0700" => "Exitne"       // Waukeen's Promenade
-    "AR0800" => "Exitne"       // Graveyard
-    "AR0900" => "Exitne"       // Temple District
-    "AR1000" => "Exitne"       // Government
-    "AR1100" => "Exitne"       // Umar Hills
-    "AR1300" => "Exitne"       // d'Arnise Keep
-    "AR1304" => "Exitne"       // d'Arnise Keep
-    "AR2000" => "Exite"        // Trademeet
-    "AR1200" => "Exite"        // Windspear Hills
+  "AR0020" => "ExitNE"       // City Gates
+  "AR0300" => "ExitNE"       // Docks
+  "AR0400" => "ExitNE"       // Slums
+  "AR0500" => "ExitNE"       // Bridge District
+  "AR0700" => "ExitNE"       // Waukeen's Promenade
+  "AR0800" => "ExitNE"       // Graveyard
+  "AR0900" => "ExitNE"       // Temple District
+  "AR1000" => "ExitNE"       // Government
+  "AR1100" => "ExitNE"       // Umar Hills
+  "AR1300" => "ExitNE"       // d'Arnise Keep
+  "AR1304" => "ExitNE"       // d'Arnise Keep
+  "AR2000" => "ExitE"        // Trademeet
+  "AR1200" => "ExitE"        // Windspear Hills
   END
 
   // add links from the new area to these area(s)
@@ -891,25 +870,42 @@
     "AR1200" => "ExitW"       // Windspear Hills
   END
 
-// call the sc#addWmpAre function
-  // returns the worldmap entry number for the new area
-  LAUNCH_ACTION_FUNCTION sc#addWmpAre RET areNum = areNum END
-
-  // remove if not desired
-  PRINT "ARLDV1 patched. New area number is: %areNum%"
-
-
-
+OUTER_SPRINT areName "ARLDV1"                // area reference, like "AR0700"
 
+OUTER_SET mapIcon = 9                        // map icon index in mapicons.bam
+OUTER_SET xCoord  = 505 + wmp_xCoordAdd_SoA  // x coordinate (+ EET offset)
+OUTER_SET yCoord  = 40  + wmp_yCoordAdd_SoA  // y coordinate (+ EET offset)
+OUTER_SET tTime   = 4                        // travel time, *4, so two means eight hours
 
+// call the sc#addWmpAre function (included with WeiDU)
+// returns the worldmap entry number for the new area
+LAUNCH_ACTION_FUNCTION sc#addWmpAre
+  INT_VAR
+    inclSv  = 1              // patch worldmap for any existing save games
+  STR_VAR
+    strName = "Deep Gardens" // area name, like "Waukeen's Promenade"
+    strDesc = "Deep Gardens" // area description which will show up when hovering the area on the worldmap
+  RET areNum = areNum
+END
 
 
+// end of main component
 
 
+//////////////////////////////////Possible Content
 
+BEGIN @180
+REQUIRE_COMPONENT ~TotDG/Setup-TotDG.tp2~ ~0~ @181
 
+COMPILE ~TotDG/DIALOGUES/DialApp.d~
+EXTEND_BOTTOM   ~ARLDV1.bcs~ ~TotDG/SCRIPTS/ARLDV1AddOn.baf~
+EXTEND_BOTTOM   ~AR1000.bcs~ ~TotDG/SCRIPTS/AR1000AddOn.baf~
+EXTEND_BOTTOM   ~AR0900.bcs~ ~TotDG/SCRIPTS/AR0900AddOn.baf~
 
 
 
 
+BEGIN ~~ // @182 Add to Worldmap
+DEPRECATED ~Deprecated: Add to worldmap is now in main component~
+//REQUIRE_COMPONENT ~TotDG/Setup-TotDG.tp2~ ~0~ @181
 
