--- TOD\ImpASG\va#maril.baf	Sun Sep 13 14:32:52 2009
+++ C:\BWP Patchstudio\patched files\TOD\ImpASG\va#maril.baf	Wed Sep 19 06:34:46 2012
@@ -198,8 +198,8 @@
 	OR(4)
 		SpellCast([GOODCUTOFF],WIZARD_TRUE_SIGHT)
 		SpellCastPriest([GOODCUTOFF],CLERIC_TRUE_SIGHT)
-		SpellCastInnate([GOODCUTOFF],HELM_TRUE_SIGHT)
-		SpellCastInnate([GOODCUTOFF],INQUIS_TRUE_SIGHT)
+		SpellCast([GOODCUTOFF],HELM_TRUE_SIGHT)
+		SpellCast([GOODCUTOFF],INQUIS_TRUE_SIGHT)
 THEN
 	RESPONSE #100
 		SetGlobalTimer("PC_True_Sight","MYAREA",60)
