--- TOD\Baf\va#shade.baf	Sun Sep 13 14:06:04 2009
+++ C:\BWP Patchstudio\patched files\TOD\Baf\va#shade.baf	Sun Mar 07 03:42:44 2010
@@ -3681,7 +3681,7 @@
 		StateCheck(SixthNearest([PC]),RZ_STATE_DISABLED)
 		!CheckStat(SixthNearest([PC]),0,SANCTUARY)
                 StateCheck(SixthNearest([PC]),STATE_INVISIBLE)
-        OR(8)
+        OR(7)
 		RandomNumGT(30,15)
 		!Heard([ENEMY],101)
 		Range(LastHeardBy(Myself),0)
@@ -3689,7 +3689,7 @@
 		!Range(LastAttackerOf(LastHeardBy(Myself)),20)
 		!CheckStat(LastAttackerOf(LastHeardBy(Myself)),0,SANCTUARY)
                 StateCheck(LastAttackerOf(LastHeardBy(Myself)),STATE_INVISIBLE)
-	OR(6)
+	OR(5)
 		RandomNumGT(34,17)
 		!See(LastAttackerOf(Myself))
 		!Range(LastAttackerOf(Myself),10)
