--- dsotsc\setup-dsotsc.tp2	Thu Jan 03 11:07:51 2013
+++ C:\BWP Patchstudio\patched files\dsotsc\setup-dsotsc.tp2	Sun Jan 27 08:24:35 2013
@@ -1,5 +1,5 @@
 // Dark Side of the Sword Coast for BGT-WeiDU
-// Original creator of DSotSC: C 'choinneach
+// Original creator of DSotSC: Cú'choinneach
 // BGT version by SirBillyBob
 // Maintainers: erebusant (up to version 215), Turambar
 
@@ -3904,7 +3904,7 @@
 COPY ~DSotSC/portralt/conchol.bmp~ ~override~
      ~DSotSC/portralt/conchos.bmp~ ~override~
 
-//! Alternate C Choinneach Portrait /////////////////////////////////////////
+//! Alternate Cú'choinneach Portrait /////////////////////////////////////////
 BEGIN @2544
 DESIGNATED 430
 GROUP @2540
