--- c#sb_silber/setup-c#sb_silber.tp2	2013-11-21 22:10:26.000000000 -0500
+++ Fixpack/c#sb_silber/setup-c#sb_silber.tp2	2016-11-22 19:16:08.584932600 -0500
@@ -1,7 +1,7 @@
 BACKUP ~C#SB_silber/backup~
 AUTHOR ~jastey@web.de~ 
 
-VERSION ~13~
+VERSION ~v13 BWP Fix~
 
 README ~C#SB_silber/readme.%LANGUAGE%.txt~ ~C#SB_silber/readme.english.txt~
 
@@ -10,7 +10,7 @@
 
 
 ALWAYS
-  ACTION_IF GAME_IS ~bg2ee~ BEGIN
+  ACTION_IF GAME_IS ~bg2ee eet~ BEGIN
   /*
    * Copies UTF8-encoded tra files over the normal ones, for use with AUTO_TRA
    */
@@ -19,6 +19,8 @@
   END
   LAF autotra_workaround END
 
+    // must also reload tra files used in LANGUAGE after replacing with UTF-8
+    LOAD_TRA ~C#SB_silber/translations/%LANGUAGE%/setup.tra~
   END
 END
 
@@ -222,12 +216,12 @@
 //as well as add new points, AS LONG AS EVERYTHING FOLLOWS THE PATTERN!
 
 COPY_EXISTING ~baldur.bcs~ ~override~
-DECOMPILE_BCS_TO_BAF
+  DECOMPILE_AND_PATCH BEGIN
 
 /*Adds a trigger variable to the "udEscape" timer so the turning hostile of Ust Natha can be triggered by the mod:*/
 REPLACE_TEXTUALLY ~\(GlobalTimerExpired("udEscape","GLOBAL")\)~
 ~\1 Global("C#SB_DrowCityHostile","GLOBAL",1)~
-COMPILE_BAF_TO_BCS
+  END
 BUT_ONLY_IF_IT_CHANGES
 
 
