--- c#sb_silber/dialoge/silberdrachenblut.d	2009-06-22 20:55:26.000000000 -0400
+++ Fixpack/c#sb_silber/dialoge/silberdrachenblut.d	2016-11-26 02:07:14.706705300 -0500
@@ -108,7 +108,7 @@
 EraseJournalEntry(@32)
 EraseJournalEntry(@49)
 SetGlobal("C#SB_DrowCityHostile","GLOBAL",1)~ SOLVED_JOURNAL @17 + silver_04
-+ ~Global("DrowCityHostile","GLOBAL",1)~ + @79 DO ~SetGlobal("C#SB_AbolethKampf","GLOBAL",0)~ DO ~EraseJournalEntry(@21)
++ ~Global("DrowCityHostile","GLOBAL",1)~ + @79 DO ~SetGlobal("C#SB_AbolethKampf","GLOBAL",0) EraseJournalEntry(@21)
 EraseJournalEntry(@23)
 EraseJournalEntry(@26)
 EraseJournalEntry(@32)
