--- TS\bgtnodlg\aBVALYGA.D	Sat Sep 19 23:36:03 2009
+++ C:\BWP Patchstudio\patched files\TS\bgtnodlg\aBVALYGA.D	Sun Jan 19 21:44:30 2014
@@ -1330,7 +1330,7 @@
 RealSetGlobalTimer("VP_ValygarRomance","GLOBAL",30)
 SetGlobal("LoveTalk","LOCALS",43)
 DestroyItem("ValNeck1")
-GiveItemCreate("ValNeck2",Protagonist,0,0,0)
+GiveItemCreate("ValNeck2",Player1,0,0,0)
 RestParty()~ EXIT
 END
 
@@ -1338,7 +1338,7 @@
   SAY ~Well, yes, it’s yours, do what you wish with it. Yet it would look beautiful on you.~
   IF ~~ THEN DO ~SetGlobal("ValygarRomanceActive","GLOBAL",3)
 DestroyItem("ValNeck1")
-GiveItemCreate("ValNeck2",Protagonist,0,0,0)
+GiveItemCreate("ValNeck2",Player1,0,0,0)
 RestParty()~ EXIT
 END
 
