--- Setup-TS.tp2	Wed Sep 23 22:53:26 2009
+++ C:\BWP Patchstudio\patched files\Setup-TS.tp2	Mon Jun 29 22:51:49 2015
@@ -1,7 +1,7 @@
 BACKUP ~TS/backup~
 
 AUTHOR ~Vlad & Domi~
-
+VERSION ~v7.04 BWP Fix~
 BEGIN ~TORTURED SOULS v7.04~
 
 //______________________________________//
@@ -2692,7 +2692,7 @@
   SAY NAME1 ~Sol~
   SAY NAME2 ~Sol~
  
-COPY~TS/cre/Sola.cre~         ~override/Sola.cre~
+COPY~TS/cre/SolaTS.cre~         ~override/SolaTS.cre~
   SAY NAME1 ~Sola~
   SAY NAME2 ~Sola~
  
@@ -3312,20 +3312,20 @@
 //Adding music
 ACTION_IF FILE_EXISTS_IN_GAME ~Bag02NeJ.ITM~
  THEN BEGIN
-		COPY_EXISTING ~SONGLIST.2da~ ~override~
+//		COPY_EXISTING ~SONGLIST.2da~ ~override~
 
-			APPEND ~SONGLIST.2da~
-				~85      Kachiko         MXKACHI.MUS~
+//			APPEND ~SONGLIST.2da~
+//				~85      Kachiko         MXKACHI.MUS~
 
-			APPEND ~SONGLIST.2da~
-				~86      Valygar         MXVALYG.MUS~
+//			APPEND ~SONGLIST.2da~
+//				~86      Valygar         MXVALYG.MUS~
 
-			APPEND ~SONGLIST.2da~
-				~87      Sime            MXSIMEL.MUS~
+//			APPEND ~SONGLIST.2da~
+//				~87      Sime            MXSIMEL.MUS~
 
-		COPY ~TS/music/MxKachi.mus~ ~music~
-		COPY ~TS/music/MxSimel.mus~ ~music~
-		COPY ~TS/music/MxValyg.mus~ ~music~
+		ADD_MUSIC ~Kachiko~	~TS/music/MxKachi.mus~
+		ADD_MUSIC ~Valygar~	~TS/music/MxSimel.mus~
+		ADD_MUSIC ~Sime~	~TS/music/MxValyg.mus~
 
 		MKDIR ~music/MxKachi~
 		MKDIR ~music/MxSimel~
@@ -3349,20 +3349,20 @@
 		COMPILE ~TS/music/zbaf~
 END
 	ELSE BEGIN
-		COPY_EXISTING ~SONGLIST.2da~ ~override~
+//		COPY_EXISTING ~SONGLIST.2da~ ~override~
 
-			APPEND ~SONGLIST.2da~
-				~84      Kachiko         MXKACHI.MUS~
+//			APPEND ~SONGLIST.2da~
+//				~84      Kachiko         MXKACHI.MUS~
 
-			APPEND ~SONGLIST.2da~
-				~85      Valygar         MXVALYG.MUS~
+//			APPEND ~SONGLIST.2da~
+//				~85      Valygar         MXVALYG.MUS~
 
-			APPEND ~SONGLIST.2da~
-				~86      Sime            MXSIMEL.MUS~
+//			APPEND ~SONGLIST.2da~
+//				~86      Sime            MXSIMEL.MUS~
 
-		COPY ~TS/music/MxKachi.mus~ ~music~
-		COPY ~TS/music/MxSimel.mus~ ~music~
-		COPY ~TS/music/MxValyg.mus~ ~music~
+		ADD_MUSIC ~Kachiko~	~TS/music/MxKachi.mus~
+		ADD_MUSIC ~Valygar~	~TS/music/MxSimel.mus~
+		ADD_MUSIC ~Sime~	~TS/music/MxValyg.mus~
 
 		MKDIR ~music/MxKachi~
 		MKDIR ~music/MxSimel~
