--- NTotSC\BAF\Area\AR7400.baf	Thu Apr 27 11:52:06 2006
+++ C:\BWP Patchstudio\patched files\NTotSC\BAF\Area\AR7400.baf	Wed Aug 26 23:19:40 2015
@@ -12,6 +12,7 @@
     TimeGT(20)
     TimeLT(4)
   Exists("NTRededg")
+  !IsActive("NTRededg")
 THEN
   RESPONSE #100
     Activate("NTRededg")
@@ -21,6 +22,7 @@
   TimeGT(4)
   TimeLT(20)
   Exists("NTRededg")
+  IsActive("NTRededg")
 THEN
   RESPONSE #100
     Deactivate("NTRededg")
