--- NTotSC\Worldmap\links.tbl	Mon Jan 01 15:39:46 2007
+++ C:\BWP Patchstudio\patched files\NTotSC\Worldmap\links.tbl	Tue Feb 03 14:13:40 2009
@@ -5,7 +5,7 @@
 ARU000    N         AR20PB      EXITU000    5         1          N        N        N        N        N        0
 ARU000    E         AR10PB      Exit20PB    6         1          N        N        N        N        N        0
 ARU000    S         AR7500      Exit10PB    4         1          N        N        N        N        N        0
-AR7500    N         AR10PB      Exit7500    4         1          N        N        N        N        N        0
+AR7500    N         AR10PB      Exit20PB    4         1          N        N        N        N        N        0
 AR7500    N         AR01PB      Exit30PB    6         1          N        N        N        N        N        0
 AR7500    E         AR30PB      Exit7500    5         1          N        N        N        N        N        0
 AR9000    E         AR60PB      Exit9000    6         1          N        N        N        N        N        0
