--- setup-wmart.tp2	Mon Aug 18 12:43:00 2014
+++ C:\BWP Patchstudio\patched files\setup-wmart.tp2	Sat Aug 08 20:46:20 2015
@@ -1,6 +1,6 @@
 BACKUP ~wmart/backup~
 AUTHOR ~"http://www.teambg.net"~
-
+VERSION ~v1 BWP Fix~
 // Language Options
 //
 // This section is optional. If you list more than one language, the
@@ -25,14 +25,14 @@
 
 /*	WMART1.CRE
 	Joluv       	*/
-COPY ~wmart/cre/WMART1.CRE~ ~Override~
+COPY ~wmart/cre/WMART1_.CRE~ ~Override~
   SAY 0x8 @1
   SAY 0xC @1
 
 
 /*	WMART2.CRE
 	Deidre      	*/
-COPY ~wmart/cre/WMART2.CRE~ ~Override~
+COPY ~wmart/cre/WMART2_.CRE~ ~Override~
   SAY 0x8 @2
   SAY 0xC @2
 
@@ -45,11 +45,11 @@
  
 
 
-COPY ~wmart/sto/WMART1.STO~ ~Override~
+COPY ~wmart/sto/WMART1_.STO~ ~Override~
   SAY 0xC @81
 
 
-COPY ~wmart/sto/WMART2.STO~ ~Override~
+COPY ~wmart/sto/WMART2_.STO~ ~Override~
   SAY 0xC @81
 
 
@@ -57,15 +57,15 @@
 
 
 COMPILE
-  ~wmart/dlg/wmart1.d~
+  ~wmart/dlg/wmart1_.d~
 
 COMPILE
-  ~wmart/dlg/wmart2.d~
+  ~wmart/dlg/wmart2_.d~
 
 /* ******************************* scripts ********* */
   
-COMPILE
-  ~wmart/baf/AR7703.baf~
+EXTEND_BOTTOM ~AR7703.bcs~  ~wmart/baf/AR7703.baf~
+
 
 
 EXTEND_BOTTOM ~AR7705.bcs~  ~wmart/baf/AR7705.baf~
