--- npckit\lib\always.tpa	Mon Jul 16 15:17:26 2007
+++ C:\BWP Patchstudio\patched files\npckit\lib\always.tpa	Wed Oct 24 04:20:25 2012
@@ -37,15 +37,15 @@
     REPLACE_TEXTUALLY ~.*\bTOTEMIC\b~                ~0x4010 TOTEMIC~
     REPLACE_TEXTUALLY ~.*\bSHAPESHIFTER\b~           ~0x4011 SHAPESHIFTER~
     REPLACE_TEXTUALLY ~.*\bBEASTFRIEND\b~            ~0x4012 BEASTFRIEND~
-    REPLACE_TEXTUALLY ~.*\bBARBARIAN\b~              ~0x0000 BARBARIAN~
-    REPLACE_TEXTUALLY ~.*\bWILDMAGE\b~               ~0x0000 WILDMAGE~
+    REPLACE_TEXTUALLY ~.*\bBARBARIAN\b~              ~0x40000000 BARBARIAN~
+    REPLACE_TEXTUALLY ~.*\bWILDMAGE\b~               ~0x80000000 WILDMAGE~
     BUT_ONLY_IF_IT_CHANGES
   
   APPEND ~kit.ids~ ~0x4000 TRUECLASS~
     UNLESS ~\bTRUECLASS\b~
-  APPEND ~kit.ids~ ~0x0000 BARBARIAN~
+  APPEND ~kit.ids~ ~0x40000000 BARBARIAN~
     UNLESS ~\bBARBARIAN\b~
-  APPEND ~kit.ids~ ~0x0000 WILDMAGE~
+  APPEND ~kit.ids~ ~0x80000000 WILDMAGE~
     UNLESS ~\bWILDMAGE\b~
     
   // adding all ToB actions to action.ids unless already present
