--- Fade\English\BE3Fade.tra	Thu Jan 06 18:29:20 2005
+++ C:\BWP Patchstudio\patched files\Fade\English\BE3Fade.tra	Fri Nov 12 16:41:17 2010
@@ -397,7 +397,7 @@
 @382 = ~I'm sorry it has come to this, <CHARNAME>. But if ever you decide you need my help again, I will be at my guildhouse.~ 
 @383 = ~I was only showing a little friendly concern, but since it's obviously not to your taste... very well, I shall refrain from making that mistake ever again.~
 @384 = ~I certainly hope you will, <CHARNAME>. I've... got used to your company, and I'd hate to have to wake up someday and find you a little heap of ash, or worse.~
-@385 = ~Oh, so the conversation of one "such as me" isn't good enough for our resident <PRO_LORDLADY) of Murder? Very well, I'll make sure your divine ears aren't bothered with it again, ever!~
+@385 = ~Oh, so the conversation of one "such as me" isn't good enough for our resident <PRO_LADYLORD> of Murder? Very well, I'll make sure your divine ears aren't bothered with it again, ever!~
 @386 = ~It still doesn't excuse them. They should be wiped from the face of Faerun.~
 
 //LT8
@@ -500,7 +500,7 @@
 @483 = ~I was only trying to see things from all perspectives, <CHARNAME>, nothing more. I can see that you were hurt deeply by what has happened, and I'm sorry for bringing it up.~
 @484 = ~No need to apologise, Fade. I shouldn't have blown up like that. You have every right to be curious... as do I. What about your upbringing? How does a Fey'ri upbringing compare to mine?~
 @485 = ~It's okay. I shouldn't have flown off the handle like that. I don't blame you for being curious. So, what of you then? What of your childhood?~
-@486 = ~Yes, I have. I'll make sure you're no longer troubled by my prattling, oh-so-gracious <PRO_LORDLADY> of Murder.~
+@486 = ~Yes, I have. I'll make sure you're no longer troubled by my prattling, oh-so-gracious <PRO_LADYLORD> of Murder.~
 @487 = ~Perhaps, <CHARNAME>. We'll see.~
 @488 = ~Yes, we should get moving.~
 
@@ -1344,7 +1344,7 @@
 @1281 = ~(follow her)~
 @1282 = ~(leave her alone)~
 @1283 = ~(She stops a little way from the rest of the group and turns. She is smiling, but it is a nervous smile, with none of the usual teasing undertones)~
-@1284 = ~I am... glad you chose to follow me, <CHARNAME >. I did wonder if you would take the hint. I... I... ~
+@1284 = ~I am... glad you chose to follow me, <CHARNAME>. I did wonder if you would take the hint. I... I... ~
 @1285 = ~(She sighs and smiles nervously again, and then begins to play with a lock of her bright crimson hair. You wait for a moment, but it is obvious that she is unsure of what to say next)~
 @1286 = ~Fade, come on, spit it out. Why did you drag me over here?~
 @1287 = ~Are you okay, Fade?~
@@ -2188,30 +2188,30 @@
 @2104 = ~(scramble away from her) Are you crazy? Give in? What in the Nine Hells were you going to do then? I just thought that... that... ~
 @2105 = ~(grab her wrists) Come on, Fade. You know you want to.~
 @2106 = ~(Her eyes flare with momentary anger, and you wonder if she will attack you there and then for denying her. She then shakes her head, and when she speaks, her voice is dismayed and desperate)~
-@2107 = ~I can't help it, <CHARNAME >! I'm sorry... This is why you have to help me. It's almost too much for me... I'm trying to resist, but it gnaws at me so... ~
+@2107 = ~I can't help it, <CHARNAME>! I'm sorry... This is why you have to help me. It's almost too much for me... I'm trying to resist, but it gnaws at me so... ~
 @2108 = ~(She takes a deep breath and closes her eyes) I must be strong. You're right. But I can't do it alone... ~
 @2109 = ~I said I'd help you, Fade, and I will. I'm sorry I pushed you. I didn't realise that your... compulsion was so strong. We'll stay here for now, so you can calm down. Everything will be better once you've rested.~
 @2110 = ~Well, you're going to have to! You're crazy, did you know that? Absolutely crazy! I should have left you back in Athkatla, with your Shadow Thief cronies. Just stay away from me from now on, okay?~
 @2111 = ~I'll help you, Fade, but I want it made quite plain that I have no intention of ever giving in to you. I don't know what instincts you are acting on at the moment, but I'm not interested, okay?~
-@2112 = ~(Her eyes flare with momentary anger at being denied, but is soon replaced by a look of resignation) You're right, <CHARNAME >. (sigh) I probably would have ended up killing you if you let me carry on... I can't help it in this place.~
+@2112 = ~(Her eyes flare with momentary anger at being denied, but is soon replaced by a look of resignation) You're right, <CHARNAME>. (sigh) I probably would have ended up killing you if you let me carry on... I can't help it in this place.~
 @2113 = ~I have to be strong, I have to fight it... but I can't do it alone, that much must be obvious to you now. Please, you have to help me.~
 @2114 = ~I said I'd help you Fade, and I will. I didn't realise that your... compulsion was so strong. We'll stay here for now, so you can calm down. Everything will be better once you've rested. Trust me.~
 @2115 = ~You would have killed me? Why?~
 @2116 = ~(She begins to slowly unlace her tunic, but as her eyes meet yours, she stops and shakes her head, looking confused) No... this is wrong... wrong. Not as drow... ~
 @2117 = ~What? No! Don't stop now!~
 @2118 = ~You're right. Not as drow. We would only end up regretting it. I said I'd help you Fade, and I will. Come on, we'll rest here for now, so we can both calm down. It'll be better once we've rested, I'm sure of it.~
-@2119 = ~You thought what? That you'd get a bit of gratification out of me, and that would be it? <CHARNAME >, I'm part succubus! You may not die if I kiss you, but if I let go... if I let that side of me take over... who knows what would happen?~
+@2119 = ~You thought what? That you'd get a bit of gratification out of me, and that would be it? <CHARNAME>, I'm part succubus! You may not die if I kiss you, but if I let go... if I let that side of me take over... who knows what would happen?~
 @2120 = ~This place isn't healthy for me! I don't know whether I want to mate with you or rip you into little bits!~
 @2121 = ~You're crazy, did you know that? Absolutely crazy! I should have left you back in Athkatla, with your Shadow Thief cronies. Just stay away from me from now on, okay?~
 @2122 = ~Is it that bad? Okay, I'll help you, even if it's just to stop you from ripping my heart out! Come on, we'll rest here for now, so we can both calm down. It'll be better once we've rested, I'm sure of it.~
-@2123 = ~No!! You think this will be a simple act of gratification? <CHARNAME >, I'm part succubus! You may not die if I kiss you, but if I let go... if I let that side of me take over... who knows what would happen?~
+@2123 = ~No!! You think this will be a simple act of gratification? <CHARNAME>, I'm part succubus! You may not die if I kiss you, but if I let go... if I let that side of me take over... who knows what would happen?~
 @2124 = ~What? I didn't... Okay, I'll help you, even if it's just to stop you from ripping my heart out! Come on, we'll rest here for now, so we can both calm down. It'll be better once we've rested, I'm sure of it.~
 @2125 = ~(keep hold of her wrists and pull her closer) I don't believe you, Fade.~
 @2126 = ~Yes, rest... but keep yourself safe. I want you to promise... if I get out of hand, if I give in and become a danger... you will kill me. I can't live like that again.~
 @2127 = ~Of course, Fade. But it will never come to that, trust me. Now rest.~
 @2128 = ~Fade, I can't promise that. I can't kill you. I'll compromise, though. If it happens, and only if, I'll knock you out until we get back to the surface, okay? No arguing. Now rest.~
-@2129 = ~(A look of dismay crosses her face) Yes, <CHARNAME >... I understand. (She turns away from you and lies on the bed, as if preparing to trance, and says nothing more)~
-@2130 = ~(sigh) I'm part succubus, <CHARNAME >. You may not die if I kiss you, but if I let go... if I let that side of me take over... who knows what would happen?~
+@2129 = ~(A look of dismay crosses her face) Yes, <CHARNAME>... I understand. (She turns away from you and lies on the bed, as if preparing to trance, and says nothing more)~
+@2130 = ~(sigh) I'm part succubus, <CHARNAME>. You may not die if I kiss you, but if I let go... if I let that side of me take over... who knows what would happen?~
 @2131 = ~This place isn't healthy for me. I don't know whether I want to mate with you or rip you into little bits! I'm possibly not... safe to be around at the moment.~
 @2132 = ~(She wrenches her wrists from you and jumps off the bed) No... No! Everything... it's over, <CHARNAME>, possibly even before it began, but it's *over*! I can't believe you want me to become... to allow that side of me to prevail... just for your own pleasure?! Not that you would have been able to have handled me in any way!!~
 @2133 = ~(She unlocks the door, wrenches it open and storms off)~
@@ -2610,7 +2610,7 @@
 @2506 = ~*Always* be watching me, Anomen? I'm sure I could have you arrested for that.~
 @2507 = ~I suppose I should have expected such a response from one of your kind. Never did I think I would have to travel with a Shadow Thief!~
 @2508 = ~Well, now, that's unfortunate, isn't it? Especially since <CHARNAME>'s joined our ranks and everything. I would say you're outnumbered, Anomen.~
-@2509 = ~Well, <CHARNAME > is the one calling the shots here, not you, so I think you can just toddle off back to your position and concentrate on being righteous, while the rest of us have some fun.~
+@2509 = ~Well, <CHARNAME> is the one calling the shots here, not you, so I think you can just toddle off back to your position and concentrate on being righteous, while the rest of us have some fun.~
 
 @2510 = ~Have you ever travelled to Sigil?~
 @2511 = ~No... never.~
