--- Setup-TS-BP.tp2	Thu Jun 21 21:24:52 2007
+++ C:\BWP Patchstudio\patched files\Setup-TS-BP.tp2	Sun Apr 18 04:08:14 2010
@@ -2,6 +2,8 @@
 
 AUTHOR ~Vlad & Domi, King Diamond (BP-compatible version)~ NO_IF_EVAL_BUG
 
+VERSION ~v6.10 BWP Fix~
+
 AUTO_TRA ~TS/Language/%s~
 
 LANGUAGE ~English~
@@ -2284,7 +2286,8 @@
   SAY NAME1 @2310
   SAY NAME2 @2310
  
-COPY ~TS/cre/Sola.cre~         ~override~
+// COPY ~TS/cre/Sola.cre~         ~override~
+COPY ~TS/cre/SolaTS.cre~         ~override~
   SAY NAME1 @2311
   SAY NAME2 @2311
 
