--- bg2fixpack\dlg\beta_tob-dlg.d	Sun May 20 04:50:44 2012
+++ C:\BWP Patchstudio\patched files\bg2fixpack\dlg\beta_tob-dlg.d	Wed Apr 24 23:45:39 2013
@@ -8,4 +8,7 @@
 ALTER_TRANS BKELDO25 BEGIN 33 END BEGIN 0 END // filename, state, trans
 BEGIN // list of changes, see below for flags
   EPILOGUE ~EXTERN BANOME25 52~
-END
\ No newline at end of file
+END
+
+// wish fixes from asc64; see also wish01.bcs changes
+REPLACE_ACTION_TEXT ~wish25~ ~\(SetupWish([0-9],1)\)~ ~/* \1 */~
\ No newline at end of file
