--- bg2fixpack\dlg\beta_soa-dlg.d	Wed Jan 23 00:51:35 2013
+++ C:\BWP Patchstudio\patched files\bg2fixpack\dlg\beta_soa-dlg.d	Wed Apr 24 23:45:39 2013
@@ -235,4 +235,61 @@
 ADD_TRANS_ACTION NALIAJ BEGIN 225 END BEGIN END ~EraseJournalEntry(11648)~
 
 // glacias charm/dispel fix, new method
-ALTER_TRANS KPGLAI01 BEGIN 0 END BEGIN END BEGIN "ACTION" ~~ END
\ No newline at end of file
+ALTER_TRANS KPGLAI01 BEGIN 0 END BEGIN END BEGIN "ACTION" ~~ END
+
+// Keldorn re-joining variable typo fix (berelinde)
+REPLACE_ACTION_TEXT ~keldorj~ ~SetGlobal("KeldornLeaves","GLOBAL",1)~  ~SetGlobal("KeldornLeave","GLOBAL",1)~
+
+// Fix Aerie's "Blanket Thieves" banter which could never occur due to a duplicated PartyRested() trigger (berelinde)
+REPLACE_TRIGGER_TEXT ~baerie~ ~PartyRested()~ ~~
+
+// Fix Haer'Dalis' "Arcane muttering" banter with Edwin which could never occur due to a duplicated PartyRested() trigger (aVENGER)
+REPLACE_TRIGGER_TEXT ~bhaerda~ ~PartyRested()~ ~~
+
+// fix for player getting control in dreams equence (argent77)
+REPLACE_TRANS_ACTION DREAM4 BEGIN 10 END BEGIN 0 END
+  ~ActionOverride(Player1,Attack("Drim02"))~
+  ~ActionOverride(Player1,MoveToObject("Drim02"))~
+  
+// lavok being picky about XP rewards
+ADD_TRANS_ACTION LAVOK BEGIN 15 END BEGIN 1 2 END ~AddexperienceParty(28750)~
+
+// mgappr03 checking for wrong variable value
+REPLACE_TRIGGER_TEXT ~MGAPPR03~ ~Global("PoorMage","GLOBAL",31)~ ~Global("PoorMage","GLOBAL",3)~
+
+// missing journal entries
+ALTER_TRANS IMOENJ   BEGIN  14 END BEGIN 0     END BEGIN "JOURNAL"            ~#843~ END
+ALTER_TRANS JAHEIRAJ BEGIN  61 END BEGIN 0     END BEGIN "SOLVED_JOURNAL"   ~#47972~ END
+ALTER_TRANS JAHEIRAJ BEGIN 295 END BEGIN 0     END BEGIN "SOLVED_JOURNAL"   ~#47972~ END
+ALTER_TRANS MGTEOS01 BEGIN  30 END BEGIN 1 2 3 END BEGIN "JOURNAL"          ~#55627~ END
+ALTER_TRANS MGTEOS01 BEGIN  38 END BEGIN 1 2   END BEGIN "JOURNAL"          ~#55628~ END
+ALTER_TRANS NALIA    BEGIN  66 END BEGIN 1     END BEGIN "UNSOLVED_JOURNAL" ~#40131~ END
+ALTER_TRANS OBSSOL02 BEGIN   4 END BEGIN 0     END BEGIN "JOURNAL"           ~#7241~ END
+ALTER_TRANS OBSSOL03 BEGIN   4 END BEGIN 0     END BEGIN "JOURNAL"          ~#23551~ END
+ALTER_TRANS RIBALD   BEGIN  34 END BEGIN 1     END BEGIN "JOURNAL"          ~#55641~ END
+
+// jaheira dupe responses
+ADD_TRANS_TRIGGER JAHEIRAJ 65 ~Global("JaheiraJerk","GLOBAL",1)~ DO 0
+ADD_TRANS_TRIGGER JAHEIRAJ 65 ~Global("JaheiraJerk","GLOBAL",0)~ DO 1
+
+// restore missing interjection
+ALTER_TRANS UHINN02 BEGIN  9 END BEGIN 0 END BEGIN "EPILOGUE" ~EXTERN UHINN01 21~ END
+EXTEND_BOTTOM UHINN01 20
+  IF ~!Dead("uhinn02") See("uhinno2")~ EXTERN ~UHINN02~ 9
+END
+
+// majordomo not recognizing nalia; don't mention where gold is stored in initial dialogue
+REPLACE_STATE_TRIGGER KPDOMO01 21
+~!InPartySlot(LastTalkedToBy,0) !Name("Nalia",LastTalkedToBy)~
+
+ADD_TRANS_TRIGGER KPDOMO01 25 ~NumTimesTalkedToGT(1)~ DO 1
+
+// gaelan asking for 20k gold, not 50k
+ALTER_TRANS GAELAN BEGIN 29 END BEGIN 2 END // filename, state, trans
+BEGIN TRIGGER ~PartyGoldGT(19999)~ END
+ALTER_TRANS GAELAN BEGIN 29 END BEGIN 3 END // filename, state, trans
+BEGIN TRIGGER ~PartyGoldLT(20000)~ END
+
+// dno't disable the tree and fubar the finale
+ALTER_TRANS PLAYER1 BEGIN 30 END BEGIN 0 END // filename, state, trans
+BEGIN ACTION ~SetGlobal("TreeLifeTalk","AR2805",2) TriggerActivation("Tran2806",TRUE)~ END
\ No newline at end of file
