--- bg2fixpack\languages\english\setup.tra	Sun Apr 14 07:02:10 2013
+++ C:\BWP Patchstudio\patched files\bg2fixpack\languages\english\setup.tra	Wed Apr 24 23:45:39 2013
@@ -211,7 +211,7 @@
 
 Disadvantages:
 -  May not use ANY magic items (potions other than healing potions, scrolls, rings, amulets, cloaks, etc.) except for weapons and armor~ []
-// fix speed factor, originally string #176
+// fix speed factor, originally string #176; identified descript for bow17 long bow +2
 @146 = ~The long bow is similar to the short bow, except that the staff is about as long as the archer is tall, usually 6 to 6.5 feet. Even though the long bow is slightly slower than the short bow, it has more range and is more accurate. The smooth staff of this bow is warm to the touch, just a hint of the energy that surges through the wood.
 
 STATISTICS:
@@ -315,7 +315,7 @@
 Save vs Spells: +3 bonus
 Constitution modifier: -2 penalty~
 
-// MGoI claims that dispel magic works against it, but does not, originally string #12164
+// MGoI claims that dispel magic works against it, but does not, originally string #12146
 @154 = ~Minor Globe of Invulnerability (Abjuration)
 Level: 4
 Range: 0
@@ -326,7 +326,7 @@
 
 This spell creates a faintly shimmering magical sphere around the caster that prevents any 1st-, 2nd-, or 3rd-level spell effects from penetrating (i.e., the area of effect of any such spells does not include the area of the minor globe of invulnerability). This includes innate abilities and effects from devices. However, any type of spell can be cast out of the magical sphere, and these pass from the caster of the globe to their subject without affecting the minor globe. Fourth and higher level spells are not affected by the globe. The globe can be brought down by forms of magical attack such as Spellstrike and Pierce Magic.~
 
-// GoI claims that dispel magic works against it, but does not, originally string #12164
+// GoI claims that dispel magic works against it, but does not, originally string #8064
 @155 = ~Globe of Invulnerability (Abjuration)
 Level: 6
 Range: 0
@@ -356,7 +356,7 @@
 Duration: Special
 Casting Time: 1 round
 Area of Effect: 1 familiar
-Saving Throw: Special
+Saving Throw: None
 
 This spell enables the caster to attempt to summon a familiar to act as his aide and companion. Familiars are typically small creatures. A creature acting as a familiar can benefit a wizard, conveying its sensory powers to its master, conversing with him, and serving as a guard/scout/spy as well. A wizard can have only one familiar at a time, and he has no control over what sort of creature answers the summoning, if any come at all.
 (Note: This spell may only be cast by the protagonist.)
@@ -405,7 +405,7 @@
 Armor Class: 4
 Hit Points: 24
 Magic Resistance: 32%
-Special abilities: Can cast cast 'Mirror Image' and 'Invisibility 10' Radius' once per day. Immune to level drain, sleep, and petrification.
+Special abilities: Can cast 'Mirror Image' and 'Invisibility 10' Radius' once per day. Immune to level drain, sleep, and petrification.
 
 CHAOTIC NEUTRAL: Cat
 Armor Class: 0
@@ -417,7 +417,7 @@
 Armor Class: 2
 Hit Points: 24
 Magic Resistance: 25%
-Special abilities: Has 100% resistance to fire, cold, and electricity. Can cast cast 'Horror' and 'Blur' once per day. Regenerates 1hp/round.~
+Special abilities: Has 100% resistance to fire, cold, and electricity. Can cast 'Horror' and 'Blur' once per day. Regenerates 1hp/round.~
 @160 = ~This is your familiar, a pseudo-dragon, currently curled up and safe within your backpack. The lizard-like creature is quite intelligent and fairly well-armored and quick... it does not do much damage in combat, but its magical claws are able to render some enemies unconscious. It is also quite resistant to magical spells.
 
 Abilities:
@@ -459,7 +459,7 @@
 Armor Class: -2
 Hit Points: 48
 Magic Resistance: 35%
-Special abilities:  Has 100% resistance to fire, cold, and electricity. It can also cast 'Polymorph Self' once per day, attack does 1d6 damage with poison (save vs. death or die). Regenerates 1hp/round. Immune to level drain, sleep, and petrification. Immune to non-magical weapons.~
+Special abilities:  Has 100% resistance to fire, cold, and electricity. It can also cast 'Polymorph Self' once per day. Regenerates 1hp/round. Immune to level drain, sleep, and petrification. Immune to non-magical weapons.~
 @166 = ~This is your familiar, a rabbit, currently curled up and safe within your backpack. The rabbit might not be one's first choice for a familiar, but this creature is incredibly quick and difficult to hit in combat... and has been augmented by magic to become quite resistant to physical harm as well as spells. The rabbit also has a keen nose for sniffing out traps on behalf of its master.
 
 Abilities:
@@ -473,7 +473,7 @@
 Armor Class: 1
 Hit Points:  48
 Magic Resistance: 65%
-Special abilities: Has 85% Detect Traps ability, 60% Hide in Shadows/Move Silently, and 50% Detect Illusions. Has 75% resistance to fire, cold, and electricity. It can also cast 'Blur' once per day. The rabbit moves as if under a constant 'Haste' spell. Immune to level drain, sleep, and petrification. spell.~
+Special abilities: Has 85% Detect Traps ability, 60% Hide in Shadows/Move Silently, and 50% Detect Illusions. Has 75% resistance to fire, cold, and electricity. It can also cast 'Blur' once per day. The rabbit moves as if under a constant 'Haste' spell. Immune to level drain, sleep, and petrification.~
 @168 = ~This is your familiar, a dust mephit currently safe within your backpack. The dust mephit is not strong in combat, but it is slightly resistant to magical attack and has the ability to cast spells to assist its master.
 
 Abilities:
@@ -494,14 +494,14 @@
 Armor Class: 4
 Hit Points: 24
 Magic Resistance: 32%
-Special abilities: Can cast cast 'Mirror Image' and 'Invisibility 10' Radius' once per day. Immune to level drain, sleep, and petrification.~
+Special abilities: Can cast 'Mirror Image' and 'Invisibility 10' Radius' once per day. Immune to level drain, sleep, and petrification.~
 @171 = ~This is your familiar, a fairy dragon, currently curled up and safe within your backpack. The mischievous fairy dragon is fairly resistant to magic... and has the ability to both protect and obscure itself with illusionary magic of its own.
 
 Abilities:
 Armor Class: 0
 Hit Points: 48
 Magic Resistance: 45%
-Special abilities: Can cast cast 'Mirror Image', 'Invisibility 10' Radius', and 'Improved Invisibility' once per day. Immune to level drain, sleep, and petrification.~
+Special abilities: Can cast 'Mirror Image', 'Invisibility 10' Radius', and 'Improved Invisibility' once per day. Immune to level drain, sleep, and petrification.~
 @172 = ~This is your cat familiar, currently curled up and safe within your backpack. The cat familiar is a master scout, able to move in the shadows with the ability of a master thief. It is also difficult to hit in combat, and the magic of the spell has made this cat resistant to both physical and magical attacks.
 
 Abilities:
@@ -522,14 +522,14 @@
 Armor Class: 2
 Hit Points: 24
 Magic Resistance: 25%
-Special abilities: Has 100% resistance to fire, cold, and electricity. Can cast cast 'Horror' and 'Blur' once per day. Regenerates 1hp/round.~
+Special abilities: Has 100% resistance to fire, cold, and electricity. Can cast 'Horror' and 'Blur' once per day. Regenerates 1hp/round.~
 @175 = ~This is your familiar, a quasit, currently grumpy yet safe within your backpack. The quasit is a small, infernal creature that is fairly quick and quite resistant to magical spells... and has the ability to cast its own spells on behalf of its master.
 
 Abilities:
 Armor Class: -2
 Hit Points: 48
 Magic Resistance: 35%
-Special abilities: 3 attacks per round 1d6 damage, save vs. death or lose one point of dexterity for 36 seconds. Has 100% resistance to fire, cold, and electricity. Can cast cast 'Horror' and 'Blur' once per day. Regenerates 1hp/round. Immune to level drain, sleep, and petrification. Immune to non-magical weapons.~
+Special abilities: 3 attacks per round 1d6 damage, save vs. death or lose one point of dexterity for 36 seconds. Has 100% resistance to fire, cold, and electricity. Can cast 'Horror' and 'Blur' once per day. Regenerates 1hp/round. Immune to level drain, sleep, and petrification. Immune to non-magical weapons.~
 
 //Description fixes for item weight and speed factor
 //Originally string #71131
@@ -545,7 +545,7 @@
 STATISTICS:
 
 Damage:  1D6 + 2
-THACO:  +1
+THAC0:  +1
 Damage type:  crushing
 Weight: 13
 Speed Factor: 6
@@ -565,7 +565,7 @@
 
 +20% Fire Resistance
 
-THACO:  +4
+THAC0:  +4
 Damage:  2D4 +4 +1-4 extra cold damage
 Damage type:  crushing
 Weight: 8
@@ -584,7 +584,7 @@
 
 STATISTICS:
 
-THACO:  +3 bonus
+THAC0:  +3 bonus
 Damage:  +3 bonus
 Weight: 8
 Speed Factor: 5
@@ -618,7 +618,7 @@
 //Originally string #57897
 @181 = ~Kuo-Toa Bolt
 
-This slender bolt is coated with a special poison, developed by Kuo-Toa alchemists.  A target hit by this bolt must making a saving throw vs. spells or be stunned for four rounds.
+This slender bolt is coated with a special poison, developed by Kuo-Toa alchemists.  A target hit by this bolt must make a saving throw vs. spells or be stunned for four rounds.
 
 STATISTICS:
 
@@ -643,7 +643,7 @@
 Type:  2-handed~
 
 //Originally string #39579
-@183 = ~Once a very powerful shield used by the clerics of Bane, most of the enchantments fled with their god's demise.  The basic magical protection of still remains, making it useful for any adventurer.
+@183 = ~Once a very powerful shield used by the clerics of Bane, most of the enchantments fled with their god's demise.  The basic magical protection still remains, making it useful for any adventurer.
 
 STATISTICS:
 
@@ -689,8 +689,7 @@
 Not Usable By:
  Druid
  Mage 
- Thief
-~
+ Thief~
 
 //Originally string #41620
 @186 = ~This two-headed flail causes acid and cold damage.  If the other head for the Flail of Ages could be found, it would make this weapon more powerful.
@@ -727,11 +726,11 @@
  Thief~
 
 //Originally string #71058
-@188 = ~The long bow is similar to the short bow, except that the staff is about as high as the archer is, usually 6 to 6.5 feet.  Even though the longbow is slightly slower then the short bow, it has more range and is more accurate.  The smooth staff of this bow is warm to the touch, just a hint of the energy that surges through the wood.
+@188 = ~The long bow is similar to the short bow, except that the staff is about as high as the archer is, usually 6 to 6.5 feet.  Even though the longbow is slightly slower than the short bow, it has more range and is more accurate.  The smooth staff of this bow is warm to the touch, just a hint of the energy that surges through the wood.
 
 STATISTICS:
 
-THACO: +3 bonus
+THAC0: +4 bonus
 Weight: 2
 Speed Factor: 4
 Proficiency Type: Long Bow
@@ -786,7 +785,7 @@
 
 Equipped Abilities: Regenerate 1 hit point every 12 seconds
 
-THACO:  +2 bonus
+THAC0:  +2 bonus
 Damage:  1D6 + 2
 Damage type:  crushing
 Weight: 3
@@ -803,7 +802,7 @@
 STATISTICS:
 
 Special Abilities: 3 points of poison damage if target fails save
-THACO:  +2 bonus
+THAC0:  +2 bonus
 Damage:  1D6 +2
 Damage type:  piercing
 Weight: 2
@@ -820,7 +819,7 @@
 STATISTICS:
 
 Special Abilities: Target loses 2 points of dexterity each hit (duration: 30 seconds)
-THACO:  +2 bonus
+THAC0:  +2 bonus
 Damage:  1D6 +2
 Damage type:  piercing
 Weight: 2
@@ -896,7 +895,7 @@
 
 //Originally string #39627
 @197 = ~Staff of the Magi
-This type of staff is thoroughly linked of the image of the spellcasting profession, rare though it is.  It is among the most powerful tools that a wizard can possess, the culmination of complex enchantment and endless study.  This particular example is good in melee, grants Invisibility, and offers protection from Charms and Evil.  It can also cast a Spell Trap, launch a Fireball-Lighting Bolt combination, and Dispel Magic. The quintessential mage item.
+This type of staff is thoroughly linked to the image of the spellcasting profession, rare though it is.  It is among the most powerful tools that a wizard can possess, the culmination of complex enchantment and endless study.  This particular example is good in melee, grants Invisibility, and offers protection from Charms and Evil.  It can also cast a Spell Trap, launch a Fireball-Lighting Bolt combination, and Dispel Magic. The quintessential mage item.
 
 STATISTICS
 Armor Class: +2 bonus
@@ -920,7 +919,7 @@
 Damage:  1D6 +1
 Damage type:  crushing
 Weight: 4
-Speed Factor: 3
+Speed Factor: 1
 Proficiency Type: Quarterstaff
 Type:  2-handed
 Requires: 5 Strength
@@ -929,7 +928,7 @@
 
 //Originally string #39629
 @198 = ~Staff of Thunder and Lightning +2
-There are stories that tell of these staves sprouting in the growth around druid groves when the forest is threatened.  Fanciful tales, but truth or not, this staff is a powerful tool.  It draws energy from storms, and there is a chance that any blow may trigger a thunderclap that knocks enemies down.  It casts Lightning Bolt, and if circumstances warrant it, it can summon a lighting storm to blast enemies to dust.  Some abilities are limited by charges.
+There are stories that tell of these staves sprouting in the growth around druid groves when the forest is threatened.  Fanciful tales, but truth or not, this staff is a powerful tool.  It draws energy from storms, and there is a chance that any blow may trigger a thunderclap that knocks enemies down.  It casts Lightning Bolt, and if circumstances warrant it, it can Call Lightning to blast enemies to dust.  Some abilities are limited by charges.
 
 STATISTICS
 
@@ -948,7 +947,7 @@
  Range: Melee 
  Saving Throw: Save vs. spells or be stunned for two rounds
 
-THACO: +2 bonus
+THAC0: +2 bonus
 Damage:  1D6 +2
 Damage type:  crushing
 Weight: 4
@@ -962,7 +961,7 @@
 
 //Originally string #39631
 @199 = ~Staff of Air
-This staff seems to draw from eleven artistic traditions, though tied less to the forest than is usually customary.  It was apparently designed to control air elementals, being able to summon one as a servant to the wielder, and potentially destroying creatures of the same type in a single blow.  When necessary, it can also spawn a cloud of gas causing anyone within the area of effect to fall into a deep slumber.  Some effects are limited by charges.
+This staff seems to draw from elven artistic traditions, though tied less to the forest than is usually customary.  It was apparently designed to control air elementals, being able to summon one as a servant to the wielder, and potentially destroying creatures of the same type in a single blow.  When necessary, it can also spawn a cloud of gas causing anyone within the area of effect to fall into a deep slumber.  Some effects are limited by charges.
 
 STATISTICS
 
@@ -1034,7 +1033,7 @@
 
 STATISTICS:
 
-THACO:  +2 bonus
+THAC0:  +2 bonus
 Damage:  1D6 + 2
 Damage type:  crushing
 Weight: 3
@@ -1051,7 +1050,7 @@
 Combat Abilities:
   50% chance of poisoning opponents with every successful attack.  The poison lasts for 1 round, and does 2 hp of damage/second unless the target makes a save vs death.
 
-THACO:  +2 bonus
+THAC0:  +2 bonus
 Damage:  1D6 + 2
 Damage type:  crushing
 Weight: 3
@@ -1076,8 +1075,7 @@
 Requires: 8 Strength
 Not Usable By:
  Cleric
- Mage
- Thief~
+ Mage~
 
 //Originally string #66203
 @205 = ~Also known as the Stream of Anguish, this is the legendary burning blade of Frithiof.     
@@ -1130,7 +1128,7 @@
 STATISTICS:
 
 Damage:  1D8 +3
-THACO:  +3 bonus
+THAC0:  +3 bonus
 Damage type:  slashing
 Weight: 3
 Speed Factor: 2
@@ -1140,12 +1138,11 @@
 Not Usable By:
  Druid
  Cleric
- Mage
-~
+ Mage~
 
 //Originally string #61593
 @208 = ~Harmonium Halberd +3
-This weapon is a powerful weapon usually given out to the most important of Harmonium captains.  The Harmonium is an faction based out of the planar city known as 'Sigil'.
+This weapon is a powerful weapon usually given out to the most important of Harmonium captains.  The Harmonium is a faction based out of the planar city known as 'Sigil'.
 
 STATISTICS:
 
@@ -1167,7 +1164,8 @@
  Mage
  Thief~
 
-//Originally string #1764
+//Originally string #1764; no longer needed so translators dont bother
+/*
 @209 = ~Battle Axe +3
 
 This is the axe of Bruenor Battlehammer, and a fine weapon it is. It has a formidable single-edged blade, and handles with the efficiency of a hand axe despite its size.
@@ -1186,7 +1184,7 @@
  Cleric
  Mage 
  Thief~
-                          
+*/
 //Originally string #21957                          
 @210 = ~Bala's Axe - Wizard Slayer:
 Bala was a foul-tempered prince of a gully dwarf clan, which was enslaved by a cruel elven mage in Thesk. Though Bala managed to escape the more cowardly traits of his race he was even more filthy and obnoxious than the average gully dwarf. Sickened of his enslavement, Bala crept into the mage's lair and cracked the elf's skull open. Although rumor has it that it was the dwarf's stench that overcame the mage, it was, in fact, his enchanted axe that dispelled the elf's magical defenses. It is obvious from the poor workmanship that Bala himself crafted the axe. The mystery lies in where its power originates. After a lengthy career of mage-slaying, Bala died, never having revealed his secret to those who were willing to risk a lice infestation by approaching him. Some say that the axe was powered by Bala's intense hatred of wizards. The axe gives the owner the ability to dispel magic once a day.
@@ -1222,7 +1220,7 @@
 Equipped Abilities:
  +1 AC bonus
  +20% resistance to slashing, piercing, and blunt damage
-THACO:  +3 bonus
+THAC0:  +3 bonus
 Damage:  1D6 + 4
 Damage type:  crushing
 Weight: 12
