--- bg2fixpack\setup-bg2fixpack.tp2	Sun Apr 14 06:57:56 2013
+++ C:\BWP Patchstudio\patched files\bg2fixpack\setup-bg2fixpack.tp2	Mon Feb 17 19:34:10 2014
@@ -1,7 +1,7 @@
 BACKUP ~bg2fixpack/backup~ // location to store files for uninstall purposes
 AUTHOR ~pcamagna@yahoo.com~ // email address displayed if install fails
 
-VERSION ~v10~
+VERSION ~v10 BWP Fix~
 
 README ~bg2fixpack/readme-bg2fixpack.html~
 
