--- vecna\Language\English\A_CTHI2A.tra	Sat Mar 18 19:43:17 2006
+++ C:\BWP Patchstudio\patched files\vecna\Language\English\A_CTHI2A.tra	Wed Sep 10 08:14:08 2014
@@ -1,7 +1,7 @@
-@0    = ~Hmmm....Me neck hairs be springin'...~
-@1    = ~Singing out! These jerkies slipped me a wet one!~
+@0    = ~Hmmm... Me neck hairs be springin'...~
+@1    = ~Signing out! These jerkies slipped me a wet one!~
 @2    = ~Knave! Come fight me in the flesh...~
-@3    = ~Come closer and fight me fair!~
+@3    = ~Come closer and fight me fairly!~
 @4    = ~Coward! I bet ye can't handle a blade...~
 @5    = ~Barb-slinger! Come taste some steel...~
 @6    = ~Somebody shank this bastard!~
