--- vecna\Language\English\A_PRI18B.tra	Sat Mar 18 19:50:55 2006
+++ C:\BWP Patchstudio\patched files\vecna\Language\English\A_PRI18B.tra	Wed Sep 10 01:41:38 2014
@@ -1,8 +1,8 @@
 @0    = ~There you are, you bastard!~
-@1    = ~Hmmm...must've been a ghost~
-@2    = ~Magic gas is a thing of the past~
+@1    = ~Hmmm... Must've been a ghost.~
+@2    = ~Magic gas is a thing of the past.~
 @3    = ~Ouch! Who did that?~
 @4    = ~A magic cloud has engulfed me!~
-@5    = ~Help! I'm...poisoned.~
-@6    = ~Hmmm....Something's not right here...~
+@5    = ~Help! I'm... poisoned.~
+@6    = ~Hmmm... Something's not right here.~
 @7    = ~Die with your boots on!~
