--- vecna\Language\English\A_VAMP01.tra	Sat Mar 18 20:25:33 2006
+++ C:\BWP Patchstudio\patched files\vecna\Language\English\A_VAMP01.tra	Wed Sep 10 01:42:13 2014
@@ -4,8 +4,8 @@
 @3    = ~Hissss--sneaky, for cattle...~
 @4    = ~To gas I return...~
 @5    = ~Come closer, if you dare!~
-@6    = ~Hmm...that missile attack may come in handy...~
+@6    = ~Hmm... that missile attack may come in handy...~
 @7    = ~Barb-slinger! You must be afraid of me...~
-@8    = ~Barb-slinger! I'll drink your blood for that~
-@9    = ~Curseds! Foiled again...~
-@10   = ~I'm not going down without a fight~
+@8    = ~Barb-slinger! I'll drink your blood for that!~
+@9    = ~Curses! Foiled again...~
+@10   = ~I'm not going down without a fight!~
