--- vecna\baf\cre\AA187GEN.baf	Wed Feb 26 12:12:06 2003
+++ C:\BWP Patchstudio\patched files\vecna\baf\cre\AA187GEN.baf	Fri Oct 28 08:54:13 2011
@@ -658,8 +658,8 @@
     Class(LastSeenBy(Myself),DRUID_ALL)
     Class(LastSeenBy(Myself),BARD)
     Class(LastSeenBy(Myself),CLERIC_ALL)
-    Name("astSeenBy(),","Planatar")  // Planatar
-    Name("astSeenBy(),","Darkplanatar")  // Darkplanatar
+    Race(LastSeenBy(Myself),PLANATAR)  // Planatar
+    Race(LastSeenBy(Myself),DARKPLANATAR)  // Darkplanatar
   OR(2)
     !StateCheck(LastSeenBy(Myself),-2146164691)
     RandomNum(4,1)
