--- vecna\Language\English\AA35COPY.tra	Sun May 02 22:25:38 2010
+++ C:\BWP Patchstudio\patched files\vecna\Language\English\AA35COPY.tra	Wed Sep 10 04:37:44 2014
@@ -1,4 +1,4 @@
-@0=~This strange machine humms with internal power, Many wierd controlls cover the machine all with bright flashing lights, no matter how hard you try you cannot read any of the numerous lables, but a large green button on the front is labled simply " COPY " in common, you could push the button if you wished~
-@1=~You opperate the green copy button, the humming grows louder then a bright flash, then the lid of the machine opens~
-@2=~The machine seems lifeless and without power now, maybe if you found another power source it could function again~
-@3=~You are too far away to use that~
\ No newline at end of file
+@0=~This strange machine, covered all over with complex controls and flashing lights, hums with power. No matter how hard you try, you cannot read any of the numerous labels, except for the one under a large green button on the front, which reads "COPY".~
+@1=~As you opperate the green button, the humming grows louder. With a bright flash the lid of the machine opens.~
+@2=~The machine seems lifeless and without power now. Maybe with another power source, it could function once again.~
+@3=~You are too far away to use that.~
\ No newline at end of file
