--- vecna\Language\English\AA37TR04.tra	Fri Apr 30 23:56:31 2010
+++ C:\BWP Patchstudio\patched files\vecna\Language\English\AA37TR04.tra	Wed Sep 10 04:40:04 2014
@@ -1,6 +1,6 @@
-@0=~This strange box seems formed from the same semi-liqid metal that makes up the rest of this area, a indented handprint seems the only controll. you could put your hand in the indentation if you wished~
-@1=~As your hand rests on the cold surface of the container you feel suddenly stronger, but a flash occurs and creatures are summoned and some items appear ontop of the metal box.~
-@2=~Some powerfull psionic force is holding the items in place. You may not remove them until the guardians are dead.~
-@3=~The holding force is gone, you place the treasure in your pack~
-@4=~The box seems to be of no further intrest.~
-@5=~You are too far away to use that~
\ No newline at end of file
+@0=~This strange box seems formed from the same semi-liqid metal that makes up the rest of this area. An indented handprint seems the only control. You could place your hand in the indentation.~
+@1=~As your hand rests on the cold surface of the container, you suddenly feel stronger. With a flash, creatures are summoned and some items appear on top of the metal box.~
+@2=~Some powerful psionic force is holding the items in place. You may not remove them until the guardians are dead.~
+@3=~The holding force is gone, you place the treasure in your pack.~
+@4=~The box seems to be of no further interest.~
+@5=~You are too far away to use that.~
\ No newline at end of file
