--- vecna\Language\English\AA40DOOR.tra	Mon Apr 12 21:56:05 2010
+++ C:\BWP Patchstudio\patched files\vecna\Language\English\AA40DOOR.tra	Wed Sep 10 04:33:11 2014
@@ -1,7 +1,7 @@
-@0=~You feel very uneasy as you approach the sarcophagus. This is obviously the resting place of a powerefull member of the twisted rune. The lid looks heavy but you could remove it if you wish to~
-@1=~You push the heavy lid aside and the foul stench of rot nearly over comes. There is some treasure, and a stone jar with a golden stopper. Power surges and creatures appear.~
-@2=~Some powerfull psionic force is holding the items in place. You may not remove them until the guardians are dead.~
-@3=~The holding force is gone, you place the treasure and key in your pack, the jar though you feel it wise to leave or destroy~
-@4=~You strike the jar a mighty blow, you hear a sound very few have heard before, the sound of a Black soal sreaming as it is dragged to hell where it belongs. That is one less lich to plauge the waking world.~
+@0=~You feel very uneasy as you approach the sarcophagus. This is obviously the resting place of a powereful member of the Twisted Rune. The lid looks heavy, but you could probably remove it.~
+@1=~You push the heavy lid aside and the foul stench of rot nearly overcomes you. There is some treasure, and a stone jar with a golden stopper. You feel a surge of arcane power.~
+@2=~Some powerful psionic force is holding the items in place. You may not remove them until the guardians are dead.~
+@3=~The holding force is gone. You place the treasure and key in your pack. It would probably be wise to either leave ot destroy the jar.~
+@4=~As you strike the jar with a mighty blow, you hear a sound very few have heard before: The terrible shriek of a black soul being dragged to hell, where it belongs. That is one less lich to plauge the waking world.~
 @5=~The sarcophagus is now empty.~
-@6=~You are too far away to use that~
\ No newline at end of file
+@6=~You are too far away to use that.~
\ No newline at end of file
