--- vecna\Language\English\AA42CHS2.tra	Fri Apr 16 20:28:00 2010
+++ C:\BWP Patchstudio\patched files\vecna\Language\English\AA42CHS2.tra	Sat Sep 20 17:53:36 2014
@@ -1 +1 @@
-@0=~As you lift the heavy lid of the chest, the sound of gears and wheels tells you that the door at the end of the hall is being opened gradually~
\ No newline at end of file
+@0=~As you lift the heavy lid off the chest, your hear the sound of gears and wheels, signaling that the door at the end of the hall is opening slowly.~
\ No newline at end of file
