--- vecna\Language\English\AA50PLNT.tra	Thu Jun 16 20:55:35 2011
+++ C:\BWP Patchstudio\patched files\vecna\Language\English\AA50PLNT.tra	Wed Sep 10 04:17:17 2014
@@ -1,5 +1,5 @@
-@0=~This is some kind if advanced control but obviously without power~
-@1=~Now a screen on the control lights,you could innitiate a sequence if you wished~
-@2=~The machine behind the control springs into life, you feel your very being has been altered~
-@3=~The sequence has finished, it would take the original programmer to reset it~
-@4=~You feel somehow different, everything makes more sense?~
\ No newline at end of file
+@0=~This is some kind if advanced control panel. It has no power.~
+@1=~A screen on the control panel lights up. You could initiate a sequence.~
+@2=~The machine behind the control panel activates. You feel your very being has been altered.~
+@3=~The sequence has finished. It would take the original programmer to reset it.~
+@4=~You feel different, somehow. Everything makes more sense, or does it?~
\ No newline at end of file
