--- vecna\Language\English\AACRE193.tra	Sun Jul 10 13:01:50 2011
+++ C:\BWP Patchstudio\patched files\vecna\Language\English\AACRE193.tra	Wed Sep 10 17:50:00 2014
@@ -1 +1 @@
-@1=~HA! More fools after the map, I shall enjoy feasting on your bones mortal!.~
\ No newline at end of file
+@1=~Ha! More fools after the map! I shall enjoy feasting on your bones, mortal!~
\ No newline at end of file
