--- vecna\Language\English\AACRE001.tra	Fri May 13 20:13:16 2011
+++ C:\BWP Patchstudio\patched files\vecna\Language\English\AACRE001.tra	Sat Sep 20 17:41:09 2014
@@ -1 +1 @@
-@1=~Well what do we have here? some adventurers with no silver cord! Looks like christmas has come early eh lads.~
\ No newline at end of file
+@1=~Well, what do we have here? Some adventurers with no silver cord! Looks like Christma--I mean Solstice Eve!-- has come early, eh lads?~
\ No newline at end of file
