--- vecna\Language\English\AACRE141.tra	Thu Mar 18 00:00:31 2010
+++ C:\BWP Patchstudio\patched files\vecna\Language\English\AACRE141.tra	Tue Sep 09 23:44:38 2014
@@ -1 +1 @@
-@115=~Fools! you may have seen through this little ruse but you will NOT take the key~
\ No newline at end of file
+@115=~Fools! You may have seen through this little ruse, but you will NOT take the key!~
\ No newline at end of file
