--- vecna\Language\English\AACRE015.tra	Wed Mar 17 23:51:24 2010
+++ C:\BWP Patchstudio\patched files\vecna\Language\English\AACRE015.tra	Sat Sep 20 18:01:41 2014
@@ -1,11 +1,11 @@
-@104=~"and the twisted rune must not let that bitch rule our actions and furthermore.." Ho brothers we have strangers in our midst, now what would a puny godchild and even more insignificant friends be doing taking a wander in Hades ?~
-@105=~I have come to end your days thing of evil !~
-@106=~Does everyone know of me and my business !~
-@107=~I think you know why I am here~
-@108=~I was taking a wander through the planes thought this would be a fine place for tea~
-@109=~Oh not another do gooder !, very well then to battle, your father would have been so disappointed.~
-@110=~Not everyone, but I am not everyone~
-@111=~Damn your snide suggestions daemon tell me what you know of the twisted rune and I might spare you this time !~
-@112=~I don?t take kindly to threats fool, to battle brothers~
-@113=~Very wise godchild, maybe you are the one to take your fathers throne one day, we will not fight today, take this trinket it is worthless to me now but its magic will bring you the answers you seek , or your swift death. Now I bid you goodbye till next time we meet. Come brothers let us find a quieter place to talk~
-@114=~Well tea you shall have..... COVERED IN BLOOD BAFOON !!!~
\ No newline at end of file
+@104=~...and the Twisted Rune must not allow that harlot to rule our actions. Furthermore--Ho, brothers! We have strangers in our midst! ...Now, what would a puny god-child and <PRO_HISHER> even less significant underlings be doing here, bumbling through Hades?~
+@105=~I have come to end your days, creature of evil!~
+@106=~Does everyone know of me and my business?!~
+@107=~I think you know why I am here.~
+@108=~I was taking a stroll through the planes, thought this would be a nice place for a warm cup of tea.~
+@109=~Oh bother. Not another bloody would-be saint! Very well, then. To battle! ...Your father would have been so disappointed.~
+@110=~Not everyone does. But I am not 'everyone'.~
+@111=~To the hells with your snide remarks, daemon! Tell me what you know of the Twisted Rune and I might just spare you this time!~
+@112=~I don't respond well to threats, fool. To battle, brothers!~
+@113=~Very wise, god-child. Maybe you are the one to take your fathers throne one day. We shall not do battle this day. Take this trinket. It is worthless to me now, but its magic will bring you the answers you seek,... or your swift demise. Now I must bid you farewell. Come, brothers! Let us continue our meeting away from any more useless distractions.~
+@114=~I have your cup of tea right here... FILLED WITH YOUR BLOOD, BUFFOON!~
\ No newline at end of file
