--- vecna\Language\English\AACRE136.tra	Fri May 13 14:09:11 2011
+++ C:\BWP Patchstudio\patched files\vecna\Language\English\AACRE136.tra	Sat Sep 20 17:41:09 2014
@@ -1 +1 @@
-@1=~Flee mortals, i have been created to guard this piece of astral space, retreat, flee, or whatever but pass by this place and you shall dieeeeeeeeee!!!!!.~
\ No newline at end of file
+@1=~Flee, mortals! I have been created to guard this scrap of astral space. Retreat, flee, or whatever--but pass by this place and you shall perish!~
\ No newline at end of file
