--- vecna\Language\English\AACRE018.tra	Wed Mar 17 23:51:54 2010
+++ C:\BWP Patchstudio\patched files\vecna\Language\English\AACRE018.tra	Wed Sep 10 00:50:23 2014
@@ -1 +1 @@
-@40=~HA! More fools after the map, I shall enjoy feasting on your bones mortal!.~
\ No newline at end of file
+@40=~Ha! More fools after the map! I shall enjoy feasting on your bones, mortal!~
\ No newline at end of file
