--- vecna\Language\English\AACRE121.tra	Thu Mar 18 00:04:09 2010
+++ C:\BWP Patchstudio\patched files\vecna\Language\English\AACRE121.tra	Sat Sep 20 17:41:09 2014
@@ -1 +1 @@
-@41=~Well done child of ball, as a reward you may keep what you find here, except for the Eye and Hand leave them here or face the displeasure of Helm.~
\ No newline at end of file
+@41=~Well done, Child of Bhaal! As a reward you may keep what you find here, except for the Eye and Hand--leave them here or risk drawing the ire of Helm.~
\ No newline at end of file
