--- vecna\Language\English\AACRE014.tra	Fri May 07 02:08:29 2010
+++ C:\BWP Patchstudio\patched files\vecna\Language\English\AACRE014.tra	Wed Sep 10 17:37:19 2014
@@ -1 +1 @@
-@1=~I know not how you have managed to get past all the defences that keep me safe untill the transformation, but you now face Zod, although i smell helm on you, it will not save you, I am no god yet but this form will suffice to see your doom.~
\ No newline at end of file
+@1=~I know not how you have managed to breach all my defenses, only to disturb my transformation. But you now face Zod! You reek of Helm, mortal... He will not save you. I may not be a god yet, but this shell will more than suffice to spell your doom!~
\ No newline at end of file
