--- vecna\Language\English\AACRE009.tra	Wed Mar 17 23:48:14 2010
+++ C:\BWP Patchstudio\patched files\vecna\Language\English\AACRE009.tra	Sat Sep 20 17:52:53 2014
@@ -1 +1 @@
-@94=~Strangers ! Good have waited a centaury guarding this dump, and not tasted man flesh for sooooo long~
\ No newline at end of file
+@94=~Intruders! Good. For a century have I guarded this dump. I have not tasted sweet man-flesh for far too long! ...that came out wrong.~
\ No newline at end of file
