--- vecna\Language\English\AACRE013.tra	Wed Mar 17 23:49:11 2010
+++ C:\BWP Patchstudio\patched files\vecna\Language\English\AACRE013.tra	Wed Sep 10 04:07:40 2014
@@ -1 +1 @@
-@40=~HA! More fools after the map, I shall enjoy feasting on your bones mortal!.~
\ No newline at end of file
+@40=~Ha! More fools after the map! I shall enjoy feasting on your bones, mortal!~
\ No newline at end of file
