--- vecna\Language\English\AAFIGHT1.tra	Sat Mar 18 19:42:27 2006
+++ C:\BWP Patchstudio\patched files\vecna\Language\English\AAFIGHT1.tra	Sat Sep 20 17:58:47 2014
@@ -1,7 +1,7 @@
-@0    = ~Help! I'm...poisoned.~
+@0    = ~Help! I'm... poisoned.~
 @1    = ~Knave! Come fight like a warrior...~
-@2    = ~Come closer and fight me fair!~
+@2    = ~Come closer and fight me fairly!~
 @3    = ~You coward! Using a ranged weapon to attack me!~
-@4    = ~Barb-slinger! I bet you`re too weak for melee...~
+@4    = ~Barb-slinger! I bet you're too weak for melee...~
 @5    = ~Barb-slinger! Let's see if you can melee...~
 @6    = ~Die with your boots on!~
