--- vecna\baf\chr\AAMG22.baf	Sun Jul 17 20:58:20 2011
+++ C:\BWP Patchstudio\patched files\vecna\baf\chr\AAMG22.baf	Fri Oct 28 08:48:23 2011
@@ -1,7 +1,7 @@
 IF
 	See(NearestEnemyOf(Myself))
 	Global("Prep","LOCALS",0)
-	!Dead("yself)")
+	!StateCheck(Myself,STATE_REALLY_DEAD)
 THEN
 	RESPONSE #100
 		ApplySpell(Myself,WIZARD_IMPROVED_MANTLE)
@@ -12,7 +12,7 @@
 IF
 	See(NearestEnemyOf(Myself))
 	Global("SpellTrigger","LOCALS",0)
-	!Dead("yself)")
+	!StateCheck(Myself,STATE_REALLY_DEAD)
 THEN
 	RESPONSE #100
 		DisplayString(Myself,39968) // Spell Trigger - Fired
@@ -58,7 +58,7 @@
 IF
 	See(NearestEnemyOf(Myself))
 	Global("Prep","LOCALS",1)
-	!Dead("yself)")
+	!StateCheck(Myself,STATE_REALLY_DEAD)
 THEN
 	RESPONSE #100
 		Spell(Myself,WIZARD_SPELL_TURNING)
@@ -79,7 +79,7 @@
 	See(NearestEnemyOf(Myself))
 	TookDamage()
 	Global("Prep2","LOCALS",0)
-	!Dead("yself)")
+	!StateCheck(Myself,STATE_REALLY_DEAD)
 THEN
 	RESPONSE #100
 		DisplayString(Myself,43050) // Chain Contingency - Improved Mantle
@@ -95,7 +95,7 @@
 	See(NearestEnemyOf(Myself))
 	TookDamage()
 	Global("Prep2","LOCALS",1)
-	!Dead("yself)")
+	!StateCheck(Myself,STATE_REALLY_DEAD)
 THEN
 	RESPONSE #100
 		DisplayString(Myself,40252) // Contingency - Protection from Magical Weapons
