--- vecna\Language\English\AAR010C1.tra	Sun Jun 19 01:08:19 2011
+++ C:\BWP Patchstudio\patched files\vecna\Language\English\AAR010C1.tra	Sat Sep 20 17:56:21 2014
@@ -1,4 +1,4 @@
-@0=~Stupid squashy creatures! A tactical quantum weapon will not destroy me ~
-@1=~Already my self repair systems are working, 30 seconds and you will be dead! ~
-@2=~What! I detect a presence approaching, all defence fields to maximum~
-@3=~oh no it cant be ! An elder i will...ARHHHHH~
\ No newline at end of file
+@0=~Your attempts are futile, mortal. Such a device will not destroy me.~
+@1=~Already, this shell is reassembling itself. In thirty seconds you will perish.~
+@2=~Something is approaching...~
+@3=~An elder being? Adjusting stratagems. I will--ARHHHHH!~
\ No newline at end of file
