--- vecna\Language\English\AAR036B.tra	Fri May 07 02:17:02 2010
+++ C:\BWP Patchstudio\patched files\vecna\Language\English\AAR036B.tra	Wed Sep 10 20:06:59 2014
@@ -1 +1 @@
-@1=~I am Loloth's pity, long have i held the weakness of pity away from my mistress.~
\ No newline at end of file
+@1=~I am Lolth's Pity. Long have I kept the weakness of mercy away from my mistress.~
\ No newline at end of file
