--- vecna\Language\English\AAR001C1.tra	Thu Jul 07 19:53:14 2011
+++ C:\BWP Patchstudio\patched files\vecna\Language\English\AAR001C1.tra	Sat Sep 20 17:41:09 2014
@@ -1,3 +1,3 @@
-@0=~Ah the intruders we had been warned of, never mind we are well prepared, come my minions and allies let us show these fools real power~
-@1=~So the dark bitch requires my help eh! Well you know the price you pice of elven turd, we will talk of my payment after these fools are meat on my plate~
-@2=~WOW the stench in here! A fake baboon, fake god, fake godling, your calling in all your favours bitch, well im here now, lets roll!~
\ No newline at end of file
+@0=~Ah, the intruder we have been warned of! Never mind... We are well prepared. Come, my minions and allies! Let us show these fools true power!~
+@1=~So the Dark Queen requires my help, eh? Well, you know the price, you piece of elven waste. We will talk of payment after these fools are meat on my platter.~
+@2=~Wow, the stench in here! A fake baboon, fake god, fake godling--You are calling in all your favours, dark one. Well, I'm here now. Let's get this over with!~
\ No newline at end of file
