--- vecna\Language\English\AAR006.tra	Fri Apr 09 11:06:49 2010
+++ C:\BWP Patchstudio\patched files\vecna\Language\English\AAR006.tra	Sat Sep 20 17:55:02 2014
@@ -1 +1 @@
-@0=~When braught close together some innate magic within the wardstone fragments triggers and after a brief flash you are left holding a complete wardstone key~
\ No newline at end of file
+@0=~When brought close together, some innate magic within the wardstone fragments triggers. After a brief flash you are left holding a complete wardstone key.~
\ No newline at end of file
