--- vecna\Language\English\AAtelep.tra	Sun Apr 13 15:12:08 2008
+++ C:\BWP Patchstudio\patched files\vecna\Language\English\AAtelep.tra	Sat Sep 20 18:02:08 2014
@@ -7,42 +7,40 @@
 Area of Effect: Party
 Saving Throw: None
 
-With this powerful spell the caster may teleport himself and his entire
-party to places they have already visited. There are a few areas, mostly
-extraplanar, from which you are not permitted to teleport.~
+With this powerful spell the caster may teleport the party to places they have already visited. There are a few areas, mostly extraplanar, from which you are not permitted to teleport.~
 
 @3    = ~Choose destination area~
-@4    = ~De´Arnise keep~
+@4    = ~De'Arnise keep~
 @5    = ~Next areas~
 @6    = ~Nowhere~
 @7    = ~Sahuagin city~
 @8    = ~Back~
 @9    = ~Sewers beneath Slums~
 @10   = ~Entrance floor~
-@11   = ~Navigator´s room~
+@11   = ~Navigator's room~
 @12   = ~Ice and Fire Room~
 @13   = ~Engine room~
 @14   = ~Beneath graveyard~
-@15   = ~Bodhi´s lair~
+@15   = ~Bodhi's lair~
 @16   = ~Harper Hold~
-@17   = ~Shadow Thief guildhouse entrance~
+@17   = ~Shadow Thief Guildhouse entrance~
 @18   = ~Playhouse~
 @19   = ~Order of the Radiant Heart ~
-@20   = ~Mekrath´s hideout~
-@21   = ~The Unseeing Eye cult lair~
-@22   = ~Unseeing Eye´s hideout~
+@20   = ~Mekrath's Hideout~
+@21   = ~The Unseeing Eye Cult Lair~
+@22   = ~Unseeing Eye's Hideout~
 @23   = ~Forgotten city~
 @24   = ~Temple of the Forgotten God~
-@25   = ~Court of de´Arnise keep~
-@26   = ~First floor of de´Arnise keep~
-@27   = ~Second floor of de´Arnise keep~
-@28   = ~Ranger´s cabin~
+@25   = ~Court of de'Arnise keep~
+@26   = ~First floor of de'Arnise keep~
+@27   = ~Second floor of de'Arnise keep~
+@28   = ~Ranger's cabin~
 @29   = ~Temple of Amaunator~
-@30   = ~Shadow Dragon´s hideout~
+@30   = ~Shadow Dragon's hideout~
 @31   = ~Garren Windspear's home~
-@32   = ~Firkraag´s dungeon~
-@33   = ~Entrance to Firkraag´s dungeon~
-@34   = ~Firkraag´s lair~
+@32   = ~Firkraag's dungeon~
+@33   = ~Entrance to Firkraag's dungeon~
+@34   = ~Firkraag's lair~
 @35   = ~Druid grove~
 @36   = ~Brothel~
 @37   = ~Kuo-toas~
@@ -55,12 +53,12 @@
 @44   = ~Beginning of maze~
 @45   = ~End of maze~
 @46   = ~Penultimate floor~
-@47   = ~Second floor of Yaga-Shura´s lair~
-@48   = ~Lycanth´s laboratory~
-@49   = ~Entrance in Sendai´s enclave~
+@47   = ~Second floor of Yaga-Shura's lair~
+@48   = ~Lycanth's laboratory~
+@49   = ~Entrance in Sendai's enclave~
 @50   = ~Crossroads~
-@51   = ~Sendai´s inner sanctum~
-@52   = ~You can´t teleport from this area.~
+@51   = ~Sendai's inner sanctum~
+@52   = ~You can't teleport from this area.~
 @53   = ~Something is disturbing your attempt to teleport from this area.~
 @54   = ~Modification of NPCs and strongholds quests.~
 @55  = ~Teleport spell for BG2:SoA~
