--- allison\english\r#allis.TRA	Mon Oct 01 16:50:26 2007
+++ C:\BWP Patchstudio\patched files\allison\english\r#allis.TRA	Fri Nov 12 16:51:59 2010
@@ -116,7 +116,7 @@
 @115  = ~Who the hell are you?~
 @116  = ~(fondle her while she's hugging you)~
 @117  = ~Well, you're welcome.~
-@118  = ~(she pushes herself away from you) My apologies, m'<LORDLADY>. I was simply so grateful to you for saving the grove.~
+@118  = ~(she pushes herself away from you) My apologies, m'<LADYLORD>. I was simply so grateful to you for saving the grove.~
 @119  = ~Whatever. Can I go now?~
 @120  = ~You're welcome, but this is hardly necessary.~
 @121  = ~Oh, sorry. I forgot that we had never met.~
