--- arestorationp\baf\l-ar01.baf	Mon Apr 21 23:54:22 2014
+++ C:\BWP Patchstudio\patched files\arestorationp\baf\l-ar01.baf	Wed Apr 23 21:10:29 2014
@@ -1,5 +1,5 @@
 IF
-	OnCreation()
+//	OnCreation()
 	Global("L-SpawnBandints","GLOBAL",0)
 THEN
 	RESPONSE #100
@@ -14,7 +14,7 @@
 END
 
 IF
-	OnCreation()
+//	OnCreation()
 	Global("L-SpawnBandints","GLOBAL",1)
 	Global("L-SpawnEler","GLOBAL",0)
 THEN
