--- arestorationp\setup-arestorationp.tp2	Wed Apr 23 03:07:09 2014
+++ C:\BWP Patchstudio\patched files\arestorationp\setup-arestorationp.tp2	Wed Apr 23 21:08:19 2014
@@ -1,6 +1,6 @@
 BACKUP ~arestorationp/backup~
 AUTHOR ~Almateria (sylwiabukato@gmail.com)~
-VERSION ~v4~
+VERSION ~v4 BWP Fix~
 AUTO_TRA ~arestorationp/%s~
 LANGUAGE ~Polski~	~polski~	~arestorationp/polski/setup.tra~
 LANGUAGE ~English~	~english~	~arestorationp/english/setup.tra~
