--- stratagems/ascension/ssl/abazdg02.ssl	2018-08-30 07:50:32.431432100 +0200
+++ Fixpack/stratagems/ascension/ssl/abazdg02.ssl	2018-08-30 07:52:13.665119700 +0200
@@ -229,8 +229,7 @@
 	OR(2)
 		!Gender(scstarget,SUMMONED)
 		HasItem("morsword",scstarget)
-	TriggerBlock(ResistMagicDamage|MR|SINecromancy|Enemy)
-	!HasItem("clck26",scstarget)
+	TriggerBlock(ResistMagicDamage|MR|Enemy)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_ABI_DALZIMS_HORRID_WILTING|400|100)
@@ -259,13 +258,14 @@
 
 IF TRIGGER
 	TargetBlock(PCsInOrder)
-	TriggerBlock(SpellTurn|Enemy|Helpless)
-	OR(5)
+	TriggerBlock(SpellTurn|SpellTurnNew|Enemy|Helpless)
+	OR(6)
 		CheckStatGT(scstarget,0,WIZARD_PROTECTION_FROM_MAGIC_ENERGY)
 		CheckStatGT(scstarget,0,SCRIPTINGSTATE2)
 		CheckStatGT(scstarget,0,CLERIC_CHAOTIC_COMMANDS)
 		CheckStatGT(scstarget,0,WIZARD_PROTECTION_FROM_MAGIC_WEAPONS)
 		CheckStatGT(scstarget,2,STONESKINS)
+		CheckStat(scstarget,4,WIZARD_SPELL_IMMUNITY)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_BREACH,WIZARD_BREACH|150|150)
@@ -275,7 +275,7 @@
 
 IF TRIGGER
 	TargetBlock(PCsInOrder)
-	OR(13)
+	OR(14)
 		CheckStatGT(scstarget,0,STONESKINS)
 		CheckStatGT(scstarget,0,WIZARD_PROTECTION_FROM_MAGIC_WEAPONS)
 		StateCheck(scstarget,STATE_IMPROVEDINVISIBILITY)
@@ -289,11 +289,13 @@
 		CheckStatGT(scstarget,0,CLERIC_CHAOTIC_COMMANDS)
 		CheckStatGT(scstarget,0,WIZARD_PROTECTION_FROM_PETRIFICATION)
 		CheckStatGT(scstarget,0,SCRIPTINGSTATE2)
+		CheckStat(scstarget,4,WIZARD_SPELL_IMMUNITY)
+		
 	BEGIN LOOP(scsvar||1;2;3;4;5;6)
 		!StateCheck(Playerscsvar,STATE_DISABLED)
 		!Allegiance(Playerscsvar,ENEMY)
 	END LOOP
-	TriggerBlock(SIAbjuration|Enemy)
+	TriggerBlock(Enemy)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_DISPEL_MAGIC|100|50)
@@ -345,11 +347,10 @@
 
 IF TRIGGER
 	TargetBlock(PCsInOrder)
-	OR(5)
+	OR(4)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_TURNING)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_DEFLECTION)
 		CheckStat(scstarget,1,WIZARD_SPELL_TRAP)
-		CheckStatGT(scstarget,0,WIZARD_SPELL_IMMUNITY)
 		CheckStatGT(scstarget,0,MINORGLOBE)
 	TriggerBlock(Enemy|Helpless)
 THEN DO
@@ -367,7 +368,7 @@
 
 IF TRIGGER
 	TargetBlock(PCSpellcasters)
-	TriggerBlock(MR100|SIEvocation|Enemy)
+	TriggerBlock(MR100|Enemy|MinorGlobe)
 	StateCheck(scstarget,STATE_MIRRORIMAGE)
 THEN DO
 	Combine()
@@ -382,7 +383,7 @@
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC)&HPPercentLT(scstarget,40))
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(MR100|SIEvocation|MinorGlobe|SpellTurn|Enemy|L1safe)
+	TriggerBlock(MR100|MinorGlobe|SpellTurnNew|Enemy)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_MAGIC_MISSILE)
