--- stratagems/lib/always.tph	2016-11-21 19:00:36.654020700 -0500
+++ Fixpack/stratagems/lib/always.tph	2016-12-01 15:01:59.419080700 -0500
@@ -303,7 +303,7 @@
 // detect whether Truesight cures blindness
 ///////////////////////////////////////////////////
 
-    ACTION_IF FILE_EXISTS_IN_GAME ~dw#truesight_blindness.mrk~ BEGIN
+    ACTION_IF FILE_EXISTS_IN_GAME ~dw#truesight_blindness.mrk~ OR FILE_EXISTS_IN_GAME ~dvimhere.mrk~ BEGIN
            OUTER_SPRINT ~sslvariables~ ~%sslvariables%~^~&TruesightCuresBlindness=True~
     END
     
