--- stratagems/mage/ssl/bg1/combatblocks/areadenial.ssl	2016-11-20 18:27:15.398470600 -0500
+++ Fixpack/stratagems/mage/ssl/bg1/combatblocks/areadenial.ssl	2016-12-01 15:11:46.501882400 -0500
@@ -6,7 +6,6 @@
 	!GlobalTimerNotExpired("areadenial","LOCALS")
 	RequireBlock(IsUndead)
 	RequireBlock(HasL5)
-	IgnoreBlock(IsEnchanter)
 	TargetBlock(EnemiesInOrder)
 	TriggerBlock(MR100|Poison)
 THEN DO
@@ -17,7 +16,6 @@
 IF TRIGGER
 	!GlobalTimerNotExpired("areadenial","LOCALS")
 	RequireBlock(HasL2)
-	IgnoreBlock(IsEnchanter)
 	CheckStatGT(Myself,0,MINORGLOBE)
 	!NumCreatureGT([ENEMY.HUMANOID],1)
 	Target(NearestEnemyOf(Myself))
@@ -29,7 +27,6 @@
 IF TRIGGER
 	!GlobalTimerNotExpired("areadenial","LOCALS")
 	RequireBlock(HasL2)
-	IgnoreBlock(IsEnchanter)
 	CheckStatGT(Myself,0,MINORGLOBE)
 	!NumCreatureGT([ENEMY.HUMANOID],1)
 	Target(NearestEnemyOf(Myself))
--- stratagems/mage/ssl/bg2/combatblocks/areadenial.ssl	2016-11-20 18:27:15.413481100 -0500
+++ Fixpack/stratagems/mage/ssl/bg2/combatblocks/areadenial.ssl	2016-12-01 15:11:46.516892800 -0500
@@ -21,7 +21,6 @@
 	RequireBlock(IsRakshasa)
 	TargetBlock(EnemiesInOrder)
 	TriggerBlock(MR100|Poison)
-	!HasItem("clck26",scstarget)
 THEN DO
         Combine()
 	SetGlobalTimer("areadenial","LOCALS",18)
@@ -32,11 +31,9 @@
 IF TRIGGER
 	!GlobalTimerNotExpired("areadenial","LOCALS")
 	RequireBlock(IsLich|IsVampire)
-	IgnoreBlock(IsEnchanter)
 	RequireBlock(HasL5)
 	TargetBlock(EnemiesInOrder)
 	TriggerBlock(MR100|Poison)
-	!HasItem("clck26",scstarget)
 	!NumCreatureGT([ENEMY.HUMANOID],1)
 THEN DO
         Combine()
