--- stratagems/mage/ssl/bg1/combatblocks/arrows.ssl	2016-11-20 18:27:15.400472000 -0500
+++ Fixpack/stratagems/mage/ssl/bg1/combatblocks/arrows.ssl	2016-11-20 18:27:39.737045400 -0500
@@ -24,7 +24,7 @@
 	Allegiance(Myself,ENEMY)
 	ConditionalTargetBlock(PCSpellcasters;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(PCSpellcasters)
-	TriggerBlock(MR100|Stoneskin|MirrorImage|MinorGlobe|ArrowSafe|SpellTurn|Enemy)
+	TriggerBlock(MR100|MinorGlobe|ArrowSafe|SpellTurnNew|Enemy)
 THEN DO
 	Action(Spell,WIZARD_MELF_ACID_ARROW|100|50)
 END
@@ -38,7 +38,7 @@
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC)&HPPercentLT(scstarget,40))
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(MR100|MinorGlobe|SpellTurn|Enemy|MagicMissile)
+	TriggerBlock(MR100|MinorGlobe|SpellTurnNew|Enemy|MagicMissile)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_CHROMATIC_ORB)
@@ -50,7 +50,7 @@
 	RequireBlock(HasL2)
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(Enemy|MR100|Stoneskin|MirrorImage|MinorGlobe|ArrowSafe|SpellTurn)
+	TriggerBlock(Enemy|MR100|MirrorImage|MinorGlobe|ArrowSafe|SpellTurnNew)
 THEN DO
 	Action(Spell,WIZARD_MELF_ACID_ARROW)
 END
@@ -63,7 +63,7 @@
 IF TRIGGER
 	TargetBlock(PlayersInOrder)
 	Allegiance(Myself,ENEMY)
-	TriggerBlock(MR100|SpellTurn|MinorGlobe)
+	TriggerBlock(MR100|SpellTurnNew|MinorGlobe)
 THEN DO
 	Action(Spell,WIZARD_FLAME_ARROW)
 	Action(Spell,WIZARD_MELF_ACID_ARROW)
--- stratagems/mage/ssl/bg2/combatblocks/arrows.ssl	2016-11-20 18:27:15.414482000 -0500
+++ Fixpack/stratagems/mage/ssl/bg2/combatblocks/arrows.ssl	2016-11-20 18:27:39.750054400 -0500
@@ -6,7 +6,7 @@
 	TargetBlock(PCSpellcasters)
 	IgnoreBlock(Indiscriminate)
 	IgnoreBlock(IsEnchanter)
-	TriggerBlock(MR100|SIEvocation|Enemy|MagicMissile|L1safe|MinorGlobe|SpellTurn)
+	TriggerBlock(MR100|ResistMagicDamage|Enemy|MagicMissile|MinorGlobe|SpellTurnNew)
 	StateCheck(scstarget,STATE_MIRRORIMAGE)
 	Allegiance(Myself,ENEMY)
 THEN DO
@@ -17,7 +17,7 @@
 	TargetBlock(EnemiesInOrder)
 	RequireBlock(Indiscriminate)
 	IgnoreBlock(IsEnchanter)
-	TriggerBlock(MR100|SIEvocation|MagicMissile|L1safe|MinorGlobe|SpellTurn)
+	TriggerBlock(MR100|ResistMagicDamage|MagicMissile|MinorGlobe|SpellTurnNew)
 	StateCheck(scstarget,STATE_MIRRORIMAGE)
 THEN DO
 	Action(Spell,WIZARD_MAGIC_MISSILE)
@@ -36,8 +36,7 @@
 	RequireBlock(HasL4)
 	ConditionalTargetBlock(PCSpellcasters;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(PCSpellcasters)
-	TriggerBlock(DirectDamageSafe)
-	TriggerBlock(MR100|SIConjuration|MirrorImage|MinorGlobe|ArrowSafe|SpellTurn|Enemy|L2safe)
+	TriggerBlock(MR100|MirrorImage|MinorGlobe|ArrowSafe|SpellTurnNew|Enemy)
 THEN DO
 	Action(MinorSequencer,3,WIZARD_MELF_ACID_ARROW,WIZARD_MELF_ACID_ARROW)
 END
@@ -47,8 +46,7 @@
 	IgnoreBlock(IsEvoker)
 	RequireBlock(HasL4)
 	TargetBlock(EnemiesInOrder)
-	TriggerBlock(DirectDamageSafe)
-	TriggerBlock(MR100|SIConjuration|MirrorImage|MinorGlobe|ArrowSafe|SpellTurn|L2safe)
+	TriggerBlock(MR100|MirrorImage|MinorGlobe|ArrowSafe|SpellTurnNew|Enemy)
 THEN DO
 	Action(MinorSequencer,3,WIZARD_MELF_ACID_ARROW,WIZARD_MELF_ACID_ARROW)
 END
@@ -61,8 +59,7 @@
 	RequireBlock(HasL2)
 	ConditionalTargetBlock(PCSpellcasters;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(PCSpellcasters)
-	TriggerBlock(DirectDamageSafe)
-	TriggerBlock(MR100|SIConjuration|MirrorImage|MinorGlobe|ArrowSafe|SpellTurn|Enemy|L2safe)
+	TriggerBlock(MR100|MirrorImage|MinorGlobe|ArrowSafe|SpellTurnNew|Enemy)
 THEN DO
 	Action(Spell,WIZARD_MELF_ACID_ARROW|100|50)
 END
@@ -72,8 +69,7 @@
 	IgnoreBlock(IsEvoker)
 	RequireBlock(HasL2)
 	TargetBlock(EnemiesInOrder)
-	TriggerBlock(DirectDamageSafe)
-	TriggerBlock(MR100|SIConjuration|MirrorImage|MinorGlobe|ArrowSafe|SpellTurn|L2safe)
+	TriggerBlock(MR100|MirrorImage|MinorGlobe|ArrowSafe|SpellTurnNew)
 THEN DO
 	Action(Spell,WIZARD_MELF_ACID_ARROW|100|50)
 END
@@ -90,8 +86,7 @@
 	IgnoreBlock(Demivrgvs)
 	IgnoreBlock(IsEnchanter)
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(DirectDamageSafe)
-	TriggerBlock(MR100|SIEvocation|MinorGlobe|SpellTurn|Enemy|MagicMissile|L1safe|ResistMagicDamage)
+	TriggerBlock(MR100|MinorGlobe|SpellTurnNew|Enemy|MagicMissile|ResistMagicDamage)
 THEN DO
 	Combine()
 	Action(MinorSequencer,2,WIZARD_MAGIC_MISSILE,WIZARD_MAGIC_MISSILE)
@@ -123,8 +118,7 @@
 	RequireBlock(Demivrgvs)
 	IgnoreBlock(IsEnchanter)
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(DirectDamageSafe)
-	TriggerBlock(MR100|SIEvocation|MinorGlobe|SpellTurn|Enemy|MagicMissile|L1safe|ResistMagicDamage)
+	TriggerBlock(MR100|MinorGlobe|SpellTurnNew|Enemy|MagicMissile|ResistMagicDamage)
 THEN DO
 	Combine()
 	Action(MinorSequencer,2,WIZARD_MAGIC_MISSILE,WIZARD_MAGIC_MISSILE)
@@ -140,8 +134,7 @@
 	IgnoreBlock(IsEnchanter)
 	RequireBlock(Demivrgvs)
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(DirectDamageSafe)
-	TriggerBlock(MR100|SIEvocation|MinorGlobe|SpellTurn|MagicMissile|L1safe|ResistMagicDamage)
+	TriggerBlock(MR100|MinorGlobe|SpellTurnNew|MagicMissile|ResistMagicDamage)
 THEN DO
 	Combine()
 	Action(MinorSequencer,2,WIZARD_MAGIC_MISSILE,WIZARD_MAGIC_MISSILE)
@@ -158,8 +151,7 @@
 	TargetBlock(EnemiesInOrderShort)
 	IgnoreBlock(IsEvoker)
 	RequireBlock(HasL2)
-	TriggerBlock(DirectDamageSafe)
-	TriggerBlock(Enemy|MR100|SIConjuration|Stoneskin|MirrorImage|MinorGlobe|ArrowSafe|SpellTurn|L2safe)
+	TriggerBlock(Enemy|MR100|MirrorImage|MinorGlobe|ArrowSafe|SpellTurnNew)
 THEN DO
 	Action(Spell,WIZARD_MELF_ACID_ARROW)
 END
@@ -169,8 +161,7 @@
 	TargetBlock(EnemiesInOrderShort)
 	IgnoreBlock(IsEvoker)
 	RequireBlock(HasL2)
-	TriggerBlock(DirectDamageSafe)
-	TriggerBlock(MR100|SIConjuration|Stoneskin|MirrorImage|MinorGlobe|ArrowSafe|SpellTurn|L2safe)
+	TriggerBlock(MR100|MirrorImage|MinorGlobe|ArrowSafe|SpellTurnNew)
 THEN DO
 	Action(Spell,WIZARD_MELF_ACID_ARROW)
 END
@@ -183,8 +174,7 @@
 	TargetBlock(PlayersInOrder)
 	Allegiance(Myself,ENEMY)
 	IgnoreBlock(IsEnchanter)
-	TriggerBlock(DirectDamageSafe)
-	TriggerBlock(MR100|SpellTurn|MinorGlobe|L1safe|MagicMissile|ResistMagicDamage)
+	TriggerBlock(MR100|SpellTurnNew|MinorGlobe|MagicMissile|ResistMagicDamage)
 THEN DO
 	Action(Spell,WIZARD_MAGIC_MISSILE)
 END
@@ -195,8 +185,7 @@
 	Allegiance(Myself,ENEMY)
 	IgnoreBlock(IsEvoker)
 	RequireBlock(HasL3)
-	TriggerBlock(DirectDamageSafe)
-	TriggerBlock(MR100|SpellTurn|MinorGlobe)
+	TriggerBlock(MR100|SpellTurnNew|MinorGlobe)
 THEN DO
 	Action(Spell,WIZARD_FLAME_ARROW)
 END
@@ -206,8 +195,7 @@
 	Allegiance(Myself,ENEMY)
 	IgnoreBlock(IsEvoker)
 	RequireBlock(HasL2)
-	TriggerBlock(DirectDamageSafe)
-	TriggerBlock(MR100|SpellTurn|MinorGlobe)
+	TriggerBlock(MR100|SpellTurnNew|MinorGlobe)
 THEN DO
 	Action(Spell,WIZARD_MELF_ACID_ARROW)
 END
@@ -216,8 +204,7 @@
 	TargetBlock(PlayersInOrder)
 	Allegiance(Myself,ENEMY)
 	IgnoreBlock(IsEnchanter)
-	TriggerBlock(DirectDamageSafe)
-	TriggerBlock(MR100|SpellTurn|MinorGlobe)
+	TriggerBlock(MR100|SpellTurnNew|MinorGlobe)
 THEN DO
 	Action(Spell,WIZARD_CHROMATIC_ORB)
 END
@@ -229,7 +216,7 @@
         IgnoreBlock(IsEvokerStreamline)
         RequireBlock(Demivrgvs)
 	TargetBlock(PlayersInOrder)
-	TriggerBlock(MR100|Enemy|SpellTurn|MinorGlobe|SINecromancy)
+	TriggerBlock(MR100|Enemy|SpellTurnNew|MinorGlobe)
 THEN DO
 	Action(Spell,WIZARD_LARLOCH_MINOR_DRAIN)
 END
