--- stratagems/mage/ssl/bg1/combatblocks/attack_antimagic.ssl	2016-11-20 18:27:15.401472500 -0500
+++ Fixpack/stratagems/mage/ssl/bg1/combatblocks/attack_antimagic.ssl	2016-11-20 18:27:39.738045900 -0500
@@ -7,10 +7,11 @@
 	IgnoreBlock(SpellsAreBG1)
 	RequireBlock(HasL3)
 	TargetBlock(PCsInOrder)
-	OR(2)
+	OR(3)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_TURNING)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_DEFLECTION)
-	TriggerBlock(Enemy|Helpless|MinorGlobe)
+		CheckStatGT(scstarget,0,MINORGLOBE)
+	TriggerBlock(Enemy|Helpless)
 	!GlobalTimerNotExpired("DMWW_dispel","LOCALS")
 THEN DO
 	SetGlobal("DMWW_dispel","LOCALS",18)
