--- stratagems/mage/spellchoices/demivrgvs/bard.2da	2012-08-26 17:08:32.000000000 -0400
+++ Fixpack/stratagems/mage/spellchoices/demivrgvs/bard.2da	2016-12-01 15:11:46.489873400 -0500
@@ -3,7 +3,7 @@
 2 MELF_ACID_ARROW       MELF_ACID_ARROW         MELF_ACID_ARROW             POWER_WORD_SLEEP        POWER_WORD_SLEEP        HORROR          GLITTERDUST     GLITTERDUST
 3 HOLD_PERSON           DIRE_CHARM              MONSTER_SUMMONING_1                                                                         
 4 MONSTER_SUMMONING_2   SPIDER_SPAWN            EMOTION_HOPELESSNESS       EMOTION_HOPELESSNESS             CONFUSION               CONFUSION                   
-5 CHAOS                 CHAOS                   DOMINATION                  SUN_FIRE                                                             
+5 CHAOS                 SUN_FIRE                DOMINATION                  SUN_FIRE                                                             
 6 DEATH_SPELL           CHAIN_LIGHTNING         POWER_WORD_SILENCE          POWER_WORD_SILENCE                                                  
 7 POWER_WORD_STUN       POWER_WORD_STUN         POWER_WORD_STUN             MORDENKAINENS_SWORD      SPHERE_OF_CHAOS         PRISMATIC_SPRAY             
 8 POWER_WORD_BLIND      POWER_WORD_BLIND        ABI_DALZIMS_HORRID_WILTING  MAZE                                                                
