--- stratagems/mage/ssl/bg2/combatblocks/barrage.ssl	2018-08-30 07:50:32.463431300 +0200
+++ Fixpack/stratagems/mage/ssl/bg2/combatblocks/barrage.ssl	2018-08-30 07:52:13.699118700 +0200
@@ -10,8 +10,7 @@
 	CheckStatGT(Myself,0,WIZARD_IMPROVED_ALACRITY)
 	TargetBlock(PCsPreferringWeak)
 	RequireBlock(IsAzamantes)
-	TriggerBlock(DirectDamageSafe)
-	TriggerBlock(MR100|SIEvocation|MinorGlobe|SpellTurn|Enemy|MagicMissile|L1safe|ResistMagicDamage)
+	TriggerBlock(MR100|MinorGlobe|SpellTurnNew|Enemy|MagicMissile|ResistMagicDamage)
 	!CheckStatGT(scstarget,30,MAGICDAMAGERESISTANCE)
 THEN DO
 	Combine()
@@ -30,8 +29,7 @@
 	Allegiance(Myself,ENEMY)
 	TargetBlock(PCSpellcasters)
 	RequireBlock(IsAzamantes)
-	TriggerBlock(DirectDamageSafe)
-	TriggerBlock(MR100|SIConjuration|MirrorImage|MinorGlobe|ArrowSafe|SpellTurn|Enemy)
+	TriggerBlock(MR100|MirrorImage|MinorGlobe|ArrowSafe|SpellTurnNew|Enemy)
 THEN DO
 	Combine()
 	Action(Literal)
@@ -49,9 +47,7 @@
 	Allegiance(Myself,ENEMY)
 	TargetBlock(PlayersInRandomOrder)
 	RequireBlock(IsAzamantes)
-	TriggerBlock(DirectDamageSafe)
-	TriggerBlock(MR100|ResistFire|SIEvocation|MinorGlobe|Enemy)
-	!StateCheck(scstarget,STATE_MIRRORIMAGE)
+	TriggerBlock(MR100|ResistFire|MinorGlobe|Enemy)
 THEN DO
 	Action(Literal)
 	Spell(scstarget,WIZARD_FIREBALL)
@@ -67,7 +63,7 @@
 	Allegiance(Myself,ENEMY)
 	TargetBlock(PlayersInRandomOrder)
 	RequireBlock(IsAzamantes)
-	TriggerBlock(SpellTurn|SIAbjuration|Enemy|Helpless)
+	TriggerBlock(SpellTurn|SpellTurnNew|Enemy|Helpless)
 	OR(5)
 		CheckStatGT(scstarget,99,WIZARD_PROTECTION_FROM_MAGIC_ENERGY)
 		!CheckStatGT(scstarget,-10,SAVEVSSPELL) // a rough indicator of a potion of Magic Shielding
@@ -91,8 +87,7 @@
 	TargetBlock(PCsInOrder)
 	CheckStatGT(Myself,0,WIZARD_IMPROVED_ALACRITY)
 	RequireBlock(IsAzamantes)
-	OR(3)
-		CheckStatGT(scstarget,0,WIZARD_SPELL_IMMUNITY)
+	OR(2)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_DEFLECTION)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_TURNING)
 	TriggerBlock(Enemy|Helpless)
