--- stratagems/beholder/ssl/behdir01.ssl	2016-11-20 18:27:15.323434500 -0500
+++ Fixpack/stratagems/beholder/ssl/behdir01.ssl	2016-12-01 15:11:46.470877000 -0500
@@ -141,7 +141,7 @@
 IF TRIGGER
 	  Global("BeholderBehavior","LOCALS",0)
           TargetBlock(EnemiesInOrder)
-          TriggerBlock(Enemy|Disabled|DirectDamageSafe)
+          TriggerBlock(Enemy|Disabled)
 THEN DO
           Combine()
           Action(EyestalkRES,dw#bhenv,Enervation|100|100)
@@ -186,7 +186,7 @@
 IF TRIGGER
 	Global("BeholderBehavior","LOCALS",1)
           TargetBlock(PCsPreferringWeak)
-          TriggerBlock(Enemy|Disabled|MinorGlobe|ResistFire|DirectDamageSafe)
+          TriggerBlock(Enemy|Disabled|MinorGlobe|ResistFire)
 THEN DO
           Combine()
           Action(Eyestalk,BEHOLDER_AGANNAZAR_SCORCHER,burning|100|50)
@@ -199,7 +199,7 @@
 IF TRIGGER
 	Global("BeholderBehavior","LOCALS",1)
           TargetBlock(PCsPreferringWeak)
-          TriggerBlock(Enemy|Disabled|MinorGlobe|DirectDamageSafe)
+          TriggerBlock(Enemy|Disabled|MinorGlobe)
 THEN DO
           Combine()
           Action(Eyestalk,BEHOLDER_MAGIC_MISSILE,MM|100|50)
@@ -245,7 +245,7 @@
 
 IF TRIGGER
 	GlobalGT("BeholderBehavior","LOCALS",1)
-	TriggerBlock(MR100|LevelDrain|Enemy|DirectDamageSafe)
+	TriggerBlock(MR100|LevelDrain|Enemy)
 	TargetBlock(PlayersInRandomOrder)
 	TargetBlock(EnemiesInOrder)
 THEN DO
@@ -256,7 +256,7 @@
 
 IF TRIGGER
 	GlobalGT("BeholderBehavior","LOCALS",1)
-	TriggerBlock(MR100|Enemy|DirectDamageSafe)
+	TriggerBlock(MR100|Enemy)
 	TargetBlock(PlayersInRandomOrder)
 	TargetBlock(EnemiesInOrder)
 THEN DO
@@ -267,7 +267,7 @@
 
 IF TRIGGER
 	GlobalGT("BeholderBehavior","LOCALS",1)
-	TriggerBlock(MR100|Enemy|DirectDamageSafe)
+	TriggerBlock(MR100|Enemy)
 	TargetBlock(PlayersInRandomOrder)
 	TargetBlock(EnemiesInOrder)
 THEN DO
