--- stratagems/beholder/ssl/behold01.ssl	2016-11-20 18:27:15.324435000 -0500
+++ Fixpack/stratagems/beholder/ssl/behold01.ssl	2016-09-16 23:30:04.429866400 -0400
@@ -133,7 +133,7 @@
 	RequireBlock(IsHiveMother)
 	!Global("maze","LOCALS",3)
 	TargetBlock(PCsPreferringStrong)
-	TriggerBlock(Enemy|Disabled|Maze|SpellTurn)
+	TriggerBlock(Enemy|Disabled|Maze|SpellTurnNew)
 	Range(scstarget,10)
 THEN DO
 	Action(SpellNoDec,WIZARD_MAZE|100|100)
