--- stratagems/priest/ssl/bg1/combatblocks/bolt_from_sky.ssl	2016-11-20 18:27:15.464517400 -0500
+++ Fixpack/stratagems/priest/ssl/bg1/combatblocks/bolt_from_sky.ssl	2016-12-01 15:11:46.537907900 -0500
@@ -13,24 +13,11 @@
 //////////////////////////////////////////////////////////////
 ///	Call Lightning
 ///////////////////////////////////////////////////////////////
-
 IF TRIGGER
 	IgnoreBlock(IsCleric)
 	RequireBlock(HasL3)
 	TargetBlock(PCsPreferringWeak)
-	TriggerBlock(Disabled|MR|ResistElectric|MinorGlobe|Enemy|SIAlteration)
-	AreaType(OUTDOOR)
-	IgnoreBlock(Demivrgvs)
-THEN DO
-	Combine()
-	Action(Spell,CLERIC_CALL_LIGHTNING|70|30)
-END
-
-IF TRIGGER
-	IgnoreBlock(IsCleric)
-	RequireBlock(HasL3)
-	TargetBlock(PCsPreferringWeak)
-	TriggerBlock(Disabled|MR|ResistElectric|MinorGlobe|Enemy|SIAlteration)
+	TriggerBlock(Disabled|MR|ResistElectric|MinorGlobe|Enemy)
 	RequireBlock(Demivrgvs)
 THEN DO
 	Combine()
@@ -44,7 +31,7 @@
 IF TRIGGER
         IgnoreBlock(IsNymph)
 	TargetBlock(PCsPreferringWeak)
-	TriggerBlock(Disabled|MR|ResistFire|Enemy|SIEvocation)
+	TriggerBlock(Disabled|MR|ResistFire|Enemy)
 	IgnoreBlock(Galactygon_FlameStrike)
 THEN DO
 	Combine()
--- stratagems/priest/ssl/bg2/combatblocks/bolt_from_sky.ssl	2016-11-20 18:27:15.492536900 -0500
+++ Fixpack/stratagems/priest/ssl/bg2/combatblocks/bolt_from_sky.ssl	2016-09-16 14:05:11.796610300 -0400
@@ -30,7 +30,7 @@
 	IgnoreBlock(IsCleric)
 	RequireBlock(HasL3)
 	TargetBlock(PCsPreferringWeak)
-	TriggerBlock(Disabled|MR|ResistElectric|MinorGlobe|Enemy|SIAlteration)
+	TriggerBlock(Disabled|MR|ResistElectric|MinorGlobe|Enemy|SpellTurnNew)
 	RequireBlock(Demivrgvs)
 THEN DO
 	Combine()
@@ -44,7 +44,7 @@
 IF TRIGGER
         IgnoreBlock(IsNymph)
 	TargetBlock(PCsPreferringWeak)
-	TriggerBlock(Disabled|MR|ResistFire|Enemy|SIEvocation)
+	TriggerBlock(Disabled|MR|ResistFire|Enemy|SpellTurnNew)
 	IgnoreBlock(Galactygon_FlameStrike)
 THEN DO
 	Combine()
@@ -73,8 +73,7 @@
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
 	Range(scstarget,15)
-	TriggerBlock(SpellTurn|MR100|Enemy|SINecromancy)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(SpellTurnNew|MR100|Enemy)
 THEN DO
 	Action(Spell,CLERIC_POISON|100|50)
 END
