--- stratagems/tactical_bg1/c2end.tpa	2016-11-20 18:27:15.543573300 -0500
+++ Fixpack/stratagems/tactical_bg1/c2end.tpa	2016-11-20 18:27:39.877144700 -0500
@@ -77,7 +77,4 @@
 END
 >>>>>>>>
      LAF install STR_VAR files=mulreplace.d inline=yes END
-     LAF ssl_to_bcs STR_VAR script=helpmul location=~tactical_bg1/ssl~ rename_to= ~%tutu_var%helpmul~ END
-     LAF extend_area_script STR_VAR area= ~%NashkelMines_MulaheysCave%~ bottom=mularea location=ssl ssl=yes END
-
 END
