--- stratagems/priest/ssl/bg1/combatblocks/chromatic.ssl	2016-11-20 18:27:15.474524500 -0500
+++ Fixpack/stratagems/priest/ssl/bg1/combatblocks/chromatic.ssl	2016-11-20 18:27:39.814100200 -0500
@@ -3,7 +3,7 @@
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC)&HPPercentLT(scstarget,40))
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(MR100|MinorGlobe|SpellTurn|Enemy|MagicMissile)
+	TriggerBlock(MR100|MinorGlobe|SpellTurnNew|Enemy)
 THEN DO
 	Combine()
 	Action(SpellRES,"spdr101"|100|50)
--- stratagems/priest/ssl/bg2/combatblocks/chromatic.ssl	2016-11-20 18:27:15.495539200 -0500
+++ Fixpack/stratagems/priest/ssl/bg2/combatblocks/chromatic.ssl	2016-11-20 18:27:39.831112100 -0500
@@ -3,7 +3,7 @@
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC)&HPPercentLT(scstarget,40))
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(MR100|MinorGlobe|SpellTurn|Enemy|MagicMissile)
+	TriggerBlock(MR100|MinorGlobe|SpellTurnNew|Enemy)
 THEN DO
 	Combine()
 	Action(SpellRES,"spdr101"|100|50)
