--- stratagems/priest/ssl/bg1/combatblocks/command_and_attack.ssl	2018-08-27 01:01:22.000000000 +0200
+++ Fixpack/stratagems/priest/ssl/bg1/combatblocks/command_and_attack.ssl	2018-08-27 18:00:49.000000000 +0200
@@ -7,7 +7,7 @@
 	IgnoreBlock(IsDruid)
 	TargetBlock(PCsInOrderShort)
 	Range(scstarget,6)
-	TriggerBlock(Disabled|Hold|MinorGlobe|MR|Enemy|Sleep|L1safe)
+	TriggerBlock(Disabled|MinorGlobe|MR|Enemy|Sleep)
 THEN DO
 	Action(Spell,CLERIC_COMMAND)
 	SetGlobalTimer("killhelpless","LOCALS",6)
