--- stratagems/mage/spellchoices/demivrgvs/conjurer.2da	2013-11-09 19:41:32.000000000 -0500
+++ Fixpack/stratagems/mage/spellchoices/demivrgvs/conjurer.2da	2016-11-30 20:12:10.156089400 -0500
@@ -5,9 +5,9 @@
 4 MONSTER_SUMMONING_2   MONSTER_SUMMONING_2        SPIDER_SPAWN                 SPIDER_SPAWN            CONFUSION                                       
 5 MONSTER_SUMMONING_3   MONSTER_SUMMONING_3        MONSTER_SUMMONING_3          CHAOS                   CHAOS              CONJURE_LESSER_AIR_ELEMENTAL CONJURE_LESSER_EARTH_ELEMENTAL CONJURE_LESSER_FIRE_ELEMENTAL
 6 CONJURE_AIR_ELEMENTAL CONJURE_EARTH_ELEMENTAL    CONJURE_FIRE_ELEMENTAL       SUMMON_NISHRUU          SUMMON_NISHRUU     WYVERN_CALL          WYVERN_CALL     
-7 PRISMATIC_SPRAY       PRISMATIC_SPRAY            SUMMON_EFREET                SUMMON_EFREET      SUMMON_HAKEASHAR     SUMMON_HAKEASHAR                 
+7 PRISMATIC_SPRAY       SUMMON_DJINNI              CACOFIEND                    SUMMON_EFREET      SUMMON_HAKEASHAR     SUMMON_HAKEASHAR                 
 8 NPC_SYMBOL_STUN       NPC_SYMBOL_FEAR            SUMMON_FIEND                 SUMMON_FIEND            SUMMON_FIEND       NPC_SYMBOL_DEATH                   
-9 GATE                  GATE                                                                                                                
+9 GATE                  GATE                       WAIL_OF_THE_BANSHEE                                                                                         
 H SUMMON_PLANATAR_EVIL  SUMMON_PLANATAR_EVIL       SUMMON_PLANATAR_EVIL         SUMMON_PLANATAR_EVIL    COMET                COMET           COMET
 IO	SUMMON_PLANATAR_EVIL	IMPROVED_ALUCRITY	COMET      	 	DRAGONS_BREATH     ENERGY_BLADES
 IR      SUMMON_PLANATAR_EVIL	SUMMON_PLANATAR_EVIL	SUMMON_PLANATAR_EVIL	IMPROVED_ALUCRITY  COMET      	 	DRAGONS_BREATH     ENERGY_BLADES
