--- stratagems/fiend/ssl/cornugon.ssl	2018-08-30 07:50:32.441431800 +0200
+++ Fixpack/stratagems/fiend/ssl/cornugon.ssl	2018-08-30 07:52:13.675119400 +0200
@@ -49,7 +49,7 @@
 IF TRIGGER
 	TargetBlock(PCsInOrder)
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(Enemy|Disabled|MR100|ResistElectric|SIEvocation)
+	TriggerBlock(Enemy|Disabled|MR100|ResistElectric|SpellTurnNew)
 THEN DO
 	Action(SpellNoDec,WIZARD_LIGHTNING_BOLT)
 END
