--- stratagems/fiend/ssl/cornugon.ssl	2018-08-27 01:01:22.000000000 +0200
+++ Fixpack/stratagems/fiend/ssl/cornugon.ssl	2018-08-27 18:00:49.000000000 +0200
@@ -49,7 +49,7 @@
 IF TRIGGER
 	TargetBlock(PCsInOrder)
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(Enemy|Disabled|MR100|ResistElectric|SIEvocation)
+	TriggerBlock(Enemy|Disabled|MR100|ResistElectric|SpellTurnNew)
 THEN DO
 	Action(SpellNoDec,WIZARD_LIGHTNING_BOLT)
 END
