--- stratagems/fiend/ssl/cornugon.ssl	2018-08-21 11:17:54.154198400 +0200
+++ Fixpack/stratagems/fiend/ssl/cornugon.ssl	2018-08-21 11:36:59.330035800 +0200
@@ -49,7 +49,7 @@
 IF TRIGGER
 	TargetBlock(PCsInOrder)
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(Enemy|Disabled|MR100|ResistElectric|SIEvocation)
+	TriggerBlock(Enemy|Disabled|MR100|ResistElectric|SpellTurnNew)
 THEN DO
 	Action(SpellNoDec,WIZARD_LIGHTNING_BOLT)
 END
