--- stratagems/genie/ssl/dao01.ssl	2016-11-20 18:27:15.390464900 -0500
+++ Fixpack/stratagems/genie/ssl/dao01.ssl	2016-12-01 15:11:46.480867300 -0500
@@ -68,7 +68,7 @@
 	ConditionalTargetBlock(PCsInOrder;CheckStatGT(scstarget,4,SAVEVSSPELL)&!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(PCsInOrder)
 	!CheckStatGT(scstarget,0,SCRIPTINGSTATE2)
-	TriggerBlock(MR100|SpellTurn|Enemy)
+	TriggerBlock(MR100|SpellTurnNew|Enemy)
 THEN DO
 	Action(Spell,WIZARD_FLESH_TO_STONE|100|50)
 END
