--- stratagems/priest/defensive_spells_druid.tph	2016-11-20 18:27:15.457512300 -0500
+++ Fixpack/stratagems/priest/defensive_spells_druid.tph	2016-11-20 18:27:39.795086600 -0500
@@ -16,53 +16,53 @@
 
 //////////////////////////////////// level 1 //////////////////////////////////////////
       1 BEGIN
-         PUSH spells BLESS
+         PUSH spells ARMOR_OF_FAITH
       END
       
 //////////////////////////////////// level 2 //////////////////////////////////////////
       2 BEGIN
-         PUSH spells ~BLESS REMOVE_FEAR~
+         PUSH spells ~ARMOR_OF_FAITH~
       END
 
 //////////////////////////////////// level 3 //////////////////////////////////////////
       3 BEGIN
-         PUSH spells ~BLESS REMOVE_FEAR FLAME_BLADE~
+         PUSH spells ~ARMOR_OF_FAITH FLAME_BLADE~
       END
 //////////////////////////////////// level 4 //////////////////////////////////////////
       4 BEGIN
-         PUSH spells ~BLESS REMOVE_FEAR ARMOR_OF_FAITH FLAME_BLADE~
+         PUSH spells ~ARMOR_OF_FAITH FLAME_BLADE~
          PATCH_MATCH RANDOM(1 2) WITH
            1 BEGIN PUSH spells BARKSKIN END
            DEFAULT END
       END
 //////////////////////////////////// level 5 //////////////////////////////////////////
       5 BEGIN
-         PUSH spells ~BLESS REMOVE_FEAR ARMOR_OF_FAITH FLAME_BLADE BARKSKIN~
+         PUSH spells ~ARMOR_OF_FAITH FLAME_BLADE BARKSKIN~
          PATCH_MATCH RANDOM(1 3) WITH
-           1 BEGIN PUSH spells REMOVE_PARALYSIS END
+           1 BEGIN PUSH spells CURE_DISEASE END
            DEFAULT END
       END
 //////////////////////////////////// level 6 //////////////////////////////////////////
       6 BEGIN
-         PUSH spells ~BLESS REMOVE_FEAR ARMOR_OF_FAITH FLAME_BLADE BARKSKIN~
+         PUSH spells ~ARMOR_OF_FAITH FLAME_BLADE BARKSKIN~
          PATCH_MATCH RANDOM(1 2) WITH
-           1 BEGIN PUSH spells REMOVE_PARALYSIS END
+           1 BEGIN PUSH spells CURE_DISEASE END
            DEFAULT END
 
       END
 //////////////////////////////////// level 7 //////////////////////////////////////////
       7 BEGIN
-         PUSH spells ~BLESS REMOVE_FEAR ARMOR_OF_FAITH BARKSKIN~
+         PUSH spells ~ARMOR_OF_FAITH BARKSKIN~
          PATCH_MATCH RANDOM(1 2) WITH
-           1 BEGIN PUSH spells REMOVE_PARALYSIS END
+           1 BEGIN PUSH spells FREE_ACTION END
            DEFAULT END
          PATCH_MATCH RANDOM(1 2) WITH
-           1 BEGIN PUSH spells DEFENSIVE_HARMONY END
+           1 BEGIN PUSH spells NEUTRALIZE_POISON END
            DEFAULT END
       END
 //////////////////////////////////// level 8 //////////////////////////////////////////
       8 9 BEGIN
-         PUSH spells ~BLESS REMOVE_FEAR ARMOR_OF_FAITH BARKSKIN REMOVE_PARALYSIS DEFENSIVE_HARMONY~
+         PUSH spells ~BLESS ARMOR_OF_FAITH BARKSKIN~
          PATCH_MATCH RANDOM(1 2) WITH
            1 BEGIN PUSH spells FREE_ACTION END
            DEFAULT END
@@ -70,61 +70,61 @@
 
 //////////////////////////////////// level 10 //////////////////////////////////////////
       10 11 BEGIN
-         PUSH spells ~BLESS REMOVE_FEAR ARMOR_OF_FAITH BARKSKIN REMOVE_PARALYSIS DEFENSIVE_HARMONY~
+         PUSH spells ~BLESS ARMOR_OF_FAITH BARKSKIN~
          PATCH_MATCH RANDOM(1 2) WITH
            1 BEGIN PUSH spells FREE_ACTION END
            DEFAULT END
          PATCH_IF fighter=0 BEGIN
            PATCH_MATCH RANDOM(1 2) WITH
-             1 BEGIN PUSH spells CHAOTIC_COMMANDS END
+             1 BEGIN PUSH spells IRONSKIN END
              DEFAULT END
          END
       END
 
 //////////////////////////////////// level 12 //////////////////////////////////////////
       12 BEGIN
-         PUSH spells ~BLESS REMOVE_FEAR ARMOR_OF_FAITH BARKSKIN REMOVE_PARALYSIS ZONE_OF_SWEET_AIR CURE_DISEASE DEFENSIVE_HARMONY FREE_ACTION~
+         PUSH spells ~ARMOR_OF_FAITH BARKSKIN ZONE_OF_SWEET_AIR CURE_DISEASE FREE_ACTION~
          PATCH_IF fighter=0 BEGIN
            PATCH_MATCH RANDOM(1 2) WITH
-             1 BEGIN PUSH spells CHAOTIC_COMMANDS END
+             1 BEGIN PUSH spells IRONSKIN END
              DEFAULT END
          END
       END
 //////////////////////////////////// level 13 //////////////////////////////////////////
       13 BEGIN
-         PUSH spells ~BLESS REMOVE_FEAR ARMOR_OF_FAITH BARKSKIN REMOVE_PARALYSIS ZONE_OF_SWEET_AIR CURE_DISEASE DEFENSIVE_HARMONY FREE_ACTION~
+         PUSH spells ~ARMOR_OF_FAITH BARKSKIN ZONE_OF_SWEET_AIR CURE_DISEASE FREE_ACTION~
          PATCH_IF fighter=0 BEGIN
            PATCH_MATCH RANDOM(1 2) WITH
-             1 BEGIN PUSH spells CHAOTIC_COMMANDS END
+             1 BEGIN PUSH spells IRONSKIN END
              DEFAULT END
          END
          PATCH_MATCH RANDOM(1 2) WITH
-             1 BEGIN PUSH spells PHYSICAL_MIRROR END
+             1 BEGIN PUSH spells SOL_SEARING_ORB END
              DEFAULT END
       END
 
 //////////////////////////////////// level 14 //////////////////////////////////////////
       14 15 BEGIN
-         PUSH spells ~BLESS REMOVE_FEAR ARMOR_OF_FAITH BARKSKIN REMOVE_PARALYSIS ZONE_OF_SWEET_AIR CURE_DISEASE DEFENSIVE_HARMONY FREE_ACTION~
-         PUSH spells ~CHAOTIC_COMMANDS IRONSKIN~
+         PUSH spells ~ARMOR_OF_FAITH BARKSKIN ZONE_OF_SWEET_AIR CURE_DISEASE FREE_ACTION~
+         PUSH spells ~IRONSKIN~
          PATCH_MATCH RANDOM(1 2) WITH
             1 BEGIN PUSH spells MAGIC_RESISTANCE END
             DEFAULT END
          PATCH_MATCH RANDOM(1 2) WITH
-            1 BEGIN PUSH spells PHYSICAL_MIRROR END
+            1 BEGIN PUSH spells SOL_SEARING_ORB END
             DEFAULT END
       END
 
 
 //////////////////////////////////// level 16 //////////////////////////////////////////
       16 17 BEGIN
-         PUSH spells ~BLESS REMOVE_FEAR ARMOR_OF_FAITH BARKSKIN REMOVE_PARALYSIS ZONE_OF_SWEET_AIR CURE_DISEASE DEFENSIVE_HARMONY FREE_ACTION~
-         PUSH spells ~CHAOTIC_COMMANDS IRONSKIN~
+         PUSH spells ~ARMOR_OF_FAITH BARKSKIN ZONE_OF_SWEET_AIR CURE_DISEASE FREE_ACTION~
+         PUSH spells ~IRONSKIN~
          PATCH_MATCH RANDOM(1 2) WITH
            1 BEGIN PUSH spells PROTECTION_FROM_FIRE END
            DEFAULT END
          PATCH_MATCH RANDOM(1 2) WITH
-            1 BEGIN PUSH spells PHYSICAL_MIRROR END
+            1 BEGIN PUSH spells SOL_SEARING_ORB END
             DEFAULT END
          PATCH_MATCH RANDOM(1 2) WITH
             1 BEGIN PUSH spells MAGIC_RESISTANCE END
@@ -134,13 +134,13 @@
 //////////////////////////////////// level 18+ //////////////////////////////////////////
 
       DEFAULT
-         PUSH spells ~BLESS REMOVE_FEAR ARMOR_OF_FAITH BARKSKIN REMOVE_PARALYSIS ZONE_OF_SWEET_AIR CURE_DISEASE DEFENSIVE_HARMONY FREE_ACTION~
-         PUSH spells ~CHAOTIC_COMMANDS IRONSKIN MAGIC_RESISTANCE REGENERATE~
+         PUSH spells ~ARMOR_OF_FAITH BARKSKIN ZONE_OF_SWEET_AIR CURE_DISEASE FREE_ACTION~
+         PUSH spells ~IRONSKIN MAGIC_RESISTANCE REGENERATE~
          PATCH_MATCH RANDOM(1 2) WITH
            1 BEGIN PUSH spells PROTECTION_FROM_FIRE END
            DEFAULT END
          PATCH_MATCH RANDOM(1 2) WITH
-            1 BEGIN PUSH spells PHYSICAL_MIRROR END
+            1 BEGIN PUSH spells SOL_SEARING_ORB END
             DEFAULT END
          PATCH_IF (hla=1 && hla_innate=0) BEGIN
           PATCH_IF level>=30 BEGIN
