--- stratagems/fiend/ssl/demalu01.ssl	2018-08-21 11:17:54.155197900 +0200
+++ Fixpack/stratagems/fiend/ssl/demalu01.ssl	2018-08-21 11:36:59.331035700 +0200
@@ -78,11 +78,10 @@
 
 IF TRIGGER
 	TargetBlock(PCsInOrder)
-	OR(5)
+	OR(4)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_TURNING)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_DEFLECTION)
 		CheckStat(scstarget,1,WIZARD_SPELL_TRAP)
-		CheckStatGT(scstarget,0,WIZARD_SPELL_IMMUNITY)
 		CheckStatGT(scstarget,0,MINORGLOBE)
 	TriggerBlock(Enemy|Helpless)
 	!GlobalGT("secretword","LOCALS",2)
