--- stratagems/fiend/ssl/demmau01.ssl	2018-08-21 11:17:54.157198000 +0200
+++ Fixpack/stratagems/fiend/ssl/demmau01.ssl	2018-08-21 11:36:59.334035700 +0200
@@ -48,7 +48,7 @@
 
 IF TRIGGER
 	TargetBlock(PCsPreferringStrong)
-	TriggerBlock(MR100|SpellTurn|Hold|SIEnchantment|Enemy|Helpless)
+	TriggerBlock(MR100|SpellTurnNew|Hold|SIEnchantment|Enemy|Helpless)
 	Target(NearestEnemyOf(Myself))
 THEN DO
 	Combine()
