--- stratagems/fiend/ssl/demmau01.ssl	2016-11-20 18:27:15.377455900 -0500
+++ Fixpack/stratagems/fiend/ssl/demmau01.ssl	2016-09-16 22:50:52.015316000 -0400
@@ -48,7 +48,7 @@
 
 IF TRIGGER
 	TargetBlock(PCsPreferringStrong)
-	TriggerBlock(MR100|SpellTurn|Hold|SIEnchantment|Enemy|Helpless)
+	TriggerBlock(MR100|SpellTurnNew|Hold|SIEnchantment|Enemy|Helpless)
 	Target(NearestEnemyOf(Myself))
 THEN DO
 	Combine()
