--- stratagems/fiend/ssl/demogor2.ssl	2018-08-21 11:17:54.158197500 +0200
+++ Fixpack/stratagems/fiend/ssl/demogor2.ssl	2018-08-21 11:36:59.335035700 +0200
@@ -302,8 +302,7 @@
 	ConditionalTargetBlock(PCsPreferringWeak;CheckStatGT(scstarget,40,RESISTMAGIC))
 	TargetBlock(PCsPreferringWeak)
 	TargetBlock(Celestials)
-	TriggerBlock(SpellTurn|SIEvocation|Enemy|Helpless)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(SpellTurnNew|Enemy|Helpless)
 THEN DO
 	Action(SpellTimerGaze,CLERIC_IMPLOSION,implosion,180|150|50)
 END
@@ -324,7 +323,7 @@
 
 IF TRIGGER
 	TargetBlock(PCsInOrder)
-	TriggerBlock(SpellTurn|SIAbjuration|Enemy|Helpless)
+	TriggerBlock(SpellTurn|SpellTurnNew|Enemy|Helpless)
 	OR(5)
 		CheckStatGT(scstarget,99,MAGICDAMAGERESISTANCE)
 		!CheckStatGT(scstarget,-10,SAVEVSSPELL) // a rough indicator of a potion of Magic Shielding
@@ -340,11 +339,10 @@
 
 IF TRIGGER
 	TargetBlock(PCsInOrder)
-	OR(5)
+	OR(4)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_TURNING)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_DEFLECTION)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_TRAP)
-		CheckStatGT(scstarget,0,WIZARD_SPELL_IMMUNITY)
 		CheckStatGT(scstarget,0,MINORGLOBE)
 	TriggerBlock(Enemy|Helpless)
 	!GlobalTimerNotExpired("DMWW_antimagic","LOCALS")
@@ -408,7 +406,7 @@
 IF TRIGGER
 	TargetBlock(EnemiesInOrderShort)
 	Range(scstarget,10)
-	TriggerBlock(MR|Enemy|Helpless|SIConjuration)
+	TriggerBlock(MR|Enemy|Helpless)
 	Alignment(scstarget,MASK_GOOD)
 	!Alignment(Myself,MASK_GOOD)
 THEN DO
@@ -422,8 +420,7 @@
 IF TRIGGER
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(MR100|SpellTurn|Slay|SINecromancy|Enemy)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(MR100|SpellTurnNew|Enemy)
 THEN DO
 	Action(SpellTimerGaze,WIZARD_FINGER_OF_DEATH,finger,18|150|50)
 END
