--- stratagems/fiend/ssl/dempitsu.ssl	2018-08-30 07:50:32.444431500 +0200
+++ Fixpack/stratagems/fiend/ssl/dempitsu.ssl	2018-08-30 07:52:13.678119100 +0200
@@ -62,7 +62,7 @@
 IF TRIGGER
 	Target([ANYONE])
 	!GlobalTimerNotExpired("meteorswarm","LOCALS")
-	TriggerBlock(Enemy|Disabled|MR100|SIEvocation|ResistFire)
+	TriggerBlock(Enemy|Disabled|ResistFire)
 THEN DO
 	SetGlobalTimer("meteorswarm","LOCALS",1000)
 	Action(SpellNoDec,WIZARD_METEOR_SWARM|100|100)
@@ -85,7 +85,7 @@
 IF TRIGGER
 	Target([ANYONE])
 	Target(SecondNearest([ANYONE]))
-	TriggerBlock(Enemy|Disabled|MR100|SIEvocation|ResistFire)
+	TriggerBlock(Enemy|Disabled|MR100|ResistFire)
 THEN DO
 	Combine()
 	Action(SpellNoDec,DEATHKNIGHT_FIREBALL|100|100)
@@ -99,7 +99,7 @@
 IF TRIGGER
 	Target([ANYONE])
 	Target(SecondNearest([ANYONE]))
-	TriggerBlock(Enemy|Disabled|MR|Panic|SIConjuration)
+	TriggerBlock(Enemy|Disabled|MR|Panic)
 THEN DO
 	Combine()
 	Action(SpellNoDec,WIZARD_NPC_SYMBOL_FEAR)
