--- stratagems/fiend/ssl/dispel.ssl	2016-11-20 18:27:15.381458400 -0500
+++ Fixpack/stratagems/fiend/ssl/dispel.ssl	2016-11-20 18:27:39.706023600 -0500
@@ -23,7 +23,7 @@
 		!CheckStatGT(Playerscsvar,0,CLERIC_INSECT_PLAGUE)
 		!Allegiance(Playerscsvar,ENEMY)
 	END LOOP
-	TriggerBlock(SIAbjuration|Enemy)
+	TriggerBlock(Enemy)
 THEN DO
 	Combine()
 	SetGlobalTimer("DMWW_dispel","LOCALS",18)
@@ -55,7 +55,7 @@
 		!CheckStatGT(Playerscsvar,0,CLERIC_INSECT_PLAGUE)
 		!Allegiance(Playerscsvar,ENEMY)
 	END LOOP
-	TriggerBlock(SIAbjuration|Enemy)
+	TriggerBlock(Enemy)
 THEN DO
 	Combine()
 	SetGlobalTimer("DMWW_dispel","LOCALS",18)
