--- stratagems/dragon/ssl/dragblue.ssl	2018-08-30 07:50:32.439432100 +0200
+++ Fixpack/stratagems/dragon/ssl/dragblue.ssl	2018-08-30 07:52:13.673119300 +0200
@@ -184,7 +184,7 @@
 		StateCheck(scstarget,STATE_IMPROVEDINVISIBILITY)
 		StateCheck(scstarget,STATE_BLESS)
 		StateCheck(scstarget,STATE_CHANT)
-		StateCheck(scstarget,STATE_HASTED)
+		CheckStat(scstarget,4,WIZARD_SPELL_IMMUNITY)
 		StateCheck(scstarget,STATE_MIRRORIMAGE)
 		CheckStatGT(scstarget,0,CLERIC_DEFENSIVE_HARMONY)
 		CheckStatGT(scstarget,0,PROTECTION_FROM_EVIL)
@@ -247,7 +247,7 @@
 	OR(2)
 		!GlobalTimerNotExpired("DMWWTimestopRunning","GLOBAL")
 		!GlobalTimerNotExpired("timestopmaze","LOCALS")
-	TriggerBlock(SpellTurn|Enemy|Helpless|SIAbjuration)
+	TriggerBlock(SpellTurnNew|Enemy|Helpless)
 	OR(2)
 		!InPartySlot(scstarget,0) // isn't Player 1
 		Global("DMWWImprisonPlayer","GLOBAL",1)
@@ -284,8 +284,7 @@
 IF TRIGGER
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(MR100|SpellTurn|Slay|SINecromancy|Enemy)
-	!HasItem("clck26",scstarget)
+	TriggerBlock(MR100|SpellTurnNew|Enemy)
 THEN DO
 	Action(Spell,WIZARD_FINGER_OF_DEATH|150|50)
 END
@@ -320,12 +319,13 @@
 
 IF TRIGGER
 	TargetBlock(PCsInOrder)
-	TriggerBlock(SpellTurn|Enemy|Helpless)
-	OR(4)
+	TriggerBlock(SpellTurn|SpellTurnNew|Enemy|Helpless)
+	OR(5)
 		CheckStatGT(scstarget,0,WIZARD_PROTECTION_FROM_MAGIC_ENERGY)
 		CheckStatGT(scstarget,50,RESISTELECTRICITY) // we're assuming that anything this high probably comes from a spell
 		CheckStatGT(scstarget,0,WIZARD_PROTECTION_FROM_MAGIC_WEAPONS)
 		CheckStatGT(scstarget,2,STONESKINS)
+		CheckStat(scstarget,4,WIZARD_SPELL_IMMUNITY)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_BREACH|200|100)
@@ -339,7 +339,7 @@
 	TargetBlock(PCsPreferringStrong)
 	TargetBlock(Celestials)
 	!Gender(scstarget,SUMMONED)
-	TriggerBlock(MR|Enemy|SIConjuration|SpellTurn)
+	TriggerBlock(MR|Enemy|SIEnchantment|SpellTurnNew)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_POWER_WORD_BLIND|200|100)
@@ -355,7 +355,7 @@
 	OR(2)
 		!GlobalTimerNotExpired("DMWWTimestopRunning","GLOBAL")
 		!GlobalTimerNotExpired("timestopmaze","LOCALS")
-	TriggerBlock(Maze|SpellTurn|MR|Enemy|Helpless|SIConjuration)
+	TriggerBlock(Maze|SpellTurnNew|MR|Enemy|Helpless)
 	Range(scstarget,6)
 	OR(2)
 		InPartySlot(scstarget,1) // isn't Player 1
@@ -394,11 +394,10 @@
 
 IF TRIGGER
 	TargetBlock(PCsInOrder)
-	OR(6)
+	OR(5)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_TURNING)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_DEFLECTION)
 		CheckStat(scstarget,1,WIZARD_SPELL_TRAP)
-		CheckStatGT(scstarget,0,WIZARD_SPELL_IMMUNITY)
 		CheckStatGT(scstarget,0,MINORGLOBE)
 		CheckStatGT(scstarget,0,CLERIC_SHIELD_OF_THE_ARCHONS)
 	TriggerBlock(Enemy|Helpless)
@@ -415,7 +414,7 @@
 
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
-	TriggerBlock(MR|Confused|SIEnchantment|Enemy|Helpless|ConfuseProof)
+	TriggerBlock(MR|Enemy|Helpless)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_CHAOS|150|50)
@@ -427,7 +426,7 @@
 
 IF TRIGGER
 	TargetBlock(PCsPreferringStrong)
-	TriggerBlock(MR100|Slow|SIAlteration|Enemy|MinorGlobe|Helpless)
+	TriggerBlock(MR100|Slow|Enemy|MinorGlobe|Helpless)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_SLOW|150|50)
@@ -441,7 +440,7 @@
 
 IF TRIGGER
 	TargetBlock(PCSpellcasters)
-	TriggerBlock(MR100|SIEvocation|Enemy|MinorGlobe|SpellTurn)
+	TriggerBlock(MR100|Enemy|MinorGlobe|SpellTurnNew)
 	StateCheck(scstarget,STATE_MIRRORIMAGE)
 THEN DO
 	Combine()
@@ -456,7 +455,7 @@
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC)&HPPercentLT(scstarget,40))
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(MR100|SIEvocation|MinorGlobe|SpellTurn|Enemy)
+	TriggerBlock(MR100|MinorGlobe|SpellTurnNew|Enemy)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_MAGIC_MISSILE)
