--- stratagems/dragon/ssl/draggree.ssl	2018-08-27 01:01:22.000000000 +0200
+++ Fixpack/stratagems/dragon/ssl/draggree.ssl	2018-08-27 18:00:49.000000000 +0200
@@ -162,12 +162,13 @@
 
 IF TRIGGER
 	TargetBlock(PCsInOrder)
-	TriggerBlock(SpellTurn|Enemy|Helpless)
+	TriggerBlock(SpellTurn|SpellTurnNew|Enemy|Helpless)
 	!GlobalTimerNotExpired("breach","LOCALS")
-	OR(3)
+	OR(4)
 		CheckStatGT(scstarget,0,WIZARD_PROTECTION_FROM_MAGIC_ENERGY)
 		CheckStatGT(scstarget,0,WIZARD_PROTECTION_FROM_MAGIC_WEAPONS)
 		CheckStatGT(scstarget,2,STONESKINS)
+		CheckStat(scstarget,4,WIZARD_SPELL_IMMUNITY)
 THEN DO
 	Combine()
 	SetGlobalTimer("breach","LOCALS",12)
@@ -182,8 +183,7 @@
 IF TRIGGER
 	ConditionalTargetBlock(PCsPreferringStrong;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	!Alignment(scstarget,MASK_EVIL)
-	TriggerBlock(MR100|SINecromancy|ResistMagicDamage|Enemy)
+	TriggerBlock(MR100|ResistMagicDamage|Enemy)
 THEN DO
 	Action(Sequencer,CLERIC_UNHOLY_BLIGHT,CLERIC_UNHOLY_BLIGHT,CLERIC_UNHOLY_BLIGHT)
 END
@@ -230,9 +230,7 @@
 IF TRIGGER
 	ConditionalTargetBlock(PCsPreferringStrong;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	!Alignment(scstarget,MASK_EVIL)
-	TriggerBlock(DirectDamageSafe)
-	TriggerBlock(MR100|SINecromancy|ResistMagicDamage|Enemy|MirrorImage)
+	TriggerBlock(MR100|ResistMagicDamage|Enemy)
 THEN DO
 	Action(Spell,CLERIC_UNHOLY_BLIGHT|200|100)
 END
@@ -244,11 +242,10 @@
 
 IF TRIGGER
 	TargetBlock(PCsInOrder)
-	OR(6)
+	OR(5)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_TURNING)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_DEFLECTION)
 		CheckStat(scstarget,1,WIZARD_SPELL_TRAP)
-		CheckStatGT(scstarget,0,WIZARD_SPELL_IMMUNITY)
 		CheckStatGT(scstarget,0,MINORGLOBE)
 		CheckStatGT(scstarget,0,CLERIC_SHIELD_OF_THE_ARCHONS)
 	TriggerBlock(Enemy|Helpless)
@@ -269,7 +266,7 @@
 IF TRIGGER
 	ConditionalTargetBlock(PCSpellcasters;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(PCSpellcasters)
-	TriggerBlock(MR100|SpellTurn|SIConjuration)
+	TriggerBlock(MR100|SpellTurnNew|SIEnchantment)
 	TriggerBlock(Enemy|Helpless|Silence)
 THEN DO
 	Action(Spell,WIZARD_POWER_WORD_SILENCE|150|50)
@@ -283,7 +280,7 @@
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(PCsInOrderShort)
 	TargetBlock(Celestials)
-	TriggerBlock(MR100|SpellTurn|Stun|Enemy|Helpless|SIConjuration)
+	TriggerBlock(MR100|SpellTurnNew|Stun|Enemy|Helpless|SIEnchantment)
 	OR(3)
 		HPPercentLT(scstarget,60)
 		Class(scstarget,MAGE)
@@ -298,7 +295,7 @@
 
 IF TRIGGER
 	TargetBlock(PCsPreferringStrong)
-	TriggerBlock(MR|Charm|SIEnchantment|SpellTurn|Enemy|Helpless)
+	TriggerBlock(MR|Charm|SIEnchantment|SpellTurnNew|Enemy|Helpless)
 	!CheckStatLT(scstarget,0,SAVEVSSPELL)
 THEN DO
 	Action(Spell,WIZARD_DOMINATION|150|50)
@@ -324,7 +321,7 @@
 
 IF TRIGGER
 	TargetBlock(PCSpellcasters)
-	TriggerBlock(MR100|SIEvocation|Enemy|MinorGlobe|SpellTurn)
+	TriggerBlock(MR100|Enemy|MinorGlobe|SpellTurnNew)
 	StateCheck(scstarget,STATE_MIRRORIMAGE)
 THEN DO
 	Combine()
@@ -338,7 +335,7 @@
 IF TRIGGER
 	ConditionalTargetBlock(PCSpellcasters;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(PCSpellcasters)
-	TriggerBlock(MR100|SIConjuration|Stoneskin|MirrorImage|MinorGlobe|ArrowSafe|SpellTurn|Enemy)
+	TriggerBlock(MR100|MinorGlobe|ArrowSafe|SpellTurnNew|Enemy)
 THEN DO
 	Combine()
 	Action(MinorSequencer,WIZARD_MAGIC_MISSILE,WIZARD_MAGIC_MISSILE)
@@ -353,7 +350,7 @@
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC)&HPPercentLT(scstarget,40))
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(MR100|SIEvocation|MinorGlobe|SpellTurn|Enemy)
+	TriggerBlock(MR100|MinorGlobe|SpellTurnNew|Enemy)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_MAGIC_MISSILE)
@@ -362,7 +359,7 @@
 IF TRIGGER
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(MR100|SIConjuration|Stoneskin|MirrorImage|MinorGlobe|ArrowSafe|SpellTurn)
+	TriggerBlock(MR100|MirrorImage|MinorGlobe|ArrowSafe|SpellTurnNew)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_MELF_ACID_ARROW)
