--- stratagems/dragon/ssl/dragsilv.ssl	2016-11-20 18:27:15.369450300 -0500
+++ Fixpack/stratagems/dragon/ssl/dragsilv.ssl	2016-09-17 00:26:11.686462500 -0400
@@ -285,7 +285,7 @@
 
 IF TRIGGER
 	TargetBlock(PCsInOrder)
-	TriggerBlock(SpellTurn|Enemy|Helpless)
+	TriggerBlock(SpellTurn|SpellTurnNew|Enemy|Helpless)
 	OR(3)
 		CheckStatGT(scstarget,50,RESISTCOLD) // we're assuming that anything this high probably comes from a spell
 		CheckStatGT(scstarget,0,WIZARD_PROTECTION_FROM_MAGIC_WEAPONS)
@@ -301,8 +301,7 @@
 IF TRIGGER
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(MR100|SpellTurn|Slay|SINecromancy|Enemy)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(MR100|SpellTurnNew|Enemy)
 THEN DO
 	Action(Spell,WIZARD_FINGER_OF_DEATH|150|50)
 END
@@ -314,8 +313,7 @@
 
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
-	TriggerBlock(Enemy|MR|ResistElectric|SIEvocation)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(Enemy|MR|ResistElectric)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_CHAIN_LIGHTNING|150|50)
@@ -355,7 +353,7 @@
 
 IF TRIGGER
 	TargetBlock(PCsPreferringStrong)
-	TriggerBlock(MR100|Slow|SIAlteration|Enemy|MinorGlobe|Helpless)
+	TriggerBlock(MR100|Slow|Enemy|MinorGlobe|Helpless)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_SLOW|150|50)
@@ -368,7 +366,7 @@
 IF TRIGGER
 	ConditionalTargetBlock(PCsPreferringWeak;HPPercentLT(scstarget,40))
 	TargetBlock(PCsPreferringWeak)
-	TriggerBlock(MR|ResistFire|Enemy|ArrowSafe|SpellTurn|Stoneskin|MinorGlobe|MirrorImage|Helpless|SIConjuration)
+	TriggerBlock(MR|ResistFire|Enemy|ArrowSafe|SpellTurnNew|MinorGlobe|MirrorImage|Helpless)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_FLAME_ARROW|150|100)
@@ -395,7 +393,6 @@
 IF TRIGGER
 	TargetBlock(PCFighters)
 	TriggerBlock(Enemy|Helpless|MR|MinorGlobe)
-	CheckStatGT(scstarget,9,STR)
 	CheckStatGT(scstarget,0,SAVEVSSPELL)
 THEN DO
 	Combine()
@@ -410,8 +407,7 @@
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC)&HPPercentLT(scstarget,40))
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(MR100|SIEvocation|MinorGlobe|SpellTurn|Enemy)
-	!HasItem("clck26",Myself)
+	TriggerBlock(MR100|MinorGlobe|SpellTurnNew|Enemy)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_CHROMATIC_ORB)
