--- stratagems/dragon/ssl/dragshad.ssl	2018-08-30 07:50:32.440431600 +0200
+++ Fixpack/stratagems/dragon/ssl/dragshad.ssl	2018-08-30 07:52:13.674119200 +0200
@@ -164,7 +164,7 @@
 
 IF TRIGGER
 	TargetBlock(PCsInOrder)
-	TriggerBlock(SpellTurn|Enemy|Helpless)
+	TriggerBlock(SpellTurn|SpellTurnNew|Enemy|Helpless)
 	OR(3)
 		CheckStatGT(scstarget,0,SCRIPTINGSTATE1)
 		CheckStatGT(scstarget,0,WIZARD_PROTECTION_FROM_MAGIC_WEAPONS)
@@ -190,7 +190,7 @@
 
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
-	TriggerBlock(MR|Confused|SIEnchantment|Enemy|Helpless)
+	TriggerBlock(MR|Enemy|Helpless)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_CHAOS|150|50)
@@ -203,7 +203,7 @@
 
 IF TRIGGER
 	TargetBlock(PCsPreferringStrong)
-	TriggerBlock(MR100|Slow|SIAlteration|Enemy|MinorGlobe|Helpless)
+	TriggerBlock(MR100|Slow|Enemy|MinorGlobe|Helpless)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_SLOW|150|50)
@@ -218,7 +218,7 @@
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC)&HPPercentLT(scstarget,40))
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(MR100|SIEvocation|MinorGlobe|SpellTurn|Enemy)
+	TriggerBlock(MR100|MinorGlobe|SpellTurnNew|Enemy)
 THEN DO
 	Action(Spell,WIZARD_CHROMATIC_ORB)
 END
