--- stratagems/abazigal/ssl/dw#abred.ssl	2016-11-20 18:27:15.318431900 -0500
+++ Fixpack/stratagems/abazigal/ssl/dw#abred.ssl	2016-11-20 18:27:39.657990400 -0500
@@ -191,7 +191,7 @@
 
 IF TRIGGER
 	TargetBlock(PCsInOrder)
-	TriggerBlock(SpellTurn|Enemy|Helpless)
+	TriggerBlock(SpellTurn|SpellTurnNew|Enemy|Helpless)
 	OR(4)
 		CheckStatGT(scstarget,0,WIZARD_PROTECTION_FROM_MAGIC_ENERGY)
 		CheckStatGT(scstarget,50,RESISTFIRE) // we're assuming that anything this high probably comes from a spell
@@ -210,7 +210,7 @@
 	TargetBlock(PCsPreferringStrong)
 	TargetBlock(Celestials)
 	!Gender(scstarget,SUMMONED)
-	TriggerBlock(MR|Enemy|SIConjuration)
+	TriggerBlock(MR|Enemy|SIEnchantment|SpellTurnNew)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_POWER_WORD_BLIND|200|100)
@@ -259,7 +259,7 @@
 IF TRIGGER
 	ConditionalTargetBlock(PCsPreferringWeak;HPPercentLT(scstarget,40))
 	TargetBlock(PCsPreferringWeak)
-	TriggerBlock(MR|ResistFire|Enemy|ArrowSafe|SpellTurn|Stoneskin|MinorGlobe|MirrorImage|Helpless|SIConjuration)
+	TriggerBlock(MR|ResistFire|Enemy|ArrowSafe|SpellTurnNew|MinorGlobe|MirrorImage|Helpless)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_FLAME_ARROW|150|50)
@@ -271,7 +271,7 @@
 
 IF TRIGGER
 	TargetBlock(PCsPreferringStrong)
-	TriggerBlock(MR100|Slow|SIAlteration|Enemy|MinorGlobe|Helpless)
+	TriggerBlock(MR100|Slow|Enemy|MinorGlobe|Helpless)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_SLOW|150|50)
@@ -283,11 +283,10 @@
 
 IF TRIGGER
 	TargetBlock(PCsInOrder)
-	OR(5)
+	OR(4)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_TURNING)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_DEFLECTION)
 		CheckStat(scstarget,1,WIZARD_SPELL_TRAP)
-		CheckStatGT(scstarget,0,WIZARD_SPELL_IMMUNITY)
 		CheckStatGT(scstarget,0,MINORGLOBE)
 	TriggerBlock(Enemy|Helpless)
 THEN DO
@@ -302,7 +301,7 @@
 
 IF TRIGGER
 	TargetBlock(PCSpellcasters)
-	TriggerBlock(MR100|SIEvocation|Enemy)
+	TriggerBlock(MR100|MinorGlobe|Enemy)
 	StateCheck(scstarget,STATE_MIRRORIMAGE)
 THEN DO
 	Combine()
@@ -316,7 +315,7 @@
 IF TRIGGER
 	ConditionalTargetBlock(PCSpellcasters;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(PCSpellcasters)
-	TriggerBlock(MR100|SIConjuration|Stoneskin|MirrorImage|MinorGlobe|ArrowSafe|SpellTurn|Enemy)
+	TriggerBlock(MR100|MirrorImage|MinorGlobe|ArrowSafe|SpellTurnNew|Enemy)
 THEN DO
 	Combine()
 	Action(MinorSequencer,WIZARD_MELF_ACID_ARROW,WIZARD_MELF_ACID_ARROW)
@@ -331,7 +330,7 @@
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC)&HPPercentLT(scstarget,40))
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(MR100|SIEvocation|MinorGlobe|SpellTurn|Enemy)
+	TriggerBlock(MR100|MinorGlobe|SpellTurnNew|Enemy)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_MAGIC_MISSILE)
@@ -340,7 +339,7 @@
 IF TRIGGER
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(MR100|SIConjuration|Stoneskin|MirrorImage|MinorGlobe|ArrowSafe|SpellTurn)
+	TriggerBlock(MR100|MirrorImage|MinorGlobe|ArrowSafe|SpellTurnNew)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_MELF_ACID_ARROW)
