--- stratagems/abazigal/ssl/dw#abdgy.ssl	2016-11-20 18:27:15.316429300 -0500
+++ Fixpack/stratagems/abazigal/ssl/dw#abdgy.ssl	2016-09-16 23:05:09.334351900 -0400
@@ -306,13 +306,14 @@
 
 IF TRIGGER
 	TargetBlock(PCsInOrder)
-	TriggerBlock(SpellTurn|Enemy|Helpless)
-	OR(5)
+	TriggerBlock(SpellTurn|SpellTurnNew|Enemy|Helpless)
+	OR(6)
 		CheckStatGT(scstarget,0,WIZARD_PROTECTION_FROM_MAGIC_ENERGY)
 		CheckStatGT(scstarget,0,SCRIPTINGSTATE2)
 		CheckStatGT(scstarget,0,CLERIC_CHAOTIC_COMMANDS)
 		CheckStatGT(scstarget,0,WIZARD_PROTECTION_FROM_MAGIC_WEAPONS)
 		CheckStatGT(scstarget,2,STONESKINS)
+		CheckStat(scstarget,4,WIZARD_SPELL_IMMUNITY)
 	!GlobalTimerNotExpired("DMWW_breach","LOCALS")
 THEN DO
 	Combine()
@@ -327,7 +328,7 @@
 IF TRIGGER
 	!GlobalTimerNotExpired("DMWW_dispel","LOCALS")
 	TargetBlock(PCsInOrder)
-	OR(14)
+	OR(15)
 		!CheckStatGT(scstarget,-10,SAVEVSSPELL) // a rough indicator of a potion of Magic Shielding
 		CheckStatGT(scstarget,0,STONESKINS)
 		CheckStatGT(scstarget,0,WIZARD_PROTECTION_FROM_MAGIC_WEAPONS)
@@ -336,6 +337,7 @@
 		StateCheck(scstarget,STATE_CHANT)
 		StateCheck(scstarget,STATE_HASTED)
 		StateCheck(scstarget,STATE_MIRRORIMAGE)
+		CheckStat(scstarget,4,WIZARD_SPELL_IMMUNITY)
 		CheckStatGT(scstarget,0,CLERIC_DEFENSIVE_HARMONY)
 		CheckStatGT(scstarget,0,PROTECTION_FROM_EVIL)
 		CheckStatGT(scstarget,0,CLERIC_PHYSICAL_MIRROR)
@@ -347,7 +349,7 @@
 		!CheckStatGT(Playerscsvar,0,WIZARD_GREATER_MALISON)
 		!Allegiance(Playerscsvar,ENEMY)
 	END LOOP
-	TriggerBlock(SIAbjuration|Enemy)
+	TriggerBlock(Enemy)
 THEN DO
 	Combine()
 	SetGlobalTimer("DMWW_dispel","LOCALS",18)
@@ -373,7 +375,7 @@
 THEN DO
 	Combine()
 	Action(SpellArea,WIZARD_LOWER_RESISTANCE|150|50)
-	Action(SpellArea,WIZARD_PIERCE_SHIELD|150|50)
+	Action(SpellArea,WIZARD_PIERCE_MAGIC|150|50)
 END
 
 ////////////////////////////////////////////////////////////////////////////
@@ -396,11 +398,10 @@
 
 IF TRIGGER
 	TargetBlock(PCsInOrder)
-	OR(5)
+	OR(4)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_TURNING)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_DEFLECTION)
 		CheckStat(scstarget,1,WIZARD_SPELL_TRAP)
-		CheckStatGT(scstarget,0,WIZARD_SPELL_IMMUNITY)
 		CheckStatGT(scstarget,0,MINORGLOBE)
 	TriggerBlock(Enemy|Helpless)
 THEN DO
@@ -436,8 +437,7 @@
 	OR(2)
 		!Gender(scstarget,SUMMONED)
 		HasItem("morsword",scstarget)
-	TriggerBlock(ResistMagicDamage|MR|SINecromancy|Enemy|MirrorImage)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(ResistMagicDamage|MR|Enemy)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_ABI_DALZIMS_HORRID_WILTING|200|100)
@@ -449,8 +449,7 @@
 
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
-	TriggerBlock(MR100|ResistFire|SIEvocation|Enemy)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(MR100|ResistFire|Enemy)
 THEN DO
 	Action(SpellArea,WIZARD_INCENDIARY_CLOUD|200|50)
 END
@@ -492,8 +491,7 @@
 IF TRIGGER
 	ConditionalTargetBlock(PCsPreferringWeak;HPPercentLT(scstarget,40))
 	TargetBlock(PCsPreferringWeak)
-	TriggerBlock(DirectDamageSafe)
-	TriggerBlock(MR|ResistFire|Enemy|ArrowSafe|SpellTurn|Stoneskin|MinorGlobe|MirrorImage|Helpless|SIConjuration)
+	TriggerBlock(MR|ResistFire|Enemy|ArrowSafe|SpellTurnNew|MinorGlobe|MirrorImage|Helpless)
 THEN DO
 	Action(Spell,WIZARD_FLAME_ARROW|150|50)
 END
@@ -505,7 +503,7 @@
 
 IF TRIGGER
 	TargetBlock(PCSpellcasters)
-	TriggerBlock(MR100|SIEvocation|Enemy)
+	TriggerBlock(MR100|Enemy)
 	StateCheck(scstarget,STATE_MIRRORIMAGE)
 THEN DO
 	Combine()
@@ -520,7 +518,7 @@
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC)&HPPercentLT(scstarget,40))
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(MR100|SIEvocation|MinorGlobe|SpellTurn|Enemy)
+	TriggerBlock(MR100|MinorGlobe|SpellTurnNew|Enemy)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_MAGIC_MISSILE)
