--- stratagems/fiend/ssl/dw#bonef.ssl	2018-08-27 01:01:22.000000000 +0200
+++ Fixpack/stratagems/fiend/ssl/dw#bonef.ssl	2018-08-27 18:00:49.000000000 +0200
@@ -62,7 +62,7 @@
 IF TRIGGER
 	TargetBlock(PCsInOrder)
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(Enemy|Disabled|MR100|ResistCold|SIEvocation)
+	TriggerBlock(Enemy|Disabled|MR100|ResistCold)
 THEN DO
 	Action(SpellNoDec,WIZARD_CONE_OF_COLD)
 END
