--- stratagems/fiend/ssl/dw#bonef.ssl	2018-08-30 07:50:32.445431500 +0200
+++ Fixpack/stratagems/fiend/ssl/dw#bonef.ssl	2018-08-30 07:52:13.679119400 +0200
@@ -62,7 +62,7 @@
 IF TRIGGER
 	TargetBlock(PCsInOrder)
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(Enemy|Disabled|MR100|ResistCold|SIEvocation)
+	TriggerBlock(Enemy|Disabled|MR100|ResistCold)
 THEN DO
 	Action(SpellNoDec,WIZARD_CONE_OF_COLD)
 END
