--- stratagems/fiend/ssl/dw#bonef.ssl	2018-08-21 11:17:54.160197400 +0200
+++ Fixpack/stratagems/fiend/ssl/dw#bonef.ssl	2018-08-21 11:36:59.337035700 +0200
@@ -62,7 +62,7 @@
 IF TRIGGER
 	TargetBlock(PCsInOrder)
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(Enemy|Disabled|MR100|ResistCold|SIEvocation)
+	TriggerBlock(Enemy|Disabled|MR100|ResistCold)
 THEN DO
 	Action(SpellNoDec,WIZARD_CONE_OF_COLD)
 END
