--- stratagems/celestial/ssl/dw#dvdev.ssl	2016-11-20 18:27:15.334424800 -0500
+++ Fixpack/stratagems/celestial/ssl/dw#dvdev.ssl	2016-09-17 00:09:37.437594700 -0400
@@ -46,7 +46,7 @@
 IF TRIGGER
 	TargetBlock(EnemiesInOrderShort)
 	Range(scstarget,10)
-	TriggerBlock(MR|Enemy|Helpless|SIConjuration)
+	TriggerBlock(MR|Enemy|Helpless)
 	Alignment(scstarget,MASK_GOOD)
 THEN DO
 	Action(SpellMyself,CLERIC_UNHOLY_WORD|200|50)
@@ -79,7 +79,7 @@
 
       IF TRIGGER
          Target(Playerscsplayer)
-         TriggerBlock(Enemy|Disabled|SpellTurn|MinorGlobe)
+         TriggerBlock(Enemy|Disabled|SpellTurnNew|MinorGlobe)
          !GlobalTimerNotExpired("doomedscsplayer","LOCALS")
          !GlobalTimerNotExpired("doomed","LOCALS")
       THEN DO
@@ -95,7 +95,7 @@
 
 IF TRIGGER
 	TargetBlock(PCSpellcasters)
-	TriggerBlock(MR100|SpellTurn|SIConjuration|Silence) // alas, can't detect the Vocalise spell
+	TriggerBlock(MR100|Silence) // alas, can't detect the Vocalise spell
 	TriggerBlock(Enemy|Helpless)
 	!StateCheck(scstarget,STATE_SILENCED)
 
@@ -119,7 +119,7 @@
 	TargetBlock(PCsInOrderShort)
 	Range(scstarget,10)
 	!CheckStat(scstarget,4,WIZARD_PROTECTION_FROM_MAGIC_WEAPONS)
-	TriggerBlock(SpellTurn|MRTouch|Enemy|SINecromancy|ResistMagicDamage)
+	TriggerBlock(Enemy|ResistMagicDamage)
 	Global("shapeshift","LOCALS",0)
 THEN DO
 	Action(SpellMyself,CLERIC_HARM|100|50)
@@ -138,7 +138,7 @@
 
 IF TRIGGER
 	TargetBlock(PCsPreferringWeak)
-	TriggerBlock(Disabled|MR|ResistFire|Enemy|SIEvocation)
+	TriggerBlock(Disabled|MR|ResistFire|Enemy|SpellTurnNew)
 THEN DO
 	Action(Spell,CLERIC_FLAME_STRIKE|80|20)
 END
@@ -149,8 +149,7 @@
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
 	Range(scstarget,15)
-	TriggerBlock(SpellTurn|MR100|Enemy|SINecromancy)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(SpellTurnNew|MR100|Enemy)
 THEN DO
 	Action(Spell,CLERIC_POISON|100|50)
 	Action(Spell,CLERIC_RIGID_THINKING|100|50) // actually Contagion
@@ -162,9 +161,7 @@
 	ConditionalTargetBlock(PCsPreferringStrong;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
 	!Alignment(Myself,MASK_GOOD)
-	Alignment(scstarget,MASK_GOOD)
-	TriggerBlock(DirectDamageSafe)
-	TriggerBlock(MR100|SINecromancy|ResistMagicDamage|Enemy|MirrorImage)
+	TriggerBlock(MR100|ResistMagicDamage|Enemy)
 THEN DO
 	Action(Spell,CLERIC_UNHOLY_BLIGHT|200|100)
 END
