--- stratagems/fiend/ssl/dw#demiv.ssl	2018-08-21 11:17:54.160197400 +0200
+++ Fixpack/stratagems/fiend/ssl/dw#demiv.ssl	2018-08-21 11:36:59.337035700 +0200
@@ -18,7 +18,7 @@
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
 	!Gender(scstarget,SUMMONED)
-	TriggerBlock(Panic|MR100|SIConjuration|Enemy|Helpless)
+	TriggerBlock(Panic|MR100|Enemy|Helpless)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_NPC_SYMBOL_FEAR|100|200)
@@ -31,7 +31,7 @@
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
 	!Gender(scstarget,SUMMONED)
-	TriggerBlock(Slay|MR|SIConjuration)
+	TriggerBlock(Slay|MR)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_NPC_SYMBOL_DEATH|100|200)
@@ -44,8 +44,7 @@
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
 	!GlobalTimerNotExpired("fireball","LOCALS")
-	TriggerBlock(MR100|ResistFire|SIEvocation|MinorGlobe|Enemy)
-	!HasItem("clck26",scstarget)
+	TriggerBlock(MR100|ResistFire|MinorGlobe|Enemy)
 THEN DO
 	SetGlobalTimer("fireball","LOCALS",1000)
 	Action(SpellNoDec,DEATHKNIGHT_FIREBALL|100|100)
@@ -62,8 +61,7 @@
 		HPPercentLT(scstarget,50)
 		Class(scstarget,MAGE)
 		Class(scstarget,MAGE_THIEF)
-	TriggerBlock(MR100|SpellTurn|Slay|SIConjuration|Enemy)
-	!HasItem("clck26",scstarget)
+	TriggerBlock(MR100|SpellTurn|Slay|SpellTurnNew|Enemy|SIEnchantment)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_POWER_WORD_KILL|100|100)
@@ -77,7 +75,7 @@
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
 	!Gender(scstarget,SUMMONED)
-	TriggerBlock(MR|SIConjuration|SpellTurn|Helpless)
+	TriggerBlock(MR|SIEnchantment|SpellTurnNew|Helpless)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_POWER_WORD_BLIND|100|100)
@@ -90,7 +88,7 @@
 IF TRIGGER
 	ConditionalTargetBlock(EnemiesInOrder;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(MR100|SpellTurn|Stun|Helpless|SIConjuration)
+	TriggerBlock(MR100|SpellTurnNew|Stun|Helpless|SIEnchantment)
 	OR(3)
 		HPPercentLT(scstarget,60)
 		Class(scstarget,MAGE)
