--- stratagems/celestial/ssl/dw#plane.ssl	2018-08-27 01:01:22.000000000 +0200
+++ Fixpack/stratagems/celestial/ssl/dw#plane.ssl	2018-08-27 18:00:49.000000000 +0200
@@ -73,7 +73,7 @@
 IF TRIGGER
 	TargetBlock(PCSpellcasters)
 	!CheckStatGT(scstarget,0,CLERIC_INSECT_PLAGUE)
-	TriggerBlock(MR100|SIConjuration|Enemy)
+	TriggerBlock(Enemy)
 THEN DO
 	Action(Spell,CLERIC_INSECT_PLAGUE|100|50)
 END
@@ -81,7 +81,7 @@
 IF TRIGGER
 	TargetBlock(PCsInOrder)
 	Range(scstarget,10)
-	TriggerBlock(MR100|SIConjuration|Enemy)
+	TriggerBlock(MR100|Enemy)
 THEN DO
 	Combine()
 	Action(Spell,CLERIC_UNHOLY_WORD|100|50)
@@ -114,7 +114,7 @@
 
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
-	TriggerBlock(Panic|MR|SIConjuration|Enemy)
+	TriggerBlock(Panic|MR|Enemy)
 THEN DO
 	Combine()
 	Action(Spell,CLERIC_SYMBOL_FEAR|100|50)
@@ -126,7 +126,7 @@
 
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
-	TriggerBlock(Slay|MR|SIConjuration|Enemy)
+	TriggerBlock(Slay|MR|Enemy)
 THEN DO
 	Combine()
 	Action(SpellRES,"sppr719"|100|50)
@@ -138,7 +138,7 @@
 
 IF TRIGGER
 	TargetBlock(PCSpellcasters)
-	TriggerBlock(MR100|SIEvocation|Enemy)
+	TriggerBlock(MR100|Enemy)
 	StateCheck(scstarget,STATE_MIRRORIMAGE)
 THEN DO
 	Action(Spell,WIZARD_MAGIC_MISSILE)
@@ -153,8 +153,7 @@
 	ConditionalTargetBlock(PCsPreferringWeak;HPPercentLT(scstarget,40)&!CheckStatGT(scstarget,50,RESISTMAGIC)&!CheckStatGT(scstarget,50,RESISTFIRE))
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC)&!CheckStatGT(scstarget,50,RESISTFIRE))
 	TargetBlock(EnemiesInOrderShort)
-	!HasItem("clck26",scstarget)
-	TriggerBlock(MR100|Enemy|ArrowSafe|SpellTurn|Stoneskin|MinorGlobe|MirrorImage|Helpless|SIConjuration)
+	TriggerBlock(MR100|Enemy|ArrowSafe|SpellTurnNew|MinorGlobe|MirrorImage|Helpless)
 THEN DO
 	Action(Spell,WIZARD_FLAME_ARROW|100|50)
 END
@@ -168,7 +167,7 @@
 	TriggerBlock(MR|Confused|SIEnchantment|Enemy|Helpless)
 THEN DO
 	Combine()
-	Action(Spell,WIZARD_CHAOS|100|50)
+	Action(Spell,WIZARD_CONFUSION|100|50)
 END
 
 /////////////////////////////////////////////////////////////////////////
@@ -178,8 +177,7 @@
 IF TRIGGER
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC)&!CheckStatGT(scstarget,50,RESISTFIRE))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(MR100|SIEvocation|Enemy|MirrorImage)
-	!HasItem("clck26",scstarget)
+	TriggerBlock(MR100|Enemy|SpellTurnNew)
 THEN DO
 	Action(Spell,CLERIC_FLAME_STRIKE|100|100)
 END
@@ -192,8 +190,7 @@
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC)&HPPercentLT(scstarget,40))
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(MR100|SIEvocation|MinorGlobe|SpellTurn|Enemy|MagicMissile|L1safe)
-	!HasItem("clck26",scstarget)
+	TriggerBlock(MR100|MinorGlobe|SpellTurnNew|Enemy|MagicMissile)
 THEN DO
 	Action(Spell,WIZARD_MAGIC_MISSILE|150|50)
 END
