--- stratagems/fiend/ssl/dw#pitfi.ssl	2016-11-20 18:27:15.384460300 -0500
+++ Fixpack/stratagems/fiend/ssl/dw#pitfi.ssl	2016-11-20 18:27:39.718032100 -0500
@@ -76,7 +76,7 @@
 	IgnoreBlock(IsSummoned)
 	TargetBlock(PCsInOrder)
 	!GlobalTimerNotExpired("meteorswarm","LOCALS")
-	TriggerBlock(Enemy|Disabled|MR100|SIEvocation|ResistFire)
+	TriggerBlock(Enemy|Disabled|ResistFire)
 THEN DO
 	Combine()
 	SetGlobalTimer("meteorswarm","LOCALS",1000)
@@ -101,7 +101,7 @@
 IF TRIGGER
 	IgnoreBlock(IsSummoned)
 	TargetBlock(PCsInOrder)
-	TriggerBlock(Enemy|Disabled|MR100|SIEvocation|ResistFire)
+	TriggerBlock(Enemy|Disabled|MR100|ResistFire|MinorGlobe)
 THEN DO
 	Combine()
 	Action(SpellNoDec,DEATHKNIGHT_FIREBALL|200|100)
@@ -114,7 +114,7 @@
 
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
-	TriggerBlock(MR|Panic|SIConjuration)
+	TriggerBlock(MR|Panic)
 THEN DO
 	Combine()
 	Action(SpellNoDec,WIZARD_NPC_SYMBOL_FEAR)
