--- stratagems/genie/ssl/efreet01.ssl	2016-11-20 18:27:15.393466500 -0500
+++ Fixpack/stratagems/genie/ssl/efreet01.ssl	2016-11-20 18:27:39.730041900 -0500
@@ -121,8 +121,7 @@
 
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
-	TriggerBlock(MR100|SIEvocation|MinorGlobe|Enemy)
-	!HasItem("clck26",scstarget)
+	TriggerBlock(MR100|MinorGlobe|Enemy)
 	!NumCreatureGT([ENEMY.HUMANOID],0)
 	Allegiance(Myself,ENEMY)
 THEN DO
@@ -137,8 +136,7 @@
 IF TRIGGER
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC)&!CheckStatGT(scstarget,50,RESISTFIRE)&Allegiance(Myself,ENEMY))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(MR100|SIEvocation|MirrorImage)
-	!HasItem("clck26",scstarget)
+	TriggerBlock(MR100|SpellTurnNew)
 THEN DO
 	Action(Spell,CLERIC_FLAME_STRIKE|100|100)
 END
@@ -149,7 +147,7 @@
 
 IF TRIGGER
 	TargetBlock(PCSpellcasters)
-	TriggerBlock(MR100|SIEvocation|Enemy)
+	TriggerBlock(MR100|MinorGlobe|SpellTurnNew|Enemy)
 	StateCheck(scstarget,STATE_MIRRORIMAGE)
 THEN DO
 	Action(Spell,WIZARD_MAGIC_MISSILE)
@@ -164,8 +162,7 @@
 	ConditionalTargetBlock(PCsPreferringWeak;HPPercentLT(scstarget,40)&!CheckStatGT(scstarget,50,RESISTMAGIC)&!CheckStatGT(scstarget,50,RESISTFIRE)&Allegiance(Myself,ENEMY))
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC)&!CheckStatGT(scstarget,50,RESISTFIRE)&Allegiance(Myself,ENEMY))
 	TargetBlock(EnemiesInOrderShort)
-	!HasItem("clck26",scstarget)
-	TriggerBlock(MR100|ArrowSafe|SpellTurn|Stoneskin|MinorGlobe|MirrorImage|Helpless|SIConjuration)
+	TriggerBlock(MR100|ArrowSafe|SpellTurnNew|MinorGlobe|Helpless)
 THEN DO
 	Action(Spell,WIZARD_FLAME_ARROW|100|50)
 END
@@ -178,8 +175,7 @@
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC)&HPPercentLT(scstarget,40)&Allegiance(Myself,ENEMY))
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC)&Allegiance(Myself,ENEMY))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(MR100|SIEvocation|MinorGlobe|SpellTurn|MagicMissile)
-	!HasItem("clck26",scstarget)
+	TriggerBlock(MR100|MinorGlobe|SpellTurnNew|MagicMissile)
 THEN DO
 	Action(Spell,WIZARD_MAGIC_MISSILE|150|50)
 END
