--- stratagems/firegiant/ssl/elemimix.ssl	2018-08-30 07:50:32.447431400 +0200
+++ Fixpack/stratagems/firegiant/ssl/elemimix.ssl	2018-08-30 07:52:13.681119200 +0200
@@ -162,8 +162,7 @@
 IF TRIGGER
 	!StateCheck(Myself,STATE_DISABLED)
 	TargetBlock(PCsPreferringWeak)
-	TriggerBlock(Disabled|MR|ResistFire|Enemy|SIEvocation)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(Disabled|MR|ResistFire|Enemy|SpellTurnNew)
 THEN DO
 	Action(ForceSpellRES,"spimix01"|80|20)
 	DisplayStringHead(Myself,72787) // Feel the wrath of Imix!
