--- stratagems/mage/ssl/bg1/combatblocks/enfeeblement.ssl	2018-08-30 07:50:32.459431600 +0200
+++ Fixpack/stratagems/mage/ssl/bg1/combatblocks/enfeeblement.ssl	2018-08-30 07:52:13.694118900 +0200
@@ -1,21 +1,19 @@
-IF TRIGGER
-	IgnoreBlock(IsInvoker)
-	RequireBlock(HasL2)
-	TargetBlock(PCFighters)
-	TriggerBlock(Enemy|Helpless|MR|MinorGlobe)
-	CheckStatGT(scstarget,9,STR)
-	CheckStatGT(scstarget,0,SAVEVSSPELL)
-THEN DO
-	Combine()
-	Action(Spell,WIZARD_RAY_OF_ENFEEBLEMENT|150|50)
-END
-
-IF TRIGGER
-	IgnoreBlock(IsNecromancer)
-	TargetBlock(PCFighters)
-	TriggerBlock(Enemy|Helpless|MR|MinorGlobe)
-	CheckStatGT(scstarget,0,SAVEVSSPELL)
-THEN DO
-	Combine()
-	Action(Spell,WIZARD_BLINDNESS|150|50)
-END
+IF TRIGGER
+	RequireBlock(HasL2)
+	TargetBlock(PCFighters)
+	TriggerBlock(Enemy|Helpless|MR|MinorGlobe)
+	CheckStatGT(scstarget,9,STR)
+	CheckStatGT(scstarget,0,SAVEVSSPELL)
+THEN DO
+	Combine()
+	Action(Spell,WIZARD_RAY_OF_ENFEEBLEMENT|150|50)
+END
+
+IF TRIGGER
+	TargetBlock(PCFighters)
+	TriggerBlock(Enemy|Helpless|MR|MinorGlobe)
+	CheckStatGT(scstarget,0,SAVEVSSPELL)
+THEN DO
+	Combine()
+	Action(Spell,WIZARD_BLINDNESS|150|50)
+END
