--- stratagems/mage/ssl/bg1/combatblocks/enfeeblement.ssl	2016-11-20 18:27:15.403474000 -0500
+++ Fixpack/stratagems/mage/ssl/bg1/combatblocks/enfeeblement.ssl	2016-11-20 18:27:39.740046900 -0500
@@ -1,5 +1,4 @@
 IF TRIGGER
-	IgnoreBlock(IsInvoker)
 	RequireBlock(HasL2)
 	TargetBlock(PCFighters)
 	TriggerBlock(Enemy|Helpless|MR|MinorGlobe)
@@ -11,7 +10,6 @@
 END
 
 IF TRIGGER
-	IgnoreBlock(IsNecromancer)
 	TargetBlock(PCFighters)
 	TriggerBlock(Enemy|Helpless|MR|MinorGlobe)
 	CheckStatGT(scstarget,0,SAVEVSSPELL)
