--- stratagems/ascension/ssl/finaluf.ssl	2018-08-21 11:17:54.138198600 +0200
+++ Fixpack/stratagems/ascension/ssl/finaluf.ssl	2018-08-21 11:36:59.311036400 +0200
@@ -110,11 +110,10 @@
 
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
-	OR(5)
+	OR(4)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_TURNING)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_DEFLECTION)
 		CheckStat(scstarget,1,WIZARD_SPELL_TRAP)
-		CheckStatGT(scstarget,0,WIZARD_SPELL_IMMUNITY)
 		CheckStatGT(scstarget,0,MINORGLOBE)
 	TriggerBlock(Helpless)
 	!GlobalGT("secretword","LOCALS",2)
