--- stratagems/ascension/ssl/finbalor.ssl	2018-08-30 07:50:32.432432100 +0200
+++ Fixpack/stratagems/ascension/ssl/finbalor.ssl	2018-08-30 07:52:13.666119600 +0200
@@ -133,13 +133,14 @@
 
 IF TRIGGER
 	TargetBlock(PCsInOrder)
-	TriggerBlock(SpellTurn|Enemy|Helpless)
-	OR(5)
+	TriggerBlock(SpellTurn|SpellTurnNew|Enemy|Helpless)
+	OR(6)
 		CheckStatGT(scstarget,0,WIZARD_PROTECTION_FROM_MAGIC_ENERGY)
 		CheckStatGT(scstarget,0,SCRIPTINGSTATE2)
 		CheckStatGT(scstarget,0,CLERIC_CHAOTIC_COMMANDS)
 		CheckStatGT(scstarget,0,WIZARD_PROTECTION_FROM_MAGIC_WEAPONS)
 		CheckStatGT(scstarget,2,STONESKINS)
+		CheckStat(scstarget,4,WIZARD_SPELL_IMMUNITY)
 	!GlobalTimerNotExpired("DMWW_breach","LOCALS")
 THEN DO
 	Combine()
@@ -157,7 +158,7 @@
 	!GlobalTimerNotExpired("implosion","LOCALS")
 	!Gender(scstarget,SUMMONED)
 	!Gender(scstarget,20)
-	TriggerBlock(SpellTurn|Helpless)
+	TriggerBlock(SpellTurnNew|Helpless)
 THEN DO
 	SetGlobalTimer("implosion","LOCALS",1000)
 	Action(SpellNoDec,CLERIC_IMPLOSION|100|100)
@@ -193,7 +194,7 @@
 	TargetBlock(EnemiesInOrder)
 	!Gender(scstarget,SUMMONED)
 	!Gender(scstarget,20)
-	TriggerBlock(Stun|MR|SIConjuration|Enemy|Helpless)
+	TriggerBlock(Stun|MR|Enemy|Helpless)
 THEN DO
 	Combine()
 	Action(SpellNoDec,WIZARD_NPC_SYMBOL_STUN|100|100)
@@ -207,7 +208,7 @@
 	TargetBlock(EnemiesInOrder)
 	!Gender(scstarget,20)
 	!Gender(scstarget,SUMMONED)
-	TriggerBlock(Slay|MR|Enemy|SIConjuration)
+	TriggerBlock(Slay|MR|Enemy)
 THEN DO
 	Combine()
 	Action(SpellNoDec,WIZARD_NPC_SYMBOL_DEATH|100|100)
@@ -219,7 +220,7 @@
 
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
-	TriggerBlock(MR|Charm|SIEnchantment|SpellTurn|Helpless)
+	TriggerBlock(MR|Charm|SIEnchantment|SpellTurnNew|Helpless)
 THEN DO
 	Combine()
 	Action(SpellNoDec,WIZARD_DOMINATION)
