--- stratagems/ascension/ssl/finglab.ssl	2018-08-27 01:01:22.000000000 +0200
+++ Fixpack/stratagems/ascension/ssl/finglab.ssl	2018-08-27 18:00:49.000000000 +0200
@@ -82,7 +82,7 @@
 
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
-	TriggerBlock(MR|SpellTurn|Stun|Helpless|SIConjuration)
+	TriggerBlock(MR|SpellTurnNew|Stun|Helpless|SIEnchantment)
 	OR(3)
 		HPPercentLT(scstarget,60)
 		Class(scstarget,MAGE)
@@ -97,8 +97,7 @@
 
 IF TRIGGER
 	TargetBlock(PCsPreferringWeak)
-	TriggerBlock(MR100|Enemy|Helpless|SINecromancy)
-	Alignment(scstarget,MASK_GOOD)	
+	TriggerBlock(MR100|Enemy|Helpless)	
 THEN DO
 	Action(SpellNoDec,CLERIC_UNHOLY_BLIGHT|200|100)
 END
