--- stratagems/ascension/ssl/finiren.ssl	2016-11-20 18:27:15.321434300 -0500
+++ Fixpack/stratagems/ascension/ssl/finiren.ssl	2016-11-20 18:27:39.661992800 -0500
@@ -79,7 +79,7 @@
 	RESPONSE #100
 			SetGlobal("ChainContingencyFired","LOCALS",1)
 			DisplayStringHead(Myself,26328)
-			ReallyForceSpell(Myself,WIZARD_MISLEAD)
+			ReallyForceSpellRES("%WIZARD_MISLEAD_INSTANT%",Myself)
 			ReallyForceSpell(Myself,WIZARD_SIMULACRUM)
 			ReallyForceSpell(Myself,WIZARD_SPELL_IMMUNITY_DIVINATION)
 			SetGlobalTimer("redefend","LOCALS",3)
