--- stratagems/melissan/ssl/finmel.ssl	2018-08-21 11:17:54.207195800 +0200
+++ Fixpack/stratagems/melissan/ssl/finmel.ssl	2018-08-21 11:36:59.390033900 +0200
@@ -377,9 +377,8 @@
 	Target("finbalth")
 	Target("finbodh")
 	!GlobalTimerNotExpired("DMWWTimestopRunning","GLOBAL")
-	TriggerBlock(SIEvocation|SlashingSafe)
+	TriggerBlock(SlashingSafe)
 	!CheckStatGT(scstarget,0,STONESKINS)
-	TriggerBlock(DirectDamageSafe)
 THEN DO
 	Action(ForceSpellRESTimer,"melis01",boneblades,12|100|150)
 END
@@ -401,8 +400,7 @@
 		HPPercentLT(scstarget,50)
 		Class(scstarget,MAGE)
 		Class(scstarget,MAGE_THIEF)
-	TriggerBlock(MR100|SpellTurn|Slay|SIConjuration|Enemy)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(MR100|SpellTurnNew|Slay|SIEnchantment|Enemy)
 THEN DO
 	Combine()
 	Action(SpellTimer,WIZARD_POWER_WORD_KILL,powerword,120|150|150)
@@ -449,12 +447,13 @@
 
 IF TRIGGER
 	TargetBlock(PCsInOrder)
-	TriggerBlock(SpellTurn|SIAbjuration|Enemy|Helpless)
-	OR(4)
+	TriggerBlock(SpellTurn|Enemy|Helpless)
+	OR(5)
 		CheckStatGT(scstarget,99,MAGICDAMAGERESISTANCE)
 		!CheckStatGT(scstarget,-10,SAVEVSSPELL) // a rough indicator of a potion of Magic Shielding
 		CheckStatGT(scstarget,0,WIZARD_PROTECTION_FROM_MAGIC_WEAPONS)
 		CheckStatGT(scstarget,2,STONESKINS)
+		CheckStat(scstarget,4,WIZARD_SPELL_IMMUNITY)
 	!GlobalTimerNotExpired("DMWWTimestopRunning","GLOBAL")
 	!GlobalTimerNotExpired("timestop","LOCALS") // embarrassing to dispel your own attacks
 THEN DO
@@ -537,7 +536,6 @@
 	Target([PC])
 	Target("finbalth")
 	Target("finbodh")
-	TriggerBlock(DirectDamageSafe)
 	GlobalGT("FiveDead","AR6200",3)
 	!GlobalTimerNotExpired("bigspell","LOCALS")
 THEN DO
@@ -555,7 +553,6 @@
 	Target("finbodh")
 	TriggerBlock(ResistFire|Enemy)
 	GlobalGT("FiveDead","AR6200",3)
-	!StateCheck(scstarget,STATE_MIRRORIMAGE)
 THEN DO
 	Combine()
 	SetGlobalTimer("bigspell","LOCALS",12)
@@ -574,8 +571,8 @@
 
 IF TRIGGER
 	TargetBlock(PCsPreferringStrong)
-	TriggerBlock(MR|Slow|SIAlteration|Enemy|MinorGlobe|Helpless)
-	!CheckStatLT(scstarget,0,SAVEVSSPELL)
+	TriggerBlock(MR|Slow|SpellTurnNew|Enemy|Helpless)
+	!CheckStatLT(scstarget,0,SAVEVSPOLY)
 	!StateCheck(scstarget,STATE_SLOWED)
 THEN DO
 	Combine()
@@ -588,11 +585,10 @@
 
 IF TRIGGER
 	TargetBlock(PCsInOrder)
-	OR(5)
+	OR(4)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_TURNING)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_DEFLECTION)
 		CheckStat(scstarget,1,WIZARD_SPELL_TRAP)
-		CheckStatGT(scstarget,0,WIZARD_SPELL_IMMUNITY)
 		CheckStatGT(scstarget,0,MINORGLOBE)
 	TriggerBlock(Enemy|Helpless)
 	!GlobalTimerNotExpired("DMWW_antimagic","LOCALS")
@@ -622,7 +618,7 @@
 	Combine()
 	SetGlobalTimer("DMWW_antimagic","LOCALS",18)
 	SetGlobalTimer("timestop_antimagic","LOCALS",24)
-	Action(SpellTimer,WIZARD_PIERCE_SHIELD,pierceshield,18|100|200)
+	Action(SpellTimer,WIZARD_PIERCE_MAGIC,pierceshield,18|100|200)
 END
 
 
@@ -634,7 +630,7 @@
 IF TRIGGER
 	TargetBlock(EnemiesInOrderShort)
 	Range(scstarget,10)
-	TriggerBlock(MR|Enemy|Helpless|SIConjuration)
+	TriggerBlock(MR|Enemy|Helpless)
 	Alignment(scstarget,MASK_GOOD)
 	OR(5)
 		Class(scstarget,BARD)
@@ -656,8 +652,7 @@
 IF TRIGGER
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(MR100|SpellTurn|Slay|SINecromancy|Enemy)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(MR100|SpellTurnNew|Slay|Enemy)
 THEN DO
 	Action(SpellTimer,WIZARD_FINGER_OF_DEATH,finger,18|150|100)
 END
