--- stratagems/mage/ssl/bg2/combatblocks/first_dispel.ssl	2016-11-20 18:27:15.419484900 -0500
+++ Fixpack/stratagems/mage/ssl/bg2/combatblocks/first_dispel.ssl	2016-11-20 18:27:39.760061500 -0500
@@ -28,7 +28,7 @@
 		!CheckStatGT(Playerscsvar,0,WIZARD_GREATER_MALISON)
 		!Allegiance(Playerscsvar,ENEMY)
 	END LOOP
-	TriggerBlock(SIAbjuration|Enemy)
+	TriggerBlock(Enemy)
 	CheckStatGT(Myself,14,Level)
 	OR(2)
 		CheckStatGT(Myself,19,Level)
