--- stratagems/mage/ssl/bg1/combatblocks/flamearrows.ssl	2018-08-27 01:01:22.000000000 +0200
+++ Fixpack/stratagems/mage/ssl/bg1/combatblocks/flamearrows.ssl	2018-08-27 18:00:49.000000000 +0200
@@ -1,21 +1,21 @@
-
-IF TRIGGER
-        RequireBlock(HasL3)
-        IgnoreBlock(IsEvoker)
-	ConditionalTargetBlock(PCsPreferringWeak;HPPercentLT(scstarget,40))
-	TargetBlock(PCsPreferringWeak)
-	TriggerBlock(MR|ResistFire|Enemy|ArrowSafe|SpellTurn|Stoneskin|MinorGlobe|MirrorImage|Helpless|SIConjuration)
-THEN DO
-	Action(Spell,WIZARD_FLAME_ARROW|150|50)
-END
-
-IF TRIGGER
-        RequireBlock(HasL3)
-        IgnoreBlock(IsEnchanter)
-        RequireBlock(Demivrgvs)
-	ConditionalTargetBlock(PCsPreferringWeak;HPPercentLT(scstarget,40))
-	TargetBlock(PCsPreferringWeak)
-	TriggerBlock(MR|ResistElectric|Enemy|SpellTurn|MinorGlobe|MirrorImage|Helpless|SIEvocation)
-THEN DO
-	Action(Spell,WIZARD_LIGHTNING_BOLT|150|50)
-END
+
+IF TRIGGER
+        RequireBlock(HasL3)
+        IgnoreBlock(IsEvoker)
+	ConditionalTargetBlock(PCsPreferringWeak;HPPercentLT(scstarget,40))
+	TargetBlock(PCsPreferringWeak)
+	TriggerBlock(MR|ResistFire|Enemy|ArrowSafe|SpellTurnNew|MinorGlobe|MirrorImage|Helpless)
+THEN DO
+	Action(Spell,WIZARD_FLAME_ARROW|150|50)
+END
+
+IF TRIGGER
+        RequireBlock(HasL3)
+        IgnoreBlock(IsEnchanter)
+        RequireBlock(Demivrgvs)
+	ConditionalTargetBlock(PCsPreferringWeak;HPPercentLT(scstarget,40))
+	TargetBlock(PCsPreferringWeak)
+	TriggerBlock(MR|ResistElectric|Enemy|SpellTurnNew|MinorGlobe|MirrorImage|Helpless)
+THEN DO
+	Action(Spell,WIZARD_LIGHTNING_BOLT|150|50)
+END
