--- stratagems/mage/ssl/bg2/prepblocks/fm_chaincont_core.ssl	2016-11-20 18:27:15.442501800 -0500
+++ Fixpack/stratagems/mage/ssl/bg2/prepblocks/fm_chaincont_core.ssl	2016-11-20 18:27:39.787080100 -0500
@@ -31,7 +31,7 @@
 			DisplayString(Myself,26328)
 			ReallyForceSpell(Myself,protectionspell)
 			ReallyForceSpell(Myself,WIZARD_ABI_DALZIMS_HORRID_WILTING)
-			ReallyForceSpell(Myself,WIZARD_SPELL_IMMUNITY_CONJURATION)
+			ReallyForceSpell(Myself,WIZARD_SPELL_IMMUNITY_ABJURATION)
 			Continue()
 		RESPONSE #50
 			RequireBlock(WithSI)
