--- stratagems/fiend/ssl/gorchr.ssl	2016-11-20 18:27:15.386461700 -0500
+++ Fixpack/stratagems/fiend/ssl/gorchr.ssl	2016-09-16 23:01:15.850997400 -0400
@@ -86,8 +86,7 @@
 IF TRIGGER
 	Global("shape","LOCALS",1)
 	TargetBlock(EnemiesInOrder)
-	TriggerBlock(MR100|ResistFire|SIEvocation|MinorGlobe|Enemy)
-	!HasItem("clck26",scstarget)
+	TriggerBlock(MR100|ResistFire|MinorGlobe|Enemy)
 THEN DO
 	Action(SpellNoDec,DEATHKNIGHT_FIREBALL)
 END
@@ -110,7 +109,6 @@
 	TargetBlock(EnemiesInOrder)
 	Range(scstarget,5)
 	TriggerBlock(MR100|ResistAcid|Enemy)
-	!HasItem("clck26",scstarget)
 THEN DO
 	Action(SpellNoDecRES,"dw#chrsl")
 END
@@ -119,7 +117,7 @@
 	Global("shape","LOCALS",3)
 	TargetBlock(PCsInOrder)
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(Enemy|Disabled|MR100|ResistCold|SIEvocation)
+	TriggerBlock(Enemy|Disabled|MR100|ResistCold)
 THEN DO
 	Action(SpellNoDecRES,"dw#chrco")
 END
