--- stratagems/dragon/ssl/gorsal.ssl	2016-11-20 18:27:15.370450800 -0500
+++ Fixpack/stratagems/dragon/ssl/gorsal.ssl	2016-11-20 18:27:39.696017400 -0500
@@ -196,7 +196,7 @@
 
 IF TRIGGER
 	TargetBlock(PCsInOrder)
-	TriggerBlock(SpellTurn|Enemy|Helpless)
+	TriggerBlock(SpellTurn|SpellTurnNew|Enemy|Helpless)
 	OR(4)
 		CheckStatGT(scstarget,0,WIZARD_PROTECTION_FROM_MAGIC_ENERGY)
 		CheckStatGT(scstarget,50,RESISTFIRE) // we're assuming that anything this high probably comes from a spell
@@ -215,7 +215,7 @@
 	TargetBlock(PCsPreferringStrong)
 	TargetBlock(Celestials)
 	!Gender(scstarget,SUMMONED)
-	TriggerBlock(MR|Enemy|SIConjuration)
+	TriggerBlock(MR|Enemy|SIEnchantment)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_POWER_WORD_BLIND|200|100)
@@ -264,7 +264,7 @@
 IF TRIGGER
 	ConditionalTargetBlock(PCsPreferringWeak;HPPercentLT(scstarget,40))
 	TargetBlock(PCsPreferringWeak)
-	TriggerBlock(MR|ResistFire|Enemy|ArrowSafe|SpellTurn|Stoneskin|MinorGlobe|MirrorImage|Helpless|SIConjuration)
+	TriggerBlock(MR|ResistFire|Enemy|ArrowSafe|SpellTurnNew|MinorGlobe|MirrorImage|Helpless)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_FLAME_ARROW|150|50)
@@ -276,7 +276,7 @@
 
 IF TRIGGER
 	TargetBlock(PCsPreferringStrong)
-	TriggerBlock(MR100|Slow|SIAlteration|Enemy|MinorGlobe|Helpless)
+	TriggerBlock(MR100|Slow|Enemy|MinorGlobe|Helpless)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_SLOW|150|50)
@@ -288,11 +288,10 @@
 
 IF TRIGGER
 	TargetBlock(PCsInOrder)
-	OR(6)
+	OR(5)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_TURNING)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_DEFLECTION)
 		CheckStat(scstarget,1,WIZARD_SPELL_TRAP)
-		CheckStatGT(scstarget,0,WIZARD_SPELL_IMMUNITY)
 		CheckStatGT(scstarget,0,MINORGLOBE)
 		CheckStatGT(scstarget,0,CLERIC_SHIELD_OF_THE_ARCHONS)
 	TriggerBlock(Enemy|Helpless)
@@ -308,7 +307,7 @@
 
 IF TRIGGER
 	TargetBlock(PCSpellcasters)
-	TriggerBlock(L1safe|MR100|SIEvocation|Enemy|MinorGlobe|SpellTurn)
+	TriggerBlock(MR100|Enemy|MinorGlobe|SpellTurnNew)
 	StateCheck(scstarget,STATE_MIRRORIMAGE)
 THEN DO
 	Combine()
@@ -322,7 +321,7 @@
 IF TRIGGER
 	ConditionalTargetBlock(PCSpellcasters;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(PCSpellcasters)
-	TriggerBlock(L2safe|MR100|SIConjuration|Stoneskin|MirrorImage|MinorGlobe|ArrowSafe|SpellTurn|Enemy)
+	TriggerBlock(MR100|MirrorImage|MinorGlobe|ArrowSafe|SpellTurnNew|Enemy)
 THEN DO
 	Combine()
 	Action(MinorSequencer,WIZARD_MAGIC_MISSILE,WIZARD_MAGIC_MISSILE)
@@ -337,7 +336,7 @@
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC)&HPPercentLT(scstarget,40))
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(MR100|SIEvocation|MinorGlobe|SpellTurn|Enemy)
+	TriggerBlock(MR100|MinorGlobe|SpellTurnNew|Enemy)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_MAGIC_MISSILE)
@@ -346,7 +345,7 @@
 IF TRIGGER
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(MR100|SIConjuration|Stoneskin|MirrorImage|MinorGlobe|ArrowSafe|SpellTurn)
+	TriggerBlock(MR100|MirrorImage|MinorGlobe|ArrowSafe|SpellTurnNew)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_MELF_ACID_ARROW)
