--- stratagems/priest/ssl/bg1/combatblocks/group_buffing.ssl	2018-08-30 07:50:32.476430800 +0200
+++ Fixpack/stratagems/priest/ssl/bg1/combatblocks/group_buffing.ssl	2018-08-30 07:52:13.713118500 +0200
@@ -1,34 +1,36 @@
-IF TRIGGER
-	RequireBlock(HasL4)
-	TargetBlock(VillainsInOrder)
-	GlobalTimerExpired("redefend","LOCALS")
-	NumCreatureGT([ENEMY],3)
-	!CheckStatGT(scstarget,0,CLERIC_DEFENSIVE_HARMONY)
-	Range(scstarget,10)
-THEN DO
-	Action(SpellMyself,CLERIC_DEFENSIVE_HARMONY|200|100)
-END
-
-IF TRIGGER
-	RequireBlock(HasL2)
-	IgnoreBlock(IsDruid)
-	TargetBlock(VillainsInOrder)
-	GlobalTimerExpired("redefend","LOCALS")
-	NumCreatureGT([ENEMY],3)
-	!StateCheck(scstarget,STATE_CHANT)
-	Range(scstarget,20)
-THEN DO
-	Action(SpellMyself,CLERIC_CHANT|100|100)
-END
-
-
-IF TRIGGER
-	TargetBlock(VillainsInOrder)
-	GlobalTimerExpired("redefend","LOCALS")
-	NumCreatureGT([ENEMY],3)
-	!StateCheck(scstarget,STATE_BLESS)
-	Range(scstarget,20)
-THEN DO
-	Action(SpellMyself,CLERIC_BLESS|100|100)
-END
-
+IF TRIGGER
+	RequireBlock(HasL4)
+	IgnoreBlock(IsDruid)
+	TargetBlock(VillainsInOrder)
+	GlobalTimerExpired("redefend","LOCALS")
+	NumCreatureGT([ENEMY],3)
+	!CheckStatGT(scstarget,0,CLERIC_DEFENSIVE_HARMONY)
+	Range(scstarget,10)
+THEN DO
+	Action(SpellMyself,CLERIC_DEFENSIVE_HARMONY|200|100)
+END
+
+IF TRIGGER
+	RequireBlock(HasL2)
+	IgnoreBlock(IsDruid)
+	TargetBlock(VillainsInOrder)
+	GlobalTimerExpired("redefend","LOCALS")
+	NumCreatureGT([ENEMY],3)
+	!StateCheck(scstarget,STATE_CHANT)
+	Range(scstarget,20)
+THEN DO
+	Action(SpellMyself,CLERIC_CHANT|100|100)
+END
+
+
+IF TRIGGER
+    IgnoreBlock(IsDruid)
+	TargetBlock(VillainsInOrder)
+	GlobalTimerExpired("redefend","LOCALS")
+	NumCreatureGT([ENEMY],3)
+	!StateCheck(scstarget,STATE_BLESS)
+	Range(scstarget,20)
+THEN DO
+	Action(SpellMyself,CLERIC_BLESS|100|100)
+END
+
--- stratagems/priest/ssl/bg2/combatblocks/group_buffing.ssl	2018-08-30 07:50:32.483431400 +0200
+++ Fixpack/stratagems/priest/ssl/bg2/combatblocks/group_buffing.ssl	2018-08-30 07:52:13.719118500 +0200
@@ -1,34 +1,36 @@
-IF TRIGGER
-	RequireBlock(HasL4)
-	TargetBlock(VillainsInOrder)
-	GlobalTimerExpired("redefend","LOCALS")
-	NumCreatureGT([ENEMY],3)
-	!CheckStatGT(scstarget,0,CLERIC_DEFENSIVE_HARMONY)
-	Range(scstarget,10)
-THEN DO
-	Action(SpellMyself,CLERIC_DEFENSIVE_HARMONY|200|100)
-END
-
-IF TRIGGER
-	RequireBlock(HasL2)
-	IgnoreBlock(IsDruid)
-	TargetBlock(VillainsInOrder)
-	GlobalTimerExpired("redefend","LOCALS")
-	NumCreatureGT([ENEMY],3)
-	!StateCheck(scstarget,STATE_CHANT)
-	Range(scstarget,20)
-THEN DO
-	Action(SpellMyself,CLERIC_CHANT|100|100)
-END
-
-
-IF TRIGGER
-	TargetBlock(VillainsInOrder)
-	GlobalTimerExpired("redefend","LOCALS")
-	NumCreatureGT([ENEMY],3)
-	!StateCheck(scstarget,STATE_BLESS)
-	Range(scstarget,20)
-THEN DO
-	Action(SpellMyself,CLERIC_BLESS|100|100)
-END
-
+IF TRIGGER
+	RequireBlock(HasL4)
+	IgnoreBlock(IsDruid)
+	TargetBlock(VillainsInOrder)
+	GlobalTimerExpired("redefend","LOCALS")
+	NumCreatureGT([ENEMY],3)
+	!CheckStatGT(scstarget,0,CLERIC_DEFENSIVE_HARMONY)
+	Range(scstarget,10)
+THEN DO
+	Action(SpellMyself,CLERIC_DEFENSIVE_HARMONY|200|100)
+END
+
+IF TRIGGER
+	RequireBlock(HasL2)
+	IgnoreBlock(IsDruid)
+	TargetBlock(VillainsInOrder)
+	GlobalTimerExpired("redefend","LOCALS")
+	NumCreatureGT([ENEMY],3)
+	!StateCheck(scstarget,STATE_CHANT)
+	Range(scstarget,20)
+THEN DO
+	Action(SpellMyself,CLERIC_CHANT|100|100)
+END
+
+
+IF TRIGGER
+    IgnoreBlock(IsDruid)
+	TargetBlock(VillainsInOrder)
+	GlobalTimerExpired("redefend","LOCALS")
+	NumCreatureGT([ENEMY],3)
+	!StateCheck(scstarget,STATE_BLESS)
+	Range(scstarget,20)
+THEN DO
+	Action(SpellMyself,CLERIC_BLESS|100|100)
+END
+
