--- stratagems/priest/ssl/bg2/combatblocks/harmony.ssl	2018-08-30 07:50:32.483431400 +0200
+++ Fixpack/stratagems/priest/ssl/bg2/combatblocks/harmony.ssl	2018-08-30 07:52:13.720117700 +0200
@@ -1,10 +1,11 @@
-IF TRIGGER
-	TargetBlock(VillainsInOrder)
-	GlobalTimerExpired("redefend","LOCALS")
-	NumCreatureGT([ENEMY],3)
-	!CheckStatGT(scstarget,0,CLERIC_DEFENSIVE_HARMONY)
-	Range(scstarget,10)
-THEN DO
-	Action(SpellMyself,CLERIC_DEFENSIVE_HARMONY|100|100)
-END
-
+IF TRIGGER
+    IgnoreBlock(IsDruid)
+	TargetBlock(VillainsInOrder)
+	GlobalTimerExpired("redefend","LOCALS")
+	NumCreatureGT([ENEMY],3)
+	!CheckStatGT(scstarget,0,CLERIC_DEFENSIVE_HARMONY)
+	Range(scstarget,10)
+THEN DO
+	Action(SpellMyself,CLERIC_DEFENSIVE_HARMONY|100|100)
+END
+
