--- stratagems/mage/ssl/bg2/combatblocks/highpowered_area_attacks.ssl	2016-11-20 18:27:15.423488200 -0500
+++ Fixpack/stratagems/mage/ssl/bg2/combatblocks/highpowered_area_attacks.ssl	2016-11-20 18:27:39.769067700 -0500
@@ -11,9 +11,8 @@
 	IgnoreBlock(IsConjurerStreamline)
 	TargetBlock(PCsInOrder)
 	Range(scstarget,10)
-	TriggerBlock(Slay|MR100|Enemy|Helpless|SINecromancy)
+	TriggerBlock(Slay|MR100|Enemy|Helpless)
 	RequireBlock(HasL9)
-	TriggerBlock(DirectDamageSafe)
 	slowspell
 THEN DO
 	Action(SpellMyself,WIZARD_WAIL_OF_THE_BANSHEE|150|50)
@@ -39,9 +38,8 @@
 	IgnoreBlock(IsConjurerStreamline)
 	TargetBlock(EnemiesInOrderShort)
 	Range(scstarget,10)
-	TriggerBlock(Slay|MR100|Helpless|SINecromancy)
+	TriggerBlock(Slay|MR100|Helpless)
 	RequireBlock(HasL9)
-	TriggerBlock(DirectDamageSafe)
 	slowspell
 THEN DO
 	Action(SpellMyself,WIZARD_WAIL_OF_THE_BANSHEE|150|50)
@@ -164,7 +162,7 @@
 IF TRIGGER
 	  IgnoreBlock(Indiscriminate)
           TargetBlock(EnemiesInOrderShort)
-          TriggerBlock(MR100|SIEvocation|Enemy)
+          TriggerBlock(MR100|Enemy)
           RequireBlock(HasL8)
           IgnoreBlock(IsEnchanter)
           IgnoreBlock(IsNecromancerStreamline)
@@ -190,7 +188,7 @@
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
 	!Gender(scstarget,SUMMONED)
-	TriggerBlock(Stun|MR100|SIConjuration|Helpless)
+	TriggerBlock(Stun|MR100|Helpless)
 	RequireBlock(HasL8)
 	IgnoreBlock(IsEvoker)
 	IgnoreBlock(IsNecromancerStreamline)
@@ -217,11 +215,10 @@
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
 	!Gender(scstarget,SUMMONED)
-	TriggerBlock(Slay|MR|SIConjuration)
+	TriggerBlock(Slay|MR)
 	RequireBlock(HasL8)
 	IgnoreBlock(IsEvoker)
 	IgnoreBlock(IsEnchanterStreamline)
-	TriggerBlock(DirectDamageSafe)
 	slowspell
 THEN DO
 	Action(SpellArea,WIZARD_NPC_SYMBOL_DEATH|100|300)
@@ -277,7 +274,7 @@
 	IgnoreBlock(IsEvoker)
 	RequireBlock(Demivrgvs)
 	IgnoreBlock(IsNecromancerStreamline)
-	TriggerBlock(MR100|Panic|Helpless|SIConjuration)
+	TriggerBlock(MR100|Panic|Helpless)
 	slowspell
 THEN DO
 	Action(Spell,WIZARD_NPC_SYMBOL_FEAR|100|300)
@@ -302,8 +299,7 @@
 	IgnoreBlock(IsEnchanterStreamline)
 	TargetBlock(EnemiesInOrder)
 	!Gender(scstarget,SUMMONED)
-	TriggerBlock(ResistMagicDamage|MR|SINecromancy|Helpless)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(ResistMagicDamage|MR|Helpless)
 	RequireBlock(HasL8)
 	slowspell
 THEN DO
@@ -318,10 +314,9 @@
 	    !Gender(scstarget,SUMMONED)
 	    HasItem("dw#morhw",scstarget)
         !GlobalTimerNotExpired("DMWWTimestopRunning","GLOBAL") // so we don't kill swords more than once in Timestop
-	TriggerBlock(MR|SINecromancy|Helpless)
+	TriggerBlock(MR|Helpless)
 	!CheckStatGT(scstarget,99,MAGICDAMAGERESISTANCE)
 	RequireBlock(HasL8)
-	TriggerBlock(DirectDamageSafe)
 	slowspell
 THEN DO
 	Action(SpellArea,WIZARD_ABI_DALZIMS_HORRID_WILTING|100|100)
@@ -362,8 +357,7 @@
    TargetBlock(EnemiesInOrderShort)
    Range(scstarget,10)
    RequireBlock(Demivrgvs)
-   TriggerBlock(MR|SIAlteration|Helpless)
-   CheckStatGT(scstarget,4,SAVEVSSPELL)
+   TriggerBlock(MR|Helpless)
    RequireBlock(HasL7)
    IgnoreBlock(IsNecromancerStreamline)
    slowspell
