--- stratagems/mage/ssl/bg1/combatblocks/highend.ssl	2018-08-30 07:50:32.460431200 +0200
+++ Fixpack/stratagems/mage/ssl/bg1/combatblocks/highend.ssl	2018-08-30 07:52:13.695118600 +0200
@@ -1,38 +1,38 @@
-////////////////////////////////////////////////////////////////////////
-//// If undead, use Cloudkill; summon a Stalker
-////////////////////////////////////////////////////////////////////////
-
-IF TRIGGER
-	RequireBlock(IsUndead)
-	RequireBlock(HasL5)
-	Allegiance(Myself,ENEMY)
-	TargetBlock(EnemiesInOrderShort)
-THEN DO
-	Combine()
-	Action(SpellArea,WIZARD_CLOUDKILL|900|100)
-END
-
-IF TRIGGER
-	RequireBlock(HasL6)
-	IgnoreBlock(IsInvoker)
-	Allegiance(Myself,ENEMY)
-	TargetBlock(EnemiesInOrderShort)
-THEN DO
-	Combine()
-	Action(SpellArea,WIZARD_INVISIBLE_STALKER|800|200)
-END
-
-
-IF TRIGGER
-	RequireBlock(HasL6)
-	IgnoreBlock(IsEnchanter)
-	IgnoreBlock(SpellsAreBG1)
-	TargetBlock(EnemiesInOrder)
-	TriggerBlock(Enemy|MR|ResistElectric|SIEvocation)
-	OR(2)
-		!StateCheck(scstarget,STATE_MIRRORIMAGE)
-		GlobalTimerNotExpired("ignoremirror","LOCALS")
-THEN DO
-	Combine()
-	Action(Spell,WIZARD_CHAIN_LIGHTNING|150|50)
-END
+////////////////////////////////////////////////////////////////////////
+//// If undead, use Cloudkill; summon a Stalker
+////////////////////////////////////////////////////////////////////////
+
+IF TRIGGER
+	RequireBlock(IsUndead)
+	RequireBlock(HasL5)
+	Allegiance(Myself,ENEMY)
+	TargetBlock(EnemiesInOrderShort)
+THEN DO
+	Combine()
+	Action(SpellArea,WIZARD_CLOUDKILL|900|100)
+END
+
+IF TRIGGER
+	RequireBlock(HasL6)
+	IgnoreBlock(IsInvoker)
+	Allegiance(Myself,ENEMY)
+	TargetBlock(EnemiesInOrderShort)
+THEN DO
+	Combine()
+	Action(SpellArea,WIZARD_INVISIBLE_STALKER|800|200)
+END
+
+
+IF TRIGGER
+	RequireBlock(HasL6)
+	IgnoreBlock(IsEnchanter)
+	IgnoreBlock(SpellsAreBG1)
+	TargetBlock(EnemiesInOrder)
+	TriggerBlock(Enemy|MR|ResistElectric)
+	OR(2)
+		!StateCheck(scstarget,STATE_MIRRORIMAGE)
+		GlobalTimerNotExpired("ignoremirror","LOCALS")
+THEN DO
+	Combine()
+	Action(Spell,WIZARD_CHAIN_LIGHTNING|150|50)
+END
