--- stratagems/mage/ssl/bg1/combatblocks/highend.ssl	2016-11-20 18:27:15.407476800 -0500
+++ Fixpack/stratagems/mage/ssl/bg1/combatblocks/highend.ssl	2016-12-01 15:11:46.509887600 -0500
@@ -28,7 +28,7 @@
 	IgnoreBlock(IsEnchanter)
 	IgnoreBlock(SpellsAreBG1)
 	TargetBlock(EnemiesInOrder)
-	TriggerBlock(Enemy|MR|ResistElectric|SIEvocation)
+	TriggerBlock(Enemy|MR|ResistElectric)
 	OR(2)
 		!StateCheck(scstarget,STATE_MIRRORIMAGE)
 		GlobalTimerNotExpired("ignoremirror","LOCALS")
