--- stratagems/mage/ssl/bg2/combatblocks/highpowered_sequencers.ssl	2018-08-21 11:17:54.198196200 +0200
+++ Fixpack/stratagems/mage/ssl/bg2/combatblocks/highpowered_sequencers.ssl	2018-08-21 11:36:59.380034300 +0200
@@ -9,8 +9,7 @@
 
 IF TRIGGER
           TargetBlock(PCsInOrder)
-          TriggerBlock(MR|Confused|SpellTurn|SIAlteration|Enemy|Helpless)
-	TriggerBlock(DirectDamageSafe)
+          TriggerBlock(MR|SpellTurnNew|Enemy|Helpless)
 	RequireBlock(HasL8)
 	IgnoreBlock(HasNoSequencer)
 THEN DO
@@ -20,8 +19,7 @@
 IF TRIGGER
         RequireBlock(Demivrgvs)
           TargetBlock(PCsInOrder)
-          TriggerBlock(MR|Confused|SpellTurn|SIAlteration|Enemy|Helpless)
-	TriggerBlock(DirectDamageSafe)
+          TriggerBlock(MR|SpellTurnNew|ResistMagicDamage|Enemy|Helpless)
 	RequireBlock(HasL8)
 	IgnoreBlock(HasNoSequencer)
 	OR(2)
@@ -35,7 +33,7 @@
           TargetBlock(PCsInOrder)
 	TargetBlock(PCsPreferringStrong)
 	IgnoreBlock(IsEvoker)
-	TriggerBlock(MR|Charm|SIEnchantment|SpellTurn|Enemy|Helpless)
+	TriggerBlock(MR|Charm|SIEnchantment|SpellTurnNew|Enemy|Helpless)
 	!CheckStatLT(scstarget,0,SAVEVSSPELL)
 	RequireBlock(HasL8)
 	IgnoreBlock(IsEvoker)
@@ -56,7 +54,7 @@
 	IgnoreBlock(IsEvoker)
 	IgnoreBlock(HasNoSequencer)
 THEN DO
-	Action(Trigger,7,WIZARD_CHAOS,WIZARD_CHAOS,WIZARD_CHAOS|100|100) // Chaos x3
+	Action(Trigger,7,WIZARD_CONFUSION,WIZARD_CONFUSION,WIZARD_CONFUSION|100|100) // Chaos x3
 END
 
 IF TRIGGER
@@ -66,12 +64,12 @@
 	IgnoreBlock(IsEvoker)
 	IgnoreBlock(HasNoSequencer)
 THEN DO
-	Action(TriggerMyself,7,WIZARD_CHAOS,WIZARD_CHAOS,WIZARD_CHAOS) // Chaos x3
+	Action(TriggerMyself,7,WIZARD_CONFUSION,WIZARD_CONFUSION,WIZARD_CONFUSION) // Chaos x3
 END
 
 IF TRIGGER
 	TargetBlock(PCsInOrder)
-	TriggerBlock(MR|ResistElectric|SIEvocation|Enemy)
+	TriggerBlock(MR|ResistElectric|Enemy)
 	TriggerBlock(DirectDamageSafe)
 	RequireBlock(HasL8)
 	IgnoreBlock(IsEvoker)
@@ -94,8 +92,7 @@
 
 IF TRIGGER
 	TargetBlock(PCsInOrder)
-	TriggerBlock(MR|ResistElectric|SIEvocation|Enemy)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(MR|ResistElectric|Enemy)
 	RequireBlock(HasL8)
 	IgnoreBlock(IsEnchanter)
 	IgnoreBlock(HasNoSequencer)
@@ -170,8 +167,7 @@
 IF TRIGGER
 	TargetBlock(PCsInOrder)
 	TargetBlock(Celestials)
-	TriggerBlock(MR|ResistMagicDamage|Enemy|SINecromancy|MinorGlobe)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(MR|ResistMagicDamage|Enemy|MinorGlobe)
 	!NumCreatureGT([ENEMY.WEAPON],0)
 	RequireBlock(HasL7)
 	IgnoreBlock(HasNoSequencer)
@@ -190,9 +186,8 @@
 IF TRIGGER
 	TargetBlock(PCsInOrder)
 	TargetBlock(Celestials)
-	TriggerBlock(MR|ResistFire|Enemy|SIEvocation|MinorGlobe)
+	TriggerBlock(MR|ResistFire|Enemy|MinorGlobe)
 	IgnoreBlock(IsEnchanter)
-	TriggerBlock(DirectDamageSafe)
 	!NumCreatureGT([ENEMY.WEAPON],0)
 	RequireBlock(HasL7)
 	IgnoreBlock(HasNoSequencer)
@@ -212,9 +207,8 @@
 IF TRIGGER
 	TargetBlock(PCsInOrder)
 	TargetBlock(Celestials)
-	TriggerBlock(MR|ResistElectric|Enemy|SIEvocation|MinorGlobe)
+	TriggerBlock(MR|ResistElectric|Enemy|MinorGlobe)
 	IgnoreBlock(IsEnchanter)
-	TriggerBlock(DirectDamageSafe)
 	!NumCreatureGT([ENEMY.WEAPON],0)
 	RequireBlock(HasL7)
 	IgnoreBlock(HasNoSequencer)
@@ -234,10 +228,9 @@
 IF TRIGGER
 	ConditionalTargetBlock(PCsPreferringWeak;HPPercentLT(scstarget,40))
 	TargetBlock(PCsPreferringWeak)
-	TriggerBlock(DirectDamageSafe)
 	RequireBlock(HasL7)
 	IgnoreBlock(IsEvoker)
-	TriggerBlock(MR|ResistFire|SIConjuration|ArrowSafe|SpellTurn|Stoneskin|MinorGlobe|Enemy|Helpless)
+	TriggerBlock(MR|ResistFire|SpellTurnNew|MinorGlobe|Enemy|Helpless)
 	IgnoreBlock(HasNoSequencer)
 THEN DO
 	Action(Sequencer,5,WIZARD_FLAME_ARROW,WIZARD_FLAME_ARROW,WIZARD_FLAME_ARROW) // 3x Flame Arrow
