--- stratagems/priest/ssl/bg1/combatblocks/holy_smite.ssl	2016-11-20 18:27:15.478527400 -0500
+++ Fixpack/stratagems/priest/ssl/bg1/combatblocks/holy_smite.ssl	2016-09-16 15:07:35.807784400 -0400
@@ -10,9 +10,7 @@
 	ConditionalTargetBlock(PCsPreferringStrong;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
 	!Alignment(Myself,MASK_GOOD)
-	Alignment(scstarget,MASK_GOOD)
-	TriggerBlock(DirectDamageSafe)
-	TriggerBlock(MR100|SINecromancy|ResistMagicDamage|Enemy)
+	TriggerBlock(MR100|ResistMagicDamage|Enemy)
 THEN DO
 	Action(SpellArea,CLERIC_UNHOLY_BLIGHT|200|100)
 END
@@ -26,9 +24,7 @@
 	ConditionalTargetBlock(PCsPreferringStrong;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
 	!Alignment(Myself,MASK_EVIL)
-	Alignment(scstarget,MASK_EVIL)
-	TriggerBlock(DirectDamageSafe)
-	TriggerBlock(MR100|SINecromancy|ResistMagicDamage|Enemy)
+	TriggerBlock(MR100|ResistMagicDamage|Enemy)
 THEN DO
 	Action(SpellArea,CLERIC_HOLY_SMITE|200|100)
 END
@@ -39,7 +35,6 @@
 	RequireBlock(HasL3)
 	ConditionalTargetBlock(PCsPreferringStrong;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(DirectDamageSafe)
 	TriggerBlock(MR100|ResistElectric|Enemy)
 THEN DO
 	Action(SpellArea,CLERIC_GLYPH_OF_WARDING|200|100)
--- stratagems/priest/ssl/bg2/combatblocks/holy_smite.ssl	2016-11-20 18:27:15.503544900 -0500
+++ Fixpack/stratagems/priest/ssl/bg2/combatblocks/holy_smite.ssl	2016-09-16 14:39:52.334164600 -0400
@@ -10,9 +10,7 @@
 	ConditionalTargetBlock(PCsPreferringStrong;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
 	!Alignment(Myself,MASK_GOOD)
-	Alignment(scstarget,MASK_GOOD)
-	TriggerBlock(DirectDamageSafe)
-	TriggerBlock(MR100|SINecromancy|ResistMagicDamage|Enemy|MirrorImage)
+	TriggerBlock(MR100|ResistMagicDamage|Enemy)
 THEN DO
         Combine()
 	Action(SpellArea,CLERIC_UNHOLY_BLIGHT|100|100)
@@ -45,9 +43,7 @@
 	ConditionalTargetBlock(PCsPreferringStrong;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
 	!Alignment(Myself,MASK_EVIL)
-	Alignment(scstarget,MASK_EVIL)
-	TriggerBlock(DirectDamageSafe)
-	TriggerBlock(MR100|SINecromancy|ResistMagicDamage|Enemy|MirrorImage)
+	TriggerBlock(MR100|ResistMagicDamage|Enemy)
 THEN DO
         Combine()
 	Action(SpellArea,CLERIC_HOLY_SMITE|100|100)
@@ -77,8 +73,7 @@
 	RequireBlock(HasL3)
 	ConditionalTargetBlock(PCsPreferringStrong;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(DirectDamageSafe)
-	TriggerBlock(MR100|ResistMagicDamage|Enemy|MirrorImage)
+	TriggerBlock(MR100|ResistElectric|Enemy)
 THEN DO
 	Action(SpellArea,CLERIC_GLYPH_OF_WARDING|200|100)
 END
