--- stratagems/priest/ssl/bg2/combatblocks/implosion.ssl	2018-08-30 07:50:32.484430700 +0200
+++ Fixpack/stratagems/priest/ssl/bg2/combatblocks/implosion.ssl	2018-08-30 07:52:13.721118200 +0200
@@ -9,8 +9,7 @@
 	ConditionalTargetBlock(PCsPreferringWeak;CheckStatGT(scstarget,40,RESISTMAGIC))
 	TargetBlock(PCsPreferringWeak)
 	TargetBlock(Celestials)
-	TriggerBlock(SpellTurn|SIEvocation|Enemy|Helpless)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(SpellTurnNew|Enemy|Helpless)
 THEN DO
 	Action(Spell,CLERIC_IMPLOSION|150|50)
 END
