--- stratagems/mage/ssl/bg2/combatblocks/imprisonment.ssl	2018-08-30 07:50:32.466431300 +0200
+++ Fixpack/stratagems/mage/ssl/bg2/combatblocks/imprisonment.ssl	2018-08-30 07:52:13.702118500 +0200
@@ -12,7 +12,7 @@
 	OR(2)
 		!GlobalTimerNotExpired("DMWWTimestopRunning","GLOBAL")
 		!GlobalTimerNotExpired("timestopmaze","LOCALS")
-	TriggerBlock(SpellTurn|Enemy|Helpless|SIAbjuration|Imprisonment)
+	TriggerBlock(SpellTurnNew|Enemy|Helpless|SIAbjuration|Imprisonment)
 	OR(2)
 		!InPartySlot(scstarget,0) // isn't Player 1
 		Global("DMWWImprisonPlayer","GLOBAL",1)
@@ -34,7 +34,7 @@
 	OR(2)
 		!GlobalTimerNotExpired("DMWWTimestopRunning","GLOBAL")
 		!GlobalTimerNotExpired("timestopmaze","LOCALS")
-	TriggerBlock(SpellTurn|Enemy|Helpless|SIAbjuration|Imprisonment)
+	TriggerBlock(SpellTurnNew|Enemy|Helpless|Imprisonment)
 	OR(2)
 		!InPartySlot(scstarget,0) // isn't Player 1
 		Global("DMWWImprisonPlayer","GLOBAL",1)
@@ -56,7 +56,7 @@
 	OR(2)
 		!GlobalTimerNotExpired("DMWWTimestopRunning","GLOBAL")
 		!GlobalTimerNotExpired("timestopmaze","LOCALS")
-	TriggerBlock(SpellTurn|Enemy|Helpless|SIAbjuration|Imprisonment)
+	TriggerBlock(SpellTurnNew|Enemy|Helpless|Imprisonment)
 	OR(2)
 		!InPartySlot(scstarget,0) // isn't Player 1
 		Global("DMWWImprisonPlayer","GLOBAL",1)
@@ -78,7 +78,7 @@
 	OR(2)
 		!GlobalTimerNotExpired("DMWWTimestopRunning","GLOBAL")
 		!GlobalTimerNotExpired("timestopmaze","LOCALS")
-	TriggerBlock(SpellTurn|Enemy|Helpless|SIAbjuration|Imprisonment)
+	TriggerBlock(SpellTurnNew|Enemy|Helpless|Imprisonment)
 	OR(2)
 		!InPartySlot(scstarget,0) // isn't Player 1
 		Global("DMWWImprisonPlayer","GLOBAL",1)
