--- stratagems/priest/ssl/bg2/combatblocks/insects.ssl	2018-08-27 01:01:22.000000000 +0200
+++ Fixpack/stratagems/priest/ssl/bg2/combatblocks/insects.ssl	2018-08-27 18:00:49.000000000 +0200
@@ -1,15 +1,15 @@
-
-///////////////////////////////////////////////////////////////
-///	Insect Plague etc
-///////////////////////////////////////////////////////////////
-
-IF TRIGGER
-	TargetBlock(PCSpellcasters)
-	TargetBlock(PCsInOrderShort)
-	TargetBlock(Celestials)
-	TriggerBlock(MR100|SIConjuration|Enemy|Helpless)
-	!CheckStatGT(scstarget,0,CLERIC_INSECT_PLAGUE)
-THEN DO
-	Action(Spell,CLERIC_INSECT_PLAGUE|200|100)
-	Action(Spell,CLERIC_SUMMON_INSECTS|200|100)
-END
+
+///////////////////////////////////////////////////////////////
+///	Insect Plague etc
+///////////////////////////////////////////////////////////////
+
+IF TRIGGER
+	TargetBlock(PCSpellcasters)
+	TargetBlock(PCsInOrderShort)
+	TargetBlock(Celestials)
+	TriggerBlock(MR100|Enemy|Helpless)
+	!CheckStatGT(scstarget,0,CLERIC_INSECT_PLAGUE)
+THEN DO
+	Action(Spell,CLERIC_INSECT_PLAGUE|200|100)
+	Action(Spell,CLERIC_SUMMON_INSECTS|200|100)
+END
