--- stratagems/mage/ssl/bg2/prepblocks/instantprep.ssl	2016-11-20 18:27:15.446504700 -0500
+++ Fixpack/stratagems/mage/ssl/bg2/prepblocks/instantprep.ssl	2016-12-01 15:11:46.531903200 -0500
@@ -136,6 +136,16 @@
 IF TRIGGER
 	IgnoreBlock(IsOptionThree)
 	optiontwosub1
+	RequireBlock(HasL7)
+	!HaveSpell(WIZARD_PROTECTION_FROM_THE_ELEMENTS)
+	IgnoreBlock(HasNoPrep)
+THEN DO
+	Action(SpellPrecast,WIZARD_PROTECTION_FROM_THE_ELEMENTS)
+END
+
+IF TRIGGER
+	IgnoreBlock(IsOptionThree)
+	optiontwosub1
 	RequireBlock(HasL9)
 	IgnoreBlock(HasNoPrep)
 THEN DO
--- stratagems/priest/ssl/bg1/prepblocks/instantprep.ssl	2016-11-20 18:27:15.489535400 -0500
+++ Fixpack/stratagems/priest/ssl/bg1/prepblocks/instantprep.ssl	2016-09-16 16:08:12.903649700 -0400
@@ -20,6 +20,7 @@
 END
 
 IF TRIGGER
+    IgnoreBlock(IsDruid)
 	IgnoreBlock(IsOptionThree)
 	optiontwosub1
 THEN DO
@@ -36,6 +37,7 @@
 END
 
 IF TRIGGER
+   IgnoreBlock(IsCleric)
    IgnoreBlock(IsOptionThree)
    RequireBlock(HasL2)
 	optiontwosub1
@@ -54,6 +56,7 @@
 END
 
 IF TRIGGER
+   IgnoreBlock(IsDruid)
    IgnoreBlock(IsOptionThree)
    RequireBlock(HasL4)
 	optiontwosub1
@@ -63,7 +66,6 @@
 
 IF TRIGGER
    IgnoreBlock(IsOptionThree)
-   IgnoreBlock(IsDruid)
    RequireBlock(HasL4)
 	optiontwosub1
 THEN DO
@@ -99,6 +101,7 @@
 END
 
 IF TRIGGER
+   IgnoreBlock(IsDruid)
    IgnoreBlock(IsOptionThree)
    RequireBlock(HasL6)
 	optiontwosub1
--- stratagems/priest/ssl/bg2/prepblocks/instantprep.ssl	2016-11-20 18:27:15.527562300 -0500
+++ Fixpack/stratagems/priest/ssl/bg2/prepblocks/instantprep.ssl	2016-09-17 22:48:40.317536900 -0400
@@ -1,5 +1,6 @@
 
 IF TRIGGER
+    IgnoreBlock(IsDruid)
 	RequireBlock(HasL7)
 	IgnoreBlock(IsOptionThree)
 	optiontwosub1
@@ -22,6 +23,19 @@
 		Action(SpellPrecast,CLERIC_REGENERATE)
 END
 
+IF TRIGGER
+    IgnoreBlock(IsCleric)
+	RequireBlock(HasL6)
+	IgnoreBlock(IsOptionThree)
+	optiontwosub1
+	!HasItem("dw#nopre",Myself)
+	IgnoreBlock(HasNoPrep)
+	!HasItem("dw#prbig",Myself)
+THEN DO
+		Action(SpellPrecast,CLERIC_REGENERATE)
+END
+
+
 IF  TRIGGER
 	RequireBlock(HasL7)
 	IgnoreBlock(IsOptionThree)
@@ -80,6 +94,7 @@
 END
 
 IF TRIGGER
+    IgnoreBlock(IsDruid)
 	IgnoreBlock(IsOptionThree)
 	optiontwosub1
 	!HasItem("dw#nopre",Myself)
@@ -100,6 +115,7 @@
 END
 
 IF TRIGGER
+    IgnoreBlock(IsCleric)
 	IgnoreBlock(IsOptionThree)
 	optiontwosub1
 	!HasItem("dw#nopre",Myself)
@@ -122,6 +138,7 @@
 END
 
 IF TRIGGER
+    IgnoreBlock(IsDruid)
 	IgnoreBlock(IsOptionThree)
 	optiontwosub1
 	!HasItem("dw#nopre",Myself)
@@ -136,7 +153,6 @@
 	optiontwosub1
 	!HasItem("dw#nopre",Myself)
 	IgnoreBlock(HasNoPrep)
-   IgnoreBlock(IsDruid)
    RequireBlock(HasL4)
 THEN DO
      Action(SpellPrecast,CLERIC_FREE_ACTION)
@@ -178,6 +194,7 @@
 END
 
 IF TRIGGER
+    IgnoreBlock(IsDruid)
 	IgnoreBlock(IsOptionThree)
 	optiontwosub1
 	!HasItem("dw#nopre",Myself)
@@ -193,10 +210,10 @@
 	!HasItem("dw#nopre",Myself)
 	IgnoreBlock(HasNoPrep)
    RequireBlock(HasL6)
-   !HaveSpell(CLERIC_GLOBE_OF_BLADES)
+   HaveSpell(CLERIC_BLADE_BARRIER)
    IgnoreBlock(IsDruid)
 THEN DO
-     Action(SpellPrecast,NPC_BLADE_BARRIER)
+     Action(SpellPrecast,CLERIC_BLADE_BARRIER)
 END
 
 IF TRIGGER
