--- stratagems/mage/ssl/bg2/combatblocks/kill-maze-stun.ssl	2018-08-21 11:17:54.200196200 +0200
+++ Fixpack/stratagems/mage/ssl/bg2/combatblocks/kill-maze-stun.ssl	2018-08-21 11:36:59.382034100 +0200
@@ -9,14 +9,13 @@
 	RequireBlock(HasL9)
 	IgnoreBlock(IsEvoker)
 	OR(2)
-		HPPercentLT(scstarget,75)
+		!CheckStatLT(scstarget,8,SAVEVSSPELL)
 		!CheckStatGT(scstarget,12,Level)
 	OR(3)
 		HPPercentLT(scstarget,50)
 		Class(scstarget,MAGE)
 		Class(scstarget,MAGE_THIEF)
-	TriggerBlock(MR100|SpellTurn|Slay|SIConjuration|Enemy)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(MR100|SpellTurn|Slay|SpellTurnNew|Enemy)
         !GlobalTimerNotExpired("DMWWTimestopRunning","GLOBAL")
 THEN DO
 	Combine()
@@ -35,8 +34,7 @@
 		HPPercentLT(scstarget,50)
 		Class(scstarget,MAGE)
 		Class(scstarget,MAGE_THIEF)
-	TriggerBlock(MR100|SpellTurn|Slay|SIConjuration)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(MR100|SpellTurn|Slay|SpellTurnNew)
         !GlobalTimerNotExpired("DMWWTimestopRunning","GLOBAL")
 THEN DO
 	Combine()
@@ -60,7 +58,7 @@
 	OR(2)
 		!GlobalTimerNotExpired("DMWWTimestopRunning","GLOBAL")
 		!GlobalTimerNotExpired("timestopmaze","LOCALS")
-	TriggerBlock(Maze|SpellTurn|MR|Enemy|Helpless|SIConjuration)
+	TriggerBlock(Maze|SpellTurn|MR|Enemy|Helpless|SpellTurnNew)
 	Range(scstarget,6)
 	OR(2)
 		InPartySlot(scstarget,1) // isn't Player 1
@@ -83,7 +81,7 @@
 	OR(2)
 		!GlobalTimerNotExpired("DMWWTimestopRunning","GLOBAL")
 		!GlobalTimerNotExpired("timestopmaze","LOCALS")
-	TriggerBlock(Maze|SpellTurn|MR|Helpless|SIConjuration)
+	TriggerBlock(Maze|SpellTurn|MR|Helpless|SpellTurnNew)
 	Range(scstarget,6)
 	OR(2)
 		InPartySlot(scstarget,1) // isn't Player 1
@@ -107,7 +105,7 @@
 	IgnoreBlock(IsEvoker)
 	IgnoreBlock(IsNecromancerStreamline)
 	TargetBlock(Celestials)
-	TriggerBlock(MR100|SpellTurn|Stun|Enemy|Helpless|SIConjuration)
+	TriggerBlock(MR100|SpellTurnNew|Stun|Enemy|Helpless)
 	OR(3)
 		HPPercentLT(scstarget,60)
 		Class(scstarget,MAGE)
@@ -123,7 +121,7 @@
 	RequireBlock(HasL7)
 	IgnoreBlock(IsEvoker)
 	IgnoreBlock(IsNecromancerStreamline)
-	TriggerBlock(MR100|SpellTurn|Stun|Helpless|SIConjuration)
+	TriggerBlock(MR100|SpellTurnNew|Stun|Helpless)
 	OR(3)
 		HPPercentLT(scstarget,60)
 		Class(scstarget,MAGE)
@@ -147,7 +145,7 @@
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(PCsInOrderShort)
 	TargetBlock(Celestials)
-	TriggerBlock(MR100|SpellTurn|Enemy|Helpless|SIEvocation)
+	TriggerBlock(MR100|SpellTurnNew|Enemy|Helpless)
 	CheckStatGT(scstarget,4,SAVEVSDEATH)
 THEN DO
         Combine()
@@ -163,7 +161,7 @@
 	IgnoreBlock(IsNecromancerStreamline)
 	TargetBlock(EnemiesInOrder)
 	TargetBlock(Celestials)
-	TriggerBlock(MR100|SpellTurn|Helpless|SIEvocation)
+	TriggerBlock(MR100|SpellTurnNew|Helpless)
 	CheckStatGT(scstarget,4,SAVEVSDEATH)
 THEN DO
         Combine()
@@ -183,8 +181,7 @@
 	RequireBlock(HasL6)
 	IgnoreBlock(IsConjurerStreamline)
 	IgnoreBlock(IsEnchanter)
-	TriggerBlock(MR100|SpellTurn|Disintegrate|SIEvocation|Enemy)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(MR100|SpellTurnNew|Disintegrate|ResistMagicDamage|Slay|Enemy)
 	OR(2)
 	        GlobalGT("DMWWDisintegratePlayer","GLOBAL",0)
 		!InPartySlot(scstarget,0) // isn't Player 1
@@ -200,8 +197,7 @@
 	RequireBlock(HasL6)
 	IgnoreBlock(IsConjurerStreamline)
 	IgnoreBlock(IsEnchanter)
-	TriggerBlock(MR100|SpellTurn|Disintegrate|SIEvocation)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(MR100|SpellTurnNew|Disintegrate|Slay|ResistMagicDamage)
 	OR(2)
 	        GlobalGT("DMWWDisintegratePlayer","GLOBAL",0)
 		!InPartySlot(scstarget,0) // isn't Player 1
