--- stratagems/mage/override/bg2/level.2da	2018-08-27 01:01:22.000000000 +0200
+++ Fixpack/stratagems/mage/override/bg2/level.2da	2018-08-27 18:00:49.000000000 +0200
@@ -56,13 +56,14 @@
 FIRRAK01	15
 PPIRENI2	30
 PPWANEV2	18
-YAGA02          27
-NEVAZIAH        18
-GORLIC01        33
-GROMG08         24
-IOGREMAG        7
-DRUSH           7
-OGRMAG01        7
-HELLSLAY        30
-FIRLCH01        33
-SENLICH         34
+YAGA02		27
+NEVAZIAH	18
+GORLIC01	33
+GROMG08		24
+IOGREMAG	7
+DRUSH		7
+OGRMAG01	7
+HELLSLAY	30
+FIRLCH01	33
+SENLICH		34
+LICH01		25
