--- stratagems/sfo/filetype/lib_spl.tpa	2016-11-20 18:27:15.537569000 -0500
+++ Fixpack/stratagems/sfo/filetype/lib_spl.tpa	2016-11-20 18:27:39.872140600 -0500
@@ -742,7 +742,7 @@
                END
              END
              PATCH_MATCH "%spell_ID%" WITH
-             CLERIC_BLADE_BARRIER CLERIC_GLOBE_OF_BLADES CLERIC_AURA_OF_FLAMING_DEATH WIZARD_FIRE_SHIELD_RED WIZARD_FIRE_SHIELD_BLUE
+             NPC_BLADE_BARRIER CLERIC_GLOBE_OF_BLADES CLERIC_AURA_OF_FLAMING_DEATH WIZARD_FIRE_SHIELD_RED WIZARD_FIRE_SHIELD_BLUE
              BEGIN
                   LPF delete_entry STR_VAR type=effect arguments="opcode=174" file_ext=SPL END
              END
