--- stratagems/mage/ssl/bg1/prepblocks/longprep_core.ssl	2018-08-27 01:01:22.000000000 +0200
+++ Fixpack/stratagems/mage/ssl/bg1/prepblocks/longprep_core.ssl	2018-08-27 18:00:49.000000000 +0200
@@ -50,3 +50,15 @@
 		Continue()
 END
 
+IF
+	Global("longprep","LOCALS",0)
+	scsvar
+	RequireBlock(HasL2)
+	RequireBlock(IsConjurer)
+	IgnoreBlock(HasNoPrep)
+	!HasItem("dw#nopre",Myself)
+THEN
+	RESPONSE #100
+		ReallyForceSpellRES("spwi226",Myself)
+		Continue()
+END
--- stratagems/mage/ssl/bg2/prepblocks/longprep_core.ssl	2018-08-27 01:01:22.000000000 +0200
+++ Fixpack/stratagems/mage/ssl/bg2/prepblocks/longprep_core.ssl	2018-08-27 18:00:49.000000000 +0200
@@ -138,3 +138,69 @@
 		Continue()
 END
 
+IF
+	Global("longprep","LOCALS",0)
+	scsvar
+	RequireBlock(HasL8)
+	IgnoreBlock(HasNoPrep)
+	!HasItem("dw#nopre",Myself)
+THEN
+	RESPONSE #100
+		ReallyForceSpellRES("spwi802",Myself)
+		Continue()
+END
+
+IF
+	Global("longprep","LOCALS",0)
+	scsvar
+	RequireBlock(HasL7)
+	RequireBlock(IsConjurer)
+	IgnoreBlock(HasL8)
+	IgnoreBlock(HasL9) 
+	IgnoreBlock(HasNoPrep)
+	!HasItem("dw#nopre",Myself)
+THEN
+	RESPONSE #100
+		ReallyForceSpellRES("spwi724",Myself)
+		Continue()
+END
+
+IF
+	Global("longprep","LOCALS",0)
+	scsvar
+	RequireBlock(HasL8)
+	RequireBlock(IsConjurer)
+	IgnoreBlock(HasL9) 
+	IgnoreBlock(HasNoPrep)
+	!HasItem("dw#nopre",Myself)
+THEN
+	RESPONSE #100
+		ReallyForceSpellRES("spwi819",Myself)
+		Continue()
+END
+
+IF
+	Global("longprep","LOCALS",0)
+	scsvar
+	RequireBlock(HasL9)
+	RequireBlock(IsConjurer)
+	IgnoreBlock(HasNoPrep)
+	!HasItem("dw#nopre",Myself)
+THEN
+	RESPONSE #100
+		ReallyForceSpellRES("spwi906",Myself)
+		Continue()
+END
+
+IF
+	Global("longprep","LOCALS",0)
+	scsvar
+	RequireBlock(HasL6)
+	RequireBlock(IsNecromancer)
+	IgnoreBlock(HasNoPrep)
+	!HasItem("dw#nopre",Myself)
+THEN
+	RESPONSE #100
+		ReallyForceSpellRES("spwi623",Myself)
+		Continue()
+END
--- stratagems/priest/ssl/bg1/prepblocks/longprep_core.ssl	2018-08-27 01:01:22.000000000 +0200
+++ Fixpack/stratagems/priest/ssl/bg1/prepblocks/longprep_core.ssl	2018-08-27 18:00:49.000000000 +0200
@@ -26,6 +26,7 @@
    HaveSpell(CLERIC_DEATH_WARD)
    RequireBlock(HasL4)
    IgnoreBlock(SpellsAreBG1)
+   IgnoreBlock(IsDruid)
 THEN
      RESPONSE #100
               ApplySpellRES("%CLERIC_DEATH_WARD%",Myself)
@@ -38,6 +39,7 @@
    HaveSpell(CLERIC_CHAOTIC_COMMANDS)
    RequireBlock(HasL5)
    IgnoreBlock(SpellsAreBG1)
+   IgnoreBlock(IsDruid)
 THEN 
      RESPONSE #100
               ApplySpellRES("%CLERIC_CHAOTIC_COMMANDS%",Myself)
--- stratagems/priest/ssl/bg2/prepblocks/longprep_core.ssl	2018-08-27 01:01:22.000000000 +0200
+++ Fixpack/stratagems/priest/ssl/bg2/prepblocks/longprep_core.ssl	2018-08-27 18:00:49.000000000 +0200
@@ -1,49 +1,50 @@
-///////////////////////////////////////////////////////////////////////
-/// Long-duration spells
-///////////////////////////////////////////////////////////////////
-
-
-//////////////////////////////////////////////////////////////
-/// Ironskin - level 9+
-//////////////////////////////////////////////////////////////
-
-IF
-  Global("longprep","LOCALS",0)
-  RequireBlock(HasL5)
-  RequireBlock(IsDruid)
-THEN
-    RESPONSE #100
-             ReallyForceSpellRes("%CLERIC_IRONSKIN_PREBUFF%",Myself)
-             Continue()
-END
-
-///////////////////////////////////////////////////////////////////////
-///	Protection from Fire
-///////////////////////////////////////////////////////////////////////
-
-IF TRIGGER
-   Global("longprep","LOCALS",0)
-   RequireBlock(HasL3)
-THEN DO
-     Action(SpellPrecast,CLERIC_PROTECTION_FROM_FIRE)
-END
-
-//////////////////////////////////////////////////////////////
-/// Death Ward and Chaotic Commands
-//////////////////////////////////////////////////////////////
-
-IF TRIGGER
-   Global("longprep","LOCALS",0)
-   RequireBlock(HasL4)
-THEN DO
-     Action(SpellPrecast,CLERIC_DEATH_WARD)
-END
-
-
-IF TRIGGER
-   Global("longprep","LOCALS",0)
-   RequireBlock(HasL5)
-THEN DO
-     Action(SpellPrecast,CLERIC_CHAOTIC_COMMANDS)
-END
-
+///////////////////////////////////////////////////////////////////////
+/// Long-duration spells
+///////////////////////////////////////////////////////////////////
+
+
+//////////////////////////////////////////////////////////////
+/// Ironskin - level 9+
+//////////////////////////////////////////////////////////////
+
+IF
+  Global("longprep","LOCALS",0)
+  RequireBlock(HasL5)
+  RequireBlock(IsDruid)
+THEN
+    RESPONSE #100
+             ReallyForceSpellRes("%CLERIC_IRONSKIN_PREBUFF%",Myself)
+             Continue()
+END
+
+///////////////////////////////////////////////////////////////////////
+///	Protection from Fire
+///////////////////////////////////////////////////////////////////////
+
+IF TRIGGER
+   Global("longprep","LOCALS",0)
+   RequireBlock(HasL3)
+THEN DO
+     Action(SpellPrecast,CLERIC_PROTECTION_FROM_FIRE)
+END
+
+//////////////////////////////////////////////////////////////
+/// Death Ward and Chaotic Commands
+//////////////////////////////////////////////////////////////
+
+IF TRIGGER
+   Global("longprep","LOCALS",0)
+   RequireBlock(HasL4)
+THEN DO
+     Action(SpellPrecast,CLERIC_DEATH_WARD)
+END
+
+
+IF TRIGGER
+   Global("longprep","LOCALS",0)
+   RequireBlock(HasL5)
+   IgnoreBlock(IsDruid)
+THEN DO
+     Action(SpellPrecast,CLERIC_CHAOTIC_COMMANDS)
+END
+
