--- stratagems/mage/ssl/bg2/combatblocks/low_level_disabling.ssl	2016-11-20 18:27:15.435496700 -0500
+++ Fixpack/stratagems/mage/ssl/bg2/combatblocks/low_level_disabling.ssl	2016-12-01 15:11:46.523897600 -0500
@@ -8,8 +8,8 @@
 	IgnoreBlock(Indiscriminate)
 	TargetBlock(PCsPreferringStrong)
 	RequireBlock(HasL3)
-	TriggerBlock(MR100|Slow|SIAlteration|Enemy|MinorGlobe|Helpless)
-	!CheckStatLT(scstarget,0,SAVEVSSPELL)
+	TriggerBlock(MR100|Slow|Enemy|MinorGlobe|Helpless)
+	!CheckStatLT(scstarget,0,SAVEVSPOLY)
 	!StateCheck(scstarget,STATE_SLOWED)
 THEN DO
 	Combine()
@@ -20,8 +20,8 @@
 	RequireBlock(Indiscriminate)
 	TargetBlock(EnemiesInOrderShort)
 	RequireBlock(HasL3)
-	TriggerBlock(MR100|Slow|SIAlteration|MinorGlobe|Helpless)
-	!CheckStatLT(scstarget,0,SAVEVSSPELL)
+	TriggerBlock(MR100|Slow|MinorGlobe|Helpless)
+	!CheckStatLT(scstarget,0,SAVEVSPOLY)
 	!StateCheck(scstarget,STATE_SLOWED)
 THEN DO
 	Combine()
@@ -37,7 +37,7 @@
 	IgnoreBlock(Indiscriminate)
 	TargetBlock(PCsPreferringStrong)
 	IgnoreBlock(IsEvoker)
-	TriggerBlock(MR|Charm|SIEnchantment|SpellTurn|Enemy|Helpless)
+	TriggerBlock(MR|Charm|SpellTurnNew|Enemy|Helpless)
 	!CheckStatLT(scstarget,0,SAVEVSSPELL)
 THEN DO
 	Combine()
@@ -52,7 +52,7 @@
 	RequireBlock(Indiscriminate)
 	TargetBlock(EnemiesInOrder)
 	IgnoreBlock(IsEvoker)
-	TriggerBlock(MR|Charm|SIEnchantment|SpellTurn|Helpless)
+	TriggerBlock(MR|Charm|SpellTurnNew|Helpless)
 	!CheckStatLT(scstarget,0,SAVEVSSPELL)
 THEN DO
 	Combine()
@@ -71,7 +71,7 @@
 	TargetBlock(EnemiesInOrder)
 	RequireBlock(HasL4)
 	IgnoreBlock(IsEvoker)
-	TriggerBlock(MR100|Confused|ConfuseProof|SIEnchantment|Helpless)
+	TriggerBlock(MR100|Confused|ConfuseProof|Helpless)
 	!CheckStatLT(scstarget,0,SAVEVSSPELL)
 	slowspell
 THEN DO
@@ -89,7 +89,7 @@
 	RequireBlock(HasL2)
 	IgnoreBlock(IsEvoker)
 	IgnoreBlock(IsNecromancerStreamline)
-	TriggerBlock(MR100|SpellTurn|Hold|SIEnchantment|Enemy|Helpless)
+	TriggerBlock(MR100|SpellTurnNew|Hold|Enemy|Helpless)
 	!CheckStatLT(scstarget,0,SAVEVSSPELL)
 	Target(NearestEnemyOf(Myself))
 THEN DO
@@ -105,7 +105,7 @@
 	RequireBlock(HasL2)
 	IgnoreBlock(IsEvoker)
 	IgnoreBlock(IsNecromancerStreamline)
-	TriggerBlock(MR100|SpellTurn|Hold|SIEnchantment|Helpless)
+	TriggerBlock(MR100|SpellTurnNew|Hold|Helpless)
 	!CheckStatLT(scstarget,0,SAVEVSSPELL)
 	Target(NearestEnemyOf(Myself))
 THEN DO
@@ -123,7 +123,7 @@
 	TargetBlock(EnemiesInOrder)
 	RequireBlock(HasL4)
 	IgnoreBlock(IsEvoker)
-	TriggerBlock(MR100|SIEnchantment|Helpless)
+	TriggerBlock(MR100|Helpless)
 	!CheckStatLT(scstarget,0,SAVEVSSPELL)
 	slowspell
 THEN DO
@@ -163,9 +163,8 @@
 	IgnoreBlock(IsEvokerStreamline)
 	IgnoreBlock(IsNecromancer)
 	IgnoreBlock(Demivrgvs)
-	TargetBlock(PCsPreferringWeak)
+	TargetBlock(PCsPreferringStrong)
 	TriggerBlock(Enemy|Helpless|MR|MinorGlobe)
-	CheckStatGT(scstarget,0,SAVEVSSPELL)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_BLINDNESS|100|200)
@@ -174,7 +173,7 @@
 IF TRIGGER
 	RequireBlock(IsNecromancerStreamline)
 	RequireBlock(Demivrgvs)
-	TargetBlock(PCsPreferringWeak)
+	TargetBlock(PCsPreferringStrong)
 	TriggerBlock(Enemy|Helpless|MR|MinorGlobe)
 	CheckStatGT(scstarget,0,SAVEVSSPELL)
 THEN DO
@@ -193,7 +192,6 @@
 	RequireBlock(HasL2)
 	TargetBlock(PCFighters)
 	TriggerBlock(Enemy|Helpless|MR|MinorGlobe)
-	CheckStatGT(scstarget,9,STR)
 	CheckStatGT(scstarget,0,SAVEVSSPELL)
 THEN DO
 	Combine()
@@ -208,7 +206,6 @@
 	RequireBlock(HasL2)
 	TargetBlock(EnemiesInOrderShort)
 	TriggerBlock(Helpless|MR|MinorGlobe)
-	CheckStatGT(scstarget,9,STR)
 	CheckStatGT(scstarget,0,SAVEVSSPELL)
 THEN DO
 	Combine()
@@ -223,7 +220,7 @@
 IF TRIGGER
           IgnoreBlock(IsEnchanterStreamline)
 	TargetBlock(PCsInOrder)
-	TriggerBlock(MR100|SINecromancy|Helpless|Enemy|MinorGlobe|Panic|L2safe)
+	TriggerBlock(MR100|Helpless|Enemy|MinorGlobe|Panic)
 	CheckStatGT(scstarget,7,SAVEVSSPELL)
 THEN DO
 	Combine()
@@ -243,7 +240,7 @@
         IgnoreBlock(IsNecromancerStreamline)
         RequireBlock(HasL2)
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(MR100|SpellTurn|Sleep|SIConjuration|MinorGlobe|Helpless|Enemy)
+	TriggerBlock(MR100|SpellTurnNew|Sleep|MinorGlobe|Helpless|Enemy)
 THEN DO
         Combine()
 	Action(Spell,WIZARD_POWER_WORD_SLEEP|100|200)
