--- stratagems/mage/ssl/bg2/combatblocks/low_level_area.ssl	2018-08-27 01:01:22.000000000 +0200
+++ Fixpack/stratagems/mage/ssl/bg2/combatblocks/low_level_area.ssl	2018-08-27 18:00:49.000000000 +0200
@@ -11,8 +11,7 @@
 	IgnoreBlock(IsEvokerStreamline)
 	TargetBlock(EnemiesInOrder)
 	RequireBlock(HasL3)
-	TriggerBlock(MR100|ResistMagicDamage|SINecromancy|MinorGlobe|Enemy)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(MR100|ResistMagicDamage|MinorGlobe|Enemy)
 	!NumCreatureGT([ENEMY.WEAPON],0)
 THEN DO
 	Action(Spell,WIZARD_SKULL_TRAP|100|100)
@@ -26,8 +25,7 @@
 	IgnoreBlock(IsEvokerStreamline)
 	TargetBlock(EnemiesInOrder)
 	RequireBlock(HasL3)
-	TriggerBlock(MR100|ResistMagicDamage|SINecromancy|MinorGlobe|Enemy)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(MR100|ResistMagicDamage|MinorGlobe|Enemy)
 	!NumCreatureGT([ENEMY.HUMANOID],1)
 	!NumCreatureGT([ENEMY.GIANTHUMANOID],1)
 	!NumCreatureGT([ENEMY.UNDEAD],1)
@@ -89,8 +87,7 @@
 	IgnoreBlock(IsNecromancerStreamline)
 	RequireBlock(HasL3)
 	TargetBlock(EnemiesInOrder)
-	TriggerBlock(MR100|ResistFire|SIEvocation|MinorGlobe|Enemy)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(MR100|ResistFire|MinorGlobe|Enemy)
 THEN DO
 	Action(Spell,WIZARD_FIREBALL|100|100)
 END
@@ -113,8 +110,7 @@
 	IgnoreBlock(IsNecromancerStreamline)
 	RequireBlock(HasL3)
 	TargetBlock(EnemiesInOrder)
-	TriggerBlock(MR100|ResistFire|SIEvocation|MinorGlobe|Enemy)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(MR100|ResistFire|MinorGlobe|Enemy)
 	!NumCreatureGT([ENEMY.HUMANOID],1)
 	!NumCreatureGT([ENEMY.GIANTHUMANOID],1)
 	!NumCreatureGT([ENEMY.MONSTER],1)
@@ -137,8 +133,7 @@
 	RequireBlock(HasL5)
 	Range(scstarget,10)
 	TargetBlock(EnemiesInOrder)
-	TriggerBlock(MR100|ResistCold|SIEvocation|Enemy)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(MR100|ResistCold|Enemy)
 	!NumCreatureGT([ENEMY.HUMANOID],1)
 	!NumCreatureGT([ENEMY.GIANTHUMANOID],1)
 	!NumCreatureGT([ENEMY.MONSTER],1)
@@ -158,8 +153,7 @@
 	IgnoreBlock(IsConjurerStreamline)
 	RequireBlock(HasL4)
 	TargetBlock(EnemiesInOrder)
-	TriggerBlock(MR100|ResistCold|SIEvocation|MinorGlobe|Enemy)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(MR100|ResistCold|Enemy)
 THEN DO
 	Action(Spell,WIZARD_ICE_STORM|200|50)
 END
@@ -172,8 +166,7 @@
 	IgnoreBlock(IsConjurerStreamline)
 	RequireBlock(HasL4)
 	TargetBlock(EnemiesInOrder)
-	TriggerBlock(MR100|ResistCold|SIEvocation|MinorGlobe|Enemy)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(MR100|ResistCold|Enemy)
 	!NumCreatureGT([ENEMY.HUMANOID],1)
 	!NumCreatureGT([ENEMY.GIANTHUMANOID],1)
 	!NumCreatureGT([ENEMY.MONSTER],1)
