--- stratagems/mage/mage.tpa	2018-08-30 07:50:32.454431200 +0200
+++ Fixpack/stratagems/mage/mage.tpa	2018-08-30 07:52:13.689118700 +0200
@@ -144,7 +144,7 @@
       RET value
 BEGIN
       SET value=0
-      PATCH_FOR_EACH skip IN xzar ppsanik BEGIN
+      PATCH_FOR_EACH skip IN xzar ppsanik wm_rhia1 wm_rhia3 BEGIN // BiG World Fixpack added Rhialto from Wild Mage Additions
          PATCH_IF ~%filename%~ STRING_EQUAL_CASE ~%skip%~ BEGIN
             SET value=1
          END
