--- stratagems/melissan/ssl/meliss01.ssl	2016-11-20 18:27:15.450507500 -0500
+++ Fixpack/stratagems/melissan/ssl/meliss01.ssl	2016-11-20 18:27:39.792084600 -0500
@@ -560,7 +560,7 @@
 	Combine()
 	SetGlobalTimer("DMWW_antimagic","LOCALS",18)
 	SetGlobalTimer("timestop_antimagic","LOCALS",24)
-	Action(SpellTimer,WIZARD_PIERCE_SHIELD,pierceshield,18|100|100)
+	Action(SpellTimer,WIZARD_PIERCE_MAGIC,pierceshield,18|100|100)
 END
 
 
@@ -610,7 +610,7 @@
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
 	!Gender(scstarget,SUMMONED)
-	TriggerBlock(Panic|MR100|SIConjuration|Enemy|Helpless)
+	TriggerBlock(Panic|MR100|Enemy|Helpless)
 THEN DO
 	Combine()
 	Action(SpellTimer,WIZARD_NPC_SYMBOL_FEAR,symbol,18|100|100)
@@ -623,8 +623,7 @@
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
 	!Gender(scstarget,SUMMONED)
-	TriggerBlock(Slay|MR|SIConjuration)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(Slay|MR)
 THEN DO
 	Combine()
 	Action(SpellTimer,WIZARD_NPC_SYMBOL_DEATH,symbol,18|100|100)
@@ -638,7 +637,7 @@
 IF TRIGGER
 	TargetBlock(EnemiesInOrderShort)
 	Range(scstarget,10)
-	TriggerBlock(MR|Enemy|Helpless|SIConjuration)
+	TriggerBlock(MR|Enemy|Helpless)
 	Alignment(scstarget,MASK_GOOD)
 	OR(5)
 		Class(scstarget,BARD)
