--- stratagems/melissan/ssl/meliss03.ssl	2016-11-20 18:27:15.452509300 -0500
+++ Fixpack/stratagems/melissan/ssl/meliss03.ssl	2016-11-20 18:27:39.793084700 -0500
@@ -560,11 +560,10 @@
 
 IF TRIGGER
 	TargetBlock(PCsInOrder)
-	OR(5)
+	OR(4)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_TURNING)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_DEFLECTION)
 		CheckStat(scstarget,1,WIZARD_SPELL_TRAP)
-		CheckStatGT(scstarget,0,WIZARD_SPELL_IMMUNITY)
 		CheckStatGT(scstarget,0,MINORGLOBE)
 	TriggerBlock(Enemy|Helpless)
 	!GlobalTimerNotExpired("DMWW_antimagic","LOCALS")
@@ -593,7 +592,7 @@
 	Combine()
 	SetGlobalTimer("DMWW_antimagic","LOCALS",18)
 	SetGlobalTimer("timestop_antimagic","LOCALS",24)
-	Action(SpellNoDec,WIZARD_SPELL_STRIKE|100|100)
+	Action(SpellNoDec,WIZARD_PIERCE_MAGIC|100|100)
 END
 
 
@@ -638,7 +637,6 @@
 IF TRIGGER
 	TargetBlock(PCsInOrder)
 	TriggerBlock(ResistFire|Enemy)
-	!StateCheck(scstarget,STATE_MIRRORIMAGE)
 THEN DO
 	Combine()
 	Action(SpellTimer,WIZARD_DRAGONS_BREATH,bigspell,12|100|200)
@@ -653,9 +651,8 @@
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(PCsInOrderShort)
 	!GlobalTimerNotExpired("DMWWTimestopRunning","GLOBAL")
-	TriggerBlock(SIEvocation|SlashingSafe)
+	TriggerBlock(SlashingSafe)
 	!CheckStatGT(scstarget,1,STONESKINS)
-	TriggerBlock(DirectDamageSafe)
 THEN DO
 	Action(ForceSpellRESTimer,"melis01",bigspell,12|100|100)
 END
@@ -671,7 +668,7 @@
 	OR(2)
 		!GlobalTimerNotExpired("DMWWTimestopRunning","GLOBAL")
 		!GlobalTimerNotExpired("timestopmaze","LOCALS")
-	TriggerBlock(Maze|SpellTurn|MR|Enemy|Helpless|SIConjuration)
+	TriggerBlock(Maze|SpellTurnNew|MR|Enemy|Helpless)
 	Range(scstarget,6)
 	OR(2)
 		InPartySlot(scstarget,1) // isn't Player 1
@@ -687,8 +684,7 @@
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
 	!Gender(scstarget,SUMMONED)
-	TriggerBlock(Slay|MR|SIConjuration)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(Slay|MR|)
 THEN DO
 	Combine()
 	Action(SpellTimer,WIZARD_NPC_SYMBOL_DEATH,symbol,18|100|100)
@@ -702,9 +698,8 @@
 IF TRIGGER
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(MR100|SpellTurn|Slay|SINecromancy|Enemy)
-	TriggerBlock(DirectDamageSafe)
-	CheckStatGT(scstarget,0,SAVEVSSPELL)
+	TriggerBlock(MR100|SpellTurnNew|Slay|Enemy)
+	CheckStatGT(scstarget,0,SAVEVSDEATH)
 THEN DO
 	Action(SpellTimer,WIZARD_FINGER_OF_DEATH,finger,12|150|50)
 END
@@ -717,8 +712,7 @@
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC)&HPPercentLT(scstarget,40))
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(DirectDamageSafe)
-	TriggerBlock(MR100|SIEvocation|MinorGlobe|SpellTurn|Enemy|MagicMissile|L1safe|ResistMagicDamage)
+	TriggerBlock(MR100|MinorGlobe|SpellTurn|Enemy|MagicMissile|ResistMagicDamage)
 THEN DO
 	Combine()
 	Action(SpellNoDec,WIZARD_MAGIC_MISSILE)
