--- stratagems/melissan/ssl/meliss02.ssl	2016-11-20 18:27:15.451508000 -0500
+++ Fixpack/stratagems/melissan/ssl/meliss02.ssl	2016-11-20 18:27:39.792084600 -0500
@@ -543,11 +543,10 @@
 
 IF TRIGGER
 	TargetBlock(PCsInOrder)
-	OR(5)
+	OR(4)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_TURNING)
 		CheckStatGT(scstarget,0,WIZARD_SPELL_DEFLECTION)
 		CheckStat(scstarget,1,WIZARD_SPELL_TRAP)
-		CheckStatGT(scstarget,0,WIZARD_SPELL_IMMUNITY)
 		CheckStatGT(scstarget,0,MINORGLOBE)
 	TriggerBlock(Enemy|Helpless)
 	!GlobalTimerNotExpired("DMWW_antimagic","LOCALS")
@@ -576,7 +575,7 @@
 	Combine()
 	SetGlobalTimer("DMWW_antimagic","LOCALS",18)
 	SetGlobalTimer("timestop_antimagic","LOCALS",24)
-	Action(SpellNoDec,WIZARD_SPELL_STRIKE|100|100)
+	Action(SpellNoDec,WIZARD_PIERCE_MAGIC|100|100)
 END
 
 
@@ -632,8 +631,7 @@
 		HPPercentLT(scstarget,50)
 		Class(scstarget,MAGE)
 		Class(scstarget,MAGE_THIEF)
-	TriggerBlock(MR100|SpellTurn|Slay|SIConjuration|Enemy)
-	!HasItem("clck26",scstarget)
+	TriggerBlock(MR100|SpellTurnNew|Slay|SIEnchantment|Enemy)
 THEN DO
 	Combine()
 	Action(SpellTimer,WIZARD_POWER_WORD_KILL,powerword,12|100|100)
@@ -649,7 +647,7 @@
 	OR(2)
 		!GlobalTimerNotExpired("DMWWTimestopRunning","GLOBAL")
 		!GlobalTimerNotExpired("timestopmaze","LOCALS")
-	TriggerBlock(Maze|SpellTurn|MR|Enemy|Helpless|SIConjuration)
+	TriggerBlock(SpellTurnNew|MR|Enemy|Helpless)
 	Range(scstarget,6)
 	OR(2)
 		InPartySlot(scstarget,1) // isn't Player 1
@@ -663,11 +661,10 @@
 ///////////////////////////////////////////////////////////////////////////
 
 IF TRIGGER
-	ConditionalTargetBlock(PCsPreferringStrong;!CheckStatGT(scstarget,50,RESISTMAGIC))
+	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(MR100|SpellTurn|Slay|SINecromancy|Enemy)
-	TriggerBlock(DirectDamageSafe)
-	CheckStatGT(scstarget,0,SAVEVSSPELL)
+	TriggerBlock(MR100|SpellTurnNew|Slay|Enemy)
+	CheckStatGT(scstarget,0,SAVEVSDEATH)
 THEN DO
 	Action(SpellTimer,WIZARD_FINGER_OF_DEATH,finger,12|150|50)
 END
