--- stratagems/mage/ssl/bg2/combatblocks/mid_level_area.ssl	2016-11-20 18:27:15.437498100 -0500
+++ Fixpack/stratagems/mage/ssl/bg2/combatblocks/mid_level_area.ssl	2016-11-20 18:27:39.782077100 -0500
@@ -5,10 +5,8 @@
 IF TRIGGER
 	IgnoreBlock(Indiscriminate)
 	RequireBlock(HasL5)
-	IgnoreBlock(IsEvoker)
 	TargetBlock(EnemiesInOrder)
-	TriggerBlock(MR|Confused|ConfuseProof|SIEnchantment|Enemy|Helpless)
-	!CheckStatLT(scstarget,0,SAVEVSSPELL)
+	TriggerBlock(MR|Enemy|Helpless)
 THEN DO
 	Combine()
 	Action(SpellArea,WIZARD_CHAOS|150|50)
@@ -17,7 +15,6 @@
 IF TRIGGER
 	IgnoreBlock(Indiscriminate)
 	RequireBlock(HasL5)
-	IgnoreBlock(IsEvoker)
 	StateCheck(Myself,STATE_BLIND)
 	TriggerBlock(PlayersInShortRange)
 THEN DO
@@ -27,10 +24,8 @@
 IF TRIGGER
 	RequireBlock(Indiscriminate)
 	RequireBlock(HasL5)
-	IgnoreBlock(IsEvoker)
 	TargetBlock(EnemiesInOrder)
-	TriggerBlock(MR|Confused|ConfuseProof|SIEnchantment|Helpless)
-	!CheckStatLT(scstarget,0,SAVEVSSPELL)
+	TriggerBlock(MR|Helpless)
 THEN DO
 	Combine()
 	Action(SpellArea,WIZARD_CHAOS|150|50)
@@ -47,8 +42,7 @@
 	RequireBlock(HasL6)
 	IgnoreBlock(IsEnchanter)
 	TargetBlock(EnemiesInOrder)
-	TriggerBlock(Enemy|MR|ResistElectric|SIEvocation)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(Enemy|MR|ResistElectric)
 	slowspell
 THEN DO
 	Combine()
@@ -60,8 +54,7 @@
 	RequireBlock(HasL6)
 	IgnoreBlock(IsEnchanter)
 	TargetBlock(EnemiesInOrder)
-	TriggerBlock(MR|ResistElectric|SIEvocation)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(MR|ResistElectric)
 	slowspell
 THEN DO
 	Combine()
@@ -80,13 +73,12 @@
 	IgnoreBlock(IsEnchanter)
 	IgnoreBlock(IsConjurerStreamline)
 	TargetBlock(EnemiesInOrder)
-	TriggerBlock(Enemy|MR|ResistFire|SIEvocation)
+	TriggerBlock(Enemy|MR|ResistFire)
 	!GlobalTimerNotExpired("DMWWTimestopRunning","GLOBAL")
 	Range(scstarget,10)
 	!NumCreatureGT([ENEMY.HUMANOID],1)
 	!NumCreatureGT([ENEMY.GIANTHUMANOID],1)
 	!NumCreatureGT([ENEMY.MONSTER],1)
-        TriggerBlock(DirectDamageSafe)
 	slowspell
 THEN DO
 	Combine()
@@ -100,10 +92,9 @@
 	IgnoreBlock(IsEnchanter)
 	IgnoreBlock(IsConjurerStreamline)
 	TargetBlock(EnemiesInOrder)
-	TriggerBlock(Enemy|MR|ResistFire|SIEvocation)
+	TriggerBlock(Enemy|MR|ResistFire)
 	!GlobalTimerNotExpired("DMWWTimestopRunning","GLOBAL")
 	Range(scstarget,10)
-        TriggerBlock(DirectDamageSafe)
 	slowspell
 THEN DO
 	Combine()
