--- stratagems/priest/ssl/bg1/combatblocks/miscast_magic.ssl	2018-08-27 01:01:22.000000000 +0200
+++ Fixpack/stratagems/priest/ssl/bg1/combatblocks/miscast_magic.ssl	2018-08-27 18:00:49.000000000 +0200
@@ -1,9 +1,8 @@
-IF TRIGGER
-   RequireBlock(HasL3)
-   IgnoreBlock(Demivrgvs)
-   TargetBlock(PCSpellcasters)
-   TriggerBlock(SpellTurn|Enemy|Disabled|MinorGlobe)
-   !CheckStatGT(scstarget,80,SPELLFAILUREMAGE)
-THEN DO
-     Action(Spell,CLERIC_MISCAST_MAGIC|100|100)
+IF TRIGGER
+   RequireBlock(HasL3)
+   TargetBlock(PCSpellcasters)
+   TriggerBlock(SpellTurnNew|Enemy|Disabled|MinorGlobe)
+   !CheckStatGT(scstarget,80,SPELLFAILUREMAGE)
+THEN DO
+     Action(Spell,CLERIC_MISCAST_MAGIC|100|100)
 END
--- stratagems/priest/ssl/bg2/combatblocks/miscast_magic.ssl	2018-08-27 01:01:22.000000000 +0200
+++ Fixpack/stratagems/priest/ssl/bg2/combatblocks/miscast_magic.ssl	2018-08-27 18:00:49.000000000 +0200
@@ -1,9 +1,8 @@
-IF TRIGGER
-   RequireBlock(HasL3)
-   IgnoreBlock(Demivrgvs)
-   TargetBlock(PCSpellcasters)
-   TriggerBlock(SpellTurn|Enemy|Disabled|MinorGlobe)
-   !CheckStatGT(scstarget,80,SPELLFAILUREMAGE)
-THEN DO
-     Action(Spell,CLERIC_MISCAST_MAGIC|100|100)
+IF TRIGGER
+   RequireBlock(HasL3)
+   TargetBlock(PCSpellcasters)
+   TriggerBlock(SpellTurnNew|Enemy|Disabled|MinorGlobe)
+   !CheckStatGT(scstarget,80,SPELLFAILUREMAGE)
+THEN DO
+     Action(Spell,CLERIC_MISCAST_MAGIC|100|100)
 END
