--- stratagems/priest/ssl/bg2/combatblocks/natures_beauty.ssl	2016-11-20 18:27:15.517554800 -0500
+++ Fixpack/stratagems/priest/ssl/bg2/combatblocks/natures_beauty.ssl	2016-11-20 18:27:39.861133400 -0500
@@ -3,7 +3,7 @@
 	RequireBlock(HasL7)
 	TargetBlock(PCsInOrder)
 	Range(scstarget,5)
-	TriggerBlock(MR100|SIIllusion|Enemy|Helpless)
+	TriggerBlock(MR100|Enemy|Helpless)
 THEN DO
 	Action(Spell,CLERIC_NATURE_BEAUTY|200|50)
 END
