--- stratagems/mage/ssl/bg2/combatblocks/power_word_blind.ssl	2016-11-20 18:27:15.438499000 -0500
+++ Fixpack/stratagems/mage/ssl/bg2/combatblocks/power_word_blind.ssl	2016-11-20 18:27:39.783078000 -0500
@@ -13,7 +13,7 @@
 	TargetBlock(PCsPreferringStrong)
 	TargetBlock(Celestials)
 	!Gender(scstarget,SUMMONED)
-	TriggerBlock(MR|Enemy|SIConjuration|SpellTurn)
+	TriggerBlock(MR|Enemy|SIEnchantment|SpellTurn|SpellTurnNew)
         !GlobalTimerNotExpired("DMWWTimestopRunning","GLOBAL")
 THEN DO
 	Combine()
@@ -27,7 +27,7 @@
         RequireBlock(HasL8)
 	TargetBlock(EnemiesInOrderShort)
 	!Gender(scstarget,SUMMONED)
-	TriggerBlock(MR|SIConjuration|SpellTurn)
+	TriggerBlock(MR|SIEnchantment|SpellTurn|SpellTurnNew)
 	!GlobalTimerNotExpired("DMWWTimestopRunning","GLOBAL")
 THEN DO
 	Combine()
