--- stratagems/mage/ssl/bg2/combatblocks/power_word_blind.ssl	2018-08-21 11:17:54.203198700 +0200
+++ Fixpack/stratagems/mage/ssl/bg2/combatblocks/power_word_blind.ssl	2018-08-21 11:36:59.386034000 +0200
@@ -1,35 +1,35 @@
-
-
-
-////////////////////////////////////////
-////	Power Word: Blind
-///////////////////////////////////////
-
-IF TRIGGER
-	IgnoreBlock(Indiscriminate)
-        IgnoreBlock(IsEvoker)
-        IgnoreBlock(IsNecromancerStreamline)
-        RequireBlock(HasL8)
-	TargetBlock(PCsPreferringStrong)
-	TargetBlock(Celestials)
-	!Gender(scstarget,SUMMONED)
-	TriggerBlock(MR|Enemy|SIConjuration|SpellTurn)
-        !GlobalTimerNotExpired("DMWWTimestopRunning","GLOBAL")
-THEN DO
-	Combine()
-	Action(Spell,WIZARD_POWER_WORD_BLIND|200|100)
-END
-
-IF TRIGGER
-	RequireBlock(Indiscriminate)
-        IgnoreBlock(IsEvoker)
-        IgnoreBlock(IsNecromancerStreamline)
-        RequireBlock(HasL8)
-	TargetBlock(EnemiesInOrderShort)
-	!Gender(scstarget,SUMMONED)
-	TriggerBlock(MR|SIConjuration|SpellTurn)
-	!GlobalTimerNotExpired("DMWWTimestopRunning","GLOBAL")
-THEN DO
-	Combine()
-	Action(Spell,WIZARD_POWER_WORD_BLIND|200|100)
-END
+
+
+
+////////////////////////////////////////
+////	Power Word: Blind
+///////////////////////////////////////
+
+IF TRIGGER
+	IgnoreBlock(Indiscriminate)
+        IgnoreBlock(IsEvoker)
+        IgnoreBlock(IsNecromancerStreamline)
+        RequireBlock(HasL8)
+	TargetBlock(PCsPreferringStrong)
+	TargetBlock(Celestials)
+	!Gender(scstarget,SUMMONED)
+	TriggerBlock(MR|Enemy|SIEnchantment|SpellTurn|SpellTurnNew)
+        !GlobalTimerNotExpired("DMWWTimestopRunning","GLOBAL")
+THEN DO
+	Combine()
+	Action(Spell,WIZARD_POWER_WORD_BLIND|200|100)
+END
+
+IF TRIGGER
+	RequireBlock(Indiscriminate)
+        IgnoreBlock(IsEvoker)
+        IgnoreBlock(IsNecromancerStreamline)
+        RequireBlock(HasL8)
+	TargetBlock(EnemiesInOrderShort)
+	!Gender(scstarget,SUMMONED)
+	TriggerBlock(MR|SIEnchantment|SpellTurn|SpellTurnNew)
+	!GlobalTimerNotExpired("DMWWTimestopRunning","GLOBAL")
+THEN DO
+	Combine()
+	Action(Spell,WIZARD_POWER_WORD_BLIND|200|100)
+END
