--- stratagems/priest/priest_shared.tph	2016-11-20 18:27:15.458513200 -0500
+++ Fixpack/stratagems/priest/priest_shared.tph	2016-09-17 22:15:27.545556800 -0400
@@ -409,8 +409,10 @@
         SET temp=$av(~%i%~)
       END
       PATCH_IF VARIABLE_IS_SET ~enlightened~ BEGIN
-         SET av_5=av_5+1
-         SET av_6=av_6+1
+         PATCH_IF level>22 BEGIN
+         SET av_5=av_5+1 END
+         PATCH_IF level>23 BEGIN
+         SET av_6=av_6+1 END
       END
       FOR(i=0;i<7;i+=1) BEGIN
          SET spell_level=i+1
