--- stratagems/gameplay/realistic_wolves.tpa	2018-08-21 11:17:54.167197600 +0200
+++ Fixpack/stratagems/gameplay/realistic_wolves.tpa	2018-08-21 11:36:59.345035700 +0200
@@ -63,9 +63,9 @@
      ACTION_FOR_EACH area IN Encounter_Plains1 Encounter_Plains2 Encounter_Canyon1 Encounter_Canyon2
                              Encounter_Cliff1 Encounter_Cliff2 Encounter_Grassland1 Encounter_Grassland2 
                              Encounter_Forest1 Encounter_Forest2 Encounter_Road BEGIN
-       OUTER_SPRINT files EVALUATE_BUFFER ~%%area%%.bcs~
+       OUTER_SPRINT files EVALUATE_BUFFER ~%%area%%.are~
        LAF swap_text STR_VAR files
-                             swaps= ~dogwi=>dw#rndwi dogwa=>dw#rndwa %tutu_scriptbg%wolf=>dw#rnwlf~ allow_missing=1
+                             swaps= ~dogwi=>dw#rndwi dogwa=>dw#rndwa %tutu_scriptbg%wolf=>dw#rnwlf~
        END
      END
    END
