--- stratagems/setup-stratagems.tp2	2018-08-30 07:50:32.491430300 +0200
+++ Fixpack/stratagems/setup-stratagems.tp2	2018-08-30 07:52:13.728117600 +0200
@@ -1,6 +1,7 @@
 BACKUP ~stratagems_external/backup/stratagems~
 SUPPORT ~For help troubleshooting installation problems, go to the Sword Coast Stratagems forum at forums.gibberlings3.net.~
-VERSION ~v31 RC2~
+VERSION ~v31 RC2 + kreso's Spell Revisions compatibility~
+
 README ~stratagems/readme-stratagems.html~
 AUTO_EVAL_STRINGS
 
@@ -174,6 +175,7 @@
 
 BEGIN @12 DESIGNATED 2010 GROUP @2
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
+REQUIRE_PREDICATE !FILE_EXISTS ~override/dvimhere.mrk~ @3352 // added by kreso
 
 LAF make_label STR_VAR label=dw#breach END
 
@@ -187,6 +189,7 @@
 
 BEGIN @13 DESIGNATED 2020 GROUP @2
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
+REQUIRE_PREDICATE !FILE_EXISTS ~override/dvimhere.mrk~ @3352 // added by kreso
 
 LAF make_label STR_VAR label=dw#antimagicII END
 
@@ -237,6 +240,7 @@
 ///////////////////////////////////////////////////////////////////////////////
 
 BEGIN @18 DESIGNATED 2050 GROUP @2
+REQUIRE_PREDICATE !FILE_EXISTS_IN_GAME ~dvimhere.mrk~ @3352
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE !FILE_EXISTS ~tobex.dll~ @16045
 REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
@@ -404,6 +408,7 @@
 
 BEGIN @3387 DESIGNATED 2160 GROUP @2
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
+REQUIRE_PREDICATE !FILE_EXISTS ~override/dvimhere.mrk~ @3352 // added by kreso
 REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
 
 
@@ -418,6 +423,7 @@
 
 BEGIN @3392 DESIGNATED 2170 GROUP @2
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
+REQUIRE_PREDICATE !FILE_EXISTS ~override/dvimhere.mrk~ @3352 // added by kreso
 REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
 
 LAF make_label STR_VAR label=dw#truesight_blindness  END
