--- stratagems/setup-stratagems.tp2	2016-11-21 19:01:07.594903100 -0500
+++ Fixpack/stratagems/setup-stratagems.tp2	2016-12-01 15:06:48.248033100 -0500
@@ -1,6 +1,6 @@
 BACKUP ~stratagems_external/backup/stratagems~
 AUTHOR ~For help troubleshooting installation problems, go to the Sword Coast Stratagems forum at forums.gibberlings3.net.~
-VERSION ~v30 BWP fix + K4thos' EET compatibility~
+VERSION ~v30 BWP fix + K4thos' EET compatibility + kreso's Spell Revisions compatibility~
 README ~stratagems/readme-stratagems.html~
 AUTO_EVAL_STRINGS
 
@@ -180,6 +180,7 @@
 
 BEGIN @13 DESIGNATED 2020 GROUP @2
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
+REQUIRE_PREDICATE !FILE_EXISTS ~override/dvimhere.mrk~ @3352 // added by kreso
 
 LAF make_label STR_VAR label=dw#antimagicII END
 
@@ -230,6 +231,7 @@
 ///////////////////////////////////////////////////////////////////////////////
 
 BEGIN @18 DESIGNATED 2050 GROUP @2
+REQUIRE_PREDICATE !FILE_EXISTS_IN_GAME ~dvimhere.mrk~ @3352
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
 REQUIRE_PREDICATE !FILE_EXISTS ~tobex.dll~ @16045
 REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
@@ -411,6 +413,7 @@
 
 BEGIN @3392 DESIGNATED 2170 GROUP @2
 REQUIRE_PREDICATE FILE_EXISTS ~override/dw#setup.mrk~ @8
+REQUIRE_PREDICATE !FILE_EXISTS ~override/dvimhere.mrk~ @3352 // added by kreso
 REQUIRE_PREDICATE !GAME_IS ~tutu tutu_totsc bgee~ @16000
 
 LAF make_label STR_VAR label=dw#truesight_blindness  END
