--- stratagems/tactical_bg2/ssl/shadel.ssl	2018-08-21 11:17:54.250195100 +0200
+++ Fixpack/stratagems/tactical_bg2/ssl/shadel.ssl	2018-08-21 11:36:59.440032400 +0200
@@ -208,7 +208,7 @@
 	TargetBlock(PCsPreferringStrong)
 	TargetBlock(Celestials)
 	!Gender(scstarget,SUMMONED)
-	TriggerBlock(MR|Enemy|SIConjuration)
+	TriggerBlock(MR|Enemy|SIEnchantment)
 THEN DO
 	Combine()
 	Action(Power,WIZARD_POWER_WORD_BLIND,powerword,30|200|100)
@@ -235,8 +235,7 @@
 IF TRIGGER
 	ConditionalTargetBlock(PCsPreferringStrong;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(MR100|SpellTurn|Slay|SINecromancy|Enemy)
-	TriggerBlock(DirectDamageSafe)
+	TriggerBlock(MR100|SpellTurnNew|Slay|Enemy)
 THEN DO
 	Action(Power,WIZARD_FINGER_OF_DEATH,finger,24|150|50)
 END
@@ -249,8 +248,7 @@
 IF TRIGGER
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(EnemiesInOrderShort)
-	TriggerBlock(DirectDamageSafe)
-	TriggerBlock(MR100|SIEvocation|MinorGlobe|SpellTurn|Enemy|L1safe)
+	TriggerBlock(MR100|MinorGlobe|SpellTurnNew|Enemy)
 THEN DO
 	Action(Power,WIZARD_CHROMATIC_ORB,orb,6)
 END
