--- stratagems/mage/ssl/bg1/combatblocks/slow.ssl	2018-08-30 07:50:32.461431600 +0200
+++ Fixpack/stratagems/mage/ssl/bg1/combatblocks/slow.ssl	2018-08-30 07:52:13.697118800 +0200
@@ -1,13 +1,13 @@
-
-IF TRIGGER
-	RequireBlock(HasL3)
-	TargetBlock(PCsPreferringStrong)
-	TriggerBlock(MR100|Slow|Enemy|MinorGlobe|Helpless)
-	!CheckStatLT(scstarget,0,SAVEVSSPELL)
-	!StateCheck(scstarget,STATE_SLOWED)
-THEN DO
-	Combine()
-	Action(SpellArea,WIZARD_SLOW|150|50)
-	Action(Scroll,"%tutu_var%scrl1l"|150|50)
-END
-
+
+IF TRIGGER
+	RequireBlock(HasL3)
+	TargetBlock(PCsPreferringStrong)
+	TriggerBlock(MR100|Slow|Enemy|MinorGlobe|Helpless)
+	!CheckStatLT(scstarget,0,SAVEVSPOLY)
+	!StateCheck(scstarget,STATE_SLOWED)
+THEN DO
+	Combine()
+	Action(SpellArea,WIZARD_SLOW|150|50)
+	Action(Scroll,"%tutu_var%scrl1l"|150|50)
+END
+
