--- stratagems/priest/ssl/bg1/combatblocks/summon_insects.ssl	2018-08-27 01:01:22.000000000 +0200
+++ Fixpack/stratagems/priest/ssl/bg1/combatblocks/summon_insects.ssl	2018-08-27 18:00:49.000000000 +0200
@@ -1,17 +1,17 @@
-
-///////////////////////////////////////////////////////////////
-///	Insect Plague etc
-///////////////////////////////////////////////////////////////
-
-IF TRIGGER
-        IgnoreBlock(IsCleric)
-	RequireBlock(HasL3)
-	IgnoreBlock(SpellsAreBG1)
-	TargetBlock(PCSpellcasters)
-	TargetBlock(PCsInOrderShort)
-	TriggerBlock(MR100Touch|SIConjuration|Enemy|Helpless)
-	!CheckStatGT(scstarget,0,CLERIC_INSECT_PLAGUE)
-	Global("shapeshift","LOCALS",0)
-THEN DO
-	Action(Spell,CLERIC_SUMMON_INSECTS|200|100)
-END
+
+///////////////////////////////////////////////////////////////
+///	Insect Plague etc
+///////////////////////////////////////////////////////////////
+
+IF TRIGGER
+        IgnoreBlock(IsCleric)
+	RequireBlock(HasL3)
+	IgnoreBlock(SpellsAreBG1)
+	TargetBlock(PCSpellcasters)
+	TargetBlock(PCsInOrderShort)
+	TriggerBlock(Enemy|Helpless)
+	!CheckStatGT(scstarget,0,CLERIC_INSECT_PLAGUE)
+	Global("shapeshift","LOCALS",0)
+THEN DO
+	Action(Spell,CLERIC_SUMMON_INSECTS|200|100)
+END
--- stratagems/priest/ssl/bg2/combatblocks/summon_insects.ssl	2018-08-27 01:01:22.000000000 +0200
+++ Fixpack/stratagems/priest/ssl/bg2/combatblocks/summon_insects.ssl	2018-08-27 18:00:49.000000000 +0200
@@ -1,17 +1,17 @@
-
-///////////////////////////////////////////////////////////////
-///	Insect Plague etc
-///////////////////////////////////////////////////////////////
-
-IF TRIGGER
-        IgnoreBlock(IsCleric)
-	RequireBlock(HasL3)
-	IgnoreBlock(SpellsAreBG1)
-	TargetBlock(PCSpellcasters)
-	TargetBlock(PCsInOrderShort)
-	TriggerBlock(MR100|SIConjuration|Enemy|Helpless)
-	!CheckStatGT(scstarget,0,CLERIC_INSECT_PLAGUE)
-	Global("shapeshift","LOCALS",0)
-THEN DO
-	Action(Spell,CLERIC_SUMMON_INSECTS|200|100)
-END
+
+///////////////////////////////////////////////////////////////
+///	Insect Plague etc
+///////////////////////////////////////////////////////////////
+
+IF TRIGGER
+        IgnoreBlock(IsCleric)
+	RequireBlock(HasL3)
+	IgnoreBlock(SpellsAreBG1)
+	TargetBlock(PCSpellcasters)
+	TargetBlock(PCsInOrderShort)
+	TriggerBlock(Enemy|Helpless)
+	!CheckStatGT(scstarget,0,CLERIC_INSECT_PLAGUE)
+	Global("shapeshift","LOCALS",0)
+THEN DO
+	Action(Spell,CLERIC_SUMMON_INSECTS|200|100)
+END
