--- stratagems/priest/ssl/bg1/combatblocks/summon_insects.ssl	2016-11-20 18:27:15.485532100 -0500
+++ Fixpack/stratagems/priest/ssl/bg1/combatblocks/summon_insects.ssl	2016-11-20 18:27:39.823106000 -0500
@@ -9,7 +9,7 @@
 	IgnoreBlock(SpellsAreBG1)
 	TargetBlock(PCSpellcasters)
 	TargetBlock(PCsInOrderShort)
-	TriggerBlock(MR100Touch|SIConjuration|Enemy|Helpless)
+	TriggerBlock(Enemy|Helpless)
 	!CheckStatGT(scstarget,0,CLERIC_INSECT_PLAGUE)
 	Global("shapeshift","LOCALS",0)
 THEN DO
--- stratagems/priest/ssl/bg2/combatblocks/summon_insects.ssl	2016-11-20 18:27:15.519556300 -0500
+++ Fixpack/stratagems/priest/ssl/bg2/combatblocks/summon_insects.ssl	2016-11-20 18:27:39.863134400 -0500
@@ -9,7 +9,7 @@
 	IgnoreBlock(SpellsAreBG1)
 	TargetBlock(PCSpellcasters)
 	TargetBlock(PCsInOrderShort)
-	TriggerBlock(MR100|SIConjuration|Enemy|Helpless)
+	TriggerBlock(Enemy|Helpless)
 	!CheckStatGT(scstarget,0,CLERIC_INSECT_PLAGUE)
 	Global("shapeshift","LOCALS",0)
 THEN DO
