--- stratagems/priest/ssl/bg2/combatblocks/symbol.ssl	2016-11-20 18:27:15.520557200 -0500
+++ Fixpack/stratagems/priest/ssl/bg2/combatblocks/symbol.ssl	2016-11-20 18:27:39.864135700 -0500
@@ -9,7 +9,7 @@
         RequireBlock(HasL7)
 	TargetBlock(EnemiesInOrder)
 	!Gender(scstarget,SUMMONED)
-	TriggerBlock(Stun|MR100|SIConjuration|Enemy|Helpless)
+	TriggerBlock(MR100|Enemy|Helpless)
 THEN DO
 	Action(SpellArea,WIZARD_NPC_SYMBOL_STUN|100|100)
 	Action(SpellAreaReplace,WIZARD_NPC_SYMBOL_STUN,WIZARD_NPC_SYMBOL_FEAR|100|100)
@@ -26,7 +26,7 @@
         RequireBlock(HasL7)
 	TargetBlock(EnemiesInOrder)
 	!Gender(scstarget,SUMMONED)
-	TriggerBlock(Stun|MR100|SIConjuration|Enemy|Helpless)
+	TriggerBlock(MR100|Enemy|Helpless)
 THEN DO
 	Action(SpellArea,WIZARD_NPC_SYMBOL_STUN|100|100)
 END
@@ -37,7 +37,7 @@
         RequireBlock(HasL7)
 	TargetBlock(EnemiesInOrder)
 	!Gender(scstarget,SUMMONED)
-	TriggerBlock(MR100|SIConjuration|Enemy|Helpless)
+	TriggerBlock(MR100|Enemy|Helpless)
 THEN DO
 	Action(SpellArea,WIZARD_NPC_SYMBOL_FEAR|100|100)
 END
@@ -50,7 +50,7 @@
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
 	!Gender(scstarget,SUMMONED)
-	TriggerBlock(Slay|MR|SIConjuration)
+	TriggerBlock(Slay|MR)
 	RequireBlock(HasL7)
 	IgnoreBlock(IsDruid)
 	OR(2)
@@ -60,7 +60,6 @@
 		HPPercentLT(scstarget,50)
 		Class(scstarget,MAGE)
 		Class(scstarget,MAGE_THIEF)
-	TriggerBlock(DirectDamageSafe)
 THEN DO
 	Combine()
 	Action(SpellArea,WIZARD_NPC_SYMBOL_DEATH|150|50)
