--- stratagems/fiend/ssl/tanari.ssl	2016-11-20 18:27:15.386461700 -0500
+++ Fixpack/stratagems/fiend/ssl/tanari.ssl	2016-09-16 22:44:38.163932900 -0400
@@ -99,7 +99,7 @@
 IF TRIGGER
 	ConditionalTargetBlock(PCsInOrder;HPPercentLT(scstarget,30))
 	TargetBlock(PCsInOrderShort)
-	TriggerBlock(Enemy|Disabled|MR|SINecromancy|SpellTurn|Helpless)
+	TriggerBlock(Enemy|Disabled|MR|SpellTurnNew|Helpless)
 	Range(scstarget,10)
 THEN DO
 	Combine()
