--- stratagems/priest/ssl/bg2/meleeblocks/touchattack.ssl	2018-08-30 07:50:32.489430000 +0200
+++ Fixpack/stratagems/priest/ssl/bg2/meleeblocks/touchattack.ssl	2018-08-30 07:52:13.726117700 +0200
@@ -1,30 +1,30 @@
-DEFAULT TRIGGER(OR(2) HasItem("harm",Myself) HasItem("dw#harmn",Myself))
-
-IF TRIGGER
-          RequireBlock(HasL6)
-	Target(WorstAC())
-	TargetBlock(PCsInOrder)
-	TargetBlock(EnemiesInOrderShort)
-	Range(scstarget,10)
-	TriggerBlock(SpellTurn|SINecromancy|MirrorImage|MR)
-	!CheckStat(scstarget,4,WIZARD_PROTECTION_FROM_MAGIC_WEAPONS)
-THEN DO
-	Action(Attack)
-END
-
-DEFAULT TRIGGER(HasItem("slaylive",Myself))
-
-IF TRIGGER
-        IgnoreBlock(IsDruid)
-        RequireBlock(HasL5)
-	Target(WorstAC())
-	TargetBlock(PCsInOrder)
-	TargetBlock(EnemiesInOrderShort)
-	Range(scstarget,10)
-	TriggerBlock(SpellTurn|SINecromancy|MR|MirrorImage|Slay)
-	!CheckStat(scstarget,4,WIZARD_PROTECTION_FROM_MAGIC_WEAPONS)
-THEN DO
-	Action(Attack)
-END
-
-DEFAULT TRIGGER()
+DEFAULT TRIGGER(OR(2) HasItem("harm",Myself) HasItem("dw#harmn",Myself))
+
+IF TRIGGER
+
+	IgnoreBlock(IsDruid)
+    RequireBlock(HasL6)
+	Target(WorstAC())
+	TargetBlock(PCsInOrder)
+	TargetBlock(EnemiesInOrderShort)
+	Range(scstarget,10)
+	TriggerBlock(Slay|MR|SpellTurnNew)
+THEN DO
+	Action(Attack)
+END
+
+DEFAULT TRIGGER(HasItem("slaylive",Myself))
+
+IF TRIGGER
+        IgnoreBlock(IsDruid)
+        RequireBlock(HasL5)
+	Target(WorstAC())
+	TargetBlock(PCsInOrder)
+	TargetBlock(EnemiesInOrderShort)
+	Range(scstarget,10)
+	TriggerBlock(Slay)
+THEN DO
+	Action(Attack)
+END
+
+DEFAULT TRIGGER()
