--- stratagems/fiend/ssl/uddeath.ssl	2018-08-30 07:50:32.446431500 +0200
+++ Fixpack/stratagems/fiend/ssl/uddeath.ssl	2018-08-30 07:52:13.680119300 +0200
@@ -23,7 +23,7 @@
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
 	!Gender(scstarget,SUMMONED)
-	TriggerBlock(Panic|MR100|SIConjuration|Enemy|Helpless)
+	TriggerBlock(Panic|MR100|Enemy|Helpless)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_NPC_SYMBOL_FEAR|100|200)
@@ -36,7 +36,7 @@
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
 	!Gender(scstarget,SUMMONED)
-	TriggerBlock(Slay|MR|SIConjuration)
+	TriggerBlock(Slay|MR)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_NPC_SYMBOL_DEATH|100|200)
@@ -49,8 +49,7 @@
 IF TRIGGER
 	TargetBlock(EnemiesInOrder)
 	!GlobalTimerNotExpired("fireball","LOCALS")
-	TriggerBlock(MR100|ResistFire|SIEvocation|MinorGlobe|Enemy)
-	!HasItem("clck26",scstarget)
+	TriggerBlock(MR100|ResistFire|MinorGlobe|Enemy)
 THEN DO
         Combine()
 	SetGlobalTimer("fireball","LOCALS",1000)
@@ -69,8 +68,7 @@
 		HPPercentLT(scstarget,50)
 		Class(scstarget,MAGE)
 		Class(scstarget,MAGE_THIEF)
-	TriggerBlock(MR100|SpellTurn|Slay|SIConjuration|Enemy)
-	!HasItem("clck26",scstarget)
+	TriggerBlock(MR100|Slay|SIEnchantment|SpellTurnNew|Enemy)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_POWER_WORD_KILL|100|100)
@@ -84,7 +82,7 @@
 IF TRIGGER
 	TargetBlock(PCsPreferringStrong)
 	!Gender(scstarget,SUMMONED)
-	TriggerBlock(MR|Enemy|SIConjuration)
+	TriggerBlock(MR|Enemy|SpellTurnNew)
 THEN DO
 	Combine()
 	Action(Spell,WIZARD_POWER_WORD_BLIND|100|100)
@@ -99,7 +97,7 @@
 
 	ConditionalTargetBlock(PCsPreferringWeak;!CheckStatGT(scstarget,50,RESISTMAGIC))
 	TargetBlock(PCsInOrderShort)
-	TriggerBlock(MR100|SpellTurn|Stun|Enemy|Helpless|SIConjuration)
+	TriggerBlock(MR100|SpellTurnNew|Stun|Enemy|Helpless|SIEnchantment)
 	OR(3)
 		HPPercentLT(scstarget,60)
 		Class(scstarget,MAGE)
