--- stratagems/mage/ssl/bg1/combatblocks/wand_of_frost.ssl	2018-08-27 01:01:22.000000000 +0200
+++ Fixpack/stratagems/mage/ssl/bg1/combatblocks/wand_of_frost.ssl	2018-08-27 18:00:49.000000000 +0200
@@ -1,20 +1,20 @@
-///////////////////////////////////////////////////////////////////
-////	This is the moment to whip out that Wand of Frost...
-///////////////////////////////////////////////////////////////////
-
-IF TRIGGER
-        RequireBlock(EasyOptionAvailable)
-        TriggerBlock(CorePlus)
-	TargetBlock(PCsPreferringWeak)
-	TriggerBlock(Disabled|MR|ResistCold|Enemy|MirrorImage)
-THEN DO
-	Action(Wand,"%tutu_var%wand06"|70|30)
-END
-
-IF TRIGGER
-        IgnoreBlock(EasyOptionAvailable)
-	TargetBlock(PCsPreferringWeak)
-	TriggerBlock(Disabled|MR|ResistCold|Enemy|MirrorImage)
-THEN DO
-	Action(Wand,"%tutu_var%wand06"|70|30)
+///////////////////////////////////////////////////////////////////
+////	This is the moment to whip out that Wand of Frost...
+///////////////////////////////////////////////////////////////////
+
+IF TRIGGER
+        RequireBlock(EasyOptionAvailable)
+        TriggerBlock(CorePlus)
+	TargetBlock(PCsPreferringWeak)
+	TriggerBlock(Disabled|MR|ResistCold|Enemy)
+THEN DO
+	Action(Wand,"%tutu_var%wand06"|70|30)
+END
+
+IF TRIGGER
+        IgnoreBlock(EasyOptionAvailable)
+	TargetBlock(PCsPreferringWeak)
+	TriggerBlock(Disabled|MR|ResistCold|Enemy|MirrorImage)
+THEN DO
+	Action(Wand,"%tutu_var%wand06"|70|30)
 END
