--- amber\tra\english\dialogs.tra	Thu Sep 17 11:33:00 2009
+++ C:\BWP Patchstudio\patched files\amber\tra\english\dialogs.tra	Fri Nov 12 16:30:45 2010
@@ -589,7 +589,7 @@
 @2031 = ~I'll order around whomever I want and whenever I want! You'd better start serving us right about now, before I lose my temper!~
 @2032 = ~GET OUT OF HERE!~
 @2033 = ~Are you daft or something, boy? I said that creature's money is no good here, and just to make it clear to you, that includes her companions as well. Now get lost.~
-@2034 = ~Are you daft or something, <BOYGIRL>? I said that creature's money is no good here...~
+@2034 = ~Are you daft or something, <GIRLBOY>? I said that creature's money is no good here...~
 @2035 = ~But just out of curiosity... What kind of a sum are we talking about here?~
 @2036 = ~I am sure 10 gold coins would do the trick, don't you agree?~
 @2037 = ~How about 50 gold coins? Would that be enough to make you forget this little incident, ma'am?~
@@ -2837,7 +2837,7 @@
 @6247 = ~To tell you the truth, it was a bit tedious - having to come all the way just to save your butt and all for nothing. Friends are useful to have around, but I did it just because <CHARNAME> insisted on it.~
 @6248 = ~Do not take it personally Amber. I understand the value of friends in a tight spot, but my loyalties lie with <CHARNAME> and it's just not wise for me to get too attached to anyone. And besides, I did come after you, didn't I?~
 @6249 = ~I saved you this time as I consider you a valuable ally, but I do not want you to get any ideas. The only reason you are still with us is because <CHARNAME> knows you can pull your load and because you have abilities that we need. Our relationship is strictly business and that is how it is going to stay.~
-@6250 = ~Maybe I should have told <CHARNAME> not to come after you, after all. If all you are going to do is whine and moralise, I don't think <CHARNAME will tolerate that for long. That is not why <PRO_HESHE> took you with us.~
+@6250 = ~Maybe I should have told <CHARNAME> not to come after you, after all. If all you are going to do is whine and moralise, I don't think <CHARNAME> will tolerate that for long. That is not why <PRO_HESHE> took you with us.~
 @6251 = ~ If that is what you want, that is what you are going to get. Just don't expect anyone to show you true commitment in a tight spot unless you are willing to do the same.~ [m#amb108]
 @6252 = ~Well, maybe I should not have come back at all, then. Luckily that mistake is an easy one to correct.~
 @6253 = ~However, I will not ditch <CHARNAME> just because <PRO_HESHE> has made a poor choice in dragging you along.~
