--- amber\dialogs\m#amber.d	Fri Aug 28 11:48:00 2009
+++ C:\BWP Patchstudio\patched files\amber\dialogs\m#amber.d	Sat Nov 28 21:54:44 2015
@@ -7534,7 +7534,7 @@
   IF ~~ THEN REPLY @6554 /* @6554 = ~Blue? I bet the person who wrote that never even saw me, or he would surely have noticed my yellow eyes. I like to think of them as rather striking.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",2)~ GOTO LT1EyesB
   IF ~~ THEN REPLY @6555 /* @6555 = ~Well, I guess you could make that mistake, although in proper lighting, my eyes are quite clearly emerald green.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",3)~ GOTO LT1EyesC
   IF ~~ THEN REPLY @6556 /* @6556 = ~Well, they could have gotten that right just by guessing; half the people in the Sword Coast probably have blue eyes like mine. Based on that description they could have caught almost anyone.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",4)~ GOTO LT1EyesD
-  IF ~~ THEN REPLY @6557 /* @6557 = ~My eyes are actually grey, although in a suitable light, you could surely mistake them for blue.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",5)~ DO ~SetGlobal("M#PCEyeColour","Locals",5)~ GOTO LT1EyesE
+  IF ~~ THEN REPLY @6557 /* @6557 = ~My eyes are actually grey, although in a suitable light, you could surely mistake them for blue.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",5)~ GOTO LT1EyesE
   IF ~~ THEN REPLY @6558 /* @6558 = ~If they had been paying any attention, they would surely have noticed that my eyes were actually violet. Or perhaps lavender in bright daylight.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",6)~ GOTO LT1EyesF
   IF ~~ THEN REPLY @6559 /* @6559 = ~Well, that was pretty far off, considering that my eyes are actually pitch black.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",7)~ GOTO LT1EyesG
   IF ~~ THEN REPLY @6560 /* @6560 = ~Well, I don't blame them for getting confused, since I don't really think there is a way to describe the colour of my eyes. Blue was as good a guess as anything.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",8)~ GOTO LT1EyesH
@@ -9306,7 +9306,7 @@
   IF ~~ THEN REPLY @6554 /* @6554 = ~Blue? I bet the person who wrote that never even saw me, or he would surely have noticed my yellow eyes. I like to think of them as rather striking.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",2)~ GOTO P2LT1EyesB
   IF ~~ THEN REPLY @6555 /* @6555 = ~Well, I guess you could make that mistake, although in proper lighting, my eyes are quite clearly emerald green.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",3)~ GOTO P2LT1EyesC
   IF ~~ THEN REPLY @6556 /* @6556 = ~Well, they could have gotten that right just by guessing; half the people in the Sword Coast probably have blue eyes like mine. Based on that description they could have caught almost anyone.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",4)~ GOTO P2LT1EyesD
-  IF ~~ THEN REPLY @6557 /* @6557 = ~My eyes are actually grey, although in a suitable light, you could surely mistake them for blue.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",5)~ DO ~SetGlobal("M#PCEyeColour","Locals",5)~ GOTO P2LT1EyesE
+  IF ~~ THEN REPLY @6557 /* @6557 = ~My eyes are actually grey, although in a suitable light, you could surely mistake them for blue.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",5)~ GOTO P2LT1EyesE
   IF ~~ THEN REPLY @6558 /* @6558 = ~If they had been paying any attention, they would surely have noticed that my eyes were actually violet. Or perhaps lavender in bright daylight.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",6)~ GOTO P2LT1EyesF
   IF ~~ THEN REPLY @6559 /* @6559 = ~Well, that was pretty far off, considering that my eyes are actually pitch black.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",7)~ GOTO P2LT1EyesG
   IF ~~ THEN REPLY @6560 /* @6560 = ~Well, I don't blame them for getting confused, since I don't really think there is a way to describe the colour of my eyes. Blue was as good a guess as anything.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",8)~ GOTO P2LT1EyesH
@@ -9658,35 +9658,35 @@
   IF ~IsValidForPartyDialog("Imoen2")~ THEN EXTERN IMOEN2J M#P2LT2
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(4)
-    	Class("Player2",FIGHTER)
-    	Class("Player2",PALADIN)
-    	Class("Player2",MONK)
-    	Kit("Player2",BARBARIAN)~ THEN EXTERN HAERDAJ M#P2LT2Hawk
+    	Class(Player2,FIGHTER)
+    	Class(Player2,PALADIN)
+    	Class(Player2,MONK)
+    	Kit(Player2,BARBARIAN)~ THEN EXTERN HAERDAJ M#P2LT2Hawk
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(2)
-    	Class("Player2",BARD)
-    	Class("Player2",THIEF)~ THEN EXTERN HAERDAJ M#P2LT2Mockingbird
+    	Class(Player2,BARD)
+    	Class(Player2,THIEF)~ THEN EXTERN HAERDAJ M#P2LT2Mockingbird
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(3)
-    	Class("Player2",CLERIC)
-    	Class("Player2",DRUID)
-    	Class("Player2",RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Owl
+    	Class(Player2,CLERIC)
+    	Class(Player2,DRUID)
+    	Class(Player2,RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Owl
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(2)
-    	Class("Player2",MAGE)
-    	Class("Player2",SORCERER)~ THEN EXTERN HAERDAJ M#P2LT2Ibis
+    	Class(Player2,MAGE)
+    	Class(Player2,SORCERER)~ THEN EXTERN HAERDAJ M#P2LT2Ibis
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(10)
-	Class("Player2",FIGHTER_MAGE)
-	Class("Player2",FIGHTER_CLERIC)
-	Class("Player2",FIGHTER_THIEF)
-	Class("Player2",FIGHTER_MAGE_THIEF)
-	Class("Player2",MAGE_THIEF)
-	Class("Player2",CLERIC_MAGE)
-	Class("Player2",CLERIC_THIEF)
-	Class("Player2",FIGHTER_DRUID)
-	Class("Player2",FIGHTER_MAGE_CLERIC)
-    	Class("Player2",CLERIC_RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Hound
+	Class(Player2,FIGHTER_MAGE)
+	Class(Player2,FIGHTER_CLERIC)
+	Class(Player2,FIGHTER_THIEF)
+	Class(Player2,FIGHTER_MAGE_THIEF)
+	Class(Player2,MAGE_THIEF)
+	Class(Player2,CLERIC_MAGE)
+	Class(Player2,CLERIC_THIEF)
+	Class(Player2,FIGHTER_DRUID)
+	Class(Player2,FIGHTER_MAGE_CLERIC)
+    	Class(Player2,CLERIC_RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Hound
   IF ~IsValidForPartyDialog("Edwin")~ THEN EXTERN EDWINJ M#P2LT2
   // IF ~IsValidForPartyDialog("Cernd")~ THEN EXTERN CERNDJ M#P2LT2 
   IF ~IsValidForPartyDialog("Anomen")~ THEN EXTERN ANOMENJ M#P2LT2 
@@ -9702,35 +9702,35 @@
   IF ~IsValidForPartyDialog("Edwin")~ THEN EXTERN EDWINJ M#P2LT2
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(4)
-    	Class("Player2",FIGHTER)
-    	Class("Player2",PALADIN)
-    	Class("Player2",MONK)
-    	Kit("Player2",BARBARIAN)~ THEN EXTERN HAERDAJ M#P2LT2Hawk
+    	Class(Player2,FIGHTER)
+    	Class(Player2,PALADIN)
+    	Class(Player2,MONK)
+    	Kit(Player2,BARBARIAN)~ THEN EXTERN HAERDAJ M#P2LT2Hawk
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(2)
-    	Class("Player2",BARD)
-    	Class("Player2",THIEF)~ THEN EXTERN HAERDAJ M#P2LT2Mockingbird
+    	Class(Player2,BARD)
+    	Class(Player2,THIEF)~ THEN EXTERN HAERDAJ M#P2LT2Mockingbird
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(3)
-    	Class("Player2",CLERIC)
-    	Class("Player2",DRUID)
-    	Class("Player2",RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Owl
+    	Class(Player2,CLERIC)
+    	Class(Player2,DRUID)
+    	Class(Player2,RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Owl
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(2)
-    	Class("Player2",MAGE)
-    	Class("Player2",SORCERER)~ THEN EXTERN HAERDAJ M#P2LT2Ibis
+    	Class(Player2,MAGE)
+    	Class(Player2,SORCERER)~ THEN EXTERN HAERDAJ M#P2LT2Ibis
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(10)
-	Class("Player2",FIGHTER_MAGE)
-	Class("Player2",FIGHTER_CLERIC)
-	Class("Player2",FIGHTER_THIEF)
-	Class("Player2",FIGHTER_MAGE_THIEF)
-	Class("Player2",MAGE_THIEF)
-	Class("Player2",CLERIC_MAGE)
-	Class("Player2",CLERIC_THIEF)
-	Class("Player2",FIGHTER_DRUID)
-	Class("Player2",FIGHTER_MAGE_CLERIC)
-    	Class("Player2",CLERIC_RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Hound
+	Class(Player2,FIGHTER_MAGE)
+	Class(Player2,FIGHTER_CLERIC)
+	Class(Player2,FIGHTER_THIEF)
+	Class(Player2,FIGHTER_MAGE_THIEF)
+	Class(Player2,MAGE_THIEF)
+	Class(Player2,CLERIC_MAGE)
+	Class(Player2,CLERIC_THIEF)
+	Class(Player2,FIGHTER_DRUID)
+	Class(Player2,FIGHTER_MAGE_CLERIC)
+    	Class(Player2,CLERIC_RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Hound
   IF ~IsValidForPartyDialog("Imoen2")~ THEN EXTERN IMOEN2J M#P2LT2
   IF ~IsValidForPartyDialog("Jaheira")~ THEN EXTERN JAHEIRAJ M#P2LT2
   IF ~IsValidForPartyDialog("Jan")~ THEN EXTERN JANJ M#P2LT2
@@ -9756,35 +9756,35 @@
   IF ~IsValidForPartyDialog("Yoshimo")~ THEN EXTERN YOSHJ M#P2LT2
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(4)
-    	Class("Player2",FIGHTER)
-    	Class("Player2",PALADIN)
-    	Class("Player2",MONK)
-    	Kit("Player2",BARBARIAN)~ THEN EXTERN HAERDAJ M#P2LT2Hawk
+    	Class(Player2,FIGHTER)
+    	Class(Player2,PALADIN)
+    	Class(Player2,MONK)
+    	Kit(Player2,BARBARIAN)~ THEN EXTERN HAERDAJ M#P2LT2Hawk
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(2)
-    	Class("Player2",BARD)
-    	Class("Player2",THIEF)~ THEN EXTERN HAERDAJ M#P2LT2Mockingbird
+    	Class(Player2,BARD)
+    	Class(Player2,THIEF)~ THEN EXTERN HAERDAJ M#P2LT2Mockingbird
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(3)
-    	Class("Player2",CLERIC)
-    	Class("Player2",DRUID)
-    	Class("Player2",RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Owl
+    	Class(Player2,CLERIC)
+    	Class(Player2,DRUID)
+    	Class(Player2,RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Owl
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(2)
-    	Class("Player2",MAGE)
-    	Class("Player2",SORCERER)~ THEN EXTERN HAERDAJ M#P2LT2Ibis
+    	Class(Player2,MAGE)
+    	Class(Player2,SORCERER)~ THEN EXTERN HAERDAJ M#P2LT2Ibis
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(10)
-	Class("Player2",FIGHTER_MAGE)
-	Class("Player2",FIGHTER_CLERIC)
-	Class("Player2",FIGHTER_THIEF)
-	Class("Player2",FIGHTER_MAGE_THIEF)
-	Class("Player2",MAGE_THIEF)
-	Class("Player2",CLERIC_MAGE)
-	Class("Player2",CLERIC_THIEF)
-	Class("Player2",FIGHTER_DRUID)
-	Class("Player2",FIGHTER_MAGE_CLERIC)
-    	Class("Player2",CLERIC_RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Hound
+	Class(Player2,FIGHTER_MAGE)
+	Class(Player2,FIGHTER_CLERIC)
+	Class(Player2,FIGHTER_THIEF)
+	Class(Player2,FIGHTER_MAGE_THIEF)
+	Class(Player2,MAGE_THIEF)
+	Class(Player2,CLERIC_MAGE)
+	Class(Player2,CLERIC_THIEF)
+	Class(Player2,FIGHTER_DRUID)
+	Class(Player2,FIGHTER_MAGE_CLERIC)
+    	Class(Player2,CLERIC_RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Hound
   IF ~IsValidForPartyDialog("Nalia")~ THEN EXTERN NALIAJ M#P2LT2
   IF ~IsValidForPartyDialog("Imoen2")~ THEN EXTERN IMOEN2J M#P2LT2
   IF ~IsValidForPartyDialog("Anomen")~ THEN EXTERN ANOMENJ M#P2LT2 
@@ -9808,35 +9808,35 @@
   IF ~IsValidForPartyDialog("Nalia")~ THEN EXTERN NALIAJ M#P2LT2
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(4)
-    	Class("Player2",FIGHTER)
-    	Class("Player2",PALADIN)
-    	Class("Player2",MONK)
-    	Kit("Player2",BARBARIAN)~ THEN EXTERN HAERDAJ M#P2LT2Hawk
+    	Class(Player2,FIGHTER)
+    	Class(Player2,PALADIN)
+    	Class(Player2,MONK)
+    	Kit(Player2,BARBARIAN)~ THEN EXTERN HAERDAJ M#P2LT2Hawk
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(2)
-    	Class("Player2",BARD)
-    	Class("Player2",THIEF)~ THEN EXTERN HAERDAJ M#P2LT2Mockingbird
+    	Class(Player2,BARD)
+    	Class(Player2,THIEF)~ THEN EXTERN HAERDAJ M#P2LT2Mockingbird
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(3)
-    	Class("Player2",CLERIC)
-    	Class("Player2",DRUID)
-    	Class("Player2",RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Owl
+    	Class(Player2,CLERIC)
+    	Class(Player2,DRUID)
+    	Class(Player2,RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Owl
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(2)
-    	Class("Player2",MAGE)
-    	Class("Player2",SORCERER)~ THEN EXTERN HAERDAJ M#P2LT2Ibis
+    	Class(Player2,MAGE)
+    	Class(Player2,SORCERER)~ THEN EXTERN HAERDAJ M#P2LT2Ibis
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(10)
-	Class("Player2",FIGHTER_MAGE)
-	Class("Player2",FIGHTER_CLERIC)
-	Class("Player2",FIGHTER_THIEF)
-	Class("Player2",FIGHTER_MAGE_THIEF)
-	Class("Player2",MAGE_THIEF)
-	Class("Player2",CLERIC_MAGE)
-	Class("Player2",CLERIC_THIEF)
-	Class("Player2",FIGHTER_DRUID)
-	Class("Player2",FIGHTER_MAGE_CLERIC)
-    	Class("Player2",CLERIC_RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Hound
+	Class(Player2,FIGHTER_MAGE)
+	Class(Player2,FIGHTER_CLERIC)
+	Class(Player2,FIGHTER_THIEF)
+	Class(Player2,FIGHTER_MAGE_THIEF)
+	Class(Player2,MAGE_THIEF)
+	Class(Player2,CLERIC_MAGE)
+	Class(Player2,CLERIC_THIEF)
+	Class(Player2,FIGHTER_DRUID)
+	Class(Player2,FIGHTER_MAGE_CLERIC)
+    	Class(Player2,CLERIC_RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Hound
   IF ~IsValidForPartyDialog("Minsc")~ THEN EXTERN MINSCJ M#P2LT2
   IF ~IsValidForPartyDialog("Imoen2")~ THEN EXTERN IMOEN2J M#P2LT2
   IF ~IsValidForPartyDialog("Mazzy")~ THEN EXTERN MAZZYJ M#P2LT2
