--- amber/dialogs/m#amber.d	2009-08-28 12:48:00.000000000 -0400
+++ Fixpack/amber/dialogs/m#amber.d	2016-11-21 23:30:47.149800300 -0500
@@ -265,12 +265,12 @@
 IF ~~ THEN BEGIN T1J
   SAY @5025 /* @5025 = ~What? You look as if you still have something to tell me.~ */
   IF ~OR(3)
-        Global("Chapter","GLOBAL",1)
-        Global("Chapter","GLOBAL",2)
-        Global("Chapter","GLOBAL",3)~ THEN REPLY @5026 /* @5026 = ~Before we go you should know that my ultimate goal is to rescue someone who is like a sister to me. Imoen was taken by the Cowled Wizards.~ */ GOTO T1K
+        Global("Chapter","GLOBAL",%bg2_chapter_1%)
+        Global("Chapter","GLOBAL",%bg2_chapter_2%)
+        Global("Chapter","GLOBAL",%bg2_chapter_3%)~ THEN REPLY @5026 /* @5026 = ~Before we go you should know that my ultimate goal is to rescue someone who is like a sister to me. Imoen was taken by the Cowled Wizards.~ */ GOTO T1K
   IF ~~ THEN REPLY @5027 /* @5027 = ~Before we go you should know that my ultimate goal is to track down an evil wizard named Irenicus.~ */ GOTO T1L
-  IF ~GlobalGT("Chapter","GLOBAL",3)~ THEN REPLY @5028 /* @5028 = ~Before we go you should know that my ultimate goal is to track down an evil wizard named Irenicus, who is in possession of something that is very important to me.~ */ GOTO T1L
-  IF ~Global("Chapter","GLOBAL",6)
+  IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_3%)~ THEN REPLY @5028 /* @5028 = ~Before we go you should know that my ultimate goal is to track down an evil wizard named Irenicus, who is in possession of something that is very important to me.~ */ GOTO T1L
+  IF ~Global("Chapter","GLOBAL",%bg2_chapter_6%)
       !Dead("c6bodhi")~ THEN REPLY @5029 /* @5029 = ~Before we go, you should know that I am currently tracking down a powerful vampire known by the name of Bodhi.~ */ GOTO T1Q
   IF ~Gender(Player2,MALE)	
       CheckStatGT(Player2,11,CHR)
@@ -1311,9 +1311,9 @@
 IF ~~ THEN BEGIN 64
   SAY @5201 /* @5201 = ~No <CHARNAME>, I will not give up my bracelet! The sword is fine as it is.~ */
   IF ~CheckStatGT(Player1,14,CHR)
-      GlobalGT("CHAPTER","GLOBAL",5)~ THEN REPLY @5202 /* @5202 = ~Amber, you will not be any less of a friend to Jill even if that stone is set back in the sword. Memories like that never fade. You do not need some trinket to remember her by.~ */ GOTO 66
+      GlobalGT("CHAPTER","GLOBAL",%bg2_chapter_5%)~ THEN REPLY @5202 /* @5202 = ~Amber, you will not be any less of a friend to Jill even if that stone is set back in the sword. Memories like that never fade. You do not need some trinket to remember her by.~ */ GOTO 66
   IF ~CheckStatLT(Player1,15,CHR)
-      GlobalGT("CHAPTER","GLOBAL",5)~ THEN REPLY @5202 /* @5202 = ~Amber, you will not be any less of a friend to Jill even if that stone is set back in the sword. Memories like that never fade. You do not need some trinket to remember her by.~ */ GOTO 65
+      GlobalGT("CHAPTER","GLOBAL",%bg2_chapter_5%)~ THEN REPLY @5202 /* @5202 = ~Amber, you will not be any less of a friend to Jill even if that stone is set back in the sword. Memories like that never fade. You do not need some trinket to remember her by.~ */ GOTO 65
   IF ~~ THEN REPLY @5203 /* @5203 = ~Surrender that bracelet now, Amber. This is no time for such foolishness.~ */ GOTO 65
   IF ~~ THEN REPLY @5204 /* @5204 = ~All right Amber, I'll respect that. Let the sword remain as it is. Do I have anything else that would be of interest?~ */ DO ~ActionOverride("M#Amber",SetDialog("m#amberj")~ EXTERN WSMITH01 M#9
 END
@@ -1687,33 +1687,33 @@
     !StateCheck(Player1,STATE_SILENCED)~ THEN BEGIN T7Begin
   SAY @5304 /* @5304 = ~So, we got your relationship with your brother sorted out, but what about Imoen? You grew up with her didn't you?~ */ 
   IF ~OR(3)
-        Global("Chapter","GLOBAL",2)
-        Global("Chapter","GLOBAL",3)
-        Global("Chapter","GLOBAL",4)~ THEN REPLY @5305 /* @5305 = ~Yes, Imoen has always been like a little sister to me. Her courage and spirit have always been encouraging even if she doesn't always take things seriously enough.~ */ GOTO T7A
+        Global("Chapter","GLOBAL",%bg2_chapter_2%)
+        Global("Chapter","GLOBAL",%bg2_chapter_3%)
+        Global("Chapter","GLOBAL",%bg2_chapter_4%)~ THEN REPLY @5305 /* @5305 = ~Yes, Imoen has always been like a little sister to me. Her courage and spirit have always been encouraging even if she doesn't always take things seriously enough.~ */ GOTO T7A
   IF ~OR(3)
-        Global("Chapter","GLOBAL",5)
-        Global("Chapter","GLOBAL",6)
-        Global("Chapter","GLOBAL",7)
+        Global("Chapter","GLOBAL",%bg2_chapter_5%)
+        Global("Chapter","GLOBAL",%bg2_chapter_6%)
+        Global("Chapter","GLOBAL",%bg2_chapter_7%)
       InPartyAllowDead("Imoen2")~ THEN REPLY @5305 /* @5305 = ~Yes, Imoen has always been like a little sister to me. Her courage and spirit have always been encouraging even if she doesn't always take things seriously enough.~ */ GOTO T7B
   IF ~OR(3)
-        Global("Chapter","GLOBAL",5)
-        Global("Chapter","GLOBAL",6)
-        Global("Chapter","GLOBAL",7)
+        Global("Chapter","GLOBAL",%bg2_chapter_5%)
+        Global("Chapter","GLOBAL",%bg2_chapter_6%)
+        Global("Chapter","GLOBAL",%bg2_chapter_7%)
       !InPartyAllowDead("Imoen2")~ THEN REPLY @5305 /* @5305 = ~Yes, Imoen has always been like a little sister to me. Her courage and spirit have always been encouraging even if she doesn't always take things seriously enough.~ */ DO ~IncrementGlobal("M#AmberLike","Locals",-1)~ GOTO T7C
   IF ~~ THEN REPLY @5306 /* @5306 = ~Yes, she lived in Candlekeep, but we're not as close as she likes to imply.~ */ DO ~IncrementGlobal("M#AmberLike","Locals",-1)~ GOTO T7D
   IF ~OR(3)
-        Global("Chapter","GLOBAL",2)
-        Global("Chapter","GLOBAL",3)
-        Global("Chapter","GLOBAL",4)~ THEN REPLY @5307 /* @5307 = ~Yes, we grew up together and yes, we are friends.~ */ GOTO T7A
+        Global("Chapter","GLOBAL",%bg2_chapter_2%)
+        Global("Chapter","GLOBAL",%bg2_chapter_3%)
+        Global("Chapter","GLOBAL",%bg2_chapter_4%)~ THEN REPLY @5307 /* @5307 = ~Yes, we grew up together and yes, we are friends.~ */ GOTO T7A
   IF ~OR(3)
-        Global("Chapter","GLOBAL",5)
-        Global("Chapter","GLOBAL",6)
-        Global("Chapter","GLOBAL",7)
+        Global("Chapter","GLOBAL",%bg2_chapter_5%)
+        Global("Chapter","GLOBAL",%bg2_chapter_6%)
+        Global("Chapter","GLOBAL",%bg2_chapter_7%)
       InPartyAllowDead("Imoen2")~ THEN REPLY @5307 /* @5307 = ~Yes, we grew up together and yes, we are friends.~ */ GOTO T7B
   IF ~OR(3)
-        Global("Chapter","GLOBAL",5)
-        Global("Chapter","GLOBAL",6)
-        Global("Chapter","GLOBAL",7)
+        Global("Chapter","GLOBAL",%bg2_chapter_5%)
+        Global("Chapter","GLOBAL",%bg2_chapter_6%)
+        Global("Chapter","GLOBAL",%bg2_chapter_7%)
       !InPartyAllowDead("Imoen2")~ THEN REPLY @5307 /* @5307 = ~Yes, we grew up together and yes, we are friends.~ */ DO ~IncrementGlobal("M#AmberLike","Locals",-1)~ GOTO T7C
 END
 
@@ -1721,124 +1721,124 @@
   SAY @5308 /* @5308 = ~Oh, is that so? Can I ask why? You seem to take her friendship quite lightly.~ */
   IF ~~ THEN REPLY @5309 /* @5309 = ~I've known her since childhood, but I guess the trials along the road exposed our differences. She refuses to take anything seriously, and that irritates me.~ */ GOTO T7I
   IF ~OR(3)
-        Global("Chapter","GLOBAL",2)
-        Global("Chapter","GLOBAL",3)
-        Global("Chapter","GLOBAL",4)~ THEN REPLY @5310 /* @5310 = ~Since I left Candlekeep, I have met people who have become much closer to me than Imoen ever was. It's not that she is a bad person, and I do wish the best for her, but after we rescue her, our paths shall separate.~ */ GOTO T7J
+        Global("Chapter","GLOBAL",%bg2_chapter_2%)
+        Global("Chapter","GLOBAL",%bg2_chapter_3%)
+        Global("Chapter","GLOBAL",%bg2_chapter_4%)~ THEN REPLY @5310 /* @5310 = ~Since I left Candlekeep, I have met people who have become much closer to me than Imoen ever was. It's not that she is a bad person, and I do wish the best for her, but after we rescue her, our paths shall separate.~ */ GOTO T7J
   IF ~OR(3)
-        Global("Chapter","GLOBAL",5)
-        Global("Chapter","GLOBAL",6)
-        Global("Chapter","GLOBAL",7)~ THEN REPLY @5311 /* @5311 = ~Since I left Candlekeep, I have met people who have become much closer to me than Imoen ever was. It's not that she is a bad person, and I do wish the best for her, but I think we are better off going our separate paths.~ */ GOTO T7J
+        Global("Chapter","GLOBAL",%bg2_chapter_5%)
+        Global("Chapter","GLOBAL",%bg2_chapter_6%)
+        Global("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN REPLY @5311 /* @5311 = ~Since I left Candlekeep, I have met people who have become much closer to me than Imoen ever was. It's not that she is a bad person, and I do wish the best for her, but I think we are better off going our separate paths.~ */ GOTO T7J
   IF ~OR(3)
-        Global("Chapter","GLOBAL",2)
-        Global("Chapter","GLOBAL",3)
-        Global("Chapter","GLOBAL",4)~ THEN REPLY @5312 /* @5312 = ~I feel naught but pity for that silly brat. And I will risk my life and that of my companions to rescue her. What else could one ask of me?~ */ GOTO T7K
+        Global("Chapter","GLOBAL",%bg2_chapter_2%)
+        Global("Chapter","GLOBAL",%bg2_chapter_3%)
+        Global("Chapter","GLOBAL",%bg2_chapter_4%)~ THEN REPLY @5312 /* @5312 = ~I feel naught but pity for that silly brat. And I will risk my life and that of my companions to rescue her. What else could one ask of me?~ */ GOTO T7K
   IF ~OR(3)
-        Global("Chapter","GLOBAL",5)
-        Global("Chapter","GLOBAL",6)
-        Global("Chapter","GLOBAL",7)~ THEN REPLY @5313 /* @5313 = ~I feel naught but pity for that silly brat. And I did risk my life and that of my companions to rescue her. What else could one ask of me?~ */ GOTO T7K
+        Global("Chapter","GLOBAL",%bg2_chapter_5%)
+        Global("Chapter","GLOBAL",%bg2_chapter_6%)
+        Global("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN REPLY @5313 /* @5313 = ~I feel naught but pity for that silly brat. And I did risk my life and that of my companions to rescue her. What else could one ask of me?~ */ GOTO T7K
   IF ~~ THEN REPLY @5314 /* @5314 = ~Oh, please. I don't give a damn about that stupid girl. She can blame only herself for getting into all this trouble. She should have stayed in Candlekeep washing dishes for Winthrop.~ */ GOTO T7L
   IF ~OR(3)
-        Global("Chapter","GLOBAL",2)
-        Global("Chapter","GLOBAL",3)
-        Global("Chapter","GLOBAL",4)~ THEN REPLY @5315 /* @5315 = ~To be honest, I'm glad that she is gone. At last I can get some respite from her. The way she always trails after me like a lost puppy - there is just no way to shake her off!~ */ GOTO T7L
+        Global("Chapter","GLOBAL",%bg2_chapter_2%)
+        Global("Chapter","GLOBAL",%bg2_chapter_3%)
+        Global("Chapter","GLOBAL",%bg2_chapter_4%)~ THEN REPLY @5315 /* @5315 = ~To be honest, I'm glad that she is gone. At last I can get some respite from her. The way she always trails after me like a lost puppy - there is just no way to shake her off!~ */ GOTO T7L
   IF ~OR(3)
-        Global("Chapter","GLOBAL",5)
-        Global("Chapter","GLOBAL",6)
-        Global("Chapter","GLOBAL",7)~ THEN REPLY @5316 /* @5316 = ~To be honest, I'm glad that she spent all that time stranded in Spellhold. Gave me some long-awaited respite from her. The way she always trails after me like a lost puppy - there is just no way to shake her off!~ */ GOTO T7L
+        Global("Chapter","GLOBAL",%bg2_chapter_5%)
+        Global("Chapter","GLOBAL",%bg2_chapter_6%)
+        Global("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN REPLY @5316 /* @5316 = ~To be honest, I'm glad that she spent all that time stranded in Spellhold. Gave me some long-awaited respite from her. The way she always trails after me like a lost puppy - there is just no way to shake her off!~ */ GOTO T7L
   IF ~OR(3)
-        Global("Chapter","GLOBAL",2)
-        Global("Chapter","GLOBAL",3)
-        Global("Chapter","GLOBAL",4)~ THEN REPLY @5317 /* @5317 = ~She opened my cage during the attack on Irenicus' dungeon. I suppose I owe her for that, but that's it. After we find her, she'll be on her own again.~ */ GOTO T7M
+        Global("Chapter","GLOBAL",%bg2_chapter_2%)
+        Global("Chapter","GLOBAL",%bg2_chapter_3%)
+        Global("Chapter","GLOBAL",%bg2_chapter_4%)~ THEN REPLY @5317 /* @5317 = ~She opened my cage during the attack on Irenicus' dungeon. I suppose I owe her for that, but that's it. After we find her, she'll be on her own again.~ */ GOTO T7M
   IF ~~ THEN REPLY @5318 /* @5318 = ~I really don't want to discuss my relationship with Imoen with anybody.~ */ GOTO T7O
 END
 
 IF ~~ THEN BEGIN T7I
   SAY @5319 /* @5319 = ~So light-heartedness ticks you off? I'll have to keep that in mind.~ */
   IF ~OR(3)
-        Global("Chapter","GLOBAL",2)
-        Global("Chapter","GLOBAL",3)
-        Global("Chapter","GLOBAL",4)~ THEN GOTO T7N
+        Global("Chapter","GLOBAL",%bg2_chapter_2%)
+        Global("Chapter","GLOBAL",%bg2_chapter_3%)
+        Global("Chapter","GLOBAL",%bg2_chapter_4%)~ THEN GOTO T7N
   IF ~OR(3)
-        Global("Chapter","GLOBAL",5)
-        Global("Chapter","GLOBAL",6)
-        Global("Chapter","GLOBAL",7)
+        Global("Chapter","GLOBAL",%bg2_chapter_5%)
+        Global("Chapter","GLOBAL",%bg2_chapter_6%)
+        Global("Chapter","GLOBAL",%bg2_chapter_7%)
       InPartyAllowDead("Imoen2")~ THEN GOTO T7N1
   IF ~OR(3)
-        Global("Chapter","GLOBAL",5)
-        Global("Chapter","GLOBAL",6)
-        Global("Chapter","GLOBAL",7)
+        Global("Chapter","GLOBAL",%bg2_chapter_5%)
+        Global("Chapter","GLOBAL",%bg2_chapter_6%)
+        Global("Chapter","GLOBAL",%bg2_chapter_7%)
       !InPartyAllowDead("Imoen2")~ THEN GOTO T7N2
 END
 
 IF ~~ THEN BEGIN T7J
   SAY @5320 /* @5320 = ~That may well be true, but I would not cast a friend aside so lightly. Especially someone who knows what you are and still remains by your side.~ */
   IF ~OR(3)
-        Global("Chapter","GLOBAL",2)
-        Global("Chapter","GLOBAL",3)
-        Global("Chapter","GLOBAL",4)~ THEN GOTO T7N
+        Global("Chapter","GLOBAL",%bg2_chapter_2%)
+        Global("Chapter","GLOBAL",%bg2_chapter_3%)
+        Global("Chapter","GLOBAL",%bg2_chapter_4%)~ THEN GOTO T7N
   IF ~OR(3)
-        Global("Chapter","GLOBAL",5)
-        Global("Chapter","GLOBAL",6)
-        Global("Chapter","GLOBAL",7)
+        Global("Chapter","GLOBAL",%bg2_chapter_5%)
+        Global("Chapter","GLOBAL",%bg2_chapter_6%)
+        Global("Chapter","GLOBAL",%bg2_chapter_7%)
       InPartyAllowDead("Imoen2")~ THEN GOTO T7N1
   IF ~OR(3)
-        Global("Chapter","GLOBAL",5)
-        Global("Chapter","GLOBAL",6)
-        Global("Chapter","GLOBAL",7)
+        Global("Chapter","GLOBAL",%bg2_chapter_5%)
+        Global("Chapter","GLOBAL",%bg2_chapter_6%)
+        Global("Chapter","GLOBAL",%bg2_chapter_7%)
       !InPartyAllowDead("Imoen2")~ THEN GOTO T7N2
 END
 
 IF ~~ THEN BEGIN T7K
   SAY @5321 /* @5321 = ~Some respect for her affection?~ */
   IF ~OR(3)
-        Global("Chapter","GLOBAL",2)
-        Global("Chapter","GLOBAL",3)
-        Global("Chapter","GLOBAL",4)~ THEN GOTO T7N
+        Global("Chapter","GLOBAL",%bg2_chapter_2%)
+        Global("Chapter","GLOBAL",%bg2_chapter_3%)
+        Global("Chapter","GLOBAL",%bg2_chapter_4%)~ THEN GOTO T7N
   IF ~OR(3)
-        Global("Chapter","GLOBAL",5)
-        Global("Chapter","GLOBAL",6)
-        Global("Chapter","GLOBAL",7)
+        Global("Chapter","GLOBAL",%bg2_chapter_5%)
+        Global("Chapter","GLOBAL",%bg2_chapter_6%)
+        Global("Chapter","GLOBAL",%bg2_chapter_7%)
       InPartyAllowDead("Imoen2")~ THEN GOTO T7N1
   IF ~OR(3)
-        Global("Chapter","GLOBAL",5)
-        Global("Chapter","GLOBAL",6)
-        Global("Chapter","GLOBAL",7)
+        Global("Chapter","GLOBAL",%bg2_chapter_5%)
+        Global("Chapter","GLOBAL",%bg2_chapter_6%)
+        Global("Chapter","GLOBAL",%bg2_chapter_7%)
       !InPartyAllowDead("Imoen2")~ THEN GOTO T7N2
 END
 
 IF ~~ THEN BEGIN T7L
   SAY @5322 /* @5322 = ~I see that you're not very different from everyone else, then. But I guess that's the way of the world: if you can't make it on your own, don't count on anyone helping you out. It's a pity, really.~ */
   IF ~OR(3)
-        Global("Chapter","GLOBAL",2)
-        Global("Chapter","GLOBAL",3)
-        Global("Chapter","GLOBAL",4)~ THEN GOTO T7N
+        Global("Chapter","GLOBAL",%bg2_chapter_2%)
+        Global("Chapter","GLOBAL",%bg2_chapter_3%)
+        Global("Chapter","GLOBAL",%bg2_chapter_4%)~ THEN GOTO T7N
   IF ~OR(3)
-        Global("Chapter","GLOBAL",5)
-        Global("Chapter","GLOBAL",6)
-        Global("Chapter","GLOBAL",7)
+        Global("Chapter","GLOBAL",%bg2_chapter_5%)
+        Global("Chapter","GLOBAL",%bg2_chapter_6%)
+        Global("Chapter","GLOBAL",%bg2_chapter_7%)
       InPartyAllowDead("Imoen2")~ THEN GOTO T7N1
   IF ~OR(3)
-        Global("Chapter","GLOBAL",5)
-        Global("Chapter","GLOBAL",6)
-        Global("Chapter","GLOBAL",7)
+        Global("Chapter","GLOBAL",%bg2_chapter_5%)
+        Global("Chapter","GLOBAL",%bg2_chapter_6%)
+        Global("Chapter","GLOBAL",%bg2_chapter_7%)
       !InPartyAllowDead("Imoen2")~ THEN GOTO T7N2
 END
 
 IF ~~ THEN BEGIN T7M
   SAY @5323 /* @5323 = ~So, it's all about getting even for you? I guess that's your reason for going after this Irenicus, too. Funny. Personally, I have rarely found revenge to be rewarding to any degree.~ */
   IF ~OR(3)
-        Global("Chapter","GLOBAL",2)
-        Global("Chapter","GLOBAL",3)
-        Global("Chapter","GLOBAL",4)~ THEN GOTO T7N
+        Global("Chapter","GLOBAL",%bg2_chapter_2%)
+        Global("Chapter","GLOBAL",%bg2_chapter_3%)
+        Global("Chapter","GLOBAL",%bg2_chapter_4%)~ THEN GOTO T7N
   IF ~OR(3)
-        Global("Chapter","GLOBAL",5)
-        Global("Chapter","GLOBAL",6)
-        Global("Chapter","GLOBAL",7)
+        Global("Chapter","GLOBAL",%bg2_chapter_5%)
+        Global("Chapter","GLOBAL",%bg2_chapter_6%)
+        Global("Chapter","GLOBAL",%bg2_chapter_7%)
       InPartyAllowDead("Imoen2")~ THEN GOTO T7N1
   IF ~OR(3)
-        Global("Chapter","GLOBAL",5)
-        Global("Chapter","GLOBAL",6)
-        Global("Chapter","GLOBAL",7)
+        Global("Chapter","GLOBAL",%bg2_chapter_5%)
+        Global("Chapter","GLOBAL",%bg2_chapter_6%)
+        Global("Chapter","GLOBAL",%bg2_chapter_7%)
       !InPartyAllowDead("Imoen2")~ THEN GOTO T7N2
 END
 
@@ -7534,7 +7534,7 @@
   IF ~~ THEN REPLY @6554 /* @6554 = ~Blue? I bet the person who wrote that never even saw me, or he would surely have noticed my yellow eyes. I like to think of them as rather striking.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",2)~ GOTO LT1EyesB
   IF ~~ THEN REPLY @6555 /* @6555 = ~Well, I guess you could make that mistake, although in proper lighting, my eyes are quite clearly emerald green.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",3)~ GOTO LT1EyesC
   IF ~~ THEN REPLY @6556 /* @6556 = ~Well, they could have gotten that right just by guessing; half the people in the Sword Coast probably have blue eyes like mine. Based on that description they could have caught almost anyone.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",4)~ GOTO LT1EyesD
-  IF ~~ THEN REPLY @6557 /* @6557 = ~My eyes are actually grey, although in a suitable light, you could surely mistake them for blue.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",5)~ DO ~SetGlobal("M#PCEyeColour","Locals",5)~ GOTO LT1EyesE
+  IF ~~ THEN REPLY @6557 /* @6557 = ~My eyes are actually grey, although in a suitable light, you could surely mistake them for blue.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",5)~ GOTO LT1EyesE
   IF ~~ THEN REPLY @6558 /* @6558 = ~If they had been paying any attention, they would surely have noticed that my eyes were actually violet. Or perhaps lavender in bright daylight.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",6)~ GOTO LT1EyesF
   IF ~~ THEN REPLY @6559 /* @6559 = ~Well, that was pretty far off, considering that my eyes are actually pitch black.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",7)~ GOTO LT1EyesG
   IF ~~ THEN REPLY @6560 /* @6560 = ~Well, I don't blame them for getting confused, since I don't really think there is a way to describe the colour of my eyes. Blue was as good a guess as anything.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",8)~ GOTO LT1EyesH
@@ -8953,19 +8953,19 @@
 IF ~~ THEN BEGIN LT12A
   SAY @6899 /* @6899 = ~Not much of a romantic, are you? Well, I suppose that someone in your situation cannot afford to be too sentimental.~ */
   IF ~~ THEN GOTO LT12D
-  IF ~GlobalGT("CHAPTER","GLOBAL",3)~ THEN GOTO LT12D1
+  IF ~GlobalGT("CHAPTER","GLOBAL",%bg2_chapter_3%)~ THEN GOTO LT12D1
 END
 
 IF ~~ THEN BEGIN LT12B
   SAY @6900 /* @6900 = ~I wonder if it's just something we have been conditioned to feel by the armies of poets who have written countless sad and longing verses about the sea. Or is it somehow a feeling that we all naturally share?~ */
   IF ~~ THEN GOTO LT12D
-  IF ~GlobalGT("CHAPTER","GLOBAL",3)~ THEN GOTO LT12D1
+  IF ~GlobalGT("CHAPTER","GLOBAL",%bg2_chapter_3%)~ THEN GOTO LT12D1
 END
 
 IF ~~ THEN BEGIN LT12C
   SAY @6901 /* @6901 = ~That's certainly one way of seeing it. I must admit that I find your optimism inspiring, <GABBER>.~ */
   IF ~~ THEN GOTO LT12D
-  IF ~GlobalGT("CHAPTER","GLOBAL",3)~ THEN GOTO LT12D1
+  IF ~GlobalGT("CHAPTER","GLOBAL",%bg2_chapter_3%)~ THEN GOTO LT12D1
 END
 
 IF ~~ THEN BEGIN LT12D
@@ -9306,7 +9306,7 @@
   IF ~~ THEN REPLY @6554 /* @6554 = ~Blue? I bet the person who wrote that never even saw me, or he would surely have noticed my yellow eyes. I like to think of them as rather striking.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",2)~ GOTO P2LT1EyesB
   IF ~~ THEN REPLY @6555 /* @6555 = ~Well, I guess you could make that mistake, although in proper lighting, my eyes are quite clearly emerald green.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",3)~ GOTO P2LT1EyesC
   IF ~~ THEN REPLY @6556 /* @6556 = ~Well, they could have gotten that right just by guessing; half the people in the Sword Coast probably have blue eyes like mine. Based on that description they could have caught almost anyone.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",4)~ GOTO P2LT1EyesD
-  IF ~~ THEN REPLY @6557 /* @6557 = ~My eyes are actually grey, although in a suitable light, you could surely mistake them for blue.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",5)~ DO ~SetGlobal("M#PCEyeColour","Locals",5)~ GOTO P2LT1EyesE
+  IF ~~ THEN REPLY @6557 /* @6557 = ~My eyes are actually grey, although in a suitable light, you could surely mistake them for blue.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",5)~ GOTO P2LT1EyesE
   IF ~~ THEN REPLY @6558 /* @6558 = ~If they had been paying any attention, they would surely have noticed that my eyes were actually violet. Or perhaps lavender in bright daylight.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",6)~ GOTO P2LT1EyesF
   IF ~~ THEN REPLY @6559 /* @6559 = ~Well, that was pretty far off, considering that my eyes are actually pitch black.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",7)~ GOTO P2LT1EyesG
   IF ~~ THEN REPLY @6560 /* @6560 = ~Well, I don't blame them for getting confused, since I don't really think there is a way to describe the colour of my eyes. Blue was as good a guess as anything.~ */ DO ~SetGlobal("M#PCEyeColour","Locals",8)~ GOTO P2LT1EyesH
@@ -9658,35 +9658,35 @@
   IF ~IsValidForPartyDialog("Imoen2")~ THEN EXTERN IMOEN2J M#P2LT2
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(4)
-    	Class("Player2",FIGHTER)
-    	Class("Player2",PALADIN)
-    	Class("Player2",MONK)
-    	Kit("Player2",BARBARIAN)~ THEN EXTERN HAERDAJ M#P2LT2Hawk
+    	Class(Player2,FIGHTER)
+    	Class(Player2,PALADIN)
+    	Class(Player2,MONK)
+    	Kit(Player2,BARBARIAN)~ THEN EXTERN HAERDAJ M#P2LT2Hawk
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(2)
-    	Class("Player2",BARD)
-    	Class("Player2",THIEF)~ THEN EXTERN HAERDAJ M#P2LT2Mockingbird
+    	Class(Player2,BARD)
+    	Class(Player2,THIEF)~ THEN EXTERN HAERDAJ M#P2LT2Mockingbird
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(3)
-    	Class("Player2",CLERIC)
-    	Class("Player2",DRUID)
-    	Class("Player2",RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Owl
+    	Class(Player2,CLERIC)
+    	Class(Player2,DRUID)
+    	Class(Player2,RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Owl
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(2)
-    	Class("Player2",MAGE)
-    	Class("Player2",SORCERER)~ THEN EXTERN HAERDAJ M#P2LT2Ibis
+    	Class(Player2,MAGE)
+    	Class(Player2,SORCERER)~ THEN EXTERN HAERDAJ M#P2LT2Ibis
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(10)
-	Class("Player2",FIGHTER_MAGE)
-	Class("Player2",FIGHTER_CLERIC)
-	Class("Player2",FIGHTER_THIEF)
-	Class("Player2",FIGHTER_MAGE_THIEF)
-	Class("Player2",MAGE_THIEF)
-	Class("Player2",CLERIC_MAGE)
-	Class("Player2",CLERIC_THIEF)
-	Class("Player2",FIGHTER_DRUID)
-	Class("Player2",FIGHTER_MAGE_CLERIC)
-    	Class("Player2",CLERIC_RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Hound
+	Class(Player2,FIGHTER_MAGE)
+	Class(Player2,FIGHTER_CLERIC)
+	Class(Player2,FIGHTER_THIEF)
+	Class(Player2,FIGHTER_MAGE_THIEF)
+	Class(Player2,MAGE_THIEF)
+	Class(Player2,CLERIC_MAGE)
+	Class(Player2,CLERIC_THIEF)
+	Class(Player2,FIGHTER_DRUID)
+	Class(Player2,FIGHTER_MAGE_CLERIC)
+    	Class(Player2,CLERIC_RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Hound
   IF ~IsValidForPartyDialog("Edwin")~ THEN EXTERN EDWINJ M#P2LT2
   // IF ~IsValidForPartyDialog("Cernd")~ THEN EXTERN CERNDJ M#P2LT2 
   IF ~IsValidForPartyDialog("Anomen")~ THEN EXTERN ANOMENJ M#P2LT2 
@@ -9702,35 +9702,35 @@
   IF ~IsValidForPartyDialog("Edwin")~ THEN EXTERN EDWINJ M#P2LT2
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(4)
-    	Class("Player2",FIGHTER)
-    	Class("Player2",PALADIN)
-    	Class("Player2",MONK)
-    	Kit("Player2",BARBARIAN)~ THEN EXTERN HAERDAJ M#P2LT2Hawk
+    	Class(Player2,FIGHTER)
+    	Class(Player2,PALADIN)
+    	Class(Player2,MONK)
+    	Kit(Player2,BARBARIAN)~ THEN EXTERN HAERDAJ M#P2LT2Hawk
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(2)
-    	Class("Player2",BARD)
-    	Class("Player2",THIEF)~ THEN EXTERN HAERDAJ M#P2LT2Mockingbird
+    	Class(Player2,BARD)
+    	Class(Player2,THIEF)~ THEN EXTERN HAERDAJ M#P2LT2Mockingbird
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(3)
-    	Class("Player2",CLERIC)
-    	Class("Player2",DRUID)
-    	Class("Player2",RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Owl
+    	Class(Player2,CLERIC)
+    	Class(Player2,DRUID)
+    	Class(Player2,RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Owl
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(2)
-    	Class("Player2",MAGE)
-    	Class("Player2",SORCERER)~ THEN EXTERN HAERDAJ M#P2LT2Ibis
+    	Class(Player2,MAGE)
+    	Class(Player2,SORCERER)~ THEN EXTERN HAERDAJ M#P2LT2Ibis
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(10)
-	Class("Player2",FIGHTER_MAGE)
-	Class("Player2",FIGHTER_CLERIC)
-	Class("Player2",FIGHTER_THIEF)
-	Class("Player2",FIGHTER_MAGE_THIEF)
-	Class("Player2",MAGE_THIEF)
-	Class("Player2",CLERIC_MAGE)
-	Class("Player2",CLERIC_THIEF)
-	Class("Player2",FIGHTER_DRUID)
-	Class("Player2",FIGHTER_MAGE_CLERIC)
-    	Class("Player2",CLERIC_RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Hound
+	Class(Player2,FIGHTER_MAGE)
+	Class(Player2,FIGHTER_CLERIC)
+	Class(Player2,FIGHTER_THIEF)
+	Class(Player2,FIGHTER_MAGE_THIEF)
+	Class(Player2,MAGE_THIEF)
+	Class(Player2,CLERIC_MAGE)
+	Class(Player2,CLERIC_THIEF)
+	Class(Player2,FIGHTER_DRUID)
+	Class(Player2,FIGHTER_MAGE_CLERIC)
+    	Class(Player2,CLERIC_RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Hound
   IF ~IsValidForPartyDialog("Imoen2")~ THEN EXTERN IMOEN2J M#P2LT2
   IF ~IsValidForPartyDialog("Jaheira")~ THEN EXTERN JAHEIRAJ M#P2LT2
   IF ~IsValidForPartyDialog("Jan")~ THEN EXTERN JANJ M#P2LT2
@@ -9756,35 +9756,35 @@
   IF ~IsValidForPartyDialog("Yoshimo")~ THEN EXTERN YOSHJ M#P2LT2
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(4)
-    	Class("Player2",FIGHTER)
-    	Class("Player2",PALADIN)
-    	Class("Player2",MONK)
-    	Kit("Player2",BARBARIAN)~ THEN EXTERN HAERDAJ M#P2LT2Hawk
+    	Class(Player2,FIGHTER)
+    	Class(Player2,PALADIN)
+    	Class(Player2,MONK)
+    	Kit(Player2,BARBARIAN)~ THEN EXTERN HAERDAJ M#P2LT2Hawk
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(2)
-    	Class("Player2",BARD)
-    	Class("Player2",THIEF)~ THEN EXTERN HAERDAJ M#P2LT2Mockingbird
+    	Class(Player2,BARD)
+    	Class(Player2,THIEF)~ THEN EXTERN HAERDAJ M#P2LT2Mockingbird
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(3)
-    	Class("Player2",CLERIC)
-    	Class("Player2",DRUID)
-    	Class("Player2",RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Owl
+    	Class(Player2,CLERIC)
+    	Class(Player2,DRUID)
+    	Class(Player2,RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Owl
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(2)
-    	Class("Player2",MAGE)
-    	Class("Player2",SORCERER)~ THEN EXTERN HAERDAJ M#P2LT2Ibis
+    	Class(Player2,MAGE)
+    	Class(Player2,SORCERER)~ THEN EXTERN HAERDAJ M#P2LT2Ibis
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(10)
-	Class("Player2",FIGHTER_MAGE)
-	Class("Player2",FIGHTER_CLERIC)
-	Class("Player2",FIGHTER_THIEF)
-	Class("Player2",FIGHTER_MAGE_THIEF)
-	Class("Player2",MAGE_THIEF)
-	Class("Player2",CLERIC_MAGE)
-	Class("Player2",CLERIC_THIEF)
-	Class("Player2",FIGHTER_DRUID)
-	Class("Player2",FIGHTER_MAGE_CLERIC)
-    	Class("Player2",CLERIC_RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Hound
+	Class(Player2,FIGHTER_MAGE)
+	Class(Player2,FIGHTER_CLERIC)
+	Class(Player2,FIGHTER_THIEF)
+	Class(Player2,FIGHTER_MAGE_THIEF)
+	Class(Player2,MAGE_THIEF)
+	Class(Player2,CLERIC_MAGE)
+	Class(Player2,CLERIC_THIEF)
+	Class(Player2,FIGHTER_DRUID)
+	Class(Player2,FIGHTER_MAGE_CLERIC)
+    	Class(Player2,CLERIC_RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Hound
   IF ~IsValidForPartyDialog("Nalia")~ THEN EXTERN NALIAJ M#P2LT2
   IF ~IsValidForPartyDialog("Imoen2")~ THEN EXTERN IMOEN2J M#P2LT2
   IF ~IsValidForPartyDialog("Anomen")~ THEN EXTERN ANOMENJ M#P2LT2 
@@ -9808,35 +9808,35 @@
   IF ~IsValidForPartyDialog("Nalia")~ THEN EXTERN NALIAJ M#P2LT2
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(4)
-    	Class("Player2",FIGHTER)
-    	Class("Player2",PALADIN)
-    	Class("Player2",MONK)
-    	Kit("Player2",BARBARIAN)~ THEN EXTERN HAERDAJ M#P2LT2Hawk
+    	Class(Player2,FIGHTER)
+    	Class(Player2,PALADIN)
+    	Class(Player2,MONK)
+    	Kit(Player2,BARBARIAN)~ THEN EXTERN HAERDAJ M#P2LT2Hawk
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(2)
-    	Class("Player2",BARD)
-    	Class("Player2",THIEF)~ THEN EXTERN HAERDAJ M#P2LT2Mockingbird
+    	Class(Player2,BARD)
+    	Class(Player2,THIEF)~ THEN EXTERN HAERDAJ M#P2LT2Mockingbird
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(3)
-    	Class("Player2",CLERIC)
-    	Class("Player2",DRUID)
-    	Class("Player2",RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Owl
+    	Class(Player2,CLERIC)
+    	Class(Player2,DRUID)
+    	Class(Player2,RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Owl
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(2)
-    	Class("Player2",MAGE)
-    	Class("Player2",SORCERER)~ THEN EXTERN HAERDAJ M#P2LT2Ibis
+    	Class(Player2,MAGE)
+    	Class(Player2,SORCERER)~ THEN EXTERN HAERDAJ M#P2LT2Ibis
   IF ~IsValidForPartyDialog("HaerDalis")
       OR(10)
-	Class("Player2",FIGHTER_MAGE)
-	Class("Player2",FIGHTER_CLERIC)
-	Class("Player2",FIGHTER_THIEF)
-	Class("Player2",FIGHTER_MAGE_THIEF)
-	Class("Player2",MAGE_THIEF)
-	Class("Player2",CLERIC_MAGE)
-	Class("Player2",CLERIC_THIEF)
-	Class("Player2",FIGHTER_DRUID)
-	Class("Player2",FIGHTER_MAGE_CLERIC)
-    	Class("Player2",CLERIC_RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Hound
+	Class(Player2,FIGHTER_MAGE)
+	Class(Player2,FIGHTER_CLERIC)
+	Class(Player2,FIGHTER_THIEF)
+	Class(Player2,FIGHTER_MAGE_THIEF)
+	Class(Player2,MAGE_THIEF)
+	Class(Player2,CLERIC_MAGE)
+	Class(Player2,CLERIC_THIEF)
+	Class(Player2,FIGHTER_DRUID)
+	Class(Player2,FIGHTER_MAGE_CLERIC)
+    	Class(Player2,CLERIC_RANGER)~ THEN EXTERN HAERDAJ M#P2LT2Hound
   IF ~IsValidForPartyDialog("Minsc")~ THEN EXTERN MINSCJ M#P2LT2
   IF ~IsValidForPartyDialog("Imoen2")~ THEN EXTERN IMOEN2J M#P2LT2
   IF ~IsValidForPartyDialog("Mazzy")~ THEN EXTERN MAZZYJ M#P2LT2
@@ -10807,19 +10807,19 @@
 IF ~~ THEN BEGIN P2LT12A
   SAY @6899 /* @6899 = ~Not much of a romantic, are you? Well, I suppose that someone in your situation cannot afford to be too sentimental.~ */
   IF ~~ THEN GOTO P2LT12D
-  IF ~GlobalGT("CHAPTER","GLOBAL",3)~ THEN GOTO P2LT12D1
+  IF ~GlobalGT("CHAPTER","GLOBAL",%bg2_chapter_3%)~ THEN GOTO P2LT12D1
 END
 
 IF ~~ THEN BEGIN P2LT12B
   SAY @6900 /* @6900 = ~I wonder if it's just something we have been conditioned to feel by the armies of poets who have written countless sad and longing verses about the sea. Or is it somehow a feeling that we all naturally share?~ */
   IF ~~ THEN GOTO P2LT12D
-  IF ~GlobalGT("CHAPTER","GLOBAL",3)~ THEN GOTO P2LT12D1
+  IF ~GlobalGT("CHAPTER","GLOBAL",%bg2_chapter_3%)~ THEN GOTO P2LT12D1
 END
 
 IF ~~ THEN BEGIN P2LT12C
   SAY @6901 /* @6901 = ~That's certainly one way of seeing it. I must admit that I find your optimism inspiring, <GABBER>.~ */
   IF ~~ THEN GOTO P2LT12D
-  IF ~GlobalGT("CHAPTER","GLOBAL",3)~ THEN GOTO P2LT12D1
+  IF ~GlobalGT("CHAPTER","GLOBAL",%bg2_chapter_3%)~ THEN GOTO P2LT12D1
 END
 
 IF ~~ THEN BEGIN P2LT12D
@@ -15970,13 +15970,13 @@
       GlobalLT("WorkingForBodhi","GLOBAL",1)~ THEN REPLY @7603 /* @7603 = ~What is your take on this? Would you rather trust the Shadow Thieves to help us or throw your lot in with this mysterious Valen's 'mistress'?~ */ GOTO ChoosePath1
   IF ~IsGabber(Player1)
       Global("AsylumPlot","GLOBAL",56)
-      GlobalLT("CHAPTER","GLOBAL",5)
+      GlobalLT("CHAPTER","GLOBAL",%bg2_chapter_5%)
       Global("TakeSaemonRoute","GLOBAL",1)
       !AreaCheck("AR1515")
       !AreaCheck("AR1516")~ THEN REPLY @7604 /* @7604 = ~Do you think that I made the right decision in trusting our lives to Saemon once again?~ */ GOTO ChoosePath2
   IF ~IsGabber(Player1)
       Global("AsylumPlot","GLOBAL",56)
-      GlobalLT("CHAPTER","GLOBAL",5)
+      GlobalLT("CHAPTER","GLOBAL",%bg2_chapter_5%)
       Global("TakeSaemonRoute","GLOBAL",1)
       OR(2)
         AreaCheck("AR1515")
@@ -19162,10 +19162,10 @@
       GlobalLT("M#AmberTalk","Locals",74)
       RealGlobalTimerExpired("M#AmberRomanceTime","GLOBAL")~ THEN GOTO WhatsUpOcean
   IF ~GlobalGT("M#AmberTalk","Locals",74)
-      GlobalLT("Chapter","GLOBAL",6)~ THEN GOTO WhatsUpUnderdark
+      GlobalLT("Chapter","GLOBAL",%bg2_chapter_6%)~ THEN GOTO WhatsUpUnderdark
   IF ~Global("M#AmberFriend","GLOBAL",1)
       Global("M#AmberTalk","Locals",20)
-      GlobalLT("Chapter","GLOBAL",6)~ THEN GOTO WhatsUpFriend
+      GlobalLT("Chapter","GLOBAL",%bg2_chapter_6%)~ THEN GOTO WhatsUpFriend
 END
 
 IF ~~ THEN BEGIN WhatsUpBad
