--- amber/scripts/m#ar0700.baf	2005-03-13 01:52:00.000000000 -0500
+++ Fixpack/amber/scripts/m#ar0700.baf	2016-11-26 02:07:11.365750600 -0500
@@ -1,31 +1,31 @@
 IF
-	Global("M#RashrynInPlace","AR0700",0)
+	Global("M#RashryInPlace","AR0700",0)
 	TimeGT(6)
 	TimeLT(22)
 THEN
 	RESPONSE #100
-	CreateCreature("M#Rashryn",[1557.2236],14)
-	SetGlobal("M#RashrynInPlace","AR0700",1)
+	CreateCreature("M#Rashry",[1557.2236],14)
+	SetGlobal("M#RashryInPlace","AR0700",1)
 END
 
 IF
-	!Dead("M#Rashryn")
-	Global("M#RashrynInPlace","AR0700",1)
+	!Dead("M#Rashry")
+	Global("M#RashryInPlace","AR0700",1)
 	TimeGT(21)
 	TimeLT(7)
 THEN
 	RESPONSE #100
-	Deactivate("M#Rashryn")
-	SetGlobal("M#RashrynInPlace","AR0700",2)
+	Deactivate("M#Rashry")
+	SetGlobal("M#RashryInPlace","AR0700",2)
 END
 
 IF
-	!Dead("M#Rashryn")
-	Global("M#RashrynInPlace","AR0700",2)
+	!Dead("M#Rashry")
+	Global("M#RashryInPlace","AR0700",2)
 	TimeGT(6)
 	TimeLT(22)
 THEN
 	RESPONSE #100
-	Activate("M#Rashryn")
-	SetGlobal("M#RashrynInPlace","AR0700",1)
+	Activate("M#Rashry")
+	SetGlobal("M#RashryInPlace","AR0700",1)
 END
