--- amber/scripts/m#girl1.baf	2003-11-06 20:56:00.000000000 -0500
+++ Fixpack/amber/scripts/m#girl1.baf	2016-11-26 02:07:11.580551500 -0500
@@ -9,12 +9,12 @@
 END
 
 IF
-	Global("M#BoyGirl","GLOBAL","1")
+	Global("M#BoyGirl","GLOBAL",1)
 	!TimeOfDay(Day)
 THEN
 	RESPONSE #100
 		EscapeAreaDestroy(30)
-		SetGlobal("M#BoyGirl","GLOBAL","0")
+		SetGlobal("M#BoyGirl","GLOBAL",0)
 END
 
 IF
