--- amber/scripts/m#imoen.baf	2009-08-21 11:07:00.000000000 -0400
+++ Fixpack/amber/scripts/m#imoen.baf	2016-11-21 20:18:22.562774800 -0500
@@ -20,7 +20,7 @@
 IF
 	InParty("M#Amber")
 	InParty("Imoen2")
-	!Global("Chapter","GLOBAL",5)
+	!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 	!AreaType(DUNGEON)
 	OR(4)
 	  Global("M#AmberImoenTalk","GLOBAL",0)
@@ -52,7 +52,7 @@
 
 IF
 	Global("M#AmberImoenBanter","GLOBAL",0)
-	GlobalGT("Chapter","GLOBAL",5)
+	GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 	Global("M#AmberRomanceActive","GLOBAL",2)
 	Global("M#AmberMatch","GLOBAL",1)
 	Global("M#AmberImoenBanterTimeActivated","Locals",0)
@@ -68,7 +68,7 @@
 	RealGlobalTimerExpired("M#AmberImoenBanterTime","GLOBAL")
 	InParty("M#Amber")
 	InParty("Imoen2")
-	!Global("Chapter","GLOBAL",5)
+	!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 	!AreaType(DUNGEON)
 	CombatCounter(0)
 	
