--- amber/dialogs/m#interject.d	2009-08-21 12:33:00.000000000 -0400
+++ Fixpack/amber/dialogs/m#interject.d	2016-11-26 02:07:11.798194200 -0500
@@ -378,7 +378,7 @@
   END
 
   IF ~~ THEN BEGIN M#AmberPatriciaK
-    SAY @2034 /* @2034 = ~Are you daft or something, <BOYGIRL>? I said that creature's money is no good here...~ */
+    SAY @2034 /* @2034 = ~Are you daft or something, <GIRLBOY>? I said that creature's money is no good here...~ */
     =
     @2035 /* @2035 = ~But just out of curiosity... What kind of a sum are we talking about here?~ */
     IF ~~ THEN REPLY @2036 /* @2036 = ~I am sure 10 gold coins would do the trick, don't you agree?~ */ GOTO M#AmberPatriciaL
