--- amber/scripts/m#jaheir.baf	2006-08-21 13:06:00.000000000 -0400
+++ Fixpack/amber/scripts/m#jaheir.baf	2016-11-21 20:19:18.126447000 -0500
@@ -15,7 +15,7 @@
 IF
 	RealGlobalTimerExpired("M#AmberJaheiraRomanceConflictTime","GLOBAL")
 	Global("M#AmberJaheiraRomanceConflict","GLOBAL",1)
-	!Global("Chapter","GLOBAL",5)
+	!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 	!AreaType(DUNGEON)
 	InParty("m#amber")
 	InParty("Jaheira")
