--- amber\scripts\m#jaheir.baf	Mon Aug 21 12:06:00 2006
+++ D:\EE_Patchstudio\patched files\amber\scripts\m#jaheir.baf	Fri Jan 15 12:51:46 2016
@@ -15,7 +15,7 @@
 IF
 	RealGlobalTimerExpired("M#AmberJaheiraRomanceConflictTime","GLOBAL")
 	Global("M#AmberJaheiraRomanceConflict","GLOBAL",1)
-	!Global("Chapter","GLOBAL",5)
+	!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 	!AreaType(DUNGEON)
 	InParty("m#amber")
 	InParty("Jaheira")
