--- amber/scripts/m#viconi.baf	2006-08-21 13:13:00.000000000 -0400
+++ Fixpack/amber/scripts/m#viconi.baf	2016-11-21 16:26:17.554494400 -0500
@@ -15,7 +15,7 @@
 IF
 	RealGlobalTimerExpired("M#AmberViconiaRomanceConflictTime","GLOBAL")
 	Global("M#AmberViconiaRomanceConflict","GLOBAL",1)
-	!Global("Chapter","GLOBAL",5)
+	!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 	!AreaType(DUNGEON)
 	InParty("m#amber")
 	InParty("Viconia")
@@ -46,7 +46,7 @@
 	OR(2)
 	  Global("M#AmberViconiaTalk","GLOBAL",0)
 	  Global("M#AmberViconiaTalk","GLOBAL",4)
-	!Global("Chapter","GLOBAL",5)
+	!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 	!AreaType(DUNGEON)
 	InParty("M#Amber")
 	InParty(Myself)
