--- setup-amber.tp2	2014-01-07 15:15:55.956875700 -0500
+++ Fixpack/setup-amber.tp2	2016-11-23 15:24:07.835179900 -0500
@@ -1,13 +1,56 @@
 BACKUP ~amber/backup~
 AUTHOR ~amber@welho.net~
-VERSION ~v4~
+
+VERSION ~v4 BWP Fix~
+
 README ~amber/readme-amber.html~
+
+ALWAYS
+  ACTION_IF GAME_IS ~eet~ BEGIN
+    OUTER_SET bg2_chapter = 12
+  END ELSE BEGIN
+    OUTER_SET bg2_chapter = 0
+  END
+  OUTER_FOR (i=1; i<=10; i=i+1) BEGIN
+    OUTER_SET bg2_chapter = bg2_chapter + 1
+    OUTER_SPRINT name_source ~bg2_chapter_%i%~
+    OUTER_SET EVAL ~%name_source%~ = bg2_chapter
+  END
+
+  ACTION_DEFINE_ASSOCIATIVE_ARRAY tra#charsets BEGIN
+    english => cp1252
+    british => cp1252
+    french => cp1252
+    spanish => cp1252
+    russian => cp1251
+    german => cp1252
+    italian => cp1252
+    polish => cp1250
+    schinese => cp936
+  END
+  ACTION_DEFINE_ARRAY tra#reload BEGIN setup setup_ee END
+  ACTION_DEFINE_ARRAY tra#noconvert BEGIN END
+  ACTION_IF "%WEIDU_OS%" STRING_EQUAL_CASE ~win32~ BEGIN // HANDLE_CHARSETS doesn't work with iconv_path containing spaces
+    AT_NOW ~if not exist "iconv" move "BiG World Fixpack\_utils\iconv" "iconv"~
+  END
+  LAF HANDLE_CHARSETS
+    INT_VAR
+      infer_charsets = 0
+    STR_VAR
+      tra_path = EVAL ~%MOD_FOLDER%/tra~
+      charset_table = tra#charsets
+      noconvert_array = tra#noconvert
+      reload_array = tra#reload
+      iconv_path = ~iconv~ // provided by BiG World Fixpack
+  END
+END
+
 LANGUAGE ~English~ ~english~ ~amber/tra/english/setup.tra~
 LANGUAGE ~Deutsch~ ~german~  ~amber/tra/german/setup.tra~
 LANGUAGE ~Spanish~ ~spanish~ ~amber/tra/spanish/setup.tra~
 
 BEGIN @150 /* @150 = ~Amber the NPC MOD for BGII:SoA~ */
-REQUIRE_PREDICATE FILE_EXISTS_IN_GAME ~mel01.cre~ @151 /* @151 = ~Please install the Throne of Bhaal expansion and the official patch before installing Amber.~ */
+REQUIRE_PREDICATE GAME_IS ~tob bg2ee eet~ @151 /* @151 = ~Please install the Throne of Bhaal expansion and the official patch before installing Amber.~ */
 
 // use new macro to handle all audio installation
 LAF HANDLE_AUDIO STR_VAR audio_path = ~amber/sounds/ambient~ oggdec_path = ~amber/sounds~ sox_path = ~amber/sounds~ END // ambient area sounds
@@ -27,7 +70,7 @@
 SET_2DA_ENTRY 0 2 3 ~m#blank.mus~
 ADD_MUSIC  ~Amber~    ~music/mxm#am.mus~
 
-ACTION_IF GAME_IS BG2EE THEN BEGIN
+ACTION_IF GAME_IS ~bg2ee eet~ THEN BEGIN
 
   COPY ~amber/ee~ ~override~ // ee-sized portraits, item description images
   LOAD_TRA ~amber/tra/%LANGUAGE%/setup_ee.tra~
@@ -136,7 +179,7 @@
  SAY @152 /* @152 = ~Fighter~ */
  SAY @153 /* @153 = ~FIGHTER:  The fighter is a champion, swordsman, soldier, and brawler.  He lives or dies by his knowledge of weapons and tactics.  Fighters can be found at the front of any battle, contesting toe-to-toe with monsters and villains.  A good fighter needs to be strong and healthy if he hopes to survive.~ */
 
-ACTION_IF GAME_IS BG2EE THEN BEGIN
+ACTION_IF GAME_IS ~bg2ee eet~ THEN BEGIN
 
   INCLUDE ~amber/lib/cd_ee_kit_appends.tph~
   LAUNCH_ACTION_FUNCTION ee_kit_appends STR_VAR class = "FIGHTER" kitname = "m#ambkit" END
@@ -145,8 +188,8 @@
 
 // Compiling scripts and dialogs
 
-COMPILE ~amber/dialogs~ USING ~amber/tra/%s/dialogs.tra~
-COMPILE ~amber/scripts~ USING ~amber/tra/%s/dialogs.tra~
+COMPILE EVALUATE_BUFFER ~amber/dialogs~ USING ~amber/tra/%s/dialogs.tra~
+COMPILE EVALUATE_BUFFER ~amber/scripts~ USING ~amber/tra/%s/dialogs.tra~
 COPY_EXISTING ~m#amber.bcs~ ~override/m#amber.bcs~
 REPLACE_TEXTUALLY 999997 ~%Amber%~
 COPY_EXISTING ~m#amberd.bcs~ ~override/m#amberd.bcs~
@@ -264,46 +307,46 @@
   END
   BUT_ONLY_IF_IT_CHANGES
 
-// The following fixes the broken kit.ids so that it can be used to detect kits properly.I stole this improved version from the Ding0.
+// The following fixes the broken kit.ids so that it can be used to detect kits properly.
 COPY_EXISTING ~kit.ids~ ~override~
-  REPLACE_TEXTUALLY ~.*\bTRUECLASS\b~              ~0x4000 TRUECLASS~
-  REPLACE_TEXTUALLY ~.*\bBERSERKER\b~              ~0x4001 BERSERKER~
-  REPLACE_TEXTUALLY ~.*\bWIZARDSLAYER\b~           ~0x4002 WIZARDSLAYER~
-  REPLACE_TEXTUALLY ~.*\bKENSAI\b~                 ~0x4003 KENSAI~
-  REPLACE_TEXTUALLY ~.*\bCAVALIER\b~               ~0x4004 CAVALIER~
-  REPLACE_TEXTUALLY ~.*\bINQUISITOR\b~             ~0x4005 INQUISITOR~
-  REPLACE_TEXTUALLY ~.*\bUNDEADHUNTER\b~           ~0x4006 UNDEADHUNTER~
-  REPLACE_TEXTUALLY ~.*\bMAGESCHOOL_ABJURER\b~     ~0x0040 MAGESCHOOL_ABJURER~
-  REPLACE_TEXTUALLY ~.*\bMAGESCHOOL_CONJURER\b~    ~0x0080 MAGESCHOOL_CONJURER~
-  REPLACE_TEXTUALLY ~.*\bMAGESCHOOL_DIVINER\b~     ~0x0100 MAGESCHOOL_DIVINER~
-  REPLACE_TEXTUALLY ~.*\bMAGESCHOOL_ENCHANTER\b~   ~0x0200 MAGESCHOOL_ENCHANTER~
-  REPLACE_TEXTUALLY ~.*\bMAGESCHOOL_ILLUSIONIST\b~ ~0x0400 MAGESCHOOL_ILLUSIONIST~
-  REPLACE_TEXTUALLY ~.*\bMAGESCHOOL_INVOKER\b~     ~0x0800 MAGESCHOOL_INVOKER~
-  REPLACE_TEXTUALLY ~.*\bMAGESCHOOL_NECROMANCER\b~ ~0x1000 MAGESCHOOL_NECROMANCER~
-  REPLACE_TEXTUALLY ~.*\bMAGESCHOOL_TRANSMUTER\b~  ~0x2000 MAGESCHOOL_TRANSMUTER~
-  REPLACE_TEXTUALLY ~.*\bMAGESCHOOL_GENERALIST\b~  ~0x4000 MAGESCHOOL_GENERALIST~
-  REPLACE_TEXTUALLY ~.*\bFERALAN\b~                ~0x4007 FERALAN~
-  REPLACE_TEXTUALLY ~.*\bSTALKER\b~                ~0x4008 STALKER~
-  REPLACE_TEXTUALLY ~.*\bBEASTMASTER\b~            ~0x4009 BEASTMASTER~
-  REPLACE_TEXTUALLY ~.*\bASSASIN\b~                ~0x400A ASSASIN~
-  REPLACE_TEXTUALLY ~.*\bBOUNTYHUNTER\b~           ~0x400B BOUNTYHUNTER~
-  REPLACE_TEXTUALLY ~.*\bSWASHBUCKLER\b~           ~0x400C SWASHBUCKLER~
-  REPLACE_TEXTUALLY ~.*\bBLADE\b~                  ~0x400D BLADE~
-  REPLACE_TEXTUALLY ~.*\bJESTER\b~                 ~0x400E JESTER~
-  REPLACE_TEXTUALLY ~.*\bSKALD\b~                  ~0x400F SKALD~
-  REPLACE_TEXTUALLY ~.*\bGODTALOS\b~               ~0x4013 GODTALOS~
-  REPLACE_TEXTUALLY ~.*\bGODHELM\b~                ~0x4014 GODHELM~
-  REPLACE_TEXTUALLY ~.*\bGODLATHANDER\b~           ~0x4015 GODLATHANDER~
-  REPLACE_TEXTUALLY ~.*\bTOTEMIC\b~                ~0x4010 TOTEMIC~
-  REPLACE_TEXTUALLY ~.*\bSHAPESHIFTER\b~           ~0x4011 SHAPESHIFTER~
-  REPLACE_TEXTUALLY ~.*\bBEASTFRIEND\b~            ~0x4012 BEASTFRIEND~
-  REPLACE_TEXTUALLY ~.*\bBARBARIAN\b~              ~0x40000000 BARBARIAN~
-  REPLACE_TEXTUALLY ~.*\bWILDMAGE\b~               ~0x80000000 WILDMAGE~
-  BUT_ONLY
-
-APPEND ~kit.ids~ ~0x4000 TRUECLASS~ UNLESS ~\bTRUECLASS\b~
-APPEND ~kit.ids~ ~0x40000000 BARBARIAN~ UNLESS ~\bBARBARIAN\b~
-APPEND ~kit.ids~ ~0x80000000 WILDMAGE~ UNLESS ~\bWILDMAGE\b~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]TRUECLASS[ %TAB%]*~              ~0x4000 TRUECLASS~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]BERSERKER[ %TAB%]*~              ~0x4001 BERSERKER~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]WIZARDSLAYER[ %TAB%]*~           ~0x4002 WIZARDSLAYER~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]KENSAI[ %TAB%]*~                 ~0x4003 KENSAI~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]CAVALIER[ %TAB%]*~               ~0x4004 CAVALIER~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]INQUISITOR[ %TAB%]*~             ~0x4005 INQUISITOR~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]UNDEADHUNTER[ %TAB%]*~           ~0x4006 UNDEADHUNTER~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]MAGESCHOOL_ABJURER[ %TAB%]*~     ~0x0040 MAGESCHOOL_ABJURER~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]MAGESCHOOL_CONJURER[ %TAB%]*~    ~0x0080 MAGESCHOOL_CONJURER~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]MAGESCHOOL_DIVINER[ %TAB%]*~     ~0x0100 MAGESCHOOL_DIVINER~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]MAGESCHOOL_ENCHANTER[ %TAB%]*~   ~0x0200 MAGESCHOOL_ENCHANTER~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]MAGESCHOOL_ILLUSIONIST[ %TAB%]*~ ~0x0400 MAGESCHOOL_ILLUSIONIST~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]MAGESCHOOL_INVOKER[ %TAB%]*~     ~0x0800 MAGESCHOOL_INVOKER~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]MAGESCHOOL_NECROMANCER[ %TAB%]*~ ~0x1000 MAGESCHOOL_NECROMANCER~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]MAGESCHOOL_TRANSMUTER[ %TAB%]*~  ~0x2000 MAGESCHOOL_TRANSMUTER~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]MAGESCHOOL_GENERALIST[ %TAB%]*~  ~0x4000 MAGESCHOOL_GENERALIST~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]FERALAN[ %TAB%]*~                ~0x4007 FERALAN~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]STALKER[ %TAB%]*~                ~0x4008 STALKER~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]BEASTMASTER[ %TAB%]*~            ~0x4009 BEASTMASTER~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]ASSASIN[ %TAB%]*~                ~0x400A ASSASIN~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]BOUNTYHUNTER[ %TAB%]*~           ~0x400B BOUNTYHUNTER~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]SWASHBUCKLER[ %TAB%]*~           ~0x400C SWASHBUCKLER~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]BLADE[ %TAB%]*~                  ~0x400D BLADE~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]JESTER[ %TAB%]*~                 ~0x400E JESTER~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]SKALD[ %TAB%]*~                  ~0x400F SKALD~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]GODTALOS[ %TAB%]*~               ~0x4013 GODTALOS~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]GODHELM[ %TAB%]*~                ~0x4014 GODHELM~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]GODLATHANDER[ %TAB%]*~           ~0x4015 GODLATHANDER~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]TOTEMIC[ %TAB%]*~                ~0x4010 TOTEMIC~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]SHAPESHIFTER[ %TAB%]*~           ~0x4011 SHAPESHIFTER~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]BEASTFRIEND[ %TAB%]*~            ~0x4012 BEASTFRIEND~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]BARBARIAN[ %TAB%]*~              ~0x40000000 BARBARIAN~
+  REPLACE_TEXTUALLY ~.*[ %TAB%]WILDMAGE[ %TAB%]*~               ~0x80000000 WILDMAGE~
+BUT_ONLY_IF_IT_CHANGES
+
+APPEND ~kit.ids~ ~0x4000 TRUECLASS~     UNLESS ~[ %TAB%]TRUECLASS[ %TAB%]*~
+APPEND ~kit.ids~ ~0x40000000 BARBARIAN~ UNLESS ~[ %TAB%]BARBARIAN[ %TAB%]*~
+APPEND ~kit.ids~ ~0x80000000 WILDMAGE~  UNLESS ~[ %TAB%]WILDMAGE[ %TAB%]*~
 
 // The following fixes the spear animations (code from G3 Tweak Pack)
 
@@ -389,8 +432,7 @@
   SAY HIDDEN_IN_SHADOWS @145
   SAY 0x1cc @144
   WRITE_SHORT 0x244 0 // doesn't change
-  WRITE_BYTE  0x246 ~%m#ambkit%~ // the internal name of your kit, surrounded by %
-  WRITE_BYTE  0x247 0x40 // doesn't change
+  WRITE_SHORT 0x246 0x4000 + ~%m#ambkit%~ // the internal name of your kit, surrounded by %
 
 COPY ~Amber/creatures/m#ambr12.cre~           ~override/m#ambr12.cre~ 
   SAY NAME1 @100
@@ -441,8 +483,7 @@
   SAY HIDDEN_IN_SHADOWS @145
   SAY 0x1cc @144
   WRITE_SHORT 0x244 0 // doesn't change
-  WRITE_BYTE  0x246 ~%m#ambkit%~ // the internal name of your kit, surrounded by %
-  WRITE_BYTE  0x247 0x40 // doesn't change
+  WRITE_SHORT 0x246 0x4000 + ~%m#ambkit%~ // the internal name of your kit, surrounded by %
 
 COPY ~Amber/creatures/m#ambr14.cre~           ~override/m#ambr14.cre~
   SAY NAME1 @100
@@ -493,8 +534,7 @@
   SAY HIDDEN_IN_SHADOWS @145
   SAY 0x1cc @144
   WRITE_SHORT 0x244 0 // doesn't change
-  WRITE_BYTE  0x246 ~%m#ambkit%~ // the internal name of your kit, surrounded by %
-  WRITE_BYTE  0x247 0x40 // doesn't change
+  WRITE_SHORT 0x246 0x4000 + ~%m#ambkit%~ // the internal name of your kit, surrounded by %
 
 COPY ~Amber/creatures/m#ambr15.cre~           ~override/m#ambr15.cre~
   SAY NAME1 @100
@@ -545,8 +585,7 @@
   SAY HIDDEN_IN_SHADOWS @145  
   SAY 0x1cc @144
   WRITE_SHORT 0x244 0 // doesn't change
-  WRITE_BYTE  0x246 ~%m#ambkit%~ // the internal name of your kit, surrounded by %
-  WRITE_BYTE  0x247 0x40 // doesn't change
+  WRITE_SHORT 0x246 0x4000 + ~%m#ambkit%~ // the internal name of your kit, surrounded by %
 
 COPY ~amber/creatures/m#vamber.cre~           ~override/m#vamber.cre~       
   SAY NAME1 @159 /* @159 = ~Amber~ */ 
