--- setup-amber.tp2	Thu Oct 03 11:59:05 2013
+++ C:\BWP Patchstudio\patched files\setup-amber.tp2	Mon Dec 09 21:37:58 2013
@@ -1,6 +1,6 @@
 BACKUP ~amber/backup~
 AUTHOR ~amber@welho.net~
-VERSION ~v3~
+VERSION ~v4 BWP Fix~
 README ~amber/readme-amber.html~
 LANGUAGE ~English~ ~english~ ~amber/tra/english/setup.tra~
 LANGUAGE ~Deutsch~ ~german~  ~amber/tra/german/setup.tra~
@@ -12,7 +12,7 @@
 // use new macro to handle all audio installation
 LAF HANDLE_AUDIO STR_VAR audio_path = ~amber/sounds/ambient~ oggdec_path = ~amber/sounds~ sox_path = ~amber/sounds~ END // ambient area sounds
 ACTION_IF FILE_EXISTS ~amber/sounds/%LANGUAGE%/m#amb001.ogg~ THEN BEGIN
-  LAF HANDLE_AUDIO STR_VAR audio_path = ~amber/sounds/%LANGUAGE%~ oggdec_path = ~amber/sounds~ sox_path = ~amber/sounds~ END // this replaces all the junk below
+  LAF HANDLE_AUDIO STR_VAR audio_path = EVAL ~amber/sounds/%LANGUAGE%~ oggdec_path = ~amber/sounds~ sox_path = ~amber/sounds~ END // this replaces all the junk below
 END
 
 LAF HANDLE_TILESETS STR_VAR tiz_path = ~amber/tiz~ tizunpack_path = ~amber/tiz~ END // unpacks area tilesets
@@ -389,8 +389,8 @@
   SAY HIDDEN_IN_SHADOWS @145
   SAY 0x1cc @144
   WRITE_SHORT 0x244 0 // doesn't change
-  WRITE_BYTE  0x246 ~%m#ambkit%~ // the internal name of your kit, surrounded by %
-  WRITE_BYTE  0x247 0x40 // doesn't change
+  WRITE_SHORT 0x246 0x4000 + ~%m#ambkit%~ // the internal name of your kit, surrounded by %
+//WRITE_BYTE  0x247 0x40 // doesn't change
 
 COPY ~Amber/creatures/m#ambr12.cre~           ~override/m#ambr12.cre~ 
   SAY NAME1 @100
@@ -441,8 +441,8 @@
   SAY HIDDEN_IN_SHADOWS @145
   SAY 0x1cc @144
   WRITE_SHORT 0x244 0 // doesn't change
-  WRITE_BYTE  0x246 ~%m#ambkit%~ // the internal name of your kit, surrounded by %
-  WRITE_BYTE  0x247 0x40 // doesn't change
+  WRITE_SHORT 0x246 0x4000 + ~%m#ambkit%~ // the internal name of your kit, surrounded by %
+//WRITE_BYTE  0x247 0x40 // doesn't change
 
 COPY ~Amber/creatures/m#ambr14.cre~           ~override/m#ambr14.cre~
   SAY NAME1 @100
@@ -493,8 +493,8 @@
   SAY HIDDEN_IN_SHADOWS @145
   SAY 0x1cc @144
   WRITE_SHORT 0x244 0 // doesn't change
-  WRITE_BYTE  0x246 ~%m#ambkit%~ // the internal name of your kit, surrounded by %
-  WRITE_BYTE  0x247 0x40 // doesn't change
+  WRITE_SHORT 0x246 0x4000 + ~%m#ambkit%~ // the internal name of your kit, surrounded by %
+//WRITE_BYTE  0x247 0x40 // doesn't change
 
 COPY ~Amber/creatures/m#ambr15.cre~           ~override/m#ambr15.cre~
   SAY NAME1 @100
@@ -545,8 +545,8 @@
   SAY HIDDEN_IN_SHADOWS @145  
   SAY 0x1cc @144
   WRITE_SHORT 0x244 0 // doesn't change
-  WRITE_BYTE  0x246 ~%m#ambkit%~ // the internal name of your kit, surrounded by %
-  WRITE_BYTE  0x247 0x40 // doesn't change
+  WRITE_SHORT 0x246 0x4000 + ~%m#ambkit%~ // the internal name of your kit, surrounded by %
+//WRITE_BYTE  0x247 0x40 // doesn't change
 
 COPY ~amber/creatures/m#vamber.cre~           ~override/m#vamber.cre~       
   SAY NAME1 @159 /* @159 = ~Amber~ */ 
