--- DrizztSaga\DLG\F_ELMINS.d	Wed Feb 14 21:18:56 2007
+++ C:\BWP Patchstudio\patched files\DrizztSaga\DLG\F_ELMINS.d	Wed Jan 04 05:11:22 2012
@@ -19,13 +19,13 @@
 IF ~~ THEN BEGIN 3 // from: 2.0
   SAY @8
   IF ~~ THEN DO ~RevealAreaOnMap("F_7777")
+EscapeArea()
 ActionOverride(Player1,LeaveAreaLUA("F_7777","",[397.1238],0))
 ActionOverride(Player2,LeaveAreaLUA("F_7777","",[397.1238],0))
 ActionOverride(Player3,LeaveAreaLUA("F_7777","",[397.1238],0))
 ActionOverride(Player4,LeaveAreaLUA("F_7777","",[397.1238],0))
 ActionOverride(Player5,LeaveAreaLUA("F_7777","",[397.1238],0))
-ActionOverride(Player6,LeaveAreaLUA("F_7777","",[397.1238],0))
-EscapeArea()~ EXIT
+ActionOverride(Player6,LeaveAreaLUA("F_7777","",[397.1238],0))~ EXIT
 END
 
 IF ~~ THEN BEGIN 4 // from: 1.1
