--- Faren\crossmod\tsujatha\fhftsu.d	Fri May 01 20:55:04 2015
+++ C:\BWP Patchstudio\patched files\Faren\crossmod\tsujatha\fhftsu.d	Fri Jul 10 23:13:36 2015
@@ -78,7 +78,7 @@
 == BTSUJAT ~You're practising lockpicking?~
 == BFHFRN ~Yeah, I need to keep in practice. And it's something to do, when we're on the road.~
 == BTSUJAT ~How did you come to learn? It seems odd for a member of the militia to turn to criminal activities.~
-== BFHFRN ~More common than you'd think, actually, but that's another story. No, after I left I headed over to Athkatla to seek my fortune and all that.
+== BFHFRN ~More common than you'd think, actually, but that's another story. No, after I left I headed over to Athkatla to seek my fortune and all that.~
 = ~And there were some friendly folks who I got in with. They taught me, I was quick, it went from there.~
 == BTSUJAT ~So you had no plan, no decisive force behind it.~
 == BFHFRN ~...you could say that. If you were feeling obnoxious.~
