--- c2/french/setup.tra	2011-12-28 13:18:46.000000000 +0100
+++ Fixpack/c2/french/setup.tra	2017-02-05 12:45:25.491314200 +0100
@@ -1,6 +1,8 @@
 @1 = ~Shadows Of Amn Item Upgrades~
 @2 = ~Throne Of Bhaal Item Upgrades~
 @3 = ~Je n'ai pas pu trouver [Data\25Dialog.bif], ce qui me fait penser que vous n'avez pas installe l'extension Throne of Bhaal. Desole !~
+@4 = ~Version d'origine (les nouvelles ameliorations sont proposees dans une liste)~
+@5 = ~Version revisee (les nouvelles ameliorations sont proposees dans le meme style que les ameliorations du jeu de base)~
 
 @100 = ~Anneau de Préservation +1~
 @101 = ~Anneau de Préservation +1 : "Anneau du Prince Troll"
@@ -1222,3 +1224,30 @@
 
 @-1038= "dem[A]nder pour chaque composant, tout [S]auter, ou choisir une selection pre-definie :"
 @-1039= "dem[A]nder pour chaque composant, [R]einstaller la configuration courante, tout s[U]pprimer, tout [S]auter, ou choisir une selection pre-definie :" 
+
+@-1040= "
+Veuillez choisir la langue dans laquelle vous souhaitez jouer au jeu.
+Si par la suite vous desirez jouer au jeu dans une langue differente, il vous
+faudra editer le fichier weidu.conf et reinstaller vos mods.
+"
+@-1041= "Tcheque"
+@-1042= "Allemand"
+@-1043= "Anglais"
+@-1044= "Espagnol"
+@-1045= "Francais"
+@-1046= "Italien"
+@-1047= "Polonais"
+@-1048= "Portugais"
+@-1049= "Turc"
+@-1050= "Japonais"
+@-1051= "Coreen"
+@-1052= "Chinois simplifie"
+
+@-1060= "
+Ce jeu est disponible en plusieurs langues mais WeiDU ne sait pas laquelle a ete choisie quand ce mod a ete installe.
+
+Veuillez indiquer quelle langue du jeu a ete utilisee quand vous avez installe ce mod. Votre choix servira jusqu'a ce que WeiDU s'arrete mais ne sera pas memorise.
+"
+@-1061= "
+Utilisation de"
+@-1062= " et "
