--- c2/german/setup.tra	2006-08-05 15:30:41.434000000 +0200
+++ Fixpack/c2/german/setup.tra	2017-02-05 12:45:12.845074100 +0100
@@ -1,7 +1,9 @@
 
-@1 = ~Verbesserte Ausrüstung für 'Die Schatten von Amn'~
-@2 = ~Verbesserte Ausrüstung für 'Thron des Bhaal'~
+@1 = ~Verbesserte Ausruestung fuer 'Die Schatten von Amn'~
+@2 = ~Verbesserte Ausruestung fuer 'Thron des Bhaal'~
 @3 = ~Konnte [Data\25Dialog.bif] nicht finden! Vermutlich haben Sie 'Thron des Bhaal' nicht installiert. Sorry!~
+@4 = ~Originale Version (Aufruestungen werden als Liste praesentiert)~
+@5 = ~Ueberarbeitete Version (Aufruestungen werden im gleichen Stil wie die existierenden praesentiert)~
 
 @100 = ~Ring der Bewahrung +1~
 @101 = ~Ring der Bewahrung +1: 'Ring des Trollprinzen'
