--- c2/german/setup.tra	2006-08-05 15:30:41.434000000 +0200
+++ Fixpack/c2/german/setup.tra	2017-02-02 13:28:38.000000000 +0100
@@ -2,6 +2,9 @@
 @1 = ~Verbesserte Ausrüstung für 'Die Schatten von Amn'~
 @2 = ~Verbesserte Ausrüstung für 'Thron des Bhaal'~
 @3 = ~Konnte [Data\25Dialog.bif] nicht finden! Vermutlich haben Sie 'Thron des Bhaal' nicht installiert. Sorry!~
+@4 = ~Originale Version (Aufruestungen werden als Liste praesentiert)~
+@5 = ~Ueberarbeitete Version (Aufruestungen werden im gleichen Stil wie die existierenden praesentiert)~
+@6 = ~Nicht in dieser Sprache verfuegbar~
 
 @100 = ~Ring der Bewahrung +1~
 @101 = ~Ring der Bewahrung +1: 'Ring des Trollprinzen'
