--- c2/lib/items_common.tpa	2013-07-31 14:32:15.902000000 +0200
+++ Fixpack/c2/lib/items_common.tpa	2017-02-05 12:51:26.062985300 +0100
@@ -1,7 +1,12 @@
-// load up ee-specific tra files on BG2EE
-ACTION_IF ((FILE_EXISTS_IN_GAME OH6000.are) AND 
-           (FILE_EXISTS ~c2/%LANGUAGE%/lolwot.tra~)) THEN BEGIN
-  LOAD_TRA ~c2/%LANGUAGE%/setup_ee.tra~
+// needed for updated item descriptions in EE games
+ACTION_MATCH ~%LANGUAGE%~ WITH
+  ~french~  BEGIN OUTER_TEXT_SPRINT regUse ~%regexpUsable_FR%~ END
+  ~german~  BEGIN OUTER_TEXT_SPRINT regUse ~%regexpUsable_DE%~ END
+  ~spanish~ BEGIN OUTER_TEXT_SPRINT regUse ~%regexpUsable_ES%~ END
+  ~polski~  BEGIN OUTER_TEXT_SPRINT regUse ~%regexpUsable_PL%~ END
+  ~russian~ BEGIN OUTER_TEXT_SPRINT regUse ~%regexpUsable_RU%~ END
+  ~korean~  BEGIN OUTER_TEXT_SPRINT regUse ~%regexpUsable_KO%~ END
+  DEFAULT         OUTER_TEXT_SPRINT regUse ~%regexpUsable_EN%~
 END
 
 // in case Fixpack is installed, change shadow temple to not consume Symbol of Amaunator as a key
@@ -74,8 +79,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2amul01.itm THEN BEGIN
   
   COPY ~c2/itm/c2amul01.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@181~ title = ~@180~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @180
-    SAY DESC      @181
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2amul01.bam~ ~override~
 
@@ -88,8 +94,22 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2anom01.itm THEN BEGIN
   
   COPY ~c2/itm/c2anom01.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@167~ title = ~@166~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @166
-    SAY DESC      @167
+    SAY DESC      ~%updated_desc%~
+    PATCH_IF (GAME_IS ~bg2ee eet~) BEGIN
+      LPF ADD_ITEM_EQEFFECT
+      INT_VAR
+        opcode      = 319   // restrict item
+        target      = 1     // self
+        power       = 1     // only usable by
+        parameter2  = 11    // actor's script name
+        timing      = 2     // while equipped
+        special     = 9138  // strref: Anomen
+      STR_VAR
+        resource    = "ANOMEN"
+      END
+    END
 
   COPY ~c2/bam/c2anom01.bam~ ~override~
 
@@ -100,8 +120,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2ax1h01.itm THEN BEGIN
   
   COPY ~c2/itm/c2ax1h01.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@185~ title = ~@184~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @184
-    SAY DESC      @185
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2ax1h01.bam~ ~override~
        ~c2/eff/c2ax1h1a.eff~ ~override~
@@ -114,8 +135,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2belt01.itm THEN BEGIN
   
   COPY ~c2/itm/c2belt01.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@189~ title = ~@188~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @188
-    SAY DESC      @189
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2belt01.bam~ ~override~
 
@@ -124,8 +146,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2blun01.itm THEN BEGIN
   
   COPY ~c2/itm/c2blun01.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@123~ title = ~@122~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @122
-    SAY DESC      @123
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2blun01.bam~ ~override~
 
@@ -134,8 +157,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2blun02.itm THEN BEGIN
   
   COPY ~c2/itm/c2blun02.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@125~ title = ~@124~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @124
-    SAY DESC      @125
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2blun02.bam~ ~override~
        ~c2/eff/c2blun02.eff~ ~override~
@@ -145,8 +169,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2blun03.itm THEN BEGIN
   
   COPY ~c2/itm/c2blun03.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@173~ title = ~@172~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @172
-    SAY DESC      @173
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2blun03.bam~ ~override~
        ~c2/eff/c2blun03.eff~ ~override~
@@ -156,8 +181,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2boot01.itm THEN BEGIN
   
   COPY ~c2/itm/c2boot01.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@115~ title = ~@114~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @114
-    SAY DESC      @115
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2boot01.bam~ ~override~
 
@@ -166,8 +192,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2boot02.itm THEN BEGIN
   
   COPY ~c2/itm/c2boot02.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@117~ title = ~@116~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @116
-    SAY DESC      @117
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2boot02.bam~ ~override~
 
@@ -176,8 +203,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2boot03.itm THEN BEGIN
   
   COPY ~c2/itm/c2boot03.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@119~ title = ~@118~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @118
-    SAY DESC      @119
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2boot03.bam~ ~override~
 
@@ -186,8 +214,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2boot04.itm THEN BEGIN
   
   COPY ~c2/itm/c2boot04.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@121~ title = ~@120~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @120
-    SAY DESC      @121
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2boot04.bam~ ~override~
 
@@ -196,8 +225,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2bow01.itm THEN BEGIN
 
   COPY ~c2/itm/c2bow01.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@127~ title = ~@126~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @126
-    SAY DESC      @127
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2bow01.bam~ ~override~
 
@@ -206,8 +236,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2brac01.itm THEN BEGIN
   
   COPY ~c2/itm/c2brac01.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@179~ title = ~@178~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @178
-    SAY DESC      @179
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2brac01.bam~ ~override~
 
@@ -216,8 +247,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2chan01.itm THEN BEGIN
   
   COPY ~c2/itm/c2chan01.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@198~ title = ~@197~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @197
-    SAY DESC      @198
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2chan01.bam~ ~override~
 
@@ -228,8 +260,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2dagg01.itm THEN BEGIN
   
   COPY ~c2/itm/c2dagg01.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@175~ title = ~@174~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @174
-    SAY DESC      @175
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2dagg01.bam~ ~override~
 
@@ -238,8 +271,21 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2hd1.itm THEN BEGIN
   
   COPY ~c2/itm/c2hd1.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@194~ title = ~@193~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @193
-    SAY DESC      @194
+    SAY DESC      ~%updated_desc%~
+    PATCH_IF (GAME_IS ~bg2ee eet~) BEGIN
+      LPF ADD_ITEM_EQEFFECT
+      INT_VAR
+        opcode      = 319   // restrict item
+        target      = 1     // self
+        power       = 1     // only usable by
+        parameter1  = 153   // tiefling
+        parameter2  = 4     // race.ids
+        timing      = 2     // while equipped
+        special     = 8332  // strref: tiefling
+      END
+    END
 
   COPY ~c2/bam/c2hd1.bam~ ~override~
 
@@ -248,16 +294,30 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2hd2.itm THEN BEGIN
   
   COPY ~c2/itm/c2hd2.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@196~ title = ~@195~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @195
-    SAY DESC      @196
+    SAY DESC      ~%updated_desc%~
+    PATCH_IF (GAME_IS ~bg2ee eet~) BEGIN
+      LPF ADD_ITEM_EQEFFECT
+      INT_VAR
+        opcode      = 319   // restrict item
+        target      = 1     // self
+        power       = 1     // only usable by
+        parameter1  = 153   // tiefling
+        parameter2  = 4     // race.ids
+        timing      = 2     // while equipped
+        special     = 8332  // strref: tiefling
+      END
+    END
 
 END
 
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2helm01.itm THEN BEGIN
   
   COPY ~c2/itm/c2helm01.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@137~ title = ~@136~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @136
-    SAY DESC      @137
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2helm01.bam~ ~override~
 
@@ -266,8 +326,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2helm02.itm THEN BEGIN
   
   COPY ~c2/itm/c2helm02.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@139~ title = ~@138~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @138
-    SAY DESC      @139
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2helm02.bam~ ~override~
 
@@ -276,8 +337,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2hide01.itm THEN BEGIN
   
   COPY ~c2/itm/c2hide01.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@141~ title = ~@140~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @140
-    SAY DESC      @141
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2hide01.bam~ ~override~
   
@@ -288,8 +350,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2keld01.itm THEN BEGIN
 
   COPY ~c2/itm/c2keld01.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@145~ title = ~@144~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @144
-    SAY DESC      @145
+    SAY DESC      ~%updated_desc%~
     PATCH_IF MOD_IS_INSTALLED ~BG2FIXPACK/SETUP-BG2FIXPACK.TP2~ ~114~  BEGIN // add immunity to stun for free acrion if OBC fixpack stuff installed
       LPF ADD_ITEM_EQEFFECT INT_VAR opcode = 101 target = 1 parameter2 =   210 timing = 2 END // immunity to pw: stun
       LPF ADD_ITEM_EQEFFECT INT_VAR opcode = 101 target = 1 parameter2 =    45 timing = 2 END // immunity to stun
@@ -297,6 +360,19 @@
       LPF ADD_ITEM_EQEFFECT INT_VAR opcode = 267 target = 1 parameter1 =  1280 timing = 2 END // prevent 'stunned' string
       LPF ADD_ITEM_EQEFFECT INT_VAR opcode = 267 target = 1 parameter1 = 14043 timing = 2 END // prevent 'stun' string
     END
+    PATCH_IF (GAME_IS ~bg2ee eet~) BEGIN
+      LPF ADD_ITEM_EQEFFECT
+      INT_VAR
+        opcode      = 319   // restrict item
+        target      = 1     // self
+        power       = 1     // only usable by
+        parameter2  = 11    // actor's script name
+        timing      = 2     // while equipped
+        special     = 9144  // strref: Keldorn
+      STR_VAR
+        resource    = "KELDORN"
+      END
+    END
 
   COPY ~c2/bam/c2keld01.bam~ ~override~
   
@@ -307,8 +383,22 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2keld02.itm THEN BEGIN
   
   COPY ~c2/itm/c2keld02.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@165~ title = ~@164~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @164
-    SAY DESC      @165
+    SAY DESC      ~%updated_desc%~
+    PATCH_IF (GAME_IS ~bg2ee eet~) BEGIN
+      LPF ADD_ITEM_EQEFFECT
+      INT_VAR
+        opcode      = 319   // restrict item
+        target      = 1     // self
+        power       = 1     // only usable by
+        parameter2  = 11    // actor's script name
+        timing      = 2     // while equipped
+        special     = 9144  // strref: Keldorn
+      STR_VAR
+        resource    = "KELDORN"
+      END
+    END
 
   COPY ~c2/bam/c2keld02.bam~ ~override~
 
@@ -317,8 +407,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2kit.itm THEN BEGIN
 
   COPY ~c2/itm/c2kit.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@191~ title = ~@190~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @190
-    SAY DESC      @191
+    SAY DESC      ~%updated_desc%~
   
   COPY ~c2/itm/c2kit.cre~ ~override~
     SAY NAME1     @192
@@ -333,8 +424,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2mazz01.itm THEN BEGIN
 
   COPY ~c2/itm/c2mazz01.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@163~ title = ~@162~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @162
-    SAY DESC      @163
+    SAY DESC      ~%updated_desc%~
     PATCH_IF MOD_IS_INSTALLED ~BG2FIXPACK/SETUP-BG2FIXPACK.TP2~ ~114~  BEGIN // add immunity to stun for free acrion if OBC fixpack stuff installed
       LPF ADD_ITEM_EQEFFECT INT_VAR opcode = 101 target = 1 parameter2 =   210 timing = 2 END // immunity to pw: stun
       LPF ADD_ITEM_EQEFFECT INT_VAR opcode = 101 target = 1 parameter2 =    45 timing = 2 END // immunity to stun
@@ -350,8 +442,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2plat01.itm THEN BEGIN
 
   COPY ~c2/itm/c2plat01.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@147~ title = ~@146~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @146
-    SAY DESC      @147
+    SAY DESC      ~%updated_desc%~
 
   APPEND ~tooltip.2da~ ~c2plat01 12033 -1 -1~ // tooltip stuff
   
@@ -362,8 +455,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2ring01.itm THEN BEGIN
 
   COPY ~c2/itm/c2ring01.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@101~ title = ~@100~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @100
-    SAY DESC      @101
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2ring01.bam~ ~override~
   
@@ -374,8 +468,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2ring02.itm THEN BEGIN
 
   COPY ~c2/itm/c2ring02.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@103~ title = ~@102~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @102
-    SAY DESC      @103
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2ring02.bam~ ~override~
   
@@ -386,8 +481,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2ring03.itm THEN BEGIN
   
   COPY ~c2/itm/c2ring03.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@105~ title = ~@104~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @104
-    SAY DESC      @105
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2ring03.bam~ ~override~
        ~c2/eff/c2ring03.eff~ ~override~
@@ -406,8 +502,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2ring04.itm THEN BEGIN
   
   COPY ~c2/itm/c2ring04.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@187~ title = ~@186~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @186
-    SAY DESC      @187
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2ring04.bam~ ~override~
   
@@ -420,32 +517,36 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2robe01.itm THEN BEGIN
   
   COPY ~c2/itm/c2robe01.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@131~ title = ~@130~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @130
-    SAY DESC      @131
+    SAY DESC      ~%updated_desc%~
 
 END
 
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2robe02.itm THEN BEGIN
   
   COPY ~c2/itm/c2robe02.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@133~ title = ~@132~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @132
-    SAY DESC      @133
+    SAY DESC      ~%updated_desc%~
 
 END
 
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2robe03.itm THEN BEGIN
   
   COPY ~c2/itm/c2robe03.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@135~ title = ~@134~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @134
-    SAY DESC      @135
+    SAY DESC      ~%updated_desc%~
 
 END
 
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2shld01.itm THEN BEGIN
   
   COPY ~c2/itm/c2shld01.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@149~ title = ~@148~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @148
-    SAY DESC      @149
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2shld01.bam~ ~override~
 
@@ -454,8 +555,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2shld02.itm THEN BEGIN
   
   COPY ~c2/itm/c2shld02.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@177~ title = ~@176~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @176
-    SAY DESC      @177
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2shld02.bam~ ~override~
 
@@ -466,8 +568,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2sper01.itm THEN BEGIN
   
   COPY ~c2/itm/c2sper01.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@151~ title = ~@150~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @150
-    SAY DESC      @151
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2sper01.bam~ ~override~
 
@@ -478,8 +581,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2sper02.itm THEN BEGIN
 
   COPY ~c2/itm/c2sper02.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@153~ title = ~@152~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @152
-    SAY DESC      @153
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2sper02.bam~ ~override~
 
@@ -490,8 +594,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2sper03.itm THEN BEGIN
 
   COPY ~c2/itm/c2sper03.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@155~ title = ~@154~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @154
-    SAY DESC      @155
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2sper03.bam~ ~override~
 
@@ -502,8 +607,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2staf01.itm THEN BEGIN
   
   COPY ~c2/itm/c2staf01.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@107~ title = ~@106~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @106
-    SAY DESC      @107
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2staf01.bam~ ~override~
        ~c2/cre/elairpr3.cre~ ~override~
@@ -521,8 +627,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2staf02.itm THEN BEGIN
   
   COPY ~c2/itm/c2staf02.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@157~ title = ~@156~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @156
-    SAY DESC      @157
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2staf02.bam~ ~override~
 
@@ -533,8 +640,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2sw1h01.itm THEN BEGIN
   
   COPY ~c2/itm/c2sw1h01.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@159~ title = ~@158~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @158
-    SAY DESC      @159
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2sw1h01.bam~ ~override~
        ~c2/eff/c2sw1h1a.eff~ ~override~ // +4 thac0 vs. evil
@@ -550,8 +658,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2sw1h02.itm THEN BEGIN
   
   COPY ~c2/itm/c2sw1h02.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@161~ title = ~@160~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @160
-    SAY DESC      @161
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2sw1h02.bam~ ~override~
 
@@ -562,8 +671,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2sw1h03.itm THEN BEGIN
   
   COPY ~c2/itm/c2sw1h03.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@183~ title = ~@182~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @182
-    SAY DESC      @183
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2sw1h03.bam~ ~override~
 
@@ -572,8 +682,9 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2sw2h01.itm THEN BEGIN
   
   COPY ~c2/itm/c2sw2h01.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@169~ title = ~@168~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @168
-    SAY DESC      @169
+    SAY DESC      ~%updated_desc%~
 
   COPY ~c2/bam/c2sw2h01.bam~ ~override~
 
@@ -582,8 +693,22 @@
 ACTION_IF NOT FILE_EXISTS_IN_GAME c2valy01.itm THEN BEGIN
   
   COPY ~c2/itm/c2valy01.itm~ ~override~
+    LPF ITEM_DESC_TO_EE STR_VAR desc = ~@129~ title = ~@128~ regexp = EVAL ~%regUse%~ RET updated_desc END
     SAY NAME2     @128
-    SAY DESC      @129
+    SAY DESC      ~%updated_desc%~
+    PATCH_IF (GAME_IS ~bg2ee eet~) BEGIN
+      LPF ADD_ITEM_EQEFFECT
+      INT_VAR
+        opcode      = 319   // restrict item
+        target      = 1     // self
+        power       = 1     // only usable by
+        parameter2  = 11    // actor's script name
+        timing      = 2     // while equipped
+        special     = 9158  // strref: Valygar
+      STR_VAR
+        resource    = "VALYGAR"
+      END
+    END
 
   COPY ~c2/bam/c2valy01.bam~ ~override~
   
