--- setup-ub.tp2	Mon Jan 20 11:54:24 2014
+++ C:\BWP Patchstudio\patched files\setup-ub.tp2	Mon Feb 03 21:45:25 2014
@@ -58,7 +58,7 @@
 
 README ~ub/readme_ub.html~
 
-VERSION ~v24~
+VERSION ~v24 BWP Fix~
 
 ////////////////////////
 ////////////////////////
@@ -137,7 +137,7 @@
 UNLESS ~CD_STATE_NOTVALID~
 
 COPY_EXISTING ~bag02.itm~ ~override/wwbag02.itm~ /* Kulyok */
-
+COPY_EXISTING ~bag02.sto~ ~override/wwbag02.sto~
 /*
 DavidW- this is unused (presumably part of Cliffette's not-quite implemented extra area content)
 COPY_EXISTING ~AR0042.tis~	~Override/arwwbo.tis~
@@ -610,7 +610,7 @@
 
 /* The Slums */
 EXTEND_TOP ar0400.bcs ~ub/encount/u!0400.baf~
-
+COMPILE ~ub/encount/slumvic.d~
 COMPILE ~ub/encount/slumvic.baf~
 
 /* Reputation Trap */
