--- ub/encount/u!vamps.baf	2015-03-22 18:05:02.000000000 -0400
+++ Fixpack/ub/encount/u!vamps.baf	2016-11-21 23:25:44.859776800 -0500
@@ -6,7 +6,7 @@
     Dead("shthass1")  // ~Ama~
     !AreaCheck("AR0700")
   CombatCounter(0)
-  GlobalLT("Chapter","GLOBAL",4)
+  GlobalLT("Chapter","GLOBAL",%bg2_chapter_4%)
   Global("VampShadWar","GLOBAL",5)
   GlobalTimerExpired("VampShadTimer","GLOBAL")
   OR(2)
@@ -44,7 +44,7 @@
     Dead("shthass1")  // ~Ama~
     !AreaCheck("AR0700")
   CombatCounter(0)
-  GlobalLT("Chapter","GLOBAL",4)
+  GlobalLT("Chapter","GLOBAL",%bg2_chapter_4%)
   Global("VampShadWar","GLOBAL",6)
   GlobalTimerExpired("VampShadTimer","GLOBAL")
   OR(2)
