--- xpmod\functions\quest_xp.tph	Mon Dec 24 22:17:12 2007
+++ C:\BWP Patchstudio\patched files\xpmod\functions\quest_xp.tph	Wed Dec 08 23:20:10 2010
@@ -32,7 +32,7 @@
 
 COPY_EXISTING_REGEXP GLOB ~.*\.bcs~ ~override~
 DECOMPILE_BCS_TO_BAF
-REPLACE_EVALUATE ~AddexperienceParty(\(.*\))~
+REPLACE_EVALUATE ~AddexperienceParty(\([0-9]*\))~
   BEGIN
 	SET "RESULT" = "%MATCH1%"
 	PATCH_IF (("%RESULT%" > 0) && (NOT (("%RESULT%" = 88999) && (("%SOURCE_RES%" STRING_EQUAL_CASE "AR0602") = 1)))) BEGIN
@@ -52,7 +52,7 @@
 	END
   END
   ~AddexperienceParty(%RESULT%)~
-REPLACE_EVALUATE ~AddXPObject(\(.*\),\(.*\))~
+REPLACE_EVALUATE ~AddXPObject(\(.*\),\([0-9]*\))~
   BEGIN
 	SET "RESULT" = "%MATCH2%"
 	PATCH_IF (("%RESULT%" > 0) && (NOT (("%RESULT%" = 88999) && (("%SOURCE_RES%" STRING_EQUAL_CASE "AR0602") = 1)))) BEGIN
@@ -78,7 +78,7 @@
 COPY_EXISTING_REGEXP GLOB ~.*\.dlg~ ~override~
 PATCH_SILENT
 DECOMPILE_DLG_TO_D
-REPLACE_EVALUATE ~xperienceParty(\(.*\))~
+REPLACE_EVALUATE ~xperienceParty(\([0-9]*\))~
   BEGIN
 	SET "RESULT" = "%MATCH1%"
 	PATCH_IF ("%RESULT%" > 0) BEGIN
@@ -98,7 +98,7 @@
 	END
   END
   ~xperienceParty(%RESULT%)~
-REPLACE_EVALUATE ~AddXPObject(\(.*\),\(.*\))~
+REPLACE_EVALUATE ~AddXPObject(\(.*\),\([0-9]*\))~
   BEGIN
 	SET "RESULT" = "%MATCH2%"
 	PATCH_IF ("%RESULT%" > 0) BEGIN
