--- Setup-RoT.tp2	Fri Nov 25 09:09:36 2016
+++ C:\BWP Patchstudio\patched files\Setup-RoT.tp2	Fri Nov 25 09:16:37 2016
@@ -9483,6 +9483,7 @@
 COPY ~RoTerror/kits/SPCL376.SPL~         ~override~
 
 /**************************************************************************/
+/* Compatibility with Rylorn mod:  Don't add Dark Paladin kit
 ADD_KIT ~Dark_Paladin~
 ~Dark_Paladin            1           1           1           1           1           1           1           1~
 ~Dark_Paladin 2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2  2 ~
@@ -9509,7 +9510,7 @@
   SAY NAME1 @1716
 
 COPY ~RoTerror/kits/SPCL378.EFF~         ~override~
-
+*/
 /**************************************************************************/
 ADD_KIT ~Planar_Druid~
 ~Planar_Druid            1           0           1           1           0           0           0           1~
