--- atweaks/lib/310-creature_colouring.tpa	2015-07-26 21:40:06.000000000 +0200
+++ Fixpack/atweaks/lib/310-creature_colouring.tpa	2017-01-22 13:33:13.022405300 +0100
@@ -40,6 +40,7 @@
 PATCH_INCLUDE ~aTweaks/lib/fj_cre_validity.tpp~                                    // Nythrun's CRE validity macro (detects corrupt and invalid CREs)
 PATCH_IF valid BEGIN
   PATCH_INCLUDE ~aTweaks/lib/fj_cre_reindex.tpp~                                   // Nythrun's CRE reindex macro (this also calls the EFF macro if needed)
+PATCH_IF (LONG_AT 0x28) = IDS_OF_SYMBOL (~animate~ ~GOLEM_IRON~) BEGIN             // modify only if standard creature animation is used
 // =============================================================================== // the actual work starts from here
 LAUNCH_PATCH_FUNCTION ~DELETE_CRE_EFFECT~                                          // delete the EFF with the designated opcode (remove any existent tint)
 INT_VAR opcode_to_delete = "51" END                                                // mark opcode #51 (Colour: Strong/Dark by RGB) for deletion
@@ -94,6 +95,7 @@
 END                                                                                // ends PATCH_FOR_EACH
 WRITE_BYTE  0x2E "23"                                                              // major color (Light Copper)
 // =============================================================================== // the actual work ends here
+END                                                                                // ends creature animation check
 END                                                                                // ends file validity check
 BUT_ONLY_IF_IT_CHANGES
 END                                                                                // ends ACTION_IF FILE_EXISTS_IN_GAME block
@@ -116,6 +118,7 @@
 PATCH_INCLUDE ~aTweaks/lib/fj_cre_validity.tpp~                                    // Nythrun's CRE validity macro (detects corrupt and invalid CREs)
 PATCH_IF valid BEGIN
   PATCH_INCLUDE ~aTweaks/lib/fj_cre_reindex.tpp~                                   // Nythrun's CRE reindex macro (this also calls the EFF macro if needed)
+PATCH_IF (LONG_AT 0x28) = IDS_OF_SYMBOL (~animate~ ~GOLEM_CLAY~) BEGIN             // modify only if standard creature animation is used
 // =============================================================================== // the actual work starts from here
 LAUNCH_PATCH_FUNCTION ~DELETE_CRE_EFFECT~                                          // delete the EFF with the designated opcode (remove any existent tint)
 INT_VAR opcode_to_delete = "51" END                                                // mark opcode #51 (Colour: Strong/Dark by RGB) for deletion
@@ -170,6 +173,7 @@
 END                                                                                // ends PATCH_FOR_EACH
 WRITE_BYTE  0x2E "5"                                                               // major color (Light Silver)
 // =============================================================================== // the actual work ends here
+END                                                                                // ends creature animation check
 END                                                                                // ends file validity check
 BUT_ONLY_IF_IT_CHANGES
 END                                                                                // ends ACTION_IF FILE_EXISTS_IN_GAME block
@@ -191,6 +195,7 @@
 PATCH_INCLUDE ~aTweaks/lib/fj_cre_validity.tpp~                                    // Nythrun's CRE validity macro (detects corrupt and invalid CREs)
 PATCH_IF valid BEGIN
   PATCH_INCLUDE ~aTweaks/lib/fj_cre_reindex.tpp~                                   // Nythrun's CRE reindex macro (this also calls the EFF macro if needed)
+PATCH_IF (LONG_AT 0x28) = IDS_OF_SYMBOL (~animate~ ~GOLEM_CLAY~) BEGIN             // modify only if standard creature animation is used
 // =============================================================================== // the actual work starts from here
 LAUNCH_PATCH_FUNCTION ~DELETE_CRE_EFFECT~                                          // delete the EFF with the designated opcode (remove any existent tint)
 INT_VAR opcode_to_delete = "51" END                                                // mark opcode #51 (Colour: Strong/Dark by RGB) for deletion
@@ -245,6 +250,7 @@
 END                                                                                // ends PATCH_FOR_EACH
 WRITE_BYTE  0x2E "4"                                                               // major color (Auburn)
 // =============================================================================== // the actual work ends here
+END                                                                                // ends creature animation check
 END                                                                                // ends file validity check
 BUT_ONLY_IF_IT_CHANGES
 END                                                                                // ends ACTION_IF FILE_EXISTS_IN_GAME block
@@ -262,6 +268,7 @@
 PATCH_INCLUDE ~aTweaks/lib/fj_cre_validity.tpp~                                    // Nythrun's CRE validity macro (detects corrupt and invalid CREs)
 PATCH_IF valid BEGIN
   PATCH_INCLUDE ~aTweaks/lib/fj_cre_reindex.tpp~                                   // Nythrun's CRE reindex macro (this also calls the EFF macro if needed)
+PATCH_IF (LONG_AT 0x28) = IDS_OF_SYMBOL (~animate~ ~GOLEM_CLAY~) BEGIN             // modify only if standard creature animation is used
 // =============================================================================== // the actual work starts from here
 LAUNCH_PATCH_FUNCTION ~DELETE_CRE_EFFECT~                                          // delete the EFF with the designated opcode (remove any existent tint)
 INT_VAR opcode_to_delete = "51" END                                                // mark opcode #51 (Colour: Strong/Dark by RGB) for deletion
@@ -316,6 +323,7 @@
 END                                                                                // ends PATCH_FOR_EACH
 WRITE_BYTE  0x2E "37"                                                              // major color (Dark Dirty Yellow)
 // =============================================================================== // the actual work ends here
+END                                                                                // ends creature animation check
 END                                                                                // ends file validity check
 BUT_ONLY_IF_IT_CHANGES
 END                                                                                // ends ACTION_IF FILE_EXISTS_IN_GAME block
@@ -335,6 +343,7 @@
 PATCH_INCLUDE ~aTweaks/lib/fj_cre_validity.tpp~                                    // Nythrun's CRE validity macro (detects corrupt and invalid CREs)
 PATCH_IF valid BEGIN
   PATCH_INCLUDE ~aTweaks/lib/fj_cre_reindex.tpp~                                   // Nythrun's CRE reindex macro (this also calls the EFF macro if needed)
+PATCH_IF (LONG_AT 0x28) = IDS_OF_SYMBOL (~animate~ ~GOLEM_CLAY~) BEGIN             // modify only if standard creature animation is used
 // =============================================================================== // the actual work starts from here
 LAUNCH_PATCH_FUNCTION ~DELETE_CRE_EFFECT~                                          // delete the EFF with the designated opcode (remove any existent tint)
 INT_VAR opcode_to_delete = "51" END                                                // mark opcode #51 (Colour: Strong/Dark by RGB) for deletion
@@ -389,6 +398,7 @@
 END                                                                                // ends PATCH_FOR_EACH
 WRITE_BYTE  0x2E "10"                                                              // major color (Light Pure Gold)
 // =============================================================================== // the actual work ends here
+END                                                                                // ends creature animation check
 END                                                                                // ends file validity check
 BUT_ONLY_IF_IT_CHANGES
 END                                                                                // ends ACTION_IF FILE_EXISTS_IN_GAME block
