--- atweaks/setup-atweaks.tp2	2016-01-13 13:36:39.000000000 -0500
+++ Fixpack/atweaks/setup-atweaks.tp2	2016-11-29 23:02:31.102891400 -0500
@@ -8,7 +8,7 @@
 
 BACKUP ~atweaks/backup~
 AUTHOR "Wisp, shsforums.net"                                                       // Author and e-mail address
-VERSION ~v4.50~                                                                    // Mod version
+VERSION ~v4.50 BWP Fix~                                                            // BiG World Fixpack added Polish translation
 
 ALWAYS                                                                             // Always do this block
   INCLUDE ~aTweaks/LIB/RR#AGLEF.TPH~                                               // Include my custom "Add Global Effect" function
@@ -58,6 +58,8 @@
 LANGUAGE ~Russian (by the Aerie.ru team and the Arcanecoast.ru team)~ ~RUSSIAN~ ~atweaks/tra/english/setup.tra~ ~atweaks/tra/english/game.tra~
                                                                                 ~atweaks/tra/russian/game.tra~
 
+LANGUAGE ~Polish (by Kinski, Bartek and Cahir)~ ~POLISH~ ~atweaks/tra/english/setup.tra~ ~atweaks/tra/english/game.tra~
+                                                         ~atweaks/tra/polish/setup.tra~ ~atweaks/tra/polish/game.tra~
 
 
 // ******************************************************* PnP tweaks *************************************************************************************
