--- atweaks/setup-atweaks.tp2	2017-05-01 15:22:38.000000000 +0200
+++ Fixpack/atweaks/setup-atweaks.tp2	2017-06-21 21:44:07.716002800 +0200
@@ -8,7 +8,7 @@
 
 BACKUP ~atweaks/backup~
 AUTHOR "Wisp, shsforums.net"                                                       // Author and e-mail address
-VERSION ~v4.52~                                                                    // Mod version
+VERSION ~v4.52 BWP Fix~                                                                    // Mod version
 
 ALWAYS                                                                             // Always do this block
   INCLUDE ~aTweaks/LIB/RR#AGLEF.TPH~                                               // Include my custom "Add Global Effect" function
