--- Horace\Dialogue\BHorace.d	Tue Sep 24 00:59:34 2002
+++ C:\BWP Patchstudio\patched files\Horace\Dialogue\BHorace.d	Sat Feb 01 10:41:17 2014
@@ -64,6 +64,9 @@
 IF ~InParty("Jan")
 See("Jan")
 !StateCheck("Jan",STATE_SLEEPING)
+InParty("Horace")
+See("Horace")
+!StateCheck("Horace",STATE_SLEEPING)
 Global("HorJan1","LOCALS",0)~ THEN BJAN Freeze
 @23
 DO ~SetGlobal("HorJan1","LOCALS",1)~
@@ -85,6 +88,9 @@
 IF ~InParty("Korgan")
 See("Korgan")
 !StateCheck("Korgan",STATE_SLEEPING)
+InParty("Horace")
+See("Horace")
+!StateCheck("Horace",STATE_SLEEPING)
 Global("HorKor1","LOCALS",0)~ THEN BKORGAN Abuse
 @30
 DO ~SetGlobal("HorKor1","LOCALS",1)~
@@ -141,6 +147,9 @@
 IF ~InParty("Edwin")
 See("Edwin")
 !StateCheck("Edwin",STATE_SLEEPING)
+InParty("Horace")
+See("Horace")
+!StateCheck("Horace",STATE_SLEEPING)
 Global("HorEdd1","LOCALS",1)
 Global("EdwintalksHoraceJames","LOCALS",1)
 Global("HorEdd2","LOCALS",0)~ THEN BEDWIN BDonation
@@ -191,6 +200,9 @@
 IF ~InParty("Mazzy")
 See("Mazzy")
 !StateCheck("Mazzy",STATE_SLEEPING)
+InParty("Horace")
+See("Horace")
+!StateCheck("Horace",STATE_SLEEPING)
 Global("HorMaz1","LOCALS",1)
 Global("HorMaz2","LOCALS",0)~ THEN BMAZZY Observation
 @68
@@ -209,6 +221,9 @@
 IF ~InParty("Aerie")
 See("Aerie")
 !StateCheck("Aerie",STATE_SLEEPING)
+InParty("Horace")
+See("Horace")
+!StateCheck("Horace",STATE_SLEEPING)
 Global("HorAer1","LOCALS",0)~ THEN BAERIE Worship
 @73
 DO ~SetGlobal("HorAer1","LOCALS",1)~
@@ -232,6 +247,9 @@
 IF ~InParty("Aerie")
 See("Aerie")
 !StateCheck("Aerie",STATE_SLEEPING)
+InParty("Horace")
+See("Horace")
+!StateCheck("Horace",STATE_SLEEPING)
 Global("HorAer1","LOCALS",1)
 Global("HorAer2","LOCALS",0)~ THEN BAERIE Apology
 @81
@@ -260,6 +278,9 @@
 IF ~InParty("Keldorn")
 See("Keldorn")
 !StateCheck("Keldorn",STATE_SLEEPING)
+InParty("Horace")
+See("Horace")
+!StateCheck("Horace",STATE_SLEEPING)
 Global("HorKeld1","LOCALS",0)~ THEN BKELDOR Conditioning
 @91
 DO ~SetGlobal("HorKeld1","LOCALS",1)~
