--- kittomes\setup-kittomes.tp2	Sun Jun 09 23:50:55 2013
+++ C:\BWP Patchstudio\patched files\kittomes\setup-kittomes.tp2	Sun Jul 21 07:27:19 2013
@@ -36,9 +36,9 @@
 //--------------------------------------------
 //Invisibile Creature
 COPY ~kittomes/CRE/S9KTAdd.cre~ ~override~
-COMPILE ~kittomes/BAF/S9KTAdd.baf~ ~override~
+COMPILE ~kittomes/BAF/S9KTAdd.baf~
 //Dialogues
-COMPILE EVALUATE_BUFFER ~kittomes/dlg~ ~override~
+COMPILE EVALUATE_BUFFER ~kittomes/dlg~
 //--------------------------------------------
 //Create Tomes; adds current descriptions of classes to them
 //====================================================================
