--- imoenRomance\dialogues\SOA\Banter\AlignLaw.d	Sun Dec 06 00:54:02 2015
+++ C:\BWP Patchstudio\patched files\imoenRomance\dialogues\SOA\Banter\AlignLaw.d	Sun Jan 10 22:40:08 2016
@@ -182,7 +182,7 @@
 	END
 
 	IF ~~ THEN BEGIN MRLawChaChaPlay
-		SAY ~Haha! Whoa there <PRO_BOYGIRL>! (She grins at you happily) I'm not sure I trust you with *total* anarchy, you nut.~
+		SAY ~Haha! Whoa there <PRO_GIRLBOY>! (She grins at you happily) I'm not sure I trust you with *total* anarchy, you nut.~
 		=
 		~I still wanna be 'Her Royal Highness' for at least a day...~
 		IF ~~ THEN GOTO MRLawChaLawEnd
