--- imoenRomance\dialogues\SOA\Banter\AlignLaw.d	Mon Jul 25 12:49:44 2016
+++ C:\BWP Patchstudio\patched files\imoenRomance\dialogues\SOA\Banter\AlignLaw.d	Sat Oct 08 19:40:23 2016
@@ -183,7 +183,7 @@
 	END
 
 	IF ~~ THEN BEGIN MRLawChaChaPlay
-		SAY ~Haha! Whoa there <PRO_BOYGIRL>! (She grins at you happily) I'm not sure I trust you with *total* anarchy, you nut.~
+		SAY ~Haha! Whoa there <PRO_GIRLBOY>! (She grins at you happily) I'm not sure I trust you with *total* anarchy, you nut.~
 		=
 		~I still wanna be 'Her Royal Highness' for at least a day...~
 		IF ~~ THEN GOTO MRLawChaLawEnd
