--- imoenRomance\dialogues\SOA\Love\lovetalks.d	Fri Jan 01 00:29:18 2016
+++ C:\BWP Patchstudio\patched files\imoenRomance\dialogues\SOA\Love\lovetalks.d	Sun Jan 10 22:40:10 2016
@@ -469,7 +469,7 @@
 	END
 
 	IF ~~ MRLoveTalk3Ears
-	 SAY ~Riiight. Okay then, ear-<PRO_BOYGIRL>, I'm gonna let you and your body parts have a little alone time.~
+	 SAY ~Riiight. Okay then, ear-<PRO_GIRLBOY>, I'm gonna let you and your body parts have a little alone time.~
 	 = ~What I was gonna say wasn't all that important anyways.~
 	 IF ~~ THEN DO ~SetGlobal("ImoenRomanceActive","GLOBAL",4)~EXIT
 	END
