--- imoenRomance\dialogues\SOA\Love\lovetalks.d	Sun Dec 07 17:26:58 2014
+++ C:\BWP Patchstudio\patched files\imoenRomance\dialogues\SOA\Love\lovetalks.d	Wed Nov 11 13:41:53 2015
@@ -469,7 +469,7 @@
 	END
 
 	IF ~~ MRLoveTalk3Ears
-	 SAY ~Riiight. Okay then, ear-<PRO_BOYGIRL>, I'm gonna let you and your body parts have a little alone time.~
+	 SAY ~Riiight. Okay then, ear-<PRO_GIRLBOY>, I'm gonna let you and your body parts have a little alone time.~
 	 = ~What I was gonna say wasn't all that important anyways.~
 	 IF ~~ THEN DO ~SetGlobal("ImoenRomanceActive","GLOBAL",3)~EXIT
 	END
