--- imoenRomance\dialogues\TOB\Love\QualityTimes.d	Wed Nov 26 20:36:20 2014
+++ C:\BWP Patchstudio\patched files\imoenRomance\dialogues\TOB\Love\QualityTimes.d	Wed Nov 11 13:41:53 2015
@@ -240,7 +240,7 @@
 
 	IF ~~ THEN BEGIN MRLT0-Sorry
 		SAY ~Good!  You oughta be!~
-		= ~Now be a good little <PRO_BOYGIRL> and leave me alone!~
+		= ~Now be a good little <PRO_GIRLBOY> and leave me alone!~
 		= ~Go back to whatever you were doing and let me watch what I want, I command it! (She pushes you away and goes back to her bird-watching.)~
 		= ~(As you watch Imoen smile contentedly, you can't help but admire the way she always manages to put you in your place.)~
 		IF ~~ THEN  EXIT
