--- imoenRomance\dialogues\SOA\Love\qualitytimes.d	Mon Nov 23 19:06:12 2015
+++ C:\BWP Patchstudio\patched files\imoenRomance\dialogues\SOA\Love\qualitytimes.d	Sun Jan 10 22:40:09 2016
@@ -237,7 +237,7 @@
 
 	IF ~~ THEN BEGIN MRLT0-Sorry
 		SAY ~Good! You oughta be!~
-		= ~Now be a good little <PRO_BOYGIRL>, go back to whatever you were doing and let me watch what I want, I command it! (She pushes you away and goes back to her bird-watching.)~
+		= ~Now be a good little <PRO_GIRLBOY>, go back to whatever you were doing and let me watch what I want, I command it! (She pushes you away and goes back to her bird-watching.)~
 		IF ~~ THEN EXIT
 	END
 
