--- Setup-SpellPackB6.TP2	Tue May 17 21:11:21 2011
+++ C:\BWP Patchstudio\patched files\Setup-SpellPackB6.TP2	Sun Feb 07 22:54:59 2016
@@ -1,7 +1,7 @@
 BACKUP ~SpellPackB6/Backup~
 AUTHOR ~galactygon@baraban.com~
+VERSION ~BWP Fix 2~
 README ~SpellPackB6/README.html~
-
 ALWAYS
 	// Macros
 	INCLUDE ~SpellPackB6/Macros/add_projectile.tph~
@@ -2353,7 +2353,8 @@
 BEGIN @15390	DESIGNATED 1539 	GROUP @10002
 REQUIRE_FILE ~Data/25Dialog.bif~ @2 // ToB  check
 
-OUTER_SPRINT add_spell_ids "CLERIC_SHIELD_OF_LATHANDER"
+APPEND ~SPELL.IDS~ ~1738 CLERIC_GREATER_SHIELD_OF_LATHANDER~
+ UNLESS ~1738 CLERIC_GREATER_SHIELD_OF_LATHANDER~
 OUTER_SPRINT add_spell_res "LCPR539" // ACTION_MACRO ~ADD_SPELL~ sets this variable to the added spell's SOURCE_RES
 OUTER_SPRINT add_spell_scroll "*" 
 OUTER_SPRINT sphere_restrictions "GUARDIAN"
@@ -3090,7 +3091,8 @@
 BEGIN @17430	DESIGNATED 1743 	GROUP @10003
 REQUIRE_FILE ~Data/25Dialog.bif~ @2 // ToB  check
 
-OUTER_SPRINT add_spell_ids "CLERIC_MIST_OF_ELDATH"
+APPEND ~SPELL.IDS~ ~1739 CLERIC_MIST_OF_ELDATH~
+ UNLESS ~1739 CLERIC_MIST_OF_ELDATH~
 OUTER_SPRINT add_spell_res "LCPR743" // ACTION_MACRO ~ADD_SPELL~ sets this variable to the added spell's SOURCE_RES
 OUTER_SPRINT add_spell_scroll "*" 
 OUTER_SPRINT sphere_restrictions "WEATHER"
@@ -3996,6 +3998,6 @@
 // Earlier components used macros to patch REGEXP GLOB spell effects. MACRO_MATRIX makes sure the new files we copied are up to date with the new effects.
-COPY_EXISTING_REGEXP GLOB ~%add_spell_res%~ ~override~
+COPY_EXISTING_REGEXP GLOB ~%add_spell_res%.spl~ ~override~
                           ~LCWI240.*.spl~ ~override~
                           ~LC_HPPA.*.spl~ ~override~
 	LAUNCH_PATCH_MACRO ~MACRO_MATRIX~
 BUT_ONLY_IF_IT_CHANGES
