--- hammers\setup-hammers.tp2	Mon Sep 15 11:17:54 2014
+++ C:\BWP Patchstudio\patched files\hammers\setup-hammers.tp2	Fri Jun 26 22:38:54 2015
@@ -1,6 +1,6 @@
 BACKUP ~hammers/backup~
 AUTHOR ~www.shsforums.net/topic/34099-thrown-hammers/~
-VERSION ~v4.1~
+VERSION ~v4.1 BWP Fix~
 README ~hammers/hammers-%LANGUAGE%.html~ ~hammers/hammers-english.html~
 
 ALWAYS
@@ -82,7 +82,7 @@
         SPRINT rz ~%rz%~ ^ ~%rx%~ ^ ~%ry%~
       END
     END ELSE BEGIN
-      FOR (i = 3; i < (cl - 1); i += 1) BEGIN //Cycle through columns
+      FOR (i = 4; i < (cl - 1); i += 1) BEGIN //Cycle through columns
         SPRINT cz ~%cz%~ ^ ~%rx%~ ^ ~%ry%~
       END
     END
