--- hammers\setup-hammers.tp2	Sun Oct 31 00:36:34 2010
+++ C:\BWP Patchstudio\patched files\hammers\setup-hammers.tp2	Sat Oct 05 12:16:08 2013
@@ -1,6 +1,6 @@
 BACKUP ~hammers/backup~
 AUTHOR ~www.shsforums.net/topic/34099-thrown-hammers/~
-VERSION ~v3~
+VERSION ~v3 BWP Fix~
 README ~hammers/hammers-%LANGUAGE%.html~ ~hammers/hammers-english.html~
 
 ALWAYS
@@ -82,7 +82,7 @@
         SPRINT rz ~%rz%~ ^ ~%rx%~ ^ ~%ry%~
       END
     END ELSE BEGIN
-      FOR (i = 3; i < (cl - 1); i += 1) BEGIN //Cycle through columns
+      FOR (i = 4; i < (cl - 1); i += 1) BEGIN //Cycle through columns
         SPRINT cz ~%cz%~ ^ ~%rx%~ ^ ~%ry%~
       END
     END
