--- bolsa\bolsa.tp2	Wed Nov 04 19:43:14 2009
+++ C:\BWP Patchstudio\patched files\bolsa\bolsa.tp2	Sun Sep 29 00:49:38 2013
@@ -1,6 +1,6 @@
 BACKUP ~bolsa/backup~
 AUTHOR ~www.shsforums.net/index.php?showforum=234~ //aimainahikari@hotmail.com
-VERSION ~v4.1~
+VERSION ~v4.1 BWP Fix~
 README ~bolsa/bolsa-%LANGUAGE%.txt~
 
 AUTO_TRA ~bolsa/%s~
@@ -78,4 +78,4 @@
   SAY NAME1 @3
   SAY NAME2 @3
 
-EXTEND_BOTTOM ~ar0702.bcs~ ~bolsa/mhbolsa.baf~
+EXTEND_BOTTOM ~ar0511.bcs~ ~bolsa/mhbolsa.baf~
