--- nephele/scripts/lk#neph.baf	2013-02-07 16:53:02.000000000 -0500
+++ Fixpack/nephele/scripts/lk#neph.baf	2016-11-04 16:10:40.583838800 -0400
@@ -204,7 +204,7 @@
     See(Player1)
     GlobalGT("LK#NephFriendTalks","GLOBAL",8)
     Global("LK#NepheleCooking","GLOBAL",0)
-    !Global("chapter","GLOBAL",5)
+    !Global("chapter","GLOBAL",%bg2_chapter_5%)
 THEN
   RESPONSE #100
     SetGlobal("LK#NepheleCooking","GLOBAL",1)
@@ -367,7 +367,7 @@
     InParty(Myself)
     See(Player1)
     AreaCheck("AR0700")
-    GlobalGT("chapter","GLOBAL",5)
+    GlobalGT("chapter","GLOBAL",%bg2_chapter_5%)
     Global("LK#NepheleWPTalk","GLOBAL",0)
 THEN
   RESPONSE #100
