--- nephele\nephele.tp2	Mon Nov 11 15:41:31 2013
+++ C:\BWP Patchstudio\patched files\nephele\nephele.tp2	Sat Dec 14 22:11:26 2013
@@ -4,7 +4,7 @@
 // Basics
 BACKUP ~nephele/backup~
 AUTHOR ~K'aeloree (kae@spellholdstudios.net)~
-VERSION ~v2.2~
+VERSION ~v2.3 BWP Fix~
 README ~nephele/nephele_readme.html~
 
 AUTO_TRA ~nephele/tra/%s~
@@ -85,7 +85,8 @@
   WRITE_EVALUATED_ASCII DEATHVAR        ~Nephele~ #32
   WRITE_EVALUATED_ASCII SCRIPT_OVERRIDE ~%DEST_RES%~ #8
   WRITE_LONG 0x18 190000
-  WRITE_BYTE  0x246 ~%lkneph%~
+  WRITE_SHORT 0x244 0
+  WRITE_SHORT 0x246 0x4000 + ~%lkneph%~
   ADD_CRE_ITEM ~slng02~ #0 #0 #0 ~IDENTIFIED~ ~WEAPON1~ EQUIP
   ADD_CRE_ITEM ~bull02~ #20 #0 #0 ~IDENTIFIED~ ~QUIVER1~
   SAY BIO @7
@@ -136,7 +137,8 @@
   WRITE_EVALUATED_ASCII DEATHVAR        ~Nephele~ #32
   WRITE_EVALUATED_ASCII SCRIPT_OVERRIDE ~%DEST_RES%~ #8
   WRITE_LONG 0x18 2500000
-  WRITE_BYTE  0x246 ~%lkneph%~
+  WRITE_SHORT 0x244 0
+  WRITE_SHORT 0x246 0x4000 + ~%lkneph%~
   ADD_CRE_ITEM ~slng04~ #0 #0 #0 ~IDENTIFIED~ ~WEAPON1~ EQUIP
   ADD_CRE_ITEM ~bull05~ #20 #0 #0 ~IDENTIFIED~ ~QUIVER1~
   SAY BIO @7
