--- cdtweaks\setup-cdtweaks.tp2	Tue Jul 12 16:31:39 2016
+++ C:\BWP Patchstudio\patched files\setup-cdtweaks.tp2	Tue Jul 12 16:31:43 2016
@@ -29,3 +29,3 @@
 
-VERSION ~Beta 4~
+VERSION ~Beta 4 BWP Fix~
 
@@ -5594,6 +5594,7 @@
 SUBCOMPONENT @215000 // wear multiple protection items
 
 INCLUDE ~cdtweaks/lib/pnp_descripts.tpa~  // text replacement macros
+INCLUDE ~cdtweaks/lib/fj_spl_itm_reindex.tpa~
 
 // first, eliminate iwd items on list without AC bonuses
 COPY_EXISTING ~itemexcl.2da~ ~override~
@@ -5655,6 +5656,7 @@
       ACTION_IF FILE_EXISTS_IN_GAME ~%orig%.itm~ THEN BEGIN
 
         COPY_EXISTING ~%orig%.itm~ ~override~
+          LPF ~FJ_SPL_ITM_REINDEX~ END
           READ_ASCII  0x3a bam (8)
           READ_SHORT  0x42 lore
           PATCH_IF (lore = 0) BEGIN
