--- dace\scripts\ar4000.baf	Wed Aug 28 16:53:30 2013
+++ C:\BWP Patchstudio\patched files\dace\scripts\ar4000.baf	Tue Sep 27 16:29:59 2016
@@ -11,8 +11,8 @@
 THEN
   RESPONSE #100
     SetGlobal("T#DaceExistsToBJoin","GLOBAL",1)
-    CreateCreature("T#DAC25",[440.1000],3)
+    CreateCreature("T#DACE25",[440.1000],3)
     ActionOverride("T#Dace",MoveToPointNoInterrupt([830.800]))
     Wait(1)
     ActionOverride("T#Dace",StartDialogueNoSet(Player1))
-END
\ No newline at end of file
+END
