--- SmilingImpCrossBanterMod\SharTeelTsuki.d	Fri Mar 21 16:23:52 2014
+++ C:\BWP Patchstudio\patched files\SmilingImpCrossBanterMod\SharTeelTsuki.d	Sun Jan 04 19:48:21 2015
@@ -24,7 +24,7 @@
 == L7TsukiB ~Is wasn't true, but becasue of the lie, I was hunted by my step-brother, Yoshida. What's more it was him who started the farce, not me. Surprisngly.~
 == B7Xshar ~Why should I believe anyone that is in league with vampires?~
 == L7TsukiB ~Believe what you want. I don't care.~
-== B7Xshar ~Tsk-tsk.. a rapist and a cowardly slave to vampires eh? Just don't stand behind me in a dark corridor in battle. I hate for there to be an accident.~'
+== B7Xshar ~Tsk-tsk.. a rapist and a cowardly slave to vampires eh? Just don't stand behind me in a dark corridor in battle. I hate for there to be an accident.~
 == L7TsukiB ~Bitch, I'm not letting you get behind me at all.~
 EXIT
 
