--- Setup-NeJ2.tp2	Mon Jul 25 11:46:34 2005
+++ C:\BWP Patchstudio\patched files\Setup-NeJ2.tp2	Thu Nov 21 18:26:35 2013
@@ -2,6 +2,8 @@
 
 AUTHOR ~Vlad & Larrienne~
 
+VERSION ~v4.2b BWP Fix~
+
 BEGIN ~Never Ending Journey v4.2 - Part 1~
 
 //______________________________________//
@@ -13,7 +15,7 @@
 REQUIRE_FILE ~Data/25Dialog.bif~ ~Please install "Throne Of Bhaal" and the official ToB patch before installation of this mod.~
 REQUIRE_FILE ~override/Animate.ids~ ~Please install the official ToB patch before installation of this mod.~
 REQUIRE_FILE ~override/MusicList.ids~ ~Please install the official ToB patch before installation of this mod.~
-REQUIRE_FILE ~override/CLASS.ids~ ~Please install the official ToB patch before installation of this mod.~
+//REQUIRE_FILE ~override/CLASS.ids~ ~Please install the official ToB patch before installation of this mod.~
 REQUIRE_FILE ~override/AreaType.ids~ ~Please install the official ToB patch before installation of this mod.~
 REQUIRE_FILE ~override/Spell.ids~ ~Please install the Baldurdash fixpack before installation of this mod.~
 
@@ -5716,8 +5718,8 @@
 COPY ~NeJ2/are/TT0079.wed~           ~override/TT0079.wed~
 COPY ~NeJ2/are/TT0079.are~           ~override/TT0079.are~
   
-COPY ~Intro/Combo1/combo1b.acm~				~music/Combo1/combo1b.acm~
-COPY ~Intro/Theme/themea.acm~					~music/Theme/themea.acm~
+// COPY ~Intro/Combo1/combo1b.acm~				~music/Combo1/combo1b.acm~
+// COPY ~Intro/Theme/themea.acm~					~music/Theme/themea.acm~
 
 /*****************
 ***** Melora *****
