--- Xan/Scripts/Baldur.baf	2007-07-07 00:30:50.000000000 -0400
+++ Fixpack/Xan/Scripts/Baldur.baf	2016-11-21 19:54:09.372860500 -0500
@@ -1,7 +1,7 @@
 IF
 Dead("O#Xan")
 !InPartyAllowDead("O#Xan")
-GlobalLT("Chapter","GLOBAL",8)
+GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)
 Global("O#XanRomanceActive","GLOBAL",2)
 !Global("O#XanMoonbladeExtinguished","GLOBAL",3)
 Global("O#XanBackFromTheDead","GLOBAL",0)
@@ -53,9 +53,9 @@
 PartyRested()
 AreaType(OUTDOOR)
 OR(3)
-Global("Chapter","GLOBAL",2)
-Global("Chapter","GLOBAL",3)
-Global("Chapter","GLOBAL",6)
+Global("Chapter","GLOBAL",%bg2_chapter_2%)
+Global("Chapter","GLOBAL",%bg2_chapter_3%)
+Global("Chapter","GLOBAL",%bg2_chapter_6%)
 THEN
 RESPONSE #100
 CreateCreatureObjectOffset("O#Hiel",Player1,[30.25]) 
