--- Xan/Scripts/O#DrowF.baf	2007-06-14 23:20:50.000000000 -0400
+++ Fixpack/Xan/Scripts/O#DrowF.baf	2016-11-21 20:18:38.003759400 -0500
@@ -1,5 +1,5 @@
 IF
-Global("Chapter","GLOBAL",5)
+Global("Chapter","GLOBAL",%bg2_chapter_5%)
 OR(4)
 GlobalGT("DemonWalks","GLOBAL",0)
 Dead("Phaere")
@@ -13,7 +13,7 @@
 END
 
 IF
-Global("Chapter","GLOBAL",5)
+Global("Chapter","GLOBAL",%bg2_chapter_5%)
 AttackedBy([GOODCUTOFF],DEFAULT)
 Global("O#IDidThis","LOCALS",0)
 THEN
@@ -23,7 +23,7 @@
 END
 
 IF
-Global("Chapter","GLOBAL",6)
+Global("Chapter","GLOBAL",%bg2_chapter_6%)
 Global("O#XanRicarQuest","GLOBAL",11)
 Global("O#ITalkedDidThis","LOCALS",0)
 OR(6)
