--- Xan/Scripts/O#Xan25D.baf	2013-03-07 09:30:06.000000000 -0500
+++ Fixpack/Xan/Scripts/O#Xan25D.baf	2016-11-21 16:33:18.579393300 -0500
@@ -105,7 +105,7 @@
 Global("O#XanToBNightTalk3","GLOBAL",0)
 InParty(Myself)
 GlobalGT("O#XanKeepEntered","GLOBAL",1)
-Global("Chapter","GLOBAL",9)
+Global("Chapter","GLOBAL",%bg2_chapter_9%)
 !AreaCheck("AR3000")
 AreaType(OUTDOOR)
 TimeOfDay(NIGHT)
