--- Xan/Dialogue/O#Xan25J.d	2013-09-11 21:23:52.000000000 -0400
+++ Fixpack/Xan/Dialogue/O#Xan25J.d	2016-11-21 16:32:57.063804800 -0500
@@ -131,50 +131,6 @@
 == SOLAR IF ~Global("O#XanBondedPathAlive","GLOBAL",2)~ THEN @2182
 END
 
-EXTEND_BOTTOM FATESP 6 #2
-
-+ ~!Dead("O#Xan") !InMyArea("O#Xan") Global("O#XanSummoned","GLOBAL",0) !Global("O#XanRT15","GLOBAL",2) !Global("O#XanBondedPathAlive","GLOBAL",2)~ + @34
-DO ~CreateVisualEffect("SPPORTAL",[1999.1228])
-Wait(2) 
-CreateCreature("O#Xan25",[1999.1228],0) 
-SetGlobal("O#XanSummoned","GLOBAL",1) 
-SetGlobal("O#XanFriendship","GLOBAL",1)~ GOTO 8
-
-+ ~!Dead("O#Xan") !InMyArea("O#Xan") Global("O#XanSummoned","GLOBAL",0) Global("O#XanRT15","GLOBAL",2)~ + @35
-DO ~CreateVisualEffect("SPPORTAL",[1999.1228]) 
-Wait(2) 
-CreateCreature("O#Xan25",[1999.1228],0) 
-SetGlobal("O#XanSummoned","GLOBAL",1)~ GOTO 8
-
-+ ~!Dead("O#Xan") !InMyArea("O#Xan") Global("O#XanJoined","GLOBAL",0) Global("O#XanSummoned","GLOBAL",0) Gender(Player1,FEMALE) Race(Player1,ELF)~ + @35 
-DO ~CreateVisualEffect("SPPORTAL",[1999.1228]) 
-Wait(2) 
-CreateCreature("O#Xan25",[1999.1228],0) 
-SetGlobal("O#XanRT15","GLOBAL",2) 
-SetGlobal("O#XanSummoned","GLOBAL",1)~ GOTO 8
-
-+ ~Dead("O#Xan") Global("O#XanSummoned","GLOBAL",0) Global("O#XanMoonbladeExtinguished","GLOBAL",3)~ + @36 
-DO ~SetGlobal("O#XanSummoned","GLOBAL",3)~ + O#XanFateSpirit1
-
-+ ~!Dead("O#Xan") !InMyArea("O#Xan") Global("O#XanSummoned","GLOBAL",0) Global("O#XanJoined","GLOBAL",0) Gender(Player1,FEMALE) Race(Player1,ELF)~ + @2166
-DO ~CreateVisualEffect("SPPORTAL",[1999.1228]) 
-Wait(2)
-CreateCreature("O#Xan25",[1999.1228],0)
-SetGlobal("O#XanBondedPathAlive","GLOBAL",2)
-SetGlobal("O#XanRomanceActive","GLOBAL",2)
-SetGlobal("O#XanSummoned","GLOBAL",1)~ GOTO 8
-
-+ ~!Dead("O#Xan") !InMyArea("O#Xan") Global("O#XanSummoned","GLOBAL",0) Global("O#XanJoined","GLOBAL",0) Gender(Player1,FEMALE) Race(Player1,ELF)~ + @36 
-DO ~SetGlobal("O#XanMoonbladeExtinguished","GLOBAL",3)
-SetGlobal("O#XanSummoned","GLOBAL",3)~ + O#XanFateSpirit1
-
-END
-
-CHAIN FATESP O#XanFateSpirit1
-@37
-= @398
-EXIT
-
 EXTEND_BOTTOM SARVOLO 9 #2
 + ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) !Global("O#XanRomanceActive","GLOBAL",2)~ + @38 + O#XanVoloBio1
 + ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) Global("O#XanRomanceActive","GLOBAL",2)~ + @38 + O#XanVoloBio2
@@ -305,117 +261,117 @@
 // Watcher's Keep interjections, ToB
 
 I_C_T GORAPR 10 O#XanApprentice1
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @69
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @69
 END
 
 I_C_T GORAPR 0 O#XanApprentice2
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @70
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @71
-== GORAPR IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @72
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @70
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @71
+== GORAPR IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @72
 END
 
 I_C_T GORARC1 3 O#XanArchivist1
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @73
-== GORARC1 IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @74
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @73
+== GORARC1 IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @74
 END
 
 I_C_T GORBAT1 7 O#XanLawfulEvilDemon1
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @75
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @75
 END
 
 I_C_T GORCAMB 17 O#XanCambion1
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @76
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @76
 END
 
 I_C_T GORCAR 18 O#XanCarston1
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @77
-== GORCAR IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @78 
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @77
+== GORCAR IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @78 
 END
 
 I_C_T GORCHR 7 O#XanChromaticDemon1
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @79
-== GORCHR IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @80
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @79
+== GORCHR IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @80
 END
 
 I_C_T GORCHR 13 O#XanChromaticDemon2
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @81
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @81
 END
 
 I_C_T GORDEMO 1 O#XanImprisonedOneMet1
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @82
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @82
 DO ~SetGlobal("O#XanSpokeToDemogorgon","GLOBAL",1)~
 END
 
 I_C_T GORDEMO 18 O#XanImprisonedOneDeceit1
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @83
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @83
 END
 
 I_C_T GORDEMO 24 O#XanImprisonedOneFailed1
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @84
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @85
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @84
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @85
 END
 
 I_C_T GORGIT 2 O#XanGithyanki1
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @86
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @86
 END
 
 I_C_T GORIMP01 4 O#XanImpDeadWizards1
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @87
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @87
 END
 
 I_C_T GORMAD1 1 O#XanYakmanLonely1
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @2163
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @2163
 END
 
 I_C_T GORMAD1 3 O#XanYakmanLonely3
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @2163
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @2163
 END
 
 I_C_T GORMAD1 4 O#XanYakmanLonely4
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @2163
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @2163
 END
 
 I_C_T GORMAD1 5 O#XanYakmanLonely5
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @2163
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @2163
 END
 
 I_C_T GORMAD1 34 O#XanTamorlinRestored1
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @2164
-== GORMAD1 IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @2165
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @2164
+== GORMAD1 IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @2165
 END
 
 I_C_T GORODR1 35 O#XanOdrenSurprise1
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @90
-== GORODR1 IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @91
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @90
+== GORODR1 IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @91
 END
 
 I_C_T GORODR1 50 O#XanOdrenFoolishness1
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @92
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @93
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @92
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @93
 END
 
 I_C_T GORODR1 53 O#XanOdrenDemogorgonActually1
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @94
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @94
 END
 
 I_C_T GORPOL1 1 O#XanMetKnightsOfTheVigil1
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @95
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @95
 END
 
 I_C_T GORPOL1 8 O#XanGreatHorrorsOdren1
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @96
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @96
 END
 
 I_C_T GORSAL 6 O#XanSaladrexAmusing1
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @97
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @97
 END
 
 I_C_T GORSUC01 6 O#XanKissSuccubus1
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7) OR(2) Global("O#XanRomanceActive","GLOBAL",1) Global("O#XanRomanceActive","GLOBAL",2)~ THEN @98
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%) OR(2) Global("O#XanRomanceActive","GLOBAL",1) Global("O#XanRomanceActive","GLOBAL",2)~ THEN @98
 END
 
 I_C_T GORTAN1 4 O#XanTanarri1
-== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @99
+== O#XAN25J IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @99
 END
 
 
@@ -423,117 +379,117 @@
 // Watcher's Keep interjections, SoA
 
 I_C_T GORAPR 10 O#XanApprentice1
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @69
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @69
 END
 
 I_C_T GORAPR 0 O#XanApprentice2
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @70
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @71
-== GORAPR IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @72
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @70
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @71
+== GORAPR IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @72
 END
 
 I_C_T GORARC1 3 O#XanArchivist1
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @73
-== GORARC1 IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @74
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @73
+== GORARC1 IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @74
 END
 
 I_C_T GORBAT1 7 O#XanLawfulEvilDemon1
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @75
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @75
 END
 
 I_C_T GORCAMB 17 O#XanCambion1
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @76
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @76
 END
 
 I_C_T GORCAR 18 O#XanCarston1
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @77
-== GORCAR IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @78 
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @77
+== GORCAR IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @78 
 END
 
 I_C_T GORCHR 7 O#XanChromaticDemon1
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @79
-== GORCHR IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @80
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @79
+== GORCHR IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @80
 END
 
 I_C_T GORCHR 13 O#XanChromaticDemon2
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @81
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @81
 END
 
 I_C_T GORDEMO 1 O#XanImprisonedOneMet1
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @82
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @82
 DO ~SetGlobal("O#XanSpokeToDemogorgon","GLOBAL",1)~
 END
 
 I_C_T GORDEMO 18 O#XanImprisonedOneDeceit1
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @83
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @83
 END
 
 I_C_T GORDEMO 24 O#XanImprisonedOneFailed1
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @84
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @85
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @84
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @85
 END
 
 I_C_T GORGIT 2 O#XanGithyanki1
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @86
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @86
 END
 
 I_C_T GORIMP01 4 O#XanImpDeadWizards1
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @87
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @87
 END
 
 I_C_T GORMAD1 1 O#XanYakmanLonely1
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @2163
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @2163
 END
 
 I_C_T GORMAD1 3 O#XanYakmanLonely3
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @2163
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @2163
 END
 
 I_C_T GORMAD1 4 O#XanYakmanLonely4
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @2163
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @2163
 END
 
 I_C_T GORMAD1 5 O#XanYakmanLonely5
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @2163
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @2163
 END
 
 I_C_T GORMAD1 34 O#XanTamorlinRestored1
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @2164
-== GORMAD1 IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @2165
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @2164
+== GORMAD1 IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @2165
 END
 
 I_C_T GORODR1 35 O#XanOdrenSurprise1
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @90
-== GORODR1 IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @91
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @90
+== GORODR1 IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @91
 END
 
 I_C_T GORODR1 50 O#XanOdrenFoolishness1
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @92
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @93
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @92
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @93
 END
 
 I_C_T GORODR1 53 O#XanOdrenDemogorgonActually1
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @94
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @94
 END
 
 I_C_T GORPOL1 1 O#XanMetKnightsOfTheVigil1
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @95
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @95
 END
 
 I_C_T GORPOL1 8 O#XanGreatHorrorsOdren1
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @96
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @96
 END
 
 I_C_T GORSAL 6 O#XanSaladrexAmusing1
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @97
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @97
 END
 
 I_C_T GORSUC01 6 O#XanKissSuccubus1
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8) OR(2) Global("O#XanRomanceActive","GLOBAL",1) Global("O#XanRomanceActive","GLOBAL",2)~ THEN @98
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%) OR(2) Global("O#XanRomanceActive","GLOBAL",1) Global("O#XanRomanceActive","GLOBAL",2)~ THEN @98
 END
 
 I_C_T GORTAN1 4 O#XanTanarri1
-== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @99
+== O#XANJ IF ~InParty("O#Xan") InMyArea("O#Xan") !StateCheck("O#Xan",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @99
 END
 
 
@@ -4532,7 +4488,7 @@
 
 // Chapter 10
 
-IF ~Global("Chapter","GLOBAL",10)~ O#XanToBThronePID1
+IF ~Global("Chapter","GLOBAL",%bg2_chapter_10%)~ O#XanToBThronePID1
 SAY @153
 = @154
 IF ~~ EXIT
@@ -5482,16 +5438,16 @@
 + ~RandomNum(10,8)~ + @2898 + O#XanR11.8
 + ~RandomNum(10,9)~ + @2898 + O#XanR11.9
 + ~RandomNum(10,10)~ + @2898 + O#XanR11.10
-+ ~Global("Chapter","GLOBAL",9) RandomNum(10,1)~ + @2899 + O#XanR12.1
-+ ~Global("Chapter","GLOBAL",9) RandomNum(10,2)~ + @2899 + O#XanR12.2
-+ ~Global("Chapter","GLOBAL",9) RandomNum(10,3)~ + @2899 + O#XanR12.3
-+ ~Global("Chapter","GLOBAL",9) RandomNum(10,4)~ + @2899 + O#XanR12.4
-+ ~Global("Chapter","GLOBAL",9) RandomNum(10,5)~ + @2899 + O#XanR12.5
-+ ~Global("Chapter","GLOBAL",9) RandomNum(10,6)~ + @2899 + O#XanR12.6
-+ ~Global("Chapter","GLOBAL",9) RandomNum(10,7)~ + @2899 + O#XanR12.7
-+ ~Global("Chapter","GLOBAL",9) RandomNum(10,8)~ + @2899 + O#XanR12.8
-+ ~Global("Chapter","GLOBAL",9) RandomNum(10,9)~ + @2899 + O#XanR12.9
-+ ~Global("Chapter","GLOBAL",9) RandomNum(10,10)~ + @2899 + O#XanR12.10
++ ~Global("Chapter","GLOBAL",%bg2_chapter_9%) RandomNum(10,1)~ + @2899 + O#XanR12.1
++ ~Global("Chapter","GLOBAL",%bg2_chapter_9%) RandomNum(10,2)~ + @2899 + O#XanR12.2
++ ~Global("Chapter","GLOBAL",%bg2_chapter_9%) RandomNum(10,3)~ + @2899 + O#XanR12.3
++ ~Global("Chapter","GLOBAL",%bg2_chapter_9%) RandomNum(10,4)~ + @2899 + O#XanR12.4
++ ~Global("Chapter","GLOBAL",%bg2_chapter_9%) RandomNum(10,5)~ + @2899 + O#XanR12.5
++ ~Global("Chapter","GLOBAL",%bg2_chapter_9%) RandomNum(10,6)~ + @2899 + O#XanR12.6
++ ~Global("Chapter","GLOBAL",%bg2_chapter_9%) RandomNum(10,7)~ + @2899 + O#XanR12.7
++ ~Global("Chapter","GLOBAL",%bg2_chapter_9%) RandomNum(10,8)~ + @2899 + O#XanR12.8
++ ~Global("Chapter","GLOBAL",%bg2_chapter_9%) RandomNum(10,9)~ + @2899 + O#XanR12.9
++ ~Global("Chapter","GLOBAL",%bg2_chapter_9%) RandomNum(10,10)~ + @2899 + O#XanR12.10
 ++ @420 + O#XanToBPIDTalk
 ++ @421 EXIT
 END
@@ -6474,27 +6430,27 @@
 + ~GlobalGT("RE_ValygarToB","GLOBAL",2) GlobalLT("RE_ValygarToB","GLOBAL",22) Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanRE_ValygarToB","GLOBAL",0)~ + @4273 DO ~SetGlobal("O#XanRE_ValygarToB","GLOBAL",1)~ + xv1
 + ~Global("O#XanRomanceActive","GLOBAL",2) Global("RE_LaranSexToB","GLOBAL",1) Global("RE_LaranSexToBXan","GLOBAL",0)~ + @2152 DO ~SetGlobal("RE_LaranSexToBXan","GLOBAL",1)~ + O#XanRELaranToB
 + ~Global("O#XanMoonbladeImprove","GLOBAL",0) OR(2) PartyHasItem("O#XanMB") PartyHasItem("O#XanMS")~ + @2149 DO ~SetGlobal("O#XanMoonbladeImprove","GLOBAL",2)~ + O#XanMIToB1
-+ ~Global("O#XanToBChosen","GLOBAL",0) Global("Chapter","GLOBAL",9)~ + @3037 DO ~SetGlobal("O#XanToBChosen","GLOBAL",1)~ + O#XanCh1
++ ~Global("O#XanToBChosen","GLOBAL",0) Global("Chapter","GLOBAL",%bg2_chapter_9%)~ + @3037 DO ~SetGlobal("O#XanToBChosen","GLOBAL",1)~ + O#XanCh1
 + ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("O#XanToBRR1","GLOBAL",0)~ + @3038 DO ~SetGlobal("O#XanToBRR1","GLOBAL",1)~ + O#XanRO1
 + ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("O#XanToBRR2","GLOBAL",0)~ + @3039 DO ~SetGlobal("O#XanToBRR2","GLOBAL",1)~ + O#XanRO2
 + ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("O#XanToBRR3","GLOBAL",0)~ + @3040 DO ~SetGlobal("O#XanToBRR3","GLOBAL",1)~ + O#XanRO3
-+ ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("Chapter","GLOBAL",9) Global("O#XanToBRR4","GLOBAL",0)~ + @3041 DO ~SetGlobal("O#XanToBRR4","GLOBAL",1)~ + O#XanRO4
-+ ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("Chapter","GLOBAL",9) Global("O#XanToBRR5","GLOBAL",0)~ + @3042 DO ~SetGlobal("O#XanToBRR5","GLOBAL",1)~ + O#XanRO5
++ ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("Chapter","GLOBAL",%bg2_chapter_9%) Global("O#XanToBRR4","GLOBAL",0)~ + @3041 DO ~SetGlobal("O#XanToBRR4","GLOBAL",1)~ + O#XanRO4
++ ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("Chapter","GLOBAL",%bg2_chapter_9%) Global("O#XanToBRR5","GLOBAL",0)~ + @3042 DO ~SetGlobal("O#XanToBRR5","GLOBAL",1)~ + O#XanRO5
 + ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("O#XanToBRR6","GLOBAL",0)~ + @3043 DO ~SetGlobal("O#XanToBRR6","GLOBAL",1)~ + O#XanRO6
-+ ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("Chapter","GLOBAL",9) Global("O#XanToBRR7","GLOBAL",0)~ + @3044 DO ~SetGlobal("O#XanToBRR7","GLOBAL",1)~ + O#XanRO7
++ ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("Chapter","GLOBAL",%bg2_chapter_9%) Global("O#XanToBRR7","GLOBAL",0)~ + @3044 DO ~SetGlobal("O#XanToBRR7","GLOBAL",1)~ + O#XanRO7
 + ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("O#XanToBRR8","GLOBAL",0)~ + @3045 DO ~SetGlobal("O#XanToBRR8","GLOBAL",1)~ + O#XanRO8
 + ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("O#XanToBRR8","GLOBAL",1)~ + @3046 DO ~SetGlobal("O#XanToBRR8","GLOBAL",2)~ + O#XanRO9
-+ ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("Chapter","GLOBAL",9) Global("O#XanToBRR8","GLOBAL",2)~ + @3047 DO ~SetGlobal("O#XanToBRR8","GLOBAL",3)~ + O#XanRO10
++ ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("Chapter","GLOBAL",%bg2_chapter_9%) Global("O#XanToBRR8","GLOBAL",2)~ + @3047 DO ~SetGlobal("O#XanToBRR8","GLOBAL",3)~ + O#XanRO10
 + ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("O#XanToBRR11","GLOBAL",0)~ + @3048 DO ~SetGlobal("O#XanToBRR11","GLOBAL",1)~ + O#XanRO11
 + ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("O#XanToBRR12","GLOBAL",0)~ + @3049 DO ~SetGlobal("O#XanToBRR12","GLOBAL",1)~ + O#XanRO12
 + ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("O#XanToBRR13","GLOBAL",0) Global("O#XanSolarCorellon1","GLOBAL",1)~ + @3050 DO ~SetGlobal("O#XanToBRR13","GLOBAL",1)~ + O#XanRO13
 + ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("O#XanToBRR14","GLOBAL",0)~ + @3051 DO ~SetGlobal("O#XanToBRR14","GLOBAL",1)~ + O#XanRO14
 + ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("O#XanToBRR15","GLOBAL",0)~ + @3052 DO ~SetGlobal("O#XanToBRR15","GLOBAL",1)~ + O#XanRO15
-+ ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("Chapter","GLOBAL",9) Global("O#XanToBRR16","GLOBAL",0)~ + @3053 DO ~SetGlobal("O#XanToBRR16","GLOBAL",1)~ + O#XanRO16
-+ ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("Chapter","GLOBAL",9) Global("O#XanToBRR17","GLOBAL",0)~ + @3054 DO ~SetGlobal("O#XanToBRR17","GLOBAL",1)~ + O#XanRO17
-+ ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("Chapter","GLOBAL",9) Global("O#XanToBRR18","GLOBAL",0)~ + @3055 DO ~SetGlobal("O#XanToBRR18","GLOBAL",1)~ + O#XanRO18
-+ ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("Chapter","GLOBAL",9) Global("O#XanToBRR19","GLOBAL",0)~ + @3056 DO ~SetGlobal("O#XanToBRR19","GLOBAL",1)~ + O#XanRO19
-+ ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("Chapter","GLOBAL",9) Global("O#XanToBRR20","GLOBAL",0)~ + @3057 DO ~SetGlobal("O#XanToBRR20","GLOBAL",1)~ + O#XanRO20
++ ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("Chapter","GLOBAL",%bg2_chapter_9%) Global("O#XanToBRR16","GLOBAL",0)~ + @3053 DO ~SetGlobal("O#XanToBRR16","GLOBAL",1)~ + O#XanRO16
++ ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("Chapter","GLOBAL",%bg2_chapter_9%) Global("O#XanToBRR17","GLOBAL",0)~ + @3054 DO ~SetGlobal("O#XanToBRR17","GLOBAL",1)~ + O#XanRO17
++ ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("Chapter","GLOBAL",%bg2_chapter_9%) Global("O#XanToBRR18","GLOBAL",0)~ + @3055 DO ~SetGlobal("O#XanToBRR18","GLOBAL",1)~ + O#XanRO18
++ ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("Chapter","GLOBAL",%bg2_chapter_9%) Global("O#XanToBRR19","GLOBAL",0)~ + @3056 DO ~SetGlobal("O#XanToBRR19","GLOBAL",1)~ + O#XanRO19
++ ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanBondedPathAlive","GLOBAL",2) Global("Chapter","GLOBAL",%bg2_chapter_9%) Global("O#XanToBRR20","GLOBAL",0)~ + @3057 DO ~SetGlobal("O#XanToBRR20","GLOBAL",1)~ + O#XanRO20
 + ~Global("O#XanRomanceActive","GLOBAL",2) GlobalGT("O#XanUnTalk","GLOBAL",8) Global("O#XanToBBond1","GLOBAL",0)~ + @1893 DO ~SetGlobal("O#XanToBBond1","GLOBAL",1)~ + O#XanBO1
 + ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanRT15","GLOBAL",2) Global("O#XanToBRant1","GLOBAL",0)~ + @1894 DO ~SetGlobal("O#XanToBRant1","GLOBAL",1)~ + O#XanRA1
 + ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanRT15","GLOBAL",2) Global("O#XanToBRant2","GLOBAL",0)~ + @1895 DO ~SetGlobal("O#XanToBRant2","GLOBAL",1)~ + O#XanRA2
@@ -6507,8 +6463,8 @@
 + ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanRT15","GLOBAL",2) Global("O#XanPP6","GLOBAL",0)~ + @710 DO ~SetGlobal("O#XanPP6","GLOBAL",1)~ + O#XanPP6
 + ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanRT15","GLOBAL",2) Global("O#XanPP7","GLOBAL",0)~ + @711 DO ~SetGlobal("O#XanPP7","GLOBAL",1)~ + O#XanPP7
 + ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanRT15","GLOBAL",2) Global("O#XanPP8","GLOBAL",0)~ + @712 DO ~SetGlobal("O#XanPP8","GLOBAL",1)~ + O#XanPP8
-+ ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanRT15","GLOBAL",2) Global("O#XanPP9","GLOBAL",0) Global("Chapter","GLOBAL",9)~ + @713 DO ~SetGlobal("O#XanPP9","GLOBAL",1)~ + O#XanPP9
-+ ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanRT15","GLOBAL",2) Global("O#XanPP10","GLOBAL",0) Global("Chapter","GLOBAL",9)~ + @714 DO ~SetGlobal("O#XanPP10","GLOBAL",1)~ + O#XanPP10
++ ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanRT15","GLOBAL",2) Global("O#XanPP9","GLOBAL",0) Global("Chapter","GLOBAL",%bg2_chapter_9%)~ + @713 DO ~SetGlobal("O#XanPP9","GLOBAL",1)~ + O#XanPP9
++ ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanRT15","GLOBAL",2) Global("O#XanPP10","GLOBAL",0) Global("Chapter","GLOBAL",%bg2_chapter_9%)~ + @714 DO ~SetGlobal("O#XanPP10","GLOBAL",1)~ + O#XanPP10
 + ~Global("O#XanRomanceActive","GLOBAL",2) Global("O#XanRT15","GLOBAL",2) Global("O#XanPP11","GLOBAL",0)~ + @1387 DO ~SetGlobal("O#XanPP11","GLOBAL",1)~ + O#XanPP11
 + ~Global("O#XanFriendship","GLOBAL",1) Global("O#XanDD1","GLOBAL",0) !InParty("lrirenic")~ + @715 DO ~SetGlobal("O#XanDD1","GLOBAL",1)~ + O#XanDD1
 + ~Global("O#XanFriendship","GLOBAL",1) Global("O#XanDD2","GLOBAL",0)~ + @716 DO ~SetGlobal("O#XanDD2","GLOBAL",1)~ + O#XanDD2
@@ -6518,18 +6474,18 @@
 + ~Global("O#XanFriendship","GLOBAL",1) Global("O#XanDD6","GLOBAL",0)~ + @720 DO ~SetGlobal("O#XanDD6","GLOBAL",1)~ + O#XanDD6
 + ~Global("O#XanFriendship","GLOBAL",1) Global("O#XanDD7","GLOBAL",0)~ + @721 DO ~SetGlobal("O#XanDD7","GLOBAL",1)~ + O#XanDD7
 + ~Global("O#XanFriendship","GLOBAL",1) Global("O#XanDD8","GLOBAL",0)~ + @722 DO ~SetGlobal("O#XanDD8","GLOBAL",1)~ + O#XanDD8
-+ ~Global("O#XanFriendship","GLOBAL",1) Global("O#XanDD9","GLOBAL",0) Global("Chapter","GLOBAL",9)~ + @723 DO ~SetGlobal("O#XanDD9","GLOBAL",1)~ + O#XanDD9
-+ ~Global("O#XanFriendship","GLOBAL",1) Global("O#XanDD10","GLOBAL",0) Global("Chapter","GLOBAL",9)~ + @724 DO ~SetGlobal("O#XanDD10","GLOBAL",1)~ + O#XanDD10
++ ~Global("O#XanFriendship","GLOBAL",1) Global("O#XanDD9","GLOBAL",0) Global("Chapter","GLOBAL",%bg2_chapter_9%)~ + @723 DO ~SetGlobal("O#XanDD9","GLOBAL",1)~ + O#XanDD9
++ ~Global("O#XanFriendship","GLOBAL",1) Global("O#XanDD10","GLOBAL",0) Global("Chapter","GLOBAL",%bg2_chapter_9%)~ + @724 DO ~SetGlobal("O#XanDD10","GLOBAL",1)~ + O#XanDD10
 + ~Global("O#XanFriendship","GLOBAL",1) Global("O#XanDD11","GLOBAL",0)~ + @1386 DO ~SetGlobal("O#XanDD11","GLOBAL",1)~ + O#XanDD11
 + ~Global("O#XanToBErevain","GLOBAL",0)~ + @1804 DO ~SetGlobal("O#XanToBErevain","GLOBAL",1)~ + O#XanEr1.0
 + ~Global("O#XanToBMother","GLOBAL",0) GlobalGT("TalkedToSolar","GLOBAL",1)~ + @1857 DO ~SetGlobal("O#XanToBMother","GLOBAL",1)~ + O#XanMo1.0
-+ ~Global("O#XanToBDemandBack","GLOBAL",0) !Global("O#XanBondedPathAlive","GLOBAL",2) Global("Chapter","GLOBAL",9)~ + @1805 DO ~SetGlobal("O#XanToBDemandBack","GLOBAL",1) RealSetGlobalTimer("O#XanToBRicarTimer","GLOBAL",500)~ + O#XanLe1.0
++ ~Global("O#XanToBDemandBack","GLOBAL",0) !Global("O#XanBondedPathAlive","GLOBAL",2) Global("Chapter","GLOBAL",%bg2_chapter_9%)~ + @1805 DO ~SetGlobal("O#XanToBDemandBack","GLOBAL",1) RealSetGlobalTimer("O#XanToBRicarTimer","GLOBAL",500)~ + O#XanLe1.0
 + ~Global("O#XanEncounterToB3","GLOBAL",3)~ + @3744 DO ~SetGlobal("O#XanEncounterToB3","GLOBAL",4)~ + O#XanEA3
 + ~InParty("Sarevok") Global("O#XanSarPID1","GLOBAL",0) Alignment("Sarevok",MASK_EVIL)~ + @725 DO ~SetGlobal("O#XanSarPID1","GLOBAL",1)~ + O#XanPIDSarevok1
 + ~InParty("Sarevok") Global("O#XanSarPID2","GLOBAL",0) !Alignment("Sarevok",MASK_EVIL)~ + @726 DO ~SetGlobal("O#XanSarPID2","GLOBAL",1)~ + O#XanPIDSarevok2
 + ~Global("O#XanDD12","GLOBAL",0) Global("O#XanFriendship","GLOBAL",1) Global("O#XanRomanceFuture","GLOBAL",1)~ + @1319 DO ~SetGlobal("O#XanDD12","GLOBAL",1)~ + O#XanDD12
 + ~Global("O#XanDD13","GLOBAL",0) Global("O#XanFriendship","GLOBAL",1) Global("O#XanRomanceFuture","GLOBAL",1)~ + @1320 DO ~SetGlobal("O#XanDD13","GLOBAL",1)~ + O#XanDD13
-+ ~Global("O#XanDD14","GLOBAL",0) Global("O#XanFriendship","GLOBAL",1) Global("Chapter","GLOBAL",9) Global("O#XanRomanceFuture","GLOBAL",1)~ + @4045 DO ~SetGlobal("O#XanDD14","GLOBAL",1)~ + O#XanDD14
++ ~Global("O#XanDD14","GLOBAL",0) Global("O#XanFriendship","GLOBAL",1) Global("Chapter","GLOBAL",%bg2_chapter_9%) Global("O#XanRomanceFuture","GLOBAL",1)~ + @4045 DO ~SetGlobal("O#XanDD14","GLOBAL",1)~ + O#XanDD14
 + ~GlobalGT("O#XanToBFirstPartDone","GLOBAL",0) Global("O#XanToBQT1","GLOBAL",0)~ + @1764 DO ~SetGlobal("O#XanToBQT1","GLOBAL",1)~ + O#XanToBQT1
 + ~Dead("O#Ezara") Global("O#XanToBQT2","GLOBAL",0)~ + @1764 DO ~SetGlobal("O#XanToBQT2","GLOBAL",1)~ + O#XanToBQT2
 + ~Global("O#XanRomanceActive","GLOBAL",2) GlobalGT("O#XanUnTalk","GLOBAL",4) GlobalLT("O#XanHangoverTalk","GLOBAL",2) Global("O#XanToBPhysical1","GLOBAL",0)~ + @1891 DO ~SetGlobal("O#XanToBPhysical1","GLOBAL",1)~ + O#XanPH1.0
