--- deArnise/Dialogue/PGNalLT.d	2008-02-26 12:39:18.000000000 -0500
+++ Fixpack/deArnise/Dialogue/PGNalLT.d	2016-11-21 16:22:13.986146900 -0500
@@ -250,9 +250,9 @@
 
 IF ~~ t4.alwaystrue
 SAY @82
-+ ~GlobalLT("Chapter","GLOBAL",4)~ + @83 + t4.street
-+ ~NumInPartyGT(2) GlobalLT("Chapter","GLOBAL",4)~ + @84 + t4.profession
-+ ~NumInPartyGT(2) GlobalLT("Chapter","GLOBAL",4)~ + @85 + t4.profession
++ ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_4%)~ + @83 + t4.street
++ ~NumInPartyGT(2) GlobalLT("Chapter","GLOBAL",%bg2_chapter_4%)~ + @84 + t4.profession
++ ~NumInPartyGT(2) GlobalLT("Chapter","GLOBAL",%bg2_chapter_4%)~ + @85 + t4.profession
 + ~!NumInPartyGT(2)~ + @86 + t4.closequarters
 ++ @87 + t4.nearme
 ++ @88 + t4.quitecivil
@@ -264,9 +264,9 @@
 
 IF ~~ t4.interrupt
 SAY @91
-+ ~GlobalLT("Chapter","GLOBAL",4)~ + @83 + t4.street
-+ ~NumInPartyGT(2) GlobalLT("Chapter","GLOBAL",4)~ + @84 + t4.profession
-+ ~NumInPartyGT(2) GlobalLT("Chapter","GLOBAL",4)~ + @85 + t4.profession
++ ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_4%)~ + @83 + t4.street
++ ~NumInPartyGT(2) GlobalLT("Chapter","GLOBAL",%bg2_chapter_4%)~ + @84 + t4.profession
++ ~NumInPartyGT(2) GlobalLT("Chapter","GLOBAL",%bg2_chapter_4%)~ + @85 + t4.profession
 + ~!NumInPartyGT(2)~ + @86 + t4.closequarters
 ++ @87 + t4.nearme
 ++ @88 + t4.quitecivil
@@ -1107,7 +1107,7 @@
 ++ @381 + t11.glorious
 ++ @382 + t11.betterman
 ++ @383 + t11.madscheme
-+ ~GlobalGT("Chapter","GLOBAL",4) !Dead("Imoen2")~ + @384 + t11.imoen
++ ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_4%) !Dead("Imoen2")~ + @384 + t11.imoen
 + ~Dead("Firkra02")~ + @385 + t11.firkraag
 + ~Class(Player1,PALADIN_ALL)~ + @386 + t11.paladin
 + ~Class(Player1,CLERIC_ALL)~ + @387 + t11.cleric
@@ -1297,7 +1297,7 @@
 ++ @381 + t11.glorious
 ++ @382 + t11.betterman
 ++ @383 + t11.madscheme
-+ ~GlobalGT("Chapter","GLOBAL",4) !Dead("Imoen2")~ + @384 + t11.imoen
++ ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_4%) !Dead("Imoen2")~ + @384 + t11.imoen
 + ~Dead("Firkra02")~ + @385 + t11.firkraag
 + ~Class(Player1,PALADIN_ALL)~ + @386 + t11.paladin
 + ~Class(Player1,CLERIC_ALL)~ + @387 + t11.cleric
@@ -1557,7 +1557,7 @@
 SAY @523 
 ++ @524 + t14.noregrets
 ++ @525 + t14.skip
-+ ~GlobalLT("Chapter","GLOBAL",4)~ + @526 + t14.imoen
++ ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_4%)~ + @526 + t14.imoen
 ++ @527 + t14.wring
 ++ @528 + t14.gorion
 ++ @529 + t14.gagged
@@ -2248,7 +2248,7 @@
 IF ~~ t20.main
 SAY @782
 IF ~~ + t20.3
-IF ~GlobalGT("Chapter","GLOBAL",3) ~ + t20.2
+IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_3%) ~ + t20.2
 END
 
 IF ~~ t20.2
