--- deArnise/Dialogue/PGNalFL.d	2008-01-06 12:32:40.000000000 -0500
+++ Fixpack/deArnise/Dialogue/PGNalFL.d	2016-11-21 16:22:25.567382700 -0500
@@ -387,7 +387,7 @@
 OR(2)
 Global("PGNaliaRomanceActive","GLOBAL",1)
 Global("PGNaliaRomanceActive","GLOBAL",2)
-!Global("Chapter","GLOBAL",5)~ naliaflirtbase1
+!Global("Chapter","GLOBAL",%bg2_chapter_5%)~ naliaflirtbase1
 SAY @82
 = @83
 
@@ -728,7 +728,7 @@
 OR(2)
 Global("PGNaliaRomanceActive","GLOBAL",1)
 Global("PGNaliaRomanceActive","GLOBAL",2)
-!Global("Chapter","GLOBAL",5)~ naliaflirtbase2
+!Global("Chapter","GLOBAL",%bg2_chapter_5%)~ naliaflirtbase2
 SAY @151
 = @152
 
@@ -1117,7 +1117,7 @@
 OR(2)
 Global("PGNaliaRomanceActive","GLOBAL",1)
 Global("PGNaliaRomanceActive","GLOBAL",2)
-Global("Chapter","GLOBAL",5)~ BEGIN noflirtinunderdark
+Global("Chapter","GLOBAL",%bg2_chapter_5%)~ BEGIN noflirtinunderdark
 SAY @211
 IF ~~ EXIT
 END
