--- setup-AzenMOD.tp2	2009-01-01 10:52:38.977500000 -0500
+++ Fixpack/setup-AzenMOD.tp2	2016-12-18 22:50:39.993261200 -0500
@@ -3,7 +3,7 @@
 
 AUTHOR	~erebusant@verizon.net~
 
-VERSION	~v5~
+VERSION	~v5 BWP Trim + Fix~
 
 AUTO_TRA ~AzenMOD/Language/%s/D~
 
@@ -29,13 +29,14 @@
 
 BEGIN @0
 
-COPY + ~Setup-AzenMod.exe~  ~WeiDU.exe~
-COPY + ~chitin.key~        ~AzenMod/chitin.az~
-COPY_LARGE + ~dialog.tlk~  ~AzenMod/dialog.az~
+// COPY + ~Setup-AzenMod.exe~  ~WeiDU.exe~
+// COPY + ~chitin.key~        ~AzenMod/chitin.az~
+// COPY_LARGE + ~dialog.tlk~  ~AzenMod/dialog.az~
 //MKDIR ~AzenMod/over.bak~
 //COPY_LARGE + ~override~ ~AzenMod/over.bak~
 
 COPY	~azenMOD/are~		~override~
+	~azenMOD/bam~		~override~
 	~azenMOD/bcs~		~override~
 	~azenMOD/portrait~	~override~
 	~azenMOD/sound~		~override~
@@ -44,34 +45,43 @@
   COPY	~azenMOD/bcs/add/shout20.bcs~	~override~	// sare override script
 END
 
-ACTION_IF NOT FILE_EXISTS_IN_GAME	~ceskel01.cre~	THEN BEGIN
-  COPY	~azenMOD/cre/add/ceskel01.cre~	~override~	// crypt skeleton
-END
-
-ACTION_IF NOT FILE_EXISTS_IN_GAME	~ibootfox.bam~	THEN BEGIN
-  COPY	~azenMOD/bam/ibootfox.bam~	~override~	//.bam for Boots of the Fox
-END
-
-ACTION_IF NOT FILE_EXISTS_IN_GAME	~isw1aze.bam~	THEN BEGIN
-  COPY	~azenMOD/bam/isw1aze.bam~	~override~	//.bam for Azurenkarath
-END
-
-ACTION_IF NOT FILE_EXISTS_IN_GAME	~isw1var.bam~	THEN BEGIN
-  COPY	~azenMOD/bam/isw1var.bam~	~override~	//.bam for Varscona
-END
-
-ACTION_IF NOT FILE_EXISTS_IN_GAME	~sw1h06.itm~	THEN BEGIN
-  COPY	~azenMOD/itm/add/sw1h06.itm~	~override~	// Varscona
-END
-
-ACTION_IF NOT FILE_EXISTS_IN_GAME	~sw1h23.itm~	THEN BEGIN
-  COPY	~azenMOD/itm/add/sw1h23.itm~	~override~	// Rashad's Talon
-END
-
-ACTION_IF NOT FILE_EXISTS_IN_GAME	~sw1h24.itm~	THEN BEGIN
-  COPY	~azenMOD/sw1h24.itm~	~override~	// Flame Tongue
-END
-
+COPY_EXISTING ~sw1h06.itm~	~override~	// Varscona
+  WRITE_ASCII 0x3a ~ISW1VAR~ #8
+  READ_LONG  0x64 "abil_off" ELSE 0
+  READ_SHORT 0x68 "abil_num" ELSE 0
+  PATCH_IF (abil_off > 0x71) BEGIN
+    FOR (index = 0; index < abil_num; index = index + 1) BEGIN
+      WRITE_ASCII ("%abil_off%" + 0x04 + (0x38 * "%index%")) ~ISW1VAR~ #8
+    END
+  END
+BUT_ONLY_IF_IT_CHANGES
+
+COPY_EXISTING ~sw1h23.itm~	~override~	// Rashad's Talon
+  LAUNCH_PATCH_FUNCTION ~ADD_ITEM_EFFECT~
+    INT_VAR
+      opcode = 40	// Slow
+      target = 2
+      probability1 = 50
+      header = 1
+      type = 1
+  END
+BUT_ONLY_IF_IT_CHANGES
+
+COPY_EXISTING ~sw1h24.itm~	~override~	// Flame Tongue
+  LAUNCH_PATCH_FUNCTION ~ADD_ITEM_EFFECT~
+    INT_VAR
+      opcode = 12	// Damage
+      target = 2
+      power = 8
+      parameter1 = 1
+      parameter2 = 524288	// Fire
+      timing = 1
+      resist_dispel = 1
+      dicesize = 6
+      header = 1
+      type = 1
+  END
+BUT_ONLY_IF_IT_CHANGES
 COPY_EXISTING	~kpchap01.sto~	~override~
 	ADD_STORE_ITEM ~JC_BT01~ AFTER ~scrl63~ #0 #0 #0 ~IDENTIFIED~ #1
 	ADD_STORE_ITEM ~SW1H06~ AFTER ~JC_BT01~ #0 #0 #0 ~IDENTIFIED~ #1
@@ -137,6 +147,10 @@
 
 BEGIN	@19
 
+COPY_EXISTING ~ceskel01.cre~	~override~	// give amulet to crypt skeleton here to avoid dependency on previous component
+  REPLACE_CRE_ITEM ~AMUL12~ #0 #0 #0 ~NONE~ ~INV1~
+BUT_ONLY_IF_IT_CHANGES
+
 COPY_EXISTING	~AR2013.ARE~	~override~
 	READ_LONG 0x54 ~actoroffset~
 	READ_SHORT 0x58 ~actornum~
