--- C:/Users/ggoldman/Downloads/setup-pcvamp.tp2	Mon Jan 25 17:09:39 2016
+++ C:/Users/ggoldman/Downloads/setup-pcvamp.p.tp2	Mon Jan 25 17:44:03 2016
@@ -1,18 +1,35 @@
 BACKUP ~pcvamp/Backup~
 AUTHOR ~Nicolas Peters - admin@baldursgate-refugium.de (http://www.baldursgate-refugium.de/)~
-VERSION "v1.1"
+VERSION "v1.1 BWP Fix 2"
 README "pcvamp/Readme.html"
-AUTO_TRA ~pcvamp/TRA/%s~
+AUTO_TRA ~pcvamp/TRA/%LANGUAGE%/%s~
+
+// Make sure strings are correctly encoded for both EE and non-EE games
+ALWAYS
+  ACTION_IF "%WEIDU_OS%" STRING_EQUAL_CASE ~win32~ THEN BEGIN
+    AT_NOW ~if not exist "iconv" mkdir iconv~
+    AT_NOW ~if not exist "iconv\iconv.exe" copy "BiG World Fixpack\_utils\iconv" "iconv"~
+  END
+  ACTION_DEFINE_ARRAY tra_reload BEGIN setup END
+  LAF HANDLE_CHARSETS
+    INT_VAR
+      infer_charset = 1
+    STR_VAR
+      tra_path      = EVAL ~%MOD_FOLDER%/TRA~
+      iconv_path    = EVAL ~iconv~
+      reload_array  = tra_reload
+  END
+END
+
 LANGUAGE ~Deutsch~ ~german~ ~pcvamp/TRA/german/setup.tra~
 LANGUAGE ~English~ ~english~ ~pcvamp/TRA/english/setup.tra~
 
 BEGIN @1 // Mal des Vampirismus
-REQUIRE_FILE ~Data/25Dialog.bif~ @31 // Für eine Installation muss Thron des Bhaal installiert sein!
-REQUIRE_FILE ~Data/25Scripts.bif~ @31 // Für eine Installation muss Thron des Bhaal installiert sein!
+REQUIRE_PREDICATE GAME_INCLUDES ~TOB~ @31 // Für eine Installation muss Thron des Bhaal installiert sein!
 
 // BMPs
 COPY ~pcvamp/BMP/amalethS.bmp~ ~override/amalethS.bmp~
 COPY ~pcvamp/BMP/taericS.bmp~ ~override/taericS.bmp~
 
 // [BAMs]
 COPY ~pcvamp/BAM/sumik04.bam~ ~override/sumik04.bam~
@@ -94,20 +111,22 @@
 // Falle durch Schattendieb und Lathanderiten
 COPY ~pcvamp/BCS/pcvampc6.bcs~ ~override/pcvampc6.bcs~
 COPY ~pcvamp/BCS/pcvampc7.bcs~ ~override/pcvampc7.bcs~
 COPY ~pcvamp/BCS/pcvampc8.bcs~ ~override/pcvampc8.bcs~
 EXTEND_BOTTOM ~AR0700.bcs~ ~pcvamp/BCS/AR0700.bcs~ // Gebietscript Waukeens Promenade ergänzen
 EXTEND_BOTTOM ~AR0702.bcs~ ~pcvamp/BCS/AR0702.bcs~ // Gebietscript Abenteuers Allerlei ergänzen
 
+/* BWP Fixpack incorporates this patch into Valen's original priest.bcs file
 // Kleriker-Script Bugfix ausführen, wenn Valen-Mod installiert ist
 ACTION_IF FILE_EXISTS_IN_GAME ~valen.cre~ THEN BEGIN
 	PRINT @20 // Valen-Mod gefunden, installiere Kleriker-Script Bugfix...
 	COPY_EXISTING_REGEXP ~.*\(PRIE\|CLER\|HEAL\|PRST\|SHAM\).*\.BCS~ ~override~
 	REPLACE_BCS_BLOCK ~pcvamp/BCS/priest.bcs~ ~pcvamp/BCS/priest2.bcs~
 END
+*/
 
 // Gebietscript Refugium (TdB)
 EXTEND_BOTTOM ~AR4500.bcs~ ~pcvamp/BCS/pcvampc3.bcs~
 
 // [Dialoge]
 COMPILE ~pcvamp/DLG/pcvamp.d~
 USING ~pcvamp/TRA/%s/pcvamp.tra~
