--- BGTNeJ2\compat\nAR0602.baf	Thu Mar 26 03:14:22 2009
+++ C:\BWP Patchstudio\patched files\BGTNeJ2\compat\nAR0602.baf	Tue Mar 04 20:24:31 2014
@@ -9,6 +9,10 @@
 THEN
 
 	RESPONSE #100
+
+		SetGlobal("NewGame","AR0602",1)
+
+		SetGlobal("ENDOFBG1","GLOBAL",2)
 
 		SetCursorState(TRUE)
 
@@ -17,10 +21,6 @@
 		SmallWait(10)
 
 		StartMovie("SOAINTRO")
-
-		SetGlobal("NewGame","AR0602",1)
-
-		SetGlobal("ENDOFBG1","GLOBAL",2)
 
 		HideAreaOnMap("AR9800")
 
