--- BGTNeJ2\Setup-BGTNeJ2.tp2	Mon Apr 05 03:19:58 2010
+++ C:\BWP Patchstudio\patched files\BGTNeJ2\Setup-BGTNeJ2.tp2	Sun Dec 08 20:15:26 2013
@@ -2,7 +2,7 @@
 
 AUTHOR ~http://www.shsforums.net/index.php?act=SF&f=193~
 
-VERSION ~v1.2~
+VERSION ~v1.2 BWP Fix~
 
 README ~BGTNeJ2/BGTNeJ2-Readme.txt~
 
@@ -339,8 +339,8 @@
               ~SHARTE4.cre~ ~override/SHARTE4.cre~
               ~SHARTE6.cre~ ~override/SHARTE6.cre~
   WRITE_SHORT 0x244 0 // doesn‘t change
-  WRITE_BYTE 0x246 ~%FIREWALKER%~ // the internal name of your kit, surrounded by %
-  WRITE_BYTE 0x247 0x40 // doesn‘t change
+  WRITE_SHORT 0x246 0x4000 + ~%FIREWALKER%~ // the internal name of your kit, surrounded by %
+//WRITE_BYTE 0x247 0x40 // doesn‘t change
   WRITE_BYTE 0x32 66 //hair color
   WRITE_BYTE 0x23b 10 //Wisdom
   WRITE_BYTE 0x23c 18 //Dexterity
