--- questpack\encounters\DING0\randenc\city_new.baf	Thu Jan 05 21:07:28 2006
+++ C:\BWP Patchstudio\patched files\questpack\encounters\DING0\randenc\city_new.baf	Sun Oct 24 18:46:16 2010
@@ -10,13 +10,13 @@
 	!DifficultyGT(NORMAL)
 THEN
 	RESPONSE #100
-		CreateCreature("VAMPIR01",[488.558],10)
-		CreateCreature("VAMPIR01",[500.454],10)
+		CreateCreature("VAMPIM01",[488.558],10)
+		CreateCreature("VAMPIF01",[500.454],10)
 		CreateCreature("VAMFLM01",[552.476],10)
 	RESPONSE #150
 		CreateCreature("VAMEMI01",[488.558],10)
-		CreateCreature("VAMPIR01",[500.454],10)
-		CreateCreature("VAMPIR01",[552.476],10)
+		CreateCreature("VAMPIM01",[500.454],10)
+		CreateCreature("VAMPIF01",[552.476],10)
 	RESPONSE #250
 		CreateCreature("VAMEMI01",[488.558],10)
 		CreateCreature("VAMEMI01",[500.454],10)
@@ -24,16 +24,16 @@
 		CreateCreature("VAMFLF01",[427.548],10)
 	RESPONSE #300
 		CreateCreature("VAMEMI01",[488.558],10)
-		CreateCreature("VAMPIR01",[500.454],10)
+		CreateCreature("VAMPIF01",[500.454],10)
 		CreateCreature("VAMFLF01",[552.476],10)
-		CreateCreature("VAMPIR01",[427.548],10)
+		CreateCreature("VAMPIM01",[427.548],10)
 		CreateCreature("VAMFLM01",[525.591],10)
 	RESPONSE #300
 		CreateCreature("VAMEMI01",[488.558],10)
-		CreateCreature("VAMPIR01",[500.454],10)
+		CreateCreature("VAMPIF01",[500.454],10)
 		CreateCreature("VAMFLM01",[552.476],10)
-		CreateCreature("VAMPIR01",[427.548],10)
-		CreateCreature("VAMPIR01",[525.591],10)
+		CreateCreature("VAMPIM01",[427.548],10)
+		CreateCreature("VAMPIM01",[525.591],10)
 END
 
 IF
@@ -48,13 +48,13 @@
 	DifficultyGT(NORMAL)
 THEN
 	RESPONSE #100
-		CreateCreature("VAMPIR01",[488.558],10)
-		CreateCreature("VAMPIR01",[500.454],10)
+		CreateCreature("VAMPIF01",[488.558],10)
+		CreateCreature("VAMPIM01",[500.454],10)
 		CreateCreature("VAMFLM01",[552.476],10)
 	RESPONSE #150
 		CreateCreature("VAMEMI01",[488.558],10)
-		CreateCreature("VAMPIR01",[500.454],10)
-		CreateCreature("VAMPIR01",[552.476],10)
+		CreateCreature("VAMPIF01",[500.454],10)
+		CreateCreature("VAMPIM01",[552.476],10)
 	RESPONSE #250
 		CreateCreature("VAMEMI01",[488.558],10)
 		CreateCreature("VAMEMI01",[500.454],10)
@@ -62,16 +62,16 @@
 		CreateCreature("VAMFLF01",[427.548],10)
 	RESPONSE #300
 		CreateCreature("VAMEMI01",[488.558],10)
-		CreateCreature("VAMPIR01",[500.454],10)
+		CreateCreature("VAMPIF01",[500.454],10)
 		CreateCreature("VAMFLF01",[552.476],10)
-		CreateCreature("VAMPIR01",[427.548],10)
+		CreateCreature("VAMPIM01",[427.548],10)
 		CreateCreature("VAMFLM01",[525.591],10)
 	RESPONSE #300
 		CreateCreature("VAMEMI01",[488.558],10)
-		CreateCreature("VAMPIR01",[500.454],10)
+		CreateCreature("VAMPIF01",[500.454],10)
 		CreateCreature("VAMFLM01",[552.476],10)
-		CreateCreature("VAMPIR01",[427.548],10)
-		CreateCreature("VAMPIR01",[525.591],10)
+		CreateCreature("VAMPIM01",[427.548],10)
+		CreateCreature("VAMPIM01",[525.591],10)
 	RESPONSE #500
 		CreateCreature("VAMFLM01",[488.558],10)
 		CreateCreature("VAMFLF01",[500.454],10)
@@ -202,8 +202,8 @@
 	RandomNumGT(5,2)
 THEN
 	RESPONSE #100
-		SetGlobal("D0RandEncRuneRevenge","GLOBAL",1)
-		SetGlobalTimer("D0RuneReinforce","MYAREA",12)
+		SetGlobal("D0RandEncRuneRevenge","GLOBAL",2)
+		SetGlobalTimer("D0RuneReinforce","MYAREA",60)
 		CreateCreature("TANTHF01",[488.558],10)
 		CreateCreature("TANTHF01",[427.548],10)
 		CreateCreature("TANWIZ02",[525.591],10)
@@ -271,7 +271,7 @@
 	GlobalTimerExpired("D0RuneReinforce","MYAREA")
 THEN
 	RESPONSE #100
-		SetGlobal("D0RandEncRuneRevenge","GLOBAL",1)
+		SetGlobal("D0RandEncRuneRevenge","GLOBAL",2)
 		CreateCreatureObjectDoor("TANTHF01",Player1,0,0,0)
 		CreateCreatureObjectDoor("TANTHF01",[PC],0,0,0)
 END
