--- questpack\tp2\quest\coronet.tph	Sun Sep 06 00:50:58 2015
+++ C:\BWP Patchstudio\patched files\questpack\tp2\quest\coronet.tph	Fri Aug 26 07:42:29 2016
@@ -125,6 +125,15 @@
   END
 BUT_ONLY
 
+COPY_EXISTING ~cut41f.bcs~ ~override~
+  DECOMPILE_AND_PATCH BEGIN
+    REPLACE_TEXTUALLY ~CreateCreature("ppdradee",[742.1323],0)~
+                      ~CreateCreature("ppdradee",[742.1323],0)
+                       CreateCreature("d0lorick",[771.597],12)~
+  END
+BUT_ONLY
+
+
 COPY_EXISTING ~mazzy.bcs~ ~override~
   DECOMPILE_AND_PATCH BEGIN
     REPLACE_TEXTUALLY ~See("surly")~ ~See("surly") !Dead("surly")~
