--- questpack\oasis\combat\scripts\d0amtmag.baf	Tue Jan 25 20:07:22 2005
+++ C:\BWP Patchstudio\patched files\questpack\oasis\combat\scripts\d0amtmag.baf	Tue Feb 16 20:59:51 2010
@@ -263,7 +263,7 @@
 	OR(3)
 		Allegiance(Myself,GOODBUTBLUE)
 		Allegiance(Myself,NEUTRAL)
-		Allegiance(Myself,0)
+		Allegiance(Myself,EVILBUTBLUE)
 THEN
 	RESPONSE #100
 		Shout(125)
@@ -707,7 +707,7 @@
 
 IF
 	Heard(NearestMyGroupOfType,124)
-	!Dead("astHeardBy(Myself))")
+	!StateCheck(LastHeardBy(Myself),STATE_REALLY_DEAD)
 	!StateCheck(LastAttackerOf(LastHeardBy(Myself)),STATE_INVISIBLE)
 	!StateCheck(LastAttackerOf(LastHeardBy(Myself)),STATE_IMPROVEDINVISIBILITY)
 	!StateCheck(LastAttackerOf(LastHeardBy(Myself)),STATE_CHARMED)
@@ -725,7 +725,7 @@
 	Heard(NearestMyGroupOfType,124)
 	See(LastHeardBy(Myself))
 	!See(LastAttackerOf(LastHeardBy(Myself)))
-	!Dead("astHeardBy(Myself))")
+	!StateCheck(LastHeardBy(Myself),STATE_REALLY_DEAD)
 	!StateCheck(LastAttackerOf(LastHeardBy(Myself)),STATE_INVISIBLE)
 	!StateCheck(LastAttackerOf(LastHeardBy(Myself)),STATE_IMPROVEDINVISIBILITY)
 	!StateCheck(LastAttackerOf(LastHeardBy(Myself)),STATE_CHARMED)
