--- questpack\oasis\combat\scripts\d0amtmer.baf	Tue Jan 25 20:07:44 2005
+++ C:\BWP Patchstudio\patched files\questpack\oasis\combat\scripts\d0amtmer.baf	Tue Feb 16 20:59:51 2010
@@ -214,7 +214,7 @@
 
 IF
 	Heard(NearestMyGroupOfType,123)
-	!Dead("astHeardBy(Myself))")
+	!StateCheck(LastHeardBy(Myself),STATE_REALLY_DEAD)
 	HPPercentGT(Myself,75)
 	See(LastHeardBy(Myself))
 	Range(LastHeardBy(Myself),10)
@@ -567,7 +567,7 @@
 
 IF
 	Heard(NearestMyGroupOfType(Myself),124)
-	!Dead("astHeardBy(Myself))")
+	!StateCheck(LastHeardBy(Myself),STATE_REALLY_DEAD)
 	!StateCheck(LastAttackerOf(LastHeardBy(Myself)),STATE_INVISIBLE)
 	!StateCheck(LastAttackerOf(LastHeardBy(Myself)),STATE_IMPROVEDINVISIBILITY)
 	!StateCheck(LastAttackerOf(LastHeardBy(Myself)),STATE_CHARMED)
@@ -585,7 +585,7 @@
 	Heard(NearestMyGroupOfType,124)
 	See(LastHeardBy(Myself))
 	!See(LastAttackerOf(LastHeardBy(Myself)))
-	!Dead("astHeardBy(Myself))")
+	!StateCheck(LastHeardBy(Myself),STATE_REALLY_DEAD)
 	!StateCheck(LastAttackerOf(LastHeardBy(Myself)),STATE_INVISIBLE)
 	!StateCheck(LastAttackerOf(LastHeardBy(Myself)),STATE_IMPROVEDINVISIBILITY)
 	!StateCheck(LastAttackerOf(LastHeardBy(Myself)),STATE_CHARMED)
@@ -619,7 +619,7 @@
 IF
 	Heard([EVILCUTOFF],124)
 	InMyGroup(LastHeardBy(Myself))
-	!Dead("astHeardBy(Myself))")
+	!StateCheck(LastHeardBy(Myself),STATE_REALLY_DEAD)
 	LevelGT(LastHeardBy(Myself),10)
 	!HPPercentLT(Myself,40)
 	OR(2)
