--- questpack\thieves\scripts\d0edwinh.baf	Fri Dec 03 14:15:12 2004
+++ C:\BWP Patchstudio\patched files\questpack\thieves\scripts\d0edwinh.baf	Tue Feb 16 20:59:51 2010
@@ -140,7 +140,7 @@
 		Class(Myself,LONG_BOW)
 		Class(Myself,BARD_ALL)
 	See("Rayic")
-	!Dead("astSeenBy(Myself))")
+	!StateCheck(LastSeenBy(Myself),STATE_REALLY_DEAD)
 	!StateCheck("Rayic",STATE_PANIC)
 	!HasBounceEffects("Rayic")
 	!LevelGT("Rayic",10)
@@ -160,7 +160,7 @@
 		Class(Myself,LONG_BOW)
 		Class(Myself,BARD_ALL)
 	See("Rayic")
-	!Dead("astSeenBy(Myself))")
+	!StateCheck(LastSeenBy(Myself),STATE_REALLY_DEAD)
 	!HasBounceEffects("Rayic")
 	!StateCheck("Rayic",STATE_STUNNED)
 	HasItem("wand04",Myself) // Wand of Paralyzation 
@@ -179,7 +179,7 @@
 		Class(Myself,LONG_BOW)
 		Class(Myself,BARD_ALL)
 	See("Rayic")
-	!Dead("astSeenBy(Myself))")
+	!StateCheck(LastSeenBy(Myself),STATE_REALLY_DEAD)
 	!HasBounceEffects("Rayic")
 	HasItem("wand06",Myself) // Wand of Frost
 THEN
@@ -197,7 +197,7 @@
 		Class(Myself,LONG_BOW)
 		Class(Myself,BARD_ALL)
 	See("Rayic")
-	!Dead("astSeenBy(Myself))")
+	!StateCheck(LastSeenBy(Myself),STATE_REALLY_DEAD)
 	!StateCheck("Rayic",STATE_SLEEPING)
 	!HasBounceEffects("Rayic")
 	!LevelGT("Rayic",10)
@@ -217,7 +217,7 @@
 		Class(Myself,LONG_BOW)
 		Class(Myself,BARD_ALL)
 	See("Rayic")
-	!Dead("astSeenBy(Myself))")
+	!StateCheck(LastSeenBy(Myself),STATE_REALLY_DEAD)
 	!LevelGT("Rayic",10)
 	!HasBounceEffects("Rayic")
 	HasItem("wand09",Myself) // Wand of Polymorphing
@@ -236,7 +236,7 @@
 		Class(Myself,LONG_BOW)
 		Class(Myself,BARD_ALL)
 	See("Rayic")
-	!Dead("astSeenBy(Myself))")
+	!StateCheck(LastSeenBy(Myself),STATE_REALLY_DEAD)
 	HasItem("wand10",Myself) // Wand of Monster Summoning
 THEN
 	RESPONSE #100
@@ -251,7 +251,7 @@
 	!GlobalTimerNotExpired("Cast","LOCALS")
 	Class(Myself,CLERIC_ALL)
 	See("Rayic")
-	!Dead("astSeenBy(Myself))")
+	!StateCheck(LastSeenBy(Myself),STATE_REALLY_DEAD)
 	OR(2)
 		!CheckStatGT("Rayic",50,RESISTMAGICFIRE)
 		!HasImmunityEffects("Rayic")
@@ -1065,7 +1065,7 @@
 		Class(Myself,LONG_BOW)
 		Class(Myself,BARD_ALL)
 	See(NearestEnemyOf(Myself))
-	!Dead("astSeenBy(Myself))")
+	!StateCheck(LastSeenBy(Myself),STATE_REALLY_DEAD)
 	!StateCheck(LastSeenBy(Myself),STATE_PANIC)
 	!HasBounceEffects(LastSeenBy(Myself))
 	!LevelGT(LastSeenBy(Myself),10)
@@ -1084,7 +1084,7 @@
 		Class(Myself,LONG_BOW)
 		Class(Myself,BARD_ALL)
 	See(NearestEnemyOf(Myself))
-	!Dead("astSeenBy(Myself))")
+	!StateCheck(LastSeenBy(Myself),STATE_REALLY_DEAD)
 	!HasBounceEffects(LastSeenBy(Myself))
 	!StateCheck(LastSeenBy(Myself),STATE_STUNNED)
 	HasItem("wand04",Myself) // Wand of Paralyzation 
@@ -1102,7 +1102,7 @@
 		Class(Myself,LONG_BOW)
 		Class(Myself,BARD_ALL)
 	See(NearestEnemyOf(Myself))
-	!Dead("astSeenBy(Myself))")
+	!StateCheck(LastSeenBy(Myself),STATE_REALLY_DEAD)
 	!HasBounceEffects(LastSeenBy(Myself))
 	HasItem("wand06",Myself) // Wand of Frost
 THEN
@@ -1119,7 +1119,7 @@
 		Class(Myself,LONG_BOW)
 		Class(Myself,BARD_ALL)
 	See(NearestEnemyOf(Myself))
-	!Dead("astSeenBy(Myself))")
+	!StateCheck(LastSeenBy(Myself),STATE_REALLY_DEAD)
 	!StateCheck(LastSeenBy(Myself),STATE_SLEEPING)
 	!HasBounceEffects(LastSeenBy(Myself))
 	!LevelGT(LastSeenBy(Myself),10)
@@ -1138,7 +1138,7 @@
 		Class(Myself,LONG_BOW)
 		Class(Myself,BARD_ALL)
 	See(NearestEnemyOf(Myself))
-	!Dead("astSeenBy(Myself))")
+	!StateCheck(LastSeenBy(Myself),STATE_REALLY_DEAD)
 	!LevelGT(LastSeenBy(Myself),10)
 	!HasBounceEffects(LastSeenBy(Myself))
 	HasItem("wand09",Myself) // Wand of Polymorphing
@@ -1156,7 +1156,7 @@
 		Class(Myself,LONG_BOW)
 		Class(Myself,BARD_ALL)
 	See(NearestEnemyOf(Myself))
-	!Dead("astSeenBy(Myself))")
+	!StateCheck(LastSeenBy(Myself),STATE_REALLY_DEAD)
 	HasItem("wand10",Myself) // Wand of Monster Summoning
 THEN
 	RESPONSE #100
@@ -1170,7 +1170,7 @@
 	!GlobalTimerNotExpired("Cast","LOCALS")
 	Class(Myself,CLERIC_ALL)
 	See(NearestEnemyOf(Myself))
-	!Dead("astSeenBy(Myself))")
+	!StateCheck(LastSeenBy(Myself),STATE_REALLY_DEAD)
 	OR(2)
 		!CheckStatGT(LastSeenBy(Myself),50,RESISTMAGICFIRE)
 		!HasImmunityEffects(LastSeenBy(Myself))
