--- questpack\thieves\scripts\d0guard4.baf	Mon Apr 11 20:31:16 2005
+++ C:\BWP Patchstudio\patched files\questpack\thieves\scripts\d0guard4.baf	Tue Feb 16 22:37:55 2010
@@ -152,7 +152,7 @@
 END
 
 IF
-	OR(4)
+	OR(3)
 		See([0.0.0.0.242])
 		See([0.0.0.0.241])
 		See([0.0.0.0.240])
@@ -167,7 +167,7 @@
 END
 
 IF
-	OR(4)
+	OR(3)
 		See([0.0.0.0.242])
 		See([0.0.0.0.241])
 		See([0.0.0.0.240])
@@ -180,6 +180,8 @@
 		IncrementGlobal("Breach","LOCALS",1)
 		ForceSpell(LastSeenBy(Myself),WIZARD_BREACH)
 END
+
+
 
 IF
 	OR(4)
