--- questpack\harpers\d0jahrev.baf	Sat Apr 16 16:48:04 2005
+++ C:\BWP Patchstudio\patched files\questpack\harpers\d0jahrev.baf	Tue Feb 16 20:59:51 2010
@@ -165,7 +165,7 @@
 	HaveSpell(CLERIC_ENTANGLE)
 	HPGT(LastSeenBy(Myself),20)
 	NumCreatureVsPartyGT([ENEMY],4)
-	AreaCheck("UTDOOR)")
+	AreaType(OUTDOOR)
 THEN
 	RESPONSE #100
 		SetGlobalTimer("Cast","LOCALS",6)
@@ -188,7 +188,7 @@
 	See(NearestEnemyOf(Myself))
 	HaveSpell(CLERIC_CALL_LIGHTNING)
 	HPGT(LastSeenBy(Myself),20)
-	AreaCheck("UTDOOR)")
+	AreaType(OUTDOOR)
 THEN
 	RESPONSE #100
 		SetGlobalTimer("Cast","LOCALS",6)
