--- questpack\AI\compile\d0pimag.baf	Wed Apr 05 21:33:50 2006
+++ C:\BWP Patchstudio\patched files\questpack\AI\compile\d0pimag.baf	Tue Feb 16 20:59:52 2010
@@ -12,7 +12,7 @@
 IF
 	Global("Seen","LOCALS",0)
 	!Allegiance(Myself,GOODCUTOFF)
-	OR(3)
+	OR(2)
 		AttackedBy([ANYONE],DEFAULT)
 		HitBy([ANYONE],CRUSHING)
 THEN
@@ -689,7 +689,7 @@
 
 IF
 	Heard(NearestMyGroupOfType,0)
-	!Dead("astHeardBy(Myself))")
+	!StateCheck(LastHeardBy(Myself),STATE_REALLY_DEAD)
 	!StateCheck(LastAttackerOf(LastHeardBy(Myself)),STATE_INVISIBLE)
 	!StateCheck(LastAttackerOf(LastHeardBy(Myself)),STATE_IMPROVEDINVISIBILITY)
 	!StateCheck(LastAttackerOf(LastHeardBy(Myself)),STATE_CHARMED)
@@ -707,7 +707,7 @@
 	Heard(NearestMyGroupOfType,0)
 	See(LastHeardBy(Myself))
 	!See(LastAttackerOf(LastHeardBy(Myself)))
-	!Dead("astHeardBy(Myself))")
+	!StateCheck(LastHeardBy(Myself),STATE_REALLY_DEAD)
 	!StateCheck(LastAttackerOf(LastHeardBy(Myself)),STATE_INVISIBLE)
 	!StateCheck(LastAttackerOf(LastHeardBy(Myself)),STATE_IMPROVEDINVISIBILITY)
 	!StateCheck(LastAttackerOf(LastHeardBy(Myself)),STATE_CHARMED)
@@ -1488,7 +1488,7 @@
 END
 
 IF
-	OR(3)
+	OR(2)
 		AttackedBy([ANYONE],DEFAULT)
 		HitBy([ANYONE],CRUSHING)
 THEN
