--- questpack\thieves\dialog3.d	Mon Mar 20 21:02:50 2006
+++ C:\BWP Patchstudio\patched files\questpack\thieves\dialog3.d	Tue Feb 16 20:59:52 2010
@@ -332,7 +332,7 @@
 
 IF ~~ THEN BEGIN BR_17
   SAY ~Linvail's influence extended further than you might expect. The decisions of the Council are... not always in the best interests of the city.~
-  IF ~~ THEN REPLY ~I see. Is there nothing that be done about this?~ GOTO BR_18
+  IF ~~ THEN REPLY ~I see. Is there nothing that can be done about this?~ GOTO BR_18
   IF ~~ THEN REPLY ~Ah, that explains it.~ GOTO BR_19
   IF ~~ THEN REPLY ~The Council is corrupt? We must put an end to this!~ GOTO BR_18
 END
