--- questpack\coronet\dialog.d	Fri Mar 24 17:37:50 2006
+++ C:\BWP Patchstudio\patched files\questpack\coronet\dialog.d	Tue Feb 16 20:59:52 2010
@@ -1522,7 +1522,7 @@
 END
 
 IF ~~ THEN BEGIN PO_27
-  SAY ~From one of the tombs? She is dead? Oh...oh...you mean that is where they were keeping here. Ah. Thank you! Thank you, my <PRO_LADYLORD>!~
+  SAY ~From one of the tombs? She is dead? Oh...oh...you mean that is where they were keeping her. Ah. Thank you! Thank you, my <PRO_LADYLORD>!~
   IF ~~ THEN EXIT
 END
 
