--- questpack\harpers\harpass1.baf	Mon Sep 12 17:00:06 2005
+++ C:\BWP Patchstudio\patched files\questpack\harpers\harpass1.baf	Tue Feb 16 20:59:52 2010
@@ -19,6 +19,7 @@
 	Global("D0Harpers","GLOBAL",3)
 THEN
 	RESPONSE #100
+		StartCutSceneMode()
 		FaceObject("Lyros")
 		ActionOverride("Lyros",FaceObject([NEUTRAL.HUMANOID.HUMAN.FIGHTER_THIEF.133.FEMALE.CHAOTIC_GOOD]))
 		Wait(1)
@@ -30,4 +31,5 @@
 		Wait(1)
 		CreateVisualEffectObject("SPFIREPI",Myself)
 		Kill(Myself)
+		EndCutSceneMode()
 END
