--- questpack\coronet\ketodialog.d	Mon Sep 12 15:33:32 2005
+++ C:\BWP Patchstudio\patched files\questpack\coronet\ketodialog.d	Tue Feb 16 20:59:52 2010
@@ -1,25 +1,25 @@
 INTERJECT D0LENDER LE_8 D0KetoCommentsLenderMethods
-== FWKETOJ IF ~IsValidForPartyDialog("FWKeto")~ THEN ~I think I've seen that kind of motivation a few times. All I can say is, "ouch."~
+== FWKETOJ IF ~IsValidForPartyDialog("Keto")~ THEN ~I think I've seen that kind of motivation a few times. All I can say is, "ouch."~
 END D0LENDER LE_8a_2
 
 INTERJECT_COPY_TRANS D0LENDER LE_45 D0KetoCommentLenderExit
-== FWKETOJ IF ~IsValidForPartyDialog("FWKeto")~ THEN ~Anybody else get the feeling he doesn't mind so much when there are "consequences?"~
+== FWKETOJ IF ~IsValidForPartyDialog("Keto")~ THEN ~Anybody else get the feeling he doesn't mind so much when there are "consequences?"~
 END
 
 INTERJECT_COPY_TRANS D0POOR PO_10 D0KetoCommentPoor
-== FWKETOJ IF ~IsValidForPartyDialog("FWKeto")
+== FWKETOJ IF ~IsValidForPartyDialog("Keto")
 !IsValidForPartyDialog("Viconia")~ THEN ~Ehhh... <CHARNAME>? That was very sweet, but... you just gave a fortune to somebody who didn't even tell you a terribly original story. I could come up with something far better and I'd only charge you for a nice dinner!~
-== FWKETOJ IF ~IsValidForPartyDialog("FWKeto")
+== FWKETOJ IF ~IsValidForPartyDialog("Keto")
 !IsValidForPartyDialog("Viconia")~ THEN ~Anyway. Hopefully she was telling the truth.~
 END
 
 INTERJECT_COPY_TRANS VICONIJ PO_10a D0KetoCommentPoor2
-== FWKETOJ IF ~IsValidForPartyDialog("FWKeto")~ THEN ~Ehhh... <CHARNAME>? That was very sweet, but... you just gave a fortune to somebody who didn't even tell you a terribly original story. I could come up with something far better and I'd only charge you for a nice dinner!~
-== VICONIJ IF ~IsValidForPartyDialog("FWKeto")
+== FWKETOJ IF ~IsValidForPartyDialog("Keto")~ THEN ~Ehhh... <CHARNAME>? That was very sweet, but... you just gave a fortune to somebody who didn't even tell you a terribly original story. I could come up with something far better and I'd only charge you for a nice dinner!~
+== VICONIJ IF ~IsValidForPartyDialog("Keto")
 IsValidForPartyDialog("Viconia")~ THEN ~"The Barking Bard," the story of a mutt who believed she was a girl who believed she could sing?~
-== FWKETOJ IF ~IsValidForPartyDialog("FWKeto")
+== FWKETOJ IF ~IsValidForPartyDialog("Keto")
 IsValidForPartyDialog("Viconia")~ THEN ~No, I was thinking of "The Great Drow Barbeque." It's an alternative-history piece.~
-== FWKETOJ IF ~IsValidForPartyDialog("FWKeto")~ THEN ~Anyway. Hopefully she was telling the truth.~
-== VICONIJ IF ~IsValidForPartyDialog("FWKeto")
+== FWKETOJ IF ~IsValidForPartyDialog("Keto")~ THEN ~Anyway. Hopefully she was telling the truth.~
+== VICONIJ IF ~IsValidForPartyDialog("Keto")
 IsValidForPartyDialog("Viconia")~ THEN ~Never forget, girl, that given fifteen seconds and a tentacle rod, I could permanently alter your perception of reality.~
 END
