--- questpack\areas\kettaatk.baf	Wed Apr 05 21:21:46 2006
+++ C:\BWP Patchstudio\patched files\questpack\areas\kettaatk.baf	Tue Feb 16 22:37:55 2010
@@ -3,8 +3,8 @@
 	DifficultyGT(EASY)
 THEN
 	RESPONSE #100
-		CreateCreature("d0battho",[644.842],13)
-		CreateCreature("d0battho",[1074.549],13)
+		CreateCreature("d0batho",[644.842],13)
+		CreateCreature("d0batho",[1074.549],13)
 		CreateCreature("stalke",[750.752],13)
 		CreateCreature("stalke",[834.701],13)
 		CreateCreature("stalke",[964.604],13)
