--- questpack\helltest\selfish.d	Sun Mar 27 23:43:22 2005
+++ C:\BWP Patchstudio\patched files\questpack\helltest\selfish.d	Tue Feb 16 20:59:52 2010
@@ -10,8 +10,8 @@
 IF ~~ THEN BEGIN 25 // from: 24.0
   SAY #55116
   IF ~~ THEN REPLY @43 GOTO self34
-  IF ~!Gender("Player2",FEMALE)~ THEN REPLY @47 GOTO self34
-  IF ~Gender("Player2",FEMALE)~ THEN REPLY @44 GOTO self34
+  IF ~!Gender(Player2,FEMALE)~ THEN REPLY @47 GOTO self34
+  IF ~Gender(Player2,FEMALE)~ THEN REPLY @44 GOTO self34
   IF ~~ THEN REPLY @45 GOTO self34
   IF ~~ THEN REPLY @46 GOTO self36
 END
@@ -153,7 +153,8 @@
   IF ~~ THEN DO ~SetGlobal("Player1Selfish","GLOBAL",3)
 IncrementGlobal("C7SinCounter","GLOBAL",1)
 SetGlobal("OpenedDoor5","AR2904",2)
-SetGlobal("OpenedDoor6","AR2904",2)~ JOURNAL @55 FLAGS 64 GOTO 32
+SetGlobal("OpenedDoor6","AR2904",2)
+ForceSpell(Player2Fill,HELL_DISPELL)~ JOURNAL @55 FLAGS 64 GOTO 32
 END
 
 END
