--- gminion\Setup-gMinion.tp2	Sat Apr 26 21:15:54 2008
+++ C:\BWP Patchstudio\patched files\gminion\Setup-gMinion.tp2	Wed Apr 24 07:54:45 2013
@@ -3,13 +3,18 @@
  *
  * AI scripts for minions summoned by the party
  *
+ * expanded for Mega-Installations by Azazello
  */
 
 BACKUP ~gminion\build1.8\backup~
 
 AUTHOR ~cirerrek@hotmail.com~
 
-VERSION ~v2~
+VERSION ~v2 BGT-Expansion~
+
+ALLOW_MISSING
+  // Silver Shadow from The Darkest Day mod - Azazello
+  ~SILVERSH.CRE~
 
 README ~gminion\build1.8\readme-gminion.html~
 
@@ -70,6 +75,260 @@
 
 EXTEND_BOTTOM ~WIZELSUM.BCS~ ~gminion\build1.8\baf\melee.baf~
 
+// Additional changes for any (ToB?) type of installation - Azazello
+EXTEND_TOP    ~WARDOG.BCS~ ~gminion\build1.8\baf\melee.baf~
+EXTEND_TOP    ~WYVERN.BCS~ ~gminion\build1.8\baf\melee.baf~
+EXTEND_BOTTOM ~BORNWAIT.BCS~ ~gminion\build1.8\baf\melee.baf~
+EXTEND_BOTTOM ~SERVSU.BCS~   ~gminion\build1.8\baf\melee.baf~
+EXTEND_BOTTOM ~WIZELSU2.BCS~ ~gminion\build1.8\baf\melee.baf~
+
+// Scripts used by Summons from...Big Picture mod - Azazello
+ACTION_IF FILE_EXISTS_IN_GAME ~BPBAG04.ITM~ THEN BEGIN
+  EXTEND_TOP    ~BPSMSHT1.BCS~ ~gminion\build1.8\baf\melee.baf~
+  EXTEND_TOP    ~BPSMSHT2.BCS~ ~gminion\build1.8\baf\melee.baf~
+  EXTEND_TOP    ~BPEFREET.BCS~ ~gminion\build1.8\baf\spell.baf~
+END
+
+// Scripts used by Summons from...Check the Bodies mod - Azazello
+ACTION_IF FILE_EXISTS_IN_GAME ~CBWTASGT.BCS~ THEN BEGIN
+  EXTEND_TOP    ~CBWTASGT.BCS~ ~gminion\build1.8\baf\melee.baf~
+END
+
+// Scripts used by Summons from...The Darkest Day mod - Azazello
+ACTION_IF FILE_EXISTS_IN_GAME ~SILVERSH.CRE~ THEN BEGIN
+// Silver Shadow from The Darkest Day mod
+  EXTEND_BOTTOM ~SILVERSH.BCS~ ~gminion\build1.8\baf\spell.baf~
+END
+
+// Scripts used by Summons from...aTweaks mod - Azazello
+ACTION_IF FILE_EXISTS_IN_GAME ~fl#dkfa.spl~ THEN BEGIN
+   EXTEND_TOP   ~FL#SKLSU.BCS~ ~gminion\build1.8\baf\melee.baf~
+   EXTEND_TOP   ~FL#SKWSU.BCS~ ~gminion\build1.8\baf\melee.baf~
+END
+
+/*
+ * Azazello
+ * If using Big Picture (BP) or Sword Coast Strategems 2 (SCS II), ensure that summons use gMinion scripting EXCLUSIVELY.
+ * {Actually, this scripting might be redundant}
+ */
+ACTION_IF (FILE_EXISTS_IN_GAME ~BPBAG04.ITM~ OR FILE_EXISTS_IN_GAME ~scsII/scsII.tph~) THEN BEGIN
+  COPY_EXISTING ~BEARBLSU.CRE~		~override~
+                ~BEARBRSU.CRE~		~override~
+                ~BEARCASU.CRE~		~override~
+                ~BEARPOSU.CRE~		~override~
+//              ~BERSER02.CRE~		~override~
+//              ~BERSER03.CRE~		~override~
+//              ~BERSERSU.CRE~		~override~
+                ~CARRIOSU.CRE~		~override~
+                ~CATJAGSU.CRE~		~override~
+                ~CATLIOSU.CRE~		~override~
+                ~CATLIOWP.CRE~		~override~
+                ~DEVAEVIL.CRE~		~override~
+                ~DEVAGOOD.CRE~		~override~
+                ~DJINNISU.CRE~		~override~
+                ~DOGWASU.CRE~		~override~
+                ~DOGWAWP.CRE~		~override~
+                ~DOGWISU.CRE~		~override~
+                ~EFREETSU.CRE~		~override~
+                ~ELAIRSU1.CRE~		~override~
+                ~ELAIRSU2.CRE~		~override~
+                ~ELAIRSU3.CRE~		~override~
+                ~ELAIRSUW.CRE~		~override~
+                ~ELEARSU1.CRE~		~override~
+                ~ELEARSU2.CRE~		~override~
+                ~ELEARSU3.CRE~		~override~
+                ~ELEARSU4.CRE~		~override~
+                ~ELEMCHAN.CRE~		~override~
+                ~ELEMSUNN.CRE~		~override~
+                ~ELEMZAAM.CRE~		~override~
+                ~ELFIRSU1.CRE~		~override~
+                ~ELFIRSU2.CRE~		~override~
+                ~ELFIRSU3.CRE~		~override~
+                ~ELFIRSU4.CRE~		~override~
+                ~ETTERCSU.CRE~		~override~
+                ~GHASTSU.CRE~		~override~
+                ~GNOLLSU.CRE~		~override~
+                ~HAKSU.CRE~		~override~
+                ~HOBGOBSU.CRE~		~override~
+                ~KOBOLDSU.CRE~		~override~
+                ~MOONDOG.CRE~		~override~
+                ~NISHRUSU.CRE~		~override~
+                ~NYMPHSU.CRE~		~override~
+                ~OGREGRSU.CRE~		~override~
+                ~OGRELESU.CRE~		~override~
+                ~OGRESU.CRE~		~override~
+                ~PLANEVIL.CRE~		~override~
+                ~PLANGOOD.CRE~		~override~
+                ~RABBITSU.CRE~		~override~
+                ~SERVSU.CRE~		~override~
+                ~SKELWASU.CRE~		~override~
+                ~SMOUNDSU.CRE~		~override~
+                ~SPIDFGSU.CRE~		~override~
+                ~SPIDGISU.CRE~		~override~
+                ~SPIDPHSU.CRE~		~override~
+                ~SPIDSWSU.CRE~		~override~
+                ~SPIRBEAR.CRE~		~override~
+                ~SPIRLION.CRE~		~override~
+                ~SPIRSNAK.CRE~		~override~
+                ~SPIRWOLF.CRE~		~override~
+                ~STALKE.CRE~		~override~
+                ~SUMDJINN.CRE~		~override~
+                ~SUMEFREE.CRE~		~override~
+                ~SUMELAIR.CRE~		~override~
+                ~SUMELEAR.CRE~		~override~
+                ~SUMELFIR.CRE~		~override~
+                ~SWAAIR01.CRE~		~override~
+                ~SWAEAR01.CRE~		~override~
+                ~SWAFIR01.CRE~		~override~
+                ~SWORD01.CRE~		~override~
+                ~SWORD02.CRE~		~override~
+                ~SWORD68.CRE~		~override~
+                ~WOLFDISU.CRE~		~override~
+                ~WOLFWWSU.CRE~		~override~
+                ~WORGSU.CRE~		~override~
+                ~WYVERNSU.CRE~		~override~
+// Silver Shadow from TDD - // added by Azazello
+                ~SILVERSH.CRE~		~override~
+    PATCH_IF (SOURCE_SIZE > 0x2d3) THEN BEGIN // protects against invalid files
+      READ_ASCII 0x248 "Override"
+      READ_ASCII 0x250 "Class"
+      READ_ASCII 0x258 "Race"
+      READ_ASCII 0x260 "General"
+      READ_ASCII 0x268 "Default"
+
+      PATCH_IF (("%Override%" STRING_COMPARE_CASE "BPWDASGT") = 0) BEGIN
+        WRITE_ASCII 0x248 ~WTASIGHT~
+      END
+      PATCH_IF (("%Class%" STRING_COMPARE_CASE "BPWDASGT") = 0) BEGIN
+        WRITE_ASCII 0x250 ~WTASIGHT~
+      END
+      PATCH_IF (("%Race%" STRING_COMPARE_CASE "BPWDASGT") = 0) BEGIN
+        WRITE_ASCII 0x258 ~WTASIGHT~
+      END
+      PATCH_IF (("%General%" STRING_COMPARE_CASE "BPWDASGT") = 0) BEGIN
+        WRITE_ASCII 0x260 ~WTASIGHT~
+      END
+      PATCH_IF (("%Default%" STRING_COMPARE_CASE "BPWDASGT") = 0) BEGIN
+        WRITE_ASCII 0x268 ~WTASIGHT~
+      END
+
+      PATCH_IF (("%Override%" STRING_COMPARE_CASE "BPWTSIGT") = 0) BEGIN
+        WRITE_ASCII 0x248 ~WTASIGHT~
+      END
+      PATCH_IF (("%Class%" STRING_COMPARE_CASE "BPWTSIGT") = 0) BEGIN
+        WRITE_ASCII 0x250 ~WTASIGHT~
+      END
+      PATCH_IF (("%Race%" STRING_COMPARE_CASE "BPWTSIGT") = 0) BEGIN
+        WRITE_ASCII 0x258 ~WTASIGHT~
+      END
+      PATCH_IF (("%General%" STRING_COMPARE_CASE "BPWTSIGT") = 0) BEGIN
+        WRITE_ASCII 0x260 ~WTASIGHT~
+      END
+      PATCH_IF (("%Default%" STRING_COMPARE_CASE "BPWTSIGT") = 0) BEGIN
+        WRITE_ASCII 0x268 ~WTASIGHT~
+      END
+
+      PATCH_IF (("%Override%" STRING_COMPARE_CASE "dw#melee") = 0) BEGIN
+        WRITE_ASCII 0x248 ~WTASIGHT~
+      END
+      PATCH_IF (("%Class%" STRING_COMPARE_CASE "dw#melee") = 0) BEGIN
+        WRITE_ASCII 0x250 ~WTASIGHT~
+      END
+      PATCH_IF (("%Race%" STRING_COMPARE_CASE "dw#melee") = 0) BEGIN
+        WRITE_ASCII 0x258 ~WTASIGHT~
+      END
+      PATCH_IF (("%General%" STRING_COMPARE_CASE "dw#melee") = 0) BEGIN
+        WRITE_ASCII 0x260 ~WTASIGHT~
+      END
+      PATCH_IF (("%Default%" STRING_COMPARE_CASE "dw#melee") = 0) BEGIN
+        WRITE_ASCII 0x268 ~WTASIGHT~
+      END
+
+      PATCH_IF (("%Override%" STRING_COMPARE_CASE "dw#range") = 0) BEGIN
+        WRITE_ASCII 0x248 ~WTARSGT~
+      END
+      PATCH_IF (("%Class%" STRING_COMPARE_CASE "dw#range") = 0) BEGIN
+        WRITE_ASCII 0x250 ~WTARSGT~
+      END
+      PATCH_IF (("%Race%" STRING_COMPARE_CASE "dw#range") = 0) BEGIN
+        WRITE_ASCII 0x258 ~WTARSGT~
+      END
+      PATCH_IF (("%General%" STRING_COMPARE_CASE "dw#range") = 0) BEGIN
+        WRITE_ASCII 0x260 ~WTARSGT~
+      END
+      PATCH_IF (("%Default%" STRING_COMPARE_CASE "dw#range") = 0) BEGIN
+        WRITE_ASCII 0x268 ~WTARSGT~
+      END
+
+      PATCH_IF (("%Override%" STRING_COMPARE_CASE "dw#wizardslay") = 0) BEGIN
+        WRITE_ASCII 0x248 ~WTASIGHT~
+      END
+      PATCH_IF (("%Class%" STRING_COMPARE_CASE "dw#wizardslay") = 0) BEGIN
+        WRITE_ASCII 0x250 ~WTASIGHT~
+      END
+      PATCH_IF (("%Race%" STRING_COMPARE_CASE "dw#wizardslay") = 0) BEGIN
+        WRITE_ASCII 0x258 ~WTASIGHT~
+      END
+      PATCH_IF (("%General%" STRING_COMPARE_CASE "dw#wizardslay") = 0) BEGIN
+        WRITE_ASCII 0x260 ~WTASIGHT~
+      END
+      PATCH_IF (("%Default%" STRING_COMPARE_CASE "dw#wizardslay") = 0) BEGIN
+        WRITE_ASCII 0x268 ~WTASIGHT~
+      END
+
+      PATCH_IF (("%Override%" STRING_COMPARE_CASE "dw#wtacor") = 0) BEGIN
+        WRITE_ASCII 0x248 ~WTASIGHT~
+      END
+      PATCH_IF (("%Class%" STRING_COMPARE_CASE "dw#wtacor") = 0) BEGIN
+        WRITE_ASCII 0x250 ~WTASIGHT~
+      END
+      PATCH_IF (("%Race%" STRING_COMPARE_CASE "dw#wtacor") = 0) BEGIN
+        WRITE_ASCII 0x258 ~WTASIGHT~
+      END
+      PATCH_IF (("%General%" STRING_COMPARE_CASE "dw#wtacor") = 0) BEGIN
+        WRITE_ASCII 0x260 ~WTASIGHT~
+      END
+      PATCH_IF (("%Default%" STRING_COMPARE_CASE "dw#wtacor") = 0) BEGIN
+        WRITE_ASCII 0x268 ~WTASIGHT~
+      END
+
+      PATCH_IF (("%Override%" STRING_COMPARE_CASE "WDASIGHT") = 0) BEGIN
+        WRITE_ASCII 0x248 ~WTASIGHT~
+      END
+      PATCH_IF (("%Class%" STRING_COMPARE_CASE "WDASIGHT") = 0) BEGIN
+        WRITE_ASCII 0x250 ~WTASIGHT~
+      END
+      PATCH_IF (("%Race%" STRING_COMPARE_CASE "WDASIGHT") = 0) BEGIN
+        WRITE_ASCII 0x258 ~WTASIGHT~
+      END
+      PATCH_IF (("%General%" STRING_COMPARE_CASE "WDASIGHT") = 0) BEGIN
+        WRITE_ASCII 0x260 ~WTASIGHT~
+      END
+      PATCH_IF (("%Default%" STRING_COMPARE_CASE "WDASIGHT") = 0) BEGIN
+        WRITE_ASCII 0x268 ~WTASIGHT~
+      END
+
+      PATCH_IF (("%Override%" STRING_COMPARE_CASE "WDARSGT") = 0) BEGIN
+        WRITE_ASCII 0x248 ~WTARSGT~
+      END
+      PATCH_IF (("%Class%" STRING_COMPARE_CASE "WDARSGT") = 0) BEGIN
+        WRITE_ASCII 0x250 ~WTARSGT~
+      END
+      PATCH_IF (("%Race%" STRING_COMPARE_CASE "WDARSGT") = 0) BEGIN
+        WRITE_ASCII 0x258 ~WTARSGT~
+      END
+      PATCH_IF (("%General%" STRING_COMPARE_CASE "WDARSGT") = 0) BEGIN
+        WRITE_ASCII 0x260 ~WTARSGT~
+      END
+      PATCH_IF (("%Default%" STRING_COMPARE_CASE "WDARSGT") = 0) BEGIN
+        WRITE_ASCII 0x268 ~WTARSGT~
+      END
+
+    END
+
+  BUT_ONLY_IF_IT_CHANGES
+END
+
 /*
  * Copy the modified .cre files to the override directory
  *
@@ -78,7 +337,7 @@
  * making it more MOD friendly.
  *
  */
-  
+
 // djinnisu has extra spells
 COPY_EXISTING ~djinnisu.cre~ ~override~
               ~sumdjinn.cre~ ~override~
