--- ajantisbg2\dialogues\ajantis_anomen.d	Sun Mar 15 09:10:12 2009
+++ C:\BWP Patchstudio\patched files\ajantisbg2\dialogues\ajantis_anomen.d	Sun Apr 13 08:20:41 2014
@@ -59,10 +59,10 @@
 IF WEIGHT #-1
 ~CombatCounter(0)
 !See([ENEMY])
-!Dead("anomen")
-InParty("anomen")
-See("anomen")
-!StateCheck("anomen",CD_STATE_NOTVALID)
+!Dead("C#Ajantis")
+InParty("C#Ajantis")
+See("C#Ajantis")
+!StateCheck("C#Ajantis",CD_STATE_NOTVALID)
 Global("C#AjantisAnomen1","GLOBAL",1)
 Global("AnomenIsNotKnight","GLOBAL",0)~ THEN ~BANOMEN~ ajantis_anomen_2
 @15 DO ~SetGlobal("C#AjantisAnomen1","GLOBAL",2)~ 
@@ -81,10 +81,10 @@
 CHAIN
 IF ~CombatCounter(0)
 !See([ENEMY])
-!Dead("C#Ajantis")
-InParty("C#Ajantis")
-See("C#Ajantis")
-!StateCheck("C#Ajantis",CD_STATE_NOTVALID)
+!Dead("anomen")
+InParty("anomen")
+See("anomen")
+!StateCheck("anomen",CD_STATE_NOTVALID)
 Global("C#AjantisCNAnomen1","GLOBAL",0)
 Global("AnomenIsNotKnight","GLOBAL",1)~ THEN ~C#AjanJ~ ajantis_anomen_CN0
 @30 DO ~SetGlobal("C#AjantisCNAnomen1","GLOBAL",1)~ 
