--- ajantisbg2/scripts/c#ajan.baf	2014-11-01 17:10:26.000000000 -0400
+++ Fixpack/ajantisbg2/scripts/c#ajan.baf	2016-11-04 14:55:56.986718500 -0400
@@ -1025,7 +1025,7 @@
 IF
 InParty(Myself) 
 !Alignment(Player1,MASK_EVIL)
-GlobalLT("chapter","GLOBAL",3)
+GlobalLT("chapter","GLOBAL",%bg2_chapter_3%)
 Global("C#AjantisBadRepTimer","GLOBAL",0)
 !Global("C#AjantisNoBadRepJoin","GLOBAL",1)
 THEN
@@ -1039,7 +1039,7 @@
 InParty(Myself) 
 OR(2)
 Alignment(Player1,MASK_EVIL)
-GlobalGT("CHAPTER","GLOBAL",2)
+GlobalGT("CHAPTER","GLOBAL",%bg2_chapter_2%)
 Global("C#AjantisBadRepTimer","GLOBAL",0)
 !Global("C#AjantisNoBadRepJoin","GLOBAL",1)
 THEN
@@ -1475,7 +1475,7 @@
 	!See([ENEMY])
 	CombatCounter(0)
 	!AreaCheck("AR2402")
-	GlobalGT("Chapter","GLOBAL",5)
+	GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 	Global("C#AjantisAdalonLeaves","GLOBAL",0)
 THEN
 	RESPONSE #100
