--- ajantisbg2/scripts/c#ar0406.baf	2008-01-09 22:50:34.000000000 -0500
+++ Fixpack/ajantisbg2/scripts/c#ar0406.baf	2016-12-07 10:05:12.820674900 -0500
@@ -1,7 +1,9 @@
 IF 
-	Exists("firkra01")
+	Global("firkra01removed","AR0406",0)
 	Global("C#AjantisPCInWindspeerHills","GLOBAL",1)
+	Exists("firkra01")
 THEN
  RESPONSE #100
 	ActionOverride("firkra01",DestroySelf())
+	SetGlobal("firkra01removed","AR0406",1)
 END
