--- ajantisbg2/dialogues/c#orden.d	2008-11-24 22:47:12.000000000 -0500
+++ Fixpack/ajantisbg2/dialogues/c#orden.d	2016-11-04 14:55:56.982715700 -0400
@@ -146,7 +146,7 @@
 Global("C#AjantisEngaged","GLOBAL",1)
 !Global("C#AjantisRomanceActive","GLOBAL",3)
 Global("C#AjantisAskedWhereOrden","GLOBAL",0)~ + @108 DO ~SetGlobal("C#AjantisAskedWhereOrden","GLOBAL",1)~ + KAYL2_report_31_afterbodhi
-  IF ~GlobalGT("chapter","GLOBAL",3)~ THEN REPLY #45842 EXIT
+  IF ~GlobalGT("chapter","GLOBAL",%bg2_chapter_3%)~ THEN REPLY #45842 EXIT
 END
 
 ADD_TRANS_TRIGGER ~KAYL2~ 33 ~False()~
