--- Coran/Dialogue/O#Cor25J.d	2011-03-08 10:34:00.000000000 -0500
+++ Fixpack/Coran/Dialogue/O#Cor25J.d	2016-11-21 20:32:06.160665300 -0500
@@ -315,14 +315,6 @@
 IF ~~ EXIT
 END
 
-EXTEND_TOP FATESP 6 #3
-+ ~!Dead("O#Coran") !InMyArea("O#Coran") Global("O#CoranSummoned","GLOBAL",0)~ + @107
-DO ~CreateVisualEffect("SPPORTAL",[1999.1228])
-Wait(2) 
-CreateCreature("O#Cor25",[1999.1228],0) 
-SetGlobal("O#CoranSummoned","GLOBAL",1)~ GOTO 8
-END
-
 EXTEND_TOP SARVOLO 9 #3
 + ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID)~ + @108 + O#CoranVoloBio1
 END
@@ -418,59 +410,59 @@
 // Watcher's Keep interjections, ToB
 
 I_C_T GORPOL1 1 O#CoranMetKnightsOfTheVigil1
-== O#COR25J IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @130
+== O#COR25J IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @130
 END
 
 I_C_T2 GORCHR 11 O#CoranChromaticDemon2
-== O#COR25J IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @131
+== O#COR25J IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @131
 END
 
 I_C_T GORIMP01 4 O#CoranImpDeadWizards1
-== O#COR25J IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @132
+== O#COR25J IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @132
 END
 
 I_C_T GORCAR 3 O#CoranCarston1
-== O#COR25J IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @133
+== O#COR25J IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @133
 END
 
 I_C_T GORCAMB 3 O#CoranCambion3
-== O#COR25J IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @134
+== O#COR25J IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @134
 END
 
 I_C_T GORDEMO 1 O#CoranImprisonedOneMet1
-== O#COR25J IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @135
+== O#COR25J IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @135
 END
 
 I_C_T GORODR1 35 O#CoranOdrenFoolishness1
-== O#COR25J IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @136
+== O#COR25J IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @136
 END
 
 // Watcher's Keep interjections, SoA
 
 I_C_T GORPOL1 1 O#CoranMetKnightsOfTheVigil1
-== O#CORANJ IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @130
+== O#CORANJ IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @130
 END
 
 I_C_T2 GORCHR 11 O#CoranChromaticDemon2
-== O#CORANJ IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @131
+== O#CORANJ IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @131
 END
 
 I_C_T GORIMP01 4 O#CoranImpDeadWizards1
-== O#CORANJ IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @132
+== O#CORANJ IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @132
 END
 
 I_C_T GORCAR 3 O#CoranCarston1
-== O#CORANJ IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @133
+== O#CORANJ IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @133
 END
 
 I_C_T GORCAMB 3 O#CoranCambion3
-== O#CORANJ IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @134
+== O#CORANJ IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @134
 END
 
 I_C_T GORDEMO 1 O#CoranImprisonedOneMet1
-== O#CORANJ IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @135
+== O#CORANJ IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @135
 END
 
 I_C_T GORODR1 35 O#CoranOdrenFoolishness1
-== O#CORANJ IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @136
+== O#CORANJ IF ~InParty("O#Coran") InMyArea("O#Coran") !StateCheck("O#Coran",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @136
 END
