--- Coran/Scripts/O#CoranS.baf	2011-03-07 19:00:46.000000000 -0500
+++ Fixpack/Coran/Scripts/O#CoranS.baf	2016-11-21 21:54:57.762824100 -0500
@@ -113,7 +113,7 @@
 !StateCheck(Myself,CD_STATE_NOTVALID)
 CombatCounter(0)
 !See([ENEMY])
-Global("Chapter","GLOBAL",4)
+Global("Chapter","GLOBAL",%bg2_chapter_4%)
 GlobalLT("O#CoranPCChapter","GLOBAL",8)
 THEN
 RESPONSE #100
@@ -128,7 +128,7 @@
 !StateCheck(Myself,CD_STATE_NOTVALID)
 CombatCounter(0)
 !See([ENEMY])
-Global("Chapter","GLOBAL",5)
+Global("Chapter","GLOBAL",%bg2_chapter_5%)
 GlobalLT("O#CoranPCChapter","GLOBAL",10)
 THEN
 RESPONSE #100
@@ -143,7 +143,7 @@
 !StateCheck(Myself,CD_STATE_NOTVALID)
 CombatCounter(0)
 !See([ENEMY])
-Global("Chapter","GLOBAL",6)
+Global("Chapter","GLOBAL",%bg2_chapter_6%)
 GlobalLT("O#CoranPCChapter","GLOBAL",12)
 THEN
 RESPONSE #100
@@ -158,7 +158,7 @@
 !StateCheck(Myself,CD_STATE_NOTVALID)
 CombatCounter(0)
 !See([ENEMY])
-Global("Chapter","GLOBAL",7)
+Global("Chapter","GLOBAL",%bg2_chapter_7%)
 GlobalLT("O#CoranPCChapter","GLOBAL",14)
 THEN
 RESPONSE #100
