--- Coran/Setup-Coran.tp2	2014-11-10 15:53:46.000000000 -0500
+++ Fixpack/Coran/Setup-Coran.tp2	2016-11-22 19:31:31.891112700 -0500
@@ -2,27 +2,34 @@
 
 AUTHOR ~http://forums.pocketplane.net~
 
-VERSION ~v5~ 
+VERSION ~v5 BWP Fix~ 
 
 README ~Coran/%LANGUAGE%/Readme-Coran.txt~ ~Coran/Readme-Coran.txt~ 
 
 ALWAYS
-  // Borrowed from Edwin Romance: convert strings to UTF-8 for BGEE/BG2EE
-  INCLUDE ~%MOD_FOLDER%/handle_charsets.tpa~
+  ACTION_IF GAME_IS ~eet~ BEGIN
+    OUTER_SET bg2_chapter = 12
+  END ELSE BEGIN
+    OUTER_SET bg2_chapter = 0
+  END
+  OUTER_FOR (i=1; i<=10; i=i+1) BEGIN
+    OUTER_SET bg2_chapter = bg2_chapter + 1
+    OUTER_SPRINT name_source ~bg2_chapter_%i%~
+    OUTER_SET EVAL ~%name_source%~ = bg2_chapter
+  END
 
   // No exception, everything needs to be converted
   ACTION_DEFINE_ARRAY O#noconvert BEGIN END
 
-  // Only setup-mod.tra needs to be reloaded
+  // any tra files used in LANGUAGE need to be reloaded after UTF-8 conversion
   ACTION_DEFINE_ARRAY O#reload BEGIN Setup-Coran END
 
-  // There is no "language" folder so the root of all languages in %MOD_FOLDER%
+  // There is no "language" folder so the root of all languages is %MOD_FOLDER%
   LAF HANDLE_CHARSETS
     INT_VAR
       infer_charset = 1
     STR_VAR
       tra_path = EVAL ~%MOD_FOLDER%~
-      // charset_table = O#charsets // Included for illustrative purposes.
       noconvert_array = O#noconvert
       reload_array = O#reload
   END
@@ -90,7 +97,7 @@
 
 COPY ~Coran/Audio~ ~override~
 
-COMPILE ~Coran/Dialogue/BO#Coran.d~
+COMPILE EVALUATE_BUFFER ~Coran/Dialogue/BO#Coran.d~
 	~Coran/Dialogue/O#Coran.d~
 	~Coran/Dialogue/O#CoranJ.d~
 	~Coran/Dialogue/O#CoranP.d~
@@ -154,14 +161,14 @@
 WRITE_ASCII 0x2cc ~O#Cor25~ #8 // dialogue
 WRITE_ASCII 0x280 ~O#Coran~ #32 //DV
 
-COMPILE ~Coran/Dialogue/BO#Cor25.d~
+COMPILE EVALUATE_BUFFER ~Coran/Dialogue/BO#Cor25.d~
 	~Coran/Dialogue/O#Cor25.d~
 	~Coran/Dialogue/O#Cor25J.d~
 	~Coran/Dialogue/O#Cor25P.d~
 
 	~Coran/Scripts/O#Cor25S.baf~
 
-ACTION_IF GAME_IS ~BG2EE~ BEGIN
+ACTION_IF GAME_IS ~bg2ee eet~ BEGIN
 COMPILE	~Coran/Dialogue/CoranEE.d~
 END
 
@@ -178,6 +185,25 @@
 APPEND ~pdialog.2da~ ~O#Coran        O#CoranP             O#CoranJ            O#CoranD             O#Cor25P             O#Cor25J             O#Cor25D       O#Cor25S~
 UNLESS ~O#Coran~
 
+ACTION_IF GAME_IS ~eet~ BEGIN
+  INCLUDE ~EET/other/EET_functions.tph~
+  LAF ~EET_NPC_TRANSITION~
+    INT_VAR
+      type = 2
+      clean_ToB = 1
+    STR_VAR
+      dv = "O#Coran"
+      override_SoA = "O#CORANS"
+      override_ToB = "O#COR25S"
+      dialog_ToB = "O#COR25"
+      traFile = EVAL "Coran/%LANGUAGE%/O#Cor25J.tra"
+      string = "@107" //Bring me Coran, the elven fighter and thief.
+      stringPosDV = "Corwin"
+      clean_ToB_DV = "Coran"
+  END
+END ELSE BEGIN
+  COMPILE ~Coran/Dialogue/FATESP.d~ USING ~Coran/%LANGUAGE%/O#Cor25J.tra~
+END
 
 BEGIN @46
 
