--- iepbanters\dialogue/banters.d	Thu Feb 07 16:44:52 2013
+++ C:\BWP Patchstudio\patched files\iepbanters\dialogue/banters.d	Sat Dec 21 14:43:07 2013
@@ -1725,7 +1725,7 @@
 
 // Valygar
 // 1. (after seeing aboleth)
-CHAIN IF ~Global("I#B_ValygarViconia_Aboleth","GLOBAL",1)~ THEN VICONIJ I#B_ValygarViconia_Aboleth
+CHAIN IF ~Global("I#B_ValygarViconia_Aboleth","GLOBAL",2)~ THEN VICONIJ I#B_ValygarViconia_Aboleth
 @512 
 DO ~IncrementGlobal("I#B_ValygarViconia_Aboleth","GLOBAL",1)~
 == VALYGARJ @513
