--- iepbanters/dialogue/banters.d	2013-02-07 10:44:52.000000000 -0500
+++ Fixpack/iepbanters/dialogue/banters.d	2016-11-21 11:25:44.306362600 -0500
@@ -638,7 +638,7 @@
 CHAIN IF ~%BGT_VAR%
 InParty("Imoen2")
 See("Imoen2")
-GlobalGT("Chapter","GLOBAL",1)
+GlobalGT("Chapter","GLOBAL",%bg2_chapter_1%)
 !StateCheck("Imoen2",CD_STATE_NOTVALID)
 !StateCheck("Jaheira",CD_STATE_NOTVALID)
 CombatCounter(0)
@@ -986,7 +986,7 @@
 OR(2)
   Alignment(Player1,MASK_GOOD)
   Alignment(Player1,NEUTRAL)
-GlobalGT("Chapter","GLOBAL",5)
+GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("I#B_AerieMazzy_PC","GLOBAL",0)~ THEN BMAZZY I#B_AerieMazzy_PC
 @253 
 DO ~SetGlobal("I#B_AerieMazzy_PC","GLOBAL",1)~
@@ -1182,7 +1182,7 @@
 Global("I#B_JahMinLoss","GLOBAL",0)
 See("Jaheira")
 CombatCounter(0)
-!Global("chapter","GLOBAL",1)
+!Global("chapter","GLOBAL",%bg2_chapter_1%)
 !See([ENEMY])~
 THEN BMINSC I#B_JahMinLoss
 @322 
@@ -1627,7 +1627,7 @@
 !StateCheck("Minsc",CD_STATE_NOTVALID)
 !StateCheck("Viconia",CD_STATE_NOTVALID)
 CombatCounter(0)
-Global("chapter","GLOBAL",5)
+Global("chapter","GLOBAL",%bg2_chapter_5%)
 Global("I#B_MinscViconia_Underdark","GLOBAL",0)~ THEN BVICONI I#B_MinscViconia_Underdark
 @436 
 DO ~SetGlobal("I#B_MinscViconia_Underdark","GLOBAL",1)~
@@ -1725,7 +1725,7 @@
 
 // Valygar
 // 1. (after seeing aboleth)
-CHAIN IF ~Global("I#B_ValygarViconia_Aboleth","GLOBAL",1)~ THEN VICONIJ I#B_ValygarViconia_Aboleth
+CHAIN IF ~Global("I#B_ValygarViconia_Aboleth","GLOBAL",2)~ THEN VICONIJ I#B_ValygarViconia_Aboleth
 @512 
 DO ~IncrementGlobal("I#B_ValygarViconia_Aboleth","GLOBAL",1)~
 == VALYGARJ @513
