--- iepbanters/scripts/minsc.baf	2013-02-07 10:44:52.000000000 -0500
+++ Fixpack/iepbanters/scripts/minsc.baf	2016-11-26 02:07:15.813998200 -0500
@@ -1,6 +1,7 @@
 // Anomen is Knight
 IF
   InParty("Anomen")
+  InParty(Myself)
   !StateCheck("Anomen",CD_STATE_NOTVALID)
   !StateCheck(Myself,CD_STATE_NOTVALID)
   GlobalGT("AnomenIsKnight","GLOBAL",0)
@@ -16,6 +17,7 @@
 
 IF
   InParty("Anomen")
+  InParty(Myself)
   !StateCheck("Anomen",CD_STATE_NOTVALID)
   !StateCheck(Myself,CD_STATE_NOTVALID)
   Global("I#B_AnoMinProud","GLOBAL",1)
@@ -30,6 +32,7 @@
 // Anomen isn't a Knight
 IF 
   InParty("Anomen")
+  InParty(Myself)
   !StateCheck("Anomen",CD_STATE_NOTVALID)
   !StateCheck(Myself,CD_STATE_NOTVALID)
   GlobalGT("AnomenIsNotKnight","GLOBAL",0)
@@ -45,6 +48,7 @@
 
 IF
   InParty("Anomen")
+  InParty(Myself)
   !StateCheck("Anomen",CD_STATE_NOTVALID)
   !StateCheck(Myself,CD_STATE_NOTVALID)
   Global("I#B_AnoMinCN","GLOBAL",1)
