--- iepbanters\scripts\minsc.baf	Thu Feb 7 16:44:52 2013
+++ C:\BWP Patchstudio\patched files\iepbanters\scripts\minsc.baf	Sun Aug 23 15:51:13 2015
@@ -1,6 +1,7 @@
 // Anomen is Knight
 IF
   InParty("Anomen")
+  InParty(Myself)
   !StateCheck("Anomen",CD_STATE_NOTVALID)
   !StateCheck(Myself,CD_STATE_NOTVALID)
   GlobalGT("AnomenIsKnight","GLOBAL",0)
@@ -16,6 +17,7 @@
 
 IF
   InParty("Anomen")
+  InParty(Myself)
   !StateCheck("Anomen",CD_STATE_NOTVALID)
   !StateCheck(Myself,CD_STATE_NOTVALID)
   Global("I#B_AnoMinProud","GLOBAL",1)
@@ -30,6 +32,7 @@
 // Anomen isn't a Knight
 IF 
   InParty("Anomen")
+  InParty(Myself)
   !StateCheck("Anomen",CD_STATE_NOTVALID)
   !StateCheck(Myself,CD_STATE_NOTVALID)
   GlobalGT("AnomenIsNotKnight","GLOBAL",0)
@@ -45,6 +48,7 @@
 
 IF
   InParty("Anomen")
+  InParty(Myself)
   !StateCheck("Anomen",CD_STATE_NOTVALID)
   !StateCheck(Myself,CD_STATE_NOTVALID)
   Global("I#B_AnoMinCN","GLOBAL",1)
