--- iepbanters\setup-iepbanters.tp2	Sun Apr 20 15:33:22 2014
+++ C:\BWP Patchstudio\patched files\iepbanters\setup-iepbanters.tp2	Sun Jun 28 21:29:10 2015
@@ -10,7 +10,7 @@
 // Error reports sent to K'aeloree
 AUTHOR ~K'aeloree (kae@spellholdstudios.net)~
 
-VERSION ~V4.2_Isaya~
+VERSION ~V4.2_Isaya BWP Fix~
 
 // Default to English readme if not found in the selected language
 README ~iepbanters/translations/%LANGUAGE%/iepbanters_readme.html~ ~iepbanters/translations/english/iepbanters_readme.html~
