--- iepbanters/setup-iepbanters.tp2	2014-04-20 06:03:22.196000000 -0400
+++ Fixpack/iepbanters/setup-iepbanters.tp2	2016-11-23 14:36:32.657122300 -0500
@@ -10,7 +10,7 @@
 // Error reports sent to K'aeloree
 AUTHOR ~K'aeloree (kae@spellholdstudios.net)~
 
-VERSION ~V4.2_Isaya~
+VERSION ~v4.2_Isaya BWP Fix~
 
 // Default to English readme if not found in the selected language
 README ~iepbanters/translations/%LANGUAGE%/iepbanters_readme.html~ ~iepbanters/translations/english/iepbanters_readme.html~
@@ -19,8 +19,18 @@
 // Isaya : Additional check to support special characters for BGEE
 // Inspired by BGQE / Edwin Romance
 ALWAYS
+  ACTION_IF GAME_IS ~eet~ BEGIN
+    OUTER_SET bg2_chapter = 12
+  END ELSE BEGIN
+    OUTER_SET bg2_chapter = 0
+  END
+  OUTER_FOR (i=1; i<=10; i=i+1) BEGIN
+    OUTER_SET bg2_chapter = bg2_chapter + 1
+    OUTER_SPRINT name_source ~bg2_chapter_%i%~
+    OUTER_SET EVAL ~%name_source%~ = bg2_chapter
+  END
 
-  // We want his to happen only once
+  // We want this to happen only once
   ACTION_IF !FILE_EXISTS ~%MOD_FOLDER%/translations/%LANGUAGE%/utf8.mrk~ BEGIN
     ACTION_IF GAME_IS ~bg2ee~ THEN BEGIN
       // Copies UTF8-encoded tra files over the normal ones, for use with AUTO_TRA
@@ -93,7 +103,7 @@
 // Ensures banters won't run in BG1 section in Baldur's Gate Trilogy
 ACTION_IF FILE_EXISTS ~DATA/BG1ARE.BIF~ THEN BEGIN
     OUTER_SPRINT "percentage_sign" "%"
-    OUTER_SPRINT ~BGT_VAR~ ~Global("ENDOFBG1","GLOBAL",2)~
+    OUTER_SPRINT ~BGT_VAR~ ~!GlobalLT("ENDOFBG1","GLOBAL",2)~
 END ELSE BEGIN
     OUTER_SPRINT "percentage_sign" "%"
     OUTER_SPRINT ~BGT_VAR~ ~~
@@ -123,7 +133,7 @@
   EVALUATE_BUFFER
   
 ACTION_IF FILE_EXISTS_IN_GAME     ~FWJASSY.DLG~ THEN BEGIN
-  EXTEND_BOTTOM ~minsc.bcs~   ~iepbanters/scripts/minsc_2.baf~
+  EXTEND_BOTTOM ~minsc.bcs~   ~iepbanters/scripts/minsc_2.baf~ EVALUATE_BUFFER
     EVALUATE_BUFFER
 END
 
@@ -142,7 +152,7 @@
 // Extended Minsc/Aerie Interaction ToB
 // --------------------------------------
 BEGIN @2
-  REQUIRE_PREDICATE GAME_IS ~bg2ee bgt tob~ @14
+  REQUIRE_PREDICATE GAME_IS ~bg2ee eet bgt tob~ @14
 
 // Add custom If Valid For Party Dialogue state
 APPEND ~state.ids~
@@ -161,7 +171,7 @@
 // Imoen "Retrospection" dialogue for BGII: ToB
 // --------------------------------------
 BEGIN @3
-  REQUIRE_PREDICATE GAME_IS ~bg2ee bgt tob~ @15
+  REQUIRE_PREDICATE GAME_IS ~bg2ee eet bgt tob~ @15
 
 APPEND ~STATE.IDS~ // adds custom IsValidForPartyDialogue state
   ~0x80101FEF CD_STATE_NOTVALID~
@@ -196,10 +206,10 @@
 ACTION_IF MOD_IS_INSTALLED ~SETUP-NPCFLIRT.TP2~ 12 // Anomen Flirt Pack for SOA
 THEN BEGIN
   COPY_EXISTING ~anomenj.dlg~ ~override~
-    DECOMPILE_DLG_TO_D
+    DECOMPILE_AND_PATCH BEGIN
       REPLACE_TEXTUALLY ~Global("FWAnomenMorningAfter","LOCALS",1)~
                         ~Global("FWAnomenMorningAfter","LOCALS",9999)~
-    COMPILE_D_TO_DLG
+    END
   BUT_ONLY_IF_IT_CHANGES
 END
 
@@ -230,7 +240,7 @@
 
 // DADROW21 interjection
 COPY_EXISTING ~DADROW21.DLG~ ~override~
-  DECOMPILE_DLG_TO_D
+  DECOMPILE_AND_PATCH BEGIN
     REPLACE_TEXTUALLY ~!Global("AerieRomanceActive","GLOBAL",1)~
                       ~~
     REPLACE_TEXTUALLY ~!Global("AerieRomanceActive","GLOBAL",2)~
@@ -243,7 +253,7 @@
                       ~~
     REPLACE_TEXTUALLY ~!Global("JaheiraRomanceActive","GLOBAL",2)~
                       ~~
-  COMPILE_D_TO_DLG
+  END
 BUT_ONLY_IF_IT_CHANGES
 
 COMPILE ~iepbanters/dialogue/restored_interjections.d~
@@ -267,10 +277,10 @@
     UNINSTALL Keto.tp2 2
     // Appending gtimes.ids to include the timer
     APPEND ~gtimes.ids~ ~600 I#BANTER_TIMER~
-    EXTEND_TOP ~baldur.bcs~ ~iepbanters/scripts/baldur.baf~
+    EXTEND_TOP ~baldur.bcs~ ~iepbanters/scripts/baldur.baf~ EVALUATE_BUFFER
     
     ACTION_IF FILE_EXISTS ~data/25dialog.bif~ THEN BEGIN
-      EXTEND_TOP ~baldur25.bcs~ ~iepbanters/scripts/baldur.baf~
+      EXTEND_TOP ~baldur25.bcs~ ~iepbanters/scripts/baldur.baf~ EVALUATE_BUFFER
     END
 
 // 15 minutes
@@ -284,10 +294,10 @@
     UNINSTALL Keto.tp2 2
     // Appending gtimes.ids to include the timer
     APPEND ~gtimes.ids~ ~900 I#BANTER_TIMER~
-    EXTEND_TOP ~baldur.bcs~ ~iepbanters/scripts/baldur.baf~
+    EXTEND_TOP ~baldur.bcs~ ~iepbanters/scripts/baldur.baf~ EVALUATE_BUFFER
 
     ACTION_IF FILE_EXISTS ~data/25dialog.bif~ THEN BEGIN
-      EXTEND_TOP ~baldur25.bcs~ ~iepbanters/scripts/baldur.baf~
+      EXTEND_TOP ~baldur25.bcs~ ~iepbanters/scripts/baldur.baf~ EVALUATE_BUFFER
     END
 
 // 20 minutes
@@ -301,10 +311,10 @@
     UNINSTALL Keto.tp2 2
     // Appending gtimes.ids 
     APPEND ~gtimes.ids~ ~1200 I#BANTER_TIMER~
-    EXTEND_TOP ~baldur.bcs~ ~iepbanters/scripts/baldur.baf~
+    EXTEND_TOP ~baldur.bcs~ ~iepbanters/scripts/baldur.baf~ EVALUATE_BUFFER
 
     ACTION_IF FILE_EXISTS ~data/25dialog.bif~ THEN BEGIN
-      EXTEND_TOP ~baldur25.bcs~ ~iepbanters/scripts/baldur.baf~
+      EXTEND_TOP ~baldur25.bcs~ ~iepbanters/scripts/baldur.baf~ EVALUATE_BUFFER
     END
 
 // 30 minutes
@@ -318,10 +328,10 @@
     UNINSTALL Keto.tp2 2
     // Appending gtimes.ids to include the timer
     APPEND ~gtimes.ids~ ~1800 I#BANTER_TIMER~
-    EXTEND_TOP ~baldur.bcs~ ~iepbanters/scripts/baldur.baf~
+    EXTEND_TOP ~baldur.bcs~ ~iepbanters/scripts/baldur.baf~ EVALUATE_BUFFER
 
     ACTION_IF FILE_EXISTS ~data/25dialog.bif~ THEN BEGIN
-      EXTEND_TOP ~baldur25.bcs~ ~iepbanters/scripts/baldur.baf~
+      EXTEND_TOP ~baldur25.bcs~ ~iepbanters/scripts/baldur.baf~ EVALUATE_BUFFER
     END
 
 // 45 minutes
@@ -335,10 +345,10 @@
     UNINSTALL Keto.tp2 2
     // Appending gtimes.ids to include the timer
     APPEND ~gtimes.ids~ ~2700 I#BANTER_TIMER~
-    EXTEND_TOP ~baldur.bcs~ ~iepbanters/scripts/baldur.baf~
+    EXTEND_TOP ~baldur.bcs~ ~iepbanters/scripts/baldur.baf~ EVALUATE_BUFFER
     
     ACTION_IF FILE_EXISTS ~data/25dialog.bif~ THEN BEGIN
-      EXTEND_TOP ~baldur25.bcs~ ~iepbanters/scripts/baldur.baf~
+      EXTEND_TOP ~baldur25.bcs~ ~iepbanters/scripts/baldur.baf~ EVALUATE_BUFFER
     END
 
 // 60 minutes
@@ -352,8 +362,8 @@
     UNINSTALL Keto.tp2 2
     // Appending gtimes.ids to include the timer
     APPEND ~gtimes.ids~ ~3600 I#BANTER_TIMER~
-    EXTEND_TOP ~baldur.bcs~ ~iepbanters/scripts/baldur.baf~
+    EXTEND_TOP ~baldur.bcs~ ~iepbanters/scripts/baldur.baf~ EVALUATE_BUFFER
     
     ACTION_IF FILE_EXISTS ~data/25dialog.bif~ THEN BEGIN
-      EXTEND_TOP ~baldur25.bcs~ ~iepbanters/scripts/baldur.baf~
+      EXTEND_TOP ~baldur25.bcs~ ~iepbanters/scripts/baldur.baf~ EVALUATE_BUFFER
     END
