--- CtB\Worldmap\areas.tbl	Sat Jun 23 14:56:46 2007
+++ C:\BWP Patchstudio\patched files\CtB\Worldmap\areas.tbl	Mon Jan 14 12:49:53 2013
@@ -10,7 +10,7 @@
 AR3540      AR3540   AR3540     4      10       3090    2100   @CTB_778005  @CTB_778105  N        //Academy of Kuldin
 AR3570      AR3570   AR3570     4      91       2276    1766   @CTB_778006  @CTB_778106  N        //Secluded Meadow
 AR3585      AR3585   AR3585     4      11       1840    1950   @CTB_778007  @CTB_778107  N        //Hunting Lodge
-AR3587      AR3587   AR3587     4      38       1322    624    @CTB_778008  @CTB_778108  N        //Druidic Circle
+AR3587      AR3587   AR3587     4      38       1422    624    @CTB_778008  @CTB_778108  N        //Druidic Circle
 AR3588      AR3588   AR3588     1      82       450     600    @CTB_778009  @CTB_778009  N        //Rauthym Island
 AR3597      AR3597   AR3597     4      9        1730    1254   @CTB_778010  @CTB_778110  N        //Neng Horse Ranch
 AR3675      AR3675   AR3675     4      92       4556    1520   @CTB_778011  @CTB_778111  N        //Brimstone Bridge
