--- NEJ2\bgtdlg\TafficJ.D	Fri Apr 08 17:45:40 2016
+++ C:\BWP Patchstudio\patched files\NEJ2\bgtdlg\TafficJ.D	Thu Jun 09 21:48:34 2016
@@ -66,36 +66,36 @@
 
 IF ~~ THEN BEGIN 9
   SAY ~I am in no mood for jests Deidre... you know my reasons for coming here! You promised me that on your return you would have the necessary knowledge for me to achieve the greatness I am destined for. Well, have you discovered the secret or not?~
-  IF ~~ THEN EXTERN ~WMART2~ 4
+  IF ~~ THEN EXTERN ~WMART2~ WMART2-4
 END
 
 IF ~~ THEN BEGIN 10
   SAY ~Don’t toy with me witch, I know your mind to be as sharp as it ever was in your youth! A simple yes or no to my question would suffice... have you the means to make Taffic into the tall hero or am I wasting my time here?~
-  IF ~~ THEN EXTERN ~WMART2~ 5
+  IF ~~ THEN EXTERN ~WMART2~ WMART2-5
 END
 
 IF ~~ THEN BEGIN 11
   SAY ~(*Angrily*) If that is true, then what was the, *I can not help you now Taffic, I have people that I must speak with first* excuse that you gave me when first we spoke? You have added yet even more delay to my achieving greatness and I’m beginning to tire of such hindrances.~
-  IF ~~ THEN EXTERN ~WMART2~ 6
+  IF ~~ THEN EXTERN ~WMART2~ WMART2-6
 END
 
 IF ~~ THEN BEGIN 12
   SAY ~(*Excitedly*) Anything Deidre! Just tell me what it is you want found and I will find it for you... won’t we <CHARNAME>?~
-  IF ~~ THEN REPLY ~*We* Taffic... how did *I* suddenly become *we*?~ EXTERN ~WMART2~ 7
-  IF ~~ THEN REPLY ~Now hold on a moment gnome... I will decide whether *we* continue with this insanity of yours or not.~ EXTERN ~WMART2~ 7
-  IF ~~ THEN REPLY ~As I expected, everything in life has a price tag attached to it. However it is for me to decide whether I wish to pay it, not you Taffic.~ EXTERN ~WMART2~ 7
-  IF ~~ THEN REPLY ~Rule No.1, Taffic, hear what the other person wants from you before agreeing to anything. That way you may get to enjoy your status as a bigger person longer.~ EXTERN ~WMART2~ 7
+  IF ~~ THEN REPLY ~*We* Taffic... how did *I* suddenly become *we*?~ EXTERN ~WMART2~ WMART2-7
+  IF ~~ THEN REPLY ~Now hold on a moment gnome... I will decide whether *we* continue with this insanity of yours or not.~ EXTERN ~WMART2~ WMART2-7
+  IF ~~ THEN REPLY ~As I expected, everything in life has a price tag attached to it. However it is for me to decide whether I wish to pay it, not you Taffic.~ EXTERN ~WMART2~ WMART2-7
+  IF ~~ THEN REPLY ~Rule No.1, Taffic, hear what the other person wants from you before agreeing to anything. That way you may get to enjoy your status as a bigger person longer.~ EXTERN ~WMART2~ WMART2-7
 END
 
 IF ~~ THEN BEGIN 13
   SAY ~Never mind <CHARNAME>’s path and destiny, it is my path, my future as a great hero that I came here to discuss! The sooner you spit out what it is you want from me, the sooner we will both be content!~
-  IF ~~ THEN REPLY ~One moment Taffic, I have more questions to ask Deidre.~ EXTERN ~WMART2~ 12
-  IF ~~ THEN REPLY ~Quiet gnome, there is more I wish to hear from the witch.~ EXTERN ~WMART2~ 12
+  IF ~~ THEN REPLY ~One moment Taffic, I have more questions to ask Deidre.~ EXTERN ~WMART2~ WMART2-12
+  IF ~~ THEN REPLY ~Quiet gnome, there is more I wish to hear from the witch.~ EXTERN ~WMART2~ WMART2-12
 END
 
 IF ~~ THEN BEGIN 14
   SAY ~At last! All this talk of the future and one’s destiny is making me impatient to begin the road to my own. Now, what is it that we are to find and where do we start looking? I presume you have some idea where this what ever it is might be?~
-  IF ~~ THEN EXTERN ~WMART2~ 14
+  IF ~~ THEN EXTERN ~WMART2~ WMART2-14
 END
 
 IF ~~ THEN BEGIN 15
@@ -137,7 +137,7 @@
 
 IF ~~ THEN BEGIN 20
   SAY ~Nooo! If the witch won’t give us the information then you must force it out of her <CHARNAME>! If you will not then I shall, there is no way I am leaving here without knowing all that she has to tell!~
-  IF ~~ THEN EXTERN ~WMART2~ 30
+  IF ~~ THEN EXTERN ~WMART2~ WMART2-30
 END
 
 IF ~Global("VP_Taffic_Grumbles","GLOBAL",2)
