--- bg2_tweaks\lib\faster_start.tpa	Sun Aug 19 22:46:22 2012
+++ C:\BWP Patchstudio\patched files\bg2_tweaks\lib\faster_start.tpa	Sat Aug 22 10:20:16 2015
@@ -1,9 +1,18 @@
+// reactivate party animations in movie02c.baf
+ACTION_IF FILE_EXISTS_IN_GAME ~ohreprty.spl~ THEN BEGIN
+  OUTER_SPRINT ohreprty ~ApplySpellRES("OHREPRTY",Player1)~
+END ELSE BEGIN
+  OUTER_SPRINT ohreprty ~~
+END
+
+COMPILE EVALUATE_BUFFER
+        ~BG2_Tweaks/baf/movie02c.baf~
+
 COMPILE ~BG2_Tweaks/baf/cut01.baf~
         ~BG2_Tweaks/baf/cut01a.baf~
         ~BG2_Tweaks/baf/cut01b.baf~
         ~BG2_Tweaks/baf/movie01b.baf~
         ~BG2_Tweaks/baf/movie02b.baf~
-        ~BG2_Tweaks/baf/movie02c.baf~
         ~BG2_Tweaks/dlg/gaelan.d~
 
 COPY_EXISTING ~ar0400.bcs~ ~override~
