--- ISNF\DIALOGUES\ISNFxRE.d	Fri May 03 21:10:20 2013
+++ C:\BWP Patchstudio\patched files\ISNF\DIALOGUES\ISNFxRE.d	Thu Feb 27 21:10:19 2014
@@ -9,12 +9,12 @@
 
 IF~~THEN BEGIN Reward001
 SAY @4 /* It's fine. I'm used to that kind of attitude. But--I suppose you are a bit surprised that someone just appeared out of the blue. */
-IF~~THEN GOTO Reward001
+IF~~THEN GOTO Reward003
 END
 
 IF~~THEN BEGIN Reward002
 SAY @5 /* Hmm, interesting. I thought you would be more surprised when someone just appears out of the blue. Anyway, I suppose I should tell you who I am. */
-IF~~THEN GOTO Reward001
+IF~~THEN GOTO Reward003
 END
 
 IF~~THEN BEGIN Reward003
