--- Reunion/Setup-Reunion.tp2	Mon Jan 23 11:25:47 2017
+++ Fixpack/Reunion/Setup-Reunion.tp2	Mon Jan 23 11:27:20 2017
@@ -1,10 +1,21 @@
 BACKUP ~Reunion/Backup~
 AUTHOR ~http://forums.pocketplane.net~
-VERSION ~v1~
+VERSION ~v2 BWP Fix~
 
 README ~Reunion/Readme-Reunion-%LANGUAGE%.txt~ ~Reunion/Readme-Reunion.txt~
 
 ALWAYS
+  ACTION_IF GAME_IS ~eet~ BEGIN
+    OUTER_SET bg2_chapter = 12
+  END ELSE BEGIN
+    OUTER_SET bg2_chapter = 0
+  END
+  OUTER_FOR (i=1; i<=10; i=i+1) BEGIN
+    OUTER_SET bg2_chapter = bg2_chapter + 1
+    OUTER_SPRINT name_source ~bg2_chapter_%i%~
+    OUTER_SET EVAL ~%name_source%~ = bg2_chapter
+  END
+
   // Borrowed from Edwin Romance: convert strings to UTF-8 for BG2EE
 
   // Setup-Reunion.tra contains in game texts as well as installation texts
@@ -14,7 +25,7 @@
 
   LAF HANDLE_CHARSETS
     INT_VAR
-      infer_charset = 1
+      infer_charsets = 1
     STR_VAR
       tra_path = EVAL ~%MOD_FOLDER%~ // Language directories and iconv directory directly in the mod folder
       // charset_table = O#charsets // Use HANDLE_CHARSETS built-in rule for charset choice
@@ -30,13 +41,13 @@
 
 BEGIN @0
 
-REQUIRE_PREDICATE FILE_EXISTS_IN_GAME ~mel01.cre~ @1
+REQUIRE_PREDICATE GAME_INCLUDES ~tob~ @1
 
 // Areas
 
 COPY ~Reunion/Areas~ ~override~
 
-ACTION_IF GAME_IS ~BG2EE~ BEGIN
+ACTION_IF GAME_IS ~bg2ee eet~ BEGIN
 ADD_JOURNAL @10000 @10001 @10002 @10003 @10004 @10005 @10006 @10007 @10008 @10009 @10010 @10011 @10012 @10013 @10014 @10015 @10016 @10017 @10018 @10019
 END
 
@@ -49,16 +60,16 @@
 COMPILE ~Reunion/Dialogue/Reunion.d~
 
 	~Reunion/Scripts/O#DAEMON.baf~ 
-        ~Reunion/Scripts/O#DAEALI.baf~ 
+	~Reunion/Scripts/O#DAEALI.baf~ 
 	~Reunion/Scripts/O#DAEFIG.baf~ 
 	~Reunion/Scripts/O#DAEGUY.baf~ 
 	~Reunion/Scripts/O#DAEMAJ.baf~ 
 	~Reunion/Scripts/O#DAEDEL.baf~ 
 
-        ~Reunion/Scripts/O#DAE0.baf~ // Outside
-        ~Reunion/Scripts/O#DAE1.baf~ // First level
-        ~Reunion/Scripts/O#DAE2.baf~ // Second level
-        ~Reunion/Scripts/O#DAE3.baf~ // Basement
+	~Reunion/Scripts/O#DAE0.baf~ // Outside
+	~Reunion/Scripts/O#DAE1.baf~ // First level
+	~Reunion/Scripts/O#DAE2.baf~ // Second level
+	~Reunion/Scripts/O#DAE3.baf~ // Basement
 
 	~Reunion/Scripts/O#DECut1.baf~ // Going to the castle
 	~Reunion/Scripts/O#DECut2.baf~ // Going home
@@ -70,7 +81,7 @@
 	~Reunion/Scripts/O#DAET2.baf~   // teleport back to Amkethran
 
 EXTEND_TOP ~Player1d.bcs~ ~Reunion/Scripts/Player1d.baf~
-EXTEND_TOP ~Nali25.bcs~ ~Reunion/Scripts/Nali25.baf~
+EXTEND_TOP ~Nali25.bcs~ ~Reunion/Scripts/Nali25.baf~ EVALUATE_BUFFER
 
 // Characters
 
@@ -336,3 +347,76 @@
 APPEND ~itemdial.2da~
 ~O#DAEGEM    14117 O#DAEGEM~
 UNLESS ~O#DAEGEM~
+
+//Worldmap
+
+MKDIR ~Worldmap~
+ACTION_IF (FILE_EXISTS ~Worldmap/map_mods_areas.tbl~) BEGIN
+  COPY ~Worldmap/map_mods_areas.tbl~ ~Worldmap~
+    APPEND_FILE ~Reunion/Worldmap/areas.tbl~
+END ELSE BEGIN
+  COPY ~Reunion/Worldmap/areas.tbl~ ~Worldmap/map_mods_areas.tbl~
+END
+
+ACTION_IF GAME_IS ~eet~ BEGIN
+  ACTION_IF (FILE_EXISTS ~Worldmap/map_mods_links.tbl~) BEGIN
+    COPY ~Worldmap/map_mods_links.tbl~ ~Worldmap~
+      APPEND_FILE ~Reunion/Worldmap/links.tbl~
+  END ELSE BEGIN
+    COPY ~Reunion/Worldmap/links.tbl~ ~Worldmap/map_mods_links.tbl~
+  END
+  COPY_EXISTING - ~worldmap.wmp~ ~.../override~
+    READ_LONG 0x30 na
+    READ_LONG 0x34 ao
+    FOR (i=0;i<na;++i) BEGIN
+      READ_ASCII ao + 0xf0*i area
+      PATCH_IF (~%area%~ STRING_EQUAL_CASE ~AR1304~) BEGIN //get data from AR1304
+        READ_LONG ao + 0xf0*i + 0x34 bam
+        READ_LONG ao + 0xf0*i + 0x38 xcor
+        READ_LONG ao + 0xf0*i + 0x3c ycor
+        READ_STRREF ao + 0xf0*i + 0x40 name
+        READ_STRREF ao + 0xf0*i + 0x44 tool
+        SET i = na
+      END
+    END
+  LAF sc#addWmpAre
+    INT_VAR
+    mapIcon = bam
+    xCoord  = xcor
+    yCoord  = ycor
+    STR_VAR
+    areName = "o#dae0"
+    strName = EVAL "%name%"
+    strDesc = EVAL "%tool%"
+  END
+  COPY - ~Reunion/Worldmap/links.tbl~ ~.../Reunion/Worldmap~
+    REPLACE_EVALUATE CASE_INSENSITIVE ~^\([A-Z0-9#_]+\)[ %TAB%]+\([A-Z]+\)[ %TAB%]+\([A-Z0-9#_]+\)[ %TAB%]+\([A-Z0-9#_]+\)[ %TAB%]+\([0-9]+\)[ %TAB%]+\([0-9]+\)[ %TAB%]+\([A-Z0-9#_]+\)[ %TAB%]+\([A-Z0-9#_]+\)[ %TAB%]+\([A-Z0-9#_]+\)[ %TAB%]+\([A-Z0-9#_]+\)[ %TAB%]+\([A-Z0-9#_]+\)[ %TAB%]+\([0-9]+\)~ BEGIN
+      PATCH_FOR_EACH var IN MATCH7 MATCH8 MATCH9 MATCH10 MATCH11 BEGIN
+        SPRINT match EVAL ~%%var%%~
+        PATCH_IF ~%match%~ STRING_EQUAL_CASE ~N~ BEGIN
+          SPRINT EVAL ~%var%~ ~~
+        END
+      END
+      INNER_ACTION BEGIN
+        COPY_EXISTING ~worldmap.wmp~ ~override~
+          LPF ADD_WORLDMAP_LINKS
+            INT_VAR
+            distance_scale = MATCH5
+            default_entry = MATCH6
+            encounter_probability = MATCH12
+            STR_VAR
+            from_area = EVAL ~%MATCH1%~
+            from_node = EVAL ~%MATCH2%~
+            to_area = EVAL ~%MATCH3%~
+            entry = EVAL ~%MATCH4%~
+            random_area1 = EVAL ~%MATCH7%~
+            random_area2 = EVAL ~%MATCH8%~
+            random_area3 = EVAL ~%MATCH9%~
+            random_area4 = EVAL ~%MATCH10%~
+            random_area5 = EVAL ~%MATCH11%~
+          END
+        BUT_ONLY
+      END
+    END ~~
+END
+
