--- Reunion/Setup-Reunion.tp2	2015-12-22 06:36:53.037521000 -0500
+++ Fixpack/Reunion/Setup-Reunion.tp2	2016-12-08 12:10:42.585867600 -0500
@@ -1,22 +1,54 @@
 BACKUP ~Reunion/Backup~
 AUTHOR ~http://forums.pocketplane.net~
-VERSION ~v1~
+VERSION ~v1 BWP Fix~
 
 README ~Reunion/Readme-Reunion.txt~ 
 
 AUTO_TRA ~Reunion/%s~
 
+ALWAYS
+  ACTION_IF GAME_IS ~eet~ BEGIN
+    OUTER_SET bg2_chapter = 12
+  END ELSE BEGIN
+    OUTER_SET bg2_chapter = 0
+  END
+  OUTER_FOR (i=1; i<=10; i=i+1) BEGIN
+    OUTER_SET bg2_chapter = bg2_chapter + 1
+    OUTER_SPRINT name_source ~bg2_chapter_%i%~
+    OUTER_SET EVAL ~%name_source%~ = bg2_chapter
+  END
+
+  // any tra files used in LANGUAGE need to be reloaded after UTF-8 conversion
+  ACTION_DEFINE_ARRAY tra#reload BEGIN Setup-Reunion END
+
+  // No exception, everything needs to be converted
+  ACTION_DEFINE_ARRAY tra#noconvert BEGIN END
+
+  ACTION_IF "%WEIDU_OS%" STRING_EQUAL_CASE ~win32~ BEGIN // HANDLE_CHARSETS doesn't work with iconv_path containing spaces
+    AT_NOW ~if not exist "iconv" move "BiG World Fixpack\_utils\iconv" "iconv"~
+  END
+  LAF HANDLE_CHARSETS
+    INT_VAR
+      infer_charsets = 1
+    STR_VAR
+      tra_path = EVAL ~%MOD_FOLDER%~
+      iconv_path = ~iconv~ // provided by BiG World Fixpack
+      reload_array = tra#reload
+      noconvert_array = tra#noconvert
+  END
+END
+
 LANGUAGE ~English~ ~English~ ~Reunion/English/Setup-Reunion.tra~
 
 BEGIN @0
 
-REQUIRE_PREDICATE FILE_EXISTS_IN_GAME ~mel01.cre~ @1
+REQUIRE_PREDICATE GAME_INCLUDES ~tob~ @1
 
 // Areas
 
 COPY ~Reunion/Areas~ ~override~
 
-ACTION_IF GAME_IS ~BG2EE~ BEGIN
+ACTION_IF GAME_IS ~bg2ee eet~ BEGIN
 ADD_JOURNAL @10000 @10001 @10002 @10003 @10004 @10005 @10006 @10007 @10008 @10009 @10010 @10011 @10012 @10013 @10014 @10015 @10016 @10017 @10018 @10019
 END
 
@@ -50,7 +82,7 @@
 	~Reunion/Scripts/O#DAET2.baf~   // teleport back to Amkethran
 
 EXTEND_TOP ~Player1d.bcs~ ~Reunion/Scripts/Player1d.baf~
-EXTEND_TOP ~Nali25.bcs~ ~Reunion/Scripts/Nali25.baf~
+EXTEND_TOP ~Nali25.bcs~ ~Reunion/Scripts/Nali25.baf~ EVALUATE_BUFFER
 
 // Characters
 
@@ -316,3 +348,75 @@
 APPEND ~itemdial.2da~
 ~O#DAEGEM    14117 O#DAEGEM~
 UNLESS ~O#DAEGEM~
+
+//Worldmap
+
+MKDIR ~Worldmap~
+ACTION_IF (FILE_EXISTS ~Worldmap/map_mods_areas.tbl~) BEGIN
+  COPY ~Worldmap/map_mods_areas.tbl~ ~Worldmap~
+    APPEND_FILE ~Reunion/Worldmap/areas.tbl~
+END ELSE BEGIN
+  COPY ~Reunion/Worldmap/areas.tbl~ ~Worldmap/map_mods_areas.tbl~
+END
+
+ACTION_IF GAME_IS ~eet~ BEGIN
+  ACTION_IF (FILE_EXISTS ~Worldmap/map_mods_links.tbl~) BEGIN
+    COPY ~Worldmap/map_mods_links.tbl~ ~Worldmap~
+      APPEND_FILE ~Reunion/Worldmap/links.tbl~
+  END ELSE BEGIN
+    COPY ~Reunion/Worldmap/links.tbl~ ~Worldmap/map_mods_links.tbl~
+  END
+  COPY_EXISTING - ~worldmap.wmp~ ~.../override~
+    READ_LONG 0x30 na
+    READ_LONG 0x34 ao
+    FOR (i=0;i<na;++i) BEGIN
+      READ_ASCII ao + 0xf0*i area
+      PATCH_IF (~%area%~ STRING_EQUAL_CASE ~AR1304~) BEGIN //get data from AR1304
+        READ_LONG ao + 0xf0*i + 0x34 bam
+        READ_LONG ao + 0xf0*i + 0x38 xcor
+        READ_LONG ao + 0xf0*i + 0x3c ycor
+        READ_STRREF ao + 0xf0*i + 0x40 name
+        READ_STRREF ao + 0xf0*i + 0x44 tool
+        SET i = na
+      END
+    END
+  LAF sc#addWmpAre
+    INT_VAR
+    mapIcon = bam
+    xCoord  = xcor
+    yCoord  = ycor
+    STR_VAR
+    areName = "o#dae0"
+    strName = EVAL "%name%"
+    strDesc = EVAL "%tool%"
+  END
+  COPY - ~Reunion/Worldmap/links.tbl~ ~.../Reunion/Worldmap~
+    REPLACE_EVALUATE CASE_INSENSITIVE ~^\([A-Z0-9#_]+\)[ %TAB%]+\([A-Z]+\)[ %TAB%]+\([A-Z0-9#_]+\)[ %TAB%]+\([A-Z0-9#_]+\)[ %TAB%]+\([0-9]+\)[ %TAB%]+\([0-9]+\)[ %TAB%]+\([A-Z0-9#_]+\)[ %TAB%]+\([A-Z0-9#_]+\)[ %TAB%]+\([A-Z0-9#_]+\)[ %TAB%]+\([A-Z0-9#_]+\)[ %TAB%]+\([A-Z0-9#_]+\)[ %TAB%]+\([0-9]+\)~ BEGIN
+      PATCH_FOR_EACH var IN MATCH7 MATCH8 MATCH9 MATCH10 MATCH11 BEGIN
+        SPRINT match EVAL ~%%var%%~
+        PATCH_IF ~%match%~ STRING_EQUAL_CASE ~N~ BEGIN
+          SPRINT EVAL ~%var%~ ~~
+        END
+      END
+      INNER_ACTION BEGIN
+        COPY_EXISTING ~worldmap.wmp~ ~override~
+          LPF ADD_WORLDMAP_LINKS
+            INT_VAR
+            distance_scale = MATCH5
+            default_entry = MATCH6
+            encounter_probability = MATCH12
+            STR_VAR
+            from_area = EVAL ~%MATCH1%~
+            from_node = EVAL ~%MATCH2%~
+            to_area = EVAL ~%MATCH3%~
+            entry = EVAL ~%MATCH4%~
+            random_area1 = EVAL ~%MATCH7%~
+            random_area2 = EVAL ~%MATCH8%~
+            random_area3 = EVAL ~%MATCH9%~
+            random_area4 = EVAL ~%MATCH10%~
+            random_area5 = EVAL ~%MATCH11%~
+          END
+        BUT_ONLY
+      END
+    END ~~
+END
