--- dq\language\english\english.tra	Wed Oct 24 08:34:36 2012
+++ C:\BWP Patchstudio\patched files\dq\language\english\english.tra	Sat Jan 12 19:44:37 2013
@@ -1,6 +1,7 @@
 //Journal entries - NOTE: Two carriage returns needed to create a journal title with the entry
 @20000 = ~Core Files: Fishing for Trouble by Yovaneth~
 @20001 = ~Optional: Major character portraits~
+@20002 = ~Optional: Add Fishing for Trouble to BP-BGT-Worldmap~
 @2 = ~Fishing for Trouble: Find Elminster
 
 I met a mage at the Crooked Crane Inn who claims to be an acquaintance of Elminster and who asked me to deliver a message to the old mage himself. I have promised nothing but have agreed to carry it in case I come across Elminster somewhere in the course of our travels.~
