--- CtB\scripts\ctb\AR3543.baf	Sun Aug 23 19:28:52 2009
+++ C:\BWP Patchstudio\patched files\CtB\scripts\ctb\AR3543.baf	Wed Jun 20 06:51:40 2012
@@ -42,7 +42,7 @@
 IF
 	NumInParty(1)
 	LevelPartyLT(20)
-	LevelPartyGT(11)
+	LevelPartyGT(9)
 THEN
 	RESPONSE #100
 		SetGlobal("CbParty1Cat2","MYAREA",1)
@@ -51,7 +51,7 @@
 IF
 	NumInParty(1)
 	LevelPartyLT(30)
-	LevelPartyGT(21)
+	LevelPartyGT(19)
 THEN
 	RESPONSE #100
 		SetGlobal("CbParty1Cat3","MYAREA",1)
@@ -59,7 +59,7 @@
 
 IF
 	NumInParty(1)
-	LevelPartyGT(31)
+	LevelPartyGT(29)
 THEN
 	RESPONSE #100
 		SetGlobal("CbParty1Cat4","MYAREA",1)
@@ -76,7 +76,7 @@
 IF
 	NumInParty(2)
 	LevelPartyLT(20)
-	LevelPartyGT(11)
+	LevelPartyGT(9)
 THEN
 	RESPONSE #100
 		SetGlobal("CbParty2Cat2","MYAREA",1)
@@ -85,7 +85,7 @@
 IF
 	NumInParty(2)
 	LevelPartyLT(30)
-	LevelPartyGT(21)
+	LevelPartyGT(19)
 THEN
 	RESPONSE #100
 		SetGlobal("CbParty2Cat3","MYAREA",1)
@@ -93,7 +93,7 @@
 
 IF
 	NumInParty(2)
-	LevelPartyGT(31)
+	LevelPartyGT(29)
 THEN
 	RESPONSE #100
 		SetGlobal("CbParty2Cat4","MYAREA",1)
@@ -110,7 +110,7 @@
 IF
 	NumInParty(3)
 	LevelPartyLT(20)
-	LevelPartyGT(11)
+	LevelPartyGT(9)
 THEN
 	RESPONSE #100
 		SetGlobal("CbParty3Cat2","MYAREA",1)
@@ -119,7 +119,7 @@
 IF
 	NumInParty(3)
 	LevelPartyLT(30)
-	LevelPartyGT(21)
+	LevelPartyGT(19)
 THEN
 	RESPONSE #100
 		SetGlobal("CbParty3Cat3","MYAREA",1)
@@ -127,7 +127,7 @@
 
 IF
 	NumInParty(3)
-	LevelPartyGT(31)
+	LevelPartyGT(29)
 THEN
 	RESPONSE #100
 		SetGlobal("CbParty3Cat4","MYAREA",1)
@@ -144,7 +144,7 @@
 IF
 	NumInParty(4)
 	LevelPartyLT(20)
-	LevelPartyGT(11)
+	LevelPartyGT(9)
 THEN
 	RESPONSE #100
 		SetGlobal("CbParty4Cat2","MYAREA",1)
@@ -153,7 +153,7 @@
 IF
 	NumInParty(4)
 	LevelPartyLT(30)
-	LevelPartyGT(21)
+	LevelPartyGT(19)
 THEN
 	RESPONSE #100
 		SetGlobal("CbParty4Cat3","MYAREA",1)
@@ -161,7 +161,7 @@
 
 IF
 	NumInParty(4)
-	LevelPartyGT(31)
+	LevelPartyGT(29)
 THEN
 	RESPONSE #100
 		SetGlobal("CbParty4Cat4","MYAREA",1)
@@ -178,7 +178,7 @@
 IF
 	NumInParty(5)
 	LevelPartyLT(20)
-	LevelPartyGT(11)
+	LevelPartyGT(9)
 THEN
 	RESPONSE #100
 		SetGlobal("CbParty5Cat2","MYAREA",1)
@@ -187,7 +187,7 @@
 IF
 	NumInParty(5)
 	LevelPartyLT(30)
-	LevelPartyGT(21)
+	LevelPartyGT(19)
 THEN
 	RESPONSE #100
 		SetGlobal("CbParty5Cat3","MYAREA",1)
@@ -195,7 +195,7 @@
 
 IF
 	NumInParty(5)
-	LevelPartyGT(31)
+	LevelPartyGT(29)
 THEN
 	RESPONSE #100
 		SetGlobal("CbParty5Cat4","MYAREA",1)
@@ -212,7 +212,7 @@
 IF
 	NumInParty(6)
 	LevelPartyLT(20)
-	LevelPartyGT(11)
+	LevelPartyGT(9)
 THEN
 	RESPONSE #100
 		SetGlobal("CbParty6Cat2","MYAREA",1)
@@ -221,7 +221,7 @@
 IF
 	NumInParty(6)
 	LevelPartyLT(30)
-	LevelPartyGT(21)
+	LevelPartyGT(19)
 THEN
 	RESPONSE #100
 		SetGlobal("CbParty6Cat3","MYAREA",1)
@@ -229,7 +229,7 @@
 
 IF
 	NumInParty(6)
-	LevelPartyGT(31)
+	LevelPartyGT(29)
 THEN
 	RESPONSE #100
 		SetGlobal("CbParty6Cat4","MYAREA",1)
