--- CtB\scripts\ctb\CBBDFENT.BAF	Wed Nov 09 12:19:36 2005
+++ C:\BWP Patchstudio\patched files\CtB\scripts\ctb\CBBDFENT.BAF	Mon Oct 17 11:34:25 2011
@@ -103,8 +103,8 @@
 		DisplayString(Myself,39968)
 		ReallyForceSpell(Myself,WIZARD_IMPROVED_HASTE)
 		ReallyForceSpell(Myself,HEAL_NO_VISUAL)
-		ReallyForceSpell(Myself,0)
-		ReallyForceSpell(Myself,0)
+		ReallyForceSpellRES("dgright",Myself)
+		ReallyForceSpellRES("dgfaith",Myself)
 		Continue()
 END
 
