--- CtB\Chores\scripts\CBIMOENQ.BAF	Thu Dec 25 05:07:38 2003
+++ C:\BWP Patchstudio\patched files\CtB\Chores\scripts\CBIMOENQ.BAF	Tue Mar 04 20:20:48 2014
@@ -2,11 +2,11 @@
     True()
 THEN
     RESPONSE #100
+        SetGlobal("CbImoenPlayed","AR3500",1)
         CutSceneId(Player1)
         Wait(1)
         ActionOverride("CBIMOENL",DisplayStringHead("CBIMOENL",@550403))
         Wait(5)
         SetCursorState(FALSE)
-        SetGlobal("CbImoenPlayed","AR3500",1)
         ActionOverride("CBIMOENL",StartDialogueNoSet(Player1))
 END
\ No newline at end of file
