--- CtB\CtbUninstall.bat	Tue Mar 07 15:40:10 2006
+++ C:\BWP Patchstudio\patched files\CtB\CtbUninstall.bat	Wed Feb 13 20:51:15 2013
@@ -23,14 +23,14 @@
 attrib -r data\CTB-WAV.bif
 del data\CTB-WAV.bif
 
-del chitin.key
-copy CTB\chitin.ctb chitin.key
-del CTB\chitin.ctb
+:: del chitin.key
+:: copy CTB\chitin.ctb chitin.key
+:: del CTB\chitin.ctb
 
-call CTB\deldir override
-md override
-xcopy /y /s /q CTB\over.bak override
-call CTB\deldir CTB\over.bak
+:: call CTB\deldir override
+:: md override
+:: xcopy /y /s /q CTB\over.bak override
+:: call CTB\deldir CTB\over.bak
 
 call CTB\deldir music\84
 call CTB\deldir music\85
@@ -55,6 +55,6 @@
 call CtB\deldir CTB-WAV
 
 @echo.
-@echo Original .key file and \override\ folder restored.
+@echo CtB Data files deleted.
 @echo.
 
