--- Kitanya\dialogues\kitya.d	Tue Jun 21 20:05:28 2005
+++ C:\BWP Patchstudio\patched files\Kitanya\dialogues\kitya.d	Sat Mar 27 08:37:07 2010
@@ -284,7 +284,7 @@
 
 IF ~~ THEN BEGIN PCKitDeath // from: PCGreed.1
   SAY @125
-  IF ~~ THEN DO ~SetGlobal("KitMadMonster","GLOBALS",1)~ EXIT
+  IF ~~ THEN DO ~SetGlobal("KitMadMonster","GLOBAL",1)~ EXIT
 END
 
 IF ~~ THEN BEGIN PCKitCharm // from:  PCGreed.2
