--- IndiNPC\Dialogue\Indi.d	Tue Dec 06 00:36:44 2005
+++ C:\BWP Patchstudio\patched files\IndiNPC\Dialogue\Indi.d	Sun Nov 28 05:32:06 2010
@@ -467,16 +467,16 @@
   IF ~Global("J#indiimoen","GLOBAL",0)
       InParty("J#Indi")
       See("J#Indi")
-      !StateCheck("j#indi",STATE_SLEEPING)~ THEN bimoen indiimoen
+      !StateCheck("j#indi",STATE_SLEEPING)~ THEN bimoen2 indiimoen
   @151
 DO ~SetGlobal("J#indiimoen","GLOBAL",1)~
   == J#indib
   @152 
-  == bimoen
+  == bimoen2
   @153
   == J#indib
   @154
-  == bimoen
+  == bimoen2
   @155
   == J#indib
   @156
