--- aurenaseph/scripts/ar0302.baf	2006-02-13 18:21:00.000000000 -0500
+++ Fixpack/aurenaseph/scripts/ar0302.baf	2016-11-20 23:15:21.412061400 -0500
@@ -2,7 +2,7 @@
         InParty("K#Auren")
         InParty("HaerDalis")
         !StateCheck("HaerDalis",STATE_SLEEPING)
-        Global("Chapter","GLOBAL",2)
+        Global("Chapter","GLOBAL",%bg2_chapter_2%)
         Global("AurenHaerDalisBanter1","AR0302",0)
 THEN
         RESPONSE #100
