--- aurenaseph/scripts/ar0404.baf	2006-02-13 18:21:00.000000000 -0500
+++ Fixpack/aurenaseph/scripts/ar0404.baf	2016-11-20 23:15:21.477766200 -0500
@@ -2,7 +2,7 @@
         InParty("K#Auren")
         InParty("Korgan")
         !StateCheck("Korgan",STATE_SLEEPING)
-        Global("Chapter","GLOBAL",2)
+        Global("Chapter","GLOBAL",%bg2_chapter_2%)
         Global("AurenKorganBanter1","AR0404",0)
 THEN
         RESPONSE #100
