--- aurenaseph\scripts\ar0404.baf	Mon Feb 13 18:21:00 2006
+++ D:\EE_Patchstudio\patched files\aurenaseph\scripts\ar0404.baf	Fri Jan 15 12:51:46 2016
@@ -2,7 +2,7 @@
         InParty("K#Auren")
         InParty("Korgan")
         !StateCheck("Korgan",STATE_SLEEPING)
-        Global("Chapter","GLOBAL",2)
+        Global("Chapter","GLOBAL",%bg2_chapter_2%)
         Global("AurenKorganBanter1","AR0404",0)
 THEN
         RESPONSE #100
