--- aurenaseph/scripts/ar0509.baf	2006-02-13 18:18:00.000000000 -0500
+++ Fixpack/aurenaseph/scripts/ar0509.baf	2016-11-20 23:15:21.599358500 -0500
@@ -2,7 +2,7 @@
         InParty("K#Auren")
         InParty("Jaheira")
         !StateCheck("Jaheira",STATE_SLEEPING)
-        Global("Chapter","GLOBAL",2)
+        Global("Chapter","GLOBAL",%bg2_chapter_2%)
         Global("AurenJaheiraBanter1","AR0509",0)
 THEN
         RESPONSE #100
