--- aurenaseph/scripts/ar0903.baf	2006-02-13 18:21:00.000000000 -0500
+++ Fixpack/aurenaseph/scripts/ar0903.baf	2016-11-20 23:15:21.794195700 -0500
@@ -2,7 +2,7 @@
         InParty("K#Auren")
         InParty("Keldorn")
         !StateCheck("Keldorn",STATE_SLEEPING)
-        Global("Chapter","GLOBAL",2)
+        Global("Chapter","GLOBAL",%bg2_chapter_2%)
         Global("AurenKeldornBanter1","AR0903",0)
 THEN
         RESPONSE #100
