--- aurenaseph/scripts/ar1000.baf	2006-02-19 15:06:34.000000000 -0500
+++ Fixpack/aurenaseph/scripts/ar1000.baf	2016-11-20 23:15:21.862268600 -0500
@@ -2,7 +2,7 @@
         InParty("K#Auren")
         InParty("Yoshimo")
         !StateCheck("Yoshimo",STATE_SLEEPING)
-        Global("Chapter","GLOBAL",2)
+        Global("Chapter","GLOBAL",%bg2_chapter_2%)
         Global("AurenYoshimoBanter1","AR1000",0)
 THEN
         RESPONSE #100
