--- aurenaseph/scripts/ar1100.baf	2006-02-13 18:18:00.000000000 -0500
+++ Fixpack/aurenaseph/scripts/ar1100.baf	2016-11-20 23:15:21.927965200 -0500
@@ -2,7 +2,7 @@
         InParty("K#Auren")
         InParty("Yoshimo")
         !StateCheck("Yoshimo",STATE_SLEEPING)
-        Global("Chapter","GLOBAL",3)
+        Global("Chapter","GLOBAL",%bg2_chapter_3%)
         Global("AurenYoshimoBanter2","AR1100",0)
 THEN
         RESPONSE #100
