--- aurenaseph\scripts\ar1202.baf	Mon Feb 13 18:20:00 2006
+++ D:\EE_Patchstudio\patched files\aurenaseph\scripts\ar1202.baf	Fri Jan 15 12:51:46 2016
@@ -2,7 +2,7 @@
         InParty("K#Auren")
         InParty("HaerDalis")
         !StateCheck("HaerDalis",STATE_SLEEPING)
-        Global("Chapter","GLOBAL",2)
+        Global("Chapter","GLOBAL",%bg2_chapter_2%)
         Global("AurenHaerDalisBanter1","AR1202",0)
 THEN
         RESPONSE #100
