--- aurenaseph\scripts\ar1600.baf	Mon Feb 13 18:20:00 2006
+++ D:\EE_Patchstudio\patched files\aurenaseph\scripts\ar1600.baf	Fri Jan 15 12:51:46 2016
@@ -2,7 +2,7 @@
         InParty("K#Auren")
         InParty("Aerie")
         !StateCheck("Aerie",STATE_SLEEPING)        
-        Global("Chapter","GLOBAL",4)
+        Global("Chapter","GLOBAL",%bg2_chapter_4%)
         Global("AurenAerieBanter2","AR1600",0)
 THEN
         RESPONSE #100
