--- aurenaseph/scripts/ar1900.baf	2006-02-13 18:20:00.000000000 -0500
+++ Fixpack/aurenaseph/scripts/ar1900.baf	2016-11-20 23:15:22.126924300 -0500
@@ -2,7 +2,7 @@
         InParty("K#Auren")
         InParty("Anomen")
         !StateCheck("Anomen",STATE_SLEEPING)
-        Global("Chapter","GLOBAL",2)
+        Global("Chapter","GLOBAL",%bg2_chapter_2%)
         Global("AurenAnomenBanter1","AR1900",0)
 THEN
         RESPONSE #100
