--- aurenaseph/scripts/ar2001.baf	2006-02-13 18:20:00.000000000 -0500
+++ Fixpack/aurenaseph/scripts/ar2001.baf	2016-11-20 23:15:22.193793900 -0500
@@ -2,7 +2,7 @@
         InParty("K#Auren")
         InParty("Jaheira")
         !StateCheck("Jaheira",STATE_SLEEPING)
-        Global("Chapter","GLOBAL",5)
+        Global("Chapter","GLOBAL",%bg2_chapter_5%)
         Global("AurenJaheiraBanter3","AR2001",0)
 THEN
         RESPONSE #100
