--- aurenaseph/scripts/ar2007.baf	2006-09-17 12:06:12.000000000 -0400
+++ Fixpack/aurenaseph/scripts/ar2007.baf	2016-11-20 23:15:22.259431000 -0500
@@ -2,7 +2,7 @@
         InParty("K#Auren")
         InParty("Jaheira")
         !StateCheck("Jaheira",STATE_SLEEPING)
-        Global("Chapter","GLOBAL",2)
+        Global("Chapter","GLOBAL",%bg2_chapter_2%)
         Global("AurenJaheiraBanter2","AR2007",0)
 THEN
         RESPONSE #100
