--- aurenaseph\scripts\ar2010.baf	Mon Feb 13 18:20:00 2006
+++ D:\EE_Patchstudio\patched files\aurenaseph\scripts\ar2010.baf	Fri Jan 15 12:51:46 2016
@@ -2,7 +2,7 @@
         InParty("K#Auren")
         InParty("Viconia")
         !StateCheck("Viconia",STATE_SLEEPING)       
-        Global("Chapter","GLOBAL",2)
+        Global("Chapter","GLOBAL",%bg2_chapter_2%)
         Global("AurenViconiaBanter1","AR2010",0)
 THEN
         RESPONSE #100
