--- aurenaseph/scripts/ar2010.baf	2006-02-13 18:20:00.000000000 -0500
+++ Fixpack/aurenaseph/scripts/ar2010.baf	2016-11-20 23:15:22.325632600 -0500
@@ -2,7 +2,7 @@
         InParty("K#Auren")
         InParty("Viconia")
         !StateCheck("Viconia",STATE_SLEEPING)       
-        Global("Chapter","GLOBAL",2)
+        Global("Chapter","GLOBAL",%bg2_chapter_2%)
         Global("AurenViconiaBanter1","AR2010",0)
 THEN
         RESPONSE #100
