--- aurenaseph/scripts/ar2200.baf	2006-02-13 18:21:00.000000000 -0500
+++ Fixpack/aurenaseph/scripts/ar2200.baf	2016-11-20 23:15:22.390830800 -0500
@@ -2,7 +2,7 @@
         InParty("K#Auren")
         InParty("Imoen")
         !StateCheck("Imoen",STATE_SLEEPING)
-        Global("Chapter","GLOBAL",5)
+        Global("Chapter","GLOBAL",%bg2_chapter_5%)
         Global("AurenImoenBanter1","AR2200",0)
 THEN
         RESPONSE #100
