--- aurenaseph/scripts/ar2812.baf	2006-02-13 18:22:00.000000000 -0500
+++ Fixpack/aurenaseph/scripts/ar2812.baf	2016-11-20 23:15:22.522449400 -0500
@@ -2,7 +2,7 @@
         InParty("K#Auren")
         InParty("Imoen")
         !StateCheck("Imoen",STATE_SLEEPING)
-        Global("Chapter","GLOBAL",5)
+        Global("Chapter","GLOBAL",%bg2_chapter_5%)
         Global("AurenImoenBanter2","AR2812",0)
 THEN
         RESPONSE #100
