--- aurenaseph\dialogues\k#aurenb.d	Fri Aug 03 12:14:34 2012
+++ D:\EE_Patchstudio\patched files\aurenaseph\dialogues\k#aurenb.d	Fri Jan 15 12:51:46 2016
@@ -7,7 +7,7 @@
 InParty(Player1) 
 !ActuallyInCombat() 
 !GlobalTimerNotExpired("FriendshipT","GLOBAL")
-!Global("Chapter","GLOBAL",5)
+!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("K#Friendship","GLOBAL", 0)~ 
 THEN K#AurenB AurenPC1
 @0 
@@ -32,7 +32,7 @@
 InParty(Player1) 
 !ActuallyInCombat() 
 !GlobalTimerNotExpired("FriendshipT","GLOBAL")
-!Global("Chapter","GLOBAL",5)
+!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("K#Friendship","GLOBAL", 1)~ 
 THEN K#AurenB AurenPC2 
 @5 
@@ -88,7 +88,7 @@
 InParty(Player1) 
 !ActuallyInCombat() 
 !GlobalTimerNotExpired("FriendshipT","GLOBAL")
-!Global("Chapter","GLOBAL",5)
+!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("K#Friendship","GLOBAL", 2)~ 
 THEN K#AurenB AurenPC3 
 @22 
@@ -143,7 +143,7 @@
 InParty(Player1) 
 !ActuallyInCombat() 
 !GlobalTimerNotExpired("FriendshipT","GLOBAL")
-!Global("Chapter","GLOBAL",5)
+!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("K#Friendship","GLOBAL", 4)~ 
 THEN K#AurenB AurenPC5 
 @41 
@@ -183,7 +183,7 @@
 InParty(Player1) 
 !ActuallyInCombat() 
 !GlobalTimerNotExpired("FriendshipT","GLOBAL")
-!Global("Chapter","GLOBAL",5)
+!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("K#Friendship","GLOBAL", 3)~ 
 THEN K#AurenB AurenPC4 
 @54 
@@ -222,7 +222,7 @@
 !ActuallyInCombat() 
 !GlobalTimerNotExpired("FriendshipT","GLOBAL")
 Global("AsylumPlot", "GLOBAL", 55)
-!Global("Chapter","GLOBAL",5)
+!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("K#Friendship","GLOBAL", 5)~ 
 THEN K#AurenB AurenPC6 
 @65 
@@ -269,7 +269,7 @@
 InParty(Player1) 
 !ActuallyInCombat() 
 !GlobalTimerNotExpired("FriendshipT","GLOBAL")
-!Global("Chapter","GLOBAL",5)
+!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("AsylumPlot", "GLOBAL", 55)
 Global("K#Friendship","GLOBAL", 6)~ 
 THEN K#AurenB AurenPC7 
@@ -301,7 +301,7 @@
 !ActuallyInCombat() 
 !GlobalTimerNotExpired("FriendshipT","GLOBAL")
 Global("AsylumPlot", "GLOBAL", 55)
-!Global("Chapter","GLOBAL",5)
+!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("K#Friendship","GLOBAL", 7)~ 
 THEN K#AurenB AurenPC8 
 @89 
@@ -693,7 +693,7 @@
 InParty("Nalia") 
 !ActuallyInCombat() 
 !GlobalTimerNotExpired("ANBanterT","GLOBAL")
-!Global("Chapter","GLOBAL",5)
+!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("K#BanterO","GLOBAL", 0)~ 
 THEN K#AurenB AN1 
 @310 
@@ -714,7 +714,7 @@
 InParty("Nalia")
 !ActuallyInCombat() 
 !GlobalTimerNotExpired("ANBanterT","GLOBAL")
-!Global("Chapter","GLOBAL",5)
+!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("NaliaKeepPlot","GLOBAL",1)
 Global("K#BanterO","GLOBAL", 1)~ 
 THEN K#AurenB AN2 
@@ -733,7 +733,7 @@
 InParty("K#Auren")
 !ActuallyInCombat() 
 !GlobalTimerNotExpired("ANBanterT","GLOBAL")
-!Global("Chapter","GLOBAL",5)
+!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("NaliaKeepPlot","GLOBAL",1)
 Global("K#BanterO","GLOBAL", 2)~ 
 THEN K#AurenB AN3 
@@ -768,7 +768,7 @@
 InParty("Nalia")
 !ActuallyInCombat() 
 !GlobalTimerNotExpired("ANBanterT","GLOBAL")
-!Global("Chapter","GLOBAL",5)
+!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("NaliaKeepPlot","GLOBAL",1)
 Global("K#BanterO","GLOBAL", 3)~ 
 THEN K#AurenB AN4
@@ -795,7 +795,7 @@
 InParty("Nalia")
 !ActuallyInCombat() 
 !GlobalTimerNotExpired("ANBanterT","GLOBAL")
-!Global("Chapter","GLOBAL",5)
+!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("NaliaKeepPlot","GLOBAL",1)
 Global("K#BanterO","GLOBAL", 4)~ 
 THEN K#AurenB AN5 
@@ -817,7 +817,7 @@
 InParty("Nalia")
 !ActuallyInCombat() 
 !GlobalTimerNotExpired("ANBanterT","GLOBAL")
-!Global("Chapter","GLOBAL",5)
+!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("NaliaKeepPlot","GLOBAL",1)
 Global("K#BanterO","GLOBAL", 5)~ 
 THEN K#AurenB AN6 
@@ -849,7 +849,7 @@
 InParty("Nalia")
 !ActuallyInCombat() 
 !GlobalTimerNotExpired("ANBanterT","GLOBAL")
-!Global("Chapter","GLOBAL",5)
+!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("NaliaKeepPlot","GLOBAL",1)
 Global("K#BanterO","GLOBAL", 6)~ 
 THEN K#AurenB AN7 
@@ -880,7 +880,7 @@
 InParty("Nalia")
 !ActuallyInCombat() 
 !GlobalTimerNotExpired("ANBanterT","GLOBAL")
-!Global("Chapter","GLOBAL",5)
+!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("NaliaKeepPlot","GLOBAL",1)
 Global("K#BanterO","GLOBAL", 7)~ 
 THEN K#AurenB AN8 
@@ -905,7 +905,7 @@
 InParty("Nalia")
 !ActuallyInCombat() 
 !GlobalTimerNotExpired("ANBanterT","GLOBAL")
-!Global("Chapter","GLOBAL",5)
+!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("NaliaKeepPlot","GLOBAL",1)
 Global("K#BanterO","GLOBAL", 8)~ 
 THEN K#AurenB AN9 
@@ -960,7 +960,7 @@
 InParty("Nalia")
 !ActuallyInCombat() 
 !GlobalTimerNotExpired("ANBanterT","GLOBAL")
-!Global("Chapter","GLOBAL",5)
+!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("NaliaKeepPlot","GLOBAL",1)
 Global("K#BanterO","GLOBAL", 9)~ 
 THEN K#AurenB AN10 
@@ -1004,7 +1004,7 @@
 InParty("Nalia")
 !ActuallyInCombat() 
 !GlobalTimerNotExpired("ANBanterT","GLOBAL")
-!Global("Chapter","GLOBAL",5)
+!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("NaliaKeepPlot","GLOBAL",1)
 Global("K#BanterO","GLOBAL", 10)~ 
 THEN K#AurenB AN11 
@@ -1044,7 +1044,7 @@
 InParty("Nalia")
 !ActuallyInCombat() 
 !GlobalTimerNotExpired("ANBanterT","GLOBAL")
-!Global("Chapter","GLOBAL",5)
+!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("NaliaKeepPlot","GLOBAL",1)
 Global("K#BanterO","GLOBAL", 11)~ 
 THEN K#AurenB AN12 
@@ -1087,7 +1087,7 @@
 InParty("Nalia")
 !ActuallyInCombat() 
 !GlobalTimerNotExpired("ANBanterT","GLOBAL")
-!Global("Chapter","GLOBAL",5)
+!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("NaliaKeepPlot","GLOBAL",1)
 Global("K#BanterO","GLOBAL", 12)~ 
 THEN K#AurenB AN13 
@@ -1131,7 +1131,7 @@
 InParty("Nalia")
 !ActuallyInCombat() 
 !GlobalTimerNotExpired("ANBanterT","GLOBAL")
-!Global("Chapter","GLOBAL",5)
+!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("NaliaKeepPlot","GLOBAL",1)
 Global("K#BanterO","GLOBAL", 13)~ 
 THEN K#AurenB AN14 
@@ -1152,7 +1152,7 @@
 InParty("Nalia")
 !ActuallyInCombat() 
 !GlobalTimerNotExpired("ANBanterT","GLOBAL")
-!Global("Chapter","GLOBAL",5)
+!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("NaliaKeepPlot","GLOBAL",1)
 Global("K#BanterO","GLOBAL", 14)~ 
 THEN K#AurenB AN15 
@@ -1173,7 +1173,7 @@
 InParty("Nalia")
 !ActuallyInCombat() 
 !GlobalTimerNotExpired("ANBanterT","GLOBAL")
-!Global("Chapter","GLOBAL",5)
+!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("NaliaKeepPlot","GLOBAL",1)
 Global("PCKeepOwner","GLOBAL",1)
 Global("K#BanterO","GLOBAL", 15)~ 
@@ -1214,7 +1214,7 @@
 InParty("Nalia")
 !ActuallyInCombat() 
 !GlobalTimerNotExpired("ANBanterT","GLOBAL")
-!Global("Chapter","GLOBAL",5)
+!Global("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("NaliaKeepPlot","GLOBAL",1)
 Global("PCKeepOwner","GLOBAL",0)
 Global("K#BanterO","GLOBAL", 15)~ 
