--- aurenaseph/dialogues/k#auren.d	2009-06-16 01:13:18.000000000 -0400
+++ Fixpack/aurenaseph/dialogues/k#auren.d	2016-11-20 23:15:22.656969800 -0500
@@ -4,7 +4,7 @@
 SAY @0 
 
 
-IF ~GlobalLT("Chapter","GLOBAL",3)~ THEN REPLY @1 GOTO InquireIrenicus
+IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_3%)~ THEN REPLY @1 GOTO InquireIrenicus
 IF ~~ THEN REPLY @2 GOTO Greet
 IF ~Class(Player1,FIGHTER)~ THEN REPLY @3 GOTO BothFighters
 IF ~~ THEN REPLY @4 GOTO GoAway
