--- AurenAseph\Scripts\K#Aure25.baf	Sat Oct 13 00:39:16 2007
+++ C:\BWP Patchstudio\patched files\AurenAseph\Scripts\K#Aure25.baf	Sat Aug 04 13:15:58 2012
@@ -219,7 +219,7 @@
 !AreaCheck("AR4500")
 THEN
 RESPONSE #100
-DisplayStringHead(Player1,~You notice a company of armed men approaching, and you realize it is Valinor and his comrades.~)
+DisplayStringHead(Player1,@1)
 SetGlobal("ValinorExist","GLOBAL",1)
 CreateCreatureObjectOffScreen("VALINOR1",Myself,0,0,0)
 CreateCreatureObjectOffScreen("GRUNT1",Myself,0,0,0)
