--- aurenaseph/scripts/k#auren.baf	2012-08-04 10:47:08.000000000 -0400
+++ Fixpack/aurenaseph/scripts/k#auren.baf	2016-11-20 23:15:22.590393800 -0500
@@ -345,9 +345,9 @@
 InParty(Myself)
 GlobalGT("K#BanterO","GLOBAL",11)
 Global("AriannaValinorExist","GLOBAL",0) 
-!Global("Chapter","GLOBAL",4) 
-!Global("Chapter","GLOBAL",5) 
-!Global("Chapter","GLOBAL",7) 
+!Global("Chapter","GLOBAL",%bg2_chapter_4%) 
+!Global("Chapter","GLOBAL",%bg2_chapter_5%) 
+!Global("Chapter","GLOBAL",%bg2_chapter_7%) 
 !AreaCheck("AR1900")
 !AreaCheck("AR1100")
 !AreaCheck("AR1400")
