--- AurenAseph\English\K#Aur25J.tra	Mon Jun 15 23:13:02 2009
+++ C:\BWP Patchstudio\patched files\AurenAseph\English\K#Aur25J.tra	Wed Dec 07 09:23:08 2011
@@ -84,7 +84,8 @@
 @83   = ~Hmm?  Oh, I'm fine.~
 @84   = ~You never really see how valuable nature is until you're immersed in it, <CHARNAME>.~
 @85   = ~(Auren glares, but it is a playful glare.)~
-@86   = ~(Auren smiles a little.)~ ~Thanks for making light of things, when at times, it seems so dark.~
+@86   = ~(Auren smiles a little.)~
+@861  = ~Thanks for making light of things, when at times, it seems so dark.~
 @87   = ~What?  No, I'm fine.  Sorry if I worried you, heh.~
 @88   = ~Auren~
 @89   = ~Sticking around is only going to get us killed.~ [K#Aure25]
