--- AurenAseph\English\K#AurenB.tra	Tue Jun 16 00:13:25 2009
+++ C:\BWP Patchstudio\patched files\AurenAseph\English\K#AurenB.tra	Tue Aug 07 01:31:44 2012
@@ -192,7 +192,8 @@
 @191  = ~You've sure had an exciting life so far, Auren.~
 @192  = ~(She winks) I hope it stays that way, Miss Aerie.~
 @193  = ~Anomen,  why do you always do that?~ [K#Aure42]
-@194  = ~Do what, my lady?~ ~You're always staring off into the distance...like you're caught up in a daydream or something...not something I would expect from a serious knight like you.~
+@194  = ~Do what, my lady?~
+@1941 = ~You're always staring off into the distance...like you're caught up in a daydream or something...not something I would expect from a serious knight like you.~
 @195  = ~To become a knight, I must give thought to many things. There are serious, important matters that occupy my mind. They are certainly not daydreams.~
 @196  = ~Ah...all right.~
 @197  = ~...~
