--- AurenAseph\Dialogues\K#AurenB.d	Tue Jun 16 00:13:25 2009
+++ C:\BWP Patchstudio\patched files\AurenAseph\Dialogues\K#AurenB.d	Fri Aug 03 15:44:34 2012
@@ -503,7 +503,7 @@
 @193 
 DO ~SetGlobal("AurenAnomenBanter1","AR1900",2)~
 == BANOMEN @194
-
+== K#AurenB @1941
 == BANOMEN @195
 == K#AurenB @196
 == K#AurenB @197
@@ -1178,7 +1178,7 @@
 Global("PCKeepOwner","GLOBAL",1)
 Global("K#BanterO","GLOBAL", 15)~ 
 THEN K#AurenB AN16 
-@545 
+@591 
 == BNALIA @294 
 == K#AurenB @592 
 == BNALIA @593
@@ -1219,7 +1219,7 @@
 Global("PCKeepOwner","GLOBAL",0)
 Global("K#BanterO","GLOBAL", 15)~ 
 THEN K#AurenB AN17 
-@545 
+@617 
 == BNALIA @294 
 == K#AurenB @592 
 == BNALIA @618
