--- AurenAseph\Setup-AurenAseph.tp2	Fri May 07 00:30:33 2010
+++ C:\BWP Patchstudio\patched files\AurenAseph\Setup-AurenAseph.tp2	Fri Sep 27 21:59:37 2013
@@ -1,5 +1,6 @@
 BACKUP ~AurenAseph/backup~
 AUTHOR ~theacefes (theacefes@gibberlings3.net)~
+VERSION ~v7.2 BWP Fix~
 README ~AurenAseph/Readme-Auren.html~
 
 AUTO_TRA ~AurenAseph\%s~
@@ -12,6 +13,10 @@
          ~German~
          ~AurenAseph\German\Setup.tra~
 
+LANGUAGE ~Russian~
+         ~russian~
+         ~AurenAseph\Russian\Setup.tra~
+
 BEGIN @0
 
 // adds ToB actions/triggers to SoA
@@ -148,7 +153,7 @@
 
 // Adds Scripts
 
-COMPILE ~AurenAseph/Scripts/K#Auren.baf~
+COMPILE ~AurenAseph/Scripts/K#Auren.baf~ USING ~AurenAseph/%s/scripts.tra~
 COMPILE ~AurenAseph/Scripts/K#Valin.baf~
 COMPILE ~AurenAseph/Scripts/K#Arian.baf~
 EXTEND_BOTTOM ~Ar0406.bcs~ ~AurenAseph/Scripts/ar0406.baf~
@@ -305,7 +310,7 @@
   Global("NaliaBio","GLOBAL",0)~
         COMPILE_BAF_TO_BCS
 
-COMPILE ~AurenAseph/Scripts/K#Aure25.baf~
+COMPILE ~AurenAseph/Scripts/K#Aure25.baf~ USING ~AurenAseph/%s/scripts.tra~
 COMPILE ~AurenAseph/Scripts/K#ValiT.baf~
 COMPILE ~AurenAseph/Scripts/K#AriaT.baf~
 COMPILE ~AurenAseph/Scripts/K#Nalia.baf~
