--- Divine_Remix\lib\macro_spellbook_ilmater.tph	Sun Jul 29 14:28:44 2012
+++ C:\BWP Patchstudio\patched files\Divine_Remix\lib\macro_spellbook_ilmater.tph	Sat Dec 14 21:57:13 2013
@@ -2,8 +2,8 @@
 DEFINE_PATCH_MACRO ~spellbook_ilmater~ BEGIN
 
   WRITE_SHORT   0x244 0
-  WRITE_BYTE    0x246 "%CDILMATR%"
-  WRITE_BYTE    0x247 0x40
+  WRITE_SHORT   0x246 0x4000 + "%CDILMATR%"
+//WRITE_BYTE    0x247 0x40
   READ_LONG 0x2A0 "sb_off"
   READ_LONG 0x2A4 "sb_num"
   READ_LONG 0x2A8 "meminfo_off"
