--- Xan/Dialogue/BO#Xan.d	2013-03-08 13:28:40.000000000 -0500
+++ Fixpack/Xan/Dialogue/BO#Xan.d	2016-11-21 19:54:16.475889400 -0500
@@ -7,8 +7,8 @@
 IF ~Global("O#XanFriendship","GLOBAL",1)
 Global("O#XanFriendTalk","GLOBAL",2)~ O#XanFT1
 SAY @0 
-+ ~GlobalLT("Chapter","GLOBAL",4)~ + @1 DO ~RealSetGlobalTimer("O#XanFriendTimer","GLOBAL",3000) SetGlobal("O#XanFriendTalk","GLOBAL",3)~ + O#XanFT1.1
-+ ~GlobalLT("Chapter","GLOBAL",4)~ + @2 DO ~RealSetGlobalTimer("O#XanFriendTimer","GLOBAL",3000) SetGlobal("O#XanFriendTalk","GLOBAL",3)~ + O#XanFT1.2
++ ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_4%)~ + @1 DO ~RealSetGlobalTimer("O#XanFriendTimer","GLOBAL",3000) SetGlobal("O#XanFriendTalk","GLOBAL",3)~ + O#XanFT1.1
++ ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_4%)~ + @2 DO ~RealSetGlobalTimer("O#XanFriendTimer","GLOBAL",3000) SetGlobal("O#XanFriendTalk","GLOBAL",3)~ + O#XanFT1.2
 ++ @3 DO ~RealSetGlobalTimer("O#XanFriendTimer","GLOBAL",3000) SetGlobal("O#XanFriendTalk","GLOBAL",3)~ + O#XanFT1.3
 ++ @4 DO ~RealSetGlobalTimer("O#XanFriendTimer","GLOBAL",3000) SetGlobal("O#XanFriendTalk","GLOBAL",3)~ + O#XanFT1.4
 END
@@ -54,7 +54,7 @@
 
 IF ~~ O#XanFT1.8
 SAY @17
-+ ~GlobalLT("Chapter","GLOBAL",4)~ + @18 + O#XanFT1.9
++ ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_4%)~ + @18 + O#XanFT1.9
 ++ @19 + O#XanFT1.10
 ++ @20 + O#XanFT1.11
 ++ @21 + O#XanFT1.12
@@ -848,8 +848,8 @@
 ++ @304 + O#XanFT7.15
 + ~!Dead("C6Bodhi")~ + @305 + O#XanFT7.17A
 + ~!Dead("C6Bodhi")~ + @306 + O#XanFT7.17
-+ ~Global("Chapter","GLOBAL",2)~ + @307 + O#XanFT7.16
-+ ~!Dead("C6Bodhi") GlobalGT("Chapter","GLOBAL",4)~ + @308 + O#XanFT7.17A
++ ~Global("Chapter","GLOBAL",%bg2_chapter_2%)~ + @307 + O#XanFT7.16
++ ~!Dead("C6Bodhi") GlobalGT("Chapter","GLOBAL",%bg2_chapter_4%)~ + @308 + O#XanFT7.17A
 + ~Dead("C6Bodhi")~ + @309 + O#XanFT7.18
 END
 
@@ -1098,7 +1098,7 @@
 SAY @398
 ++ @399 + O#XanFT9.1
 ++ @400 + O#XanFT9.2
-+ ~Global("Chapter","GLOBAL",2)~ + @401 + O#XanFT9.3
++ ~Global("Chapter","GLOBAL",%bg2_chapter_2%)~ + @401 + O#XanFT9.3
 + ~PartyGoldGT(999)~ + @402 + O#XanFT9.4
 + ~PartyGoldLT(1000)~ + @403 + O#XanFT9.4
 ++ @3532 + O#XanFT9.4
@@ -1110,7 +1110,7 @@
 ++ @406 + O#XanFT9.6
 ++ @407 + O#XanFT9.4
 ++ @408 + O#XanFT9.2
-+ ~Global("Chapter","GLOBAL",2)~ + @409 + O#XanFT9.3
++ ~Global("Chapter","GLOBAL",%bg2_chapter_2%)~ + @409 + O#XanFT9.3
 END
 
 IF ~~ O#XanFT9.2
@@ -5779,7 +5779,7 @@
 IF ~Global("O#XanRomanceActive","GLOBAL",2)
 Global("O#XanST15","GLOBAL",2)~ O#XanST15
 SAY @2069 
-+ ~GlobalGT("Chapter","GLOBAL",4)~ + @2070 DO ~SetGlobal("O#XanST15","GLOBAL",3)~ + O#XanST15.1
++ ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_4%)~ + @2070 DO ~SetGlobal("O#XanST15","GLOBAL",3)~ + O#XanST15.1
 ++ @2071 DO ~SetGlobal("O#XanST15","GLOBAL",3)~ + O#XanST15.2
 ++ @2072 DO ~SetGlobal("O#XanST15","GLOBAL",3)~ + O#XanST15.3
 ++ @2073 DO ~SetGlobal("O#XanST15","GLOBAL",3)~ + O#XanST15.4
@@ -5874,8 +5874,8 @@
 ++ @2098 DO ~RealSetGlobalTimer("O#XanRomanceTimer","GLOBAL",3000) SetGlobal("O#XanLoveTalk","GLOBAL",3)~ + O#XanLT1.1
 ++ @2099 DO ~RealSetGlobalTimer("O#XanRomanceTimer","GLOBAL",3000) SetGlobal("O#XanLoveTalk","GLOBAL",3)~ + O#XanLT1.2
 ++ @2100 DO ~RealSetGlobalTimer("O#XanRomanceTimer","GLOBAL",3000) SetGlobal("O#XanLoveTalk","GLOBAL",3)~ + O#XanLT1.3
-+ ~Global("Chapter","GLOBAL",2)~ + @2101 DO ~RealSetGlobalTimer("O#XanRomanceTimer","GLOBAL",3000) SetGlobal("O#XanLoveTalk","GLOBAL",3)~ + O#XanLT1.4
-+ ~Global("Chapter","GLOBAL",3)~ + @2101 DO ~RealSetGlobalTimer("O#XanRomanceTimer","GLOBAL",3000) SetGlobal("O#XanLoveTalk","GLOBAL",3)~ + O#XanLT1.5
++ ~Global("Chapter","GLOBAL",%bg2_chapter_2%)~ + @2101 DO ~RealSetGlobalTimer("O#XanRomanceTimer","GLOBAL",3000) SetGlobal("O#XanLoveTalk","GLOBAL",3)~ + O#XanLT1.4
++ ~Global("Chapter","GLOBAL",%bg2_chapter_3%)~ + @2101 DO ~RealSetGlobalTimer("O#XanRomanceTimer","GLOBAL",3000) SetGlobal("O#XanLoveTalk","GLOBAL",3)~ + O#XanLT1.5
 ++ @2102 DO ~RealSetGlobalTimer("O#XanRomanceTimer","GLOBAL",3000) SetGlobal("O#XanLoveTalk","GLOBAL",3)~ + O#XanLT1.6
 END
 
@@ -8589,8 +8589,8 @@
 SAY @3072
 ++ @3073 + O#XanDS5
 ++ @3074 + O#XanDS1
-+ ~GlobalLT("Chapter","GLOBAL",4)~ + @3075 + O#XanDS1
-+ ~Global("Chapter","GLOBAL",6)~ + @3076 + O#XanDS1
++ ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_4%)~ + @3075 + O#XanDS1
++ ~Global("Chapter","GLOBAL",%bg2_chapter_6%)~ + @3076 + O#XanDS1
 END
 
 IF ~~ O#XanDS3
