--- Xan/Setup-Xan.tp2	2014-11-10 14:26:12.000000000 -0500
+++ Fixpack/Xan/Setup-Xan.tp2	2016-11-27 22:10:48.529752900 -0500
@@ -2,27 +2,34 @@
 
 AUTHOR ~http://forums.pocketplane.net~
 
-VERSION ~v15~ 
+VERSION ~v15 BWP Fix~
 
 README ~Xan/%LANGUAGE%/Readme-Xan.txt~ ~Xan/Readme-Xan.txt~ 
 
 ALWAYS
-  // Borrowed from Edwin Romance: convert strings to UTF-8 for BGEE/BG2EE
-  INCLUDE ~%MOD_FOLDER%/handle_charsets.tpa~
+  ACTION_IF GAME_IS ~eet~ BEGIN
+    OUTER_SET bg2_chapter = 12
+  END ELSE BEGIN
+    OUTER_SET bg2_chapter = 0
+  END
+  OUTER_FOR (i=1; i<=10; i=i+1) BEGIN
+    OUTER_SET bg2_chapter = bg2_chapter + 1
+    OUTER_SPRINT name_source ~bg2_chapter_%i%~
+    OUTER_SET EVAL ~%name_source%~ = bg2_chapter
+  END
 
   // No exception, everything needs to be converted
   ACTION_DEFINE_ARRAY O#noconvert BEGIN END
 
-  // Only setup-Xan.tra needs to be reloaded
+  // Only Setup-Xan.tra needs to be reloaded
   ACTION_DEFINE_ARRAY O#reload BEGIN Setup-Xan END
 
   // There is no "language" folder so the root of all languages in %MOD_FOLDER%
   LAF HANDLE_CHARSETS
     INT_VAR
-      infer_charset = 1
+      infer_charsets = 1
     STR_VAR
       tra_path = EVAL ~%MOD_FOLDER%~
-      // charset_table = O#charsets // Included for illustrative purposes.
       noconvert_array = O#noconvert
       reload_array = O#reload
   END
@@ -37,7 +44,7 @@
 
 BEGIN @0
 
-ACTION_IF GAME_IS ~BG2EE~ BEGIN
+ACTION_IF GAME_IS ~bg2ee eet~ BEGIN
 LOAD_TRA ~Xan/%LANGUAGE%/Setup-Xan-EE.tra~
 ADD_JOURNAL @3612 @3613 @3614 @3615 @3616 @3617 @3618 @3619 @3620 USING ~Xan/%LANGUAGE%/O#XanJ.tra~ 
 ADD_JOURNAL @3900 @3901 @3902 @3903 @3904 @3905 @3906 @3907 @3908 USING ~Xan/%LANGUAGE%/O#XanJ.tra~ 
@@ -186,7 +193,7 @@
 SAY NAME2 @85
 SAY DESC @77
 
-COMPILE ~Xan/Dialogue/BO#Xan.d~
+COMPILE EVALUATE_BUFFER ~Xan/Dialogue/BO#Xan.d~
 	~Xan/Dialogue/O#Xan.d~
 	~Xan/Dialogue/O#XanJ.d~
 	~Xan/Dialogue/O#XanP.d~
@@ -226,12 +233,12 @@
 EXTEND_TOP ~AR1800.bcs~ ~Xan/Scripts/AR1800.baf~
 EXTEND_TOP ~AR2203.bcs~ ~Xan/Scripts/AR2203.baf~
 EXTEND_TOP ~AR2500.bcs~ ~Xan/Scripts/AR2500.baf~
-EXTEND_TOP ~Baldur.bcs~ ~Xan/Scripts/Baldur.baf~
+EXTEND_TOP ~Baldur.bcs~ ~Xan/Scripts/Baldur.baf~ EVALUATE_BUFFER
 
 ACTION_IF FILE_EXISTS_IN_GAME ~mel01.cre~ THEN
 BEGIN
 
-ACTION_IF GAME_IS ~BG2EE~ BEGIN
+ACTION_IF GAME_IS ~bg2ee eet~ BEGIN
 ADD_JOURNAL @1509 @1510 @1511 @1512 @1513 @1514 @1515 @1516 @1517 @1518 @1519 @1520 @1521 @1522 @1523 @4289 USING ~Xan/%LANGUAGE%/O#Xan25J.tra~ 
 END
 
@@ -310,7 +317,7 @@
 SAY NAME1 @68
 SAY NAME2 @68
 
-COMPILE ~Xan/Dialogue/BO#Xan25.d~
+COMPILE EVALUATE_BUFFER ~Xan/Dialogue/BO#Xan25.d~
 	~Xan/Dialogue/O#Xan25.d~
 	~Xan/Dialogue/O#Xan25J.d~
 
@@ -345,9 +352,9 @@
 EXTEND_TOP ~Baldur25.bcs~ ~Xan/Scripts/Baldur25.baf~
 EXTEND_TOP ~Cut218g.bcs~  ~Xan/Scripts/Cut218g.baf~
 EXTEND_TOP ~Player1d.bcs~ ~Xan/Scripts/Player1d.baf~
-EXTEND_TOP ~Valy25.bcs~   ~Xan/Scripts/O#Xan25V.baf~
+EXTEND_TOP ~Valy25.bcs~   ~Xan/Scripts/O#Xan25V.baf~ EVALUATE_BUFFER
 
-ACTION_IF GAME_IS ~BG2EE~ BEGIN
+ACTION_IF GAME_IS ~bg2ee eet~ BEGIN
 COMPILE	~Xan/Dialogue/XanEE.d~
 END
 
@@ -374,6 +381,26 @@
 
 LAF HANDLE_AUDIO END
 
+ACTION_IF GAME_IS ~eet~ BEGIN
+  INCLUDE ~EET/other/EET_functions.tph~
+  LAF ~EET_NPC_TRANSITION~
+    INT_VAR
+      type = 2 // BG2 NPC without BG1 content
+      clean_ToB = 1
+    STR_VAR
+      dv = "O#XAN"
+      override_SoA = "O#XANS"
+      override_ToB = "O#XAN25S"
+      dialog_ToB = "O#XAN25"
+      traFile = EVAL "Xan/%LANGUAGE%/O#Xan25J.tra"
+      string = "@34" //Bring me Xan, the elven mage.
+      stringPosDV = "Xzar" // position in Fate Spirit's dialogue before Xzar
+      clean_ToB_DV = "Xan" // remove EET's default Fate Spirit dialogue for BG1-only Xan
+  END
+END ELSE BEGIN
+  COMPILE ~Xan/Dialogue/FATESP.d~ USING ~Xan/%LANGUAGE%/O#Xan25J.tra~
+END
+
 BEGIN @48
 SUBCOMPONENT @47
 
