--- ascension\dlg\irenic2.d	Sat May 04 23:27:06 2002
+++ C:\BWP Patchstudio\patched files\ascension\dlg\irenic2.d	Wed Nov 11 13:41:53 2015
@@ -116,7 +116,7 @@
 END
 
 IF ~False()~ THEN BEGIN a12 // from: FINBODH:25.0 FINBODH:23.1
-  SAY @455 /* ~You think to betray me, Bodhi?  Foolish girl.  Melissan is the one who holds power here, and you hope to find succor at the feet of this fool <PRO_BOYGIRL>?  Stand by me or die with <PRO_HIMHER>!~ #82677 */
+  SAY @455 /* ~You think to betray me, Bodhi?  Foolish girl.  Melissan is the one who holds power here, and you hope to find succor at the feet of this fool <PRO_GIRLBOY>?  Stand by me or die with <PRO_HIMHER>!~ #82677 */
   IF ~~ THEN EXTERN ~FINBODH~ a24
 END
 
