--- DSoA\Gnome\baf\KRGNOME.baf	Sat Jul 17 22:41:28 2004
+++ C:\BWP Patchstudio\patched files\DSoA\Gnome\baf\KRGNOME.baf	Mon Oct 17 11:08:17 2011
@@ -268,7 +268,7 @@
 		DisplayString(Myself,39968) // Spell Trigger - Fired
 		ReallyForceSpell(LastSeenBy(Myself),WIZARD_SPELL_STRIKE)
 		ReallyForceSpell(LastSeenBy(Myself),WIZARD_POWER_WORD_KILL)
-		ReallyForceSpell(LastSeenBy(Myself),0)
+		ReallyForceSpell(LastSeenBy(Myself),WIZARD_ICE_STORM)
 		SetGlobal("KR_OFFENSIVE_TIME","LOCALS",0)
 		SetGlobal("SpellTrigger","LOCALS",1)
 END
