--- garrick-tt\setup-garrick-tt.tp2	Sun Mar 15 23:46:16 2015
+++ C:\BWP Patchstudio\patched files\garrick-tt\setup-garrick-tt.tp2	Sun Jul 05 17:20:25 2015
@@ -1,7 +1,7 @@
 BACKUP ~garrick-tt/backup~
 AUTHOR ~mirandir AT orange DOT fr~
 AUTO_TRA ~garrick-tt/tra/%LANGUAGE%/%s~
-VERSION ~1.23~
+VERSION ~1.23 BWP Fix~
 
 README ~garrick-tt/readme/%LANGUAGE%-readme.html~ ~garrick-tt/readme/english-readme.html~
 
@@ -218,9 +218,9 @@
 COMPILE ~garrick-tt/d/nashkel.d~ EVALUATE_BUFFER
 
 ACTION_IF GAME_IS ~bgee~ THEN BEGIN
-        COMPILE ~garrick-tt/baf/ar4801.baf~ EVALUATE_BUFFER
+        EXTEND_TOP ~ar4801.bcs~ ~garrick-tt/baf/ar4801.baf~ EVALUATE_BUFFER
 END ELSE BEGIN
-        COMPILE ~garrick-tt/baf/ar3701.baf~ EVALUATE_BUFFER
+        EXTEND_TOP ~ar3701.bcs~ ~garrick-tt/baf/ar3701.baf~ EVALUATE_BUFFER
 END
 
 COPY ~garrick-tt/sto/gtt#nas.sto~ ~override~
@@ -242,7 +242,10 @@
 // dialogues avec Tenya
 COMPILE ~garrick-tt/d/tenya.d~ EVALUATE_BUFFER // 1
 COPY_EXISTING ~tenya.bcs~ ~override/tenya.bcs~
-COMPILE ~garrick-tt/baf/tenya.baf~ EVALUATE_BUFFER
+DECOMPILE_AND_PATCH BEGIN
+	REPLACE_TEXTUALLY CASE_SENSITIVE EXACT_MATCH ~GlobalLT("TenyaHit","GLOBAL",2)~ ~GlobalLT("TenyaHit","GLOBAL",2)
+  !Kit(Player1,GODTALOS)~
+END
 COMPILE ~garrick-tt/d/tenya2.d~ EVALUATE_BUFFER // 2
 
 // BGEE change les conditions d'apparition de Tenya à la Porte, il faut donc adapter
