--- imnesvale\scripts\impossible\duAR1100.BAF	Tue Mar 14 16:49:54 2006
+++ C:\BWP Patchstudio\patched files\imnesvale\scripts\impossible\duAR1100.BAF	Sun May 16 18:49:44 2010
@@ -26,7 +26,7 @@
 //Oh yeah, only spawns IF you killed the shade lord.
 
 If
-	Global("Chapter", "GLOBAL", "6")
+	Global("Chapter", "GLOBAL", 6)
 	Global("duImnesInit", "GLOBAL", 0)
 	Global("ShadowLordDead", "GLOBAL", 1)
 Then
