--- imnesvale\dialogs\imnes.d	Tue Mar 14 12:41:28 2006
+++ C:\BWP Patchstudio\patched files\imnesvale\dialogs\imnes.d	Tue Nov 01 08:56:49 2016
@@ -63,7 +63,7 @@
 IF ~~ THEN REPLY @27 DO ~StartCutSceneMode() StartCutScene("ducut2")~ EXIT
 END
 
-IF ~NumTimesTalkedTo(1)~ THEN BEGIN whoops
+IF ~Global("duJugSpawnForKilling","GLOBAL",1)~ THEN BEGIN whoops
 SAY @28
 IF ~~ THEN REPLY @29 EXIT
 IF ~~ THEN REPLY @30 EXIT
