--- Bdash\BAF_r\rTR1008.BAF	Sun Sep 20 05:17:31 2009
+++ C:\BWP Patchstudio\patched files\Bdash\BAF_r\rTR1008.BAF	Wed Mar 31 06:32:34 2010
@@ -15,6 +15,7 @@
 		PlaySound("PORTAL2")
 		Wait(2)
 		ActionOverride(Player1,LeaveAreaLUAPanic("AR1008","",[495.755],10))
+		TakePartyItemNum("MISC45",1) // Rogue Stone
 		ActionOverride(Player1,LeaveAreaLUA("AR1008","",[495.755],10))
 		ActionOverride(Player2,LeaveAreaLUA("AR1008","",[431.730],10))
 		ActionOverride(Player3,LeaveAreaLUA("AR1008","",[459.777],10))
