--- Bdash\Language\english\setup.tra	Sun Sep 20 04:40:56 2009
+++ C:\BWP Patchstudio\patched files\Bdash\Language\english\setup.tra	Tue Sep 22 07:16:25 2009
@@ -25471,3 +25471,105 @@
 @12016 = ~As years pass, tales of your exploits shall spread far and wide, and bards will sing your name along with heroes and villains of legend.~
 @12017 = ~Your life may be mortal, your time now finite, but the mark you have left upon the Realms will endure.~
 
+@12018 = ~Dak’kon’s Zerth Blade~
+@12019 = ~Dak’kon’s Zerth Blade +2
+This sword was the weapon of choice of the famous githzerai Dak’kon.  Dak’kon was killed by a powerful entity while working for a creature known as the ’Nameless One’.  
+ 
+STATISTICS:
+
+Equipped Abilities:
++1 AC bonus
+able to memorize one extra 1st, 2nd, 3rd, and 4th level mage spell
+
+THAC0: +2 bonus
+Damage:  1D10 +2
+Damage type:  slashing
+Weight:  4
+Speed Factor: 2
+Proficiency Type: Katana
+Type:  1-handed
+Requires: 6 Strength
+Not Usable By:
+ Druid
+ Cleric~
+@12020 = ~Dak’kon’s Streaming Blade~
+@12021 = ~Dak’kon’s Streaming Blade +2
+This sword was the weapon of choice of the famous githzerai Dak’kon.  Dak’kon was killed by a powerful entity while working for a creature known as the ’Nameless One’.  
+ 
+STATISTICS:
+
+Equipped Abilities:
++2 AC bonus
+able to memorize two extra 1st, 2nd, 3rd, and 4th level mage spell
+
+THAC0: +2 bonus
+Damage:  1D10 +2
+Damage type:  slashing
+Weight:  4
+Speed Factor: 2
+Proficiency Type: Katana
+Type:  1-handed
+Requires: 6 Strength
+Not Usable By:
+ Druid
+ Cleric~
+@12022 = ~Dak’kon’s Streaming Blade +3
+This sword was the weapon of choice of the famous githzerai Dak’kon.  Dak’kon was killed by a powerful entity while working for a creature known as the ’Nameless One’.  
+ 
+STATISTICS:
+
+Equipped Abilities:
++3 AC bonus
+able to memorize three extra 1st, 2nd, 3rd, and two extra 4th level mage spell
+
+THAC0: +3 bonus
+Damage:  1D10 +3
+Damage type:  slashing
+Weight:  4
+Speed Factor: 2
+Proficiency Type: Katana
+Type:  1-handed
+Requires: 6 Strength
+Not Usable By:
+ Druid
+ Cleric~
+@12023 = ~Dak’kon’s Streaming Blade +4
+This sword was the weapon of choice of the famous githzerai Dak’kon.  Dak’kon was killed by a powerful entity while working for a creature known as the ’Nameless One’.  
+ 
+STATISTICS:
+
+Equipped Abilities:
++3 AC bonus
+able to memorize three extra 1st, 2nd, 3rd, 4th, and one extra 5th level mage spell
+
+THAC0: +4 bonus
+Damage:  1D10 +4
+Damage type:  slashing
+Weight:  4
+Speed Factor: 2
+Proficiency Type: Katana
+Type:  1-handed
+Requires: 6 Strength
+Not Usable By:
+ Druid
+ Cleric~
+@12024 = ~Dak’kon’s Streaming Blade +5
+This sword was the weapon of choice of the famous githzerai Dak’kon.  Dak’kon was killed by a powerful entity while working for a creature known as the ’Nameless One’.  After the Nameless One’s battle with the powerful entity, he restored Dak’kon to life and freed him of the oath he had to the Nameless One. This is the blade that was formed after Dak’kon was free from his oath.
+ 
+STATISTICS:
+
+Equipped Abilities:
++4 AC bonus
+able to memorize three extra 1st, 2nd, 3rd, 4th, and two extra 5th level mage spells
+
+THAC0: +5 bonus
+Damage:  1D10 +5
+Damage type:  slashing
+Weight:  4
+Speed Factor: 2
+Proficiency Type: Katana
+Type:  1-handed
+Requires: 6 Strength
+Not Usable By:
+ Druid
+ Cleric~
\ No newline at end of file
