--- finch\scripts\append_ar3300.baf	Sat Nov 05 11:14:56 2005
+++ C:\BWP Patchstudio\patched files\finch\scripts\append_ar3300.baf	Fri Nov 28 06:54:05 2008
@@ -1,35 +1,35 @@
 IF
 
 	Global("SUFinchExists", "GLOBAL", 0)
-	!Global("SUFinchExists", "FW3357", 1)
+	!Global("SUFinchExists", "AR6757", 1)
 	XPLT(Player1,1501)
 
 THEN
 	RESPONSE #100
 	SetGlobal("SUFinchExists", "GLOBAL", 1)
 	CreateCreature("sufinch1", [3000.2500],0)
-	ActionOverride("sufinch", MoveBetweenAreas("FW3357",[515.405],0))
+	ActionOverride("sufinch", MoveBetweenAreas("AR6757",[515.405],0))
 END
 
 IF
 	Global("SUFinchExists", "GLOBAL", 0)
-	!Global("SUFinchExists", "FW3357", 1)
+	!Global("SUFinchExists", "AR6757", 1)
 	XPGT(Player1,1500)
         XPLT(Player1,10001)
 THEN
 	RESPONSE #100
 	SetGlobal("SUFinchExists", "GLOBAL", 1)
 	CreateCreature("sufinch2", [3000.2500],0)
-	ActionOverride("sufinch",MoveBetweenAreas("FW3357",[515.405],0))
+	ActionOverride("sufinch",MoveBetweenAreas("AR6757",[515.405],0))
 END
 
 IF
 	Global("SUFinchExists", "GLOBAL", 0)
-        !Global("SUFinchExists", "FW3357", 1)
+        !Global("SUFinchExists", "AR6757", 1)
 	XPGT(Player1,10000)
 THEN
 	RESPONSE #100
 	SetGlobal("SUFinchExists", "GLOBAL", 1)
 	CreateCreature("sufinch4", [3000.2500],0)
-	ActionOverride("sufinch",MoveBetweenAreas("FW3357",[515.405],0))
+	ActionOverride("sufinch",MoveBetweenAreas("AR6757",[515.405],0))
 END
