--- Setup-FinchNPCv3.TP2	Sun Jul 30 21:07:38 2006
+++ C:\BWP Patchstudio\patched files\Setup-FinchNPCv3.TP2	Wed Jul 11 20:36:02 2012
@@ -2,6 +2,8 @@
 BACKUP ~finch/Backup~
 AUTHOR ~llamababe@pocketplane.net~
 
+VERSION ~BGT Conversion v1.1~
+
 LANGUAGE ~English~
          ~English~
          ~finch/English/sufinch.TRA~
@@ -168,7 +170,7 @@
   SAY NAME1 @46
   SAY NAME2 @46
 
-ACTION_IF FILE_EXISTS_IN_GAME ~_bimoen.dlg~
+ACTION_IF FILE_EXISTS_IN_GAME ~BIMOEN2.dlg~
  THEN BEGIN
 END
 ELSE BEGIN
@@ -176,16 +178,16 @@
 END
 
 APPEND ~Interdia.2da~
-~Imoen        _bimoen~
-UNLESS ~Imoen~
+~Imoen        bimoen2~
+UNLESS ~IMOEN2~
 
 //Go out on a limb and try to assign some death variables
 
-COPY_EXISTING ~_NOBL2.cre~ ~override/_NOBL2.cre~
+COPY_EXISTING ~NOBL2.cre~ ~override/NOBL2.cre~
   WRITE_ASCII 0x0280 ~NOBL2~
 BUT_ONLY_IF_IT_CHANGES
 
-COPY_EXISTING ~_NOBW2.cre~ ~override/_NOBW2.cre~
+COPY_EXISTING ~NOBW2.cre~ ~override/NOBW2.cre~
   WRITE_ASCII 0x0280 ~NOBW2~
 BUT_ONLY_IF_IT_CHANGES
 
@@ -226,7 +228,7 @@
 END
 
 //TotSC interjections
-ACTION_IF FILE_EXISTS ~override/_ANDRIS.dlg~
+ACTION_IF FILE_EXISTS ~override/ANDRIS.dlg~
        THEN BEGIN
        COMPILE ~finch/d/sufinch2.d~
           USING ~finch/english/sufinch.tra~
@@ -314,33 +316,34 @@
 
 // Shameless editing of Glasses of Identification
 
-COPY_EXISTING ~misc3p.itm~ ~override/misc3p.itm~
-  SAY NAME1 @60
-  SAY NAME2 @60
-  SAY UNIDENTIFIED_DESC @61
-  SAY DESC @61
+// Commenting because it requires an engine hack to work properly in a BGT game
+// COPY_EXISTING ~misc3p.itm~ ~override/misc3p.itm~
+//   SAY NAME1 @60
+//   SAY NAME2 @60
+//   SAY UNIDENTIFIED_DESC @61
+//   SAY DESC @61
 
 
-//COPY_EXISTING ~FW3357.are~ ~override/FW3357.are~
-//  WRITE_ASCII 0x94 ~_AR3357~
+//COPY_EXISTING ~AR6757.are~ ~override/AR6757.are~
+//  WRITE_ASCII 0x94 ~AR6757~
 //BUT_ONLY_IF_IT_CHANGES
 
-COPY_EXISTING ~FW4002.are~ ~override/FW4002.are~
-  WRITE_ASCII 0x94 ~_AR4002~
+COPY_EXISTING ~AR9902.are~ ~override/AR9902.are~
+  WRITE_ASCII 0x94 ~AR9902~
 BUT_ONLY_IF_IT_CHANGES
 
-COPY_EXISTING ~FW4804.are~ ~override/FW4804.are~
-  WRITE_ASCII 0x94 ~_AR4804~
+COPY_EXISTING ~AR3704.are~ ~override/AR3704.are~
+  WRITE_ASCII 0x94 ~AR3704~
 BUT_ONLY_IF_IT_CHANGES
 
-EXTEND_BOTTOM ~_AR4900.bcs~ ~finch/scripts/append_ar4900.baf~
-EXTEND_TOP ~_AR1803.bcs~ ~finch/scripts/append_ar1803.baf~
-EXTEND_TOP ~_AR1900.bcs~ ~finch/scripts/append_ar1900.baf~
-EXTEND_BOTTOM ~_AR3300.bcs~ ~finch/scripts/append_ar3300.baf~
-EXTEND_BOTTOM ~_AR2400.bcs~ ~finch/scripts/append_ar2400.baf~
-EXTEND_BOTTOM ~_AR2600.bcs~ ~finch/scripts/append_ar2600.baf~
-EXTEND_TOP ~_AR4002.bcs~ ~finch/scripts/append_ar4002.baf~
-EXTEND_TOP ~_AR4804.bcs~ ~finch/scripts/append_ar4804.baf~
+EXTEND_BOTTOM ~AR3800.bcs~ ~finch/scripts/append_ar4900.baf~
+EXTEND_TOP ~AR8603.bcs~ ~finch/scripts/append_ar1803.baf~
+EXTEND_TOP ~AR8700.bcs~ ~finch/scripts/append_ar1900.baf~
+EXTEND_BOTTOM ~AR6700.bcs~ ~finch/scripts/append_ar3300.baf~
+EXTEND_BOTTOM ~AR8900.bcs~ ~finch/scripts/append_ar2400.baf~
+EXTEND_BOTTOM ~AR0015.bcs~ ~finch/scripts/append_ar2600.baf~
+EXTEND_TOP ~AR9902.bcs~ ~finch/scripts/append_ar4002.baf~
+EXTEND_TOP ~AR3704.bcs~ ~finch/scripts/append_ar4804.baf~
 EXTEND_TOP ~J#INDI.bcs~ ~finch/scripts/append_j#indi.baf~
 COMPILE ~finch/scripts/sufinch.baf~
 COMPILE ~finch/scripts/sufinchd.baf~
