--- finch\d\sufinch2.d	Mon Jul 24 22:14:50 2006
+++ C:\BWP Patchstudio\patched files\finch\d\sufinch2.d	Fri Nov 28 05:46:55 2008
@@ -1,49 +1,49 @@
-I_C_T _ANDRIS 1 SuFinchIsland
+I_C_T ANDRIS 1 SuFinchIsland
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @816
 END
 
-I_C_T _ANDRIS 3 SuFinchIsland2
+I_C_T ANDRIS 3 SuFinchIsland2
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @816
 END
 
-I_C_T _APPAR 0 SuFinchDurlagsTower
+I_C_T APPAR 0 SuFinchDurlagsTower
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @817
 END
 
-I_C_T _BAYARD 11 SuFinchBayard1
+I_C_T BAYARD 11 SuFinchBayard1
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @820
 END
 
-I_C_T _BAYARD 13 SuFinchBayard2
+I_C_T BAYARD 13 SuFinchBayard2
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @820
 END
 
-I_C_T _BAYARD 14 SuFinchBayard3
+I_C_T BAYARD 14 SuFinchBayard3
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @820
 END
 
-I_C_T2 _CLAIRD 2 SuFinchClareDeLain
+I_C_T2 CLAIRD 2 SuFinchClareDeLain
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @833
 END
 
-I_C_T _FEAR 0 SuFinchFearRiddle
+I_C_T FEAR 0 SuFinchFearRiddle
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1073
 END
 
-I_C_T2 _FEAR 3 SuFinchFearRiddleSolution
+I_C_T2 FEAR 3 SuFinchFearRiddleSolution
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1074
 END
 
-I_C_T _IKE2 4 SuFinchDurlagTour
+I_C_T IKE2 4 SuFinchDurlagTour
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1106
 END
