--- finch\scripts\sufinch.baf	Sun Jul 30 21:01:54 2006
+++ C:\BWP Patchstudio\patched files\finch\scripts\sufinch.baf	Fri Aug 24 19:52:17 2012
@@ -1,6 +1,6 @@
 
 IF
-	AreaCheck("FW3357")
+	AreaCheck("AR6757")
 	!Dead("Karlat")
 	Allegiance(Myself,NEUTRAL)
 	Allegiance("Karlat",ENEMY)
@@ -13,7 +13,7 @@
 END
 
 IF
-	AreaCheck("FW3357")
+	AreaCheck("AR6757")
 	!Dead("Karlat")
 	Allegiance(Myself,NEUTRAL)
 	HPPercentLT(Player1,50)
@@ -23,7 +23,7 @@
 END
 
 IF
-	AreaCheck("FW3357")
+	AreaCheck("AR6757")
 	!Dead("Karlat")
 	Allegiance(Myself,NEUTRAL)
 	Global("SUFinchKarl", "GLOBAL", 1)
@@ -71,6 +71,7 @@
 END
 
 IF
+InParty(Myself)
 Global("SUFinchBook1Comment", "LOCALS", 0)
 PartyHasItem("SUFBOOK1")
 CombatCounter(0)
@@ -81,6 +82,7 @@
 END
 
 IF
+InParty(Myself)
 Global("SUFinchBook2Comment", "LOCALS", 0)
 PartyHasItem("SUFBOOK2")
 CombatCounter(0)
@@ -91,6 +93,7 @@
 END
 
 IF
+InParty(Myself)
 Global("SUFinchBook3Comment", "LOCALS", 0)
 PartyHasItem("SUFBOOK3")
 CombatCounter(0)
@@ -101,6 +104,7 @@
 END
 
 IF
+InParty(Myself)
 Global("SUFinchBook4Comment", "LOCALS", 0)
 PartyHasItem("SUFBOOK4")
 CombatCounter(0)
@@ -111,6 +115,7 @@
 END
 
 IF
+InParty(Myself)
 Global("SUFinchBookBComment", "LOCALS", 0)
 PartyHasItem("SUFBOOKB")
 CombatCounter(0)
@@ -121,6 +126,7 @@
 END
 
 IF
+InParty(Myself)
 Global("SUFinchBook5Comment", "LOCALS", 0)
 PartyHasItem("SUFBOOK5")
 CombatCounter(0)
@@ -147,12 +153,12 @@
 END
 
 IF
-        CombatCounter(0)
-        Global("SUFinchQuestFailed", "GLOBAL",1)
-        Global("SUFinchPissed", "GLOBAL",0)
+	InParty(Myself)
+    CombatCounter(0)
+    Global("SUFinchQuestFailed", "GLOBAL",1)
+    Global("SUFinchPissed", "GLOBAL",0)
 THEN
 	RESPONSE #100
-
-	StartDialogueNoSet(Player1)
+		StartDialogueNoSet(Player1)
 END
 	
