--- finch\d\sufinchj.D	Sun Jul 30 22:01:01 2006
+++ C:\BWP Patchstudio\patched files\finch\d\sufinchj.D	Fri Nov 28 05:47:06 2008
@@ -269,309 +269,309 @@
 IF ~~ THEN EXIT
 END
 
-I_C_T _ALATOS 0 SuFinchThievesGuild
+I_C_T ALATOS 0 SuFinchThievesGuild
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @813
 END
 
-I_C_T _ALYTH 2 SuFinchElfsong
+I_C_T ALYTH 2 SuFinchElfsong
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @814
 END
 
-I_C_T _AMNIS4 2 SuFinchOutsideNashMines
+I_C_T AMNIS4 2 SuFinchOutsideNashMines
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @815
 END
 
-I_C_T _ARDROU 1 SuFinchSIrenesCallMum
+I_C_T ARDROU 1 SuFinchSIrenesCallMum
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @818
 END
 
-I_C_T _BART6 1 SuFinchBartCandlekeep
+I_C_T BART6 1 SuFinchBartCandlekeep
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @819
 END
 
-I_C_T _BELAND 5 SuFinchHallOfWonders1
+I_C_T BELAND 5 SuFinchHallOfWonders1
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @821
 END
 
-I_C_T _BELT 11 SuFinchBall
+I_C_T BELT 11 SuFinchBall
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @822
 END
 
-I_C_T _BENTAN 1 SuFinchIlmater1
+I_C_T BENTAN 1 SuFinchIlmater1
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @823
 END
 
-I_C_T _BENTAN 2 SuFinchIlmater2
+I_C_T BENTAN 2 SuFinchIlmater2
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @823
 END
 
-I_C_T _BENTAN 3 SuFinchIlmater3
+I_C_T BENTAN 3 SuFinchIlmater3
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @823
 END
 
-I_C_T _BOYBE1 1 SuFinchEldothUglyStick
-== _ELDOTJ IF ~InParty("Eldoth") !Dead("Eldoth") !StateCheck("Eldoth",STATE_SLEEPING) InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
+I_C_T BOYBE1 1 SuFinchEldothUglyStick
+== ELDOTJ IF ~InParty("Eldoth") !Dead("Eldoth") !StateCheck("Eldoth",STATE_SLEEPING) InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @824
 END
 
-I_C_T _CADDER 0 SuFinchCadderly
+I_C_T CADDER 0 SuFinchCadderly
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @825
-== _CADDER IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN @826
+== CADDER IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN @826
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN @827
-== _CADDER IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN @828
+== CADDER IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN @828
 END
 
-I_C_T _CAEDMO 1 SuFinchFollowed
+I_C_T CAEDMO 1 SuFinchFollowed
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @829
 END
 
-I_C_T _CHANTH 2 SuFinchBookOfWisdom
+I_C_T CHANTH 2 SuFinchBookOfWisdom
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @830
 =
 @831
 END
 
-I_C_T _CHELAN 2 SuFinchHappyFisherman
+I_C_T CHELAN 2 SuFinchHappyFisherman
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @832
 END
 
-I_C_T _CHELAN 3 SuFinchHappyFisherman1
+I_C_T CHELAN 3 SuFinchHappyFisherman1
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @832
 END
 
-I_C_T _CORAN 6 SuFinchCoran1
+I_C_T CORAN 6 SuFinchCoran1
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @834
 END
 
-I_C_T _CUTCHAR 1 SuFinchExcavate
+I_C_T CUTCHAR 1 SuFinchExcavate
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @835
 END
 
-I_C_T2 _FAHRIN 1 SuFinchScrollOWisdom
+I_C_T2 FAHRIN 1 SuFinchScrollOWisdom
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @836
 END
 
-I_C_T2 _FAHRIN 2 SuFinchScrollOWisdom1
+I_C_T2 FAHRIN 2 SuFinchScrollOWisdom1
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @836
 END
 
-I_C_T _KEEPER 2 SuFinchCandlekeepGate1
+I_C_T KEEPER 2 SuFinchCandlekeepGate1
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @906
-== _KEEPER IF ~~ THEN @907
+== KEEPER IF ~~ THEN @907
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @908
 END
 
-I_C_T _KEEPER 3 SuFinchCandlekeepGate2
+I_C_T KEEPER 3 SuFinchCandlekeepGate2
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @906
-== _KEEPER IF ~~ THEN @907
+== KEEPER IF ~~ THEN @907
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @908
 END
 
-I_C_T _KEEPER 7 SuFinchCandlekeepGate3
+I_C_T KEEPER 7 SuFinchCandlekeepGate3
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @906
-== _KEEPER IF ~~ THEN @907
+== KEEPER IF ~~ THEN @907
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @908
 END
 
 // New Interjections
-I_C_T _FARM3 3 SuFinchFarmer1
+I_C_T FARM3 3 SuFinchFarmer1
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1066
 END
 
-I_C_T _FARM4 1 SuFinchFarmer2
+I_C_T FARM4 1 SuFinchFarmer2
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1067
 END
 
-I_C_T _FARMBR 6 SuFinchFarmerBrun
+I_C_T FARMBR 6 SuFinchFarmerBrun
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1068
 END
 
-I_C_T _FARTHI 0 SuFinchDollGirl
+I_C_T FARTHI 0 SuFinchDollGirl
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1069
 END
 
-I_C_T _FCOOK2 0 SuFinchShagCook1
+I_C_T FCOOK2 0 SuFinchShagCook1
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1070
 END
 
-I_C_T _FCOOK2 1 SuFinchShagCook2
+I_C_T FCOOK2 1 SuFinchShagCook2
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1071
 END
 
-I_C_T _FCOOK3 1 SuFinchCook3
+I_C_T FCOOK3 1 SuFinchCook3
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1072
 END
 
-I_C_T _FECOOK 0 SuFinchCookBrat
+I_C_T FECOOK 0 SuFinchCookBrat
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1075
-== _FECOOK IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
+== FECOOK IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1076
 END
 
-I_C_T _FELONI 1 SuFinchBully
+I_C_T FELONI 1 SuFinchBully
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1077
 END
 
-I_C_T _FENRUS 10 SuFinchDurlagStory
+I_C_T FENRUS 10 SuFinchDurlagStory
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1078
 =
 @1079
 END
 
-I_C_T _FERGUS 1 SuFinchAngelSkinRing
+I_C_T FERGUS 1 SuFinchAngelSkinRing
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1080
 END
 
-I_C_T2 _FIREBE 2 SuFinchFirebeard1
+I_C_T2 FIREBE 2 SuFinchFirebeard1
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1081
 =
 @1082
 END
 
-I_C_T2 _FIREBE 3 SuFinchFirebeard1
+I_C_T2 FIREBE 3 SuFinchFirebeard1
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1083
 =
 @1084
 END
 
-I_C_T _FIREBE 5 SuFinchFirebeard2
+I_C_T FIREBE 5 SuFinchFirebeard2
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1085
 END
 
-I_C_T _FIREBE 6 SuFinchFirebeard3
+I_C_T FIREBE 6 SuFinchFirebeard3
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1086
 END
 
-I_C_T _FLAM10 3 SuFinchFlamingBall
+I_C_T FLAM10 3 SuFinchFlamingBall
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1087
 END
 
-I_C_T2 _FORTHE 1 SuFinchRudeBard
+I_C_T2 FORTHE 1 SuFinchRudeBard
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1089
 END
 
-I_C_T2 _FORTHE 8 SuFinchRudeBard2
+I_C_T2 FORTHE 8 SuFinchRudeBard2
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1090
 END
 
-I_C_T2 _FORTHE 10 SuFinchRudeBard2
+I_C_T2 FORTHE 10 SuFinchRudeBard2
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1090
 END
 
-I_C_T _FTOWN2 1 SuFinchWarning
+I_C_T BGFTOWN2 1 SuFinchWarning
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1091
 END
 
-I_C_T _FTOWN3 0 SuFinchFarmer3
+I_C_T BGFTOWN3 0 SuFinchFarmer3
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1092
 END
 
-I_C_T _FTOWBA 0 SuFinchMassMurder
+I_C_T FTOWBA 0 SuFinchMassMurder
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1093
 =
 @1094
 END
 
-I_C_T _FTOWFR 13 SuFinchPopularity
+I_C_T FTOWFR 13 SuFinchPopularity
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1095
 END
 
-I_C_T2 _GALILE 2 SuFinchGalileo
+I_C_T2 GALILE 2 SuFinchGalileo
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1096
 END
 
-I_C_T2 _GALLOR 2 SuFinchNoBribeExcavation
+I_C_T2 GALLOR 2 SuFinchNoBribeExcavation
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1097
 END
 
-I_C_T _GALTOK 2 SuFinchNashkelMinesKobolds
+I_C_T GALTOK 2 SuFinchNashkelMinesKobolds
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1098
 =
 @1099
 END
 
-I_C_T _GAMES2 2 SuFinchBartendersAxe
+I_C_T GAMES2 2 SuFinchBartendersAxe
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1100
 END
 
-I_C_T _GAZIB 0 SuFinchOompah
+I_C_T GAZIB 0 SuFinchOompah
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1101
 END
 
-I_C_T _GNARL 4 SuFinchHead
+I_C_T GNARL 4 SuFinchHead
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1102
 END
 
-I_C_T2 _GREYWO 6 SuFinchGreywolf
+I_C_T2 GREYWO 6 SuFinchGreywolf
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1103
 END
 
-I_C_T2 _HAFIZ 6 SuFinchHafiz
+I_C_T2 HAFIZ 6 SuFinchHafiz
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1104
 END
 
-I_C_T2 _HICK 1 SuFinchAmnishSoldiers
+I_C_T2 HICK 1 SuFinchAmnishSoldiers
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1105
 END
 
-I_C_T _ITHTYL 4 SuFinchIthtyl
+I_C_T ITHTYL 4 SuFinchIthtyl
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1107
 =
@@ -580,53 +580,53 @@
 @1109
 END
 
-I_C_T _KARP 2 SuFinchKarpMines
+I_C_T KARP 2 SuFinchKarpMines
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1110
 =
 @1111
 END
 
-I_C_T2 _KERRAC 3 SuFinchHistorians
+I_C_T2 KERRAC 3 SuFinchHistorians
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1112
 END
 
-I_C_T _LAMBRID 2 SuFinchBGBridge
+I_C_T FLAMBRID 2 SuFinchBGBridge
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1113
 END
 
-I_C_T _LANDRI 9 SuFinchLandrin
-== _LANDRI IF ~Global("SuFinchLandrin", "LOCALS", 0) InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN @1114
+I_C_T LANDRI 9 SuFinchLandrin
+== LANDRI IF ~Global("SuFinchLandrin", "LOCALS", 0) InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN @1114
 == SUFINCHJ IF ~Global("SuFinchLandrin", "LOCALS", 0) InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN @1115
-== _LANDRI IF ~Global("SuFinchLandrin", "LOCALS", 0) InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN @1116
-== _LANDRI IF ~Global("SuFinchLandrin", "LOCALS", 0) InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN @1117
+== LANDRI IF ~Global("SuFinchLandrin", "LOCALS", 0) InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN @1116
+== LANDRI IF ~Global("SuFinchLandrin", "LOCALS", 0) InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN @1117
 == SUFINCHJ IF ~Global("SuFinchLandrin", "LOCALS", 0) InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN @1118
-== _LANDRI IF ~Global("SuFinchLandrin", "LOCALS", 0) InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN @1119 DO ~SetGlobal("SuFinchLandrin", "LOCALS", 1)~
+== LANDRI IF ~Global("SuFinchLandrin", "LOCALS", 0) InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN @1119 DO ~SetGlobal("SuFinchLandrin", "LOCALS", 1)~
 END
 
-I_C_T2 _LAOLA 3 SuFinchLaola
+I_C_T2 LAOLA 3 SuFinchLaola
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1120
 END
 
-I_C_T2 _LARYSS 1 SuFinchBrage
+I_C_T2 LARYSS 1 SuFinchBrage
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1121
 END
 
-I_C_T _LENA 2 SuFinchSickSamuel
+I_C_T LENA 2 SuFinchSickSamuel
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1122
 END
 
-I_C_T _MCOOK4 0 SuFinchDukesCook
+I_C_T MCOOK4 0 SuFinchDukesCook
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1123
 END
 
-I_C_T _TENYA 16 SuFinchUmberleeTalosQuest1
+I_C_T TENYA 16 SuFinchUmberleeTalosQuest1
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @1124
 =
@@ -635,22 +635,22 @@
 
 //End New Interjections
 
-I_C_T _SHARTJ 1 SuFinchSTAngelo
+I_C_T SHARTJ 1 SuFinchSTAngelo
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @837
 END
 
-I_C_T _TAZOK 0 SuFinchSeesTazok
+I_C_T BGTAZOK 0 SuFinchSeesTazok
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @838
 END
 
-I_C_T _TAZOK 33 SuFinchSeesTazok2
+I_C_T BGTAZOK 33 SuFinchSeesTazok2
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @838
 END
 
-I_C_T _ZOMBIE 1 SuFinchZombie
+I_C_T ZOMBIE 1 SuFinchZombie
 == SUFINCHJ IF ~InParty("sufinch") !Dead("sufinch") !StateCheck("sufinch",STATE_SLEEPING)~ THEN
 @839
 END
@@ -717,7 +717,7 @@
 
 END
 
-APPEND _NOBL2
+APPEND NOBL2
 
 IF WEIGHT #-1 ~InParty("sufinch")
 !StateCheck("sufinch", STATE_SLEEPING)
@@ -767,11 +767,11 @@
 
 END
 
-APPEND _NOBW2
+APPEND NOBW2
 
 IF WEIGHT #-1 ~InParty("sufinch")
 !StateCheck("sufinch", STATE_SLEEPING)
-AreaCheck("FW4804")
+AreaCheck("AR3704")
 Global("SUFinchNobWQuest2", "LOCALS",1)
 Global("SUFinchQuestFailed", "GLOBAL",0)
 !Global("SUFinchStartedLibrary", "GLOBAL",1)
@@ -819,7 +819,7 @@
 
 END
 
-APPEND _SAMANT
+APPEND SAMANT
 
 IF WEIGHT #-1 ~InParty("sufinch")
 !StateCheck("sufinch", STATE_SLEEPING)
@@ -847,15 +847,15 @@
 Gender(Player1, Male)
 Global("SUFinchQuestFailed", "GLOBAL",0)
 !Global("SUFinchStartedLibrary", "GLOBAL",1)
-Global("SUFinchNobWQuest1", "LOCALS",0)~ THEN _NOBW2 SuFinchQuestNWMan1
+Global("SUFinchNobWQuest1", "LOCALS",0)~ THEN NOBW2 SuFinchQuestNWMan1
 @869 DO ~SetGlobal("SUFinchNobWQuest1", "LOCALS",1)~
 == SUFINCHJ @870
 END
-++ @871 EXTERN _NOBW2 SuFinchNobWCharmer
-++ @872 EXTERN _NOBW2 SuFinchNobWGettingMiffy2
-++ @873 EXTERN _NOBW2 SuFinchNobWPlacated2
-++ @874 EXTERN _NOBW2 SuFinchNobWMyHome
-++ @875 EXTERN _NOBW2 SuFinchDirectToTrask3
+++ @871 EXTERN NOBW2 SuFinchNobWCharmer
+++ @872 EXTERN NOBW2 SuFinchNobWGettingMiffy2
+++ @873 EXTERN NOBW2 SuFinchNobWPlacated2
+++ @874 EXTERN NOBW2 SuFinchNobWMyHome
+++ @875 EXTERN NOBW2 SuFinchDirectToTrask3
 
 
 CHAIN
@@ -864,15 +864,15 @@
 Gender(Player1, Female)
 Global("SUFinchQuestFailed", "GLOBAL",0)
 !Global("SUFinchStartedLibrary", "GLOBAL",1)
-Global("SUFinchNobWQuest1", "LOCALS",0)~ THEN _NOBW2 SuFinchQuestNWWoman1
+Global("SUFinchNobWQuest1", "LOCALS",0)~ THEN NOBW2 SuFinchQuestNWWoman1
 @876 DO ~SetGlobal("SUFinchNobWQuest1", "LOCALS",1)~
 == SUFINCHJ @877
 END
-++ @878 EXTERN _NOBW2 SuFinchNobWPlacated2
-++ @879 EXTERN _NOBW2 SuFinchNobWGettingMiffy2
-++ @873 EXTERN _NOBW2 SuFinchNobWPlacated2
-++ @874 EXTERN _NOBW2 SuFinchNobWMyHome
-++ @875 EXTERN _NOBW2 SuFinchDirectToTrask3
+++ @878 EXTERN NOBW2 SuFinchNobWPlacated2
+++ @879 EXTERN NOBW2 SuFinchNobWGettingMiffy2
+++ @873 EXTERN NOBW2 SuFinchNobWPlacated2
+++ @874 EXTERN NOBW2 SuFinchNobWMyHome
+++ @875 EXTERN NOBW2 SuFinchDirectToTrask3
 
 
 CHAIN
@@ -881,14 +881,14 @@
 Global("SUFinchNobWQuest1", "LOCALS",1)
 Global("SUFinchQuestFailed", "GLOBAL",0)
 !Global("SUFinchStartedLibrary", "GLOBAL",1)
-Global("SUFinchNobWQuest2", "LOCALS",0)~ THEN _NOBW2 SuFinchQuestNW2
+Global("SUFinchNobWQuest2", "LOCALS",0)~ THEN NOBW2 SuFinchQuestNW2
 @880 DO ~SetGlobal("SUFinchNobWQuest2", "LOCALS",1)~
 == SUFINCHJ @881
 END
-++ @882 EXTERN _NOBW2 SuFinchNobWRich
+++ @882 EXTERN NOBW2 SuFinchNobWRich
 ++ @883 EXIT
-++ @884 EXTERN _NOBW2 SuFinchNobWNoSoupForYou2
-++ @885 EXTERN _NOBW2 SuFinchDirectToTrask3
+++ @884 EXTERN NOBW2 SuFinchNobWNoSoupForYou2
+++ @885 EXTERN NOBW2 SuFinchDirectToTrask3
 
 
 CHAIN
@@ -898,10 +898,10 @@
 Global("SUFinchQuestFailed", "GLOBAL",0)
 !Global("SUFinchStartedLibrary", "GLOBAL",1)
 Global("SUFinchMetTrask", "GLOBAL", 0)
-Global("SUFinchBerrunQuest", "LOCALS",0)~ THEN _BERRUN SuFinchQuest
+Global("SUFinchBerrunQuest", "LOCALS",0)~ THEN BERRUN SuFinchQuest
 @886 DO ~SetGlobal("SUFinchBerrunQuest", "LOCALS",1)~
 == SUFINCHJ @887
-== _BERRUN @888
+== BERRUN @888
 =
 @889 DO ~AddJournalEntry(@68, QUEST)~
 EXIT
@@ -911,17 +911,17 @@
 !StateCheck("sufinch", STATE_SLEEPING)
 Global("SUFinchQuestFailed", "GLOBAL",0)
 !Global("SUFinchStartedLibrary", "GLOBAL",1)
-Global("SUFinchNobMQuest1", "LOCALS",0)~ THEN _NOBL2 SuFinchQuestNM1
+Global("SUFinchNobMQuest1", "LOCALS",0)~ THEN NOBL2 SuFinchQuestNM1
 @890 DO ~SetGlobal("SUFinchNobMQuest1", "LOCALS",1)~
 =
 @891
 == SUFINCHJ @892
 END
-++ @893 EXTERN _NOBL2 SuFinchNobMPlacated
-++ @894 EXTERN _NOBL2 SuFinchNobMGettingMiffy
-++ @895 EXTERN _NOBL2 SuFinchNobMJealous
-++ @896 EXTERN _NOBL2 SuFinchNobMGettingMiffy
-++ @897  EXTERN _NOBL2 SuFinchDirectToTrask
+++ @893 EXTERN NOBL2 SuFinchNobMPlacated
+++ @894 EXTERN NOBL2 SuFinchNobMGettingMiffy
+++ @895 EXTERN NOBL2 SuFinchNobMJealous
+++ @896 EXTERN NOBL2 SuFinchNobMGettingMiffy
+++ @897  EXTERN NOBL2 SuFinchDirectToTrask
 
 
 CHAIN
@@ -930,19 +930,19 @@
 Global("SUFinchNobMQuest1", "LOCALS",1)
 Global("SUFinchQuestFailed", "GLOBAL",0)
 !Global("SUFinchStartedLibrary", "GLOBAL",1)
-Global("SUFinchNobMQuest2", "LOCALS",0)~ THEN _NOBL2 SuFinchQuestNM2
+Global("SUFinchNobMQuest2", "LOCALS",0)~ THEN NOBL2 SuFinchQuestNM2
 @898 DO ~SetGlobal("SUFinchNobMQuest2", "LOCALS",1)~
 == SUFINCHJ @899
 END
-++ @900 EXTERN _NOBL2 SuFinchNobMPlacated2
+++ @900 EXTERN NOBL2 SuFinchNobMPlacated2
 ++ @901 EXIT
-++ @884 EXTERN _NOBL2 SuFinchNobMNoSoupForYou1
-++ @885  EXTERN _NOBL2 SuFinchDirectToTrask2
+++ @884 EXTERN NOBL2 SuFinchNobMNoSoupForYou1
+++ @885  EXTERN NOBL2 SuFinchDirectToTrask2
 
 
 CHAIN
-IF ~~ THEN _NOBL2 SuFinchNobMGettingMiffy
+IF ~~ THEN NOBL2 SuFinchNobMGettingMiffy
 @902
 == SUFINCHJ @903
-== _NOBL2 @904
+== NOBL2 @904
 EXIT
