--- finch\scripts\sufinchd.baf	Mon Feb 21 16:36:40 2005
+++ C:\BWP Patchstudio\patched files\finch\scripts\sufinchd.baf	Fri Nov 28 05:47:20 2008
@@ -24,12 +24,12 @@
 
 IF
 InParty(Myself)
-InParty("Imoen")
-!Dead("Imoen")
-See("Imoen")
+InParty("IMOEN2")
+!Dead("IMOEN2")
+See("IMOEN2")
 Global("SUFinchImoenPrank","GLOBAL",0)
 THEN
 RESPONSE #100
 SetGlobal("SUFinchImoenPrank","GLOBAL",1)
-Interact("Imoen")
+Interact("IMOEN2")
 END
