--- DruidSor\Setup-DruidSor.tp2	Mon Jul 08 11:17:50 2013
+++ C:\BWP Patchstudio\patched files\DruidSor\Setup-DruidSor.tp2	Wed Jun 10 03:07:23 2015
@@ -7,13 +7,9 @@
 ASK_EVERY_COMPONENT
 LANGUAGE ~English~
          ~english~
-         ~languages/english~                    // English
          ~druidsor/languages/english/setup.tra~ // English
 LANGUAGE ~Francais (Translation by zooloo of the D'Oghmatiques)~
          ~french~
-         ~languages/english~                    // English
-         ~druidsor/languages/english/setup.tra~ // English
-         ~languages/french~                    // French
          ~druidsor/languages/french/setup.tra~ // French
 
 
