--- yoshimo/yoshimo.tp2	2014-04-20 06:04:47.195000000 -0400
+++ Fixpack/yoshimo/yoshimo.tp2	2016-11-21 23:05:40.095066900 -0500
@@ -21,8 +21,18 @@
 // Isaya : Additional check to support special characters for BGEE
 // Inspired by BGQE / Edwin Romance
 ALWAYS
+  ACTION_IF GAME_IS ~eet~ BEGIN
+    OUTER_SET bg2_chapter = 12
+  END ELSE BEGIN
+    OUTER_SET bg2_chapter = 0
+  END
+  OUTER_FOR (i=1; i<=10; i=i+1) BEGIN
+    OUTER_SET bg2_chapter = bg2_chapter + 1
+    OUTER_SPRINT name_source ~bg2_chapter_%i%~
+    OUTER_SET EVAL ~%name_source%~ = bg2_chapter
+  END
 
-  // We want his to happen only once
+  // We want this to happen only once
   ACTION_IF !FILE_EXISTS ~%MOD_FOLDER%/translations/%LANGUAGE%/utf8.mrk~ BEGIN
     ACTION_IF GAME_IS ~bg2ee~ THEN BEGIN
       // Copies UTF8-encoded tra files over the normal ones, for use with AUTO_TRA
@@ -73,7 +83,7 @@
   	~yoshimo/translations/%s/yoshj.tra~
 
 // Compiling scripts...
-EXTEND_BOTTOM ~yoshimo.bcs~ ~yoshimo/scripts/yoshimo.baf~
+EXTEND_BOTTOM ~yoshimo.bcs~ ~yoshimo/scripts/yoshimo.baf~ EVALUATE_BUFFER
 
 // Adding new triggers, changing timing etc for existing Yoshimo talks
 COPY_EXISTING ~byoshim.dlg~ ~override~
