--- kivan/tra/english/bp#kiv25.tra	2014-10-08 13:23:08.406956800 -0400
+++ Fixpack/kivan/tra/english/BP#KIV25.tra	2016-11-26 02:07:16.781233700 -0500
@@ -8,7 +8,7 @@
 @7    = ~If not for me, you would be preaching to Cespenar in the Nine Hells by now, assuming you would live so long. We struck a bargain, <CHARNAME> and I, and it was fortunate that <PRO_HESHE> thought it a fair one.~
 @8    = ~Once a merchant, always a merchant.~
 @9    = ~A merchant who deals in life and death is better by far than a lickspittle of a mad god.~
-@10   = ~Kivan, my pious companion, will you forsake Shevarash when the new <PRO_LORDLADY> of Murder is raised?~
+@10   = ~Kivan, my pious companion, will you forsake Shevarash when the new <PRO_LADYLORD> of Murder is raised?~
 @11   = ~I will do no such thing. Shevarash is my patron, and he led me in my hour of sorrow.~ [KIVAN87]
 @12   = ~As did <CHARNAME>.~
 @13   = ~My allegiance is to <CHARNAME> in the flesh, not to Bhaal's leavings.~
