--- kivan/d/p#kivj.d	2014-10-08 11:42:38.111858800 -0400
+++ Fixpack/kivan/d/p#kivj.d	2016-11-21 16:26:10.796057600 -0500
@@ -1361,7 +1361,7 @@
 ++ @303  EXTERN P#KIVJ KiHervo1.4
 ++ @304 EXTERN P#KIVJ KiHervo1.5
 ++ @305 EXTERN P#KIVJ KiHervo1.6
-+ ~GlobalLT("Chapter","GLOBAL",3)~+ @306 EXTERN P#KIVJ KiHervo1.7
++ ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_3%)~+ @306 EXTERN P#KIVJ KiHervo1.7
 
 
 
@@ -2017,12 +2017,12 @@
 END
 
 I_C_T MESSEN 2  P#KivanFlydian1
-== P#KIVJ  IF ~InParty("P#kivan") InMyArea("P#KIVAN") Global("chapter","GLOBAL",2)~ THEN @466
+== P#KIVJ  IF ~InParty("P#kivan") InMyArea("P#KIVAN") Global("chapter","GLOBAL",%bg2_chapter_2%)~ THEN @466
 END
 
 I_C_T MESSEN 15  P#KivanFlydian2
-== P#KIVJ  IF ~InParty("P#kivan") InMyArea("P#KIVAN") Global("chapter","GLOBAL",2)~ THEN @467
-== JAHEIRAJ  IF ~ InParty("jaheira")  InParty("P#kivan") InMyArea("P#KIVAN") Global("chapter","GLOBAL",2)~ THEN @468
+== P#KIVJ  IF ~InParty("P#kivan") InMyArea("P#KIVAN") Global("chapter","GLOBAL",%bg2_chapter_2%)~ THEN @467
+== JAHEIRAJ  IF ~ InParty("jaheira")  InParty("P#kivan") InMyArea("P#KIVAN") Global("chapter","GLOBAL",%bg2_chapter_2%)~ THEN @468
 END
 
 I_C_T2 CROTHF01 13  P#KivanAulava1
