--- Kivan\BAF\P#KIRIBF.BAF	Sat Dec 24 10:47:20 2005
+++ C:\BWP Patchstudio\patched files\Kivan\BAF\P#KIRIBF.BAF	Sat Sep 21 12:46:58 2013
@@ -60,6 +60,8 @@
 EraseJournalEntry(~The Lover's Ring
 Upon having his memory loss 'cured' by paying us less gold pieces than he intended, the merchant of antiquities in the Promenade recalled that the Jysstev family owned a ring similar to the one we found in the sewers. The Jysstevs, according to him, are old money. Perhaps, they'd pay me for bringing the news.~)
 EraseJournalEntry(~The Lover's Ring
+Upon having his memory loss 'cured' by a small bribe, the merchant of antiquities in the Promenade recalled that the Jysstev family owned a ring similar to the one we found in the sewers. The Jysstevs, according to him, are old money. Perhaps, they'd pay me for returning the ring.~)
+EraseJournalEntry(~The Lover's Ring
 Lady Jysstev was quite annoyed when I mentioned the Lover's Ring to her. Perhaps I should have talked to someone else. I will have to wait and see...~)
 END
 
