--- kivan/baf/p#kiv01.baf	2014-10-08 11:34:10.253782300 -0400
+++ Fixpack/kivan/baf/p#kiv01.baf	2016-11-21 16:26:10.794062000 -0500
@@ -387,7 +387,7 @@
   Global("P#ToBSwitch","GLOBAL",0)
   OR(2)
     AreaCheck("AR4000")
-    GlobalGT("Chapter","GLOBAL",7)
+    GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)
 THEN
   RESPONSE #100
     ChangeAIScript("P#KI25",OVERRIDE)
@@ -878,7 +878,7 @@
   GlobalLT("P#KivanOffence","GLOBAL",10)
   AreaCheck("AR0400")
   Global("P#LavokDeadTimer","GLOBAL",0)
-  GlobalLT("Chapter","GLOBAL",5)
+  GlobalLT("Chapter","GLOBAL",%bg2_chapter_5%)
   Race(Player1,HUMAN)
   OR(2)
     Dead("lavok")
@@ -954,7 +954,7 @@
     AreaCheck("AR1700")
     AreaCheck("AR1800")
     AreaCheck("AR2600")
-  GlobalGT("Chapter","GLOBAL",4)
+  GlobalGT("Chapter","GLOBAL",%bg2_chapter_4%)
   Global("P#KivanTethir","GLOBAL",0)
 THEN
   RESPONSE #100
@@ -1420,7 +1420,7 @@
   !Dead("P#kivan")
   InMyArea("P#Kivan")
   GlobalLT("P#DeherianaLives","GLOBAL",2)
-  GlobalLT("Chapter","GLOBAL",5)
+  GlobalLT("Chapter","GLOBAL",%bg2_chapter_5%)
   Global("P#KivanFriendStart","GLOBAL",1)
   Global("P#KiFriendTalk","GLOBAL",0)
   Global("P#KivanFriend","GLOBAL",0)
@@ -1480,7 +1480,7 @@
   InParty("P#kivan")
   !Dead("P#kivan")
   InMyArea("P#Kivan")
-  GlobalGT("Chapter","GLOBAL",1)
+  GlobalGT("Chapter","GLOBAL",%bg2_chapter_1%)
   Global("P#KivanCatch","GLOBAL",0)
 THEN
   RESPONSE #100
@@ -1494,8 +1494,8 @@
   InMyArea("P#Kivan")
   CombatCounter(0)
   !See([ENEMY])
-  GlobalGT("Chapter","GLOBAL",1)
-  GlobalLT("Chapter","GLOBAL",4)
+  GlobalGT("Chapter","GLOBAL",%bg2_chapter_1%)
+  GlobalLT("Chapter","GLOBAL",%bg2_chapter_4%)
   RealGlobalTimerExpired("P#KiCatchTalkTime","GLOBAL")
   !AreaType(DUNGEON)
   TimeOfDay(DAY)
@@ -1552,7 +1552,7 @@
   InParty("P#kivan")
   !Dead("P#kivan")
   InMyArea("P#Kivan")
-  GlobalGT("Chapter","GLOBAL",5)
+  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
   Global("P#KivanFriendStart","GLOBAL",1)
   OR(2)
     GlobalGT("P#KiFriendTalk","GLOBAL",9)
@@ -1608,7 +1608,7 @@
   InParty("P#kivan")
   !Dead("P#kivan")
   InMyArea("P#Kivan")
-  GlobalGT("Chapter","GLOBAL",5)
+  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
   Global("P#KivanMatch","GLOBAL",1)
   Global("P#DeherianaDead","GLOBAL",1)
   Global("P#KivanSweetTalk","GLOBAL",0)
@@ -1900,7 +1900,7 @@
   PartyHasItem("MISC5R") // The Lover's Ring
   !AreaCheck("AR0404")
   !AreaType(DUNGEON)
-  GlobalLT("Chapter","GLOBAL",4)
+  GlobalLT("Chapter","GLOBAL",%bg2_chapter_4%)
 THEN
   RESPONSE #100
     IncrementGlobal("P#KivanLoversQuest","GLOBAL",1)
