--- kivan/d/p#kipid.d	2013-11-10 18:22:38.486718300 -0500
+++ Fixpack/kivan/d/p#kipid.d	2016-11-21 16:26:10.795043200 -0500
@@ -32,7 +32,7 @@
 CombatCounter(0)~ P#KiFlirt
 SAY @4
 
-+ ~GlobalLT("Chapter","GLOBAL",4) GlobalLT("P#KivanLove","GLOBAL",4)~ + @568 + P#KivanInitiateRomance
++ ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_4%) GlobalLT("P#KivanLove","GLOBAL",4)~ + @568 + P#KivanInitiateRomance
 
 + ~OR(2) !Global("P#DeherianaDead","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")~ + @5 EXIT
 
@@ -99,43 +99,43 @@
 + ~OR(2) !Global("P#DeherianaDead","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL") !InParty("viconia") Gender(Player1,FEMALE) !Race(Player1,ELF) RandomNum(9,8)~ + @11 + X#KiMaleEl9
 + ~OR(2) !Global("P#DeherianaDead","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL") !InParty("viconia") Gender(Player1,FEMALE) !Race(Player1,ELF) RandomNum(9,9)~ + @11 + X#KiMaleEl10
 
-+ ~Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)
  RandomNum(8,1)~ + @12 + P#KIWho1
-+ ~Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)
  RandomNum(8,2)~ + @12 + P#KIWho2
-+ ~Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)
  RandomNum(8,3)~ + @12 + P#KIWho3
-+ ~Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)
  RandomNum(8,4)~ + @12 + P#KIWho4
-+ ~Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)
  RandomNum(8,5)~ + @12 + P#KIWho5
-+ ~Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)
  RandomNum(8,6)~ + @12 + P#KIWho6
-+ ~Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)
  RandomNum(8,7)~ + @12 + P#KIWho7
-+ ~Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)
@@ -152,233 +152,233 @@
 
 + ~Global("P#KivanRomanceActive","Global",1) PartyHasItem("P#Dream") Global("P#DreamCatcher","GLOBAL",1)~ + @14 + KiFlirt5
 
-+ ~RandomNum(8,1) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,1) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @15 + P#KIHood1
-+ ~RandomNum(8,2) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,2) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @15 + P#KIHood2
-+ ~RandomNum(8,3) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,3) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @15 + P#KIHood3
-+ ~RandomNum(8,4) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,4) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @15 + P#KIHood4
-+ ~RandomNum(8,5) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,5) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @15 + P#KIHood5
-+ ~RandomNum(8,6) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,6) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @15 + P#KIHood6
-+ ~RandomNum(8,7) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,7) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @15 + P#KIHood7
-+ ~RandomNum(8,8) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,8) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @15 + P#KIHood8
 
-+ ~RandomNum(8,1) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,1) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @16 + P#KIWalk1
-+ ~RandomNum(8,2) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,2) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @16 + P#KIWalk2
-+ ~RandomNum(8,3) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,3) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @16 + P#KIWalk3
-+ ~RandomNum(8,4) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,4) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @16 + P#KIWalk4
-+ ~RandomNum(8,5) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,5) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @16 + P#KIWalk5
-+ ~RandomNum(8,6) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,6) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @16 + P#KIWalk6
-+ ~RandomNum(8,7) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,7) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @16 + P#KIWalk7
-+ ~RandomNum(8,8) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,8) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @16 + P#KIWalk8
 
-+ ~RandomNum(8,1) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,1) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @17 + P#KIHand1
-+ ~RandomNum(8,2) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,2) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @17 + P#KIHand2
-+ ~RandomNum(8,3) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,3) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @17 + P#KIHand3
-+ ~RandomNum(8,4) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,4) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @17 + P#KIHand4
-+ ~RandomNum(8,5) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,5) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @17 + P#KIHand5
-+ ~RandomNum(8,6) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,6) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @17 + P#KIHand6
-+ ~RandomNum(8,7) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,7) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @17 + P#KIHand7
-+ ~RandomNum(8,8) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,8) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @17 + P#KIHand8
 
-+ ~RandomNum(8,1) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,1) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @18 + P#KIClap1
-+ ~RandomNum(8,2) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,2) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @18 + P#KIClap2
-+ ~RandomNum(8,3) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,3) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @18 + P#KIClap3
-+ ~RandomNum(8,4) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,4) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @18 + P#KIClap4
-+ ~RandomNum(8,5) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,5) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @18 + P#KIClap5
-+ ~RandomNum(8,6) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,6) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @18 + P#KIClap6
-+ ~RandomNum(8,7) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,7) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @18 + P#KIClap7
-+ ~RandomNum(8,8) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,8) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @18 + P#KIClap8
 
-+ ~RandomNum(8,1) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,1) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @19 + P#KIShoul1
-+ ~RandomNum(8,2) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,2) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @19 + P#KIShoul2
-+ ~RandomNum(8,3) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,3) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @19 + P#KIShoul3
-+ ~RandomNum(8,4) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,4) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @19 + P#KIShoul4
-+ ~RandomNum(8,5) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,5) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @19 + P#KIShoul5
-+ ~RandomNum(8,6) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,6) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @19 + P#KIShoul6
-+ ~RandomNum(8,7) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,7) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @19 + P#KIShoul7
-+ ~RandomNum(8,8) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,8) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @19 + P#KIShoul8
 
-+ ~RandomNum(8,1) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,1) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @20 + P#KIKiss1
-+ ~RandomNum(8,2) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,2) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @20 + P#KIKiss2
-+ ~RandomNum(8,3) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,3) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @20 + P#KIKiss3
-+ ~RandomNum(8,4) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,4) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @20 + P#KIKiss4
-+ ~RandomNum(8,5) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,5) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @20 + P#KIKiss5
-+ ~RandomNum(8,6) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,6) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @20 + P#KIKiss6
-+ ~RandomNum(8,7) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,7) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @20 + P#KIKiss7
-+ ~RandomNum(8,8) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,8) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @20 + P#KIKiss8
 
-+ ~RandomNum(8,1) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,1) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @21 + P#KIHair1
-+ ~RandomNum(8,2) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,2) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @21 + P#KIHair2
-+ ~RandomNum(8,3) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,3) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @21 + P#KIHair3
-+ ~RandomNum(8,4) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,4) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @21 + P#KIHair6
-+ ~RandomNum(8,5) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,5) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @21 + P#KIHair4
-+ ~RandomNum(8,6) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,6) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @21 + P#KIHair5
-+ ~RandomNum(8,7) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,7) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @21 + P#KIHair7
-+ ~RandomNum(8,8) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",5)
++ ~RandomNum(8,8) Global("P#KivanMatch","GLOBAL",1)  GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)
 Global("P#DisableKivanFlirts","GLOBAL",0) !Global("P#KivanRomanceInactive","GLOBAL",1)
 Global("P#KivanRomanceActive","GLOBAL",1) RealGlobalTimerExpired("P#KivanFlirtTimer","GLOBAL")
 GlobalLT("P#KivanOffence","GLOBAL",10)~ + @21 + P#KIHair8
@@ -1701,7 +1701,7 @@
 ++ @406 + P#Grave1
 ++ @407  + P#Grave2
 ++ @408  + P#Grave3
-+ ~GlobalGT("Chapter","GLOBAL",4)~ + @409  + P#Grave4
++ ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_4%)~ + @409  + P#Grave4
 ++ @410  + P#Grave5
 END
 
