--- kivan/d/p#kiitm.d	2006-02-15 21:58:06.218750000 -0500
+++ Fixpack/kivan/d/p#kiitm.d	2016-11-21 10:20:32.539220600 -0500
@@ -96,9 +96,9 @@
 ++ @20  + NeirosGreets2.1.1A
 ++ @21  + NeirosGreets2.1.1B
 ++ @22  + NeirosGreets2.1.1C
-+ ~GlobalLT("Chapter","GLOBAL",4)~ + @23 + NeirosGreets2.1.1D
-+ ~GlobalLT("Chapter","GLOBAL",4)~ + @24 + NeirosGreets2.1.1E
-+ ~GlobalLT("Chapter","GLOBAL",4)~ + @25 + NeirosGreets2.1.1F
++ ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_4%)~ + @23 + NeirosGreets2.1.1D
++ ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_4%)~ + @24 + NeirosGreets2.1.1E
++ ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_4%)~ + @25 + NeirosGreets2.1.1F
 END
 
 IF ~~ NeirosGreets2.1.1F
@@ -202,9 +202,9 @@
 + ~Gender(Player1,FEMALE)~ + @21  + NeirosGreets1.1.1B
 + ~Gender(Player1,FEMALE)~ + @22  + NeirosGreets1.1.1C
 + ~Gender(Player1,MALE)~ + @22  + NeirosGreets1.1.1M
-+ ~GlobalLT("Chapter","GLOBAL",4)~ + @23 + NeirosGreets1.1.1D
-+ ~GlobalLT("Chapter","GLOBAL",4)~ + @24 + NeirosGreets1.1.1E
-+ ~GlobalLT("Chapter","GLOBAL",4)~ + @25 + NeirosGreets1.1.1F
++ ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_4%)~ + @23 + NeirosGreets1.1.1D
++ ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_4%)~ + @24 + NeirosGreets1.1.1E
++ ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_4%)~ + @25 + NeirosGreets1.1.1F
 END
 
 IF ~~ NeirosGreets1.2
@@ -218,9 +218,9 @@
 ++ @20  + NeirosGreets1.1.1A
 ++ @21  + NeirosGreets1.1.1B
 ++ @22  + NeirosGreets1.1.1C
-+ ~GlobalLT("Chapter","GLOBAL",4)~ + @23 + NeirosGreets1.1.1D
-+ ~GlobalLT("Chapter","GLOBAL",4)~ + @24 + NeirosGreets1.1.1E
-+ ~GlobalLT("Chapter","GLOBAL",4)~ + @25 + NeirosGreets1.1.1F
++ ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_4%)~ + @23 + NeirosGreets1.1.1D
++ ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_4%)~ + @24 + NeirosGreets1.1.1E
++ ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_4%)~ + @25 + NeirosGreets1.1.1F
 END
 
 IF ~~ NeirosMakesBow
