--- mur'neth\dialogues\bgt\a#murfin.d	Wed Nov 30 16:13:36 2005
+++ C:\BWP Patchstudio\patched files\mur'neth\dialogues\bgt\a#murfin.d	Sun Nov 28 05:28:15 2010
@@ -26,11 +26,11 @@
 !StateCheck("IMOEN2",STATE_SLEEPING)
 Global("FinchMurn2","LOCALS",0)~ THEN BSUFINCH ScribblySort
 @967 DO ~SetGlobal("FinchMurn2","LOCALS",1)~
-== BIMOEN @968 
+== BIMOEN2 @968 
 == BSUFINCH @969
-== BIMOEN @970
+== BIMOEN2 @970
 == BSUFINCH @971
-== BIMOEN @972
+== BIMOEN2 @972
 =
 @973
 =
