--- mur'neth\dialogues\bgt\ba#murn.d	Wed Nov 30 16:09:20 2005
+++ C:\BWP Patchstudio\patched files\mur'neth\dialogues\bgt\ba#murn.d	Sun Nov 28 05:25:53 2010
@@ -363,16 +363,16 @@
 !Dead("IMOEN2")~ THEN ~BA#MURN~ Murn_IMOEN
 @133
 DO ~SetGlobal("A#MURN_IMOEN","LOCALS",1)~
-== ~BIMOEN~ @134
+== ~BIMOEN2~ @134
 == BA#MURN @135
-== ~BIMOEN~ @136
+== ~BIMOEN2~ @136
 == ~BQUAYL~ IF ~InParty("QUAYLE")
 See("QUAYLE")
 !StateCheck("QUAYLE",STATE_SLEEPING)
 !Dead("QUAYLE")~ THEN @137
-== ~BIMOEN~ @138
+== ~BIMOEN2~ @138
 == BA#MURN @139
-== ~BIMOEN~ @140
+== ~BIMOEN2~ @140
 == BA#MURN @141
 EXIT
 
