--- keldorn_rom/setup-keldorn_rom.tp2	2015-01-14 14:59:14.000000000 -0500
+++ Fixpack/keldorn_rom/setup-keldorn_rom.tp2	2016-11-21 10:34:06.144638100 -0500
@@ -3,5 +3,5 @@
 AUTHOR ~berelinde@gmail.com jastey@web.de~ 
 
-VERSION ~beta_jastey150114~
+VERSION ~beta_jastey150114 BWP Fix~
 
 README ~keldorn_rom/docs/readme-keldorn_rom.txt~ 
@@ -9,6 +9,17 @@
 AUTO_TRA ~keldorn_rom/tra/autotra/%s~
 
 ALWAYS
+  ACTION_IF GAME_IS ~eet~ BEGIN
+    OUTER_SET bg2_chapter = 12
+  END ELSE BEGIN
+    OUTER_SET bg2_chapter = 0
+  END
+  OUTER_FOR (i=1; i<=10; i=i+1) BEGIN
+    OUTER_SET bg2_chapter = bg2_chapter + 1
+    OUTER_SPRINT name_source ~bg2_chapter_%i%~
+    OUTER_SET EVAL ~%name_source%~ = bg2_chapter
+  END
+
 
 
 /* all following actions are only processed ONCE for the whole mod, independent of un- and reinstalling of single components */
@@ -45,7 +56,7 @@
   LOAD_TRA ~keldorn_rom\Tra/autotra/%LANGUAGE%/game.tra~
   LOAD_TRA ~keldorn_rom\Tra/autotra/%LANGUAGE%/setup.tra~
 
-ACTION_IF GAME_IS ~bg2ee~ THEN BEGIN
+ACTION_IF GAME_IS ~bg2ee eet~ THEN BEGIN
   LOAD_TRA ~keldorn_rom\Tra/autotra/%LANGUAGE%/game_ee.tra~
 END
 
@@ -369,7 +380,7 @@
 COPY ~keldorn_rom/items/descr_bam/cb!zscm.bam~ ~override/cb!zscm.bam~ 
 END
 
-ACTION_IF GAME_IS ~bg2ee~ THEN BEGIN
+ACTION_IF GAME_IS ~bg2ee eet~ THEN BEGIN
 COPY ~keldorn_rom/items/descr_bam/cb!krng.bmp~ ~override/cb!krng.bmp~ 
 COPY ~keldorn_rom/items/descr_bam/cb!zaky.bmp~ ~override/cb!zaky.bmp~ 
 COPY ~keldorn_rom/items/descr_bam/cb!zscm.bmp~ ~override/cb!zscm.bmp~ 
@@ -493,7 +504,7 @@
 COMPILE ~keldorn_rom/dialogue/b!brega.d~ 
 COMPILE ~keldorn_rom/dialogue/b!banter.d~ 
 COMPILE ~keldorn_rom/dialogue/b!night.d~ 
-COMPILE ~keldorn_rom/dialogue/b!interjections.d~ 
+COMPILE EVALUATE_BUFFER ~keldorn_rom/dialogue/b!interjections.d~ 
 COMPILE ~keldorn_rom/dialogue/b!wounds.d~ 
 COMPILE EVALUATE_BUFFER ~keldorn_rom/dialogue/b!tob_timer_talks.d~ 
 COMPILE ~keldorn_rom/dialogue/b!tob_morning.d~ 
@@ -535,6 +546,7 @@
 //EXTEND_TOP    ~keldorn.bcs~ ~keldorn_rom/script/b!keldorn_rom.baf~        // romance-related bcs additions 
 //    EVALUATE_BUFFER
 //EXTEND_TOP    ~keldord.bcs~ ~keldorn_rom/script/b!keldorn_rom_d.baf~      // Keldorn's night talks 
+//    EVALUATE_BUFFER
 EXTEND_BOTTOM ~baldur.bcs~  ~keldorn_rom/script/b!baldur.baf~             // Summons Keldorn when his timer expires 
 EXTEND_TOP    ~ar0300.bcs~  ~keldorn_rom/script/b!ar0300.baf~             // Spawns Zasheida 
 EXTEND_TOP    ~ar0313.bcs~  ~keldorn_rom/script/b!ar0313.baf~             // Spawns the porter for Keldorn's quest 
@@ -597,7 +609,7 @@
 COPY ~keldorn_rom/areas/b!inn1.mos~ ~override/b!inn1.mos~ 
 END
 
-ACTION_IF GAME_IS ~bg2ee~ THEN BEGIN
+ACTION_IF GAME_IS ~bg2ee eet~ THEN BEGIN
 COPY ~keldorn_rom/areas/b!inn1_ee.mos~ ~override/b!inn1.mos~ 
 COPY_EXISTING ~ar0041.mos~ ~override\b!aley.mos~
 COPY_EXISTING ~ar0318.mos~ ~override\b!hovl.mos~
@@ -662,6 +674,7 @@
     EXTEND_TOP ~keldorn.bcs~ ~keldorn_rom/script/with_music/b!keldorn_rom.baf~ 
         EVALUATE_BUFFER 
     EXTEND_TOP ~keldord.bcs~ ~keldorn_rom/script/with_music/b!keldorn_rom_d.baf~ 
+        EVALUATE_BUFFER 
     
     COPY_EXISTING ~keldorn.bcs~ ~override~ 
         DECOMPILE_BCS_TO_BAF 
@@ -688,7 +701,7 @@
     EXTEND_TOP ~keldorn.bcs~ ~keldorn_rom/script/b!keldorn_rom.baf~ 
         EVALUATE_BUFFER 
     EXTEND_TOP ~keldord.bcs~ ~keldorn_rom/script/b!keldorn_rom_d.baf~ 
-    
+        EVALUATE_BUFFER 
     EXTEND_TOP ~keld25.bcs~ ~keldorn_rom/script/b!keldorn_rom_tob.baf~ 
         EVALUATE_BUFFER 
 
@@ -738,7 +751,7 @@
 */
 
 
-  ACTION_IF GAME_IS ~bg2ee~ THEN BEGIN
+  ACTION_IF GAME_IS ~bg2ee eet~ THEN BEGIN
       COPY ~keldorn_rom/bmp/senka_m_ee.bmp~  ~override/nkeldorm.bmp~
       COPY ~keldorn_rom/bmp/senka_s_ee.bmp~  ~override/nkeldors.bmp~
   END
