--- NTotSC\D\NTTORDIO.D	Wed Apr 26 20:48:06 2006
+++ C:\BWP Patchstudio\patched files\NTotSC\_depends\Vault\D\NTTORDIO.D	Sun Aug 18 20:09:55 2013
@@ -49,7 +49,7 @@
 IF ~Global("TorkionNeedYou","Global",1)
 Global("TorkionReadyToFight","Global",0)
 !PartyHasItem("DSPOTN01")
-!PartyHasItem("SW1H19")
+!PartyHasItem("SW1HVAM")
 ~ THEN BEGIN 7
   SAY @19
   IF ~~ THEN GOTO 8
@@ -66,12 +66,12 @@
 IF ~Global("TorkionNeedYou","Global",1)
 Global("TorkionReadyToFight","Global",0)
 PartyHasItem("DSPOTN01")
-PartyHasItem("SW1H19")
+PartyHasItem("SW1HVAM")
 ~ THEN BEGIN 9
   SAY @23
   IF ~~ THEN DO ~SetGlobal("TorkionNeedYou","Global",2)
 TakePartyItem("DSPOTN01")
-TakePartyItem("SW1H19")
+TakePartyItem("SW1HVAM")
 DestroyItem("DSPOTN01")
 AddexperienceParty(18000)
 DropInventory()
@@ -83,7 +83,7 @@
 IF ~Global("TorkionNeedYou","Global",1)
 Global("TorkionReadyToFight","Global",0)
 !PartyHasItem("DSPOTN01")
-PartyHasItem("SW1H19")
+PartyHasItem("SW1HVAM")
 ~ THEN BEGIN 10
   SAY @25
   IF ~~ THEN GOTO 8
@@ -92,7 +92,7 @@
 IF ~Global("TorkionNeedYou","Global",1)
 Global("TorkionReadyToFight","Global",0)
 PartyHasItem("DSPOTN01")
-!PartyHasItem("SW1H19")
+!PartyHasItem("SW1HVAM")
 ~ THEN BEGIN 11
   SAY @26
   IF ~~ THEN GOTO 8
