--- vic\Dung\SCITEM.BAF	Fri Aug 12 17:09:58 2005
+++ C:\BWP Patchstudio\patched files\vic\Dung\SCITEM.BAF	Thu Apr 01 19:22:02 2010
@@ -108,7 +108,7 @@
 		Class(Myself,MAGE_ALL)
 		Class(Myself,BARD_ALL)
 	See(NearestEnemyOf(Myself))
-	!Dead("astSeenBy(Myself))")
+	!StateCheck(LastSeenBy(Myself),STATE_REALLY_DEAD)
 	!StateCheck(LastSeenBy(Myself),STATE_PANIC)
 	!HasBounceEffects(LastSeenBy(Myself))
 	!LevelGT(LastSeenBy(Myself),10)
@@ -125,7 +125,7 @@
 		Class(Myself,MAGE_ALL)
 		Class(Myself,BARD_ALL)
 	See(NearestEnemyOf(Myself))
-	!Dead("astSeenBy(Myself))")
+	!StateCheck(LastSeenBy(Myself),STATE_REALLY_DEAD)
 	!HasBounceEffects(LastSeenBy(Myself))
 	!StateCheck(LastSeenBy(Myself),STATE_STUNNED)
 	HasItem("wand04",Myself) // Wand of Paralyzation 
@@ -141,7 +141,7 @@
 		Class(Myself,MAGE_ALL)
 		Class(Myself,BARD_ALL)
 	See(NearestEnemyOf(Myself))
-	!Dead("astSeenBy(Myself))")
+	!StateCheck(LastSeenBy(Myself),STATE_REALLY_DEAD)
 	!HasBounceEffects(LastSeenBy(Myself))
 	HasItem("wand06",Myself) // Wand of Frost
 THEN
@@ -156,7 +156,7 @@
 		Class(Myself,MAGE_ALL)
 		Class(Myself,BARD_ALL)
 	See(NearestEnemyOf(Myself))
-	!Dead("astSeenBy(Myself))")
+	!StateCheck(LastSeenBy(Myself),STATE_REALLY_DEAD)
 	!StateCheck(LastSeenBy(Myself),STATE_SLEEPING)
 	!HasBounceEffects(LastSeenBy(Myself))
 	!LevelGT(LastSeenBy(Myself),10)
@@ -173,7 +173,7 @@
 		Class(Myself,MAGE_ALL)
 		Class(Myself,BARD_ALL)
 	See(NearestEnemyOf(Myself))
-	!Dead("astSeenBy(Myself))")
+	!StateCheck(LastSeenBy(Myself),STATE_REALLY_DEAD)
 	!LevelGT(LastSeenBy(Myself),10)
 	!HasBounceEffects(LastSeenBy(Myself))
 	HasItem("wand09",Myself) // Wand of Polymorphing
@@ -189,7 +189,7 @@
 		Class(Myself,MAGE_ALL)
 		Class(Myself,BARD_ALL)
 	See(NearestEnemyOf(Myself))
-	!Dead("astSeenBy(Myself))")
+	!StateCheck(LastSeenBy(Myself),STATE_REALLY_DEAD)
 	HasItem("wand10",Myself) // Wand of Monster Summoning
 THEN
 	RESPONSE #100
@@ -201,7 +201,7 @@
 IF
 	Class(Myself,CLERIC_ALL)
 	See(NearestEnemyOf(Myself))
-	!Dead("astSeenBy(Myself))")
+	!StateCheck(LastSeenBy(Myself),STATE_REALLY_DEAD)
 	OR(2)
 		!CheckStatGT(LastSeenBy(Myself),50,RESISTMAGICFIRE)
 		!HasImmunityEffects(LastSeenBy(Myself))
