--- Setup-NeJ2Biffer.tp2	Thu Mar 30 08:21:10 2006
+++ C:\BWP Patchstudio\patched files\NeJ2\_depends\BGT\Setup-NeJ2Biffer.tp2	Sat Jun 21 08:55:04 2008
@@ -3,6 +3,7 @@
 AUTHOR ~Horred the Plague)~
 //NOTE:I'm not the author, but this really means 'Who to send the nasty E-Mails to'
 
+VERSION ~v3 BWP Fix~
 
 BEGIN ~Biffing NeJv4.2 files. Run this INSTEAD OF Setup-NeJ2.exe!                      Run Setup-NeJ2.exe FIRST to uninstall!~
 
@@ -40,7 +41,7 @@
 COPY ~music/combo1/combo1a.acm~ ~override/GUImusic~
 COPY ~music/combo1/combo1b.acm~ ~override/GUImusic~
 COPY ~music/theme/themea.acm~  ~override/GUImusic~
-AT_INTERACTIVE_EXIT ~Setup-NeJ2.exe~
+//AT_INTERACTIVE_EXIT ~Setup-NeJ2.exe~
 
 
 
