--- keto/d/ketoquest.d	2004-11-21 22:52:56.000000000 -0500
+++ Fixpack/keto/d/ketoquest.d	2016-11-21 16:26:10.794062000 -0500
@@ -123,8 +123,8 @@
 IF ~Class(Player1, BARD_ALL)~ THEN REPLY @3055 GOTO KetoUnderstandsTotally
 IF ~Class(Player1, BARD_ALL)~ THEN REPLY @3056 GOTO KetoArtisticSelfishness
 IF ~!Class(Player1, BARD_ALL)~ THEN REPLY @3057 GOTO KetoNotKidding
-IF ~GlobalLT("Chapter","GLOBAL",5)~ THEN REPLY @3058 GOTO KetoSelfish2
-IF ~GlobalGT("Chapter","GLOBAL",5)~ THEN REPLY @3059 GOTO KetoWaste
+IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_5%)~ THEN REPLY @3058 GOTO KetoSelfish2
+IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)~ THEN REPLY @3059 GOTO KetoWaste
 ++ @3060 GOTO KetoWalksOut
 END
 
@@ -526,8 +526,8 @@
 ++ @3200 GOTO KetoUnderstandsKinda
 IF ~Class(Player1, BARD_ALL)~ THEN REPLY @3055 GOTO KetoUnderstandsTotally
 IF ~!Class(Player1, BARD_ALL)~ THEN REPLY @3201 GOTO KetoNotKidding
-IF ~GlobalLT("Chapter","GLOBAL",5)~ THEN REPLY @3202 GOTO KetoSelfish
-IF ~GlobalGT("Chapter","GLOBAL",5)~ THEN REPLY @3203 GOTO KetoWaste
+IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_5%)~ THEN REPLY @3202 GOTO KetoSelfish
+IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)~ THEN REPLY @3203 GOTO KetoWaste
 ++ @3204 GOTO KetoWalksOut
 END
 
@@ -550,8 +550,8 @@
 =
 @3212
 ++ @3213 GOTO KetoQuestHappy
-IF ~GlobalLT("Chapter","GLOBAL",5)~ THEN REPLY @3214 GOTO KetoSelfish
-IF ~GlobalGT("Chapter","GLOBAL",5)~ THEN REPLY @3215 GOTO KetoWaste
+IF ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_5%)~ THEN REPLY @3214 GOTO KetoSelfish
+IF ~GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)~ THEN REPLY @3215 GOTO KetoWaste
 ++ @3216 GOTO KetoUnderstandsKinda
 ++ @3217 GOTO KetoWalksOut2
 END
