--- alternatives\dialogue\dialogue.d	Sun Jun 07 21:42:18 2015
+++ C:\BWP Patchstudio\patched files\alternatives\dialogue\dialogue.d	Fri Nov 13 09:30:54 2015
@@ -2170,7 +2170,7 @@
 
 APPEND ~B!LEYT~ 
 
-IF ~Global("B!Tourist","GLOBAL",2)~ THEN BEGIN l0 
+IF ~Global("B!BonkedLeyt","GLOBAL",0)~ THEN BEGIN l0 
 SAY @597  
 + ~OR(2) CheckStatGT(Player1,13,CHR) CheckStatGT(Player2,13,STR)~ + @598 + l2
 + ~!CheckStatGT(Player1,13,CHR) !CheckStatGT(Player2,13,STR)~ + @598 + l3
