--- alternatives/dialogue/dialogue.d	2015-06-07 22:42:18.377314000 -0400
+++ Fixpack/alternatives/dialogue/dialogue.d	2016-11-21 21:53:24.991330700 -0500
@@ -351,7 +351,7 @@
 + ~PartyGoldLT(15000)~ + @121 + aster23a
 + ~!PartyGoldLT(15000)~ + @122 + aster23b
 + ~!PartyGoldLT(15000)~ + @123 DO ~
-//SetGlobal("Chapter","GLOBAL",3) 
+//SetGlobal("Chapter","GLOBAL",%bg2_chapter_3%) 
 SetGlobal("B!Alternatives","GLOBAL",6) TakePartyGold(15000) DestroyGold(15000)~ + aster23c
 END
 
@@ -1204,7 +1204,7 @@
 
 
 IF WEIGHT #-1 ~Global("B!Alternatives","GLOBAL",10) 
-    Global("Chapter","GLOBAL",6)
+    Global("Chapter","GLOBAL",%bg2_chapter_6%)
     InPartySlot(LastTalkedToBy,0)
     !Global("PaladinOrder","GLOBAL",1)
     ReputationLT(LastTalkedToBy,10)
@@ -2170,7 +2170,7 @@
 
 APPEND ~B!LEYT~ 
 
-IF ~Global("B!Tourist","GLOBAL",2)~ THEN BEGIN l0 
+IF ~Global("B!BonkedLeyt","GLOBAL",0)~ THEN BEGIN l0 // Lollorian changed global check from 2 to 0
 SAY @597  
 + ~OR(2) CheckStatGT(Player1,13,CHR) CheckStatGT(Player2,13,STR)~ + @598 + l2
 + ~!CheckStatGT(Player1,13,CHR) !CheckStatGT(Player2,13,STR)~ + @598 + l3
@@ -2438,7 +2438,7 @@
 ++ @685 + ahb3b 
 ++ @686 + ahb3a 
 ++ @687 + ahb3a 
-+ ~GlobalLT("Chapter","GLOBAL",4)~ + @688 + ahb3c 
++ ~GlobalLT("Chapter","GLOBAL",%bg2_chapter_4%)~ + @688 + ahb3c 
 ++ @689 + ahb3d
 END 
 
