--- AR4300.BAF	2016-01-28 23:29:19
+++ AR4300.BAF	2016-01-28 23:28:51
@@ -1,13 +1,6 @@
-IF
-	Global("PoorMrColquetleFamily","GLOBAL",0)
-THEN
-	RESPONSE #100
-		SetGlobal("PoorMrColquetleFamily","GLOBAL",1)
-END
-
 IF
 	Global("TeldornMessSpawn","GLOBAL",0)
 	GlobalGT("Chapter","GLOBAL",%tutu_chapter_4%)
 THEN
 	RESPONSE #100
 		CreateCreature("TELMESS1",[2142.1648]%FACE_8%) // Teldorn Messenger
@@ -18,21 +11,25 @@
 		CreateCreature("STONEGOL",[2080.1580]%FACE_10%) // Stone Golem
 		SetGlobal("TeldornMessSpawn","GLOBAL",1)
 END
 
 IF
+	Global("DoorS001Active","AR3200",0)
 	GlobalLT("CearwinQuest","GLOBAL",2)
-	Global("DoorS001active","GLOBAL",0)
 THEN
 	RESPONSE #100
 		TriggerActivation("DoorS001",FALSE)
+		HideAreaOnMap("ARS002")
+		HideAreaOnMap("ARS005")
+		SetGlobal("DoorS001Active","AR3200",1)
 		Continue()
 END
 
 IF
-	Global("DoorS001active","GLOBAL",0)
+	Global("DoorS001Active","AR3200",1)
 	GlobalGT("CearwinQuest","GLOBAL",1)
 THEN
 	RESPONSE #100
 		TriggerActivation("DoorS001",TRUE)
-		SetGlobal("DoorS001active","GLOBAL",1)
+		SetGlobal("DoorS001Active","AR3200",2)
+		Continue()
 END
\ No newline at end of file
