--- Tiax/Dialogue/O#Tia25J.d	2007-01-17 23:42:22.000000000 -0500
+++ Fixpack/Tiax/Dialogue/O#Tia25J.d	2016-11-21 19:54:00.508150600 -0500
@@ -34,14 +34,6 @@
 == O#TIA25J IF ~InParty("O#Tiax") InMyArea("O#Tiax") !StateCheck("O#Tiax",CD_STATE_NOTVALID)~ THEN @7
 END
 
-EXTEND_BOTTOM FATESP 6 #4
-+ ~!Dead("O#Tiax") !InMyArea("O#Tiax") Global("O#TiaxSummoned","GLOBAL",0)~ + @8
-DO ~CreateVisualEffect("SPPORTAL",[1999.1228])
-Wait(2) 
-CreateCreature("O#Tiax25",[1999.1228],0) 
-SetGlobal("O#TiaxSummoned","GLOBAL",1)~ GOTO 8
-END
-
 EXTEND_BOTTOM SARVOLO 9 #4
 + ~InParty("O#Tiax") InMyArea("O#Tiax") !StateCheck("O#Tiax",CD_STATE_NOTVALID)~ + @9 + O#TiaxVoloBio1
 END
@@ -61,33 +53,33 @@
 // 0
 
 I_C_T GORPOL1 1 O#TiaxMetKnightsOfTheVigil1
-== O#TIA25J IF ~InParty("O#Tiax") InMyArea("O#Tiax") !StateCheck("O#Tiax",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @14
+== O#TIA25J IF ~InParty("O#Tiax") InMyArea("O#Tiax") !StateCheck("O#Tiax",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @14
 END
 
 // 1
 
 I_C_T2 GORCHR 11 O#TiaxChromaticDemon2
-== O#TIA25J IF ~InParty("O#Tiax") InMyArea("O#Tiax") !StateCheck("O#Tiax",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @15
+== O#TIA25J IF ~InParty("O#Tiax") InMyArea("O#Tiax") !StateCheck("O#Tiax",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @15
 END
 
 I_C_T GORIMP01 4 O#TiaxImpDeadWizards1
-== O#TIA25J IF ~InParty("O#Tiax") InMyArea("O#Tiax") !StateCheck("O#Tiax",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @16
+== O#TIA25J IF ~InParty("O#Tiax") InMyArea("O#Tiax") !StateCheck("O#Tiax",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @16
 END
 
 // 3
 
 I_C_T GORCAR 3 O#TiaxCarston1
-== O#TIA25J IF ~InParty("O#Tiax") InMyArea("O#Tiax") !StateCheck("O#Tiax",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @17
+== O#TIA25J IF ~InParty("O#Tiax") InMyArea("O#Tiax") !StateCheck("O#Tiax",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @17
 END
 
 // 5
 
 I_C_T GORDEMO 1 O#TiaxImprisonedOneMet1
-== O#TIA25J IF ~InParty("O#Tiax") InMyArea("O#Tiax") !StateCheck("O#Tiax",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @18
+== O#TIA25J IF ~InParty("O#Tiax") InMyArea("O#Tiax") !StateCheck("O#Tiax",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @18
 END
 
 I_C_T GORODR1 49 O#TiaxOdrenFoolishness1
-== O#TIA25J IF ~InParty("O#Tiax") InMyArea("O#Tiax") !StateCheck("O#Tiax",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",7)~ THEN @19
+== O#TIA25J IF ~InParty("O#Tiax") InMyArea("O#Tiax") !StateCheck("O#Tiax",CD_STATE_NOTVALID) GlobalGT("Chapter","GLOBAL",%bg2_chapter_7%)~ THEN @19
 END
 
 // Watcher's Keep interjections, SoA
@@ -95,33 +87,33 @@
 // 0
 
 I_C_T GORPOL1 1 O#TiaxMetKnightsOfTheVigil1
-== O#TIAXJ IF ~InParty("O#Tiax") InMyArea("O#Tiax") !StateCheck("O#Tiax",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @14
+== O#TIAXJ IF ~InParty("O#Tiax") InMyArea("O#Tiax") !StateCheck("O#Tiax",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @14
 END
 
 // 1
 
 I_C_T2 GORCHR 11 O#TiaxChromaticDemon2
-== O#TIAXJ IF ~InParty("O#Tiax") InMyArea("O#Tiax") !StateCheck("O#Tiax",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @20
+== O#TIAXJ IF ~InParty("O#Tiax") InMyArea("O#Tiax") !StateCheck("O#Tiax",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @20
 END
 
 I_C_T GORIMP01 4 O#TiaxImpDeadWizards1
-== O#TIAXJ IF ~InParty("O#Tiax") InMyArea("O#Tiax") !StateCheck("O#Tiax",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @16
+== O#TIAXJ IF ~InParty("O#Tiax") InMyArea("O#Tiax") !StateCheck("O#Tiax",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @16
 END
 
 // 3
 
 I_C_T GORCAR 3 O#TiaxCarston1
-== O#TIAXJ IF ~InParty("O#Tiax") InMyArea("O#Tiax") !StateCheck("O#Tiax",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @17
+== O#TIAXJ IF ~InParty("O#Tiax") InMyArea("O#Tiax") !StateCheck("O#Tiax",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @17
 END
 
 // 5
 
 I_C_T GORDEMO 1 O#TiaxImprisonedOneMet1
-== O#TIAXJ IF ~InParty("O#Tiax") InMyArea("O#Tiax") !StateCheck("O#Tiax",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @18
+== O#TIAXJ IF ~InParty("O#Tiax") InMyArea("O#Tiax") !StateCheck("O#Tiax",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @18
 END
 
 I_C_T GORODR1 49 O#TiaxOdrenFoolishness1
-== O#TIAXJ IF ~InParty("O#Tiax") InMyArea("O#Tiax") !StateCheck("O#Tiax",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",8)~ THEN @19
+== O#TIAXJ IF ~InParty("O#Tiax") InMyArea("O#Tiax") !StateCheck("O#Tiax",CD_STATE_NOTVALID) GlobalLT("Chapter","GLOBAL",%bg2_chapter_8%)~ THEN @19
 END
 
 // ToB PID
@@ -160,11 +152,11 @@
 + ~RandomNum(5,3)~ + @27 + O#TiaxToB5.3
 + ~RandomNum(5,4)~ + @27 + O#TiaxToB5.4
 + ~RandomNum(5,5)~ + @27 + O#TiaxToB5.5
-+ ~RandomNum(5,1) Global("Chapter","GLOBAL",9)~ + @28 + O#TiaxToB9.1
-+ ~RandomNum(5,2) Global("Chapter","GLOBAL",9)~ + @28 + O#TiaxToB9.2
-+ ~RandomNum(5,3) Global("Chapter","GLOBAL",9)~ + @28 + O#TiaxToB9.3
-+ ~RandomNum(5,4) Global("Chapter","GLOBAL",9)~ + @28 + O#TiaxToB9.4
-+ ~RandomNum(5,5) Global("Chapter","GLOBAL",9)~ + @28 + O#TiaxToB9.5
++ ~RandomNum(5,1) Global("Chapter","GLOBAL",%bg2_chapter_9%)~ + @28 + O#TiaxToB9.1
++ ~RandomNum(5,2) Global("Chapter","GLOBAL",%bg2_chapter_9%)~ + @28 + O#TiaxToB9.2
++ ~RandomNum(5,3) Global("Chapter","GLOBAL",%bg2_chapter_9%)~ + @28 + O#TiaxToB9.3
++ ~RandomNum(5,4) Global("Chapter","GLOBAL",%bg2_chapter_9%)~ + @28 + O#TiaxToB9.4
++ ~RandomNum(5,5) Global("Chapter","GLOBAL",%bg2_chapter_9%)~ + @28 + O#TiaxToB9.5
 + ~GlobalGT("BeginChallenge4","GLOBAL",3) Global("O#TiaxMetCyric ","GLOBAL",0)~ + @29 DO ~SetGlobal("O#TiaxMetCyric ","GLOBAL",1)~ + O#TiaxToB6
 + ~Global("O#TiaxShutUp","GLOBAL",0)~ + @30 DO ~SetGlobal("O#TiaxShutUp","GLOBAL",1)~ + O#TiaxToB7
 + ~Global("O#TiaxShutUp","GLOBAL",1)~ + @31 DO ~SetGlobal("O#TiaxShutUp","GLOBAL",0)~ + O#TiaxToB8
