--- Tiax/Dialogue/O#TiaxJ.d	2007-01-17 23:26:46.000000000 -0500
+++ Fixpack/Tiax/Dialogue/O#TiaxJ.d	2016-11-21 16:34:20.788502300 -0500
@@ -331,16 +331,16 @@
 + ~Gender(Player1,FEMALE) RandomNum(5,3)~ + @81 + O#Tiax8.3
 + ~Gender(Player1,FEMALE) RandomNum(5,4)~ + @81 + O#Tiax8.4
 + ~Gender(Player1,FEMALE) RandomNum(5,5)~ + @81 + O#Tiax8.5
-+ ~RandomNum(5,1) Global("Chapter","GLOBAL",5)~ + @82 + O#Tiax12.1
-+ ~RandomNum(5,2) Global("Chapter","GLOBAL",5)~ + @82 + O#Tiax12.2
-+ ~RandomNum(5,3) Global("Chapter","GLOBAL",5)~ + @82 + O#Tiax12.3
-+ ~RandomNum(5,4) Global("Chapter","GLOBAL",5)~ + @82 + O#Tiax12.4
-+ ~RandomNum(5,5) Global("Chapter","GLOBAL",5)~ + @82 + O#Tiax12.5
-+ ~RandomNum(5,1) GlobalGT("Chapter","GLOBAL",5)~ + @83 + O#Tiax13.1
-+ ~RandomNum(5,2) GlobalGT("Chapter","GLOBAL",5)~ + @83 + O#Tiax13.2
-+ ~RandomNum(5,3) GlobalGT("Chapter","GLOBAL",5)~ + @83 + O#Tiax13.3
-+ ~RandomNum(5,4) GlobalGT("Chapter","GLOBAL",5)~ + @83 + O#Tiax13.4
-+ ~RandomNum(5,5) GlobalGT("Chapter","GLOBAL",5)~ + @83 + O#Tiax13.5
++ ~RandomNum(5,1) Global("Chapter","GLOBAL",%bg2_chapter_5%)~ + @82 + O#Tiax12.1
++ ~RandomNum(5,2) Global("Chapter","GLOBAL",%bg2_chapter_5%)~ + @82 + O#Tiax12.2
++ ~RandomNum(5,3) Global("Chapter","GLOBAL",%bg2_chapter_5%)~ + @82 + O#Tiax12.3
++ ~RandomNum(5,4) Global("Chapter","GLOBAL",%bg2_chapter_5%)~ + @82 + O#Tiax12.4
++ ~RandomNum(5,5) Global("Chapter","GLOBAL",%bg2_chapter_5%)~ + @82 + O#Tiax12.5
++ ~RandomNum(5,1) GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)~ + @83 + O#Tiax13.1
++ ~RandomNum(5,2) GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)~ + @83 + O#Tiax13.2
++ ~RandomNum(5,3) GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)~ + @83 + O#Tiax13.3
++ ~RandomNum(5,4) GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)~ + @83 + O#Tiax13.4
++ ~RandomNum(5,5) GlobalGT("Chapter","GLOBAL",%bg2_chapter_5%)~ + @83 + O#Tiax13.5
 + ~Global("O#TiaxShutUp","GLOBAL",0)~ + @84 DO ~SetGlobal("O#TiaxShutUp","GLOBAL",0)~ + O#Tiax9
 + ~Global("O#TiaxShutUp","GLOBAL",1)~ + @85 DO ~SetGlobal("O#TiaxShutUp","GLOBAL",1)~ + O#Tiax10
 + ~Global("O#TiaxQuestQuestion","GLOBAL",0)~ + @86 DO ~SetGlobal("O#TiaxQuestQuestion","GLOBAL",1)~ + O#Tiax11
