--- Tiax/Setup-Tiax.tp2	2014-11-10 14:18:04.000000000 -0500
+++ Fixpack/Tiax/Setup-Tiax.tp2	2016-11-21 16:35:31.512329100 -0500
@@ -1,3 +1,3 @@
 BACKUP ~Tiax/Backup~
 AUTHOR ~http://forums.pocketplane.net~
-VERSION ~v4~
+VERSION ~v4 BWP Fix~
@@ -5,6 +5,17 @@
 README ~Tiax/%LANGUAGE%/Readme-Tiax.txt~ ~Tiax/Readme-Tiax.txt~ 
 
 ALWAYS
+  ACTION_IF GAME_IS ~eet~ BEGIN
+    OUTER_SET bg2_chapter = 12
+  END ELSE BEGIN
+    OUTER_SET bg2_chapter = 0
+  END
+  OUTER_FOR (i=1; i<=10; i=i+1) BEGIN
+    OUTER_SET bg2_chapter = bg2_chapter + 1
+    OUTER_SPRINT name_source ~bg2_chapter_%i%~
+    OUTER_SET EVAL ~%name_source%~ = bg2_chapter
+  END
+
   // Borrowed from Edwin Romance: convert strings to UTF-8 for BGEE/BG2EE
   INCLUDE ~%MOD_FOLDER%/handle_charsets.tpa~
 
@@ -34,7 +45,7 @@
 
 BEGIN @0
 
-ACTION_IF GAME_IS ~BG2EE~ BEGIN
+ACTION_IF GAME_IS ~bg2ee eet~ BEGIN
 LOAD_TRA ~Tiax/%LANGUAGE%/Setup-Tiax-EE.tra~
 ADD_JOURNAL @24 USING ~Tiax/%LANGUAGE%/O#Tiax.tra~ 
 END
@@ -89,7 +100,7 @@
 SAY SPELL_DISRUPTED @43
 SAY BIO @44
 
-COMPILE ~Tiax/Dialogue/BO#Tiax.d~
+COMPILE EVALUATE_BUFFER ~Tiax/Dialogue/BO#Tiax.d~
 	~Tiax/Dialogue/O#Tiax.d~
 	~Tiax/Dialogue/O#TiaxJ.d~
 	~Tiax/Dialogue/O#TiaxP.d~
@@ -149,14 +160,14 @@
 SAY SPELL_DISRUPTED @43
 SAY BIO @44
 
-COMPILE ~Tiax/Dialogue/BO#Tia25.d~
+COMPILE EVALUATE_BUFFER ~Tiax/Dialogue/BO#Tia25.d~
 	~Tiax/Dialogue/O#Tiax25.d~
 	~Tiax/Dialogue/O#Tia25J.d~
 	~Tiax/Dialogue/O#Tia25P.d~
 
 	~Tiax/Scripts/O#Tia25S.baf~
 
-ACTION_IF GAME_IS ~BG2EE~ BEGIN
+ACTION_IF GAME_IS ~bg2ee eet~ BEGIN
 COMPILE	~Tiax/Dialogue/TiaxEE.d~
 END
 
@@ -180,3 +191,23 @@
 UNLESS ~O#Tiax~
 
 LAF HANDLE_AUDIO END
+
+ACTION_IF GAME_IS ~eet~ BEGIN
+  INCLUDE ~EET/other/EET_functions.tph~
+  LAF ~EET_NPC_TRANSITION~
+    INT_VAR
+      type = 2
+      clean_ToB = 1
+    STR_VAR
+      dv = "O#Tiax"
+      override_SoA = "O#TIAXS"
+      override_ToB = "O#TIA25S"
+      dialog_ToB = "O#TIAX25"
+      traFile = EVAL "Tiax/%LANGUAGE%/O#Tia25J.tra"
+      string = "@8" //Bring me Tiax, the gnome thief/cleric.
+      stringPosDV = "Viconia"
+      clean_ToB_DV = "Tiax"
+  END
+END ELSE BEGIN
+  COMPILE ~Tiax/Dialogue/FATESP.d~ USING ~Tiax/%LANGUAGE%/O#Tia25J.tra~
+END
