--- NeJ2pat\dlgfix.d	Wed Aug 15 23:13:38 2007
+++ C:\BWP Patchstudio\patched files\NeJ2pat\dlgfix.d	Tue Aug 18 16:30:32 2009
@@ -91,7 +91,9 @@
 
 REPLACE ~TuEzek~
 
-IF ~GlobalGT("VP_Necro_Test","GLOBAL",1)~ THEN BEGIN 15
+IF ~OR(2)
+GlobalGT("VP_Necro_Test","GLOBAL",1)
+PartyHasItem("TuNeBody")~ THEN BEGIN 15
   SAY ~You return, and do you have that which I sent you for?~ 
   IF ~PartyHasItem("TuNeBody")~ THEN REPLY ~I do Ezekiel. It is yours to do with as you please.~ GOTO 16
   IF ~PartyHasItem("TuNeBody")~ THEN REPLY ~That was some challenge but yes I have the body, looks to be in perfect health... oops... I meant perfect death or at least in pristine spell component condition.~ GOTO 16
