--- BP\ASCEND\BAF\balelit1.BAF	Mon Mar 27 21:02:11 2006
+++ C:\BWP Patchstudio\patched files\BP\ASCEND\BAF\balelit1.BAF	Sun Dec 20 19:42:27 2015
@@ -132,6 +132,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -144,6 +145,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -156,6 +158,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -168,6 +171,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -180,6 +184,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -193,6 +198,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -206,6 +212,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -219,6 +226,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -232,6 +240,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -245,6 +254,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -258,6 +268,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -275,6 +286,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -292,6 +304,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -309,6 +322,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -326,6 +340,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -343,6 +358,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -360,6 +376,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -377,6 +394,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -394,6 +412,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
