--- BP\BAF\RE-BAF\BEHOLDER\BPBEHELD.BAF	Sat May 25 07:48:39 2013
+++ C:\BWP Patchstudio\patched files\BP\BAF\RE-BAF\BEHOLDER\BPBEHELD.BAF	Sun Dec 20 19:42:27 2015
@@ -380,6 +380,7 @@
     !Detect([GOODCUTOFF])
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -715,6 +716,7 @@
     Allegiance(LastSeenBy(Myself),GOODBUTRED)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -1117,6 +1119,7 @@
   RESPONSE #50
     Continue()
   RESPONSE #50
+    NoAction()
 END
 
 IF
@@ -1147,5 +1150,6 @@
     DisplayStringHead(Myself,@1)
     ChangeSpecifics(Myself,CALLED_FOR_BOULDER)
   RESPONSE #45
+    NoAction()
 END
 
