--- BP\ASCEND\TOUGH\GROM\BAF\CORE\BPCARC2B.BAF	Sat May 25 07:39:19 2013
+++ C:\BWP Patchstudio\patched files\BP\ASCEND\TOUGH\GROM\BAF\CORE\BPCARC2B.BAF	Sun Dec 20 19:42:27 2015
@@ -56,6 +56,7 @@
     !Detect([GOODCUTOFF])
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -238,6 +239,7 @@
     Allegiance(LastSeenBy(Myself),GOODBUTRED)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -246,6 +248,7 @@
   RESPONSE #80
     Continue()
   RESPONSE #20
+    NoAction()
 END
 
 IF
@@ -302,6 +305,7 @@
   RESPONSE #25
     Continue()
   RESPONSE #75
+    NoAction()
 END
 
 IF
