--- BP\BAF\RE-BAF\CORE\BPCCLE2B.BAF	Tue Oct 15 06:29:34 2013
+++ C:\BWP Patchstudio\patched files\BP\BAF\RE-BAF\CORE\BPCCLE2B.BAF	Sun Dec 20 19:42:27 2015
@@ -413,6 +413,7 @@
     !Detect([GOODCUTOFF])
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -512,6 +513,7 @@
   See([GOODCUTOFF.UNDEAD])
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -963,6 +965,7 @@
     Allegiance(LastSeenBy(Myself),GOODBUTRED)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -971,6 +974,7 @@
   Specifics(LastSeenBy(Myself),65)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 
@@ -980,6 +984,7 @@
   RESPONSE #80
     Continue()
   RESPONSE #20
+    NoAction()
 END
 
 
@@ -1694,6 +1699,7 @@
     Allegiance(LastSeenBy(Myself),GOODBUTRED)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -1702,6 +1708,7 @@
   Specifics(LastSeenBy(Myself),65)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 
@@ -1711,6 +1718,7 @@
   RESPONSE #80
     Continue()
   RESPONSE #20
+    NoAction()
 END
 
 
@@ -1773,6 +1781,7 @@
   RESPONSE #25
     Continue()
   RESPONSE #75
+    NoAction()
 END
 
 IF
