--- BP\BAF\RE-BAF\CORE\BPCMNK2B.BAF	Sat Jun 08 05:06:39 2013
+++ C:\BWP Patchstudio\patched files\BP\BAF\RE-BAF\CORE\BPCMNK2B.BAF	Sun Dec 20 19:42:27 2015
@@ -177,6 +177,7 @@
     !Detect([GOODCUTOFF])
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -525,6 +526,7 @@
     Allegiance(LastSeenBy(Myself),GOODBUTRED)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -533,6 +535,7 @@
   Specifics(LastSeenBy(Myself),65)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 
@@ -542,6 +545,7 @@
   RESPONSE #80
     Continue()
   RESPONSE #20
+    NoAction()
 END
 
 
@@ -827,6 +831,7 @@
   RESPONSE #25
     Continue()
   RESPONSE #75
+    NoAction()
 END
 
 IF
