--- BP\BAF\NO_TDD\DRUID\BPDRUD6A.BAF	Sat Jun 08 05:06:38 2013
+++ C:\BWP Patchstudio\patched files\BP\BAF\NO_TDD\DRUID\BPDRUD6A.BAF	Sun Dec 20 19:42:27 2015
@@ -367,6 +367,7 @@
     !Detect([GOODCUTOFF])
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -782,6 +783,7 @@
     Allegiance(LastSeenBy(Myself),GOODBUTRED)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 
@@ -791,6 +793,7 @@
   Specifics(LastSeenBy(Myself),65)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -822,6 +825,7 @@
   RESPONSE #80
     Continue()
   RESPONSE #20
+    NoAction()
 END
 
 
@@ -1242,6 +1246,7 @@
     Allegiance(LastSeenBy(Myself),GOODBUTRED)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 
@@ -1251,6 +1256,7 @@
   Specifics(LastSeenBy(Myself),65)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -1259,6 +1265,7 @@
   RESPONSE #80
     Continue()
   RESPONSE #20
+    NoAction()
 END
 
 
@@ -1427,6 +1434,7 @@
   RESPONSE #25
     Continue()
   RESPONSE #75
+    NoAction()
 END
 
 IF
