--- BP\BAF\NO_TDD\CORE\BPDUEALL.BAF	Sat Jun 08 05:06:38 2013
+++ C:\BWP Patchstudio\patched files\BP\BAF\NO_TDD\CORE\BPDUEALL.BAF	Sun Dec 20 19:42:27 2015
@@ -60,6 +60,7 @@
     !Detect([GOODCUTOFF])
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -511,6 +512,7 @@
     Allegiance(LastSeenBy(Myself),GOODBUTRED)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 
@@ -520,6 +522,7 @@
   RESPONSE #80
     Continue()
   RESPONSE #20
+    NoAction()
 END
 
 
@@ -624,6 +627,7 @@
   RESPONSE #25
     Continue()
   RESPONSE #75
+    NoAction()
 END
 
 IF
