--- BP\TDD\BAF\NO_STRINGS\BPDUEALL.BAF	Sat May 25 10:19:41 2013
+++ C:\BWP Patchstudio\patched files\BP\TDD\BAF\NO_STRINGS\BPDUEALL.BAF	Sun Dec 20 19:42:27 2015
@@ -185,6 +185,7 @@
     !Detect([GOODCUTOFF])
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -483,6 +484,7 @@
     Allegiance(LastSeenBy(Myself),GOODBUTRED)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -491,6 +493,7 @@
   RESPONSE #80
     Continue()
   RESPONSE #20
+    NoAction()
 END
 
 IF
@@ -587,6 +590,7 @@
   RESPONSE #25
     Continue()
   RESPONSE #75
+    NoAction()
 END
 
 IF
