--- BP\BAF\NEW_BAF\UNDEAD\BPLICH01.baf	Sat May 25 07:39:19 2013
+++ C:\BWP Patchstudio\patched files\BP\BAF\NEW_BAF\UNDEAD\BPLICH01.baf	Sun Dec 20 19:42:28 2015
@@ -329,6 +329,7 @@
     !Detect([GOODCUTOFF])
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 
@@ -653,6 +654,7 @@
     Allegiance(LastSeenBy(Myself),GOODBUTRED)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -661,6 +663,7 @@
   RESPONSE #80
     Continue()
   RESPONSE #20
+    NoAction()
 END
 
 IF
@@ -1121,6 +1124,7 @@
   RESPONSE #25
     Continue()
   RESPONSE #75
+    NoAction()
 END
 
 IF
