--- BP\BAF\RE-BAF\CORE\BPMAGE6C.BAF	Tue Oct 15 06:29:34 2013
+++ C:\BWP Patchstudio\patched files\BP\BAF\RE-BAF\CORE\BPMAGE6C.BAF	Sun Dec 20 19:42:28 2015
@@ -417,6 +417,7 @@
     !Detect([GOODCUTOFF])
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -742,6 +743,7 @@
     Allegiance(LastSeenBy(Myself),GOODBUTRED)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -750,6 +752,7 @@
   Specifics(LastSeenBy(Myself),65)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 
@@ -759,6 +762,7 @@
   RESPONSE #80
     Continue()
   RESPONSE #20
+    NoAction()
 END
 
 
@@ -1152,6 +1156,7 @@
     Allegiance(LastSeenBy(Myself),GOODBUTRED)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -1160,6 +1165,7 @@
   Specifics(LastSeenBy(Myself),65)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 
@@ -1169,6 +1175,7 @@
   RESPONSE #80
     Continue()
   RESPONSE #20
+    NoAction()
 END
 
 
@@ -1240,6 +1247,7 @@
   RESPONSE #25
     Continue()
   RESPONSE #75
+    NoAction()
 END
 
 IF
