--- BP\BAF\RE-BAF\CORE\BPMAGE3.BAF	Tue Oct 15 05:07:46 2013
+++ C:\BWP Patchstudio\patched files\BP\BAF\RE-BAF\CORE\BPMAGE3.BAF	Sun Dec 20 19:42:28 2015
@@ -494,6 +494,7 @@
     !Detect([GOODCUTOFF])
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -1130,6 +1131,7 @@
     Allegiance(LastSeenBy(Myself),GOODBUTRED)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -1138,6 +1140,7 @@
   Specifics(LastSeenBy(Myself),65)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 
@@ -1163,6 +1166,7 @@
   RESPONSE #80
     Continue()
   RESPONSE #20
+    NoAction()
 END
 
 
@@ -1699,6 +1703,7 @@
     Allegiance(LastSeenBy(Myself),GOODBUTRED)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -1707,6 +1712,7 @@
   Specifics(LastSeenBy(Myself),65)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 
@@ -1716,6 +1722,7 @@
   RESPONSE #80
     Continue()
   RESPONSE #20
+    NoAction()
 END
 
 
@@ -1784,6 +1791,7 @@
   RESPONSE #25
     Continue()
   RESPONSE #75
+    NoAction()
 END
 
 IF
