--- BP\BAF\RE-BAF\CORE\BPPRI12A.BAF	Tue Oct 15 06:29:34 2013
+++ C:\BWP Patchstudio\patched files\BP\BAF\RE-BAF\CORE\BPPRI12A.BAF	Sun Dec 20 19:42:28 2015
@@ -421,6 +421,7 @@
     !Detect([GOODCUTOFF])
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -506,6 +507,7 @@
   See([GOODCUTOFF.UNDEAD])
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -1492,6 +1494,7 @@
     Allegiance(LastSeenBy(Myself),GOODBUTRED)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -1500,6 +1503,7 @@
   Specifics(LastSeenBy(Myself),65)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -1571,6 +1575,7 @@
   RESPONSE #80
     Continue()
   RESPONSE #20
+    NoAction()
 END
 
 
@@ -2238,6 +2243,7 @@
     Allegiance(LastSeenBy(Myself),GOODBUTRED)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -2246,6 +2252,7 @@
   Specifics(LastSeenBy(Myself),65)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -2254,6 +2261,7 @@
   RESPONSE #80
     Continue()
   RESPONSE #20
+    NoAction()
 END
 
 
@@ -2385,6 +2393,7 @@
   RESPONSE #25
     Continue()
   RESPONSE #75
+    NoAction()
 END
 
 IF
