--- BP\BAF\NEW_BAF\SHADOW\BPSHADOW.baf	Fri Jun 07 08:53:01 2013
+++ C:\BWP Patchstudio\patched files\BP\BAF\NEW_BAF\SHADOW\BPSHADOW.baf	Sun Dec 20 19:42:28 2015
@@ -276,6 +276,7 @@
     !Detect([GOODCUTOFF])
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -347,6 +348,7 @@
   Global("BPTURNED","LOCALS",1)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -701,6 +703,7 @@
     Allegiance(LastSeenBy(Myself),GOODBUTRED)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -1196,6 +1199,7 @@
   RESPONSE #25
     Continue()
   RESPONSE #75
+    NoAction()
 END
 
 IF
