--- BP\BAF\NEW_BAF\CORE\BPTHIEF.BAF	Wed Jan 01 20:38:30 2014
+++ C:\BWP Patchstudio\patched files\BP\BAF\NEW_BAF\CORE\BPTHIEF.BAF	Sun Dec 20 19:42:28 2015
@@ -57,6 +57,7 @@
   !Allegiance(Myself,EVILCUTOFF)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -64,6 +65,7 @@
   !Detect([GOODCUTOFF])
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -408,6 +410,7 @@
   !Detect([GOODCUTOFF])
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -758,6 +761,7 @@
     Allegiance(LastSeenBy(Myself),GOODBUTRED)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 
@@ -767,6 +771,7 @@
   RESPONSE #80
     Continue()
   RESPONSE #20
+    NoAction()
 END
 
 
@@ -863,6 +868,7 @@
   RESPONSE #25
     Continue()
   RESPONSE #75
+    NoAction()
 END
 
 IF
