--- BP\BAF\SHOUTS\NO_STRINGS_2\BPWTSIGT.BAF	Sun Feb 16 21:26:02 2014
+++ C:\BWP Patchstudio\patched files\BP\BAF\SHOUTS\NO_STRINGS_2\BPWTSIGT.BAF	Sun Dec 20 19:42:29 2015
@@ -228,6 +228,7 @@
     !Detect([GOODCUTOFF])
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -420,6 +421,7 @@
   Global("BPTURNED","LOCALS",1)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -703,6 +705,7 @@
     Allegiance(LastSeenBy(Myself),GOODBUTRED)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 
@@ -712,6 +715,7 @@
   Specifics(LastSeenBy(Myself),65)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -720,6 +724,7 @@
   RESPONSE #80
     Continue()
   RESPONSE #20
+    NoAction()
 END
 
 IF
@@ -1363,6 +1368,7 @@
   RESPONSE #25
     Continue()
   RESPONSE #75
+    NoAction()
 END
 
 IF
