--- BP\BAF\RE-BAF\CORE\BPWTRSGT.BAF	Sat Jun 08 05:06:40 2013
+++ C:\BWP Patchstudio\patched files\BP\BAF\RE-BAF\CORE\BPWTRSGT.BAF	Sun Dec 20 19:42:29 2015
@@ -190,6 +190,7 @@
     !Detect([GOODCUTOFF])
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -219,6 +220,7 @@
     TurnedBy([GOODCUTOFF.0.0.PALADIN_ALL])
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -584,6 +586,7 @@
     Allegiance(LastSeenBy(Myself),GOODBUTRED)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -592,6 +595,7 @@
   Specifics(LastSeenBy(Myself),65)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -600,6 +604,7 @@
   RESPONSE #80
     Continue()
   RESPONSE #20
+    NoAction()
 END
 
 
@@ -850,6 +855,7 @@
     Allegiance(LastSeenBy(Myself),GOODBUTRED)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -858,6 +864,7 @@
   Specifics(LastSeenBy(Myself),65)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -866,6 +873,7 @@
   RESPONSE #80
     Continue()
   RESPONSE #20
+    NoAction()
 END
 
 
@@ -965,6 +973,7 @@
   RESPONSE #25
     Continue()
   RESPONSE #75
+    NoAction()
 END
 
 IF
