--- BP\BAF\SHOUTS\STRINGS\BPWZASGT.BAF	Sat Jun 08 05:06:41 2013
+++ C:\BWP Patchstudio\patched files\BP\BAF\SHOUTS\STRINGS\BPWZASGT.BAF	Sun Dec 20 19:42:29 2015
@@ -232,6 +232,7 @@
     !Detect([GOODCUTOFF])
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -518,6 +519,7 @@
     Allegiance(LastSeenBy(Myself),GOODBUTRED)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 
@@ -527,6 +529,7 @@
   Specifics(LastSeenBy(Myself),65)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -535,6 +538,7 @@
   RESPONSE #80
     Continue()
   RESPONSE #20
+    NoAction()
 END
 
 IF
@@ -651,6 +655,7 @@
   RESPONSE #25
     Continue()
   RESPONSE #75
+    NoAction()
 END
 
 IF
