--- BP\ASCEND\BAF\finaluf.BAF	Mon Mar 27 21:24:52 2006
+++ C:\BWP Patchstudio\patched files\BP\ASCEND\BAF\finaluf.BAF	Sun Dec 20 19:42:27 2015
@@ -102,6 +102,7 @@
     !Detect([GOODCUTOFF])
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -180,6 +181,7 @@
     StateCheck(LastSeenBy(Myself),STATE_DEAD)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -188,6 +190,7 @@
   RESPONSE #80
     Continue()
   RESPONSE #20
+    NoAction()
 END
 
 IF
@@ -298,6 +301,7 @@
   RESPONSE #25
     Continue()
   RESPONSE #75
+    NoAction()
 END
 
 IF
