--- BP\ASCEND\BAF\finnabas.BAF	Mon Mar 27 22:02:46 2006
+++ C:\BWP Patchstudio\patched files\BP\ASCEND\BAF\finnabas.BAF	Sun Dec 20 19:42:27 2015
@@ -262,6 +262,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -274,6 +275,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -286,6 +288,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -298,6 +301,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -310,6 +314,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -318,6 +323,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -326,6 +332,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -341,6 +348,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -363,6 +371,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -385,6 +394,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -407,6 +417,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -429,6 +440,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
