--- BP\ASCEND\BAF\finsucc.BAF	Mon Mar 27 21:24:52 2006
+++ C:\BWP Patchstudio\patched files\BP\ASCEND\BAF\finsucc.BAF	Sun Dec 20 19:42:27 2015
@@ -401,6 +401,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -414,6 +415,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -427,6 +429,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -440,6 +443,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -453,6 +457,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -466,6 +471,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -479,6 +485,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -492,6 +499,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -505,6 +513,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -513,6 +522,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -521,6 +531,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -536,6 +547,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -558,6 +570,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -580,6 +593,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -602,6 +616,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -624,6 +639,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -646,6 +662,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -668,6 +685,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
