--- BP\RITUAL\BAF\ritb1.BAF	Mon Mar 27 20:57:59 2006
+++ C:\BWP Patchstudio\patched files\BP\RITUAL\BAF\ritb1.BAF	Sun Dec 20 19:42:29 2015
@@ -33,6 +33,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -43,6 +44,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -53,6 +55,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -63,6 +66,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -73,6 +77,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -83,6 +88,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -93,6 +99,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -103,6 +110,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
