--- BP\RITUAL\BAF\ritw1.BAF	Mon Mar 27 20:57:59 2006
+++ C:\BWP Patchstudio\patched files\BP\RITUAL\BAF\ritw1.BAF	Sun Dec 20 19:42:29 2015
@@ -31,6 +31,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -41,6 +42,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -51,6 +53,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -61,6 +64,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -71,6 +75,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -81,6 +86,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -91,6 +97,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -101,6 +108,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -111,6 +119,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -121,6 +130,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -131,6 +141,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -141,6 +152,7 @@
   False()
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
