--- BP\BAF\NEW_BAF\CORE\WBARSGT.BAF	Wed Jan 01 20:38:30 2014
+++ C:\BWP Patchstudio\patched files\BP\BAF\NEW_BAF\CORE\WBARSGT.BAF	Sun Dec 20 19:42:29 2015
@@ -190,6 +190,7 @@
     !Detect([GOODCUTOFF])
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -220,6 +221,7 @@
     TurnedBy([GOODCUTOFF.0.0.PALADIN_ALL])
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -636,6 +638,7 @@
     Allegiance(LastSeenBy(Myself),GOODBUTRED)
 THEN
   RESPONSE #100
+    NoAction()
 END
 
 IF
@@ -644,6 +647,7 @@
   RESPONSE #80
     Continue()
   RESPONSE #20
+    NoAction()
 END
 
 IF
@@ -1350,6 +1354,7 @@
   RESPONSE #25
     Continue()
   RESPONSE #75
+    NoAction()
 END
 
 IF
