--- bg1ub\setup-bg1ub.tp2	Sun Mar 01 04:51:31 2015
+++ C:\BWP Patchstudio\patched files\bg1ub\setup-bg1ub.tp2	Fri Jul 03 23:08:19 2015
@@ -275,7 +275,7 @@
 MONSTHIT.EFF
 UNDEDHIT.EFF
 
-VERSION ~v14.0_beta_150228~
+VERSION ~v14.0_beta_150228 BWP Fix~
 README ~bg1ub/BG1UB-Readme.html~
 
 ALWAYS
@@ -1772,8 +1772,8 @@
 /////\\\\\/////\\\\\/////\\\\\/////\\\\\/////\\\\\/////\\\\\
 
 BEGIN @18
-FORBID_FILE ~override/NTSHILOC.CRE~ @90003
-FORBID_FILE ~data/NTotSC1.bif~ @90003
+//FORBID_FILE ~override/NTSHILOC.CRE~ @90003
+//FORBID_FILE ~data/NTotSC1.bif~ @90003
 
 COPY ~bg1ub/bg1ubplaceholder~ ~override/BG1UBShiloChen.UB~
 
