--- bg1ub\setup-bg1ub.tp2	Tue Jul 01 20:54:08 2014
+++ C:\BWP Patchstudio\patched files\bg1ub\setup-bg1ub.tp2	Wed Jun 24 22:38:03 2015
@@ -1771,8 +1771,8 @@
 /////\\\\\/////\\\\\/////\\\\\/////\\\\\/////\\\\\/////\\\\\
 
 BEGIN @18
-FORBID_FILE ~override/NTSHILOC.CRE~ @90003
-FORBID_FILE ~data/NTotSC1.bif~ @90003
+// FORBID_FILE ~override/NTSHILOC.CRE~ @90003
+// FORBID_FILE ~data/NTotSC1.bif~ @90003
 
 COPY ~bg1ub/bg1ubplaceholder~ ~override/BG1UBShiloChen.UB~
 
