--- Setup-NeJ2v602.tp2	Thu Nov 24 11:57:41 2005
+++ C:\BWP Patchstudio\patched files\Setup-NeJ2v602.tp2	Thu Apr 15 08:04:39 2010
@@ -2,7 +2,9 @@
 
 AUTHOR ~Vlad & Larrienne~
 
-BEGIN ~Never Ending Journey v6.02 (cummulative patch)~
+VERSION ~v6.02 BWP Fix~
+
+BEGIN ~Never Ending Journey (cummulative patch)~
 
 REQUIRE_FILE ~override/HROTHJ.DLG~ ~Please install the NeJ2 v4.2b first.~
 REQUIRE_FILE ~override/DARJ.DLG~ ~Please install the NeJ2 v4.2b first.~
