--- Gavin\gavin.tp2	Sun May 31 19:58:32 2015
+++ C:\BWP Patchstudio\patched files\Gavin\gavin.tp2	Wed Jun 24 23:28:09 2015
@@ -15,11 +15,15 @@
 /* enable all error messages; nothing suppressed. comment this out for release version */
 //MODDER
 
-VERSION ~v9~
+VERSION ~v9 BWP Fix~
 
 /* launch the readme file immediately. */
 README ~gavin/readme-gavin.html~
 
+ALLOW_MISSING // allow if BG Graphics night WEDs are not installed
+AR6700N.WED
+FW3300N.WED
+
 ALWAYS
 
 /* These actions are processed for every component: reading of variables */
@@ -247,6 +251,7 @@
 END
 
 COPY_EXISTING ~%Beregost%.wed~ override
+              ~%Beregost%N.wed~ override
   sz = SOURCE_SIZE
   PATCH_IF sz > 0x48 BEGIN
     READ_LONG 0x8 ovr_count //Read initial offsets
