--- kelsey\j#clinn.BAF	Fri Oct 17 06:56:47 2003
+++ C:\BWP Patchstudio\patched files\kelsey\j#clinn.BAF	Thu Apr 01 15:38:10 2010
@@ -52,7 +52,7 @@
 END
 
 IF
-  GlobalTimerNotExpired("J#CastAttack","OCALS")
+  GlobalTimerNotExpired("J#CastAttack","LOCALS")
   Range(NearestEnemyOf(Myself),8)
 THEN
   RESPONSE #100
@@ -61,7 +61,7 @@
 END
 
 IF
-  GlobalTimerNotExpired("J#CastAttack","OCALS")
+  GlobalTimerNotExpired("J#CastAttack","LOCALS")
   !Range(NearestEnemyOf(Myself),8)
   HPPercentGT(Myself,50)
 THEN
@@ -70,7 +70,7 @@
 END
 
 IF
-  GlobalTimerNotExpired("J#CastAttack","OCALS")
+  GlobalTimerNotExpired("J#CastAttack","LOCALS")
   !Range(NearestEnemyOf(Myself),8)
   HPPercentLT(Myself,51)
 THEN
